<table border="1" class="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th></th>
      <th>Title:</th>
      <th>Aptamer Chemistry:</th>
      <th>Image</th>
      <th>Target:</th>
      <th>Antigen/Target Category:</th>
      <th>Affinity (Kd):</th>
      <th>Binding Conditions/Buffer:</th>
      <th>Binding Temp:</th>
      <th>Refolding Program:</th>
      <th>Specificity:</th>
      <th>Molecular Weight:</th>
      <th>GC Content:</th>
      <th>Extinction Coefficient:</th>
      <th>Ki:</th>
      <th>LOD:</th>
      <th>Comments:</th>
      <th>Length:</th>
      <th>Reference:</th>
      <th>Sequence:</th>
      <th>IC50:</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th>0</th>
      <td>PAI-1 01</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/612.jpg" width="120" ></td>
      <td>Plasminogen Activator Inhibitor-1</td>
      <td>Protein</td>
      <td>339 pM (reported value)</td>
      <td>PBSMCT (DPBS with 2.5 mM MgCl2, 1 mM CaCl2, 0.05% TWEEN-20)</td>
      <td>23°C</td>
      <td>None</td>
      <td></td>
      <td>18766.23 g/mole</td>
      <td>51.67%</td>
      <td>579300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>60</td>
      <td>Wang, J., Gong, Q., Maheshwari, N., Eisenstein, M., Luz Arcila, M., Kosik, K., & Soh, T. Particle Display: A Quantitative Screening Method for Generating High-Affinity Aptamers. Angew. Chem. Int. Ed. 2014, 53, 4796ƒ??4801</td>
      <td>dCp,dAp,dTp,dTp,dGp,dAp,dGp,dAp,dTp,dAp,dGp,dCp,dTp,dAp,dGp,dTp,dTp,dGp,dTp,dAp,dGp,dCp,dTp,dGp,dCp,dGp,dTp,dCp,dAp,dTp,dAp,dGp,dGp,dCp,dTp,dGp,dGp,dGp,dTp,dTp,dGp,dGp,dGp,dTp,dCp,dTp,dAp,dGp,dTp,dGp,dGp,dTp,dTp,dGp,dGp,dGp,dTp,dGp,dTp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>1</th>
      <td>Fibrinogen (85A) (T-Modified)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/634.jpg" width="120" ></td>
      <td>Fibrinogen</td>
      <td>Protein</td>
      <td>6.2 µM (reported value)</td>
      <td>300 mM NaCl, 5 mM MgCl2, 20 mM Tris-HCl at pH 7.6</td>
      <td>23°C</td>
      <td>\n                                N/A \n</td>
      <td>60-Fold increase in ability to bind Glycosylated Fibrinogen vs. Non-Glycosylated.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>All Thymines are modified in the structure to incorporate Boronic Acid.</td>
      <td>90</td>
      <td>Li M., Huanh Z., Du L., Altier C., Fang H., Wang B. 2008. Selecting Aptamers for a Glycoprotein through the Incorporation of the Boronic Acid Moiety. Journal of American Chemical SOciety 138(38): 12636-12638.</td>
      <td>dCp,dCp,dTp,dTp,dCp,dGp,dTp,dTp,dGp,dTp,dCp,dTp,dGp,dCp,dCp,dTp,dTp,dCp,dGp,dTp,dAp,dGp,dCp,dGp,dGp,dAp,dTp,dCp,dGp,dAp,dAp,dTp,dTp,dAp,dCp,dGp,dCp,dGp,dTp,dTp,dAp,dAp,dCp,dGp,dGp,dCp,dAp,dAp,dCp,dCp,dGp,dAp,dTp,dAp,dAp,dCp,dGp,dGp,dGp,dAp,dCp,dCp,dGp,dAp,dTp,dTp,dGp,dCp,dAp,dCp,dAp,dCp,dCp,dCp,dTp,dTp,dCp,dAp,dGp,dAp,dAp,dTp,dTp,dCp,dGp,dCp,dAp,dCp,dCp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>2</th>
      <td>Crude Extracellular Mixture of E. coli - trans</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/596.png" width="120" ></td>
      <td>Crude Extracellular Matrix of E. coli</td>
      <td>Protein</td>
      <td>N/A nM (reported value)</td>
      <td>Selection Buffer (1 mM HEPES (pH 7.4), 150 mM NaCl, 15 mM MgCl2, and 0.01% Tween 20), 60 min @ r.t.</td>
      <td>23°C</td>
      <td>\n                                N/A \n</td>
      <td></td>
      <td>21729.14 g/mole</td>
      <td>51.43%</td>
      <td>667800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>This is a trans acting DNAzyme that cleaves a detached substrate in the presence of the CEM of E. coli.</td>
      <td>70</td>
      <td>Aguirre et al. "A Sensitive DNA Enzyme-Based Fluorescent Assay for Bacterial Detection" Biomolecules, 3 (2013): 563-577.</td>
      <td>dGp,dAp,dTp,dGp,dTp,dGp,dCp,dGp,dTp,dTp,dGp,dTp,dCp,dGp,dAp,dGp,dAp,dCp,dCp,dTp,dGp,dCp,dGp,dAp,dCp,dCp,dGp,dGp,dAp,dAp,dCp,dAp,dCp,dTp,dAp,dCp,dAp,dCp,dTp,dGp,dTp,dGp,dTp,dGp,dGp,dGp,dGp,dAp,dTp,dGp,dGp,dAp,dTp,dTp,dTp,dCp,dTp,dTp,dTp,dAp,dCp,dAp,dGp,dTp,dTp,dGp,dTp,dGp,dTp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>3</th>
      <td>Ni2+ (AR1)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/206.png" width="120" ></td>
      <td>Ni2+</td>
      <td>Small Organic</td>
      <td>~100 µM (reported value)</td>
      <td>None</td>
      <td>23°C</td>
      <td>None</td>
      <td></td>
      <td>31742.3 g/mole</td>
      <td>54.08%</td>
      <td>985700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>98</td>
      <td>Seetharaman et al. "Immobolized RNA switches for the analysis of complex chemical and biological mixtures." Nature Biotechnology, 19(2001): 336-341.</td>
      <td>rGp,rGp,rAp,rUp,rAp,rAp,rUp,rAp,rGp,rCp,rCp,rGp,rUp,rAp,rGp,rGp,rUp,rUp,rGp,rCp,rGp,rAp,rAp,rAp,rGp,rCp,rGp,rAp,rCp,rCp,rCp,rUp,rGp,rAp,rUp,rGp,rAp,rGp,rAp,rAp,rAp,rGp,rCp,rCp,rAp,rAp,rAp,rGp,rCp,rCp,rGp,rUp,rAp,rGp,rCp,rGp,rCp,rAp,rGp,rAp,rUp,rGp,rAp,rUp,rCp,rUp,rCp,rGp,rCp,rCp,rAp,rUp,rCp,rAp,rGp,rUp,rAp,rCp,rCp,rGp,rAp,rAp,rAp,rCp,rGp,rGp,rUp,rAp,rGp,rCp,rGp,rAp,rGp,rAp,rGp,rCp,rUp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>4</th>
      <td>Hemoglobin Aptamer</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/616.jpg" width="120" ></td>
      <td>Hemoglobin</td>
      <td>Protein</td>
      <td>7.6 nM (reported value)</td>
      <td>Aptamer binding took place in human blood sample produced by centrifuging 100 µL of whole human blood, collecting the pellet of red blood cells, resuspending the pellet in 50 µL of deionized water, and straining the resulting suspension.</td>
      <td>°C</td>
      <td>None</td>
      <td></td>
      <td>22553.68 g/mole</td>
      <td>55.56%</td>
      <td>715400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>72</td>
      <td>J Li et al. On chip, aptamer-based sandwich assay for detection of glycated hemoglobins via magnetic beads. Biosensors and Bioelectronics 79 (2016): 887-893</td>
      <td>dGp,dGp,dCp,dAp,dGp,dGp,dAp,dAp,dGp,dAp,dCp,dAp,dAp,dAp,dCp,dAp,dCp,dCp,dAp,dGp,dGp,dTp,dGp,dAp,dGp,dGp,dGp,dAp,dGp,dAp,dCp,dGp,dAp,dCp,dGp,dCp,dGp,dAp,dGp,dTp,dGp,dTp,dTp,dAp,dGp,dAp,dTp,dGp,dGp,dTp,dAp,dGp,dCp,dTp,dGp,dTp,dTp,dGp,dGp,dTp,dCp,dTp,dGp,dTp,dGp,dGp,dTp,dGp,dCp,dTp,dGp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>5</th>
      <td>ATP C20</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/572.jpg" width="120" ></td>
      <td>ATP</td>
      <td>Small Organic</td>
      <td>39 µM (reported value)</td>
      <td>(50 mM Trisƒ??HCl, pH 7.5, 150 mM NaCl, 5 mM MgCl2) + 1 mg/ml tRNA and 0.1 mg/ml BSA</td>
      <td>24°C</td>
      <td>\n                                80C for 5 min, cooled to 24C \n</td>
      <td></td>
      <td>25511.67 g/mole</td>
      <td>53.33%</td>
      <td>760400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>349</td>
      <td>Weill L., Louis D., and Sargueil B., 2004.Selection and Evolution of NTP-Specific Aptamers. Nucleic Acids Research, 2004 Vol. 32 No. 17, 5045-5058</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>6</th>
      <td>S6-1b</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7193.png" width="120" ></td>
      <td>Membrane protein of glioma cell line SHG44</td>
      <td>Protein</td>
      <td>3.1-44 nM (reported value)</td>
      <td>4.5 g/L glucose, 0.01 M PBS, 5 mM MgCl2, 0.1 mg/mL yeast tRNA, and 1 mg/mL BSA.</td>
      <td>4°C</td>
      <td>None</td>
      <td></td>
      <td>16291.6 g/mole</td>
      <td>47.17%</td>
      <td>513000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>53</td>
      <td>Lin, N.; Wu, L.; Xu, X.; Wu, Q.; Wang, Y.; Shen, H.; Song, Y.; Wang, H., Zhu, Z.; Kang, D.; Yang, C. Aptamer Generated by Cell-SELEX for Specific Targeting of Human Glioma Cells. ACS Appl. Mater. Interfaces. 2021 Mar 3; 13(8): 9306-9315. DOI: 10.1021/acsami.0c11878.</td>
      <td>dCp,dAp,dCp,dAp,dGp,dGp,dTp,dTp,dCp,dCp,dAp,dGp,dGp,dTp,dAp,dAp,dTp,dAp,dCp,dCp,dTp,dAp,dAp,dGp,dGp,dGp,dTp,dAp,dTp,dGp,dCp,dTp,dCp,dTp,dCp,dGp,dCp,dCp,dTp,dAp,dTp,dTp,dAp,dTp,dAp,dTp,dGp,dGp,dAp,dGp,dCp,dAp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>7</th>
      <td>Ochratoxin A (H12)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/392.png" width="120" ></td>
      <td>Ochratoxin A (OTA)</td>
      <td>Small Organic</td>
      <td>96 nM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td>This aptamer showed minimal cross-reactivity with ochratoxin B and phenylalanine.</td>
      <td>20569.34 g/mole</td>
      <td>63.64%</td>
      <td>649700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>The limit of detection for the enzyme-linked assay was 1 nM with a linear working range from 1-80 nM.</td>
      <td>67</td>
      <td>None</td>
      <td>dGp,dGp,dGp,dAp,dGp,dGp,dAp,dCp,dGp,dAp,dAp,dGp,dCp,dGp,dGp,dAp,dAp,dCp,dCp,dGp,dGp,dGp,dTp,dGp,dTp,dGp,dGp,dGp,dTp,dGp,dCp,dCp,dTp,dTp,dGp,dAp,dTp,dCp,dCp,dAp,dGp,dGp,dGp,dAp,dGp,dTp,dCp,dTp,dCp,dAp,dGp,dAp,dAp,dGp,dAp,dCp,dAp,dCp,dGp,dCp,dCp,dCp,dGp,dAp,dCp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>8</th>
      <td>Ampicillin (AMP18)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/491.png" width="120" ></td>
      <td>Ampicillin</td>
      <td>Small Organic</td>
      <td>9.8 nM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>\n                                Heated at 90°C for 3 min and then cooled at 4°C for 1 hour. \n</td>
      <td>Only AMP17 was evaluated for specificity.</td>
      <td>5895.91 g/mole</td>
      <td>42.11%</td>
      <td>180800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>19</td>
      <td>None</td>
      <td>dTp,dTp,dTp,dAp,dGp,dTp,dTp,dGp,dGp,dGp,dGp,dTp,dTp,dCp,dAp,dGp,dTp,dTp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>9</th>
      <td>Anti-EGFR (E03)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/357.png" width="120" ></td>
      <td>mEGFR and hEGFR</td>
      <td>Protein</td>
      <td>29 nM (reported value)</td>
      <td>None</td>
      <td>25°C</td>
      <td>None</td>
      <td></td>
      <td>29988.85 g/mole</td>
      <td>51.61%</td>
      <td>921300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>96</td>
      <td>N. Li et al. "Inhibition of Cell Proliferation by an Anti-EGFR Aptamer." PLoS One 6(2011): e20299.</td>
      <td>rGp,rGp,fCp,rGp,fCp,fUp,fCp,fCp,rGp,rAp,fCp,fCp,fUp,fUp,rAp,rGp,fUp,fCp,fUp,fCp,fUp,rGp,fUp,rGp,fCp,fUp,rAp,rGp,fUp,rAp,fUp,rAp,fUp,fCp,rGp,fCp,rAp,fCp,rGp,rGp,rAp,fUp,fUp,fUp,rAp,rAp,fUp,fCp,rGp,fCp,fCp,rGp,fUp,rAp,rGp,rAp,rAp,rAp,rAp,rGp,fCp,rAp,fUp,rGp,fUp,fCp,rAp,rAp,rAp,rGp,fCp,fCp,rGp,rGp,rAp,rAp,fCp,fCp,rGp,fUp,rGp,fUp,rAp,rGp,fCp,rAp,fCp,rAp,rGp,fCp,rAp,rGp,rAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>10</th>
      <td>C-Reactive Protein (CRP)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/353.png" width="120" ></td>
      <td>C-Reactive Protein (CRP)</td>
      <td>Protein</td>
      <td>3.51 nM (reported value)</td>
      <td>None</td>
      <td>25°C</td>
      <td>\n                                Not mentioned in the reference. \n</td>
      <td></td>
      <td>22571.71 g/mole</td>
      <td>51.39%</td>
      <td>704600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>80</td>
      <td>None</td>
      <td>dGp,dGp,dCp,dAp,dGp,dGp,dAp,dAp,dGp,dAp,dCp,dAp,dAp,dAp,dCp,dAp,dCp,dGp,dAp,dTp,dGp,dGp,dGp,dGp,dGp,dGp,dGp,dTp,dAp,dTp,dGp,dAp,dTp,dTp,dTp,dGp,dAp,dTp,dGp,dTp,dGp,dGp,dTp,dTp,dGp,dTp,dTp,dGp,dCp,dAp,dTp,dGp,dAp,dTp,dCp,dGp,dTp,dGp,dGp,dTp,dCp,dTp,dGp,dTp,dGp,dGp,dTp,dGp,dCp,dTp,dGp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>11</th>
      <td>SP3</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7169.jpg" width="120" ></td>
      <td>Sepiapterin reductase deficiency (SR)</td>
      <td>Other</td>
      <td>37.3 nM (reported value)</td>
      <td>50 mM Tris, pH 7.5, 150 mM NaCl and 2 mM MgCl2</td>
      <td>°C</td>
      <td>None</td>
      <td></td>
      <td>18637.1 g/mole</td>
      <td>47.46%</td>
      <td>623400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>59</td>
      <td>Eissa S, Alkhaldi S, Chinnappan R, Siddiqua A, Abduljabbar M, Abdel Rahman AM, Dasouki M, Zourob M. Selection, characterization, and electrochemical biosensing application of DNA aptamers for sepiapterin. Talanta. 2020 Aug 15;216:120951. doi: 10.1016/j.talanta.2020.120951. Epub 2020 Mar 20. PMID: 32456943.</td>
      <td>dAp,dAp,dGp,dGp,dAp,dTp,dGp,dAp,dCp,dTp,dGp,dAp,dCp,dGp,dAp,dCp,dGp,dAp,dTp,dGp,dGp,dGp,dAp,dAp,dTp,dGp,dGp,dAp,dGp,dAp,dGp,dGp,dAp,dAp,dGp,dGp,dGp,dGp,dAp,dTp,dAp,dTp,dAp,dGp,dGp,dTp,dAp,dTp,dGp,dTp,dTp,dGp,dGp,dAp,dTp,dGp,dTp,dAp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>12</th>
      <td>2mHNE-5</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7087.png" width="120" ></td>
      <td>HNE</td>
      <td>Protein</td>
      <td>480 nM (reported value)</td>
      <td>Binding BUffer (20mM HEPES, pH 7.5, 150mM NaCL, 6mM KCL, 2mM MgCl2)</td>
      <td>RT°C</td>
      <td>None</td>
      <td></td>
      <td>20649.39 g/mole</td>
      <td>64.52%</td>
      <td>672360 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>each dNTP modified with 2'-OMe</td>
      <td>93</td>
      <td>Liu, Zhixia, et al. "Evolved polymerases facilitate selection of fully 2'-OMe-modified aptamers" Royal Society of Chemistry, 8 (2017): 8179-8182</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>13</th>
      <td>H5 Avian Influenza Virus (HAS15-5)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/226.png" width="120" ></td>
      <td>Hemagglutinin of the Subtype H5 Avian Influenza Virus</td>
      <td>Protein</td>
      <td>N/A nM (reported value)</td>
      <td>Wash Buffer (PBS Buffer (pH 7.4), 140 mM NaCl, and 2.7 mM KCl)</td>
      <td>RT°C</td>
      <td>None</td>
      <td></td>
      <td>29103.68 g/mole</td>
      <td>46.67%</td>
      <td>916900 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>This aptamer binds to the hemagglutinin receptor-binding domain and suppresses RBC agglutination.</td>
      <td>90</td>
      <td>Park et al. "Selection of an Antiviral RNA Aptamer Against Hemagglutinin of the Subtype H5 Avian Influenza Virus." Nucleic Acid Therapeutics, 21(2011): 395-402.</td>
      <td>rGp,rGp,rGp,rUp,rUp,rCp,rAp,rCp,rUp,rGp,rCp,rAp,rGp,rAp,rCp,rUp,rUp,rGp,rAp,rCp,rGp,rAp,rAp,rGp,rCp,rUp,rUp,rAp,rCp,rAp,rAp,rAp,rCp,rAp,rAp,rGp,rAp,rGp,rCp,rAp,rAp,rAp,rAp,rAp,rGp,rGp,rGp,rAp,rGp,rUp,rUp,rGp,rAp,rCp,rGp,rUp,rAp,rGp,rAp,rCp,rUp,rGp,rUp,rGp,rCp,rGp,rGp,rAp,rAp,rUp,rGp,rGp,rAp,rUp,rCp,rCp,rAp,rCp,rAp,rUp,rCp,rUp,rAp,rCp,rGp,rAp,rAp,rUp,rUp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>14</th>
      <td>Integrin ??3 subunit (??3-1)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/321.png" width="120" ></td>
      <td>Integrin ??3 subunit</td>
      <td>Protein</td>
      <td>6.5 nM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td>No binding was observed to integrin aV??6 at 250 nM, whereas the aptamer recognized integrin aV??3 and aIIb??3 at 100 pM.</td>
      <td>26160.56 g/mole</td>
      <td>59.26%</td>
      <td>784800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>See also aptamer aV-1 which binds to the aV subunit.</td>
      <td>81</td>
      <td>Gong, Q. et al,(2012). Selection strategy to generate aptamer pairs that bind to distinct sites on protein targets. Analytical Chemistry, 2012(84), 5365-5371.</td>
      <td>rGp,rGp,rGp,rAp,rGp,rGp,rAp,fCp,rGp,rAp,fUp,rGp,fCp,rGp,rGp,fCp,fCp,fCp,rAp,rGp,rAp,fUp,fUp,rAp,fCp,fUp,rGp,fUp,rGp,rGp,rAp,rGp,fUp,rGp,rGp,fUp,fUp,rGp,fUp,fCp,fUp,rGp,fCp,rGp,rAp,rAp,fUp,fCp,fCp,fUp,fUp,fCp,rGp,fUp,fCp,fCp,rAp,fCp,fCp,fCp,rAp,rAp,fUp,rAp,rGp,fCp,rAp,rGp,rAp,fCp,rGp,rAp,fCp,fUp,fCp,rGp,fCp,fCp,fCp,rGp,rAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>15</th>
      <td>Fibrinogen (85A/Ap90)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/215.png" width="120" ></td>
      <td>Fibrinogen</td>
      <td>Protein</td>
      <td>63.8 nM (reported value)</td>
      <td>None</td>
      <td>25°C</td>
      <td>None</td>
      <td>Kd = 390 nM against Deglycosylated Fibrinogen.  Kd = 70.7 nM against Periodated Fibrinogen.  Kd = 165 nM against Sialidase-treated Fibrinogen.</td>
      <td>27523.83 g/mole</td>
      <td>55.56%</td>
      <td>840700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Kd = 6.4 nM if thymidines are boronic acid modified.</td>
      <td>90</td>
      <td>None</td>
      <td>dCp,dCp,dTp,dTp,dCp,dGp,dTp,dTp,dGp,dTp,dCp,dTp,dGp,dCp,dCp,dTp,dTp,dCp,dGp,dTp,dAp,dGp,dGp,dAp,dCp,dCp,dGp,dCp,dAp,dGp,dAp,dCp,dAp,dTp,dCp,dGp,dAp,dCp,dGp,dCp,dAp,dGp,dGp,dGp,dAp,dAp,dAp,dTp,dTp,dCp,dCp,dGp,dCp,dAp,dAp,dGp,dTp,dCp,dCp,dAp,dGp,dCp,dCp,dAp,dAp,dAp,dTp,dGp,dCp,dCp,dAp,dCp,dCp,dCp,dTp,dTp,dCp,dAp,dGp,dAp,dAp,dTp,dTp,dCp,dGp,dCp,dAp,dCp,dCp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>16</th>
      <td>ErbB2 (SE15-8)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/202.png" width="120" ></td>
      <td>ErbB2 in Breast Cancer Cells</td>
      <td>Protein</td>
      <td>3.49 nM (reported value)</td>
      <td>None</td>
      <td>N/A°C</td>
      <td>None</td>
      <td>MDA-MB-453 breast cancer cell line, T47D breast cancer cells, and KPL-4 breast cancer cells (allErbB2 positive)showed staining, while ErbB2 negative cell line MCF-7 did not.</td>
      <td>11239.85 g/mole</td>
      <td>73.53%</td>
      <td>341100 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>34</td>
      <td>None</td>
      <td>rAp,rGp,fCp,fCp,rGp,fCp,rGp,rAp,rGp,rGp,rGp,rGp,rAp,rGp,rGp,rGp,rAp,fUp,rAp,rGp,rGp,rGp,fUp,rAp,rGp,rGp,rGp,fCp,rGp,fCp,rGp,rGp,fCp,fUp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>17</th>
      <td>Thrombin (15mer-29-mer dimer with 20T linker)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/447.png" width="120" ></td>
      <td>Thrombin</td>
      <td>Protein</td>
      <td>0.35 nM (reported value)</td>
      <td>Binding Buffer: 50 mM Tris-HCl, 300 mM NaCl, 5 mM KCl, pH 8.0.</td>
      <td>25°C</td>
      <td>None</td>
      <td></td>
      <td>19958.03 g/mole</td>
      <td>42.19%</td>
      <td>588600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>This aptamer is a dimerized version of the 15-mer and 29-mer thrombin aptamers connected by a 20-mer thymidine linker. This aptamer showed improved affinity and inhibitory effects to the monomer components.</td>
      <td>70</td>
      <td>None</td>
      <td>dGp,dGp,dTp,dTp,dGp,dGp,dTp,dGp,dTp,dGp,dGp,dTp,dTp,dGp,dGp,dTp,dTp,dTp,dTp,dTp,dTp,dTp,dTp,dTp,dTp,dTp,dTp,dTp,dTp,dTp,dTp,dTp,dTp,dTp,dTp,dAp,dGp,dTp,dCp,dCp,dGp,dTp,dGp,dGp,dTp,dAp,dGp,dGp,dGp,dCp,dAp,dGp,dGp,dTp,dTp,dGp,dGp,dGp,dGp,dTp,dGp,dAp,dCp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>18</th>
      <td>ATP (ATP-40-1)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/97.png" width="120" ></td>
      <td>ATP</td>
      <td>Small Organic</td>
      <td>700 nM (reported value)</td>
      <td>Binding Buffer (300mM NaCl, 20 mM Tris (pH 7.6), 5mM MgCl2)</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>12928.86 g/mole</td>
      <td>60.00%</td>
      <td>388400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>40</td>
      <td>sassanfar and Szostak. "An RNA motif that binds ATP."</td>
      <td>rGp,rGp,rGp,rUp,rUp,rGp,rGp,rGp,rAp,rAp,rGp,rAp,rAp,rAp,rCp,rUp,rGp,rUp,rGp,rGp,rCp,rAp,rCp,rUp,rUp,rCp,rGp,rGp,rUp,rGp,rCp,rCp,rAp,rGp,rCp,rAp,rAp,rCp,rCp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>19</th>
      <td>S22</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7214.jpg" width="120" ></td>
      <td>MUC-1</td>
      <td>Protein</td>
      <td>0.010 - 1000 nM (reported value)</td>
      <td>Bound to a gold electrode in order to transmit aptasensor data.</td>
      <td>25°C</td>
      <td>\n                                mFold \n</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>This aptamer is intended to act as an Electrochemical Aptasenor. In order to function properly, the aptamer needs to be connected to a magnetic bead on the 5' terminus.</td>
      <td>75</td>
      <td>https://doi.org/10.1016/j.bios.2015.04.001</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>20</th>
      <td>TTF1 (A)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/512" width="120" ></td>
      <td>Thyroid Transcription Factor 1</td>
      <td>Protein</td>
      <td>3.36 nM (reported value)</td>
      <td>None</td>
      <td>4°C</td>
      <td>None</td>
      <td></td>
      <td>24171.74 g/mole</td>
      <td>50.00%</td>
      <td>753300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>78</td>
      <td>None</td>
      <td>dGp,dGp,dTp,dAp,dTp,dTp,dGp,dAp,dGp,dGp,dGp,dTp,dCp,dGp,dCp,dAp,dTp,dCp,dTp,dCp,dAp,dAp,dAp,dAp,dGp,dGp,dGp,dGp,dTp,dGp,dAp,dTp,dTp,dGp,dCp,dTp,dTp,dGp,dCp,dAp,dCp,dAp,dAp,dTp,dGp,dAp,dCp,dAp,dGp,dGp,dGp,dTp,dAp,dGp,dGp,dAp,dCp,dAp,dGp,dAp,dTp,dGp,dGp,dCp,dTp,dCp,dTp,dAp,dAp,dCp,dTp,dCp,dTp,dCp,dCp,dTp,dCp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>21</th>
      <td>Anterior Gradient Homolog 2 (C14B0)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/478.png" width="120" ></td>
      <td>Anterior Gradient Homolog 2 (AGR2)</td>
      <td>Protein</td>
      <td>8.5 nM (reported value)</td>
      <td>None</td>
      <td>N/A°C</td>
      <td>\n                                Denatured at 95° C for 5 minutes, subsequently cooled on ice for 10 minutes. Then incubated at RT for 10 minutes. \n</td>
      <td></td>
      <td>33748.78 g/mole</td>
      <td>63.89%</td>
      <td>995300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Aptamer is expected to form a G-quartet.</td>
      <td>108</td>
      <td>None</td>
      <td>dTp,dCp,dTp,dCp,dGp,dGp,dAp,dCp,dGp,dCp,dGp,dTp,dGp,dTp,dGp,dGp,dTp,dCp,dGp,dGp,dCp,dGp,dGp,dGp,dTp,dGp,dGp,dGp,dAp,dGp,dTp,dTp,dGp,dTp,dGp,dGp,dGp,dGp,dGp,dGp,dGp,dGp,dGp,dTp,dGp,dGp,dGp,dAp,dGp,dGp,dGp,dTp,dTp,dCp,dTp,dTp,dTp,dGp,dTp,dTp,dTp,dGp,dAp,dTp,dCp,dTp,dTp,dTp,dCp,dTp,dCp,dGp,dCp,dTp,dGp,dCp,dCp,dTp,dGp,dGp,dCp,dCp,dCp,dTp,dAp,dGp,dAp,dGp,dTp,dGp,dTp,dCp,dGp,dCp,dTp,dGp,dCp,dCp,dTp,dGp,dGp,dCp,dAp,dGp,dAp,dGp,dTp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>22</th>
      <td>RhD antigen‑specificity 2</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7163.jpg" width="120" ></td>
      <td>RhD+ red blood cells</td>
      <td>Cells</td>
      <td>737.7±161.8 nM (reported value)</td>
      <td>10 mM Tris-HCl, 50 mM KCl, pH 9.0), 1.5 mM MgCl2, 400 µM each dNTP, 2.5 U Taq DNA polymerase (Promega Corporation), and 50 pmol each biotinylated primer</td>
      <td>25°C</td>
      <td>None</td>
      <td></td>
      <td>12430.1 g/mole</td>
      <td>62.50%</td>
      <td>399400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>40</td>
      <td>\nZhang Y, Xu H, Wang X, Wang L, Liu R, Li L, Zhou H. Single‑strained DNA aptamers mask RhD antigenic epitopes on human RhD+ red blood cells to escape alloanti‑RhD immunological recognition. Mol Med Rep. 2020 Apr;21(4):1841-1848. doi: 10.3892/mmr.2020.10985. Epub 2020 Feb 12. PMID: 32319623; PMCID: PMC7057830.</td>
      <td>dGp,dGp,dGp,dTp,dAp,dGp,dCp,dAp,dGp,dCp,dCp,dCp,dCp,dGp,dCp,dGp,dGp,dAp,dGp,dGp,dGp,dTp,dCp,dGp,dGp,dCp,dTp,dAp,dTp,dAp,dAp,dGp,dAp,dAp,dCp,dCp,dAp,dAp,dGp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>23</th>
      <td>M16</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7223.png" width="120" ></td>
      <td>Pb2</td>
      <td>Other</td>
      <td>nM (reported value)</td>
      <td>15 μL of NaCl solution (0.5 mM), 30 μL of aptamer (1 μM)</td>
      <td>23°C</td>
      <td>\n                                mFold \n</td>
      <td>Highly specific, mutant versions (M4, M16, and M4-16) of this aptamer are even more so.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>This aptamer is a Pb^2 binding aptamer that is a mutant variation of T30695. The other mutant variants are designated as M4, and M4-16. Most of its binding ability was determined in silico. The calculated binding free energy of this aptamer to Pb^2 is −1417.17−kj /mol.\n</td>
      <td>16</td>
      <td>https://doi.org/10.1016/j.molliq.2019.111159.\n</td>
      <td>dGp,dGp,dGp,dTp,dGp,dGp,dGp,dTp,dGp,dGp,dGp,dTp,dGp,dGp,dGp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>24</th>
      <td>ERK1/ ERK2 (Family II - Truncated)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/481.png" width="120" ></td>
      <td>Extracellular Regulated Kinase 1 and 2 (ERK 1 and ERK2)</td>
      <td>Protein</td>
      <td>N/A nM (reported value)</td>
      <td>None</td>
      <td>30°C</td>
      <td>\n                                Denatured in water at 95°C for 3 min and slowly cooling in binding buffer from 55°C to 30°C. \n</td>
      <td>Binds to ERK 1/2 but not JNK or P38.</td>
      <td>20370.42 g/mole</td>
      <td>42.86%</td>
      <td>651800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Truncated version of ERK 1/2 Family II apt</td>
      <td>69</td>
      <td>Seiwert, S., et al. "RNA aptamers as pathway-specific MAP kinase inhibitors." Chemistry & Biology, 7(2000): 833-834.</td>
      <td>rGp,rGp,rAp,rAp,rAp,rGp,rAp,rCp,rGp,rCp,rUp,rAp,rGp,rCp,rGp,rAp,rAp,rUp,rUp,rGp,rGp,rUp,rUp,rCp,rCp,rUp,rCp,rGp,rAp,rAp,rAp,rGp,rGp,rGp,rGp,rAp,rAp,rAp,rGp,rCp,rGp,rUp,rUp,rAp,rUp,rUp,rAp,rAp,rGp,rAp,rAp,rAp,rCp,rCp,rAp,rAp,rAp,rAp,rUp,rUp,rUp,rCp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>25</th>
      <td>RIG-I (CL9)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/216.png" width="120" ></td>
      <td>RIG-I</td>
      <td>Protein</td>
      <td>37.2 nM (reported value)</td>
      <td>1X PBS with 1 mM MgCl2</td>
      <td>4°C</td>
      <td>None</td>
      <td></td>
      <td>29522.39 g/mole</td>
      <td>50.00%</td>
      <td>900600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>2'-F-RNA (C's and U's Only)</td>
      <td>92</td>
      <td>Hwang et al. "5'-Triphosphate-RNA-independent activation of RIG-1 via RNA aptamer with enhanced antiviral activity." Nucleic Acids Research 40(2011): 2724 - 2733</td>
      <td>rGp,rGp,rGp,rAp,rGp,rAp,rGp,fCp,rGp,rGp,rAp,rAp,rGp,fCp,rGp,fUp,rGp,fCp,fUp,rGp,rGp,rGp,fCp,fCp,rAp,fCp,fCp,rAp,fUp,fCp,fCp,rGp,fUp,rAp,rAp,fCp,fUp,rAp,rGp,fCp,fUp,rAp,rAp,fUp,rAp,fCp,fUp,fUp,rGp,fUp,fUp,rAp,fUp,fCp,fUp,fUp,fUp,fUp,fUp,rAp,fUp,fUp,fUp,fUp,fCp,rAp,fUp,rAp,rAp,fCp,fCp,fCp,rAp,rGp,rAp,rGp,rGp,fUp,fCp,rGp,rAp,fUp,rGp,rGp,rAp,fUp,fCp,fCp,fCp,fCp,fCp,fCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>26</th>
      <td>Tenascin-C (GBI-10)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/610" width="120" ></td>
      <td>Tenascin-C (TNC)</td>
      <td>Protein</td>
      <td>150 nM (reported value)</td>
      <td>Binding Buffer (10 mM Tris-HCl (pH 7.5), 150 mM NaCl, 1 mM MgCl2, 1 mM CaCl2, 0.1% (wt/vol) BSA, 0.1 % (vol/vol) Triton X-100, 0.1% (vol/vol) Tween 20)</td>
      <td>4°C</td>
      <td>None</td>
      <td>Selected using the TNC positive cell line U251.</td>
      <td>21127.69 g/mole</td>
      <td>55.07%</td>
      <td>626300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>At least a 10-fold decrease in Kd at 22° and 37°.</td>
      <td>69</td>
      <td>None</td>
      <td>dGp,dGp,dCp,dTp,dGp,dTp,dTp,dGp,dTp,dGp,dAp,dGp,dCp,dCp,dTp,dCp,dCp,dTp,dCp,dCp,dCp,dAp,dGp,dAp,dGp,dGp,dGp,dAp,dAp,dGp,dAp,dCp,dTp,dTp,dTp,dAp,dGp,dGp,dTp,dTp,dCp,dGp,dGp,dTp,dTp,dCp,dAp,dCp,dGp,dTp,dCp,dCp,dCp,dGp,dCp,dTp,dTp,dAp,dTp,dTp,dCp,dTp,dTp,dAp,dCp,dTp,dCp,dCp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>27</th>
      <td>Human Norovirus SMV-25</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/579.jpg" width="120" ></td>
      <td>Human Norovirus</td>
      <td>Other</td>
      <td>232 nM (reported value)</td>
      <td>500 ml of binding buffer [consisting of phosphate buffered saline (pH 7.1) supplemented with 100 mg/L CaCl2, 100 mg/L MgCl2 and 0.05% Tween 20] at Room Temperature</td>
      <td>24°C</td>
      <td>None</td>
      <td>See page 8 of source material. Aptamer binds to several strains of the Human Norovirus</td>
      <td>24864.21 g/mole</td>
      <td>43.21%</td>
      <td>770500 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>245</td>
      <td>Escudero-Abarca BI, Suh SH, Moore MD, Dwivedi HP, Jaykus L-A (2014) Selection, Characterization and Application of Nucleic Acid Aptamers for the Capture and Detection of Human Norovirus Strains. PLoS ONE 9(9): e106805. doi:10.1371/journal.pone.0106805</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>28</th>
      <td>FMN-induced molecular switch</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/394.png" width="120" ></td>
      <td>flavin mononucleotide (FMN)</td>
      <td>Small Organic</td>
      <td>na nM (reported value)</td>
      <td>Reaction buffer [50 mM Tris.HCl (pH 7.5) and 20 mM MgCl2] for 20 hr</td>
      <td>23°C</td>
      <td>None</td>
      <td></td>
      <td>23333.13 g/mole</td>
      <td>58.33%</td>
      <td>710900 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>72</td>
      <td>None</td>
      <td>rGp,rGp,rGp,rCp,rGp,rAp,rCp,rCp,rCp,rUp,rGp,rAp,rUp,rGp,rAp,rGp,rCp,rCp,rUp,rUp,rAp,rGp,rGp,rAp,rUp,rAp,rUp,rGp,rCp,rUp,rUp,rCp,rGp,rGp,rCp,rAp,rGp,rAp,rAp,rGp,rGp,rAp,rCp,rGp,rUp,rCp,rGp,rAp,rAp,rAp,rCp,rGp,rGp,rUp,rGp,rAp,rAp,rAp,rGp,rCp,rCp,rGp,rUp,rAp,rGp,rGp,rUp,rUp,rGp,rCp,rCp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>29</th>
      <td>Apta 5</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7181.png" width="120" ></td>
      <td>Carcinoembryonic antigen (CEA)</td>
      <td>Protein</td>
      <td>37.8 nM (reported value)</td>
      <td>Not given.</td>
      <td>37°C</td>
      <td>None</td>
      <td>Apta 5 displayed high specificity and affinity for T84 cells (human carcinoma cell line derived from a lung metastasis of a colon carcinoma).</td>
      <td>21404.8 g/mole</td>
      <td>65.71%</td>
      <td>621800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>70</td>
      <td>Melo, M.; Correa, C.; Cunha, P.; Goes, A.; Gomes, D.; Andrade, A. DNA aptamers selection for carcinoembryonic antigen (CEA). Bioorg. Med. Chem. Lett. 2020 May 23; 30(15): 127278. DOI: 10.1016/j.bmcl.2020.127278</td>
      <td>dTp,dCp,dGp,dCp,dGp,dCp,dGp,dAp,dGp,dTp,dCp,dGp,dTp,dCp,dTp,dGp,dGp,dGp,dAp,dGp,dCp,dTp,dAp,dCp,dGp,dTp,dTp,dTp,dAp,dGp,dCp,dGp,dAp,dGp,dTp,dCp,dCp,dGp,dAp,dCp,dGp,dCp,dTp,dCp,dGp,dGp,dTp,dGp,dCp,dCp,dTp,dCp,dTp,dTp,dCp,dCp,dCp,dGp,dCp,dAp,dTp,dCp,dGp,dTp,dCp,dCp,dTp,dCp,dCp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>30</th>
      <td>Clone 488</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/560.jpg" width="120" ></td>
      <td>SelB</td>
      <td>Protein</td>
      <td>nM (reported value)</td>
      <td>50 mM potassium phosphate, pH 7.0, 5 mM Mg(OAc)2, 0.1 mM EDTA, 1 mM DTT, 0.5 mM GTP,0.02% Tween 20, 400 U/mL RNAsin</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>20508.24 g/mole</td>
      <td>67.16%</td>
      <td>621000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>210</td>
      <td>Klug, S. J. et al. "In vitro selection of RNA aptamers that bind special elongation factor SelB, a protein with multiple RNA-binding sites, reveals one major interaction domain at the carboxyl terminus." RNA, 1995 5: 1180 - 1190.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>31</th>
      <td>CBSA-5335 SF</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7177.jpg" width="120" ></td>
      <td>Cocaine</td>
      <td>Small Organic</td>
      <td>36 nM (reported value)</td>
      <td>10% saliva in 10 mM Tris, 0.1 mM MgCl2, pH 7.4</td>
      <td>23°C</td>
      <td>\n                                N/A \n</td>
      <td>Demonstrated specificity to cocaine over benzoylecgonine, anhydroecgonine methyl ester, cocaethylene and nicotine</td>
      <td>5793.8 g/mole</td>
      <td>55.56%</td>
      <td>198200 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>The sequence below is one fragment of the cocaine split aptamer that contains two separate fragments, please view "Related Aptamers" for second fragment of the sequence. "X" in sequence stands for C3 Spacer</td>
      <td>19</td>
      <td>Yu, H., Canoura, J., Guntupalli, B., Lou, X., & Xiao, Y. (2017). A cooperative-binding split aptamer assay for rapid, specific and ultra-sensitive fluorescence detection of cocaine in saliva. Chemical Science, 8(1), 131–141. doi:10.1039/c6sc01833e</td>
      <td>dGp,dAp,dGp,dAp,dCp,dAp,dAp,dGp,dGp,dGp,dAp,dCp,dAp,dAp,dGp,dGp,dAp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>32</th>
      <td>HIV-1 Integrase (P5)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/129.png" width="120" ></td>
      <td>HIV-1 Integrase</td>
      <td>Protein</td>
      <td>10 nM (reported value)</td>
      <td>Buffer A (50 mM HEPES, 1 mM DTT, 1 mM APMSF)    Binding Buffer (50 mM Na-HEPES (pH 7.5), 250 mM NaCl, 2 mM DTT, 10 mM MnCl2, 5 mM CHAPS)</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>24933.1 g/mole</td>
      <td>54.55%</td>
      <td>767300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>77</td>
      <td>Allen, P., et al. "Isolation of High-Affinity RNA Ligands to HIV-1 Integrase from a Random Pool." Virology, 209(1995): 327-336.</td>
      <td>rGp,rGp,rGp,rAp,rGp,rCp,rUp,rCp,rAp,rGp,rAp,rAp,rUp,rAp,rAp,rAp,rCp,rGp,rCp,rUp,rCp,rAp,rAp,rCp,rCp,rAp,rGp,rUp,rCp,rUp,rUp,rGp,rUp,rGp,rGp,rCp,rUp,rUp,rUp,rGp,rAp,rAp,rAp,rGp,rAp,rGp,rAp,rGp,rGp,rAp,rGp,rUp,rGp,rUp,rUp,rCp,rGp,rAp,rCp,rAp,rUp,rGp,rAp,rGp,rGp,rCp,rCp,rCp,rGp,rGp,rAp,rUp,rCp,rCp,rGp,rGp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>33</th>
      <td>Ethanolamine (14.3)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/39.png" width="120" ></td>
      <td>Ethanolamine</td>
      <td>Small Organic</td>
      <td>6 nM (reported value)</td>
      <td>Binding Buffer (100 mM NaCl, 20 mM Tris/HCl (pH 7.6), 2 mM MgCl2, 5 mM KCl, 1 mM CaCl2, 0.02% Tween 20).</td>
      <td>21°C</td>
      <td>None</td>
      <td></td>
      <td>29786.42 g/mole</td>
      <td>45.83%</td>
      <td>940500 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>96</td>
      <td>Mann, Doerthe, et al. "In vitro selection of DNA aptamers binding ethanolamine."   Biochemical and Biophysical Research Communication, 338 (2005): 1928-1934.    Tradmarked in patent, "New DNA aptamers specific for compunds that contain the ethylamino group, useful for diagnosis, prevention and treatment of ethanolaminosis, schizophrenia and neurodegeneration (Original document: DE 102005052275 (B4))"</td>
      <td>dAp,dTp,dAp,dCp,dCp,dAp,dGp,dCp,dTp,dTp,dAp,dTp,dTp,dCp,dAp,dAp,dTp,dTp,dTp,dGp,dAp,dGp,dGp,dCp,dGp,dGp,dGp,dTp,dGp,dGp,dGp,dTp,dGp,dGp,dGp,dTp,dTp,dGp,dAp,dAp,dTp,dAp,dTp,dGp,dCp,dTp,dGp,dAp,dTp,dTp,dAp,dCp,dCp,dCp,dCp,dAp,dTp,dCp,dGp,dGp,dAp,dGp,dAp,dAp,dCp,dGp,dTp,dTp,dAp,dAp,dGp,dGp,dCp,dGp,dCp,dTp,dTp,dCp,dAp,dGp,dAp,dTp,dAp,dGp,dTp,dAp,dAp,dGp,dTp,dGp,dCp,dAp,dAp,dTp,dCp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>34</th>
      <td>Gonadotropin-releasing hormone 1 (60-mer S42)</td>
      <td>L-DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/78.png" width="120" ></td>
      <td>Gonadoliberin/Gonadotropin-releasing hormone 1 (GnRH)</td>
      <td>Peptide</td>
      <td>45 nM (reported value)</td>
      <td>None</td>
      <td>25°C</td>
      <td>None</td>
      <td>None</td>
      <td>48667.48 g/mole</td>
      <td>0.00%</td>
      <td>555200 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>60</td>
      <td>Leva, S., et al. "GnRH Binding RNA and DNA Spiegelmers: A Novel Approach toward GnRH Antagonism." Chemistry and Biology, 9 (2002): 351-359.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>35</th>
      <td>CRP-U^gu4</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7121.jpg" width="120" ></td>
      <td>Human C-reactive protein (CRP)</td>
      <td>Protein</td>
      <td>6.2 nM (reported value)</td>
      <td>40 mM HEPES, pH7.5, 125 mM NaCl, 5 mM KCl, 1 mM MgCl2, 0.01% Tween 20</td>
      <td>25°C</td>
      <td>None</td>
      <td></td>
      <td>9203 g/mole</td>
      <td>68.42%</td>
      <td>283200 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>30</td>
      <td>Minagawa H, Kataoka Y, Fujita H, Kuwahara M, Horii K, Shiratori I, Waga I. Modified DNA Aptamers for C-Reactive Protein and Lactate Dehydrogenase-5 with Sub-Nanomolar Affinities. International Journal of Molecular Sciences. 2020; 21(8):2683. https://doi.org/10.3390/ijms21082683</td>
      <td>dCp,dAp,dGp,dCp,dGp,dCp,dCp,dAp,dAp,dAp,dGp,dGp,dCp,dGp,dGp,dAp,dAp,dGp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>36</th>
      <td>Interleukin 11 receptor (IL-11Ra) Human Prostate</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/249.png" width="120" ></td>
      <td>Interleukin 11 receptor (IL-11Ra)</td>
      <td>Tissue</td>
      <td>N/A nM (reported value)</td>
      <td>None</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>948.15 g/mole</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Peptide is similar to interleukin 11 (IL-11) and binds to its receptor (IL-11Rα)</td>
      <td>9</td>
      <td>Arap et al. "Steps toward mapping the human vasculature  by phage display." Nature Medicine, 8(2002): 121-127.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>37</th>
      <td>NC23</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7132.jpg" width="120" ></td>
      <td>N‑Cadherin Protein</td>
      <td>Protein</td>
      <td>174 nM (reported value)</td>
      <td>1X PBS</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>23361.2 g/mole</td>
      <td>35.53%</td>
      <td>740200 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>76</td>
      <td>Yang, L., Gao, T., Li, W., Luo, Y., Ullah, S., Fang, X., … Pei, R. (2020). Ni-Nitrilotriacetic Acid Affinity SELEX Method for Selection of DNA Aptamers Specific to the N-Cadherin Protein. ACS Combinatorial Science. doi:10.1021/acscombsci.0c00165</td>
      <td>dAp,dTp,dAp,dCp,dCp,dAp,dGp,dCp,dTp,dTp,dAp,dTp,dTp,dCp,dAp,dAp,dTp,dTp,dAp,dGp,dAp,dCp,dGp,dAp,dGp,dTp,dTp,dTp,dTp,dAp,dTp,dTp,dCp,dAp,dTp,dTp,dTp,dTp,dCp,dCp,dAp,dTp,dGp,dTp,dCp,dGp,dTp,dAp,dGp,dCp,dTp,dTp,dAp,dCp,dCp,dGp,dTp,dGp,dAp,dGp,dAp,dTp,dAp,dGp,dTp,dAp,dAp,dGp,dTp,dGp,dCp,dAp,dAp,dTp,dCp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>38</th>
      <td>Bacillus anthracis spores (Ba)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/486.png" width="120" ></td>
      <td>Bacillus anthracis spores</td>
      <td>Other</td>
      <td>1.5x10^12 CFU/mL (reported value)</td>
      <td>None</td>
      <td>37°C</td>
      <td>\n                                Stir win MOPS buffer at 60° C for one day. Then, slowly cool to room temperature. \n</td>
      <td></td>
      <td>24831.14 g/mole</td>
      <td>53.75%</td>
      <td>776600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>80</td>
      <td>None</td>
      <td>dAp,dCp,dCp,dCp,dCp,dTp,dGp,dCp,dAp,dTp,dCp,dCp,dTp,dTp,dTp,dGp,dCp,dTp,dGp,dGp,dAp,dGp,dAp,dGp,dGp,dAp,dAp,dTp,dGp,dTp,dAp,dTp,dAp,dAp,dGp,dGp,dAp,dTp,dGp,dTp,dTp,dCp,dCp,dGp,dGp,dGp,dCp,dGp,dTp,dGp,dTp,dGp,dGp,dGp,dTp,dAp,dAp,dGp,dTp,dCp,dAp,dGp,dTp,dCp,dTp,dAp,dGp,dAp,dGp,dGp,dGp,dCp,dCp,dCp,dCp,dAp,dGp,dAp,dAp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>39</th>
      <td>Plasmodium falciparum lactate dehydrogenase (pL2)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/311.png" width="120" ></td>
      <td>Plasmodium falciparum lactate dehydrogenase  (PfLDH)</td>
      <td>Protein</td>
      <td>49.6 nM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td></td>
      <td>11750.66 g/mole</td>
      <td>57.89%</td>
      <td>358500 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>38</td>
      <td>None</td>
      <td>dGp,dAp,dAp,dCp,dTp,dCp,dAp,dTp,dTp,dGp,dGp,dCp,dTp,dGp,dGp,dAp,dGp,dGp,dCp,dGp,dGp,dCp,dAp,dGp,dTp,dAp,dCp,dCp,dGp,dCp,dTp,dTp,dGp,dAp,dGp,dTp,dTp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>40</th>
      <td>Human Immunoglobulin G (IgG) (Apt 8)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/44.jpg" width="120" ></td>
      <td>hlgG-Fc</td>
      <td>Protein</td>
      <td>75 nM (reported value)</td>
      <td>Binding Buffer (145mM NaCl, 5.4 mM KCl, 0.8mM MgCl, 1.8 mM CaCl, and 20 mM Tris-HCl (pH 7.6))    Modifications at the 5' and 3' end have been made by the authors for various testing.</td>
      <td>N/A°C</td>
      <td>None</td>
      <td>None</td>
      <td>7442.54 g/mole</td>
      <td>60.87%</td>
      <td>226600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>23</td>
      <td>Miyakawa, Shin, et al. "Structural and molecular basis for hyperspecificity of RNA aptamer to human immunoglobulin G." RNA Journal 14,(2008): 1154-1163.</td>
      <td>rGp,rGp,rAp,rGp,rGp,fUp,rGp,fCp,fUp,fCp,fCp,rGp,rAp,rAp,rAp,rGp,rGp,rAp,rAp,fCp,fUp,fCp,fCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>41</th>
      <td>Tumor-derived endothelial cells (tRRL)</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/384.png" width="120" ></td>
      <td>Tumor-derived endothelial cells (TDECs)</td>
      <td>Cells</td>
      <td>N/A nM (reported value)</td>
      <td>None</td>
      <td>N/A°C</td>
      <td>None</td>
      <td>None</td>
      <td>1141.4 g/mole</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>10</td>
      <td>None</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>42</th>
      <td>U8</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/522.jpg" width="120" ></td>
      <td>U87-EGFRvIII</td>
      <td>Cells</td>
      <td>4.35 nM (reported value)</td>
      <td>0.45 g of glucose, 10 mg yeast tRNA, 5 mg salmon sperm DNA, 0.1 g BSA and 0.5 ml of 1 M MgCl2 in 100 ml of 1X phosphate-buffered saline [PBS]</td>
      <td>°C</td>
      <td>None</td>
      <td></td>
      <td>23560.41 g/mole</td>
      <td>39.47%</td>
      <td>734600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>228</td>
      <td>"Cell-SELEX Aptamer for Highly Specific Radionuclide Molecular Imaging of Glioblastoma In Vivo." PLOS One 2014, 9:3.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>43</th>
      <td>GE54</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/554.png" width="120" ></td>
      <td>Rabies Virus glycoprotein(RABV)</td>
      <td>Protein</td>
      <td>307 nM (reported value)</td>
      <td>4.5g/L glucose, 5mM MgCl2, Dulbecco's PBS, yeast tRNA, 1.0g/L bovine serum albumin</td>
      <td>37°C</td>
      <td>\n                                95C for 5 minutes, followed by 4C for 10 minutes. \n</td>
      <td></td>
      <td>13247.7 g/mole</td>
      <td>34.88%</td>
      <td>398000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>129</td>
      <td>Hong-Ru Liang, Gui-Qiu Hu, Xiang-Hong Xue, Lu Li, Xue-Xing Zheng,Yu-Wei Gao, Song-Tao Yang, Xian-Zhu Xia, "Selection of an aptamer against rabies virus: A new class of molecules with antiviral activity". Virus Research 184 (2014) 7-13.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>44</th>
      <td>Cocaine (MNS-4.1)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/418.png" width="120" ></td>
      <td>Cocaine</td>
      <td>Small Organic</td>
      <td>0.4 µM (reported value)</td>
      <td>Phosphate buffer: 20 mM, pH7, 200 mM NaCl.</td>
      <td>RT°C</td>
      <td>None</td>
      <td>Did not bind to metabolites benzoyl ecgonine or ecgonine methyl ester.</td>
      <td>11833.78 g/mole</td>
      <td>47.37%</td>
      <td>392900 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>This aptamer is predicted to form a three-way junction in the presence of cocaine.</td>
      <td>38</td>
      <td>None</td>
      <td>dGp,dGp,dGp,dAp,dGp,dAp,dCp,dAp,dAp,dGp,dGp,dAp,dAp,dAp,dAp,dTp,dCp,dCp,dTp,dTp,dCp,dAp,dAp,dTp,dGp,dAp,dAp,dGp,dTp,dGp,dGp,dGp,dTp,dCp,dGp,dAp,dCp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>45</th>
      <td>CCFM641-5</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/631.jpg" width="120" ></td>
      <td>Unknown Membrane Protien</td>
      <td>Protein</td>
      <td>10.69 +/- 0.89 nM (reported value)</td>
      <td>PBS at 37 C for 45 min with slight agitation</td>
      <td>37°C</td>
      <td>\n                                ssDNA library/pool was heat denatured at 95 C for 10 min and rapidly chilled for 10 min in an ice bath before incubation. \n</td>
      <td>~70% specificity to B. bifidum in comparison to other bacterial species. Including:  B. longum, B. animalis, B. breve, B. adolescentis and L. plantarum.</td>
      <td>12346 g/mole</td>
      <td>67.50%</td>
      <td>367800 L/(moleÂ·cm)</td>
      <td></td>
      <td></td>
      <td>Truncation reduced specificity.  Proteinase treatment of target cell-line reduced binding, hinting that target is a protein membrane.  ELAA limit of detection 10^4 cfu/mL.</td>
      <td>40</td>
      <td>Hu, Lujun, et al. "Selection, Characterization and Interaction Studies of a DNA Aptamer for the Detection of Bifidobacterium bifidum" International Journal of Molecular Sciences 10.3390(2017): 833.</td>
      <td>dTp,dGp,dCp,dGp,dTp,dGp,dAp,dGp,dCp,dGp,dGp,dTp,dAp,dGp,dGp,dCp,dCp,dCp,dCp,dTp,dAp,dCp,dGp,dAp,dCp,dCp,dCp,dAp,dCp,dTp,dGp,dTp,dGp,dGp,dTp,dTp,dGp,dGp,dGp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>46</th>
      <td>Keratinocyte Growth Factor (14F)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/138.png" width="120" ></td>
      <td>Keratinocyte Growth Factor (KGF)</td>
      <td>Protein</td>
      <td>0.3 pM (reported value)</td>
      <td>Binding Buffer (DPBS, 0.5 mM MgCl2, 0.2% BSA, 0.02% sodium azide for PC-3, serum-free MEM growth medium, 0.1% BSA, 25 mM HEPES (pH 7.4), 1 ug/ml heparin for NIH3T3/KGFR)</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>22180.13 g/mole</td>
      <td>56.52%</td>
      <td>664400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>69</td>
      <td>Pagratis, N., et al. "Potent 2'-amino-, and 2'-fluoro-2'- deoxyribonucleotide RNA inhibitors of keratinocyte growth factor." Nature Biotechnology, 25(1997):68-73.</td>
      <td>rGp,rGp,rGp,rAp,rGp,rGp,rAp,fCp,rGp,rAp,fUp,rGp,fCp,rGp,rGp,fUp,rGp,rGp,fUp,fCp,fUp,fCp,fCp,fCp,rAp,rAp,fUp,fUp,fCp,fUp,rAp,rAp,rAp,fCp,fUp,fUp,fUp,fCp,fUp,fCp,fCp,rAp,fUp,fCp,rGp,fUp,rAp,fUp,fCp,fUp,rGp,rGp,rGp,fCp,rAp,rGp,rAp,fCp,rGp,rAp,fCp,fUp,fCp,rGp,fCp,fCp,fCp,rGp,rAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>47</th>
      <td>Adenosine/ATP (DH25.42)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/98.jpg" width="120" ></td>
      <td>Adenosine and ATP</td>
      <td>Small Organic</td>
      <td>6 µM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>\n                                Heated to 75°C for 5 min and then cooled to room temperature for 20-30 min. \n</td>
      <td></td>
      <td>7868.16 g/mole</td>
      <td>64.00%</td>
      <td>251000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Forms a G-quartet</td>
      <td>25</td>
      <td>Huizenga and Szostak. "A DNA Aptamer That Binds Adenosine and ATP." Biochemistry, 34 (1995): 656-665.</td>
      <td>dCp,dCp,dTp,dGp,dGp,dGp,dGp,dGp,dAp,dGp,dTp,dAp,dTp,dTp,dGp,dCp,dGp,dGp,dAp,dGp,dGp,dAp,dAp,dGp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>48</th>
      <td>Aptamer TY04</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/584.jpg" width="120" ></td>
      <td>Multiple myeloma cells (MM.1S)</td>
      <td>Cells</td>
      <td></td>
      <td>yeast tRNA (0.1mg/ml; Sigma) and BSA (1 mg/ml; Sigma) into the wash buffer</td>
      <td>37°C</td>
      <td>None</td>
      <td>Aptermer TY04 inhibits the growth of MM cell lines, including MM.1S, NCI-H929, KM3, and OPM2.</td>
      <td>25587.67 g/mole</td>
      <td>45.78%</td>
      <td>794400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>83</td>
      <td>Dai, H. et al. "Aptamer TY04 inhibits the growth of multiple myeloma cells via cell cycle arrest."  Tumor Biology 2014: 1-8.</td>
      <td>dAp,dTp,dCp,dGp,dTp,dCp,dTp,dGp,dCp,dTp,dCp,dCp,dGp,dTp,dCp,dCp,dAp,dAp,dTp,dAp,dTp,dAp,dTp,dCp,dAp,dAp,dAp,dGp,dGp,dCp,dGp,dAp,dAp,dTp,dTp,dTp,dTp,dGp,dTp,dCp,dAp,dAp,dGp,dGp,dTp,dGp,dTp,dTp,dAp,dAp,dAp,dCp,dGp,dAp,dTp,dAp,dGp,dTp,dCp,dCp,dCp,dTp,dAp,dCp,dCp,dTp,dTp,dTp,dGp,dGp,dTp,dGp,dTp,dGp,dAp,dGp,dGp,dTp,dCp,dGp,dTp,dGp,dCp</td>
      <td>3.89 µM (reported value)</td>
    </tr>
    <tr>
      <th>49</th>
      <td>Bfl-1 Anti-Apoptotic Protein (Apt27)</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/411.png" width="120" ></td>
      <td>Bfl-1 Anti-Apoptotic Ptrotein</td>
      <td>Protein</td>
      <td>N/A nM (reported value)</td>
      <td>N/A</td>
      <td>N/A°C</td>
      <td>None</td>
      <td>Does not interact wit Bcl-2, Bcl-xL, or Mcl-1 in yeast.</td>
      <td>1547 g/mole</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>13</td>
      <td>None</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>50</th>
      <td>Acetylcholine (ACh 6R)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/442.png" width="120" ></td>
      <td>Acetylcholine</td>
      <td>Small Organic</td>
      <td>N/A nM (reported value)</td>
      <td>None</td>
      <td>37°C</td>
      <td>None</td>
      <td>Cross-reactivity for related small organics acetic acid, acetylchloride, choline, and malathion.</td>
      <td>22123.35 g/mole</td>
      <td>54.17%</td>
      <td>666100 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>72</td>
      <td>None</td>
      <td>dAp,dTp,dCp,dCp,dGp,dTp,dCp,dAp,dCp,dAp,dCp,dCp,dTp,dGp,dCp,dTp,dCp,dTp,dCp,dAp,dGp,dGp,dGp,dGp,dAp,dTp,dCp,dAp,dCp,dAp,dTp,dTp,dCp,dTp,dTp,dGp,dAp,dCp,dGp,dGp,dTp,dGp,dTp,dGp,dAp,dTp,dAp,dCp,dAp,dGp,dTp,dGp,dCp,dCp,dTp,dGp,dGp,dTp,dGp,dTp,dTp,dGp,dGp,dCp,dTp,dCp,dCp,dCp,dGp,dTp,dAp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>51</th>
      <td>Gastrin-Releasing Peptide (GRP-1)</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/386.png" width="120" ></td>
      <td>Gastrin-Releasing Peptide Receptor (GRPR) aka BB2 Receptor)</td>
      <td>Protein</td>
      <td>N/A nM (reported value)</td>
      <td>Peptide binding assays: Phosphate-buffered saline (PBS) with 1% fetal calf serum (FCS).</td>
      <td>RT°C</td>
      <td>None</td>
      <td>None</td>
      <td>1417.7 g/mole</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Aptamer was conjugated to survivin siRNA through the terminal cysteine residue, which was successfully delivered to MDA-MB 231 cells.</td>
      <td>36</td>
      <td>None</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>52</th>
      <td>CD133-B19</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7207.png" width="120" ></td>
      <td>CD133</td>
      <td>Protein</td>
      <td>145.1±75.4 nM (reported value)</td>
      <td>None</td>
      <td>°C</td>
      <td>\n                                m-Fold \n</td>
      <td></td>
      <td>6006.61 g/mole</td>
      <td>36.84%</td>
      <td>194200 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>19</td>
      <td>Shigdar, S. et al. Nov 19, 2012. "RNA aptamers targeting cancer stem cell marker CD133." Elsevier. Cancer Letters.</td>
      <td>rCp,rAp,rGp,rAp,rAp,rCp,rGp,rUp,rAp,rUp,rAp,rCp,rUp,rAp,rUp,rUp,rCp,rUp,rGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>53</th>
      <td>Huizenga ATP-Binding aptamer</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/642.png" width="120" ></td>
      <td>ATP</td>
      <td>Protein</td>
      <td>956 µM (reported value)</td>
      <td>None</td>
      <td>°C</td>
      <td>None</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>36</td>
      <td>V. C. ??zalp, D. ??am, F. J. Hernandez, L. I. Hernandez, T.  Sch??fer, and H. A. ??ktem, Small molecule detection by lateral flow strips via aptamer-gated silica nanoprobes, Analyst, 2016, 141, 2595.</td>
      <td>dCp,dAp,dCp,dCp,dTp,dGp,dGp,dGp,dGp,dGp,dAp,dGp,dTp,dAp,dTp,dTp,dGp,dCp,dGp,dGp,dAp,dGp,dGp,dAp,dAp,dGp,dGp,dTp,dTp,dCp,dCp,dAp,dGp,dGp,dTp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>54</th>
      <td>Aptamer ID 2</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/542.jpg" width="120" ></td>
      <td>PDGF-BB</td>
      <td>Protein</td>
      <td>2.5 nM (reported value)</td>
      <td>0.1 mM Na2HPO4, 1.8 mM KH2PO4, 137 mM NaCl, 2.7 mM KCl, and 1 mM MgCl2 (PBSM), pH 7.4</td>
      <td>23°C</td>
      <td>None</td>
      <td></td>
      <td>24307.75 g/mole</td>
      <td>52.50%</td>
      <td>728600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>3 rounds of positive selection at increasing selection stringencies</td>
      <td>240</td>
      <td>Cho et al. "Quantitative selection of DNA aptamers through microfluidic selection and high-throughput sequencing." PNAS 2010, 1-6.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>55</th>
      <td>Thrombopoietin Receptor (AF12505)</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/270.png" width="120" ></td>
      <td>Thrombopoietin Receptor</td>
      <td>Protein</td>
      <td>2 nM (reported value)</td>
      <td>N/A</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>1754.06 g/mole</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>The authors also created a dimerized version of AF12505 (AF13948). This peptide aptamer possessed an IC50 of 0.5 nM and EC50 4000 times that of the monomer.</td>
      <td>14</td>
      <td>Cwirla et al. "Peptide Agonist of the Thrombopoietin Receptor as Potent as the Natural Cytokine." Science, 276(1997): 1696-1699.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>56</th>
      <td>Human Colorectal Cancer WiDr Cell Line (HEW-K16)</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/186.jpeg" width="120" ></td>
      <td>Human Colorectal Cancer (WiDr Cell Line)</td>
      <td>Cells</td>
      <td>N/A nM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td></td>
      <td>1878.16 g/mole</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>13</td>
      <td>None</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>57</th>
      <td>Aptamer-1</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7228.jpg" width="120" ></td>
      <td>S/RBD receptor binding</td>
      <td>Protein</td>
      <td>nM (reported value)</td>
      <td>n Dulbecco's phosphate-buffered saline (DPBS) containing 1 mM MgCl2 (DPBS/Mg)</td>
      <td>23°C</td>
      <td>\n                                mFold \n</td>
      <td>Highly Specific</td>
      <td>23758.36 g/mole</td>
      <td>53.95%</td>
      <td>6531273 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Binds to the receptor-binding domain (RBD) of COVID-19 virus and inhibits the S/RBD interaction between virus and host cell.</td>
      <td>228</td>
      <td>https://doi.org/10.1002/anie.202100345</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>58</th>
      <td>HIV-1 Reverse Transcriptase (R1T)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/417.png" width="120" ></td>
      <td>HIV-1 RT</td>
      <td>Protein</td>
      <td>1.1 nM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td></td>
      <td>12825.34 g/mole</td>
      <td>63.41%</td>
      <td>390800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>42</td>
      <td>None</td>
      <td>dCp,dGp,dCp,dCp,dTp,dGp,dAp,dTp,dTp,dAp,dGp,dCp,dGp,dAp,dTp,dAp,dCp,dTp,dCp,dAp,dGp,dGp,dCp,dGp,dTp,dTp,dGp,dGp,dGp,dTp,dGp,dGp,dGp,dTp,dGp,dGp,dGp,dTp,dGp,dGp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>59</th>
      <td>NaA43</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/603.jpg" width="120" ></td>
      <td>Sodium Ion</td>
      <td>Other</td>
      <td>39.1 mM (reported value)</td>
      <td>Dulbeccoƒ??s modification of Eagleƒ??s medium (DMEM) supplemented with 10% Fetal Bovine Serum (FBS), 100 U/mL penicillin, and 100 µg/mL streptomycin,</td>
      <td>37°C</td>
      <td>None</td>
      <td>Tested against the following ions: Li, K, Rb, Cs, NH4, Mg, Ca, Sr, Ba, Mn, Co, Ni, Cu, Zn, Cd, Hg, Pb, In, La, Eu, Sm, Yb</td>
      <td>19035.35 g/mole</td>
      <td>63.93%</td>
      <td>602700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>SUBSTRATE SEQUENCE (spaces added before and after the ribonucleic A):   dCpdTpdCpdTpdApdTpdGpdTpdApdTp   rAp   dGpdGpdApdApdGpdTpdApdCpdCpdGpdCpdCpdGpdCp</td>
      <td>61</td>
      <td>Torabi, S., Wu, P., McGhee, C., Chen, L., Hwang, K., Zheng, N., Cheng, J., & Lu, Y. (2015) In Vitro Selection of a Sodium-Specific DNAzyme and its Application in Intracellular Sensing. PNAS 112, (19):5903-5908</td>
      <td>dCp,dTp,dCp,dTp,dAp,dTp,dGp,dTp,dAp,dTp,rAp,dGp,dGp,dAp,dAp,dGp,dTp,dAp,dCp,dCp,dGp,dCp,dCp,dGp,dCp,dGp,dCp,dGp,dGp,dCp,dGp,dGp,dTp,dAp,dCp,dCp,dAp,dGp,dGp,dTp,dCp,dAp,dAp,dAp,dGp,dGp,dTp,dGp,dGp,dGp,dTp,dGp,dAp,dGp,dGp,dGp,dGp,dAp,dCp,dGp,dCp,dCp,dAp,dAp,dGp,dAp,dGp,dTp,dCp,dCp,dCp,dCp,dGp,dCp,dGp,dGp,dTp,dTp,dAp,dCp,dAp,dTp,dAp,dGp,dAp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>60</th>
      <td>Atrazine R12.23</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/582.jpg" width="120" ></td>
      <td>Atrazine</td>
      <td>Small Organic</td>
      <td>0.62 nM (reported value)</td>
      <td>500 µL of buffer consisting of 100 mM sodium chloride, 20 mM Tris-HCl, and 2 mM magnesium chloride</td>
      <td>24°C</td>
      <td>None</td>
      <td>Analysis showed significantly higher binding of R12.23 to atrazine in solution than to four of the five negative targets used in the selection (see Table 3 of source material)</td>
      <td>24464.91 g/mole</td>
      <td>50.63%</td>
      <td>749900 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>237</td>
      <td>Williams, R.; Crihfield, C.; Gattu, S.; Holland, L.; Sooter, L. In Vitro Selection of a Single-Stranded DNA Molecular Recognition Element against Atrazine. International Journal of Molecular Sciences, 2014, 15, 14332-14347. ISSN 1422-0067 doi: 10.3990/ijms150814332</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>61</th>
      <td>Fcγ receptor IIIα / CD16α (CLN0020)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/495.png" width="120" ></td>
      <td>FcγrIIIα / FcgrIIIa / CD16a</td>
      <td>Protein</td>
      <td>22 nM (reported value)</td>
      <td>Binding Buffer: DPBS with 0.05% BSA</td>
      <td>37°C</td>
      <td>None</td>
      <td>Did not bind to CD16β.</td>
      <td>18824.27 g/mole</td>
      <td>56.67%</td>
      <td>606600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>60</td>
      <td>None</td>
      <td>dGp,dGp,dAp,dGp,dGp,dGp,dAp,dAp,dAp,dAp,dGp,dTp,dTp,dAp,dTp,dCp,dAp,dGp,dGp,dCp,dCp,dAp,dCp,dTp,dGp,dCp,dGp,dGp,dGp,dGp,dGp,dTp,dCp,dTp,dAp,dTp,dAp,dCp,dGp,dTp,dGp,dAp,dGp,dGp,dAp,dAp,dGp,dAp,dAp,dGp,dTp,dGp,dGp,dGp,dCp,dAp,dGp,dGp,dTp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>62</th>
      <td>XA-A9</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7150.jpg" width="120" ></td>
      <td>Thy-1 membrane glycoprotein</td>
      <td>Protein</td>
      <td>15 nM (reported value)</td>
      <td>10 mL of 20 mM Tris buffer (150 mM NaCl, pH 8.0)</td>
      <td>22°C</td>
      <td>None</td>
      <td></td>
      <td>12931.4 g/mole</td>
      <td>59.52%</td>
      <td>393800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>42</td>
      <td>Wang H, Li X, Lai LA, Brentnall TA, Dawson DW, Kelly KA, Chen R, Pan S. X-aptamers targeting Thy-1 membrane glycoprotein in pancreatic ductal adenocarcinoma. Biochimie. 2021 Feb;181:25-33. doi: 10.1016/j.biochi.2020.11.018. Epub 2020 Nov 23. PMID: 33242496; PMCID: PMC7863625.</td>
      <td>dTp,dTp,dGp,dCp,dCp,dCp,dAp,dCp,dAp,dGp,dAp,dTp,dCp,dTp,dGp,dTp,dGp,dGp,dAp,dAp,dGp,dCp,dCp,dGp,dAp,dAp,dCp,dCp,dGp,dCp,dGp,dTp,dGp,dCp,dTp,dAp,dGp,dTp,dCp,dGp,dTp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>63</th>
      <td>upanap-12</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/472.png" width="120" ></td>
      <td>Urokinase-type plasminogen activator</td>
      <td>Protein</td>
      <td>N/A nM (reported value)</td>
      <td>None</td>
      <td>37°C</td>
      <td>None</td>
      <td>Human urokinase-type plasminogen activator</td>
      <td>25480.22 g/mole</td>
      <td>44.30%</td>
      <td>805500 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>79</td>
      <td>None</td>
      <td>rGp,rGp,rGp,rGp,fCp,fCp,rAp,fCp,fCp,rAp,rAp,fCp,rGp,rAp,fCp,rAp,fUp,fUp,fUp,rGp,fCp,rGp,rAp,fCp,fUp,rGp,fUp,fUp,rAp,fUp,rAp,rAp,fCp,fCp,fUp,rAp,rAp,fCp,rAp,rGp,fCp,rGp,rAp,fCp,rGp,fUp,rAp,rAp,rAp,rGp,rAp,fUp,rAp,rGp,fUp,fUp,rGp,rAp,fUp,rAp,fUp,rAp,rAp,rAp,fUp,rAp,rGp,fUp,rGp,fCp,fCp,fCp,rAp,fUp,rGp,rGp,rAp,fUp,fCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>64</th>
      <td>NC3S</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7136.jpg" width="120" ></td>
      <td>N‑Cadherin</td>
      <td>Protein</td>
      <td>20.08 ± 0.72 nM (reported value)</td>
      <td>4.5 g L−1 glucose and 5 mM MgCl2 in Dulbecco’s PBS without calcium and magnesium, yeast tRNA and BSA</td>
      <td>4, 25, 37°C</td>
      <td>None</td>
      <td></td>
      <td>14499.4 g/mole</td>
      <td>44.68%</td>
      <td>454400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>47</td>
      <td>Gao T , Ding P , Li W , Wang Z , Lin Q , Pei R . Isolation of DNA aptamers targeting N-cadherin and high-efficiency capture of circulating tumor cells by using dual aptamers. Nanoscale. 2020 Nov 19;12(44):22574-22585. doi: 10.1039/d0nr06180h. PMID: 33174555.</td>
      <td>dTp,dTp,dGp,dCp,dAp,dCp,dTp,dAp,dTp,dGp,dTp,dTp,dTp,dTp,dAp,dGp,dCp,dTp,dAp,dGp,dGp,dGp,dTp,dTp,dCp,dCp,dCp,dTp,dCp,dCp,dGp,dGp,dAp,dGp,dAp,dTp,dAp,dGp,dTp,dAp,dAp,dGp,dTp,dGp,dCp,dAp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>65</th>
      <td>Anti-NF-kB p65 (R1)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/387.png" width="120" ></td>
      <td>NF-kB p65</td>
      <td>Protein</td>
      <td>11 nM (reported value)</td>
      <td>Selection buffer: 10mM HEPES at pH 7.5, 100mM NaCl, 1mM DTT</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>26750.22 g/mole</td>
      <td>48.19%</td>
      <td>837500 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>83</td>
      <td>Wurster, S. E, and Maher, L. J. "Selection and characterization of anti-NF-kB p65 aptamers." RNA, 14(2008): 1037-1047.</td>
      <td>rGp,rAp,rAp,rGp,rCp,rUp,rUp,rAp,rCp,rAp,rAp,rGp,rAp,rAp,rGp,rGp,rAp,rCp,rAp,rGp,rCp,rAp,rCp,rGp,rAp,rAp,rUp,rAp,rAp,rAp,rAp,rCp,rCp,rUp,rGp,rCp,rGp,rUp,rAp,rAp,rAp,rUp,rCp,rCp,rGp,rCp,rCp,rCp,rCp,rAp,rUp,rUp,rUp,rGp,rUp,rGp,rUp,rAp,rAp,rGp,rGp,rGp,rUp,rAp,rGp,rUp,rGp,rGp,rGp,rUp,rCp,rGp,rAp,rAp,rUp,rUp,rCp,rCp,rGp,rCp,rUp,rCp,rAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>66</th>
      <td>CCRF-CEM (sgc8)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/638" width="120" ></td>
      <td>Acute lymphoblastic leukemia (CCRF-CEM)</td>
      <td>Cells</td>
      <td>0.8 nM (reported value)</td>
      <td>None</td>
      <td>4°C</td>
      <td>None</td>
      <td>Recognizes and binds to cell lines Molt-4 (T cell Acute Lymphoblastic Leukemia [ALL]), Sup-T1 (T cell ALL), and Jurkat (T cell ALL).</td>
      <td>27106.7 g/mole</td>
      <td>42.05%</td>
      <td>859200 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>This aptamer was found to bind to protein tyrosine kinase 7 (PTK7) (Shangguan 2008).</td>
      <td>88</td>
      <td>None</td>
      <td>dAp,dTp,dAp,dCp,dCp,dAp,dGp,dCp,dTp,dTp,dAp,dTp,dTp,dCp,dAp,dAp,dTp,dTp,dAp,dGp,dTp,dCp,dAp,dCp,dAp,dCp,dTp,dTp,dAp,dGp,dAp,dGp,dTp,dTp,dCp,dTp,dAp,dAp,dCp,dTp,dGp,dCp,dTp,dGp,dCp,dGp,dCp,dCp,dGp,dCp,dCp,dGp,dGp,dGp,dAp,dAp,dAp,dAp,dTp,dAp,dCp,dTp,dGp,dTp,dAp,dCp,dGp,dGp,dTp,dTp,dAp,dGp,dAp,dTp,dAp,dGp,dTp,dAp,dAp,dGp,dTp,dGp,dCp,dAp,dAp,dTp,dCp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>67</th>
      <td>DTA-64</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/626.jpg" width="120" ></td>
      <td>DEK</td>
      <td>Protein</td>
      <td>N/A µM (reported value)</td>
      <td>Binding buffer (20 mM Tris pH 7.6; 100 mM NaCl; 5 mM MgCl2); Room Temperature (30 min - 1 hr, depending on round of testing).</td>
      <td>°C</td>
      <td>None</td>
      <td></td>
      <td>12593.25 g/mole</td>
      <td>46.34%</td>
      <td>399100 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>41</td>
      <td>N Mor-Vaknin et al. DEK-targeting DNA aptamers as therapeutics  for inflammatory arthritis. Nature Communications 8(2017): 14252.</td>
      <td>dGp,dGp,dGp,dGp,dTp,dTp,dAp,dAp,dAp,dTp,dAp,dTp,dTp,dCp,dCp,dCp,dAp,dCp,dAp,dTp,dTp,dGp,dCp,dCp,dTp,dGp,dCp,dGp,dCp,dCp,dAp,dGp,dTp,dAp,dCp,dAp,dAp,dAp,dTp,dAp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>68</th>
      <td>LZH5B</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7080.png" width="120" ></td>
      <td>HepG2 Liver Cancer Cells</td>
      <td>Cells</td>
      <td>12 nM (reported value)</td>
      <td>1x PBS with 5mM MgCl2</td>
      <td>37°C</td>
      <td>\n                                Heated to 95C for 5 minutes and snap cooled to 4C. \n</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Hachimoji DNA. Currently using PNA "pTn" as placeholders for  \n2-amino-8-(1′-b-D-2′-deoxyribofuranosyl)-imidazo-[1,2a]-1,3,5-triazin-[8H]-4-one bases (Hachimoji base P)</td>
      <td>43</td>
      <td>Zhang, L., Wang, S., Yang, Z., Hoshika, S., Xie, S., Li, J., …Tan, W. (2019). An Aptamer‐Nanotrain Assembled from 6‐Letter DNA Delivers Doxorubicin Selectively to Liver Cancer Cells. Angewandte Chemie. doi:10.1002/ange.201909691</td>
      <td>dGp,dTp,dGp,dAp,dCp,dGp,dCp,dAp,dGp,dCp,dAp,dGp,dCp,dTp,dAp,dCp,dTp,dGp,dGp,dGp,dCp,dCp,dCp,dTp,dGp,dGp,dTp,dTp,dCp,dTp,dGp,dTp,dGp,dCp,dTp,dGp,dGp,dAp,dCp,dAp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>69</th>
      <td>Cyclic Guanosine Monophosphate (AR2)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/192.png" width="120" ></td>
      <td>cGMP</td>
      <td>Protein</td>
      <td>N/A nM (reported value)</td>
      <td>None</td>
      <td>23°C</td>
      <td>None</td>
      <td></td>
      <td>29203.75 g/mole</td>
      <td>55.56%</td>
      <td>900600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>94</td>
      <td>Seetharaman et al. "Immobilized RNA switches for the analysis of complex chemical and biological mixture." Nature Biotechnology, 19(2001): 336-341.</td>
      <td>rGp,rGp,rAp,rUp,rAp,rAp,rUp,rAp,rGp,rCp,rCp,rGp,rUp,rAp,rGp,rGp,rUp,rUp,rGp,rCp,rGp,rAp,rAp,rAp,rGp,rCp,rGp,rAp,rCp,rCp,rCp,rUp,rGp,rAp,rUp,rGp,rAp,rGp,rCp,rCp,rCp,rUp,rGp,rCp,rGp,rAp,rUp,rGp,rCp,rAp,rGp,rAp,rAp,rAp,rGp,rGp,rUp,rGp,rCp,rUp,rGp,rAp,rCp,rGp,rAp,rCp,rAp,rCp,rAp,rUp,rCp,rGp,rAp,rAp,rAp,rCp,rGp,rGp,rUp,rAp,rGp,rCp,rGp,rAp,rGp,rAp,rGp,rCp,rUp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>70</th>
      <td>2F11-14</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/581.jpg" width="120" ></td>
      <td>Ebola virus inhibitory domain (eVP35 IID) (EBOV) (eVP35)</td>
      <td>Protein</td>
      <td>7.1 nM (reported value)</td>
      <td>10mM HEPES (pH 7.0), 150mM NaCl, 2mM TCEP (tris(2-carboxyethyl)phosphine), 1mM MgCl2</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>25439.76 g/mole</td>
      <td>56.41%</td>
      <td>797500 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>248</td>
      <td>Jennifer M. Binning, Tianjiao Wang, Priya Luthra, Reed S. Shabman, Dominika M. Borek, Gai  Liu, Wei Xu, Daisy W. Leung, Christopher F. Basler, and Gaya K. Amarasinghe, ƒ??Development of RNA aptamers targeting ebolavirus VP35ƒ? Biochemistry 2013, 52, 8406-8419.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>71</th>
      <td>Z23 Anti-Vesicular Stomatitis Virus</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/606.jpg" width="120" ></td>
      <td>Vesicular Stomatitis Virus</td>
      <td>Other</td>
      <td>EC50 =/< 80 nM (reported value)</td>
      <td>90% High Glucose DMEM, 10% Fetal Bovine Serum. Tested in vivo.</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>12015.87 g/mole</td>
      <td>48.72%</td>
      <td>380500 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>39</td>
      <td>Muharemagic D, Zamay A, Ghobadloo SM, Evgin L, Savitskaya A, et al. (2014) Aptamer-Facilitated Protection of Oncolytic Virus from Neutralizing Antibodies. Molecular Therapy--Nucleic Acids 3, e167. doi: 10.1038/mtna.2014.19</td>
      <td>dGp,dGp,dGp,dAp,dCp,dCp,dTp,dAp,dTp,dCp,dAp,dGp,dGp,dCp,dGp,dAp,dTp,dGp,dTp,dGp,dAp,dAp,dAp,dAp,dCp,dTp,dCp,dTp,dTp,dAp,dTp,dAp,dCp,dCp,dAp,dCp,dTp,dGp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>72</th>
      <td>Anti Aflatoxin M1</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/503.png" width="120" ></td>
      <td>Aflatoxin M1</td>
      <td>Protein</td>
      <td>0.0182 nM (reported value)</td>
      <td>N/A</td>
      <td>N/A°C</td>
      <td>\n                                N/A \n</td>
      <td>Counter target is  Ochratoxin A</td>
      <td>6380.23 g/mole</td>
      <td>38.10%</td>
      <td>201700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>affinity in this paper was determined by cyclic voltammetry  and measured current Vs E/V.  Kd was determined by finding the highest peak and converting its concentration of aptamer and target to nM of target. Aptamer sequence is shorter than most. The Nguyen et al. paper claims it came from a university in Vietnam but cites no paper for further investigation into its length.</td>
      <td>21</td>
      <td>Nguyen. Label Free detection of Aflatoxin M1 with electrochemical Fe3O4/polyaniline-based aptasensor</td>
      <td>dAp,dCp,dTp,dGp,dCp,dTp,dAp,dGp,dAp,dGp,dAp,dTp,dTp,dTp,dTp,dCp,dCp,dAp,dCp,dAp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>73</th>
      <td>Kanamycin (LGA11)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/429.png" width="120" ></td>
      <td>Kanamycin and other aminoglycosides (AMGs)</td>
      <td>Small Organic</td>
      <td>52.8 nM (reported value)</td>
      <td>Binding Buffer: PBS with 10% (v/v) methanol</td>
      <td>RT°C</td>
      <td>None</td>
      <td>Binds to several aminoglycosides: kanamycin, paromycin streptomycin, apramycin, tobramycin, neomycin, and gentamycin.</td>
      <td>25568.23 g/mole</td>
      <td>56.96%</td>
      <td>771400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>87</td>
      <td>None</td>
      <td>rGp,rGp,rGp,rAp,rGp,rAp,rAp,rGp,rGp,fCp,rGp,rGp,fCp,rGp,fCp,rGp,fUp,rAp,rGp,rGp,fCp,rGp,rAp,rGp,fCp,fUp,fUp,fUp,rAp,rGp,rGp,fCp,fCp,fCp,rGp,rAp,fCp,rAp,fUp,fUp,fCp,fCp,fCp,fCp,fUp,rAp,rAp,rAp,rAp,rAp,rAp,rGp,fCp,fUp,fUp,rGp,fUp,fUp,fCp,fUp,rAp,fUp,fUp,fCp,rGp,fUp,rGp,rGp,rAp,fCp,rGp,rAp,fUp,rGp,fCp,rGp,fCp,rAp,fCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>74</th>
      <td>XMU3</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7149.jpg" width="120" ></td>
      <td>epithelial cell adhesion  molecule (EpCAM)</td>
      <td>Cells</td>
      <td>9.3 ± 1.3 nM (reported value)</td>
      <td>PBS containing 0.55 mM MgCl2 and 0.2 mg/mL yeast tRNA, pH = 7.4</td>
      <td>25°C</td>
      <td>None</td>
      <td></td>
      <td>28366.3 g/mole</td>
      <td>60.87%</td>
      <td>860500 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>92</td>
      <td>Huang, M., Li, T., Xu, Y., Wei, X., Song, J., Lin, B., … Yang, C. J. (2020). Activation of Aptamers with Gain‐of‐Function by Small‐Molecule‐Clipping of Intramolecular Motifs. Angewandte Chemie International Edition. doi:10.1002/anie.202013570</td>
      <td>dTp,dCp,dCp,dAp,dGp,dCp,dAp,dCp,dTp,dCp,dCp,dAp,dCp,dGp,dCp,dAp,dTp,dAp,dAp,dCp,dCp,dCp,dCp,dGp,dTp,dCp,dCp,dTp,dCp,dGp,dGp,dAp,dAp,dAp,dGp,dAp,dGp,dCp,dGp,dCp,dGp,dGp,dGp,dCp,dCp,dTp,dGp,dGp,dGp,dAp,dCp,dGp,dGp,dCp,dTp,dAp,dCp,dTp,dGp,dCp,dCp,dCp,dGp,dGp,dAp,dTp,dTp,dTp,dAp,dTp,dGp,dTp,dGp,dTp,dTp,dAp,dTp,dGp,dCp,dGp,dTp,dGp,dCp,dGp,dAp,dCp,dGp,dGp,dTp,dGp,dAp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>75</th>
      <td>Amyloid Peptide BetaA4(1ƒ?? 40) (B55)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7093" width="120" ></td>
      <td>Amyloid Peptide BetaA4(1ƒ?? 40)</td>
      <td>Peptide</td>
      <td>29 nM (reported value)</td>
      <td>Binding Buffer (10 mM Trisƒ??HCl (pH 7.5), 150 mM NaCl, 5 mM MgCl2, 1 mM EDTA). Incubated at room temperature.</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>34199.41 g/mole</td>
      <td>54.21%</td>
      <td>1012800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>This aptamer is specific to polymeric amyloid.</td>
      <td>107</td>
      <td>Ylera, Francisco, et al. "Selection of RNA Aptamers to the Alzheimer's Disease Amyloid Peptide." Biochemical and Biophysical Research Communities, 290 (2002): 1583-1588.</td>
      <td>rGp,rGp,rGp,rAp,rAp,rUp,rUp,rCp,rGp,rAp,rGp,rCp,rUp,rCp,rGp,rGp,rUp,rAp,rCp,rCp,rUp,rUp,rUp,rAp,rCp,rCp,rGp,rUp,rAp,rAp,rGp,rGp,rCp,rCp,rUp,rGp,rUp,rCp,rUp,rUp,rCp,rGp,rUp,rUp,rUp,rGp,rAp,rCp,rAp,rGp,rCp,rGp,rGp,rCp,rUp,rUp,rGp,rUp,rUp,rGp,rAp,rCp,rCp,rCp,rUp,rCp,rAp,rCp,rAp,rCp,rUp,rUp,rUp,rGp,rUp,rAp,rCp,rCp,rUp,rGp,rCp,rUp,rGp,rCp,rCp,rAp,rAp,rCp,rUp,rGp,rCp,rAp,rGp,rGp,rCp,rAp,rUp,rGp,rCp,rAp,rAp,rGp,rCp,rUp,rUp,rGp,rGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>76</th>
      <td>Dopamine (DNA version of dopa1.30/c.30)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/403.png" width="120" ></td>
      <td>Dopamine</td>
      <td>Small Organic</td>
      <td>0.7 µM (reported value)</td>
      <td>None</td>
      <td>N/A°C</td>
      <td>None</td>
      <td>Aptamer has a similar affinity to related molecules dopamine and norepinephrine but much lower affinity for tyramine</td>
      <td>17982.68 g/mole</td>
      <td>60.34%</td>
      <td>561000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>This aptamer is a homolog of dopamine aptamer dopa1.30/c.30.</td>
      <td>65</td>
      <td>None</td>
      <td>dGp,dTp,dCp,dTp,dCp,dTp,dGp,dTp,dGp,dTp,dGp,dCp,dGp,dCp,dCp,dAp,dGp,dAp,dGp,dAp,dAp,dCp,dAp,dCp,dTp,dGp,dGp,dGp,dGp,dCp,dAp,dGp,dAp,dTp,dAp,dTp,dGp,dGp,dGp,dCp,dCp,dAp,dGp,dCp,dAp,dCp,dAp,dGp,dAp,dAp,dTp,dGp,dAp,dGp,dGp,dCp,dCp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>77</th>
      <td>Cyclic Adenosine Monophosphate (AR4)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/196.jpg" width="120" ></td>
      <td>cAMP</td>
      <td>Small Organic</td>
      <td>N/A nM (reported value)</td>
      <td>None</td>
      <td>23°C</td>
      <td>None</td>
      <td></td>
      <td>29204.74 g/mole</td>
      <td>54.44%</td>
      <td>905300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>93</td>
      <td>Seetharaman et al. "Immbolized RNA switches for the analysis of complex chemical and biological mixtures." Nature Biotechnology, 19(2001): 336-341.</td>
      <td>rGp,rGp,rAp,rUp,rAp,rAp,rUp,rAp,rGp,rCp,rCp,rGp,rUp,rAp,rGp,rGp,rUp,rUp,rGp,rCp,rGp,rAp,rAp,rAp,rGp,rCp,rGp,rAp,rCp,rCp,rCp,rUp,rGp,rAp,rUp,rGp,rAp,rGp,rCp,rCp,rUp,rGp,rUp,rGp,rGp,rAp,rAp,rAp,rCp,rAp,rGp,rAp,rCp,rGp,rUp,rGp,rGp,rCp,rAp,rCp,rAp,rUp,rGp,rAp,rCp,rUp,rAp,rCp,rGp,rUp,rCp,rGp,rAp,rAp,rAp,rCp,rGp,rGp,rUp,rAp,rGp,rCp,rGp,rAp,rGp,rAp,rGp,rCp,rUp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>78</th>
      <td>Antigen binding fragment of antivesicular stomatitis virus polyclonal antibodies (C5S)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/390.png" width="120" ></td>
      <td>Antigen binding fragment (Fab) of antivesicular stomatitis virus (VSV) rabbit polyclonal antibodies</td>
      <td>Protein</td>
      <td>N/A nM (reported value)</td>
      <td>DPBS</td>
      <td>RT°C</td>
      <td>None</td>
      <td></td>
      <td>12604.2 g/mole</td>
      <td>67.50%</td>
      <td>393300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>This aptamer binds to neutralizing antibodies that target the vesicular stomatitis virus (VSV). Incubating Vero cells in an aptamer-VSV-neutralizing antibody mixture recovered viral infectivity by more than 50%.</td>
      <td>41</td>
      <td>None</td>
      <td>dTp,dGp,dTp,dGp,dCp,dCp,dAp,dAp,dAp,dGp,dAp,dGp,dAp,dGp,dTp,dGp,dGp,dTp,dGp,dGp,dGp,dGp,dGp,dGp,dGp,dTp,dGp,dGp,dGp,dCp,dGp,dGp,dAp,dAp,dCp,dTp,dCp,dGp,dCp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>79</th>
      <td>Staphylococcus Aureus RNA Aptamer</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/593.jpg" width="120" ></td>
      <td>Staphylococcus Aureus</td>
      <td>Cells</td>
      <td>See Reference nM (reported value)</td>
      <td>Tris-HCl, pH 7.5, 150 mM NaCl, 1.5 mM MgCl2, 2 mM DTT, 1 % BSA</td>
      <td>23°C</td>
      <td>None</td>
      <td></td>
      <td>28512.88 g/mole</td>
      <td>53.41%</td>
      <td>875000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>2'- fluoro-2'-deoxyribonucleotide pyrimidines</td>
      <td>264</td>
      <td>Seung Ryul Han & Seong-Wook Lee, (2014). In vitro selection of RNA aptamer specific to Staphylococcus aureus. Ann Microbiol 64:883-885 DOI 10.1007/s13213-013-0720-z</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>80</th>
      <td>AP32-4</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7175.jpg" width="120" ></td>
      <td>Metronidazole</td>
      <td>Small Organic</td>
      <td>77.22 ± 11.27 nM (reported value)</td>
      <td>200mL binding buffer: 50 mM Tris-HCl containing 5 mM KCl, 100 mM NaCl and 1 mMMgCl2, pH 7.4</td>
      <td>22°C</td>
      <td>None</td>
      <td></td>
      <td>4639 g/mole</td>
      <td>53.33%</td>
      <td>143000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>15</td>
      <td>Wei, Hao et al. “Screening and application of a truncated aptamer for high-sensitive fluorescent detection of metronidazole.” Analytica chimica acta vol. 1128 (2020): 203-210. doi:10.1016/j.aca.2020.07.003</td>
      <td>dCp,dTp,dGp,dTp,dTp,dTp,dGp,dGp,dTp,dAp,dGp,dGp,dCp,dAp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>81</th>
      <td>Kit-129</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/502.jpg" width="120" ></td>
      <td>CD117 (c-kit)</td>
      <td>Protein</td>
      <td>12.21 nM (reported value)</td>
      <td>buffer composed of Dulbeccoƒ??s phosphate-buffered saline with  calcium and magnesium and 5 mM MgCl2,  0.45% glucose and 100 mg/ml tRNA. 1 hr incubation with cells</td>
      <td>4°C</td>
      <td>\n                                heated the library for  10 min at 95°C, followed by snap cooling on ice for 10 min \n</td>
      <td>Kit-129 has high specificity for  c-kit expressing  cells based on Flow Cytometry</td>
      <td>8839.75 g/mole</td>
      <td>65.52%</td>
      <td>262400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>29</td>
      <td>Development of an Efficient Targeted Cell-SELEX  Procedure for DNA Aptamer Reagents   Susanne Meyer et al. 2013</td>
      <td>dGp,dCp,dTp,dCp,dAp,dAp,dCp,dGp,dCp,dGp,dGp,dGp,dAp,dCp,dGp,dGp,dCp,dTp,dCp,dTp,dCp,dCp,dCp,dAp,dTp,dTp,dGp,dAp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>82</th>
      <td>K19</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/520.jpg" width="120" ></td>
      <td>Siglec-5</td>
      <td>Protein</td>
      <td>12.37 nM (reported value)</td>
      <td>4.5 g/L glucose, 5 mM MgCl2, 0.1 mg/ml yeast tRNA and 1 mg/ml Bovine Serum Albumin (BSA) into Phosphate buffered saline (PBS)</td>
      <td>°C</td>
      <td>None</td>
      <td></td>
      <td>29566.33 g/mole</td>
      <td>41.05%</td>
      <td>946000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>285</td>
      <td>Yang, M. et al. "Developing aptamer probes for acute myelogenous leukemia detection and surface protein biomarker discovery." Journal of Hematology & Oncology 2014, 7:5.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>83</th>
      <td>Short Mitomer2</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/576.jpg" width="120" ></td>
      <td>Mitochondria</td>
      <td>Other</td>
      <td>. nM (reported value)</td>
      <td>20mM Tris-HCl, 2mM KCl, 40mM NaCl, 1mM MgCl2, 250mM Sucrose, pH=7.5</td>
      <td>37°C</td>
      <td>\n                                80C for 10 min, and cooled down slowly to 4C \n</td>
      <td></td>
      <td>4817.85 g/mole</td>
      <td>80.00%</td>
      <td>134900 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Binding activity measured by fluorescence intensity within referenced paper.</td>
      <td>50</td>
      <td>Yuri Tawaraya, Mamoru Hyodo, Mst Naznin Ara, Yuma Yamada, and Hideyoshi Harashima, ƒ??RNA Aptamers for Targeting Mitochondria Using a Mitochondria-Based SELEX Methodƒ? Biol. Pharm. Bull. 37(8) 1411-1415 (2014)</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>84</th>
      <td>StrepApt 2</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/550.jpg" width="120" ></td>
      <td>Streptavidin</td>
      <td>Protein</td>
      <td>77 nM (reported value)</td>
      <td>NaCl (100 mM), MgCl2 (2 mM), KCl (5 mM), CaCl2 (1 mM), Tris-HCl (pH 7.6, 20 mM)</td>
      <td>°C</td>
      <td>\n                                Heated to 95C for 8 minutes, cool on ice for 10 minutes and left at room temperature for at  least 10 minutes. \n</td>
      <td></td>
      <td>8864.76 g/mole</td>
      <td>68.97%</td>
      <td>267200 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Ka = 3.98 x 10^4  kd = 3.07 x 10^-3    Best KD with highest Ka.</td>
      <td>87</td>
      <td>Ruigrok, Vincent J. B., et al. "Kinetic and Stoichiometric Characterisation of Streptavidin-Binding Aptamers." ChemBioChem 13 (2012): 829-836. Print.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>85</th>
      <td>theophylline-induced molecular switch</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/397.png" width="120" ></td>
      <td>theophylline</td>
      <td>Small Organic</td>
      <td>na nM (reported value)</td>
      <td>Reaction buffer [50 mM Tris.HCl (pH 7.5) and 20 mM MgCl2] for 20 hr</td>
      <td>23°C</td>
      <td>None</td>
      <td></td>
      <td>24536.86 g/mole</td>
      <td>60.53%</td>
      <td>736500 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>76</td>
      <td>None</td>
      <td>rGp,rGp,rGp,rCp,rGp,rAp,rCp,rCp,rCp,rUp,rGp,rAp,rUp,rGp,rAp,rGp,rCp,rCp,rUp,rUp,rAp,rUp,rAp,rCp,rCp,rAp,rGp,rCp,rCp,rGp,rAp,rAp,rAp,rGp,rGp,rCp,rCp,rCp,rUp,rUp,rGp,rGp,rCp,rAp,rGp,rAp,rCp,rGp,rUp,rCp,rGp,rAp,rAp,rAp,rCp,rGp,rGp,rUp,rGp,rAp,rAp,rAp,rGp,rCp,rCp,rGp,rUp,rAp,rGp,rGp,rUp,rUp,rGp,rCp,rCp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>86</th>
      <td>Liver-Specific Ligands (236)</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/200.png" width="120" ></td>
      <td>Liver-Specific Ligands</td>
      <td>Tissue</td>
      <td>n/a nM (reported value)</td>
      <td>Selection Buffer (PBS, 1 M NaCl, and 2% Triton X-100)    Wash Buffers (0.1% Sodium Borohydride, 0.1 M Phosphate Buffer) (PBS, 5% BSA, 0.1% cold water fish gelatin, and 5% normal goat serum in PBS)    Incubation Buffer (PBS and 1.1% BSA-C)</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>916.24 g/mole</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>77% liver uptake</td>
      <td>7</td>
      <td>Ludtke et al. "In Vivo Selection and Validation of Liver-Specific Ligands Using a New T7 Phage Peptide Display System." Drug Delivery, 14(2007): 357-369.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>87</th>
      <td>Plasmodium vivax lactate dehydrogenase (pL2)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/310.png" width="120" ></td>
      <td>Plasmodium vivax lactate dehydrogenase  (PvLDH)</td>
      <td>Protein</td>
      <td>31.7 nM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td></td>
      <td>11750.66 g/mole</td>
      <td>57.89%</td>
      <td>358500 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>38</td>
      <td>None</td>
      <td>dGp,dAp,dAp,dCp,dTp,dCp,dAp,dTp,dTp,dGp,dGp,dCp,dTp,dGp,dGp,dAp,dGp,dGp,dCp,dGp,dGp,dCp,dAp,dGp,dTp,dAp,dCp,dCp,dGp,dCp,dTp,dTp,dGp,dAp,dGp,dTp,dTp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>88</th>
      <td>Galactose-bisboronic acid aptamer-sensor</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7104.jpg" width="120" ></td>
      <td>Galactose</td>
      <td>Small Organic</td>
      <td>13 nM (reported value)</td>
      <td>Shinkai's receptor 50uM with 2% methanol.</td>
      <td>24 °C°C</td>
      <td>\n                                Mix and incubate for 5 min at 95°C, and cool down at room temperature 24°C > 30 min \n</td>
      <td>N/A</td>
      <td>15612.13 g/mole</td>
      <td>58.82%</td>
      <td>520380 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>sensor mixtures (fluorescein-labeled aptamer + competitor dabcyl-labeled oligonucleotide) to the addition of Shinkai’s receptor in the presence of increasing amounts of sugar, with the actual concentration of a receptor*complex at any particular glucose concentrations calculated from the Kd of Shinkai’s receptor.</td>
      <td>51</td>
      <td>Yang KA, Barbu M, Halim M, et al. Recognition and sensing of low-epitope targets via ternary complexes with oligonucleotides and synthetic receptors. Nat Chem. 2014;6(11):1003-1008. doi:10.1038/nchem.2058</td>
      <td>dCp,dTp,dCp,dTp,dCp,dGp,dGp,dGp,dAp,dCp,dGp,dAp,dCp,dCp,dAp,dCp,dTp,dAp,dCp,dGp,dCp,dAp,dTp,dAp,dGp,dTp,dTp,dTp,dCp,dTp,dAp,dTp,dCp,dGp,dCp,dCp,dAp,dGp,dGp,dAp,dAp,dGp,dGp,dGp,dTp,dCp,dGp,dTp,dCp,dCp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>89</th>
      <td>Tr-1</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7094.jpg" width="120" ></td>
      <td>IL2-CD25 Complex</td>
      <td>Protein</td>
      <td>67.8 nM (reported value)</td>
      <td>HEPES-buffered saline, pH=7.5 with 2mmol/L CaCl2</td>
      <td>37°C</td>
      <td>\n                                Transcribed RNA aptamer library was folded at 2 µM concentration in AIM-V (BSA) medium by\nstep-wise incubation at 95°C for 5 minutes, 65°C for 10 minutes, followed by 37°C for 20\nminutes. \n \n</td>
      <td></td>
      <td>13299.14 g/mole</td>
      <td>66.67%</td>
      <td>593700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>51</td>
      <td>Veeramani, Suresh,et al."An RNA aptamer-based biomarker platform demonstrates high soluble CD25 occupancy by IL2 in the serum of follicular lymphoma patients" American Association for Cancer Research 7, (2019) 1511-1522</td>
      <td>rGp,rGp,rGp,rAp,rGp,rGp,rAp,rCp,rGp,rAp,rUp,rGp,rCp,rGp,rGp,rUp,rCp,rCp,rUp,rGp,rUp,rCp,rGp,rUp,rCp,rUp,rGp,rUp,rUp,rCp,rGp,rUp,rCp,rCp,rCp,rCp,rAp,rGp,rAp,rCp,rGp,rAp,rCp,rUp,rCp,rGp,rCp,rCp,rCp,rGp,rAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>90</th>
      <td>C-Reactive Protein (CRP1-1)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/453.png" width="120" ></td>
      <td>C-Reactive Protein (CRP)</td>
      <td>Protein</td>
      <td>2.25 nM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td>None</td>
      <td>33559.36 g/mole</td>
      <td>53.85%</td>
      <td>1033300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>115</td>
      <td>None</td>
      <td>rGp,rGp,rGp,rCp,rGp,rAp,rAp,rUp,rUp,rCp,rGp,rGp,rGp,rAp,rCp,rUp,rUp,rCp,rGp,rAp,rUp,rCp,rCp,rGp,rUp,rAp,rGp,rUp,rAp,rCp,rCp,rCp,rAp,rCp,rCp,rAp,rGp,rGp,rCp,rAp,rUp,rAp,rCp,rAp,rCp,rCp,rAp,rGp,rCp,rAp,rCp,rGp,rCp,rGp,rGp,rAp,rGp,rCp,rCp,rAp,rAp,rGp,rGp,rAp,rAp,rAp,rAp,rAp,rUp,rAp,rGp,rUp,rAp,rAp,rAp,rCp,rUp,rAp,rGp,rCp,rAp,rCp,rUp,rCp,rAp,rGp,rUp,rGp,rCp,rUp,rCp,rGp,rUp,rAp,rUp,rGp,rCp,rGp,rGp,rAp,rAp,rGp,rCp,rUp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>91</th>
      <td>HIV-1 Nuceocapsid</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/132.jpg" width="120" ></td>
      <td>HIV-1 Nuceocapsid</td>
      <td>Protein</td>
      <td>2.3 nM (reported value)</td>
      <td>Binding Buffer (50 mM Tris, pH 7.5, 100 mM NaCl, 1 mM MgCl2, 30 µM ZnCl and 10 mM DTT)    SSC buffer (17.5 mM sodium citrate (pH 7.0), 150 mM sodium chloride)</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>12814.65 g/mole</td>
      <td>57.50%</td>
      <td>381200 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>40</td>
      <td>Berglund, A., et al. "A High Affinity Binding Site for the HIV-1 Nucleocapsid Protein." Nucleic Acids Research, 25(1997):1042-1049.</td>
      <td>rGp,rGp,rAp,rGp,rAp,rCp,rAp,rUp,rCp,rCp,rCp,rUp,rCp,rUp,rUp,rGp,rUp,rUp,rGp,rGp,rUp,rGp,rGp,rUp,rGp,rCp,rCp,rGp,rUp,rGp,rUp,rGp,rGp,rAp,rUp,rGp,rUp,rCp,rUp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>92</th>
      <td>TEPP</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7232.jpg" width="120" ></td>
      <td>Tfr (transferrin receptor) / EpCAM (epithelial cell adhesion molecule)</td>
      <td>Small Organic</td>
      <td>Tfr:  110 ± 22.0, EpCAM: 85.6 ± 28.5 nM (reported value)</td>
      <td>0.1 M sodium acetate (CH3COONa), 0.05 M sodium chloride (NaCl), and 5 mM MgCl2</td>
      <td>37°C</td>
      <td>\n                                The dissociation constant (Kd) of the aptamer-DOX conjugate was determined by measuring binding to native protein targets using flow cytometric analysis. The cells of interest (5 × 105) were first incubated with binding buffer for 30 min [PBS containing 10% FCS, 0.1 mg/mL transfer ribonucleic acid (tRNA), and 0.1 mg/mL bovine serum albumin], followed by two washes with binding buffer, before incubation with serial concentrations of TYE665-labeled aptamer-DOX conjugates prepared as described in the previous section (0–400 nM) in a 100-μL volume of binding buffer for 60 min at 37°C. The cells were washed three times with PBS, resuspended in 100 μL of PBS, and subjected to flow cytometric analyses (FACS Canto II flow cytometer; Becton Dickinson). \n</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Bifunctional aptamer</td>
      <td>33</td>
      <td>Macdonald, J., Denoyer, D., Henri, J., Jamieson, A., Burvenich, I. J. G., Pouliot, N., and S. Shigar. 2020. Bifunctional Aptamer–Doxorubicin Conjugate Crosses the Blood–Brain Barrier and Selectively Delivers Its Payload to EpCAM-Positive Tumor Cells. Nucleic Acid Therapeutics 30:2: 117-128. DOI: 10.1089/nat.2019.0807</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>93</th>
      <td>VEGF-Stimulated HUVECs</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/354.png" width="120" ></td>
      <td>VEGF-Stimulated human umbilical vein endothelial cells (HUVECs)</td>
      <td>Protein</td>
      <td>N/A nM (reported value)</td>
      <td>None</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>2006.41 g/mole</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>15</td>
      <td>None</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>94</th>
      <td>H6</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7197.jpg" width="120" ></td>
      <td>Triclosan</td>
      <td>Protein</td>
      <td>369.82 nM (reported value)</td>
      <td>100 mM NaCl, 2 mM KCl, 5 mM MgCl2,\n1 mM CaCl2, and 20 mM Tris-HCl, pH 7.5</td>
      <td>22°C</td>
      <td>None</td>
      <td></td>
      <td>23407.2 g/mole</td>
      <td>44.00%</td>
      <td>758500 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>75</td>
      <td>Li S, Clarkson M, McNatty K. Selection and characterisation of triclosan-specific aptamers using a fluorescence microscope-imaging assay. Anal Bioanal Chem. 2020 Oct;412(26):7285-7294. doi: 10.1007/s00216-020-02863-7. Epub 2020 Aug 11. PMID: 32780154.</td>
      <td>dAp,dTp,dAp,dCp,dGp,dAp,dGp,dCp,dTp,dTp,dGp,dTp,dTp,dCp,dAp,dAp,dTp,dAp,dGp,dTp,dCp,dTp,dGp,dAp,dAp,dGp,dGp,dGp,dAp,dTp,dTp,dGp,dAp,dGp,dTp,dAp,dTp,dCp,dCp,dGp,dGp,dGp,dGp,dAp,dGp,dAp,dTp,dGp,dAp,dTp,dGp,dTp,dCp,dGp,dTp,dTp,dGp,dGp,dTp,dGp,dAp,dTp,dAp,dGp,dTp,dAp,dAp,dGp,dAp,dGp,dCp,dAp,dAp,dTp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>95</th>
      <td>R10C1</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7178.png" width="120" ></td>
      <td>Legionella pneumophila (Lp)</td>
      <td>Other</td>
      <td>135 nM (reported value)</td>
      <td>1X PBS with 0.1 mg/ml salmon sperm DNA, 1% bovine serum albumin, and 0.05% Tween 20.</td>
      <td>25°C</td>
      <td>None</td>
      <td></td>
      <td>18260.8 g/mole</td>
      <td>55.00%</td>
      <td>550100 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>60</td>
      <td>Saad, M.; Chinerman, D.; Tabrizian, M.; Faucher, S. Identification of two aptamers binding to Legionella pneumophila with high affinity and specificity. Sci Rep. 2020 Jun 4; 10: 9145. doi: 10.1038/s4598-020-65973-3. PMCID: PMC7272621. PMID: 32499557.</td>
      <td>dGp,dCp,dAp,dAp,dTp,dGp,dGp,dTp,dAp,dCp,dGp,dGp,dTp,dAp,dCp,dTp,dTp,dCp,dCp,dCp,dCp,dAp,dCp,dCp,dCp,dCp,dAp,dCp,dGp,dCp,dTp,dGp,dCp,dTp,dCp,dCp,dCp,dAp,dAp,dAp,dAp,dGp,dTp,dGp,dCp,dAp,dCp,dGp,dCp,dTp,dAp,dCp,dTp,dTp,dTp,dGp,dCp,dTp,dAp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>96</th>
      <td>IR-A48</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/605.png" width="120" ></td>
      <td>Insulin Receptor</td>
      <td>Protein</td>
      <td>3.5 nM (reported value)</td>
      <td>Krebs-Ringer HEPES Buffer. Tested in vivo.</td>
      <td>37°C</td>
      <td>\n                                95 C for 5 minutes, allowed to cool to room temperature. \n</td>
      <td></td>
      <td>10129.43 g/mole</td>
      <td>70.37%</td>
      <td>313200 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>The 'N's in the structure indicate 5-[N-(1-naphthylmethyl)carboxamide]-2-deoxyuridine (Nap-dU) used in place of Thymine to improve binding efficiency.</td>
      <td>33</td>
      <td>Yunn N, Koh A, Han S, Lim JH, Park S, et al. (2015) Agonistic Aptamer to the Insulin Receptor Leads to Biased Signaling and Functional Selectivity Through Allosteric Modulation. Nucleic Acids Research 1. doi:10.1093/narlgkv767</td>
      <td>dCp,dGp,dCp,dCp,dTp,dGp,dGp,dGp,dAp,dAp,dGp,dAp,dCp,dAp,dAp,dCp,dCp,dCp,dAp,dGp,dGp,dCp,dAp,dGp,dGp,dCp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>97</th>
      <td>Cholic Acid (9)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/94.jpg" width="120" ></td>
      <td>Cholic Acid</td>
      <td>Small Organic</td>
      <td>50 pM (reported value)</td>
      <td>selection buffer (50 mM Trisƒ??HCl, 300 mM NaCl, 30 mM KCl, 5 mM MgCl2, pH 7.6</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>30924.2 g/mole</td>
      <td>41.00%</td>
      <td>985700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>100</td>
      <td>Kato, T., et al. "In vitro selection of DNA aptamers which bind to cholic acid." Biochimica et Biophysica Acta (BBA) - Gene Structure and Expression, 1493 (2000):</td>
      <td>dGp,dTp,dAp,dCp,dCp,dAp,dGp,dCp,dTp,dTp,dAp,dTp,dTp,dCp,dAp,dAp,dTp,dTp,dAp,dCp,dCp,dGp,dCp,dGp,dAp,dAp,dGp,dAp,dAp,dGp,dTp,dGp,dTp,dCp,dAp,dTp,dTp,dGp,dTp,dTp,dTp,dTp,dGp,dGp,dAp,dGp,dAp,dTp,dTp,dCp,dGp,dAp,dAp,dGp,dCp,dGp,dCp,dTp,dGp,dTp,dAp,dCp,dAp,dCp,dAp,dGp,dGp,dTp,dAp,dAp,dTp,dGp,dAp,dAp,dGp,dCp,dCp,dTp,dTp,dCp,dTp,dAp,dAp,dGp,dAp,dTp,dAp,dGp,dTp,dAp,dTp,dGp,dTp,dTp,dCp,dAp,dTp,dCp,dAp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>98</th>
      <td>RhD antigen‑specificity 1</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7162.jpg" width="120" ></td>
      <td>RhD+ red blood cells</td>
      <td>Cells</td>
      <td>580.5±142.0 nM (reported value)</td>
      <td>10 mM Tris-HCl, 50 mM KCl, pH 9.0), 1.5 mM MgCl2, 400 µM each \ndNTP, 2.5 U Taq DNA polymerase (Promega Corporation), and 50 pmol each biotinylated primer</td>
      <td>25°C</td>
      <td>None</td>
      <td></td>
      <td>12280.9 g/mole</td>
      <td>65.00%</td>
      <td>363800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>40</td>
      <td>Zhang Y, Xu H, Wang X, Wang L, Liu R, Li L, Zhou H. Single‑strained DNA aptamers mask RhD antigenic epitopes on human RhD+ red blood cells to escape alloanti‑RhD immunological recognition. Mol Med Rep. 2020 Apr;21(4):1841-1848. doi: 10.3892/mmr.2020.10985. Epub 2020 Feb 12. PMID: 32319623; PMCID: PMC7057830.</td>
      <td>dGp,dGp,dCp,dCp,dTp,dGp,dGp,dTp,dCp,dTp,dGp,dTp,dTp,dAp,dGp,dCp,dCp,dGp,dGp,dGp,dTp,dAp,dGp,dCp,dAp,dGp,dCp,dCp,dCp,dCp,dGp,dGp,dCp,dAp,dCp,dCp,dTp,dAp,dTp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>99</th>
      <td>OSM (ADR58)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/84.png" width="120" ></td>
      <td>Human Oncostatin M (OSM)</td>
      <td>Protein</td>
      <td>7 nM (reported value)</td>
      <td>None</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>22572.51 g/mole</td>
      <td>68.57%</td>
      <td>651800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Can specifically antagonize OSM by binding to the gp130 receptor.</td>
      <td>70</td>
      <td>None</td>
      <td>rGp,rGp,rGp,rAp,rGp,rGp,rAp,fCp,rGp,rAp,fUp,rGp,fCp,rGp,rGp,rAp,fUp,fCp,rGp,fCp,fCp,fCp,fUp,rGp,rAp,rAp,fCp,fCp,rGp,rGp,fCp,fCp,fCp,rAp,rGp,fCp,rAp,rGp,rAp,fCp,fUp,rGp,fCp,fUp,rGp,rAp,fCp,rGp,rGp,fCp,rAp,fCp,rGp,rAp,fUp,fCp,fCp,rGp,fCp,rAp,fUp,fCp,rGp,fUp,fCp,fCp,fUp,fCp,fCp,fCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>100</th>
      <td>Apt69</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/646.jpg" width="120" ></td>
      <td>Thymidine Kinase 1</td>
      <td>Protein</td>
      <td>510 pM (reported value)</td>
      <td>PBS</td>
      <td>°C</td>
      <td>None</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>39</td>
      <td>Aptamer-Based Sandwich Assay for Measurement of Thymidine Kinase 1 in Serum of Cancerous Patients  Mahmood Nazari, Seyed Latif Mousavi Gargari, Abbas Sahebghadam Lotfi, Mohammad Javad Rassaee, and Ramezan Ali Taheri  Biochemistry 2019 58 (18), 2373-2383</td>
      <td>dGp,dCp,dGp,dTp,dGp,dCp,dAp,dGp,dCp,dAp,dTp,dGp,dAp,dCp,dCp,dAp,dCp,dCp,dGp,dAp,dTp,dGp,dTp,dCp,dAp,dCp,dAp,dCp,dTp,dTp,dAp,dAp,dTp,dGp,dAp,dGp,dAp,dAp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>101</th>
      <td>AP-1 cJun/cJun Aptamer-19</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/586.jpg" width="120" ></td>
      <td>Activator Protein-1 Family cJun/cJun Homodimer</td>
      <td>Protein</td>
      <td>0.5 nM (reported value)</td>
      <td>binding buffer (20 mM Tris (pH 7.9), 0.1 mM EDTA, 10% glycerol, 1 mM DTT, 150 mM KCl, 5 mM MgCl2, 0.04% NP-40)</td>
      <td>24°C</td>
      <td>None</td>
      <td>kD for counter-target cJun/cFos = 85 nM</td>
      <td>24206.74 g/mole</td>
      <td>53.85%</td>
      <td>762100 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>234</td>
      <td>Walters RD, McSwiggen DT, Goodrich JA, Kugel JF (2014) Selection and Characterization of a DNA Aptamer That Can Discriminate between cJun/cJun and cJun/cFos. PLoS ONE 9(6): e101015. doi:10.1371/journal.pone.0101015</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>102</th>
      <td>AptHis-C</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7189.png" width="120" ></td>
      <td>Hexahistidine</td>
      <td>Small Organic</td>
      <td>20.8 nM (reported value)</td>
      <td>1% BSA, 0.1% Tween-20, 0.2 mg/mL of tRNA, DPBS, pH 7.4.</td>
      <td>RT°C</td>
      <td>None</td>
      <td></td>
      <td>12223.9 g/mole</td>
      <td>56.41%</td>
      <td>384000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>39</td>
      <td>Wu, S.; Liu, Y.; Sun, H.; Zhong, M.; Dai, B.; Pan, B.; Shen, Z. An ssDNA aptamer specific for detection and purification of hexahistidine-tagged proteins. Anal. Biochem. 2020 Oct 15; 607: 113893. DOI: 10.1016/j.ab.2020.113893.</td>
      <td>dTp,dGp,dGp,dCp,dAp,dAp,dGp,dAp,dGp,dGp,dGp,dTp,dGp,dTp,dGp,dCp,dTp,dTp,dAp,dAp,dGp,dGp,dTp,dGp,dGp,dAp,dCp,dAp,dCp,dGp,dGp,dTp,dGp,dGp,dCp,dTp,dTp,dAp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>103</th>
      <td>HPV-HF cell line (14)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/316.png" width="120" ></td>
      <td>HPV-nontumorigenic HF cell line</td>
      <td>Cells</td>
      <td>7.1 nM (reported value)</td>
      <td>1% BSA in PBS with salts</td>
      <td>4°C</td>
      <td>None</td>
      <td></td>
      <td>27814.14 g/mole</td>
      <td>48.31%</td>
      <td>893700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Aptamer able to penetrate cell surface and internalized in cell cytoplasm independent of protein binding.</td>
      <td>98</td>
      <td>Graham JC, Zarbl H (2012) Use of Cell-SELEX to Generate DNA Aptamers as Molecular Probes of HPV-Associated Cervical Cancer Cells. PLoS ONE 7(4):  e36103. doi:10.1371/journal.pone.0036103</td>
      <td>dAp,dTp,dAp,dCp,dCp,dAp,dGp,dCp,dTp,dTp,dAp,dTp,dTp,dCp,dAp,dAp,dTp,dTp,dGp,dGp,dGp,dCp,dGp,dGp,dGp,dGp,dAp,dGp,dTp,dAp,dGp,dGp,dGp,dAp,dGp,dAp,dGp,dGp,dGp,dGp,dTp,dTp,dTp,dCp,dCp,dAp,dTp,dCp,dGp,dGp,dCp,dGp,dAp,dCp,dAp,dGp,dAp,dGp,dGp,dAp,dGp,dTp,dTp,dAp,dTp,dGp,dTp,dGp,dTp,dGp,dTp,dAp,dGp,dAp,dTp,dAp,dGp,dTp,dAp,dAp,dGp,dTp,dGp,dCp,dAp,dAp,dTp,dCp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>104</th>
      <td>HER-2 (S6)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/284.png" width="120" ></td>
      <td>HER-2 (SK-BR-3 cell line)</td>
      <td>Cells</td>
      <td>94.6 nM (reported value)</td>
      <td>None</td>
      <td>4°C</td>
      <td>None</td>
      <td>None</td>
      <td>31658 g/mole</td>
      <td>51.02%</td>
      <td>976700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>98</td>
      <td>Kang, H. (2009). Isolation of RNA aptamers targeting Her-2-overexpressing breast cancer cells using cell-selex. Bull. Korean Chem. Soc., 30(8), 182-1831.</td>
      <td>rGp,rGp,rGp,rAp,rGp,rAp,rUp,rAp,rCp,rCp,rAp,rGp,rCp,rUp,rUp,rAp,rUp,rUp,rCp,rAp,rAp,rUp,rUp,rUp,rGp,rGp,rAp,rUp,rGp,rGp,rGp,rGp,rAp,rGp,rAp,rUp,rCp,rCp,rGp,rUp,rUp,rGp,rAp,rGp,rUp,rAp,rAp,rGp,rCp,rGp,rGp,rGp,rCp,rGp,rUp,rGp,rUp,rCp,rUp,rCp,rUp,rCp,rUp,rGp,rCp,rCp,rGp,rCp,rCp,rUp,rUp,rGp,rCp,rUp,rAp,rUp,rGp,rGp,rGp,rGp,rAp,rGp,rAp,rUp,rAp,rGp,rUp,rAp,rAp,rGp,rUp,rGp,rCp,rAp,rAp,rUp,rCp,rUp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>105</th>
      <td>ADP</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/262.png" width="120" ></td>
      <td>ADP</td>
      <td>Small Organic</td>
      <td></td>
      <td>Selection Buffer ((50 mM HEPES [pH 7.4], 25 mM MgCl2, 150 mM NaCl) containing 10 g/ml tRNA)</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>27540.71 g/mole</td>
      <td>58.82%</td>
      <td>837700 L/(mole·cm)</td>
      <td></td>
      <td>5-Jan µM (reported value)</td>
      <td>Contains a hammerhead ribozyme motif.</td>
      <td>85</td>
      <td>Srinivasan et al. "ADP-Specific Sensors Enable Universal Assay of Protein Kinase Activity." Chemistry & Biology, 11(2004): 499-508.</td>
      <td>rGp,rGp,rGp,rCp,rGp,rAp,rCp,rCp,rCp,rUp,rGp,rAp,rUp,rGp,rAp,rGp,rCp,rAp,rCp,rAp,rCp,rGp,rAp,rGp,rGp,rGp,rGp,rGp,rAp,rAp,rAp,rCp,rCp,rCp,rCp,rGp,rGp,rAp,rCp,rAp,rAp,rUp,rCp,rAp,rGp,rAp,rCp,rAp,rCp,rGp,rGp,rUp,rGp,rUp,rUp,rCp,rGp,rAp,rAp,rAp,rCp,rGp,rGp,rUp,rGp,rAp,rAp,rAp,rGp,rCp,rCp,rGp,rUp,rAp,rGp,rGp,rUp,rUp,rGp,rCp,rCp,rCp,rUp,rUp,rUp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>106</th>
      <td>Brain Vasculature</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/221.png" width="120" ></td>
      <td>Brain Vasculature</td>
      <td>Tissue</td>
      <td>N/A nM (reported value)</td>
      <td>None</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>1049.27 g/mole</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>9</td>
      <td>None</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>107</th>
      <td>von Willebrand Factor (VWF R9.3)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/179.png" width="120" ></td>
      <td>von Willebrand Factor (VWF)</td>
      <td>Protein</td>
      <td>1.2 nM (reported value)</td>
      <td>None</td>
      <td>37°C</td>
      <td>None</td>
      <td>Specifically inhibited ristocetin-induced platelet aggregation and not collagen-induced, ADP-induced, or thrombin-induced platelet aggregation.</td>
      <td>24492.61 g/mole</td>
      <td>57.89%</td>
      <td>731800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Binds to the A1 domain of vWF and completely inhibits ristocetin-induced platelet aggregation at a concentration of 250 nM.</td>
      <td>82</td>
      <td>None</td>
      <td>rGp,rGp,rGp,rAp,rGp,rGp,rAp,fCp,rGp,rAp,fUp,rGp,fCp,rGp,rGp,rAp,fUp,fCp,rGp,fCp,rGp,fCp,fUp,fCp,fUp,fCp,fCp,fUp,rGp,fCp,fUp,fUp,rAp,rAp,rGp,fCp,rAp,rGp,fCp,fUp,rAp,fUp,fCp,rAp,rAp,rAp,fUp,rAp,rGp,fCp,fCp,fCp,rAp,fCp,fUp,fCp,rAp,rGp,rAp,fCp,rGp,rAp,fCp,fUp,fCp,rGp,fCp,fUp,rGp,rAp,rGp,rGp,rAp,fUp,fCp,fCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>108</th>
      <td>VEGF (SL2-B aptamer)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/303.png" width="120" ></td>
      <td>VEGF (165)</td>
      <td>Protein</td>
      <td>0.5 nM (reported value)</td>
      <td>PBS and 0.005% tween-20 solution mixture was used as the running buffer, and 50 mM NaOH as the regeneration buffer.</td>
      <td>25°C</td>
      <td>None</td>
      <td></td>
      <td>8034.25 g/mole</td>
      <td>61.54%</td>
      <td>242600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>26</td>
      <td>Kaur H, Yung L-YL (2012) Probing High Affinity Sequences of DNA Aptamer against VEGF165. PLoS ONE 7(2): e31196. doi:10.1371/journal.pone.0031196</td>
      <td>dCp,dAp,dAp,dTp,dTp,dGp,dGp,dGp,dCp,dCp,dCp,dGp,dTp,dCp,dCp,dGp,dTp,dAp,dTp,dGp,dGp,dTp,dGp,dGp,dGp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>109</th>
      <td>M4-16</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7224.png" width="120" ></td>
      <td>Pb2</td>
      <td>Other</td>
      <td>nM (reported value)</td>
      <td>15 μL of NaCl solution (0.5 mM), 30 μL of aptamer (1 μM)</td>
      <td>25°C</td>
      <td>\n                                N/A Structure could not be determined \n</td>
      <td>Highly specific</td>
      <td>5123.31 g/mole</td>
      <td>75.00%</td>
      <td>166900 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>This aptamer is a Pb^2 binding aptamer that is a mutant variation of T30695. The other mutant variants are designated as M4, and M16. Most of its binding ability was determined in silico. The calculated binding free energy of this aptamer to Pb^2 is −1417.17−kj /mol.</td>
      <td>16</td>
      <td>https://doi.org/10.1016/j.molliq.2019.111159.\n</td>
      <td>dGp,dGp,dGp,dAp,dGp,dGp,dGp,dTp,dGp,dGp,dGp,dTp,dGp,dGp,dGp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>110</th>
      <td>1501s malaria aptamer</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7081.png" width="120" ></td>
      <td>PvLDH (Malaria)</td>
      <td>Protein</td>
      <td>670 nM (reported value)</td>
      <td>Not described</td>
      <td>25°C</td>
      <td>\n                                Not described \n</td>
      <td>Specific for PvLDH over PfLDH</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Modified with cubane functionalized thymines</td>
      <td>35</td>
      <td>Cheung, Y.-W., Röthlisberger, P., Mechaly, A. E., Weber, P., Levi-Acobas, F., Lo, Y., … Tanner, J. A. (2020). Evolution of abiotic cubane chemistries in a nucleic acid aptamer allows selective recognition of a malaria biomarker. Proceedings of the National Academy of Sciences, 202003267. doi:10.1073/pnas.2003267117</td>
      <td>dGp,dGp,dTp,dAp,dTp,dAp,dGp,dAp,dCp,dCp,dCp,dCp,dTp,dGp,dAp,dGp,dTp,dCp,dCp,dTp,dAp,dCp,dCp,dGp,dAp,dGp,dGp,dGp,dCp,dAp,dCp,dGp,dGp,dCp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>111</th>
      <td>Anti-acetylcholine Autoantibodies (SE RNA)</td>
      <td>2'-NH2-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/133.jpg" width="120" ></td>
      <td>Anti-acetylcholine Autoantibodies</td>
      <td>Protein</td>
      <td>60 nM (reported value)</td>
      <td>Binding Buffer (30 mM Tris-HCl (pH 7.5), 150 mM NaCl, 10 mM MgCl2, 2 mM DTT, 1% BSA)</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>28339.52 g/mole</td>
      <td>68.75%</td>
      <td>455500 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>48</td>
      <td>Lee and Sullenger. "Isolation of a nuclease-resistant decoy RNA that can protect human acetylcholine receptors from myasthenic antibodies." Nature Biotechnology, 15(1997):</td>
      <td>rGp,rGp,rGp,rGp,rGp,rAp,rGp,rGp,rAp,rGp,rGp,rAp,rGp,rGp,rAp,rAp,rGp,rAp,rGp,rGp,rGp,rGp,rAp,rGp,rGp,rAp,rAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>112</th>
      <td>Hepatitis B Virus (HBV) Polymerase (P protein) (A9)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/253.png" width="120" ></td>
      <td>Hepatitis B Virus (HBV) Polymerase (P protein)</td>
      <td>Protein</td>
      <td>N/A nM (reported value)</td>
      <td>binding buffer (0.1 M sodium phosphate (pH 7.4), 150 mM NaCl, 20 mM imidazol, 0.1% (v/v) NP-40, 100 mg/ml yeast tRNA), 30 °C</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>19585.92 g/mole</td>
      <td>36.07%</td>
      <td>626700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>63</td>
      <td>Feng et al. "A SELEX-Screened Aptamer of Human Hepatitis B Virus  RNA Encapsidation Signal Suppresses Viral Replication." PLoS One, 6(2011): e27862</td>
      <td>rUp,rGp,rUp,rUp,rCp,rAp,rUp,rGp,rUp,rCp,rCp,rUp,rAp,rCp,rUp,rGp,rUp,rUp,rCp,rAp,rAp,rAp,rCp,rAp,rAp,rAp,rAp,rAp,rAp,rAp,rCp,rUp,rGp,rUp,rGp,rCp,rAp,rCp,rAp,rAp,rAp,rAp,rAp,rUp,rAp,rAp,rAp,rUp,rUp,rGp,rGp,rGp,rGp,rCp,rAp,rUp,rGp,rGp,rAp,rCp,rAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>113</th>
      <td>MT1-MMP (swiggle)</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/476.png" width="120" ></td>
      <td>Membrane-type 1 matrix metalloptroteinase (MT1-MMP)</td>
      <td>Protein</td>
      <td>N/A nM (reported value)</td>
      <td>N/A</td>
      <td>N/A°C</td>
      <td>\n                                N/A \n</td>
      <td>Specific to MT1-MMP over MT2-MMP, MT3-MMP, MT5-MMP,and Cdk4.</td>
      <td>1237.57 g/mole</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>10</td>
      <td>None</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>114</th>
      <td>Streptomycin (LGA11)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/432.png" width="120" ></td>
      <td>Streptomycin and other Aminoglycosides (AMGs)</td>
      <td>Small Organic</td>
      <td>49.9 nM (reported value)</td>
      <td>Binding Buffer: PBS with 10% (v/v) methanol</td>
      <td>RT°C</td>
      <td>None</td>
      <td>Binds to several aminoglycosides: kanamycin, paromycin streptomycin, apramycin, tobramycin, neomycin, and gentamycin.</td>
      <td>25568.23 g/mole</td>
      <td>56.96%</td>
      <td>771400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>87</td>
      <td>None</td>
      <td>rGp,rGp,rGp,rAp,rGp,rAp,rAp,rGp,rGp,fCp,rGp,rGp,fCp,rGp,fCp,rGp,fUp,rAp,rGp,rGp,fCp,rGp,rAp,rGp,fCp,fUp,fUp,fUp,rAp,rGp,rGp,fCp,fCp,fCp,rGp,rAp,fCp,rAp,fUp,fUp,fCp,fCp,fCp,fCp,fUp,rAp,rAp,rAp,rAp,rAp,rAp,rGp,fCp,fUp,fUp,rGp,fUp,fUp,fCp,fUp,rAp,fUp,fUp,fCp,rGp,fUp,rGp,rGp,rAp,fCp,rGp,rAp,fUp,rGp,fCp,rGp,fCp,rAp,fCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>115</th>
      <td>S-adenosyl homocysteine (CTH-5)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/106.png" width="120" ></td>
      <td>S-adenosyl homocysteine</td>
      <td>Small Organic</td>
      <td>0.1 nM (reported value)</td>
      <td>RNA Buffer (300 mM NaCl, 25 mM Tris (pH 7.6), 5 mM MgCl2)   HBS Buffer (10 mM HEPES (pH 7.4), 150 mM NaCl, 3.4 mM EDTA, and 0.005% P-20)  Carbonate Buffer (0.05 M NaHCO3 0.125 M NaCl (pH 8.3))</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>13669.26 g/mole</td>
      <td>64.29%</td>
      <td>410600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>42</td>
      <td>Gebhardt, K., et al. "RNA Aptamers to S-Adenosylhomocysteine: Kinetic Properties, Divalent Cation  Dependency, and Comparison with Anti-S-adenosylhomocysteine Antibody." Biochemistry, 39 (2000): 7255-7265.</td>
      <td>rGp,rGp,rGp,rCp,rGp,rGp,rAp,rUp,rGp,rAp,rGp,rAp,rCp,rGp,rCp,rUp,rUp,rGp,rGp,rCp,rGp,rUp,rGp,rUp,rGp,rCp,rUp,rGp,rUp,rGp,rGp,rAp,rGp,rAp,rGp,rUp,rCp,rAp,rUp,rCp,rCp,rGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>116</th>
      <td>Paramylon-binding Fluorogenic Peptide</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/643.png" width="120" ></td>
      <td>Paramylon in E. gracilis</td>
      <td>Protein</td>
      <td>Oct-60 µM (reported value)</td>
      <td>AF-6 medium , mannitol (0.3 M)</td>
      <td>37°C</td>
      <td>\n                                N/A \n</td>
      <td><10 fold against Kuzuko starch</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Peptide Aptamer Sequence: Met-Thr-Thr-Cys-Ala-Arg-Arg-Phe-Arg-Ukn-Ile-Gly-Trp-Arg-Phe-Cys-Ser-Trp    Identified by ribosome display, synthesized by solid-phase chemistry</td>
      <td>54</td>
      <td>1. Takanori Maeno, Takanori Uzawa, Izumi Kono, Kazunori Okano, Takanori Iino, Keisuke Fukita, Yuki Oshikawa, Taro Ogawa, Osamu Iwata, Takuro Ito, Kengo Suzuki, Keisuke Goda, Yoichiroh Hosokawa. Targeted delivery of fluorogenic peptide aptamers into live microalgae by femtosecond laser photoporation at single-cell resolution. Scientific Reports, 2018; 8 (1) DOI: 10.1038/s41598-018-26565-4</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>117</th>
      <td>Avian Myeloblastosis Virus (a.1.1)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/173.png" width="120" ></td>
      <td>Avian Myeloblastosis Virus (AMV) RT</td>
      <td>Protein</td>
      <td>0.5 nM (reported value)</td>
      <td>Binding Buffer (50 mM Tris-HC1 (pH 7.7), 200 mM potassium acetate, and 10 mM DTT)</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>13704.24 g/mole</td>
      <td>55.81%</td>
      <td>408400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>43</td>
      <td>Chen and Gold. "Selection of High-Affinity RNA Ligands to Reverse Transcriptase: Inhibition of  cDNA Synthesis and RNase H Activity." Journal of Biochemistry, 33(1994): 8746-8756.</td>
      <td>rCp,rGp,rUp,rCp,rCp,rCp,rGp,rUp,rGp,rCp,rGp,rCp,rAp,rAp,rAp,rAp,rGp,rUp,rUp,rCp,rUp,rUp,rAp,rGp,rCp,rGp,rCp,rUp,rAp,rGp,rCp,rAp,rGp,rUp,rCp,rCp,rUp,rAp,rGp,rUp,rUp,rGp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>118</th>
      <td>Thrombin Aptamer 2</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7217.png" width="120" ></td>
      <td>Thrombin</td>
      <td>Protein</td>
      <td>0.027 nM (reported value)</td>
      <td>TE (20 mM Tris-HCl, 140 mM NaCl, 5 mM \nKCl, 5 mM MgCl2 and 1mM CaCl2, pH 7.4)</td>
      <td>25°C</td>
      <td>None</td>
      <td></td>
      <td>15448.88 g/mole</td>
      <td>36.73%</td>
      <td>435546 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>5'SH-(CH2)6-TTTTTTTTTTTTTTTTTTTTAGTCCGTGGTAGGGCAGGTTGGGGTGACT '3</td>
      <td>147</td>
      <td>doi:10.1039/C7TB01276D</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>119</th>
      <td>Biotin</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/152.jpg" width="120" ></td>
      <td>Biotin</td>
      <td>Small Organic</td>
      <td>5.7 µM (reported value)</td>
      <td>Selection Buffer (0.1 M KCl, 5 mM MgCl2, and 10 mM Na-HEPES (pH 7.4))</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>35326.27 g/mole</td>
      <td>51.82%</td>
      <td>1080100 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>110</td>
      <td>Wilson et al. "Functional Requirements for Specific Ligand Recognition by a Biotin-Binding RNA Pseudoknot." Journal of Biochemistry, 37(1998): 14410-14419.</td>
      <td>rGp,rGp,rAp,rAp,rCp,rAp,rCp,rUp,rAp,rUp,rCp,rCp,rGp,rAp,rUp,rGp,rGp,rCp,rAp,rCp,rCp,rGp,rAp,rCp,rCp,rAp,rUp,rAp,rGp,rGp,rCp,rUp,rCp,rGp,rGp,rGp,rUp,rUp,rGp,rCp,rCp,rAp,rGp,rAp,rGp,rGp,rUp,rUp,rCp,rCp,rAp,rCp,rAp,rCp,rUp,rUp,rUp,rCp,rAp,rUp,rCp,rGp,rAp,rAp,rAp,rAp,rGp,rCp,rCp,rUp,rAp,rUp,rGp,rCp,rUp,rAp,rGp,rGp,rCp,rAp,rAp,rUp,rGp,rAp,rCp,rAp,rUp,rGp,rGp,rAp,rCp,rUp,rCp,rCp,rUp,rUp,rGp,rGp,rUp,rCp,rAp,rUp,rUp,rAp,rGp,rGp,rAp,rUp,rCp,rGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>120</th>
      <td>SL1025</td>
      <td>Chimeric</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/621.png" width="120" ></td>
      <td>Interleukin 6 (IL-6), cytokine</td>
      <td>Protein</td>
      <td>0.2 nM (reported value)</td>
      <td>PBS with 1%BSA, 0.05% Tween 20, 5 mM MgCl2</td>
      <td>25°C</td>
      <td>None</td>
      <td></td>
      <td>6908.54 g/mole</td>
      <td>68.18%</td>
      <td>227300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>[dGp][dGp][mCp][dAp][dGp][mGp][Bn-dUp][Bn-dUp][Pe-dUp][dGp][dGp][Nap-dUp][dAp][Bn-dUp][Bn-dUp][mAp][dAp][dCp][mAp][mCp][dGp][Bn-dUp][Bn-dUp][dAp][dAp][dGp][Bn-dUp][mCp][dGp][Bn-dUp][dGp][dGp]    Refer to Figure in the publication and the visualization above for the sequence of the aptamer. Due to complex modifications to the sequence proposed by the publication authors, the structure of the aptamer cannot be estimated.</td>
      <td>66</td>
      <td>Gupta, Shashi, et al. "Chemically modified DNA aptamers bind interleukin-6 with high affinity and inhibit signaling by blocking its interaction with interleukin-6 receptor." Journal of Biological Chemistry 289.12 (2014): 8706-8719.</td>
      <td>dGp,dGp,dAp,dGp,dUp,dUp,dUp,dGp,dGp,dUp,dAp,dUp,dUp,dAp,dCp,dGp,dUp,dUp,dAp,dAp,dGp,dUp,dGp,dUp,dGp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>121</th>
      <td>Kanamycin A (#13_82)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/498.png" width="120" ></td>
      <td>Kanamycin A</td>
      <td>Small Organic</td>
      <td>5.1 nM (reported value)</td>
      <td>None</td>
      <td>21°C</td>
      <td>\n                                90°C for 8 min, immediately cooled, 4 °C for 10 min. \n</td>
      <td></td>
      <td>30429.82 g/mole</td>
      <td>47.96%</td>
      <td>961600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Lowercase letters denote constant regions. This aptamer was selected using capture-SELEX in which the DNA was immoblized using DNA complimentary to the middle constant region and was eluted by the target during the selection process.</td>
      <td>98</td>
      <td>None</td>
      <td>dAp,dTp,dAp,dCp,dCp,dAp,dGp,dCp,dTp,dTp,dAp,dTp,dTp,dCp,dAp,dAp,dTp,dTp,dCp,dAp,dGp,dGp,dGp,dCp,dGp,dGp,dTp,dAp,dTp,dGp,dAp,dGp,dGp,dCp,dTp,dCp,dGp,dAp,dTp,dCp,dAp,dAp,dGp,dGp,dTp,dCp,dGp,dGp,dAp,dGp,dCp,dGp,dAp,dGp,dAp,dAp,dTp,dTp,dTp,dTp,dTp,dTp,dCp,dGp,dCp,dGp,dGp,dAp,dGp,dTp,dCp,dGp,dGp,dCp,dTp,dGp,dGp,dAp,dTp,dCp,dAp,dGp,dAp,dTp,dAp,dGp,dTp,dAp,dAp,dGp,dTp,dGp,dCp,dAp,dAp,dTp,dCp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>122</th>
      <td>Apramycin (LS13)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/427.png" width="120" ></td>
      <td>Apramycin and other aminolycosides (AMGs)</td>
      <td>Protein</td>
      <td>35.9 nM (reported value)</td>
      <td>Binding Buffer: PBS with 10% (v/v) methanol</td>
      <td>RT°C</td>
      <td>None</td>
      <td>Binds to several aminoglycosides: kanamycin, paromycin streptomycin, apramycin, tobramycin, neomycin, and gentamycin.</td>
      <td>25474.89 g/mole</td>
      <td>46.84%</td>
      <td>788000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>87</td>
      <td>None</td>
      <td>rGp,rGp,rGp,rAp,rGp,rAp,rAp,rGp,rGp,fCp,rGp,rGp,fCp,rGp,fCp,rGp,fUp,rAp,rGp,rGp,fCp,rGp,rAp,rGp,fCp,fUp,fUp,fUp,rAp,fCp,fCp,rAp,rGp,fUp,fUp,fUp,fUp,rAp,fUp,fUp,fUp,rGp,fUp,fUp,fUp,fUp,rAp,fUp,fUp,rGp,fUp,fUp,rAp,fUp,rAp,fUp,rGp,fCp,fUp,fUp,rAp,fUp,fUp,fCp,rGp,fUp,rGp,rGp,rAp,fCp,rGp,rAp,fUp,rGp,fCp,rGp,fCp,rAp,fCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>123</th>
      <td>Mouse Pancreas Vasculature</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/245.png" width="120" ></td>
      <td>Mouse Pancreas Vasculature</td>
      <td>Tissue</td>
      <td>N/A nM (reported value)</td>
      <td>None</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>1205.44 g/mole</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>9</td>
      <td>D Rajotte et al. "  Molecular Heterogeneity of the Vascular Endothelium Revealed by In Vivo  Phage Display." Journal of Clinical Investigations, 102(1998): 430-437.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>124</th>
      <td>Phospholamban (Apt-9)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/188.png" width="120" ></td>
      <td>Phospholamban</td>
      <td>Protein</td>
      <td>20 nM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td></td>
      <td>12546.21 g/mole</td>
      <td>55.00%</td>
      <td>400500 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>40</td>
      <td>None</td>
      <td>dTp,dTp,dGp,dGp,dGp,dGp,dAp,dGp,dGp,dGp,dGp,dCp,dAp,dCp,dTp,dGp,dGp,dGp,dCp,dAp,dGp,dTp,dGp,dTp,dAp,dAp,dTp,dTp,dTp,dAp,dCp,dGp,dAp,dAp,dAp,dGp,dCp,dGp,dAp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>125</th>
      <td>Neomycin (LS13)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/436.png" width="120" ></td>
      <td>Neomycin and other Aminoglycosides (AMGs)</td>
      <td>Small Organic</td>
      <td>37 nM (reported value)</td>
      <td>Binding Buffer: PBS with 10% (v/v) methanol</td>
      <td>RT°C</td>
      <td>None</td>
      <td>Binds to several aminoglycosides: kanamycin, paromycin streptomycin, apramycin, tobramycin, neomycin, and gentamycin.</td>
      <td>25474.89 g/mole</td>
      <td>46.84%</td>
      <td>788000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>87</td>
      <td>None</td>
      <td>rGp,rGp,rGp,rAp,rGp,rAp,rAp,rGp,rGp,fCp,rGp,rGp,fCp,rGp,fCp,rGp,fUp,rAp,rGp,rGp,fCp,rGp,rAp,rGp,fCp,fUp,fUp,fUp,rAp,fCp,fCp,rAp,rGp,fUp,fUp,fUp,fUp,rAp,fUp,fUp,fUp,rGp,fUp,fUp,fUp,fUp,rAp,fUp,fUp,rGp,fUp,fUp,rAp,fUp,rAp,fUp,rGp,fCp,fUp,fUp,rAp,fUp,fUp,fCp,rGp,fUp,rGp,rGp,rAp,fCp,rGp,rAp,fUp,rGp,fCp,rGp,fCp,rAp,fCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>126</th>
      <td>Mycoplasma bovis detection aptamer (WKB-14)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/544.jpg" width="120" ></td>
      <td>P48 of M. Bovis</td>
      <td>Protein</td>
      <td>15.61 nM (reported value)</td>
      <td>Added to wells of plate coated with P48 protein solution (P48 protein dissolved in 100 uL of 0.05 carbonate-bicarbonate buffer)</td>
      <td>37°C</td>
      <td>None</td>
      <td>Relatively similar to mAB 10E to P48</td>
      <td>24687.08 g/mole</td>
      <td>48.75%</td>
      <td>769800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>240</td>
      <td>Fu, P., et. al. Enzyme linked aptamer assay: based on a competition format for sensitive detection of antibodies to Mycoplasma bovis in serum.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>127</th>
      <td>StrepApt5</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/551.jpg" width="120" ></td>
      <td>Streptavidin</td>
      <td>Protein</td>
      <td>35 nM (reported value)</td>
      <td>NaCl (100 mM), MgCl2 (2 mM), KCl (5 mM), CaCl2 (1 mM), Tris-HCl (pH 7.6, 20 mM)</td>
      <td>°C</td>
      <td>\n                                heated to 95C for 8 minutes, cool on ice for 10 minutes and left at room temperature for at  least 10 minutes. \n</td>
      <td></td>
      <td>7957.19 g/mole</td>
      <td>65.38%</td>
      <td>242700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Ka = 1.04 x 10^4   Kd = 0.36 x 10-3    Best KD with lowest Kd.</td>
      <td>78</td>
      <td>Ruigrok, Vincent J. B., et al. "Kinetic and Stoichiometric Characterisation of Streptavidin-Binding Aptamers." ChemBioChem 13 (2012): 829-836. Print.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>128</th>
      <td>Brain Penetrating Aptamer (A15)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/475.png" width="120" ></td>
      <td>Brain capillary endothelia and parenchyma</td>
      <td>Tissue</td>
      <td>N/A nM (reported value)</td>
      <td>None</td>
      <td>37°C</td>
      <td>\n                                Denatured at 94°C for 3 min then slowly cooled to room temperature. \n</td>
      <td></td>
      <td>22979.59 g/mole</td>
      <td>59.15%</td>
      <td>690300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>This aptamer was generated from in vivo SELEX and was shown to enter brain endothelial cells in vitro and brain parenchyma in vivo.</td>
      <td>72</td>
      <td>None</td>
      <td>rGp,rGp,rGp,rAp,rGp,rGp,rAp,fCp,rGp,rAp,fUp,rGp,fCp,rGp,rGp,fCp,rGp,fUp,rAp,fUp,fUp,rGp,fCp,rGp,fCp,rGp,rAp,rGp,rGp,rAp,fUp,fUp,rAp,fUp,fCp,fCp,rGp,fCp,fUp,fCp,rAp,fUp,fCp,rGp,fUp,fUp,rGp,fUp,fUp,rGp,fUp,fUp,rGp,fUp,rGp,fCp,rAp,rGp,rAp,fCp,rGp,rAp,fCp,fUp,fCp,rGp,fCp,fCp,fCp,rGp,rAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>129</th>
      <td>Arsenic(III) (Ars-3)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/460.png" width="120" ></td>
      <td>Arsenic(III)</td>
      <td>Small Organic</td>
      <td>4.95 nM (reported value)</td>
      <td>None</td>
      <td>25°C</td>
      <td>None</td>
      <td></td>
      <td>30767.08 g/mole</td>
      <td>42.00%</td>
      <td>981200 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>100</td>
      <td>None</td>
      <td>dGp,dGp,dTp,dAp,dAp,dTp,dAp,dCp,dGp,dAp,dCp,dTp,dCp,dAp,dCp,dTp,dAp,dTp,dAp,dGp,dGp,dGp,dAp,dGp,dAp,dTp,dAp,dCp,dCp,dAp,dGp,dCp,dTp,dTp,dAp,dTp,dTp,dCp,dAp,dAp,dTp,dTp,dTp,dTp,dAp,dCp,dAp,dGp,dAp,dAp,dCp,dAp,dAp,dCp,dCp,dAp,dAp,dCp,dGp,dTp,dCp,dGp,dCp,dTp,dCp,dCp,dGp,dGp,dGp,dTp,dAp,dCp,dTp,dTp,dCp,dTp,dTp,dCp,dAp,dTp,dCp,dGp,dAp,dGp,dAp,dTp,dAp,dGp,dTp,dAp,dAp,dGp,dTp,dGp,dCp,dAp,dAp,dTp,dCp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>130</th>
      <td>G-NB3</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7170.jpg" width="120" ></td>
      <td>Activated Protein C</td>
      <td>Protein</td>
      <td>0.17 nM (reported value)</td>
      <td>1X PBS</td>
      <td>22°C</td>
      <td>None</td>
      <td></td>
      <td>25630.6 g/mole</td>
      <td>52.44%</td>
      <td>807400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>82</td>
      <td>Hamedani, N. S., Müller, J., Tolle, F., Rühl, H., Pezeshkpoor, B., Liphardt, K., … Pötzsch, B. (2020). Selective Modulation of the Protease Activated Protein C Using Exosite Inhibiting Aptamers. Nucleic Acid Therapeutics. doi:10.1089/nat.2020.0844</td>
      <td>dGp,dAp,dTp,dTp,dGp,dTp,dTp,dAp,dCp,dTp,dGp,dTp,dCp,dAp,dCp,dGp,dAp,dGp,dGp,dAp,dTp,dTp,dGp,dGp,dGp,dGp,dGp,dTp,dTp,dGp,dGp,dGp,dTp,dGp,dGp,dAp,dTp,dAp,dGp,dGp,dCp,dTp,dGp,dGp,dCp,dGp,dTp,dCp,dGp,dGp,dGp,dGp,dCp,dAp,dGp,dGp,dTp,dCp,dAp,dGp,dTp,dAp,dTp,dAp,dGp,dCp,dAp,dCp,dAp,dTp,dTp,dAp,dGp,dTp,dTp,dCp,dAp,dGp,dAp,dTp,dAp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>131</th>
      <td>LPS (B2)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/301.png" width="120" ></td>
      <td>Lipopolysaccharide (LPS)</td>
      <td>Small Organic</td>
      <td>12 nM (reported value)</td>
      <td>Selection buffer: (50 mM Tris-acetate, pH 8.2)</td>
      <td>15°C</td>
      <td>None</td>
      <td></td>
      <td>26426.08 g/mole</td>
      <td>61.63%</td>
      <td>803400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>86</td>
      <td>Kim, S. (2012). Harnessing aptamers for electrochemical detection of endotoxin. Analytical Biochemistry ,424(2012), 12-20. doi: 10.1016/j.ab.2012.02.016</td>
      <td>dCp,dTp,dTp,dCp,dTp,dGp,dCp,dCp,dCp,dGp,dCp,dCp,dTp,dCp,dCp,dTp,dTp,dCp,dCp,dTp,dAp,dGp,dCp,dCp,dGp,dGp,dAp,dTp,dCp,dGp,dCp,dGp,dCp,dTp,dGp,dGp,dCp,dCp,dAp,dGp,dAp,dTp,dGp,dAp,dTp,dAp,dTp,dAp,dAp,dAp,dGp,dGp,dGp,dTp,dCp,dAp,dGp,dCp,dCp,dCp,dCp,dCp,dCp,dAp,dGp,dGp,dAp,dGp,dAp,dCp,dGp,dAp,dGp,dAp,dTp,dAp,dGp,dGp,dCp,dGp,dGp,dAp,dCp,dAp,dCp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>132</th>
      <td>Human Wild-type PAI-1 (WT-15)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/228.png" width="120" ></td>
      <td>Human Wild-type PAI-1</td>
      <td>Protein</td>
      <td>177 pM (reported value)</td>
      <td>None</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>24575.68 g/mole</td>
      <td>53.95%</td>
      <td>755600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>76</td>
      <td>Blake et al. "Antimetastatic Potential of PAI-1-Specific RNA Aptamers." Oligonucleotides, 19(2009): 117-128.</td>
      <td>rGp,rGp,rGp,rAp,rGp,rGp,rAp,fCp,rGp,rAp,fUp,rGp,fCp,rGp,rGp,rAp,fUp,fCp,rAp,rAp,fCp,fUp,fCp,rAp,fCp,fCp,rGp,fUp,rAp,rGp,rGp,fUp,fCp,fUp,rAp,rGp,fUp,rGp,rAp,rGp,rAp,rAp,fCp,fUp,fUp,fCp,rAp,rAp,rGp,fUp,fCp,fUp,rAp,fCp,fUp,fCp,rAp,rGp,rAp,fCp,rGp,rAp,fCp,fUp,fCp,rGp,fCp,fUp,rGp,rAp,rGp,rGp,rAp,fUp,fCp,fCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>133</th>
      <td>Osteopontin (2-O-Me OPN-R3)</td>
      <td>2'-O-Me-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/371.png" width="120" ></td>
      <td>Human Osteopontin</td>
      <td>Protein</td>
      <td>18 nM (reported value)</td>
      <td>None</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>13079.35 g/mole</td>
      <td>45.00%</td>
      <td>402900 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>41</td>
      <td>None</td>
      <td>rGp,rGp,rAp,rAp,rGp,rAp,rAp,rGp,rAp,rAp,rAp,rAp,rAp,rAp,rGp,rAp,rGp,rGp,rAp,rAp,rGp,rGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>134</th>
      <td>Tyrosine-Cp*Rh(III) sensor</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7106.jpg" width="120" ></td>
      <td>Tyrosine</td>
      <td>Small Organic</td>
      <td>2.9 nM (reported value)</td>
      <td>100 μM concentration of monomeric Cp*Rh(III)</td>
      <td>24 °C°C</td>
      <td>\n                                Mix and incubate for 5 min at 95oC, and cool down at room temperature > 30 min \n</td>
      <td>N/A</td>
      <td>11616.53 g/mole</td>
      <td>65.79%</td>
      <td>380810 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>38</td>
      <td>Yang KA, Barbu M, Halim M, et al. Recognition and sensing of low-epitope targets via ternary complexes with oligonucleotides and synthetic receptors. Nat Chem. 2014;6(11):1003-1008. doi:10.1038/nchem.2058</td>
      <td>dCp,dTp,dCp,dTp,dCp,dGp,dGp,dGp,dAp,dCp,dGp,dAp,dCp,dGp,dGp,dCp,dCp,dCp,dGp,dAp,dTp,dCp,dTp,dCp,dAp,dGp,dAp,dGp,dTp,dAp,dGp,dTp,dCp,dGp,dTp,dCp,dCp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>135</th>
      <td>Leptin (Lep3)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/500.png" width="120" ></td>
      <td>Leptin</td>
      <td>Protein</td>
      <td>0.32 µM (reported value)</td>
      <td>TGK 1X: 25 mM Tris, 192 mM glycine, and 5 mM potassium phosphate (pH 8.3)</td>
      <td>20°C</td>
      <td>\n                                Heated to 94°C and cooled at a rate of 0.5°C/s to 20°C. \n</td>
      <td>Showed ~10 fold discrimination between β-lactoglobulin (Kd=2.3 μM) and bovine catalase (Kd=3.1 μM).</td>
      <td>24925.14 g/mole</td>
      <td>56.79%</td>
      <td>744800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>This aptamer was selected using nonequilibrium capillary electrophoresis of equilibrium mixtures (NECEEM). Binding affinity against immobilized leptin was much lower (1.50 μM).</td>
      <td>84</td>
      <td>None</td>
      <td>dCp,dTp,dTp,dCp,dTp,dGp,dCp,dCp,dCp,dGp,dCp,dCp,dTp,dCp,dCp,dTp,dTp,dCp,dCp,dGp,dTp,dTp,dAp,dAp,dTp,dGp,dGp,dGp,dGp,dGp,dAp,dTp,dCp,dTp,dCp,dGp,dCp,dGp,dGp,dCp,dCp,dGp,dTp,dTp,dCp,dTp,dTp,dGp,dTp,dTp,dGp,dCp,dTp,dTp,dAp,dTp,dAp,dCp,dAp,dGp,dGp,dAp,dGp,dAp,dCp,dGp,dAp,dGp,dAp,dTp,dAp,dGp,dGp,dCp,dGp,dGp,dAp,dCp,dAp,dCp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>136</th>
      <td>C5a (C5C6)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/352.png" width="120" ></td>
      <td>Human complement C5 component (C5a)</td>
      <td>Protein</td>
      <td></td>
      <td>None</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>12189.15 g/mole</td>
      <td>55.26%</td>
      <td>365900 L/(mole·cm)</td>
      <td>100 nM (reported value)</td>
      <td></td>
      <td>None</td>
      <td>39</td>
      <td>None</td>
      <td>fCp,rGp,rAp,fUp,rGp,fCp,rGp,rGp,fUp,fCp,fUp,fCp,rAp,fUp,rGp,fCp,rGp,fUp,fCp,rGp,rAp,rGp,fUp,rGp,fUp,rGp,rAp,rGp,fUp,fUp,fUp,rAp,fCp,fCp,fUp,fUp,fCp,rGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>137</th>
      <td>Moloney Murine Leukemia Virus</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/174.png" width="120" ></td>
      <td>Moloney Murine Leukemia Virus (M-MLV) RT</td>
      <td>Protein</td>
      <td>6.9 nM (reported value)</td>
      <td>Binding Buffer (50 mM Tris-HC1 (pH 7.7), 200 mM potassium acetate, and 10 mM DTT)</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>10441.29 g/mole</td>
      <td>57.58%</td>
      <td>302800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>33</td>
      <td>Chen and Gold. "Selection of High-Affinity RNA Ligands to Reverse Transcriptase: Inhibition of  cDNA Synthesis and RNase H Activity." Journal of Biochemistry, 33(1994): 8746-8756.</td>
      <td>rUp,rUp,rAp,rCp,rCp,rAp,rCp,rGp,rCp,rGp,rCp,rUp,rCp,rUp,rUp,rAp,rAp,rCp,rUp,rGp,rCp,rUp,rAp,rGp,rCp,rGp,rCp,rCp,rAp,rUp,rGp,rGp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>138</th>
      <td>Paromomycin (LS11)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/437.png" width="120" ></td>
      <td>Paromomycin and other Aminoglycosides (AMGs)</td>
      <td>Small Organic</td>
      <td>19.8 nM (reported value)</td>
      <td>Binding Buffer: PBS with 10% (v/v) methanol</td>
      <td>RT°C</td>
      <td>None</td>
      <td>Binds to several aminoglycosides: kanamycin, paromycin streptomycin, apramycin, tobramycin, neomycin, and gentamycin.</td>
      <td>25474.89 g/mole</td>
      <td>46.84%</td>
      <td>788000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>87</td>
      <td>None</td>
      <td>rGp,rGp,rGp,rAp,rGp,rAp,rAp,rGp,rGp,fCp,rGp,rGp,fCp,rGp,fCp,rGp,fUp,rAp,rGp,rGp,fCp,rGp,rAp,rGp,fCp,fUp,fUp,fUp,rAp,fCp,fCp,rAp,rGp,fUp,fUp,fUp,fUp,rAp,fUp,fUp,fUp,rGp,fUp,fUp,fUp,fUp,rAp,fUp,fUp,rGp,fUp,fUp,rAp,fUp,rAp,fUp,rGp,fCp,fUp,fUp,rAp,fUp,fUp,fCp,rGp,fUp,rGp,rGp,rAp,fCp,rGp,rAp,fUp,rGp,fCp,rGp,fCp,rAp,fCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>139</th>
      <td>Salmonella Typhimurium OmpC Protein (T-2)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/514.jpg" width="120" ></td>
      <td>S. Typhimurium OmpC Protein</td>
      <td>Protein</td>
      <td>20 nM (reported value)</td>
      <td>SELEX binding buffer (30 mM Tris-HCl, pH 7.5, 150 mM NaCl, 1.5 mM MgCl2, 2 mM dithiothreitol, and 1% BSA)</td>
      <td>°C</td>
      <td>None</td>
      <td></td>
      <td>13276.35 g/mole</td>
      <td>54.76%</td>
      <td>429700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>126</td>
      <td>Han, S. R. and Lee, S. "In Vitro Selection of RNA Aptamer Specific to Salmonella Typhimurium." Journal of Microbiology and Biotechnology, 23.6(2013): 878-84.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>140</th>
      <td>L-Arginine(44Arg11)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/187.png" width="120" ></td>
      <td>L-Arginine</td>
      <td>Small Organic</td>
      <td>60 nM (reported value)</td>
      <td>Binding Buffer (250 mM NaCl, 50 mM Tris-HCl (pH=8.3), 5 mM MgCl2)</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>14215.62 g/mole</td>
      <td>56.82%</td>
      <td>438100 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>44</td>
      <td>Famulok, M., " Molecular Recognition of Amino Acids by RNA-Aptamers: An L-Citrulline Binding RNA Motif and its Evolution into an L-Arginine Binder." Journal of the American Chemical Society, 116 (1994): 1698-1706.</td>
      <td>rGp,rAp,rCp,rGp,rAp,rGp,rAp,rAp,rGp,rGp,rAp,rGp,rCp,rGp,rCp,rUp,rGp,rGp,rUp,rUp,rCp,rUp,rAp,rCp,rUp,rAp,rGp,rCp,rAp,rGp,rGp,rUp,rAp,rGp,rGp,rUp,rCp,rAp,rCp,rUp,rCp,rGp,rUp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>141</th>
      <td>Anti Anterior Gradient Homolog 2</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/508.png" width="120" ></td>
      <td>Anterior Gradient Homolog 2 (AGR2)</td>
      <td>Protein</td>
      <td>13.16 nM (reported value)</td>
      <td>200 mL, 1X PBS buffer  containing 137 mM NaCl, 2.7 mM KCl, 10 mM Na2HPO4 and  2.0 mM KH2PO4, pH 7.4</td>
      <td>37°C</td>
      <td>\n                                heat at 95C for 5 minutes, then, cool on ice for 10 minutes and finally incubate at room temperature for 10 minutes \n</td>
      <td>non conjugated GST beads were used as a counter</td>
      <td>27207.59 g/mole</td>
      <td>62.07%</td>
      <td>804600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>: The Mfold structure shown here is the secondary structure of the original aptamer and its primers at selection conditions. However, in the reference paper the authors do not give this secondary structure but a structure consisting of the sequenced random region  with a Streptavidin aptamer. That, is a design for a molecular beacon not necessarily the aptamer sequence.</td>
      <td>87</td>
      <td>Wu et al. Identification, Characterization and Application of a G-Quadruplex  Structured DNA Aptamer against Cancer  Biomarker Protein Anterior Gradient Homolog 2 (PLOSone)</td>
      <td>dTp,dCp,dTp,dCp,dGp,dGp,dAp,dCp,dGp,dCp,dGp,dTp,dGp,dTp,dGp,dGp,dTp,dCp,dGp,dGp,dGp,dTp,dGp,dGp,dGp,dAp,dGp,dTp,dTp,dGp,dTp,dGp,dGp,dGp,dGp,dGp,dGp,dGp,dGp,dGp,dTp,dGp,dGp,dGp,dAp,dGp,dGp,dGp,dTp,dTp,dCp,dTp,dTp,dTp,dGp,dTp,dTp,dTp,dGp,dAp,dTp,dCp,dTp,dTp,dTp,dCp,dTp,dCp,dGp,dCp,dTp,dGp,dCp,dCp,dTp,dGp,dGp,dCp,dCp,dCp,dTp,dAp,dGp,dAp,dGp,dTp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>142</th>
      <td>E. Coli Release Factor 1 (Clone II-1)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/122.jpg" width="120" ></td>
      <td>E. Coli Release Factor 1</td>
      <td>Small Organic</td>
      <td>30 nM (reported value)</td>
      <td>Elution Buffer (250 mM imidazole)</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>27219.37 g/mole</td>
      <td>52.94%</td>
      <td>824600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Recombinant RF1 derived from e.coli (K-12 strain) was used to develop this aptamer.</td>
      <td>85</td>
      <td>Sando, et al. "In vitro selection of RNA aptamer against Escherichia coli release factor 1." Bioorganic & Medicinal Chemistry Letters, 17(2007): 1216-1220.</td>
      <td>rGp,rGp,rAp,rCp,rCp,rGp,rAp,rGp,rAp,rAp,rGp,rUp,rUp,rAp,rCp,rCp,rCp,rUp,rGp,rUp,rAp,rAp,rUp,rCp,rUp,rUp,rAp,rGp,rGp,rAp,rUp,rGp,rAp,rAp,rUp,rCp,rGp,rCp,rAp,rUp,rGp,rCp,rUp,rCp,rUp,rAp,rGp,rCp,rGp,rAp,rCp,rCp,rUp,rUp,rUp,rUp,rCp,rGp,rGp,rCp,rUp,rUp,rCp,rGp,rGp,rCp,rGp,rUp,rAp,rCp,rGp,rCp,rAp,rCp,rAp,rUp,rCp,rGp,rCp,rAp,rGp,rCp,rAp,rAp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>143</th>
      <td>Aptamer MA</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7233.jpg" width="120" ></td>
      <td>methylenedioxypyrovalerone (MDPV)</td>
      <td>Small Organic</td>
      <td>7 nM (reported value)</td>
      <td>10 mM Tris–HCl, 0.5 mM MgCl2, 20 mM NaCl, pH 7.4</td>
      <td>23°C</td>
      <td>\n                                heated at 95◦C for 5min, slowly cooled to room temperature over 30 min \n</td>
      <td>Highly Specific</td>
      <td>14134.82 g/mole</td>
      <td>47.83%</td>
      <td>441918 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>This aptamer intrinsically binds Cy7 within its TWJ domain, but MDPV efficiently displaces the dye, resulting in a change in absorbance within seconds. This allows this aptamer to act as a label free assay.</td>
      <td>46</td>
      <td>Haixiang Yu, Weijuan Yang, Obtin Alkhamis, Juan Canoura, Kyung-Ae Yang, Yi Xiao, In vitro isolation of small-molecule-binding aptamers with intrinsic dye-displacement functionality, Nucleic Acids Research, Volume 46, Issue 8, 4 May 2018, Page e43, https://doi.org/10.1093/nar/gky026</td>
      <td>dCp,dTp,dTp,dAp,dCp,dGp,dAp,dCp,dTp,dCp,dAp,dGp,dGp,dCp,dAp,dTp,dTp,dTp,dTp,dGp,dCp,dCp,dGp,dGp,dGp,dTp,dAp,dAp,dCp,dGp,dAp,dAp,dGp,dTp,dTp,dAp,dCp,dTp,dGp,dTp,dCp,dGp,dTp,dAp,dAp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>144</th>
      <td>Dapt1</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7111.jpg" width="120" ></td>
      <td>Degraded Clostridium Difficile Toxin B</td>
      <td>Other</td>
      <td>60 nM (reported value)</td>
      <td>20 mM Tris-HCl, 100 mM NaCl, 5 mM KCl and 10 mM MgCl2, 1 mM DTT, 0.05%\nTween-20, pH 7.5</td>
      <td>RT°C</td>
      <td>\n                                first denatured at 95 ºC for 5 min and cooled on ice for 10 min,\nwas added \n</td>
      <td></td>
      <td>15531.14 g/mole</td>
      <td>48.00%</td>
      <td>519520 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>50</td>
      <td>Liu, Meng et. al. "In Vitro Selection of a DNA Aptamer Targeting Degraded Protein Fragments for Biosensing" Angewandte Chemie 10.(2020)</td>
      <td>dAp,dAp,dTp,dAp,dGp,dTp,dCp,dTp,dGp,dGp,dTp,dTp,dCp,dGp,dGp,dTp,dGp,dCp,dTp,dTp,dGp,dGp,dTp,dGp,dCp,dGp,dAp,dTp,dGp,dTp,dGp,dTp,dTp,dAp,dGp,dTp,dCp,dAp,dGp,dAp,dCp,dTp,dCp,dTp,dTp,dGp,dTp,dGp,dTp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>145</th>
      <td>C7</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7133.jpg" width="120" ></td>
      <td>Rituximab</td>
      <td>Protein</td>
      <td>8.8 nM (reported value)</td>
      <td>20 mM Tris pH 7.6, 100 mM NaCl, 2\nmM MgCl2, 5 mM KCl, 1 mM CaCl2, 0.02% v/v Tween 20</td>
      <td>22°C</td>
      <td>None</td>
      <td></td>
      <td>12380 g/mole</td>
      <td>50.00%</td>
      <td>373900 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>40</td>
      <td>Kohlberger M, Wildner S, Regl C, Huber CG, Gadermaier G. Rituximab-specific DNA aptamers are able to selectively recognize heat-treated antibodies. PLoS One. 2020 Nov 5;15(11):e0241560. doi: 10.1371/journal.pone.0241560. PMID: 33151990; PMCID: PMC7644011.</td>
      <td>dGp,dGp,dCp,dCp,dAp,dTp,dTp,dGp,dTp,dGp,dGp,dAp,dCp,dTp,dTp,dCp,dTp,dTp,dTp,dGp,dGp,dGp,dTp,dAp,dAp,dTp,dTp,dCp,dAp,dGp,dGp,dGp,dGp,dCp,dTp,dCp,dGp,dAp,dTp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>146</th>
      <td>Gentamicin (LGA11)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/428.png" width="120" ></td>
      <td>Gentamicin and other Aminoglycosides (AMGs)</td>
      <td>Small Organic</td>
      <td>22.3 nM (reported value)</td>
      <td>Binding Buffer: PBS with 10% (v/v) methanol</td>
      <td>RT°C</td>
      <td>None</td>
      <td>Binds to several aminoglycosides: kanamycin, paromycin streptomycin, apramycin, tobramycin, neomycin, and gentamycin.</td>
      <td>25568.23 g/mole</td>
      <td>56.96%</td>
      <td>771400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>87</td>
      <td>None</td>
      <td>rGp,rGp,rGp,rAp,rGp,rAp,rAp,rGp,rGp,fCp,rGp,rGp,fCp,rGp,fCp,rGp,fUp,rAp,rGp,rGp,fCp,rGp,rAp,rGp,fCp,fUp,fUp,fUp,rAp,rGp,rGp,fCp,fCp,fCp,rGp,rAp,fCp,rAp,fUp,fUp,fCp,fCp,fCp,fCp,fUp,rAp,rAp,rAp,rAp,rAp,rAp,rGp,fCp,fUp,fUp,rGp,fUp,fUp,fCp,fUp,rAp,fUp,fUp,fCp,rGp,fUp,rGp,rGp,rAp,fCp,rGp,rAp,fUp,rGp,fCp,rGp,fCp,rAp,fCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>147</th>
      <td>Thiol-modified thrombin Aptamer</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7216.png" width="120" ></td>
      <td>Thrombin</td>
      <td>Protein</td>
      <td></td>
      <td>0.1 M Na2HPO4, 0.1 M KH2PO4 and 0.1 M\nNaCl. 20 mM Tris–HCl buffer (pH 7.4) containing 140 mM NaCl,\n5 mM KCl and 1 mM MgCl2</td>
      <td>25°C</td>
      <td>\n                                mFold \n</td>
      <td>N/A</td>
      <td>5039.41 g/mole</td>
      <td>60.00%</td>
      <td>142641 L/(mole·cm)</td>
      <td></td>
      <td>0.3 fM (reported value)</td>
      <td>Modified Thrombin Aptamer. \n5' SH-(CH2)6−GGTTGGTGTGGTTGG '3</td>
      <td>2</td>
      <td>doi:10.1016/j.aca.2013.09.060</td>
      <td>dGp,dGp,dTp,dTp,dGp,dGp,dTp,dGp,dTp,dGp,dGp,dTp,dTp,dGp,dGp,dGp,dGp,dTp,dTp,dGp,dGp,dTp,dGp,dTp,dGp,dGp,dTp,dTp,dGp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>148</th>
      <td>RNA aptamer-1-based sensitive detection of SARS-CoV nucleocapsid protein</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7086.png" width="120" ></td>
      <td>SARS-CoV Nucleocapsid (N) Protein</td>
      <td>Protein</td>
      <td>1.65 nM (reported value)</td>
      <td>30mM Tris-HCl, pH 7.5, 150 mM NaCl, 10 mM MgCl2, 2mM DTT, 1%BSA</td>
      <td>°C</td>
      <td>None</td>
      <td></td>
      <td>20566.55 g/mole</td>
      <td>58.82%</td>
      <td>1021300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>85</td>
      <td>Ahn, Dae-Gyun,et al. "RNA aptamer-based sensitive detection of SARS coronavirus nucleocapsid protein" The Royal Society of Chemistry, (2009):1896-1901.</td>
      <td>rGp,rGp,rGp,rAp,rGp,rAp,rGp,rCp,rGp,rGp,rAp,rAp,rGp,rCp,rGp,rUp,rGp,rCp,rUp,rGp,rGp,rGp,rCp,rCp,rUp,rGp,rUp,rCp,rGp,rUp,rUp,rCp,rGp,rCp,rUp,rGp,rUp,rGp,rUp,rCp,rUp,rUp,rGp,rCp,rUp,rAp,rCp,rGp,rUp,rUp,rAp,rCp,rGp,rUp,rUp,rAp,rCp,rAp,rCp,rGp,rGp,rUp,rUp,rGp,rGp,rCp,rAp,rUp,rAp,rAp,rCp,rCp,rCp,rAp,rGp,rAp,rGp,rGp,rUp,rCp,rGp,rAp,rUp,rGp,rGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>149</th>
      <td>Interleukin-6 receptor (AIR-3)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/473.png" width="120" ></td>
      <td>Soluable interleukin-6 receptor (sIL-6R)</td>
      <td>Protein</td>
      <td>19.7 nM (reported value)</td>
      <td>None</td>
      <td>37°C</td>
      <td>\n                                None listed \n</td>
      <td>No binding to IL-6 or lysozyme observed.</td>
      <td>34310.65 g/mole</td>
      <td>45.28%</td>
      <td>1085700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Full-length product. Truncated 19-mer version (AIR-3A)is listed related aptamers.</td>
      <td>117</td>
      <td>None</td>
      <td>rGp,rGp,rAp,rAp,rGp,rAp,rAp,rAp,rGp,rAp,rGp,rGp,rUp,rCp,rUp,rGp,rAp,rGp,rAp,rCp,rAp,rUp,rUp,rCp,rUp,rCp,rUp,rUp,rAp,rUp,rAp,rGp,rGp,rGp,rGp,rAp,rGp,rGp,rCp,rUp,rGp,rUp,rGp,rGp,rUp,rGp,rAp,rGp,rGp,rGp,rAp,rAp,rUp,rAp,rUp,rUp,rAp,rAp,rGp,rAp,rGp,rAp,rAp,rUp,rUp,rAp,rAp,rCp,rGp,rGp,rUp,rCp,rUp,rAp,rGp,rUp,rUp,rCp,rAp,rCp,rCp,rUp,rCp,rGp,rAp,rCp,rUp,rUp,rCp,rUp,rGp,rGp,rAp,rGp,rUp,rUp,rGp,rAp,rCp,rGp,rUp,rUp,rGp,rCp,rUp,rUp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>150</th>
      <td>Phosphorylated ERK2</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/263.png" width="120" ></td>
      <td>Phosphorylated ERK2</td>
      <td>Protein</td>
      <td>N/A nM (reported value)</td>
      <td>None</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>30687.59 g/mole</td>
      <td>48.42%</td>
      <td>959300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>95</td>
      <td>Vaish et al. "Monitoring post-translational modifications of proteins with allosteric ribozymes." Nature Biotechnology, 20(2002): 810-815.</td>
      <td>rGp,rGp,rCp,rGp,rUp,rGp,rAp,rCp,rCp,rUp,rGp,rAp,rUp,rGp,rAp,rGp,rUp,rCp,rAp,rCp,rGp,rCp,rAp,rGp,rAp,rCp,rGp,rCp,rUp,rAp,rGp,rCp,rGp,rAp,rAp,rUp,rUp,rGp,rGp,rUp,rUp,rCp,rCp,rUp,rCp,rGp,rAp,rAp,rAp,rGp,rGp,rGp,rGp,rAp,rAp,rAp,rGp,rCp,rGp,rUp,rUp,rAp,rUp,rUp,rAp,rAp,rGp,rAp,rAp,rAp,rCp,rCp,rAp,rAp,rAp,rAp,rUp,rGp,rUp,rGp,rUp,rUp,rAp,rCp,rGp,rAp,rAp,rAp,rCp,rGp,rUp,rUp,rCp,rCp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>151</th>
      <td>Interleukin 10 (IL-10) (R5A1)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/291.png" width="120" ></td>
      <td>Interleukin -10 (IL-10)</td>
      <td>Protein</td>
      <td>12 nM (reported value)</td>
      <td>150 mmol/L NaCl, 20 mmol/L KCl, and 2 mmol/L HEPES heated to 82°C  and slowly cooling to room temperature</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>25585.42 g/mole</td>
      <td>56.25%</td>
      <td>765600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Aptamer was truncated and modified with 2'F and 2'O-methyl, but no KD's were given for these adaptations.</td>
      <td>88</td>
      <td>Berezhnoy, A, et al. (2012). Isolation and optimization of murine il-10 receptor blocking oligonucleotide aptamers using high- throughput sequencing. The American Society of Gene & Cell Therapy</td>
      <td>rGp,rGp,rGp,rCp,rUp,rCp,rAp,rUp,rGp,rCp,rAp,rCp,rGp,rUp,rUp,rUp,rGp,rCp,rUp,rCp,rCp,rUp,rGp,rUp,rAp,rAp,rUp,rUp,rGp,rGp,rCp,rGp,rUp,rAp,rUp,rGp,rUp,rAp,rAp,rCp,rCp,rCp,rAp,rGp,rGp,rCp,rAp,rCp,rCp,rAp,rAp,rAp,rCp,rAp,rCp,rCp,rCp,rCp,rAp,rGp,rGp,rCp,rCp,rGp,rGp,rGp,rCp,rCp,rAp,rUp,rGp,rAp,rUp,rCp,rCp,rAp,rCp,rAp,rUp,rAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>152</th>
      <td>Paromomycin (LGA11)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/431.png" width="120" ></td>
      <td>Paromomycin and other Aminoglycosides (AMGs)</td>
      <td>Small Organic</td>
      <td>21.4 nM (reported value)</td>
      <td>Binding Buffer: PBS with 10% (v/v) methanol</td>
      <td>RT°C</td>
      <td>None</td>
      <td>Binds to several aminoglycosides: kanamycin, paromycin streptomycin, apramycin, tobramycin, neomycin, and gentamycin.</td>
      <td>25568.23 g/mole</td>
      <td>56.96%</td>
      <td>771400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>87</td>
      <td>None</td>
      <td>rGp,rGp,rGp,rAp,rGp,rAp,rAp,rGp,rGp,fCp,rGp,rGp,fCp,rGp,fCp,rGp,fUp,rAp,rGp,rGp,fCp,rGp,rAp,rGp,fCp,fUp,fUp,fUp,rAp,rGp,rGp,fCp,fCp,fCp,rGp,rAp,fCp,rAp,fUp,fUp,fCp,fCp,fCp,fCp,fUp,rAp,rAp,rAp,rAp,rAp,rAp,rGp,fCp,fUp,fUp,rGp,fUp,fUp,fCp,fUp,rAp,fUp,fUp,fCp,rGp,fUp,rGp,rGp,rAp,fCp,rGp,rAp,fUp,rGp,fCp,rGp,fCp,rAp,fCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>153</th>
      <td>Mycolactone Aptamer</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/620.png" width="120" ></td>
      <td>mycolactone</td>
      <td>Protein</td>
      <td>6.3 µM (reported value)</td>
      <td>1X concentration (1X PBS [pH 7.4]  and 10 mMMgCl2)</td>
      <td>°C</td>
      <td>None</td>
      <td>SpecificApta-3683 (0.5mM)mediated  by the presence of mycolactone (100 mM) over a 3 minute time course was done</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>0</td>
      <td>Sakyi, Samuel A., et al. "RNA Aptamer That Specifically Binds to Mycolactone and Serves as a Diagnostic Tool for Diagnosis of Buruli Ulcer." PLOS Negl Trop Dis 10.10 (2016): e0004950</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>154</th>
      <td>PL1</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7115.jpg" width="120" ></td>
      <td>PD-L1</td>
      <td>Protein</td>
      <td>81.46-110  nM (reported value)</td>
      <td>4.5 g L^-1 glucose and 5 mM MgCl2 in Dulbecco’s PBS without calcium and magnesium</td>
      <td>4°C</td>
      <td>None</td>
      <td></td>
      <td>12601.1 g/mole</td>
      <td>28.95%</td>
      <td>389600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>84</td>
      <td>None</td>
      <td>dAp,dTp,dAp,dCp,dCp,dAp,dGp,dCp,dTp,dTp,dAp,dTp,dTp,dCp,dAp,dAp,dTp,dTp,dGp,dTp,dAp,dGp,dAp,dGp,dTp,dAp,dTp,dAp,dAp,dAp,dAp,dAp,dGp,dAp,dGp,dTp,dGp,dAp,dTp,dGp,dAp,dTp,dCp,dTp,dTp,dTp,dTp,dGp,dTp,dAp,dGp,dGp,dTp,dTp,dTp,dTp,dTp,dTp,dAp,dGp,dAp,dTp,dAp,dGp,dTp,dAp,dAp,dGp,dTp,dGp,dCp,dAp,dAp,dTp,dCp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>155</th>
      <td>s5rev</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7092.png" width="120" ></td>
      <td>CD44</td>
      <td>Protein</td>
      <td>238 nM (reported value)</td>
      <td>50 mM NaH2PO4/\nNa2HPO4 buffer pH 7.5 containing 300 mM NaCl and 0.01%\n(v/v) Tween 20</td>
      <td>23°C</td>
      <td>\n                                The ssDNA pool from the\nprevious cycle was resuspended in selection buffer, denatured for 5 min at 95C, immediately cooled on ice for\n10 min, and equilibrated for 10 min at RT. \n</td>
      <td>Tested for specificity against CD44 over MDM2, MDMX, and BSA</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>72</td>
      <td>Pęcak, Aleksandra, et al. “Anti-CD44 DNA Aptamers Selectively Target Cancer Cells.” Nucleic Acid Therapeutics, vol. 30, no. 5, 2020, pp. 289–298., doi:10.1089/nat.2019.0833.</td>
      <td>dCp,dAp,dTp,dGp,dCp,dTp,dTp,dCp,dCp,dCp,dCp,dAp,dGp,dGp,dGp,dAp,dGp,dAp,dTp,dGp,dAp,dCp,dCp,dGp,dGp,dGp,dGp,dCp,dGp,dTp,dAp,dCp,dAp,dCp,dCp,dGp,dTp,dCp,dGp,dCp,dGp,dGp,dCp,dAp,dCp,dAp,dTp,dGp,dTp,dCp,dTp,dGp,dAp,dAp,dTp,dGp,dCp,dGp,dTp,dTp,dTp,dAp,dGp,dTp,dCp,dTp,dCp,dTp,dGp,dTp,dGp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>156</th>
      <td>CD4 (Aptamer 7)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/120.jpg" width="120" ></td>
      <td>CD4</td>
      <td>Protein</td>
      <td>0.5 nM (reported value)</td>
      <td>Elution Buffer: (50 mM sodium phosphate, 150 mM NaCl, 2 M urea and 0.1% NaN3 (pH 7.2))    Binding Binding: (1 ml PBS containing 2 mM MgCl2)</td>
      <td>RT°C</td>
      <td>None</td>
      <td></td>
      <td>28124.72 g/mole</td>
      <td>48.28%</td>
      <td>875000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>89</td>
      <td>Davis, K., et al. "Staining of cell surface human CD4 with 2'-F-pyrimidine-containing RNA aptamers for flow cytometry." Nucleic Acids Research, 26(1998): 3915-3924.</td>
      <td>rGp,rGp,rGp,rAp,rGp,rAp,fCp,rAp,rAp,rGp,rAp,rAp,fUp,rAp,rAp,rAp,fCp,rGp,fCp,fUp,fCp,rAp,rAp,fUp,rGp,fUp,rGp,fUp,fCp,rGp,fUp,fCp,fCp,fUp,rGp,rGp,fUp,rAp,fCp,rGp,rAp,fUp,fUp,fUp,fUp,rGp,rGp,fUp,rAp,fUp,rAp,fUp,rAp,rAp,fCp,fCp,rGp,fUp,rGp,rGp,fCp,fUp,fUp,fUp,fUp,fCp,rGp,rAp,fCp,rAp,rGp,rGp,rAp,rGp,rGp,fCp,fUp,fCp,rAp,fCp,rAp,rAp,fCp,rAp,rGp,rGp,fCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>157</th>
      <td>Integrin αV subunit (aV-1)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/320.png" width="120" ></td>
      <td>Integrin aV subunit</td>
      <td>Protein</td>
      <td>2.7 nM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td>No binding was observed to integrin αIIbβ3 at 250 nM, whereas the aptamer recognized integrin αVβ3 and αVβ6 at 100 pM.</td>
      <td>26009.42 g/mole</td>
      <td>58.02%</td>
      <td>774900 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>See also aptamer β3-1 which binds to the β3 subunit.</td>
      <td>81</td>
      <td>Gong, Q. et al,(2012). Selection strategy to generate aptamer pairs that bind to distinct sites on protein targets. Analytical Chemistry, 2012(84), 5365-5371.</td>
      <td>rGp,rGp,rGp,rAp,rGp,rGp,rAp,fCp,rGp,rAp,fUp,rGp,fCp,rGp,rGp,fCp,rAp,fCp,rAp,fCp,rAp,fUp,fUp,fCp,fCp,fCp,rGp,fUp,fCp,fCp,fUp,fCp,rGp,rAp,fUp,rAp,fCp,rGp,fUp,fCp,fUp,rAp,rGp,rGp,fCp,fUp,fUp,rAp,rGp,fUp,rGp,fCp,fCp,rAp,fCp,fUp,fUp,rGp,fCp,fUp,fUp,rAp,rAp,fUp,fCp,fCp,rAp,rGp,rAp,fCp,rGp,rAp,fCp,fUp,fCp,rGp,fCp,fCp,fCp,rGp,rAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>158</th>
      <td>Anti-VEGF165 Unnatural Base DNA Aptamer #4</td>
      <td>Chimeric</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/633.jpg" width="120" ></td>
      <td>VEGF165</td>
      <td>Protein</td>
      <td>0.9 pM (reported value)</td>
      <td>1?? PBS (1 mM KH2PO4, 3 mM Na2HPO4 and 155 mM NaCl, pH 7.4, Gibco)</td>
      <td>37°C</td>
      <td>None</td>
      <td>Modified Aptamer inhibited VEGF165's ability to bind VEGF receptor.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Reference has 10 Aptamer Constructs with varying properties. Using an unnatural extra "base-pair" they improve afinity to VEGF165 over Naked DNA. Adding stabilization hairpins & G-C pairs they improved their Aptamers binding ability.</td>
      <td>56</td>
      <td>Kimoto M., Nakamura M., Hirao I. July 7 2016. Post-ExSELEX stabilization of an unnatural-base DNA aptamer targeting VEGF165 toward pharmaceutical applications. Nucleic Acids Research 44(15): 7487-7494.</td>
      <td>dGp,dCp,dGp,dGp,dTp,dAp,dAp,dAp,dCp,dTp,dGp,dCp,dGp,dTp,dCp,dCp,dGp,dAp,dAp,dGp,dGp,dGp,dGp,dCp,dTp,dGp,dCp,dAp,dGp,dTp,dGp,dAp,dCp,dCp,dCp,dGp,dAp,dAp,dTp,dGp,dGp,dGp,dTp,dCp,dCp,dGp,dCp,dCp,dGp,dCp,dGp,dAp,dAp,dGp,dCp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>159</th>
      <td>G5a3N.4</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/555.jpg" width="120" ></td>
      <td>HPV-16E7 protein</td>
      <td>Protein</td>
      <td>1.9 µM (reported value)</td>
      <td>1X PBS</td>
      <td>37°C</td>
      <td>\n                                65C for 8 min in 1X PBS \n</td>
      <td>N/A</td>
      <td>23105.1 g/mole</td>
      <td>40.00%</td>
      <td>717900 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>225</td>
      <td>Julia D. Toscano-Garibay, Maria L. Benitez-Hess, and Luis M. Alvarez-Salas. 2011. Isolation and Characterization of an RNA Aptamer for the HPV-16 E7 Oncoprotein. Archives of Medical Research 42 (2011) 88-96</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>160</th>
      <td>Bovine Thrombin (T7 05 RNA)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/559" width="120" ></td>
      <td>Bovine Thrombin</td>
      <td>Protein</td>
      <td>164 nM (reported value)</td>
      <td>SHMCK+  buffer (20 mM Hepes (pH 7.35), 120 mM NaCl, 5 mM KCl, 1 mM CaCl2, 1 mM MgCl2 and 0.1% geletin) containing 0.05% Tween 20.     Binding Buffer (SHMCK+  buffer containing 0.05% Tween 20)</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>21852.22 g/mole</td>
      <td>44.12%</td>
      <td>684600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>68</td>
      <td>None</td>
      <td>rGp,rCp,rAp,rAp,rUp,rGp,rGp,rUp,rAp,rCp,rGp,rGp,rUp,rAp,rCp,rUp,rUp,rCp,rCp,rUp,rUp,rUp,rGp,rGp,rAp,rAp,rGp,rAp,rUp,rAp,rGp,rCp,rUp,rGp,rGp,rAp,rGp,rAp,rAp,rCp,rUp,rAp,rAp,rCp,rCp,rAp,rAp,rAp,rAp,rGp,rUp,rGp,rCp,rAp,rCp,rGp,rCp,rUp,rAp,rCp,rUp,rUp,rUp,rGp,rCp,rUp,rAp,rAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>161</th>
      <td>SARS-CoV-2 Aptamer 1C - Apta-switch antisense</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7098.png" width="120" ></td>
      <td>RBD of the Spike glycoprotein from SARS-CoV-2</td>
      <td>Protein</td>
      <td>5.8 nM (reported value)</td>
      <td>10 mM Tris, 0.5 mM NaCl, 1 mM EDTA, pH 7.4</td>
      <td>23°C</td>
      <td>\n                                Main aptamer: 80°C for 3 minutes, gentle cooling at 23°C for 5 minutes. Antisense Apta-switch component: 40°C for 10 minutes, then combine with refolded main aptamer and incubate for 2 hours at 23°C. Wash with 1X DPBS, store at 4°C. \n</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Apta-switch component (antisense strand) to SARS-CoV Spike protein aptamer (ID #: 7058). Originally reported with poly(dT) spacer and linker for functionalization.</td>
      <td>15</td>
      <td>Singh NK, Ray P, Carlin AF, Magallanes C, Morgan SC, Laurent LC, Aronoff-Spencer ES, Hall DA. "Hitting the diagnostic sweet spot: Pont-of-care SARS-CoV-2 salivary antigen testing with an off-the-shelf gluocometer". medRxiv preprint. doi: https://doi.org/10.1101/2020.09.24.20200394.</td>
      <td>dTp,dGp,dTp,dCp,dCp,dAp,dTp,dTp,dAp,dAp,dCp,dGp,dCp,dCp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>162</th>
      <td>S21</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7122.jpg" width="120" ></td>
      <td>Cas9</td>
      <td>Protein</td>
      <td>0.63 nM (reported value)</td>
      <td>20 mM HEPES, pH 6.5, 100 mM NaCl, 5 mM MgCl2, 0.1 mM EDTA</td>
      <td>RT°C</td>
      <td>None</td>
      <td></td>
      <td>14334.3 g/mole</td>
      <td>61.70%</td>
      <td>418100 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>47</td>
      <td>None</td>
      <td>dGp,dAp,dGp,dGp,dCp,dTp,dCp,dTp,dCp,dAp,dCp,dAp,dTp,dCp,dGp,dCp,dGp,dCp,dCp,dTp,dTp,dTp,dCp,dCp,dCp,dCp,dAp,dGp,dCp,dTp,dTp,dTp,dCp,dGp,dGp,dTp,dGp,dTp,dGp,dAp,dGp,dAp,dGp,dCp,dCp,dTp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>163</th>
      <td>Human Hepatocellular carcinoma (TLS11a)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/286.png" width="120" ></td>
      <td>Human Hepatocellular carcinoma cell line (LH86)</td>
      <td>Cells</td>
      <td>7.16 nM (reported value)</td>
      <td>None</td>
      <td>4°C</td>
      <td>None</td>
      <td>Binds to hepatocellular carcinoma LH86 cell line.  Negative control was normal liver cells, Hu1082.</td>
      <td>19339.58 g/mole</td>
      <td>50.79%</td>
      <td>589200 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>63</td>
      <td>Meng, L. (2012). Targeted delivery of chemotherapy agents using liver cancer-specific aptamer. PloS One, 7(4), doi: 10.1371</td>
      <td>dAp,dCp,dAp,dGp,dCp,dAp,dTp,dCp,dCp,dCp,dCp,dAp,dTp,dGp,dTp,dGp,dAp,dAp,dCp,dAp,dAp,dTp,dCp,dGp,dCp,dAp,dTp,dTp,dGp,dTp,dGp,dAp,dTp,dTp,dGp,dTp,dTp,dAp,dCp,dGp,dGp,dTp,dTp,dTp,dCp,dCp,dGp,dCp,dCp,dTp,dCp,dAp,dTp,dGp,dGp,dAp,dCp,dGp,dTp,dGp,dCp,dTp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>164</th>
      <td>Tetracycline (cb28 minimer)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/116.png" width="120" ></td>
      <td>Tetracycline</td>
      <td>Small Organic</td>
      <td>~1 µM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>\n                                Heated to 90°C for 3 min and cooled at RT for 5 min. \n</td>
      <td>Binds to tetracycline analogues 4-epi-tetracycline, oxytetracycline, and 7-chlortetracycline. Does not effectively bind to doxycycline or minocycline.</td>
      <td>19333.74 g/mole</td>
      <td>50.00%</td>
      <td>604400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>60</td>
      <td>None</td>
      <td>rGp,rGp,rCp,rCp,rUp,rAp,rAp,rAp,rAp,rCp,rAp,rUp,rAp,rCp,rCp,rAp,rGp,rAp,rUp,rUp,rUp,rCp,rGp,rAp,rUp,rCp,rUp,rGp,rGp,rAp,rGp,rAp,rGp,rGp,rUp,rGp,rAp,rAp,rGp,rAp,rAp,rUp,rUp,rCp,rGp,rAp,rCp,rCp,rAp,rCp,rCp,rUp,rAp,rGp,rGp,rCp,rCp,rGp,rGp,rUp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>165</th>
      <td>HIV-1 Reverse Transcriptase (4.3)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/539" width="120" ></td>
      <td>HIV-1 RT</td>
      <td>Protein</td>
      <td>190 pM (reported value)</td>
      <td>Binding buffer: 10 mM tris, 50 mM NaCl, and 1 mM EDTA (pH 7.5)</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>24799.14 g/mole</td>
      <td>51.25%</td>
      <td>780100 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>80</td>
      <td>None</td>
      <td>dAp,dGp,dCp,dAp,dGp,dCp,dAp,dCp,dAp,dGp,dAp,dGp,dGp,dTp,dCp,dAp,dGp,dAp,dTp,dGp,dGp,dCp,dAp,dGp,dGp,dTp,dTp,dTp,dCp,dGp,dAp,dCp,dGp,dTp,dAp,dCp,dAp,dAp,dTp,dGp,dCp,dTp,dAp,dTp,dGp,dGp,dAp,dGp,dGp,dCp,dTp,dTp,dTp,dAp,dTp,dGp,dAp,dTp,dCp,dGp,dCp,dCp,dTp,dAp,dTp,dGp,dCp,dGp,dTp,dGp,dCp,dTp,dAp,dCp,dCp,dGp,dTp,dGp,dAp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>166</th>
      <td>HBA2</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7188.png" width="120" ></td>
      <td>Avian Influenza Virus Subtype H5N1</td>
      <td>Other</td>
      <td>122 nM (reported value)</td>
      <td>Tris buffer (20 mM, pH 7.4).</td>
      <td>Not given.°C</td>
      <td>None</td>
      <td></td>
      <td>18672 g/mole</td>
      <td>70.00%</td>
      <td>560400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>HBA2 was found to inhibit hemagglutination, which is caused by the virus surface protein hemagglutinin. This aptamer can be used as both a diagnostic and a therapeutic agent for H5N1.</td>
      <td>60</td>
      <td>Kim, S.; Choi, J.; Kim, A.; Lee, S.; Yoon, M. Development of ssDNA Aptamers for Diagnosis and Inhibition of the Highly Pathogenic Avian Influenza Virus Subtype H5N1. Biomolecules 2020 July 28; 10(8): 1116. DOI: 10.3390/biom10081116.</td>
      <td>dAp,dTp,dGp,dCp,dGp,dGp,dAp,dTp,dCp,dCp,dCp,dGp,dCp,dGp,dCp,dGp,dGp,dGp,dTp,dCp,dTp,dGp,dAp,dGp,dGp,dAp,dGp,dTp,dGp,dCp,dGp,dCp,dGp,dGp,dTp,dGp,dCp,dCp,dAp,dGp,dTp,dGp,dAp,dGp,dTp,dGp,dCp,dGp,dCp,dGp,dAp,dAp,dGp,dCp,dTp,dTp,dGp,dCp,dGp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>167</th>
      <td>Aptamer A14</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7089.pdf" width="120" ></td>
      <td>HA protein of CJ01 H1N1 Influenza Virus</td>
      <td>Protein</td>
      <td>2.98 nM (reported value)</td>
      <td>S Buffer (Washing and Binding): 40nM HEPES (pH 7.5); 102mM NaCl; 5mM KCl; 5mM MgCl2, 0.05% Tween-20</td>
      <td>25°C</td>
      <td>\n                                95 C for 5 min, slowly cooled to 25 C \n</td>
      <td></td>
      <td>13169.52 g/mole</td>
      <td>69.77%</td>
      <td>446380 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>43</td>
      <td>Kwon, J., Narayan, C., Kim, C., Han, M. J., Kim, M., & Jang, S. K. (2019). Development of a Subtype-Specific Diagnostic System for Influenza Virus H3N2 Using a Novel Virus-Based Systematic Evolution of Ligands by Exponential Enrichment (Viro-SELEX). Journal of Biomedical Nanotechnology, 15(7), 1609–1621. https://doi.org/10.1166/jbn.2019.2789\n\nGold L, Ayers D, Bertino J, Bock C, Bock A, Brody EN, et al. (2010) Aptamer-Based Multiplexed Proteomic Technology for Biomarker Discovery. PLoS ONE 5(12): e15004. https://doi.org/10.1371/journal.pone.0015004</td>
      <td>dCp,dCp,dTp,dGp,dCp,dCp,dTp,dTp,dTp,dGp,dGp,dCp,dAp,dGp,dCp,dCp,dGp,dGp,dGp,dCp,dGp,dAp,dAp,dGp,dGp,dCp,dAp,dAp,dCp,dCp,dCp,dGp,dAp,dCp,dAp,dCp,dCp,dGp,dAp,dCp,dAp,dGp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>168</th>
      <td>GHS-R binding peptide (G5-1)</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/331.png" width="120" ></td>
      <td>growth hormone secretagogue receptor</td>
      <td>Protein</td>
      <td>2.4 nM (reported value)</td>
      <td>None</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>3922.6 g/mole</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>28</td>
      <td>None</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>169</th>
      <td>HIV-1 R5 SU Glycoprotein gp120 (B40)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/365.png" width="120" ></td>
      <td>HIV-1 R5 gp120</td>
      <td>Protein</td>
      <td>21 nM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td></td>
      <td>37608.12 g/mole</td>
      <td>48.72%</td>
      <td>1150000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>122</td>
      <td>None</td>
      <td>rGp,rGp,rGp,rAp,rGp,rAp,fCp,rAp,rAp,rGp,rAp,fCp,fUp,rAp,rGp,rAp,fCp,rGp,fCp,fUp,fCp,rAp,rAp,fUp,rGp,fUp,rGp,rGp,rGp,fCp,fCp,rAp,fCp,rGp,fCp,fCp,fCp,rGp,rAp,fUp,fUp,fUp,fUp,rAp,fCp,rGp,fCp,fUp,fUp,fUp,fUp,rAp,fCp,fCp,fCp,rGp,fCp,rAp,fCp,rGp,fCp,rGp,rAp,fUp,fUp,rGp,rGp,fUp,fUp,fUp,rGp,fUp,fUp,fUp,fUp,fCp,rGp,rAp,fCp,rAp,fUp,rGp,rAp,rGp,rAp,fCp,fUp,fCp,rAp,fCp,rAp,rAp,fCp,rAp,rGp,fUp,fUp,fCp,fCp,fCp,fUp,fUp,fUp,rAp,rGp,fUp,rGp,rAp,rGp,rGp,rGp,fUp,fUp,rAp,rAp,fUp,fUp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>170</th>
      <td>Vaccinia virusƒ??infected A549 cells (TV08)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/272.png" width="120" ></td>
      <td>Vaccinia virus (WR Strain)ƒ??infected A549 cells</td>
      <td>Cells</td>
      <td>2.7 nM (reported value)</td>
      <td>None</td>
      <td>4°C</td>
      <td>None</td>
      <td>The aptamer was also able to discriminate between infected and uninfected Hep 3B, Hep G2, HeLa, HCT 116, and CaOV3 cell lines and vaccinia IHDJ-infected cells.</td>
      <td>25569.65 g/mole</td>
      <td>45.78%</td>
      <td>784600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>83</td>
      <td>None</td>
      <td>dAp,dTp,dCp,dGp,dTp,dCp,dTp,dGp,dCp,dTp,dCp,dCp,dGp,dTp,dCp,dCp,dAp,dAp,dTp,dAp,dTp,dGp,dAp,dTp,dGp,dAp,dCp,dAp,dCp,dCp,dTp,dGp,dCp,dAp,dTp,dAp,dAp,dTp,dTp,dTp,dAp,dTp,dAp,dGp,dTp,dGp,dAp,dGp,dTp,dCp,dTp,dTp,dGp,dAp,dTp,dTp,dCp,dAp,dCp,dGp,dCp,dTp,dGp,dCp,dAp,dTp,dTp,dTp,dGp,dGp,dTp,dGp,dTp,dGp,dAp,dGp,dGp,dTp,dCp,dGp,dTp,dGp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>171</th>
      <td>HA12-16</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/591.jpg" width="120" ></td>
      <td>Avian Influenza Glycoprotein Hemagglutinin (gHA1)</td>
      <td>Protein</td>
      <td>See Source (Non-Kd) nM (reported value)</td>
      <td>20 mM sodium phosphate, pH 7.5, 500 mM NaCl, and 20 mM imidazole</td>
      <td>23°C</td>
      <td>\n                                85C for 5 minutes \n</td>
      <td></td>
      <td>30328.3 g/mole</td>
      <td>50.00%</td>
      <td>939200 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>282</td>
      <td>Kwon H-M, Lee KH, Han BW, Han MR, Kim DH, et al. (2014) An RNA Aptamer That Specifically Binds to the Glycosylated Hemagglutinin of Avian Influenza Virus and Suppresses Viral Infection in Cells. PLoS ONE 9(5): e97574. doi:10.1371/journal.pone.0097574</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>172</th>
      <td>B23 LH aptamer</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/644.jpg" width="120" ></td>
      <td>luteinising hormone</td>
      <td>Protein</td>
      <td>321 nM (reported value)</td>
      <td>PBS</td>
      <td>25°C</td>
      <td>None</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Predicted to form a G-quadruplex.</td>
      <td>35</td>
      <td>Liang S, Kinghorn AB, Voliotis M, et al. Measuring luteinising hormone pulsatility with a robotic aptamer-enabled electrochemical reader. Nat Commun. 2019;10(1):852. Published 2019 Feb 20. doi:10.1038/s41467-019-08799-6</td>
      <td>dTp,dAp,dTp,dGp,dGp,dTp,dAp,dTp,dGp,dCp,dTp,dGp,dTp,dGp,dTp,dGp,dGp,dTp,dAp,dTp,dGp,dGp,dGp,dGp,dTp,dGp,dGp,dCp,dGp,dTp,dGp,dCp,dTp,dCp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>173</th>
      <td>Penetratin</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/467.png" width="120" ></td>
      <td>Cell-penetrating peptide (CPP)</td>
      <td>Cells</td>
      <td>N/A nM (reported value)</td>
      <td>N/A</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>2455.04 g/mole</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Crosses cell membranes through an energy-independent mechanism.</td>
      <td>16</td>
      <td>None</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>174</th>
      <td>Enthalpy Driven Aptamer SYL-H2C</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7101.png" width="120" ></td>
      <td>EpCAM</td>
      <td>Protein</td>
      <td>34.7 nM (reported value)</td>
      <td>PBS, pH=7.4 (including 136.8 mM NaCl, 10.1 mM Na2HPO4, 2.7 mM KCl, 1.8 mM KH2PO4, 0.55 mM MgCl2)</td>
      <td>37 °C</td>
      <td>\n                                95C for 10 min, cooled on ice for 10 min \n</td>
      <td></td>
      <td>12500.12 g/mole</td>
      <td>62.50%</td>
      <td>403290 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>126</td>
      <td>Huang, M., Song, J., Huang, P., Chen, X., Wang, W., Zhu, Z., Song, Y., & Yang, C. (2019). Molecular Crowding Evolution for Enabling Discovery of Enthalpy-Driven Aptamers for Robust Biomedical Applications. Analytical Chemistry, 91(16), 10879–10886. https://doi.org/10.1021/acs.analchem.9b02697</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>175</th>
      <td>L-Tyrosinamide (pe35)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/113.png" width="120" ></td>
      <td>L-Tyrosinamide</td>
      <td>Small Organic</td>
      <td>45 µM (reported value)</td>
      <td>Binding Buffer (20 mM Tris (pH 7.6), 300 mM NaCl, 5 mM MgCl2)</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>15261.94 g/mole</td>
      <td>55.10%</td>
      <td>463600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>49</td>
      <td>Vianini et al. "In vitro selection of DNA aptamers that bind L-tyrosinamide." Bioorganic & Medicinal Chemistry, 9 (2001): 2543-2548.</td>
      <td>dAp,dAp,dTp,dTp,dCp,dGp,dCp,dTp,dAp,dGp,dCp,dTp,dGp,dGp,dAp,dGp,dCp,dTp,dTp,dGp,dGp,dAp,dTp,dTp,dGp,dAp,dTp,dGp,dTp,dGp,dGp,dTp,dGp,dTp,dGp,dTp,dGp,dAp,dGp,dTp,dGp,dCp,dGp,dGp,dTp,dGp,dCp,dCp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>176</th>
      <td>V. parahemolyticus (A3P)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/285.png" width="120" ></td>
      <td>Vibrio parahemolyticus</td>
      <td>Cells</td>
      <td>16.88 nM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td></td>
      <td>26893.59 g/mole</td>
      <td>41.38%</td>
      <td>870500 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>87</td>
      <td>None</td>
      <td>dAp,dTp,dAp,dGp,dGp,dAp,dGp,dTp,dCp,dAp,dCp,dGp,dAp,dCp,dGp,dAp,dCp,dCp,dAp,dGp,dAp,dAp,dTp,dCp,dTp,dAp,dAp,dAp,dAp,dAp,dTp,dGp,dGp,dGp,dCp,dAp,dAp,dAp,dGp,dAp,dAp,dAp,dCp,dAp,dGp,dTp,dGp,dAp,dCp,dTp,dCp,dGp,dTp,dTp,dGp,dAp,dGp,dAp,dTp,dAp,dCp,dTp,dTp,dAp,dTp,dGp,dTp,dGp,dCp,dGp,dTp,dCp,dTp,dAp,dCp,dCp,dTp,dCp,dTp,dTp,dGp,dAp,dCp,dTp,dAp,dAp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>177</th>
      <td>L-Citrulline (44Cit11)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/109.png" width="120" ></td>
      <td>L-Citrulline</td>
      <td>Small Organic</td>
      <td>68 µM (reported value)</td>
      <td>Binding Buffer (250 mM NaCl, 50 mM Tris-HCl (pH=8.3), 5 mM MgCl2)</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>14217.6 g/mole</td>
      <td>52.27%</td>
      <td>445300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>44</td>
      <td>Famulok, M., "  Molecular Recognition of Amino Acids by RNA-Aptamers: An L-Citrulline Binding RNA Motif and its Evolution into an L-Arginine Binder." Journal of the American Chemical Society, 116 (1994): 1698-1706.</td>
      <td>rGp,rAp,rCp,rGp,rAp,rGp,rAp,rAp,rGp,rGp,rAp,rGp,rUp,rGp,rCp,rUp,rGp,rGp,rUp,rUp,rAp,rUp,rAp,rCp,rUp,rAp,rGp,rCp,rGp,rGp,rUp,rUp,rAp,rGp,rGp,rUp,rCp,rAp,rCp,rUp,rCp,rGp,rUp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>178</th>
      <td>IR-A43</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7158.jpg" width="120" ></td>
      <td>recombinant insulin receptor extracellular domain</td>
      <td>Protein</td>
      <td>70 nM (reported value)</td>
      <td>40 mM HEPES (pH 7.5), 102 mM NaCl, 5 mM KCl, 5 mM MgCl2, and\n0.05% Tween-20</td>
      <td>22°C</td>
      <td>None</td>
      <td></td>
      <td>22946.6 g/mole</td>
      <td>60.00%</td>
      <td>711800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>81</td>
      <td>Yunn NO, Park M, Park S, Lee J, Noh J, Shin E, Ryu SH. A hotspot for enhancing insulin receptor activation revealed by a conformation-specific allosteric aptamer. Nucleic Acids Res. 2021 Jan 25;49(2):700-712. doi: 10.1093/nar/gkaa1247. PMID: 33410883; PMCID: PMC7826266.</td>
      <td>dTp,dAp,dTp,dGp,dAp,dGp,dTp,dGp,dAp,dCp,dCp,dGp,dTp,dCp,dCp,dGp,dCp,dCp,dTp,dGp,dAp,dCp,dCp,dGp,dCp,dAp,dGp,dAp,dCp,dGp,dGp,dCp,dAp,dCp,dAp,dGp,dCp,dGp,dAp,dCp,dCp,dCp,dGp,dGp,dGp,dCp,dCp,dAp,dCp,dGp,dAp,dCp,dAp,dAp,dCp,dAp,dGp,dCp,dCp,dAp,dCp,dAp,dCp,dCp,dAp,dCp,dCp,dAp,dGp,dCp,dCp,dAp,dAp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>179</th>
      <td>DNA Blocker (Part of Lysozyme Aptasensor)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/319.png" width="120" ></td>
      <td>Lysozyme</td>
      <td>Other</td>
      <td></td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td>BSA was used as a negative control.</td>
      <td>15128.91 g/mole</td>
      <td>44.90%</td>
      <td>488200 L/(mole·cm)</td>
      <td></td>
      <td>0.1 fM (reported value)</td>
      <td>Contains a lysozyme-binding aptamer (capitalized in the structure) and a region complimentary to DNAzymeMB (highlighted; see related aptamers).</td>
      <td>49</td>
      <td>Fu et al. "An ultrasensitive peroxidase DNAzyme-associated aptasensor that utilizes a target-triggered enzymatic signal amplification strategy." Chemical Communications, 47(2011): 9876-9878.</td>
      <td>dAp,dTp,dCp,dTp,dAp,dCp,dGp,dAp,dAp,dTp,dTp,dCp,dAp,dTp,dCp,dAp,dGp,dGp,dGp,dCp,dTp,dAp,dAp,dAp,dGp,dAp,dGp,dTp,dGp,dCp,dAp,dGp,dAp,dGp,dTp,dTp,dAp,dCp,dTp,dTp,dAp,dGp,dCp,dCp,dCp,dGp,dTp,dGp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>180</th>
      <td>GBM128</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/589.jpg" width="120" ></td>
      <td>Glioblastoma</td>
      <td>Cells</td>
      <td>20 nM (reported value)</td>
      <td>The washing buffer (WB) used contained 4.5 g/L glucose, and  5 mM MgCl2 in PBS (pH = 7.4). The binding buffer used during  selection was prepared by adding yeast tRNA (0.1 mg/mL) and  BSA (1 mg/mL) to washing buffer to reduce background binding.</td>
      <td>4°C</td>
      <td>None</td>
      <td></td>
      <td>25689.73 g/mole</td>
      <td>49.40%</td>
      <td>812600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>277</td>
      <td>Kang D, Wang J, Zhang W, Song Y, Li X, et al. (2012) Selection of DNA Aptamers against Glioblastoma Cells with High Affinity and Specificity. PLoS  ONE 7(10): e42731. doi:10.1371/journal.pone.0042731</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>181</th>
      <td>Human Matrix Metalloprotease 9 (hMMP-9) (F3Bomf)</td>
      <td>Chimeric</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/378.png" width="120" ></td>
      <td>Human matrix metalloprotease 9 (hMMP-9)</td>
      <td>Protein</td>
      <td>20 nM (reported value)</td>
      <td>None</td>
      <td>23°C</td>
      <td>None</td>
      <td>Labels: Grade I Pilocytic astrocytoma, Grade I Meningioma, Grade II Astrocytoma, Grade II Ependymoma, Grade III Anaplastic astrocytoma, Grade IV Glioblastoma, Grade IV medulloblastoma, Grade IV lymphoma.  Does not label healthy brain tissue.</td>
      <td>11563.12 g/mole</td>
      <td>72.22%</td>
      <td>311900 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>This aptamer is a truncated and modified version of FB3</td>
      <td>36</td>
      <td>None</td>
      <td>fUp,fCp,fCp,fCp,fUp,fCp,fCp,fCp,fUp,fCp,fCp,fCp,fCp,fUp,fUp,fCp,fCp,fUp,fCp,fCp,fCp,fCp,fCp,fCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>182</th>
      <td>Isocarbophos (SS4-54)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/298.png" width="120" ></td>
      <td>Isocarbophos</td>
      <td>Small Organic</td>
      <td>0.9 µM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td>This aptamer binds preferentially to the organophosphates phorate, profenofos, benzoate and omethoate. It has reduced binding to imidacloprid, acetamiprid, chlorpyrifos, methamidophos, dichlorvos, parathion, dimethoate, monocrotophos and phoxim.</td>
      <td>16675.9 g/mole</td>
      <td>46.30%</td>
      <td>517600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>"A molecular beacon (5' FAM-CTGCACAAGAATCGCTGCAG-3') was designed, which was partly complementary with the middle constant nucleotides-sequence of the selected ssDNAs."</td>
      <td>54</td>
      <td>None</td>
      <td>dAp,dAp,dGp,dCp,dTp,dTp,dTp,dTp,dTp,dTp,dGp,dAp,dCp,dTp,dGp,dAp,dCp,dTp,dGp,dCp,dAp,dGp,dCp,dGp,dAp,dTp,dTp,dCp,dTp,dTp,dGp,dAp,dTp,dCp,dGp,dCp,dCp,dAp,dCp,dGp,dGp,dTp,dCp,dTp,dGp,dGp,dAp,dAp,dAp,dAp,dAp,dGp,dAp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>183</th>
      <td>Nestin Protein expressed in Brain Tumors (clone 17)</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/257.png" width="120" ></td>
      <td>Nestin Protein expressed in Brain Tumors</td>
      <td>Protein</td>
      <td>N/A nM (reported value)</td>
      <td>Binding Buffer (PBS containing 1% BSA)    Wash Buffer (PBS with 0.05% Tween 20), 4°C</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>837.98 g/mole</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>7</td>
      <td>Beck et al. "Indentification of a peptide that interacts with Nestin protein expressed in brain cancer cells." Biomaterials, 32(2011): 8518-8528.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>184</th>
      <td>Thrombin (29mer)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/315.png" width="120" ></td>
      <td>Thrombin</td>
      <td>Protein</td>
      <td>0.5 nM (reported value)</td>
      <td>None</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>9085.94 g/mole</td>
      <td>62.07%</td>
      <td>283600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Image does not show G quartet formation.  The 29mer version has higher affinity than the 15mer version, but less inhibitory activity.</td>
      <td>32</td>
      <td>Tasset, D. (1997). Oligonucleotide inhibitors of human thrombin that bind distinct epitopes. J. Mol. Biol., 1997(272), 688-698.</td>
      <td>dAp,dGp,dTp,dCp,dCp,dGp,dTp,dGp,dGp,dTp,dAp,dGp,dGp,dGp,dCp,dAp,dGp,dGp,dTp,dTp,dGp,dGp,dGp,dGp,dTp,dGp,dAp,dCp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>185</th>
      <td>U31</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/524.jpg" width="120" ></td>
      <td>U87-EGFRvIII</td>
      <td>Cells</td>
      <td>8.1 nM (reported value)</td>
      <td>0.45 g of glucose, 10 mg yeast tRNA, 5 mg salmon sperm DNA, 0.1 g BSA and 0.5 ml of 1 M MgCl2 in 100 ml of 1X phosphate-buffered saline  [PBS]</td>
      <td>°C</td>
      <td>None</td>
      <td></td>
      <td>23503.34 g/mole</td>
      <td>42.11%</td>
      <td>719900 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>228</td>
      <td>Wu, X. et al. "Cell-SELEX Aptamer for Highly Specific Radionuclide Molecular Imaging of Glioblastoma In Vivo." PLOS One 2014, 9:3.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>186</th>
      <td>Apt-3</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7173.jpg" width="120" ></td>
      <td>C promoter binding factor 1 (CBF1)</td>
      <td>Protein</td>
      <td>34 nM (reported value)</td>
      <td>100 mM Tris-HCl (pH 8.0), 100 mM NaCl, 5 mM KCl, 1 mM MgCl2, 0.1% Tween 20, 1 mg/mL BSA</td>
      <td>22°C</td>
      <td>None</td>
      <td></td>
      <td>22004.3 g/mole</td>
      <td>54.17%</td>
      <td>693600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>73</td>
      <td>Tezuka-Kagajo, Mari et al. “Development of Human CBF1-Targeting Single-Stranded DNA Aptamers with Antiangiogenic Activity In Vitro.” Nucleic acid therapeutics vol. 30,6 (2020): 365-378. doi:10.1089/nat.2020.0875</td>
      <td>dAp,dTp,dAp,dGp,dGp,dAp,dGp,dTp,dCp,dGp,dAp,dCp,dCp,dGp,dAp,dCp,dCp,dAp,dGp,dAp,dCp,dCp,dAp,dAp,dCp,dCp,dCp,dTp,dAp,dCp,dGp,dCp,dGp,dTp,dAp,dCp,dCp,dAp,dAp,dCp,dCp,dAp,dGp,dAp,dTp,dGp,dAp,dCp,dCp,dTp,dAp,dCp,dGp,dTp,dGp,dCp,dGp,dTp,dCp,dTp,dAp,dCp,dAp,dTp,dCp,dTp,dAp,dGp,dAp,dCp,dTp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>187</th>
      <td>FMN-induced molecular switch</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/395.png" width="120" ></td>
      <td>flavin mononucleotide (FMN)</td>
      <td>Small Organic</td>
      <td>na nM (reported value)</td>
      <td>Reaction buffer [50 mM Tris.HCl (pH 7.5) and 20 mM MgCl2] for 20 hr</td>
      <td>23°C</td>
      <td>None</td>
      <td></td>
      <td>23333.13 g/mole</td>
      <td>58.33%</td>
      <td>710900 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>72</td>
      <td>Garrett A. Soukup and Ronald R. Breaker. "Engineering precision RNA molecular switches" Proc. Natl. Acad. Sci. 96(1999):3584-3589.</td>
      <td>rGp,rGp,rGp,rCp,rGp,rAp,rCp,rCp,rCp,rUp,rGp,rAp,rUp,rGp,rAp,rGp,rCp,rCp,rUp,rUp,rAp,rGp,rGp,rAp,rUp,rAp,rUp,rGp,rCp,rUp,rUp,rCp,rGp,rGp,rCp,rAp,rGp,rAp,rAp,rGp,rGp,rAp,rCp,rGp,rUp,rCp,rGp,rAp,rAp,rAp,rCp,rGp,rGp,rUp,rGp,rAp,rAp,rAp,rGp,rCp,rCp,rGp,rUp,rAp,rGp,rGp,rUp,rUp,rGp,rCp,rCp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>188</th>
      <td>FMN-inhibited molecular switch</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/400.png" width="120" ></td>
      <td>flavin mononucleotide (FMN)</td>
      <td>Small Organic</td>
      <td>na nM (reported value)</td>
      <td>Reaction buffer [50 mM Tris.HCl (pH 7.5) and 20 mM MgCl2] for 20 hr</td>
      <td>23°C</td>
      <td>None</td>
      <td></td>
      <td>23027.95 g/mole</td>
      <td>57.75%</td>
      <td>702300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>72</td>
      <td>None</td>
      <td>rGp,rGp,rGp,rCp,rGp,rAp,rCp,rCp,rCp,rUp,rGp,rAp,rUp,rGp,rAp,rGp,rAp,rUp,rGp,rAp,rGp,rGp,rAp,rUp,rAp,rUp,rGp,rCp,rUp,rUp,rCp,rGp,rGp,rCp,rAp,rGp,rAp,rAp,rGp,rGp,rCp,rUp,rCp,rUp,rCp,rGp,rAp,rAp,rAp,rCp,rGp,rGp,rUp,rGp,rAp,rAp,rAp,rGp,rCp,rCp,rGp,rUp,rAp,rGp,rGp,rUp,rUp,rGp,rCp,rCp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>189</th>
      <td>Glucose-bisboronic acid aptamer-sensor</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7102.jpg" width="120" ></td>
      <td>Glucose</td>
      <td>Small Organic</td>
      <td>5 nM (reported value)</td>
      <td>Shinkai's receptor 50uM with 2% methanol.</td>
      <td>24 °C°C</td>
      <td>\n                                Mix and incubate for 5 min at 95°C, and cool down at room temperature 24°C > 30 min \n</td>
      <td>N/A</td>
      <td>13478.75 g/mole</td>
      <td>61.36%</td>
      <td>453870 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>sensor mixtures\n(fluorescein-labeled aptamer + competitor dabcyl-labeled oligonucleotide) to the addition of Shinkai’s\nreceptor in the presence of increasing amounts of sugar, with the actual concentration of a\nreceptor*complex at any particular glucose concentrations calculated from the Kd of Shinkai’s receptor.</td>
      <td>44</td>
      <td>Yang KA, Barbu M, Halim M, et al. Recognition and sensing of low-epitope targets via ternary complexes with oligonucleotides and synthetic receptors. Nat Chem. 2014;6(11):1003-1008. doi:10.1038/nchem.2058</td>
      <td>dCp,dTp,dCp,dTp,dCp,dGp,dGp,dGp,dAp,dCp,dGp,dAp,dCp,dAp,dGp,dCp,dCp,dGp,dAp,dGp,dTp,dTp,dGp,dAp,dTp,dTp,dCp,dAp,dAp,dCp,dAp,dGp,dCp,dCp,dGp,dAp,dGp,dTp,dCp,dGp,dTp,dCp,dCp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>190</th>
      <td>Antibac1</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/624.jpg" width="120" ></td>
      <td>peptidoglycan</td>
      <td>Other</td>
      <td>268.5 nM (reported value)</td>
      <td>Selection buffer (PBS containing 1.4 mM MgCl2 and 0.05% BSA); Room temperature (45 mins)</td>
      <td>°C</td>
      <td>None</td>
      <td></td>
      <td>16833.83 g/mole</td>
      <td>72.73%</td>
      <td>486900 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Antibac 1 was tested on the strains A. baumanni, L. monocitogenes, E. coli, S. aureus, and K. pneumonia, among others. The Kd listed above is for A. baumanni.</td>
      <td>55</td>
      <td>A Graziani et al. High efficiency binding aptamers for a wide range of sepsis 1 bacterial agents. JMB Papers in Press. 2017: JMB16-11004.</td>
      <td>dTp,dCp,dGp,dCp,dGp,dCp,dGp,dAp,dGp,dTp,dCp,dGp,dTp,dCp,dTp,dGp,dGp,dGp,dGp,dAp,dCp,dAp,dGp,dGp,dGp,dAp,dGp,dTp,dGp,dCp,dGp,dCp,dTp,dGp,dCp,dTp,dCp,dCp,dCp,dCp,dCp,dCp,dGp,dCp,dAp,dTp,dCp,dGp,dTp,dCp,dCp,dTp,dCp,dCp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>191</th>
      <td>Escherichia coli (E. coli) O157</td>
      <td>Chimeric</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/269.png" width="120" ></td>
      <td>Escherichia coli (E. coli) O157:H7</td>
      <td>Cells</td>
      <td>110 nM (reported value)</td>
      <td>Binding Buffer (30 mM Trisƒ??HCl (pH 7.5), 150 mM NaCl, 1.5 mM MgCl2, 2 mM dithiothreitol, and 1% BSA). Incubated at 37°C for 24 hours.</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>20656.14 g/mole</td>
      <td>40.63%</td>
      <td>662700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>64</td>
      <td>Lee at al. "In vitro selection of Escherichia coli O157:H7-specific RNA aptamer." Biochemical and Biophysical Research Communications, 417(2012): 214-220.</td>
      <td>rGp,rGp,rGp,fUp,fCp,fUp,fUp,fCp,fCp,fUp,rGp,rGp,rAp,fCp,fUp,rGp,fUp,fCp,rGp,rAp,rAp,rAp,rAp,fUp,fUp,fCp,rAp,rGp,fUp,rAp,fUp,fCp,rGp,rGp,rGp,rAp,rGp,rGp,fUp,fUp,rAp,fCp,rGp,fUp,rAp,fUp,fUp,fUp,rGp,rGp,fUp,fUp,fUp,rAp,fUp,rAp,rGp,rAp,fUp,rAp,rGp,fUp,rAp,rAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>192</th>
      <td>Human Influenza A virus H3N2 (P30-10-16)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/121.jpg" width="120" ></td>
      <td>Human Influenza A virus H3N2</td>
      <td>Cells</td>
      <td>0.188 nM (reported value)</td>
      <td>Binding Buffer (20 nM HEPES/KOH (pH 7?ú4), 105 mM NaCl)    PIPES buffer (5 mM PIPES/NaOH, 145 mM NaCl (pH 7?ú5))    Acidic buffer (145 mM NaCl, 20 mM sodium citrate, 1 U RNase inhibitor µl-1 (pH 5?ú0))</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>23621.16 g/mole</td>
      <td>54.05%</td>
      <td>710100 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>74</td>
      <td>Gopinath, S., et al. "An RNA aptamer that distinguishes between closely related human influenza viruses and inhibits haemagglutinin-mediated membrane fusion." Journal of General Virology, 87(2005): 479-487.</td>
      <td>rGp,rGp,rGp,rAp,rGp,rAp,rAp,rUp,rUp,rCp,rCp,rGp,rAp,rCp,rCp,rAp,rGp,rAp,rAp,rGp,rGp,rGp,rUp,rUp,rAp,rGp,rCp,rAp,rGp,rUp,rCp,rGp,rGp,rCp,rAp,rUp,rGp,rCp,rGp,rGp,rUp,rAp,rCp,rAp,rGp,rAp,rCp,rAp,rGp,rAp,rCp,rCp,rUp,rUp,rUp,rCp,rCp,rUp,rCp,rUp,rCp,rUp,rCp,rCp,rUp,rUp,rCp,rCp,rUp,rCp,rUp,rUp,rCp,rUp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>193</th>
      <td>A23.2</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/559.png" width="120" ></td>
      <td>Staphylococcus aureus enterotoxin A (SEA)</td>
      <td>Protein</td>
      <td>235 nM (reported value)</td>
      <td>20 mM Tris, 100 mM NaCl, 5 mM KCl, 1 mM CaCl2 and 1 mM MgCl2 ƒ?› 6H2O, pH 7.4</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>24732.2 g/mole</td>
      <td>38.75%</td>
      <td>804300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>ssDNA aptamers specific for SEA were selected using a 12-round SELEX procedure</td>
      <td>240</td>
      <td>Huang, Y. et al. "Selection, identification and application of a DNA aptamer against Staphylococcus aureus enterotoxin A." Anal. Methods, 2014, 6, 690-697.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>194</th>
      <td>Lysozyme (Apt 60)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/457.png" width="120" ></td>
      <td>Hen egg white lysozyme and recombinant human lysozyme</td>
      <td>Protein</td>
      <td>3-Feb nM (reported value)</td>
      <td>None</td>
      <td>25°C</td>
      <td>None</td>
      <td>Tran et al. found good specificity against BSA, ovomucoid, and ovotransferrin in capillary electrophoresis assays.</td>
      <td>18587.09 g/mole</td>
      <td>58.33%</td>
      <td>590400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>This a truncated version of the 80-mer aptamer developed by Tran et al. It was found to have a comparable binding affinity to the 80-mer version by Zou et al. Further truncations to 50- and 40-nucleotides reduced the binding affinity to ~15 nM.</td>
      <td>64</td>
      <td>None</td>
      <td>dAp,dGp,dCp,dAp,dGp,dCp,dAp,dCp,dAp,dGp,dAp,dGp,dGp,dTp,dCp,dAp,dGp,dAp,dTp,dGp,dGp,dCp,dAp,dGp,dGp,dTp,dAp,dAp,dGp,dCp,dAp,dGp,dGp,dCp,dGp,dGp,dCp,dTp,dCp,dAp,dCp,dAp,dAp,dAp,dAp,dCp,dCp,dAp,dTp,dTp,dCp,dGp,dCp,dAp,dTp,dGp,dCp,dGp,dGp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>195</th>
      <td>OX40 (Aptamer 9.8)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/360.png" width="120" ></td>
      <td>Murine OX40 (a tumor necrosis factor (TNF) receptor on the surface of T Cells)</td>
      <td>Protein</td>
      <td>8 nM (reported value)</td>
      <td>None</td>
      <td>37°C</td>
      <td>\n                                Heated to 95°C for 5 min, then cooled slowly to room temperature. \n</td>
      <td></td>
      <td>25857.45 g/mole</td>
      <td>58.75%</td>
      <td>783300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>82</td>
      <td>None</td>
      <td>rGp,rGp,rGp,rAp,rGp,rGp,rAp,fCp,rGp,rAp,fUp,rGp,fCp,rGp,rGp,fCp,rAp,rGp,fUp,fCp,fUp,rGp,fCp,rAp,fUp,fCp,rGp,fUp,rAp,rGp,rGp,rAp,rAp,fUp,fCp,rGp,fCp,fCp,rAp,fCp,fCp,rGp,fUp,rAp,fUp,rAp,fCp,fUp,fUp,fUp,fCp,fCp,fCp,rAp,fCp,fCp,rAp,rGp,rAp,fCp,rGp,rAp,fCp,fUp,fCp,rGp,fCp,fUp,rGp,rAp,rGp,rGp,rAp,fUp,fCp,fCp,rGp,rAp,rGp,rAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>196</th>
      <td>HER-2 (HB5)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/380.png" width="120" ></td>
      <td>HER2 peptide</td>
      <td>Protein</td>
      <td>18.9 nM (reported value)</td>
      <td>Hanks buffer with 0.1 mg/mL salmon sperm DNA and 1 mg/mL of BSA</td>
      <td>37°C</td>
      <td>None</td>
      <td>None</td>
      <td>26271.12 g/mole</td>
      <td>43.02%</td>
      <td>811300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Aptamer was complexed with doxorubicin and selectively delivered doxorubicin to HER2-positive cells in vitro.</td>
      <td>89</td>
      <td>None</td>
      <td>dAp,dAp,dCp,dCp,dGp,dCp,dCp,dCp,dAp,dAp,dAp,dTp,dCp,dCp,dCp,dTp,dAp,dAp,dGp,dAp,dGp,dTp,dCp,dTp,dGp,dCp,dAp,dCp,dTp,dTp,dGp,dTp,dCp,dAp,dTp,dTp,dTp,dTp,dGp,dTp,dAp,dTp,dAp,dTp,dGp,dTp,dAp,dTp,dTp,dTp,dGp,dGp,dTp,dTp,dTp,dTp,dTp,dGp,dGp,dCp,dTp,dCp,dTp,dCp,dAp,dCp,dAp,dGp,dAp,dCp,dAp,dCp,dAp,dCp,dTp,dAp,dCp,dAp,dCp,dAp,dCp,dGp,dCp,dAp,dCp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>197</th>
      <td>Cd(II)-specific aptamer (CAP)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/627.jpg" width="120" ></td>
      <td>Cadmium (II)</td>
      <td>Other</td>
      <td>2.15 nM (reported value)</td>
      <td>4-(2-Hydroxyethyl)piperazine-1-ethanesulfonic acid sodium salt (HEPES) buffer (pH 7.3).</td>
      <td>23°C</td>
      <td>\n                                N/A \n</td>
      <td></td>
      <td>10916.17 g/mole</td>
      <td>42.86%</td>
      <td>332500 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>35</td>
      <td>Zhu, YF., Wang, YS., Zhou, B. et al. Anal Bioanal Chem (2017). doi:10.1007/s00216-017-0436-1.</td>
      <td>dGp,dGp,dGp,dGp,dAp,dCp,dTp,dGp,dTp,dTp,dGp,dTp,dGp,dGp,dTp,dAp,dTp,dTp,dAp,dTp,dTp,dTp,dTp,dTp,dGp,dGp,dTp,dTp,dGp,dTp,dGp,dCp,dAp,dGp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>198</th>
      <td>C-Reactive Protein</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/454.png" width="120" ></td>
      <td>C-Reactive Protein (CRP)</td>
      <td>Protein</td>
      <td>125 nM (reported value)</td>
      <td>SPR Binding Buffer: HEPES at pH 6.5 with 0.005% Tween 20</td>
      <td>25°C</td>
      <td>None</td>
      <td>None</td>
      <td>14346.65 g/mole</td>
      <td>56.82%</td>
      <td>444700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>44</td>
      <td>None</td>
      <td>rGp,rCp,rCp,rUp,rGp,rUp,rAp,rAp,rGp,rGp,rUp,rGp,rGp,rUp,rCp,rGp,rGp,rUp,rGp,rUp,rGp,rGp,rCp,rGp,rAp,rGp,rUp,rGp,rUp,rGp,rUp,rUp,rAp,rGp,rGp,rAp,rGp,rAp,rGp,rAp,rUp,rUp,rGp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>199</th>
      <td>a-Bungarotoxin (Clone 24)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/363.png" width="120" ></td>
      <td>a-Bungarotoxin</td>
      <td>Protein</td>
      <td>7.5 µM (reported value)</td>
      <td>None</td>
      <td>24°C</td>
      <td>\n                                Heated to 90°C for 5 min, then cooled on ice. \n</td>
      <td></td>
      <td>12466.15 g/mole</td>
      <td>55.00%</td>
      <td>398100 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Selected using Monolex (1-step selection method).</td>
      <td>41</td>
      <td>None</td>
      <td>dGp,dCp,dGp,dAp,dGp,dGp,dTp,dGp,dTp,dTp,dCp,dGp,dAp,dGp,dAp,dGp,dTp,dTp,dAp,dGp,dGp,dGp,dGp,dCp,dGp,dAp,dCp,dAp,dTp,dGp,dAp,dCp,dCp,dAp,dAp,dAp,dCp,dGp,dTp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>200</th>
      <td>Gentamycin (LS13)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/434.png" width="120" ></td>
      <td>Gentamycin and other Aminoglycosides (AMGs)</td>
      <td>Small Organic</td>
      <td>26.7 nM (reported value)</td>
      <td>Binding Buffer: PBS with 10% (v/v) methanol</td>
      <td>RT°C</td>
      <td>None</td>
      <td>Binds to several aminoglycosides: kanamycin, paromycin streptomycin, apramycin, tobramycin, neomycin, and gentamycin.</td>
      <td>25474.89 g/mole</td>
      <td>46.84%</td>
      <td>788000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>87</td>
      <td>None</td>
      <td>rGp,rGp,rGp,rAp,rGp,rAp,rAp,rGp,rGp,fCp,rGp,rGp,fCp,rGp,fCp,rGp,fUp,rAp,rGp,rGp,fCp,rGp,rAp,rGp,fCp,fUp,fUp,fUp,rAp,fCp,fCp,rAp,rGp,fUp,fUp,fUp,fUp,rAp,fUp,fUp,fUp,rGp,fUp,fUp,fUp,fUp,rAp,fUp,fUp,rGp,fUp,fUp,rAp,fUp,rAp,fUp,rGp,fCp,fUp,fUp,rAp,fUp,fUp,fCp,rGp,fUp,rGp,rGp,rAp,fCp,rGp,rAp,fUp,rGp,fCp,rGp,fCp,rAp,fCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>201</th>
      <td>A09</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/568.jpg" width="120" ></td>
      <td>Penetrating the Blood Brain Barrier (BBB)</td>
      <td>Other</td>
      <td>nM (reported value)</td>
      <td>10mM HEPES, pH 7.4, 50mM NaCl, 1mM MgCl2, 1mM CaCl2, 0.01% BSA</td>
      <td>37°C</td>
      <td>\n                                Denature at 94C for 3 minutes, and then slowly refold at room temperature \n</td>
      <td></td>
      <td>17891.96 g/mole</td>
      <td>78.18%</td>
      <td>521300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>180</td>
      <td>Congsheng Cheng, Yong Hong Chen, Kim A Lennox, Mark A Behlke, Beverly L Davidson, "In vivo SELEX for Identification of Brain-penetrating Aptamers" Molecular Therapy-Nucleic Acids (2013)2, e67, American Society of Gene & Cell Therapy</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>202</th>
      <td>Periostin Aptamer (PNDA-3)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/530.jpg" width="120" ></td>
      <td>Periostin</td>
      <td>Protein</td>
      <td>1.07 nM (reported value)</td>
      <td>40 mM/1 HEPES, 102 mM/1 NaCl, 5 mM/1mol/1g Cl2,5 mM KCl</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>15666.2 g/mole</td>
      <td>52.94%</td>
      <td>483400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>5' benzylaminocarbonyl-d U modification</td>
      <td>154</td>
      <td>Lee, Y., Kim, I., Park, S., Kim, Y., Wu, S., Lee, J., Noh, D., Kim, K., Ryu, S. and Suh, P. "Periostin-binding DNA aptamer inhibits breast cancer growth and metastasis." The American Society of Gene & Cell Therapy 21.5 (2013): 1004-13.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>203</th>
      <td>Interferon-? (2'NH2-30)</td>
      <td>2'-NH2-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/52.jpg" width="120" ></td>
      <td>Interferon-y/IFN-y/IFNG</td>
      <td>Protein</td>
      <td>2.7 nM (reported value)</td>
      <td>None</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>38579.5 g/mole</td>
      <td>55.71%</td>
      <td>710100 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>71</td>
      <td>Kubik, Mark F., et al. "Isolation and Characterization of  2ƒ??-Fluoro-,  2ƒ??-Amino-,  and 2ƒ??-Fluoro-/Amino-Modified  RNA Ligands to  Human  IFN-7 That Inhibit  Receptor  Binding." Journal of Immunology, 159 (1997): 259-267.</td>
      <td>rGp,rGp,rGp,rAp,rGp,rGp,rAp,rGp,rAp,rGp,rGp,rGp,rAp,rGp,rGp,rAp,rAp,rAp,rAp,rGp,rAp,rAp,rGp,rGp,rGp,rGp,rGp,rAp,rGp,rGp,rAp,rAp,rGp,rGp,rGp,rAp,rGp,rAp,rGp,rAp,rGp,rGp,rAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>204</th>
      <td>Kanamycin B (K8)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/117.png" width="120" ></td>
      <td>Kanamycin B</td>
      <td>Small Organic</td>
      <td>180 nM (reported value)</td>
      <td>None</td>
      <td>20°C</td>
      <td>None</td>
      <td>None</td>
      <td>35819.5 g/mole</td>
      <td>52.25%</td>
      <td>1095000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>111</td>
      <td>Kwon et al., "In Vitro Selection of RNA against Kanamycin B." Molecules and Cells, 11 (2001): 303-311.</td>
      <td>rGp,rGp,rGp,rAp,rGp,rCp,rUp,rCp,rGp,rGp,rUp,rAp,rCp,rCp,rGp,rAp,rAp,rUp,rUp,rCp,rUp,rCp,rGp,rCp,rCp,rCp,rUp,rAp,rUp,rAp,rGp,rGp,rGp,rGp,rUp,rGp,rUp,rUp,rGp,rAp,rGp,rGp,rGp,rAp,rAp,rAp,rUp,rGp,rUp,rGp,rUp,rGp,rCp,rGp,rAp,rCp,rAp,rAp,rGp,rGp,rUp,rGp,rCp,rGp,rGp,rUp,rGp,rGp,rCp,rCp,rAp,rGp,rAp,rAp,rCp,rUp,rUp,rUp,rUp,rCp,rGp,rUp,rUp,rCp,rUp,rCp,rAp,rUp,rCp,rAp,rAp,rAp,rAp,rGp,rCp,rUp,rUp,rUp,rGp,rCp,rAp,rGp,rAp,rGp,rGp,rAp,rUp,rCp,rCp,rUp,rUp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>205</th>
      <td>VEGF-stimulated HUVECs</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/231.jpg" width="120" ></td>
      <td>VEGF-stimulated human umbilical vein endothelial cells (HUVECs)</td>
      <td>Tissue</td>
      <td>N/A nM (reported value)</td>
      <td>None</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>2017.31 g/mole</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>15</td>
      <td>None</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>206</th>
      <td>Botulinum neurotoxin type A (BoNT A) Aptamer (S132B-C22)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/528.jpg" width="120" ></td>
      <td>BoNT A Light Chain</td>
      <td>Protein</td>
      <td>87 nM (reported value)</td>
      <td>PBS pH 7.4, 300 mM NaCl, 3mM MgCl2, 5mM DTT, 1mg/ml BSA, 0.2 % Tween, 1mg/ml heparin</td>
      <td>25°C</td>
      <td>\n                                None \n</td>
      <td>Tested for specificity to His-Sec7, His-ERK2, streptavidin, and lysozyme. The aptamer had 5% retention to lysozyme and 3% retention to His-Sec7 (see Fig. 1).</td>
      <td>28858.79 g/mole</td>
      <td>51.61%</td>
      <td>924400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Aptamer was developed to inhibit endopeptidase activity of BoNT A.</td>
      <td>279</td>
      <td>Chang, T., Blank, M., Janardhanan, P., Singh, B., Mello, C., Blind, M., Cai, S. "In vitro selection of RNA aptamers that inhibit the activity of type A botulinum neurotoxin." Biochemical and Biophysical Research Communications 396(2010): 854-860.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>207</th>
      <td>TTA1.2</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7109.png" width="120" ></td>
      <td>Tenascin-C (TN-C)</td>
      <td>Protein</td>
      <td>2.4 nM (reported value)</td>
      <td>TBSMC buffer (20 mM Tris, pH 7.4; 137 mM NaCl; 1 mM CaCl2; 1 mM MgCl2)</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Aptamer was modified post selection from a 2'F library. Note that there is an internal PEG4 linker between G9 and C10. This linker stabilizes the secondary structure and is critical for binding.</td>
      <td>38</td>
      <td>Schmidt KS, Borkowski S, Kurreck J, Stephens AW,\nBald R, Hecht M, et al. Application of locked nucleic\nacids to improve aptamer in vivo stability and targeting function. Nucleic Acids Res 2004; 32:5757-\n65; PMID:15509871; http://dx.doi.org/10.1093/nar/\ngkh862</td>
      <td>fCp,rGp,fCp,rGp,fUp,fCp,rGp,fCp,fCp,rGp,fUp,fUp,fUp,fUp,fUp,fUp,fUp,fUp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>208</th>
      <td>Mouse Lipocalin-2 (mLcn2) (Oligo 569)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/379.png" width="120" ></td>
      <td>Mouse Lipocalin-2 (mLcn2)</td>
      <td>Protein</td>
      <td>340 nM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td>Does not bind to human Lcn2 (NGAL) or the chicken siderophore-binding lipocalin (Ex-FABP).</td>
      <td>13405.11 g/mole</td>
      <td>54.76%</td>
      <td>404800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Kd value given refers to mLcn2 in solution. Kd value for immobilized mLcn2 was 38 nM.</td>
      <td>42</td>
      <td>None</td>
      <td>rCp,rCp,rUp,rCp,rCp,rGp,rGp,rCp,rUp,rCp,rAp,rUp,rAp,rCp,rCp,rUp,rUp,rUp,rUp,rCp,rGp,rAp,rAp,rGp,rAp,rCp,rAp,rAp,rGp,rCp,rUp,rUp,rCp,rGp,rAp,rCp,rAp,rGp,rGp,rAp,rGp,rGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>209</th>
      <td>1-F</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/636.png" width="120" ></td>
      <td>CD117</td>
      <td>Protein</td>
      <td>4.24 nM (reported value)</td>
      <td>1X DPBS</td>
      <td>23°C</td>
      <td>\n                                no-refolding \n</td>
      <td>No binding detected by flow cytometry against cell lines: U937, CA46, 468, LNCAP. No binding detected by fluorescence microscopy against U937.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>79</td>
      <td>Zhao, N, Pei, S-N, Qi, J, Zeng, Z, Iyer, SP, Lin, P, et al. (2015). Oligonucleotide aptamer-drug conjugates for targeted therapy of acute myeloid leukemia. Biomaterials 67: 42-51</td>
      <td>dGp,dAp,dGp,dGp,dCp,dAp,dTp,dAp,dCp,dCp,dAp,dGp,dCp,dTp,dTp,dAp,dTp,dTp,dAp,dTp,dTp,dGp,dGp,dGp,dGp,dCp,dCp,dGp,dGp,dGp,dGp,dCp,dAp,dAp,dGp,dGp,dGp,dGp,dGp,dGp,dGp,dGp,dTp,dAp,dCp,dCp,dGp,dTp,dGp,dGp,dTp,dAp,dGp,dGp,dAp,dCp,dAp,dGp,dAp,dTp,dAp,dGp,dTp,dAp,dAp,dGp,dTp,dGp,dCp,dAp,dAp,dTp,dCp,dTp,dGp,dCp,dGp,dAp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>210</th>
      <td>Chitin (Chi#52)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/480.png" width="120" ></td>
      <td>Chitin</td>
      <td>Other</td>
      <td>N/A nM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>\n                                Heated to 95°C and allowed to cool to RT. \n</td>
      <td>Aptamer also bound to cellulose with 45% efficiency (bound the chitin bead with 73% efficiency). Binding to cellulose was present in each of the selected aptamers and the initial library, suggesting that the primer regions contributed cellulose binding.</td>
      <td>32572.08 g/mole</td>
      <td>60.00%</td>
      <td>1004200 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Highlighted region in the structure represents the 61-nt randomized region. It is unknown whether the aptamer will bind without primers.</td>
      <td>118</td>
      <td>None</td>
      <td>dTp,dAp,dGp,dGp,dGp,dAp,dAp,dTp,dTp,dCp,dGp,dTp,dCp,dGp,dAp,dCp,dGp,dGp,dAp,dTp,dCp,dCp,dCp,dCp,dTp,dAp,dAp,dGp,dGp,dGp,dGp,dGp,dGp,dAp,dCp,dTp,dCp,dAp,dGp,dCp,dAp,dTp,dTp,dTp,dTp,dGp,dTp,dGp,dCp,dGp,dGp,dGp,dCp,dGp,dGp,dCp,dGp,dCp,dTp,dAp,dAp,dCp,dAp,dCp,dAp,dAp,dTp,dCp,dAp,dGp,dAp,dTp,dAp,dGp,dAp,dGp,dCp,dGp,dGp,dGp,dGp,dTp,dTp,dCp,dTp,dCp,dCp,dAp,dGp,dGp,dTp,dCp,dGp,dAp,dCp,dGp,dCp,dAp,dTp,dGp,dCp,dGp,dCp,dCp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>211</th>
      <td>L-Arginine (ag.06)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/439" width="120" ></td>
      <td>L-Arginine</td>
      <td>Small Organic</td>
      <td>330 nM (reported value)</td>
      <td>None</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>31347.87 g/mole</td>
      <td>55.67%</td>
      <td>952800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>97</td>
      <td>None</td>
      <td>rGp,rGp,rAp,rGp,rCp,rUp,rCp,rAp,rGp,rCp,rCp,rUp,rUp,rCp,rAp,rCp,rUp,rGp,rCp,rAp,rUp,rGp,rAp,rUp,rAp,rAp,rAp,rCp,rCp,rGp,rAp,rUp,rGp,rCp,rUp,rGp,rGp,rGp,rCp,rGp,rAp,rUp,rUp,rCp,rUp,rCp,rCp,rUp,rGp,rAp,rAp,rGp,rUp,rAp,rGp,rGp,rGp,rGp,rAp,rAp,rGp,rAp,rGp,rUp,rUp,rGp,rUp,rCp,rAp,rUp,rGp,rUp,rAp,rUp,rGp,rGp,rGp,rGp,rGp,rCp,rAp,rCp,rCp,rAp,rCp,rGp,rGp,rUp,rCp,rGp,rGp,rAp,rUp,rCp,rCp,rUp,rGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>212</th>
      <td>RNase H1 (VI-2)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/115.png" width="120" ></td>
      <td>RNase H1</td>
      <td>Protein</td>
      <td>11 nM (reported value)</td>
      <td>Buffer A (30 mM Trisƒ??HCl (pH 8.5), 40 mM ammonium sulfate, 20 mM magnesium chloride, and 0.01% 2-mercaptoethanol)</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>12149.88 g/mole</td>
      <td>66.67%</td>
      <td>356700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>39</td>
      <td>Pileur, et al. "Selective inhibitory DNA aptamers of the human RNase H1." Nucleic Acids Research, 31 (2003): 5776-5788.</td>
      <td>dCp,dGp,dGp,dTp,dCp,dGp,dCp,dTp,dCp,dCp,dGp,dTp,dGp,dTp,dGp,dGp,dCp,dTp,dTp,dGp,dGp,dGp,dTp,dTp,dGp,dGp,dGp,dTp,dGp,dTp,dGp,dGp,dCp,dAp,dGp,dTp,dGp,dAp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>213</th>
      <td>HupB-4T</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/637.png" width="120" ></td>
      <td>HupB</td>
      <td>Protein</td>
      <td>1.72 µM (reported value)</td>
      <td>10 mM Tris pH 7.5, 10 mM MgCl2, 50 mM KCl, 25 mM NaCl</td>
      <td>23°C</td>
      <td>None</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>The aptamer forms a G-Quadruplex</td>
      <td>21</td>
      <td>None</td>
      <td>dGp,dGp,dGp,dTp,dAp,dGp,dCp,dAp,dGp,dAp,dTp,dGp,dGp,dGp,dGp,dGp,dGp,dGp,dGp,dGp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>214</th>
      <td>Moenomycin A (C2)</td>
      <td>2'-NH2-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/79.png" width="120" ></td>
      <td>Moenomycin A</td>
      <td>Small Organic</td>
      <td>350 nM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td></td>
      <td>25776.83 g/mole</td>
      <td>70.59%</td>
      <td>514500 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>51</td>
      <td>None</td>
      <td>rGp,rGp,rAp,rGp,rGp,rGp,rAp,rGp,rGp,rGp,rAp,rGp,rGp,rAp,rGp,rGp,rGp,rGp,rGp,rAp,rGp,rGp,rGp,rGp,rAp,rGp,rAp,rGp,rGp,rGp,rAp,rGp,rGp,rAp,rGp,rGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>215</th>
      <td>Human Norovirus SMV-19</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/577.jpg" width="120" ></td>
      <td>Human Norovirus</td>
      <td>Other</td>
      <td>191 nM (reported value)</td>
      <td>500 ml of binding buffer [consisting of phosphate buffered saline (pH 7.1) supplemented with 100 mg/L CaCl2, 100 mg/L MgCl2 and 0.05% Tween 20] at Room Temperature</td>
      <td>24°C</td>
      <td>None</td>
      <td>See page 8 of source material. Aptamer specific to several strains of Human Norovirus.</td>
      <td>25028.31 g/mole</td>
      <td>46.91%</td>
      <td>788400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>245</td>
      <td>Escudero-Abarca BI, Suh SH, Moore MD, Dwivedi HP, Jaykus L-A (2014) Selection, Characterization and Application of Nucleic Acid Aptamers for the Capture and Detection of Human Norovirus Strains. PLoS ONE 9(9): e106805. doi:10.1371/journal.pone.0106805</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>216</th>
      <td>Prion Protein</td>
      <td>2'-NH2-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/156.jpg" width="120" ></td>
      <td>Prion Protein</td>
      <td>Protein</td>
      <td>100 nM (reported value)</td>
      <td>Binding Buffer (8um urea, 20 mm sodium phosphate, 500 mm sodium chloride (pH 7.8))    Selection Buffer (155 mM NaCl, 1.1 mM KH2PO4, 3.0 mM Na2HPO4, 1.0 mM MgCl2 (pH 7.4))</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>42720.3 g/mole</td>
      <td>56.98%</td>
      <td>889500 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>86</td>
      <td>Proske et al. "Prion-Protein-Specific Aptamer Reduces PrPSc Formation." Chem-Biochem, 3(2002): 717-725.</td>
      <td>rGp,rGp,rGp,rAp,rGp,rAp,rAp,rAp,rGp,rGp,rGp,rAp,rAp,rGp,rGp,rAp,rGp,rGp,rGp,rAp,rGp,rGp,rAp,rGp,rGp,rGp,rAp,rGp,rGp,rAp,rGp,rGp,rAp,rAp,rGp,rGp,rGp,rGp,rGp,rGp,rGp,rGp,rGp,rAp,rGp,rAp,rAp,rGp,rAp,rAp,rGp,rGp,rGp,rAp,rGp,rGp,rAp,rGp,rGp,rAp,rAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>217</th>
      <td>GVI-7</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7140.png" width="120" ></td>
      <td>GCRV-infected CIK</td>
      <td>Cells</td>
      <td>176.63 nM (reported value)</td>
      <td>Binding Buffer [10% FBS] from Life Technologies.</td>
      <td>4°C</td>
      <td>None</td>
      <td></td>
      <td>27920.1 g/mole</td>
      <td>45.56%</td>
      <td>874000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Grass Carp Reovirus Infected Cytokine-Induced Killer Cells (GCRV-infected CIK cells). Grass carp (Ctenopharyngon idellus) kidney cells (CIK).</td>
      <td>90</td>
      <td>Yu, Qing et al. “Generation and characterization of aptamers against grass carp reovirus infection for the development of rapid detection assay.” Journal of fish diseases vol. 44,1 (2021): 33-44. doi:10.1111/jfd.13265</td>
      <td>dGp,dTp,dCp,dTp,dGp,dAp,dAp,dGp,dTp,dAp,dGp,dAp,dCp,dGp,dCp,dAp,dGp,dGp,dAp,dGp,dTp,dGp,dAp,dAp,dCp,dCp,dCp,dAp,dCp,dCp,dTp,dCp,dAp,dGp,dGp,dGp,dCp,dAp,dTp,dCp,dTp,dTp,dAp,dCp,dAp,dTp,dTp,dTp,dCp,dTp,dTp,dCp,dTp,dAp,dAp,dGp,dTp,dTp,dGp,dTp,dTp,dAp,dCp,dCp,dAp,dTp,dGp,dTp,dTp,dTp,dAp,dGp,dTp,dCp,dAp,dCp,dAp,dCp,dCp,dTp,dGp,dAp,dGp,dTp,dAp,dAp,dGp,dCp,dGp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>218</th>
      <td>Mn2+ (AR1)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/208.jpg" width="120" ></td>
      <td>Mn2+</td>
      <td>Small Organic</td>
      <td>~1 mM (reported value)</td>
      <td>None</td>
      <td>23°C</td>
      <td>None</td>
      <td></td>
      <td>31742.3 g/mole</td>
      <td>54.08%</td>
      <td>985700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>98</td>
      <td>Seetharaman et al. "Immobolized RNA switches for the analysis of complex chemical and biological mixtures." Nature Biotechnology, 19(2001): 336-341.</td>
      <td>rGp,rGp,rAp,rUp,rAp,rAp,rUp,rAp,rGp,rCp,rCp,rGp,rUp,rAp,rGp,rGp,rUp,rUp,rGp,rCp,rGp,rAp,rAp,rAp,rGp,rCp,rGp,rAp,rCp,rCp,rCp,rUp,rGp,rAp,rUp,rGp,rAp,rGp,rAp,rAp,rAp,rGp,rCp,rCp,rAp,rAp,rAp,rGp,rCp,rCp,rGp,rUp,rAp,rGp,rCp,rGp,rCp,rAp,rGp,rAp,rUp,rGp,rAp,rUp,rCp,rUp,rCp,rGp,rCp,rCp,rAp,rUp,rCp,rAp,rGp,rUp,rAp,rCp,rCp,rGp,rAp,rAp,rAp,rCp,rGp,rGp,rUp,rAp,rGp,rCp,rGp,rAp,rGp,rAp,rGp,rCp,rUp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>219</th>
      <td>Influenza B Hemagglutinin (A20)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/443.png" width="120" ></td>
      <td>Influenza B Hemagglutinin (HA)</td>
      <td>Protein</td>
      <td>720 pM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td>No binding observed with Influenza A/Panama and A/Aichi strains.</td>
      <td>36951.24 g/mole</td>
      <td>64.04%</td>
      <td>1096100 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>This aptamer inhibited HA-mediated membrane fusion in human red blood cell ghost membranes.</td>
      <td>114</td>
      <td>None</td>
      <td>rGp,rGp,rGp,rAp,rGp,rCp,rUp,rCp,rAp,rGp,rCp,rCp,rUp,rUp,rCp,rAp,rCp,rUp,rGp,rCp,rAp,rCp,rUp,rCp,rCp,rGp,rGp,rCp,rUp,rGp,rGp,rUp,rGp,rGp,rAp,rCp,rGp,rCp,rGp,rGp,rUp,rAp,rCp,rGp,rAp,rGp,rCp,rAp,rAp,rUp,rUp,rUp,rGp,rUp,rAp,rCp,rCp,rGp,rGp,rAp,rUp,rGp,rGp,rAp,rUp,rGp,rUp,rUp,rCp,rGp,rGp,rGp,rCp,rAp,rGp,rCp,rGp,rGp,rUp,rGp,rUp,rGp,rGp,rCp,rAp,rGp,rGp,rGp,rAp,rUp,rGp,rAp,rGp,rCp,rGp,rGp,rCp,rAp,rCp,rCp,rAp,rCp,rGp,rGp,rUp,rCp,rGp,rGp,rAp,rUp,rCp,rCp,rAp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>220</th>
      <td>Crude Extracellular Mixture of E. coli - cis</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/256.png" width="120" ></td>
      <td>Crude Extracellular Mixture (CEM) of E. coli</td>
      <td>Protein</td>
      <td>N/A nM (reported value)</td>
      <td>Selection Buffer (50 mm HEPES (pH 7.5), 150 mM NaCl,  15 mM MgCl2, and 0.01% Tween 20), r.t.</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>38623.03 g/mole</td>
      <td>52.80%</td>
      <td>1183600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>111</td>
      <td>Ali et al. "Fluorogenic DNAzyme Probes as Bacterial Indicators." Angewandte Chemie International Edition, 50 (2011): 3751-3754.</td>
      <td>dCp,dAp,dCp,dGp,dGp,dAp,dTp,dCp,dCp,dTp,dGp,dAp,dCp,dAp,dAp,dGp,dGp,dAp,dTp,dGp,dTp,dGp,dTp,dGp,dCp,dGp,dTp,dTp,dGp,dTp,dCp,dGp,dAp,dGp,dAp,dCp,dCp,dTp,dGp,dCp,dGp,dAp,dCp,dCp,dGp,dGp,dAp,dAp,dCp,dAp,dCp,dTp,dAp,dCp,dAp,dCp,dTp,dGp,dTp,dGp,dTp,dGp,dGp,dGp,dAp,dTp,dGp,dGp,dAp,dTp,dTp,dTp,dCp,dTp,dTp,dTp,dAp,dCp,dAp,dGp,dTp,dTp,dGp,dTp,dGp,dTp,dGp,dCp,dAp,dGp,dCp,dTp,dCp,dCp,dGp,dTp,dCp,dCp,dGp,dAp,dCp,dTp,dCp,dTp,dTp,dCp,dCp,dTp,dAp,dGp,dCp,rAp,dGp,dGp,dTp,dTp,dCp,dGp,dAp,dTp,dCp,dAp,dAp,dGp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>221</th>
      <td>Flavin Mononucleotide (AR5)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/204.png" width="120" ></td>
      <td>FMN</td>
      <td>Small Organic</td>
      <td>N/A nM (reported value)</td>
      <td>None</td>
      <td>23°C</td>
      <td>None</td>
      <td></td>
      <td>28595.37 g/mole</td>
      <td>52.27%</td>
      <td>897000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>89</td>
      <td>Seetharaman et al. "Immobilized RNA switches for the analysis of complex chemical and biological mixtures." Nature Biotechnology, 19(2001): 336-341.</td>
      <td>rGp,rGp,rAp,rUp,rAp,rAp,rUp,rAp,rGp,rCp,rCp,rGp,rUp,rAp,rGp,rGp,rUp,rUp,rGp,rCp,rGp,rAp,rAp,rAp,rGp,rCp,rGp,rAp,rCp,rCp,rCp,rUp,rGp,rAp,rUp,rGp,rAp,rGp,rCp,rCp,rUp,rUp,rAp,rGp,rGp,rAp,rUp,rAp,rUp,rGp,rCp,rAp,rUp,rGp,rAp,rUp,rGp,rCp,rAp,rGp,rAp,rAp,rGp,rGp,rAp,rCp,rGp,rUp,rCp,rGp,rAp,rAp,rAp,rCp,rGp,rGp,rUp,rAp,rGp,rCp,rGp,rAp,rGp,rAp,rGp,rCp,rUp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>222</th>
      <td>MUC1 Tumor Marker (MUC1-5TR-1)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/445.png" width="120" ></td>
      <td>MUC1 recombinant protein with five repeats of the variable tandem repeat region</td>
      <td>Protein</td>
      <td>47.3 nM (reported value)</td>
      <td>None</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>22353.64 g/mole</td>
      <td>45.83%</td>
      <td>748000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>74</td>
      <td>None</td>
      <td>dGp,dGp,dGp,dAp,dGp,dAp,dCp,dAp,dAp,dGp,dAp,dAp,dTp,dAp,dAp,dAp,dCp,dGp,dCp,dTp,dCp,dAp,dAp,dGp,dAp,dAp,dGp,dTp,dGp,dAp,dAp,dAp,dAp,dTp,dGp,dAp,dCp,dAp,dGp,dAp,dAp,dCp,dAp,dCp,dAp,dAp,dCp,dAp,dTp,dTp,dCp,dGp,dAp,dCp,dAp,dGp,dGp,dAp,dGp,dGp,dCp,dTp,dCp,dAp,dCp,dAp,dAp,dCp,dAp,dGp,dGp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>223</th>
      <td>Apt-07S</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7165.jpg" width="120" ></td>
      <td>HepG2 (Kd = 194.7nM) and SMMC-7721 (Kd = 224.2nM)</td>
      <td>Cells</td>
      <td>194.7 and 224.2 nM (reported value)</td>
      <td>1×PBS–1 mM MgCl2–0.1 μg/μL ytRNA–0.1 μg/μL wDNA–X pmol ssDNA library.</td>
      <td>4°C</td>
      <td>None</td>
      <td></td>
      <td>14594.5 g/mole</td>
      <td>42.55%</td>
      <td>462500 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>47</td>
      <td>Yu XX, Ge KL, Liu N, Zhang JY, Xue ML, Ge YL. Selection and Characterization of a Novel DNA Aptamer, Apt-07S Specific to Hepatocellular Carcinoma Cells. Drug Des Devel Ther. 2020 Apr 20;14:1535-1545. doi: 10.2147/DDDT.S244149. PMID: 32368012; PMCID: PMC7182459.</td>
      <td>dGp,dTp,dAp,dCp,dTp,dGp,dTp,dCp,dAp,dAp,dTp,dTp,dGp,dGp,dAp,dAp,dGp,dTp,dGp,dGp,dTp,dGp,dTp,dTp,dAp,dCp,dGp,dTp,dTp,dGp,dTp,dGp,dTp,dAp,dGp,dTp,dCp,dAp,dAp,dAp,dTp,dCp,dAp,dGp,dTp,dGp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>224</th>
      <td>Theophylline (mTCT8-4)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/150.jpg" width="120" ></td>
      <td>Theophylline</td>
      <td>Small Organic</td>
      <td>100 nM (reported value)</td>
      <td>Buffer A (100 mM Hepes (pH 7.3), 5 mM MgCI2, and 0.5 M NaCI)</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>12097.29 g/mole</td>
      <td>52.63%</td>
      <td>364600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Truncated version from Figure 2 in paper. Arrows indicate truncation.    Binds to theophylline with 10,000 times greater affinity than caffeine, which differs only by a methyl group.</td>
      <td>38</td>
      <td>None</td>
      <td>rAp,rGp,rUp,rGp,rAp,rUp,rAp,rCp,rCp,rAp,rGp,rCp,rAp,rUp,rCp,rGp,rUp,rCp,rUp,rUp,rGp,rAp,rUp,rGp,rCp,rCp,rCp,rUp,rUp,rGp,rGp,rCp,rAp,rGp,rCp,rAp,rCp,rUp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>225</th>
      <td>2-PA</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/563.jpg" width="120" ></td>
      <td>Anthrax Protective Antigen (PA)</td>
      <td>Protein</td>
      <td>1.51 nM (reported value)</td>
      <td>50 uL Human Serum, 5 mM HEPES</td>
      <td>37°C</td>
      <td>\n                                n/a \n</td>
      <td>n/a</td>
      <td>8865.8 g/mole</td>
      <td>58.62%</td>
      <td>280800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>96</td>
      <td>Byul Nim Oh, Sungeun Lee, Hye-Yeon Park, Jin-Ook Baeg, Moon-Young Yoon, and Jinheung Kim. 2011. Reference: Sensitive fluorescence assay of anthrax protective antigen with two new DNA aptamers and their binding properties. Analyst, 2011, 136, 3384. DOI: 10.1039/c0an00978d</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>226</th>
      <td>Estrogen Aptamer (AptER-1)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/517.jpg" width="120" ></td>
      <td>Apo-estrogen receptor alpha</td>
      <td>Protein</td>
      <td>16 nM (reported value)</td>
      <td>12 mM HEPES/pH 7.6, 150 mM NaCl, and 10 mM MgCl2</td>
      <td>°C</td>
      <td>None</td>
      <td></td>
      <td>20019.01 g/mole</td>
      <td>58.46%</td>
      <td>635300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>195</td>
      <td>Xu, D., Chatakonda, V., Kourtidis, A., Conklin, D., and Shi, H. "In Search of Novel Drug Target Sites on Estrogen Receptors Using RNA Aptamers." Nucleic acid therapeutics 24.3 (2014): 227-38.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>227</th>
      <td>Beta-Estradiol Aptamer (BES.1)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/609.jpg" width="120" ></td>
      <td>Beta-Estradiol (Estrogen)</td>
      <td>Small Organic</td>
      <td>6 µM (reported value)</td>
      <td>None</td>
      <td>23°C</td>
      <td>\n                                95°C for 5 minutes, allow aptamer to cool to room temperature over 10 minutes \n</td>
      <td>Specific to beta-estradiol over deoxycholic acid. Shows cross-reactivity towards dehydroisoandrosterone 3-sulfate and deoxycorticosterone 21-glucoside.</td>
      <td>14720.56 g/mole</td>
      <td>58.33%</td>
      <td>446300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Affinity estimated based on fluorescence response data provided in Supplementary Table 4.</td>
      <td>48</td>
      <td>None</td>
      <td>dGp,dGp,dCp,dTp,dCp,dTp,dCp,dGp,dGp,dGp,dAp,dCp,dGp,dAp,dCp,dAp,dTp,dGp,dGp,dAp,dTp,dTp,dTp,dTp,dCp,dCp,dAp,dTp,dCp,dAp,dAp,dCp,dGp,dAp,dAp,dGp,dTp,dGp,dCp,dGp,dTp,dCp,dCp,dGp,dTp,dCp,dCp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>228</th>
      <td>Platelet-Derived Growth Factor (PDGF) B-chain (36t)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/372.png" width="120" ></td>
      <td>Platelet-Derived Growth Factor -BB (PDGF-BB)</td>
      <td>Protein</td>
      <td>0.093 nM (reported value)</td>
      <td>None</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>12266 g/mole</td>
      <td>56.41%</td>
      <td>382300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Contains a terminal 3'-3'-linked thymidine residue to protect from 3' exonuclease degradation.</td>
      <td>40</td>
      <td>None</td>
      <td>dCp,dAp,dCp,dAp,dGp,dGp,dCp,dTp,dAp,dCp,dGp,dGp,dCp,dAp,dCp,dGp,dTp,dAp,dGp,dAp,dGp,dCp,dAp,dTp,dCp,dAp,dCp,dCp,dAp,dTp,dGp,dAp,dTp,dCp,dCp,dTp,dGp,dTp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>229</th>
      <td>Lead (Pb) binding aptamer</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/327.png" width="120" ></td>
      <td>Lead (Pb)</td>
      <td>Other</td>
      <td></td>
      <td>None</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>4726.13 g/mole</td>
      <td>60.00%</td>
      <td>143300 L/(mole·cm)</td>
      <td></td>
      <td>1 nM (reported value)</td>
      <td>Aptamer forms a G-quartet in the presence of lead ions.</td>
      <td>15</td>
      <td>None</td>
      <td>dGp,dGp,dTp,dTp,dGp,dGp,dTp,dGp,dTp,dGp,dGp,dTp,dTp,dGp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>230</th>
      <td>Interferon-Gamma</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/619.png" width="120" ></td>
      <td>Interferon-gamma (IFN-c)</td>
      <td>Protein</td>
      <td>74.5 nM (reported value)</td>
      <td>Salmon sperm DNA (0.1 mg/mL) and BSA (1 mg/mL) were  added to the binding buffer (PBS) to reduce the background  interference</td>
      <td>°C</td>
      <td>\n                                The procedures of selection were as follows: in the  first selection round, the ssDNA pool (200 pmol) was first heated at  95uC for 5 min and then cooled to 0uC for 15 min immediately. \n</td>
      <td>To evaluate the binding specificity of aptamers to target, FITClabeled  aptamer was incubated with IFN-c, paraformaldehydefixed  IFN-c, trypsin, IgG or BSA coated magnetic beads,  respectively. The beads were washed twice with 0.5 ml of binding  buffer, suspended in 0.2 ml of binding buffer, and ana</td>
      <td>18001.76 g/mole</td>
      <td>47.46%</td>
      <td>585100 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>59</td>
      <td>Cao, B., Hu, Y., Duan, J., Ma, J., Xu, D., & Yang, X. D. (2014). Selection of a novel DNA aptamer for assay of intracellular interferon-gamma. PloS one, 9(5), e98214.</td>
      <td>dCp,dCp,dGp,dCp,dCp,dCp,dAp,dAp,dAp,dTp,dCp,dCp,dCp,dTp,dAp,dAp,dGp,dAp,dGp,dAp,dAp,dGp,dAp,dCp,dTp,dGp,dTp,dAp,dAp,dTp,dGp,dAp,dCp,dAp,dTp,dCp,dAp,dAp,dAp,dCp,dCp,dAp,dGp,dAp,dCp,dAp,dCp,dAp,dCp,dTp,dAp,dCp,dAp,dCp,dAp,dCp,dGp,dCp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>231</th>
      <td>Sp20</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7210.jpg" width="120" ></td>
      <td>Shigella sonnei</td>
      <td>Other</td>
      <td>14.32 nM (reported value)</td>
      <td>Tris buffer</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>24563.62 g/mole</td>
      <td>53.75%</td>
      <td>740500 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>dual aptamer with Sp1</td>
      <td>80</td>
      <td>None</td>
      <td>dTp,dGp,dAp,dGp,dCp,dCp,dCp,dAp,dAp,dGp,dCp,dCp,dCp,dTp,dGp,dGp,dTp,dAp,dTp,dGp,dTp,dCp,dGp,dTp,dCp,dGp,dAp,dTp,dTp,dAp,dTp,dTp,dGp,dTp,dTp,dTp,dCp,dAp,dGp,dTp,dTp,dCp,dAp,dGp,dTp,dTp,dCp,dCp,dCp,dCp,dCp,dGp,dCp,dGp,dTp,dTp,dCp,dCp,dGp,dAp,dGp,dAp,dTp,dCp,dCp,dCp,dAp,dAp,dAp,dGp,dTp,dAp,dGp,dAp,dCp,dCp,dTp,dGp,dCp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>232</th>
      <td>Insulin Binding Aptamer (IBA)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/608.jpg" width="120" ></td>
      <td>Insulin</td>
      <td>Protein</td>
      <td>n/a nM (reported value)</td>
      <td>50 mM Tris-HCl, 10 mM KCl, 100 mM NaCl</td>
      <td>23°C</td>
      <td>None</td>
      <td>Specificity tested against Streptavidin, Avidin, and BSA.</td>
      <td>9493.19 g/mole</td>
      <td>66.67%</td>
      <td>288200 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>FAM-Labeled at 3' end.</td>
      <td>30</td>
      <td>Pu, Y.; Zhu, Z.; Han, D.; Liu, H.; Liu, J.; Liao, J.; Zhang, K.; & Tan, W. Insulin-Binding Aptamer-Conjugated Graphene Oxide for Insulin Detection. Analyst, 2011, 136, 4138-4140</td>
      <td>dGp,dGp,dTp,dGp,dGp,dTp,dGp,dGp,dGp,dGp,dGp,dGp,dGp,dGp,dTp,dTp,dGp,dGp,dTp,dAp,dGp,dGp,dGp,dTp,dGp,dTp,dCp,dTp,dTp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>233</th>
      <td>Anti-VEGF165</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/361.png" width="120" ></td>
      <td>Recombinant Human Vascular Endothelial Growth Factor</td>
      <td>Protein</td>
      <td>0.92 nM (reported value)</td>
      <td>SPR Buffer A: 10 mM phosphate buffered saline; 138 mM NaCl; 2.7 mM KCL; pH 7.4 with 0.005% Tween 20.</td>
      <td>RT°C</td>
      <td>\n                                Denatured at 80°C for 2 min and slowly cooled to room temperature. \n</td>
      <td></td>
      <td>7981.22 g/mole</td>
      <td>61.54%</td>
      <td>249100 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>78</td>
      <td>None</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>234</th>
      <td>K16-1</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7205.png" width="120" ></td>
      <td>Kanamycin A</td>
      <td>Small Organic</td>
      <td>342±74 nM (reported value)</td>
      <td>50% 100 mM sodium chloride, 20 mM tris-hydrochloride, 2 mM magnesium chloride, 5 mM potassium chloride, and 1 mM calcium chloride (pH 7.6); 50% 1X Cut Smart Buffer (New England Biolabs) 50 mM potassium \nacetate, 20 mM tris-acetate, 10 mM magnesium acetate, 100 µg/mL bovine serum albumin (pH 7.9)</td>
      <td>20°C</td>
      <td>\n                                mFold \n</td>
      <td></td>
      <td>28183.1 g/mole</td>
      <td>56.67%</td>
      <td>861900 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>This Aptamer binds to a 9 nucleotide long capture strand in order to function as a biosensor for Kanamycin A</td>
      <td>270</td>
      <td>(This content is a preprint and has not been peer-reviewed.)</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>235</th>
      <td>Mercury II (Hg2+)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/325.png" width="120" ></td>
      <td>Mercury II (Hg2+)</td>
      <td>Other</td>
      <td></td>
      <td>None</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>6590.34 g/mole</td>
      <td>36.36%</td>
      <td>178600 L/(mole·cm)</td>
      <td></td>
      <td>40 nM (reported value)</td>
      <td>None</td>
      <td>22</td>
      <td>A. Ono and H. Togashi. "Highly selective oligonucleotide-based sensor for mercury(II) in aqueous solutions." Angew. Chem. Int. Ed. 43(2004):4300-4302.</td>
      <td>dTp,dTp,dCp,dTp,dTp,dTp,dCp,dTp,dTp,dCp,dCp,dCp,dCp,dTp,dTp,dGp,dTp,dTp,dTp,dGp,dTp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>236</th>
      <td>5TRG2</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/635.jpg" width="120" ></td>
      <td>MUC1-5TR-GalNAc; Tn antigen</td>
      <td>Protein</td>
      <td>18.6 nM (reported value)</td>
      <td>150 nM NaCl, 5nM MgCl2, pH 7.4</td>
      <td>37°C</td>
      <td>\n                                Refolded in binding conditions \n</td>
      <td>See figure 3 in article reference</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Target is O-linked glycan-peptide specific to epithelial cancer cells.  Sequence is 5'-dGpdApdGpdApdCpdApdApdGpdApdApdTpdApdApdApdCpdGpdCpdTpdCpdApdApdGpdGpdCpdTpdApdTpdApdGpdCpdApdCpdApdTpdGpdGpdGpdTpdApdApdApdApdCpdGpdApdCpdTpdTpdCpdGpdApdCpdApdGpdGpdApdGpdGpdCpdTpdCpdApdCpdApdApdCpdApdGpdGpdCp-3'</td>
      <td>70</td>
      <td>Ferreira, C.S.; Cheung, M.C.; Missailidis, S.; Bisland, S.; Gariepy, J. Phototoxic aptamers selectively enter and kill epithelial cancer cells. Nucleic acids research 2009, 37, 866-876.</td>
      <td>dGp,dAp,dGp,dAp,dCp,dAp,dAp,dGp,dAp,dAp,dTp,dAp,dAp,dAp,dCp,dGp,dCp,dTp,dCp,dAp,dAp,dGp,dGp,dCp,dTp,dAp,dTp,dAp,dGp,dCp,dAp,dCp,dAp,dTp,dGp,dGp,dGp,dTp,dAp,dAp,dAp,dAp,dCp,dGp,dAp,dCp,dTp,dTp,dCp,dGp,dAp,dCp,dAp,dGp,dGp,dAp,dGp,dGp,dCp,dTp,dCp,dAp,dCp,dAp,dAp,dCp,dAp,dGp,dGp,dCp,dGp,dAp,dGp,dAp,dCp,dAp,dAp,dGp,dAp,dAp,dTp,dAp,dAp,dAp,dCp,dGp,dCp,dTp,dCp,dAp,dAp,dGp,dGp,dCp,dTp,dAp,dTp,dAp,dGp,dCp,dAp,dCp,dAp,dTp,dGp,dGp,dGp,dTp,dAp,dAp,dAp,dAp,dCp,dGp,dAp,dCp,dTp,dTp,dCp,dGp,dAp,dCp,dAp,dGp,dGp,dAp,dGp,dGp,dCp,dTp,dCp,dAp,dCp,dAp,dAp,dCp,dAp,dGp,dGp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>237</th>
      <td>T-2</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7155.jpg" width="120" ></td>
      <td>thyroglobulin</td>
      <td>Protein</td>
      <td>3.18 nM (reported value)</td>
      <td>TGK, Tris-HCl, BBS, and PBS</td>
      <td>22°C</td>
      <td>None</td>
      <td></td>
      <td>12388 g/mole</td>
      <td>65.00%</td>
      <td>380500 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>40</td>
      <td>Zhu, C., Li, L., Fang, S., Zhao, Y., Zhao, L., Yang, G., & Qu, F. (2020). Selection and Characterization of an ssDNA Aptamer against Thyroglobulin. Talanta, 121690. doi:10.1016/j.talanta.2020.121690</td>
      <td>dCp,dGp,dCp,dGp,dTp,dGp,dAp,dGp,dCp,dGp,dGp,dGp,dGp,dAp,dGp,dGp,dCp,dGp,dAp,dTp,dGp,dCp,dCp,dCp,dAp,dGp,dGp,dCp,dTp,dAp,dAp,dCp,dTp,dTp,dGp,dAp,dCp,dTp,dCp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>238</th>
      <td>JH6</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/513.jpg" width="120" ></td>
      <td>NB4 AML cell line</td>
      <td>Cells</td>
      <td>2.77 nM (reported value)</td>
      <td>4.5 g/L glucose, 5 mM MgCl2, 0.1 mg/ml yeast tRNA and 1 mg/ml Bovine Serum Albumin (BSA) into Phosphate buffered saline (PBS)</td>
      <td>°C</td>
      <td>None</td>
      <td></td>
      <td>24176.7 g/mole</td>
      <td>56.41%</td>
      <td>757900 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>234</td>
      <td>Yang, M. et al. "Developing aptamer probes for acute myelogenous leukemia detection and surface protein biomarker discovery." Journal of Hematology & Oncology 2014, 7:5.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>239</th>
      <td>Mitomer1</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/574.jpg" width="120" ></td>
      <td>Mitochondria</td>
      <td>Other</td>
      <td>. nM (reported value)</td>
      <td>20mM Tris-HCl, 2mM KCl, 40mM NaCl, 1mM MgCl2, 250mM Sucrose, pH=7.5</td>
      <td>37°C</td>
      <td>\n                                80C for 10 min, and cooled down slowly to 4C \n</td>
      <td></td>
      <td>9769.72 g/mole</td>
      <td>70.00%</td>
      <td>292400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>98</td>
      <td>Yuri Tawaraya, Mamoru Hyodo, Mst Naznin Ara, Yuma Yamada, and Hideyoshi Harashima, ƒ??RNA Aptamers for Targeting Mitochondria Using a Mitochondria-Based SELEX Methodƒ? Biol. Pharm. Bull. 37(8) 1411-1415 (2014)</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>240</th>
      <td>Chloramphenicol-binding aptamer(c7)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/329.jpg" width="120" ></td>
      <td>chloramphenicol (cam)</td>
      <td>Small Organic</td>
      <td>0.766 µM (reported value)</td>
      <td>None</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>24817.1 g/mole</td>
      <td>57.50%</td>
      <td>774600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>80</td>
      <td>None</td>
      <td>dAp,dGp,dCp,dAp,dGp,dCp,dAp,dCp,dAp,dGp,dAp,dGp,dGp,dTp,dCp,dAp,dGp,dAp,dTp,dGp,dAp,dCp,dTp,dTp,dCp,dAp,dGp,dTp,dGp,dAp,dGp,dTp,dTp,dGp,dTp,dCp,dCp,dCp,dAp,dCp,dGp,dGp,dTp,dCp,dGp,dGp,dCp,dGp,dAp,dGp,dTp,dCp,dGp,dGp,dTp,dGp,dGp,dTp,dAp,dGp,dCp,dCp,dTp,dAp,dTp,dGp,dCp,dGp,dTp,dGp,dCp,dTp,dAp,dCp,dCp,dGp,dTp,dGp,dAp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>241</th>
      <td>Staphylococcus aureus (SA23)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/161.png" width="120" ></td>
      <td>Staphylococcus aureus</td>
      <td>Cells</td>
      <td>61.5 nM (reported value)</td>
      <td>Screening buffer: 0.1 mg/ml salmon sperm DNA (0.1 mg/ml tRNA, 1%BSA, 1  PBS, 0.05% (v/v) Tween-20</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>27104.61 g/mole</td>
      <td>52.27%</td>
      <td>842400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>88</td>
      <td>None</td>
      <td>dGp,dCp,dAp,dAp,dTp,dGp,dGp,dTp,dAp,dCp,dGp,dGp,dTp,dAp,dCp,dTp,dTp,dCp,dCp,dGp,dGp,dGp,dCp,dTp,dGp,dGp,dCp,dCp,dAp,dGp,dAp,dTp,dCp,dAp,dGp,dAp,dCp,dCp,dCp,dCp,dGp,dGp,dAp,dTp,dGp,dAp,dTp,dCp,dAp,dTp,dCp,dCp,dTp,dTp,dGp,dTp,dGp,dAp,dGp,dAp,dAp,dCp,dCp,dAp,dCp,dAp,dAp,dAp,dAp,dGp,dTp,dGp,dCp,dAp,dCp,dGp,dCp,dTp,dAp,dCp,dTp,dTp,dTp,dGp,dCp,dTp,dAp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>242</th>
      <td>PCB106 (Aptamer 9.3)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/230.png" width="120" ></td>
      <td>Polychlorinated biphenyl 106 (PCB)</td>
      <td>Small Organic</td>
      <td>99 nM (reported value)</td>
      <td>Binding Buffer (100 mM NaCl, 20 mM   Trisƒ??HCl, 2 mM MgCl2, 5 mM KCl, 1 mM CaCl2, pH 7.6)</td>
      <td>r.t.°C</td>
      <td>None</td>
      <td></td>
      <td>24336.74 g/mole</td>
      <td>57.50%</td>
      <td>735000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>90</td>
      <td>Mehta et al. "Selection and characterization of PCB-binding DNA aptamers." Analytical Chemistry, 84(2012): 1669-1676.</td>
      <td>dAp,dGp,dCp,dAp,dGp,dCp,dAp,dCp,dAp,dGp,dAp,dGp,dGp,dTp,dCp,dAp,dGp,dAp,dTp,dGp,dCp,dAp,dCp,dTp,dCp,dGp,dGp,dAp,dCp,dCp,dCp,dCp,dAp,dTp,dTp,dCp,dTp,dCp,dCp,dTp,dTp,dCp,dCp,dAp,dTp,dCp,dCp,dCp,dTp,dCp,dAp,dTp,dCp,dCp,dGp,dTp,dCp,dCp,dAp,dCp,dCp,dCp,dTp,dAp,dTp,dGp,dCp,dGp,dTp,dGp,dCp,dTp,dAp,dCp,dCp,dGp,dTp,dGp,dAp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>243</th>
      <td>Profenofos (SS2-55)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/293.png" width="120" ></td>
      <td>Profenofos</td>
      <td>Small Organic</td>
      <td>1 µM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td>This aptamer binds preferentially to the organophosphates phorate, profenofos, benzoate and omethoate. It has reduced binding to imidacloprid, acetamiprid, chlorpyrifos, methamidophos, dichlorvos, parathion, dimethoate, monocrotophos and phoxim.</td>
      <td>16990.09 g/mole</td>
      <td>49.09%</td>
      <td>526900 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>"A molecular beacon (5' FAM-CTGCACAAGAATCGCTGCAG-3') was designed, which was partly complementary with the middle constant nucleotides-sequence of the selected ssDNAs."</td>
      <td>55</td>
      <td>None</td>
      <td>dAp,dAp,dGp,dCp,dTp,dTp,dGp,dCp,dTp,dTp,dTp,dAp,dTp,dAp,dGp,dCp,dCp,dTp,dGp,dCp,dAp,dGp,dCp,dGp,dAp,dTp,dTp,dCp,dTp,dTp,dGp,dAp,dTp,dCp,dGp,dGp,dAp,dAp,dAp,dAp,dGp,dGp,dCp,dTp,dGp,dAp,dGp,dAp,dGp,dCp,dTp,dAp,dCp,dGp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>244</th>
      <td>MCP-1 (ADR7)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/465" width="120" ></td>
      <td>Monocyte chemoattractant protein-1 (mouse)</td>
      <td>Protein</td>
      <td>180 pM (reported value)</td>
      <td>Selection Buffer (110 mM NaCl, 1 mM MgCl2, 20 mM HEPES (pH 7.0), 1 mM CaCl2, 5 mM KCl, 0.001% HSA, 0.05% Tween 20)</td>
      <td>37°C</td>
      <td>\n                                None given. \n</td>
      <td></td>
      <td>22580.5 g/mole</td>
      <td>62.86%</td>
      <td>670700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>70</td>
      <td>Rhodes, Andrew, et al. "The generation and characterisation of antagonist RNA aptamers to MCP-1." FEBS Letter, 506 (2001): 85-90.</td>
      <td>rGp,rGp,rGp,rAp,rGp,rGp,rAp,fCp,rGp,rAp,fUp,rGp,fCp,rGp,rGp,rGp,rAp,rAp,fCp,fUp,fCp,rAp,fCp,fCp,rGp,rGp,rGp,rAp,rAp,rGp,rAp,rAp,rGp,fCp,fCp,fCp,rGp,fUp,fUp,fCp,fCp,rGp,fUp,fCp,rAp,fCp,rAp,rGp,rAp,fCp,rAp,fUp,rGp,fUp,fUp,fCp,fCp,rGp,fCp,rAp,fUp,fCp,rGp,fUp,fCp,fCp,fUp,fCp,fCp,fCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>245</th>
      <td>DL650-E3</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/638.jpg" width="120" ></td>
      <td>PC-3</td>
      <td>Cells</td>
      <td>146 nM (reported value)</td>
      <td>DPBS with Ca2+ and Mg2+ (ThermoFisher, Waltham, MA)</td>
      <td>37°C</td>
      <td>\n                                65°C for 5 min and 4°C for 5 min \n</td>
      <td>PrEC cells only exhibit minimal uptake of E3</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>During SELEX, aptamers were refolded at 85C for 5 min in HEPES buffered saline with 1 mM MgCl2, 1 mM CaCl2, 2.7 mM KCl, then cooled on ice.  Aptamer also tested against cell lines 22RV1 (Kd = 204 nM) and VCaP (Kd = 358 nM)   Sequence is regular RNA: GGCUUUCGGGCUUUCGGCAACAUCAGCCCCUCAGCC</td>
      <td>33</td>
      <td>Powell Gray B, Kelly L, Ahrens DP et al. Tunable cytotoxic aptamer-drug conjugates for the treatment of prostate cancer. Proc. Natl. Acad. Sci. U.S.A. doi: 10.1073/pnas.1717705115 (2018)</td>
      <td>dGp,dGp,dCp,dUp,dUp,dUp,dCp,dGp,dGp,dGp,dCp,dUp,dUp,dUp,dCp,dAp,dAp,dCp,dAp,dUp,dCp,dAp,dGp,dCp,dCp,dCp,dCp,dUp,dCp,dAp,dGp,dCp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>246</th>
      <td>G1</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7144.jpg" width="120" ></td>
      <td>Glutathione S-transferase (GST)</td>
      <td>Protein</td>
      <td>59 nM (reported value)</td>
      <td>20 mM Tris-HCl, 100 mM NaCl, 5 mM MgCl2, and 0.1% bovine\nserum albumin with 10µg/mL yeast tRNA and 10µg/mL salmon sperm DNA</td>
      <td>25°C</td>
      <td>None</td>
      <td></td>
      <td>23560.4 g/mole</td>
      <td>37.66%</td>
      <td>750800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>77</td>
      <td>Wang Y, Li Z, Yu H. Aptamer-Based Western Blot for Selective Protein Recognition. Front Chem. 2020 Oct 29;8:570528. doi: 10.3389/fchem.2020.570528. PMID: 33195056; PMCID: PMC7658645.</td>
      <td>dAp,dTp,dAp,dCp,dGp,dAp,dCp,dTp,dCp,dAp,dCp,dTp,dAp,dTp,dTp,dAp,dGp,dGp,dGp,dAp,dCp,dAp,dAp,dAp,dGp,dCp,dTp,dGp,dAp,dCp,dAp,dAp,dCp,dCp,dCp,dTp,dTp,dTp,dCp,dAp,dTp,dAp,dAp,dTp,dCp,dTp,dAp,dAp,dCp,dTp,dAp,dCp,dAp,dTp,dTp,dTp,dAp,dTp,dGp,dTp,dGp,dCp,dTp,dAp,dGp,dAp,dCp,dTp,dAp,dCp,dTp,dGp,dAp,dCp,dTp,dAp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>247</th>
      <td>#AP-Cell 1</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7203.png" width="120" ></td>
      <td>Aspartate β-Hydroxylase (ASPH)</td>
      <td>Protein</td>
      <td>47.51 ± 5.86 nM (reported value)</td>
      <td>None</td>
      <td>°C</td>
      <td>\n                                m-fold \n</td>
      <td></td>
      <td>15879.27 g/mole</td>
      <td>55.77%</td>
      <td>517300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>52</td>
      <td>None</td>
      <td>dCp,dGp,dGp,dGp,dAp,dCp,dAp,dAp,dGp,dAp,dCp,dAp,dAp,dCp,dGp,dAp,dAp,dCp,dGp,dAp,dAp,dCp,dAp,dGp,dGp,dAp,dAp,dGp,dAp,dGp,dAp,dAp,dCp,dCp,dGp,dGp,dAp,dAp,dTp,dGp,dCp,dAp,dGp,dAp,dCp,dGp,dTp,dCp,dAp,dGp,dGp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>248</th>
      <td>SECURA-3</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7139.jpg" width="120" ></td>
      <td>Pituitary tumour transforming gene 1 (PTTG1)</td>
      <td>Protein</td>
      <td>16.41 ± 6.4 nM (reported value)</td>
      <td>20 mM Tris-HCl (pH 8.6), 8 M Urea, 150 mM NaCl, 10% Glycerol, 1.5X Roche Protease Inhibitor Cocktail EDTA-free).\n10 mM HEPES-KOH (pH 7.4) and 150 mM NaCl</td>
      <td>95, 25°C</td>
      <td>None</td>
      <td></td>
      <td>30229.1 g/mole</td>
      <td>58.59%</td>
      <td>907400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>99</td>
      <td>Prabu, S. S., Ch’ng, E. S., Woon, P. Y., Chen, J.-H., Tang, T.-H., & Citartan, M. (2020). Unravelling the diagnostic and therapeutic potentialities of a novel RNA aptamer isolated against human pituitary tumour transforming gene 1 (PTTG1) protein. Analytica Chimica Acta. doi:10.1016/j.aca.2020.09.038</td>
      <td>rGp,rGp,rAp,rGp,rCp,rUp,rCp,rAp,rGp,rCp,rCp,rUp,rUp,rCp,rAp,rCp,rUp,rGp,rCp,rGp,rAp,rUp,rCp,rAp,rCp,rCp,rGp,rUp,rGp,rUp,rCp,rGp,rGp,rUp,rGp,rGp,rCp,rUp,rAp,rGp,rUp,rCp,rGp,rAp,rUp,rGp,rUp,rAp,rUp,rCp,rCp,rCp,rGp,rAp,rUp,rCp,rAp,rCp,rCp,rAp,rUp,rUp,rGp,rGp,rGp,rGp,rGp,rCp,rAp,rAp,rGp,rUp,rUp,rUp,rAp,rGp,rAp,rCp,rAp,rGp,rGp,rCp,rAp,rCp,rCp,rAp,rCp,rGp,rGp,rUp,rCp,rGp,rGp,rAp,rUp,rCp,rCp,rAp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>249</th>
      <td>A15</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/564.jpg" width="120" ></td>
      <td>Penetrating the Blood Brain Barrier (BBB)</td>
      <td>Other</td>
      <td>. nM (reported value)</td>
      <td>10mM HEPES, pH 7.4, 50mM NaCl, 1mM MgCl2, 1mM CaCl2, 0.01% BSA</td>
      <td>37°C</td>
      <td>\n                                Denature at 94C for 3 minutes, and then slowly refold at room temperature \n</td>
      <td></td>
      <td>17706.87 g/mole</td>
      <td>77.78%</td>
      <td>523600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>202</td>
      <td>Congsheng Cheng, Yong Hong Chen, Kim A Lennox, Mark A Behlke, Beverly L Davidson, "In vivo SELEX for Identification of Brain-penetrating Aptamers" Molecular Therapy-Nucleic Acids (2013)2, e67, American Society of Gene & Cell Therapy</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>250</th>
      <td>GWDB-30</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7099.png" width="120" ></td>
      <td>Calreticulin</td>
      <td>Protein</td>
      <td>nM (reported value)</td>
      <td>None</td>
      <td>°C</td>
      <td>None</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Target unclear. References are made to binding calprotectin as well as calreticulin. These are structurally unrelated proteins. 30 sequences are listed, it is unknown which of these have been validated.</td>
      <td>78</td>
      <td>https://patents.google.com/patent/CN105063056A/en</td>
      <td>dAp,dCp,dGp,dTp,dTp,dCp,dGp,dGp,dAp,dTp,dCp,dGp,dCp,dAp,dCp,dTp,dAp,dGp,dAp,dGp,dCp,dCp,dCp,dAp,dAp,dAp,dTp,dCp,dTp,dCp,dAp,dGp,dAp,dAp,dAp,dCp,dCp,dCp,dAp,dTp,dTp,dCp,dTp,dCp,dTp,dAp,dCp,dAp,dCp,dAp,dCp,dCp,dTp,dTp,dGp,dTp,dAp,dCp,dCp,dTp,dTp,dCp,dTp,dGp,dAp,dAp,dCp,dGp,dTp,dTp,dCp,dTp,dCp,dAp,dCp,dGp,dCp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>251</th>
      <td>L-Selectin (LD201)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/49.png" width="120" ></td>
      <td>L-Selectin</td>
      <td>Protein</td>
      <td>1.8 nM (reported value)</td>
      <td>None</td>
      <td>22°C</td>
      <td>None</td>
      <td></td>
      <td>24243.79 g/mole</td>
      <td>46.84%</td>
      <td>754700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>79</td>
      <td>Hicke, Brain, et al. "DNA Aptamers Block L-Selectin Function In Vivo  Inhibition of Human Lymphocyte Traf?cking in SCID Mic." Journal of Clinical Investigation, 98 (1996): 2688-2692.</td>
      <td>dCp,dTp,dAp,dCp,dCp,dTp,dAp,dCp,dGp,dAp,dTp,dCp,dTp,dGp,dAp,dCp,dTp,dAp,dGp,dCp,dCp,dAp,dAp,dGp,dGp,dTp,dAp,dAp,dCp,dCp,dAp,dGp,dTp,dAp,dCp,dAp,dAp,dGp,dGp,dTp,dGp,dCp,dTp,dAp,dAp,dAp,dCp,dGp,dTp,dAp,dAp,dTp,dGp,dGp,dCp,dTp,dTp,dCp,dGp,dGp,dCp,dTp,dTp,dAp,dCp,dTp,dCp,dTp,dCp,dAp,dTp,dGp,dTp,dAp,dGp,dTp,dTp,dCp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>252</th>
      <td>Anti-FLAg M2 Antibody Aptamer (ABA)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/333.jpg" width="120" ></td>
      <td>Anti-FLAG M2 Antibody</td>
      <td>Protein</td>
      <td>80 nM (reported value)</td>
      <td>None</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>9160.02 g/mole</td>
      <td>40.00%</td>
      <td>274800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>30</td>
      <td>None</td>
      <td>dTp,dCp,dGp,dAp,dTp,dTp,dTp,dCp,dCp,dTp,dTp,dAp,dGp,dTp,dTp,dGp,dTp,dCp,dTp,dTp,dCp,dCp,dTp,dTp,dAp,dGp,dTp,dGp,dAp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>253</th>
      <td>Membrane-Bound IgM (TD05.17)</td>
      <td>Chimeric</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/351.png" width="120" ></td>
      <td>Membrane-bound IgM (mIgM )/ B-cell receptor (BCR )/ Leukemia / Lymphoma</td>
      <td>Protein</td>
      <td>43 nM (reported value)</td>
      <td>None</td>
      <td>4°C</td>
      <td>None</td>
      <td>None</td>
      <td>10256.7 g/mole</td>
      <td>56.76%</td>
      <td>349100 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>38</td>
      <td>None</td>
      <td>dAp,dGp,dGp,dAp,dGp,dGp,dAp,dTp,dAp,dGp,dTp,dTp,dCp,dGp,dGp,dTp,dGp,dGp,dCp,dTp,dGp,dTp,dTp,dCp,dAp,dGp,dGp,dGp,dTp,dCp,dTp,dCp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>254</th>
      <td>S2.2</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/622.jpg" width="120" ></td>
      <td>MUC1</td>
      <td>Protein</td>
      <td>0.135 nM (reported value)</td>
      <td>PBS buffer (pH 7.2); Room Temperature (overnight)</td>
      <td>°C</td>
      <td>None</td>
      <td></td>
      <td>7664.03 g/mole</td>
      <td>52.00%</td>
      <td>232300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>25</td>
      <td>D Seleci et al. Aptamer mediated niosomal drug delivery. RSC Adv 6(2016): 87910-87918.</td>
      <td>dGp,dCp,dAp,dGp,dTp,dTp,dGp,dAp,dTp,dCp,dCp,dTp,dTp,dTp,dGp,dGp,dAp,dTp,dAp,dCp,dCp,dCp,dTp,dGp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>255</th>
      <td>Carbonic Anhydrase IX Aptamer</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/641.jpg" width="120" ></td>
      <td>Carbonic anhydrase IX</td>
      <td>Protein</td>
      <td>n.d. nM (reported value)</td>
      <td>1x PBS at 37C</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>80</td>
      <td>Zhu, Lianhua et al. ƒ??CAIX aptamer-functionalized targeted nanobubbles for ultrasound molecular imaging of various tumorsƒ? International journal of nanomedicine vol. 13 6481-6495. 16 Oct. 2018, doi:10.2147/IJN.S176287</td>
      <td>dAp,dGp,dCp,dAp,dGp,dCp,dAp,dCp,dAp,dGp,dAp,dGp,dGp,dTp,dCp,dAp,dGp,dAp,dTp,dGp,dTp,dGp,dGp,dTp,dGp,dCp,dGp,dCp,dAp,dGp,dTp,dGp,dAp,dTp,dGp,dTp,dGp,dGp,dTp,dTp,dGp,dGp,dTp,dCp,dCp,dTp,dAp,dTp,dGp,dCp,dGp,dTp,dGp,dCp,dTp,dAp,dCp,dCp,dGp,dTp,dCp,dCp,dTp,dAp,dTp,dGp,dCp,dGp,dTp,dGp,dCp,dTp,dAp,dCp,dCp,dGp,dTp,dGp,dAp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>256</th>
      <td>Sialyllactose (Sl-11)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/90.png" width="120" ></td>
      <td>Sialyllactose</td>
      <td>Other</td>
      <td>4.9 µM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>\n                                Denatured at 95°C for 5 min and cool at RT for 30 min. \n</td>
      <td></td>
      <td>31347.28 g/mole</td>
      <td>56.86%</td>
      <td>947500 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Except for the primer 1 region, thymidines contained an amino linker with terminal amine at the C5 position. This was an attempt to offset the repulsive force between the negatively-charged target and DNA backbone.</td>
      <td>102</td>
      <td>Masud, M, et al. "Sialyllactose-binding modified DNA aptamer bearing additional functionality by SELEX." Bioorganic & Medicinal Chemistry, 12 (2004): 1111-1120.</td>
      <td>dGp,dTp,dAp,dCp,dGp,dAp,dAp,dTp,dTp,dCp,dAp,dCp,dGp,dAp,dGp,dGp,dTp,dTp,dGp,dCp,dCp,dAp,dGp,dCp,dGp,dGp,dGp,dGp,dCp,dCp,dAp,dGp,dCp,dCp,dAp,dCp,dTp,dTp,dCp,dTp,dGp,dTp,dCp,dAp,dGp,dTp,dGp,dAp,dAp,dTp,dTp,dCp,dCp,dTp,dGp,dCp,dTp,dCp,dGp,dTp,dAp,dTp,dAp,dTp,dCp,dTp,dAp,dCp,dTp,dCp,dGp,dCp,dCp,dCp,dGp,dCp,dCp,dTp,dGp,dCp,dGp,dAp,dGp,dCp,dAp,dTp,dGp,dGp,dAp,dGp,dTp,dCp,dGp,dGp,dAp,dTp,dCp,dCp,dTp,dCp,dTp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>257</th>
      <td>Aptamer-6</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7227.png" width="120" ></td>
      <td>S/RBD receptor binding</td>
      <td>Protein</td>
      <td>27.6  nM (reported value)</td>
      <td>n Dulbecco's phosphate-buffered saline \n(DPBS) containing 1 mM MgCl2 (DPBS/Mg)</td>
      <td>23°C°C</td>
      <td>\n                                mFold \n</td>
      <td>highly specific</td>
      <td>23902.47 g/mole</td>
      <td>55.26%</td>
      <td>1294263 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Binds to the receptor-binding domain (RBD) of COVID-19 virus and inhibits the S/RBD interaction between virus and host cell.</td>
      <td>76</td>
      <td>https://doi.org/10.1002/anie.202100345</td>
      <td>dAp,dTp,dCp,dTp,dAp,dGp,dAp,dGp,dTp,dGp,dAp,dCp,dGp,dCp,dAp,dGp,dCp,dAp,dGp,dGp,dGp,dCp,dTp,dTp,dGp,dGp,dGp,dTp,dTp,dGp,dGp,dGp,dAp,dAp,dTp,dAp,dAp,dAp,dGp,dAp,dTp,dGp,dTp,dGp,dGp,dGp,dAp,dGp,dGp,dCp,dGp,dGp,dCp,dGp,dAp,dAp,dCp,dAp,dTp,dGp,dGp,dAp,dCp,dAp,dCp,dGp,dGp,dTp,dGp,dGp,dCp,dTp,dTp,dAp,dGp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>258</th>
      <td>Zinc</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/149.png" width="120" ></td>
      <td>Zinc</td>
      <td>Small Organic</td>
      <td>1.2 mM (reported value)</td>
      <td>Buffer A (0.4 M NaCl, 20 mM HEPES-Na (pH 7.0), and 1 mM MgCl2)</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>30777.6 g/mole</td>
      <td>49.47%</td>
      <td>966700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>95</td>
      <td>Ciesiolka et al., "Selection of an RNA domain that binds Zn2+." Journal of RNA, 1(1995): 538-550.</td>
      <td>rGp,rGp,rGp,rAp,rGp,rAp,rGp,rGp,rAp,rUp,rAp,rCp,rUp,rAp,rCp,rUp,rGp,rUp,rCp,rAp,rUp,rAp,rCp,rGp,rUp,rUp,rAp,rGp,rGp,rCp,rUp,rGp,rUp,rAp,rGp,rGp,rCp,rGp,rAp,rGp,rGp,rUp,rGp,rAp,rAp,rAp,rUp,rGp,rAp,rGp,rCp,rGp,rGp,rUp,rAp,rAp,rUp,rAp,rGp,rCp,rCp,rUp,rCp,rAp,rGp,rCp,rGp,rUp,rAp,rGp,rCp,rAp,rUp,rAp,rUp,rGp,rCp,rAp,rUp,rGp,rAp,rAp,rUp,rUp,rCp,rGp,rAp,rAp,rGp,rCp,rUp,rUp,rCp,rGp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>259</th>
      <td>Human Umbilical Vein Endothelial Cells</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/254.png" width="120" ></td>
      <td>Human Umbilical Vein Endothelial Cells</td>
      <td>Cells</td>
      <td>N/A nM (reported value)</td>
      <td>N/A</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>600.85 g/mole</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>5</td>
      <td>Agemy et al. "Targeted nanoparticle enhanced proapoptotic peptide  as potential therapy for glioblastoma." PNAS, doi: 10.1073/pnas.1114518108.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>260</th>
      <td>Gelatinase (MMP-9)</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/234.jpg" width="120" ></td>
      <td>Matrix Metalloproteinase-9 (MMP-9)</td>
      <td>Protein</td>
      <td>N/A nM (reported value)</td>
      <td>Incubated overnight at 4 ° C in binding buffer (50 mM TBS buffer (Tris-HCl/0.1 M NaCl, pH 7.5) containing 1% BSA).</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>1268.54 g/mole</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Inhibits the migration of human endothelial and tumor cells.</td>
      <td>10</td>
      <td>None</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>261</th>
      <td>NS-1 Aptamer</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/516.jpg" width="120" ></td>
      <td>Influenza A virus (IAV) non-stuctural protein (NS1)</td>
      <td>Protein</td>
      <td>18.91 nM (reported value)</td>
      <td>50 mM Tris/Cl (pH 8.0), 150 mM NaCl, 1.5 mM MgCl2, 2 mM DTT and 1% (w/v) BSA</td>
      <td>°C</td>
      <td>None</td>
      <td></td>
      <td>27570.81 g/mole</td>
      <td>67.05%</td>
      <td>834900 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>264</td>
      <td>Woo, H., Kim, K., Lee, J., Shim, H., Cho, S., Lee, W., Ko, H., Keum, Y., Kim, S., Pathinayake, P., Kim, C., and Jeong, Y. "Single-stranded DNA aptamer that specifically binds to the influenza virus NS1 protein suppresses interferon antagonism." Antiviral Research 100.2(2013): 337-345.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>262</th>
      <td>SQ-2</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7123.jpg" width="120" ></td>
      <td>ALPPL-2</td>
      <td>Protein</td>
      <td>22.25-24.74 nM (reported value)</td>
      <td>4.5 g/L glucose, 5 mmol/L MgCl2, 0.1 mg/mL yeast tRNA, and 1 mg/mL bovine serum albumin (BSA) in Dulbecco's PBS</td>
      <td>4°C</td>
      <td>None</td>
      <td></td>
      <td>22995.6 g/mole</td>
      <td>38.16%</td>
      <td>711400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>76</td>
      <td>Dua, P., Kang, H. S., Hong, S.-M., Tsao, M.-S., Kim, S., & Lee, D. -k. (2013). Alkaline Phosphatase ALPPL-2 Is a Novel Pancreatic Carcinoma-Associated Protein. Cancer Research, 73(6), 1934–1945. doi:10.1158/0008-5472.can-12-3682</td>
      <td>rAp,rUp,rAp,rCp,rCp,rAp,rGp,rCp,rUp,rUp,rAp,rUp,rUp,rCp,rAp,rAp,rUp,rUp,rGp,rCp,rCp,rUp,rGp,rAp,rAp,rAp,rAp,rGp,rCp,rUp,rAp,rUp,rCp,rGp,rCp,rCp,rCp,rAp,rAp,rUp,rUp,rCp,rGp,rCp,rAp,rGp,rUp,rGp,rAp,rUp,rAp,rUp,rCp,rCp,rUp,rUp,rUp,rAp,rAp,rGp,rAp,rUp,rAp,rGp,rUp,rAp,rAp,rGp,rUp,rGp,rCp,rAp,rAp,rUp,rCp,rUp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>263</th>
      <td>Taq DNA polymerase (TQ21)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7118" width="120" ></td>
      <td>Taq DNA polymerase</td>
      <td>Protein</td>
      <td>36 pM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td>Has affinity for Tth DNA polymerase with Kd of 10 nM.</td>
      <td>23951.56 g/mole</td>
      <td>47.44%</td>
      <td>703200 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>78</td>
      <td>Sumedha and Dang. "Oligonucleotide inhibitors of Taq DNA polymerase facilitate detection of low copy number targets by PCR." Journal of Molecular Biology, 264 (1996): 268-78.</td>
      <td>dTp,dTp,dCp,dTp,dCp,dGp,dGp,dTp,dTp,dGp,dGp,dTp,dCp,dTp,dCp,dTp,dGp,dGp,dCp,dGp,dGp,dAp,dGp,dCp,dGp,dAp,dTp,dCp,dAp,dTp,dCp,dTp,dCp,dAp,dGp,dAp,dGp,dCp,dAp,dTp,dTp,dCp,dTp,dTp,dAp,dGp,dCp,dGp,dTp,dTp,dTp,dTp,dGp,dTp,dTp,dCp,dTp,dTp,dGp,dTp,dGp,dTp,dAp,dTp,dGp,dAp,dTp,dTp,dCp,dGp,dCp,dTp,dTp,dTp,dTp,dCp,dCp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>264</th>
      <td>Apt_NDV01</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7190.png" width="120" ></td>
      <td>Newcastle disease virus (NDV) (avian disease)</td>
      <td>Other</td>
      <td>31 nM (reported value)</td>
      <td>PBS (pH 7.2) with 150 mM NaCl and 5 mM MgCl2.</td>
      <td>RT°C</td>
      <td>None</td>
      <td></td>
      <td>24984.1 g/mole</td>
      <td>58.75%</td>
      <td>775800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>80</td>
      <td>Marnissi, B.; Kamali-Moghaddam, M.; Ghram, A.; Hmila, I. Generation of ssDNA aptamers as diagnostic tool for Newcastle avian virus. PLoS One. 2020 Aug 13; 15(8): e0237253. DOI: 10.1371/journal.pone.0237253.</td>
      <td>dAp,dGp,dTp,dGp,dCp,dAp,dAp,dGp,dCp,dAp,dGp,dTp,dAp,dTp,dTp,dCp,dGp,dGp,dTp,dCp,dGp,dGp,dGp,dTp,dCp,dTp,dTp,dGp,dCp,dAp,dGp,dGp,dTp,dCp,dCp,dCp,dGp,dTp,dAp,dGp,dGp,dAp,dGp,dGp,dGp,dGp,dCp,dCp,dAp,dTp,dTp,dGp,dGp,dAp,dGp,dTp,dGp,dGp,dGp,dGp,dTp,dAp,dAp,dAp,dGp,dCp,dTp,dGp,dAp,dTp,dGp,dCp,dGp,dTp,dGp,dAp,dTp,dGp,dCp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>265</th>
      <td>Anti-NF-kB p65 (R2)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/388.png" width="120" ></td>
      <td>NF-kB p65</td>
      <td>Protein</td>
      <td>25 nM (reported value)</td>
      <td>Selection buffer: 10mM HEPES at pH 7.5, 100mM NaCl, 1mM DTT</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>25801.63 g/mole</td>
      <td>51.25%</td>
      <td>803700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>80</td>
      <td>Wurster, S. E, and Maher, L. J. "Selection and characterization of anti-NF-kB p65 aptamers." RNA, 14(2008): 1037-1047.</td>
      <td>rGp,rAp,rAp,rGp,rCp,rUp,rUp,rUp,rCp,rAp,rCp,rAp,rCp,rAp,rAp,rCp,rAp,rAp,rGp,rGp,rCp,rCp,rCp,rGp,rGp,rGp,rAp,rCp,rUp,rGp,rUp,rAp,rUp,rUp,rAp,rGp,rGp,rGp,rAp,rAp,rAp,rUp,rUp,rAp,rGp,rAp,rGp,rUp,rAp,rCp,rAp,rGp,rAp,rCp,rAp,rGp,rUp,rCp,rGp,rCp,rCp,rGp,rUp,rGp,rGp,rGp,rUp,rCp,rGp,rAp,rAp,rUp,rUp,rCp,rCp,rGp,rCp,rUp,rCp,rAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>266</th>
      <td>E-Selectin</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/267.png" width="120" ></td>
      <td>E-Selectin</td>
      <td>Protein</td>
      <td></td>
      <td>Assay Buffer (PBS, 0.1% BSA, 424 µM Ca(NO3)2, 406 µM MgSO4, 0.02% Tween 20), 4 °C.</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>1699.99 g/mole</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>12</td>
      <td>Martens et al. "Peptides Which Bind to E-selectin and Block Neutrophil Adhesion." The Journal of Biological Chemistry, 270 (1995):21129-21136.</td>
      <td></td>
      <td>4 nM (reported value)</td>
    </tr>
    <tr>
      <th>267</th>
      <td>VEGF (3R02 bivalent)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/493.png" width="120" ></td>
      <td>Vascular Endothelial Growth Factor (VEGF)</td>
      <td>Protein</td>
      <td>30 pM (reported value)</td>
      <td>Binding buffer: 10 mM Tris-HCl, 100 mM NaCl, and 5 mM KCl (pH 7.4)</td>
      <td>20°C</td>
      <td>\n                                Heat to 95°C for 5 minutes in binding buffer, then gradually cool. \n</td>
      <td></td>
      <td>18836.24 g/mole</td>
      <td>56.67%</td>
      <td>560300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Bivalent.  Each monomer is expected to form an anti-parallel G-quadruplex.</td>
      <td>60</td>
      <td>None</td>
      <td>dTp,dGp,dTp,dGp,dGp,dGp,dGp,dGp,dTp,dGp,dGp,dAp,dCp,dTp,dGp,dGp,dGp,dTp,dGp,dGp,dGp,dTp,dAp,dCp,dCp,dTp,dTp,dTp,dTp,dTp,dTp,dTp,dTp,dTp,dTp,dTp,dGp,dTp,dGp,dGp,dGp,dGp,dGp,dTp,dGp,dGp,dAp,dCp,dTp,dGp,dGp,dGp,dTp,dGp,dGp,dGp,dTp,dAp,dCp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>268</th>
      <td>Human Gastric Adenocarcinoma</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/265.png" width="120" ></td>
      <td>Human Gastric Adenocarcinoma</td>
      <td>Tissue</td>
      <td>N/A nM (reported value)</td>
      <td>None</td>
      <td>N/A°C</td>
      <td>None</td>
      <td>Only binds to the endothelial cells of human gastric adenocarcinoma blood vessels, not to blood vessels of human hepatocarcinoma, hemangioma, or normal spleen tissues.</td>
      <td>991.17 g/mole</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>9</td>
      <td>None</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>269</th>
      <td>cAMP</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/104.png" width="120" ></td>
      <td>cAMP</td>
      <td>Small Organic</td>
      <td>10 µM (reported value)</td>
      <td>None</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>10044.17 g/mole</td>
      <td>51.61%</td>
      <td>316000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>4</td>
      <td>Kozumi and Breaker. "Molecular Recognition of cAMP by an RNA Aptamer." Biochemistry, 39 (2000): 8983-8992.</td>
      <td>rGp,rGp,rAp,rAp,rGp,rAp,rGp,rAp,rUp,rGp,rGp,rCp,rGp,rAp,rCp,rUp,rAp,rAp,rAp,rAp,rCp,rGp,rAp,rCp,rUp,rUp,rGp,rUp,rCp,rGp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>270</th>
      <td>IFNα-3</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7120.jpg" width="120" ></td>
      <td>Cytokine interferon alpha (IFN-α)</td>
      <td>Protein</td>
      <td>3.96 ± 0.36 nM (reported value)</td>
      <td>10 mM Tris pH 7.5, 10 mM MgCl2, 50 mM KCl, 25 mM NaCl</td>
      <td>RT°C</td>
      <td>None</td>
      <td></td>
      <td>24529.8 g/mole</td>
      <td>52.50%</td>
      <td>730600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>80</td>
      <td>None</td>
      <td>dGp,dTp,dCp,dTp,dTp,dGp,dAp,dCp,dTp,dAp,dGp,dTp,dTp,dAp,dCp,dGp,dCp,dCp,dCp,dAp,dCp,dAp,dAp,dGp,dAp,dAp,dCp,dCp,dTp,dGp,dCp,dTp,dTp,dGp,dCp,dCp,dAp,dTp,dGp,dGp,dTp,dGp,dAp,dCp,dGp,dCp,dCp,dGp,dAp,dTp,dCp,dAp,dTp,dGp,dCp,dTp,dTp,dTp,dTp,dTp,dGp,dGp,dTp,dCp,dAp,dTp,dTp,dCp,dAp,dGp,dTp,dTp,dGp,dGp,dCp,dGp,dCp,dCp,dTp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>271</th>
      <td>2'-O-Methyl modified RNA aptamer</td>
      <td>2'-O-Me-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/512.jpg" width="120" ></td>
      <td>p53</td>
      <td>Protein</td>
      <td>N/A nM (reported value)</td>
      <td>0.15 mM Hepes [pH 7.4], 0.3 mM KCl, 6 uM dithiothreitol [DTT], 6 uM MgCl2, 1.1 uM ethylenediaminetetraacetic acid [EDTA], 6% glycerol, and 3 uM phenylmethanesulfonyl fluoride [PMSF]</td>
      <td>23°C</td>
      <td>None</td>
      <td></td>
      <td>25257.7 g/mole</td>
      <td>48.00%</td>
      <td>753400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>75</td>
      <td>Shwetha, N. et al. "Aptamerƒ??nanoparticle-based chemiluminescence for p53 protein." Analytical Biochemistry 2013, 441: 73-79.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>272</th>
      <td>von Willebrand Factor (VWF R9.14)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/482.png" width="120" ></td>
      <td>von Willebrand Factor (VWF)</td>
      <td>Protein</td>
      <td>12 nM (reported value)</td>
      <td>None</td>
      <td>37°C</td>
      <td>None</td>
      <td>Specifically inhibited ristocetin-induced platelet aggregation and not collagen-induced, ADP-induced, or thrombin-induced platelet aggregation.</td>
      <td>24563.62 g/mole</td>
      <td>59.21%</td>
      <td>731400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>82</td>
      <td>None</td>
      <td>rGp,rGp,rGp,rAp,rGp,rGp,rAp,fCp,rGp,rAp,fUp,rGp,fCp,rGp,rGp,fUp,rGp,rGp,rAp,fCp,rGp,rAp,rAp,fCp,fUp,rGp,fCp,fCp,fCp,fUp,fCp,rAp,rGp,fCp,fUp,rAp,fCp,fUp,fUp,fUp,fCp,rAp,fUp,rGp,fUp,fUp,rGp,fCp,fUp,rGp,rAp,fCp,rGp,fCp,rAp,fCp,rAp,rGp,rAp,fCp,rGp,rAp,fCp,fUp,fCp,rGp,fCp,fUp,rGp,rAp,rGp,rGp,rAp,fUp,fCp,fCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>273</th>
      <td>Islets of Langerhans</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/250.png" width="120" ></td>
      <td>Islets of Langerhans</td>
      <td>Tissue</td>
      <td>N/A nM (reported value)</td>
      <td>None</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>1174.47 g/mole</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>9</td>
      <td>Yao et al. "Targeting Pancreatic Islets with Phage Display Assisted by Laser Pressure Catapult Microdissection." American Journal of Pathology, 166(2005): 625-636.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>274</th>
      <td>T30695</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7221.png" width="120" ></td>
      <td>Pb2</td>
      <td>Other</td>
      <td>nM (reported value)</td>
      <td>15 μL of NaCl solution (0.5 mM), 30 μL of aptamer (1 μM)</td>
      <td>23°C</td>
      <td>\n                                mFold \n</td>
      <td>Highly specific, mutant versions (M4, M16, and M4-16) of this aptamer are even more so.</td>
      <td>5105.28 g/mole</td>
      <td>75.00%</td>
      <td>157200 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>This aptamer is a Pb^2 binding aptamers with mutant versions designated as M4, M16, and M4-16. Most of its binding ability was determined in silico. The calculated binding free energy of this aptamer to Pb^2 is −1397.01 kj /mol.</td>
      <td>16</td>
      <td>https://doi.org/10.1016/j.molliq.2019.111159.</td>
      <td>dGp,dGp,dGp,dTp,dGp,dGp,dGp,dTp,dGp,dGp,dGp,dTp,dGp,dGp,dGp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>275</th>
      <td>Azo-1</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7192.png" width="120" ></td>
      <td>Human Thrombin</td>
      <td>Protein</td>
      <td>Not given. nM (reported value)</td>
      <td>20 mM Tris-HCl (pH 7.4), 1 mM MgCl2, 10 mM NaCl, and 0.05%Tween-20.</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>21442.7 g/mole</td>
      <td>55.71%</td>
      <td>658400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Each U in the sequence is a dU^Az residue. These are azobenzene-modified deoxyuridine.</td>
      <td>70</td>
      <td>Morihiro, K.; Hasegawa, O.; Kasahara, Y.; Mori, S.; Kasai, T.; Kuwahara, M.; Obika, S. Azobenzene-modified DNA aptamers evolved by capillary electrophoresis (CE)-SELEX method. Bioorg. Med. Chem. Lett. 2021 Jan 1; 31: 127607. DOI: 10.1016/j.bmcl.2020.127607.</td>
      <td>dTp,dCp,dGp,dCp,dCp,dTp,dTp,dGp,dCp,dCp,dGp,dGp,dAp,dTp,dCp,dGp,dCp,dAp,dGp,dAp,dGp,rUp,dAp,dAp,dGp,dGp,dCp,dAp,dGp,dCp,rUp,rUp,dAp,dCp,dAp,dAp,dAp,dCp,dAp,rUp,rUp,dAp,dAp,dGp,dGp,dCp,dAp,dCp,dAp,dGp,rUp,dGp,dGp,rUp,dCp,dCp,dGp,rUp,dGp,dAp,dGp,dCp,dCp,rUp,dGp,dAp,dCp,dAp,dCp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>276</th>
      <td>EP23</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7161.jpg" width="120" ></td>
      <td>Epithelial cellular activating molecule (EpCAM)</td>
      <td>Cells</td>
      <td>39.89 nM (reported value)</td>
      <td>PBS containing 5 mM MgCl2</td>
      <td>25 and 37°C</td>
      <td>None</td>
      <td></td>
      <td>5631.6 g/mole</td>
      <td>47.37%</td>
      <td>165300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>19</td>
      <td>Bell DR, Weber JK, Yin W, Huynh T, Duan W, Zhou R. In silico design and validation of high-affinity RNA aptamers targeting epithelial cellular adhesion molecule dimers. Proc Natl Acad Sci U S A. 2020 Apr 14;117(15):8486-8493. doi: 10.1073/pnas.1913242117. Epub 2020 Mar 31. Erratum in: Proc Natl Acad Sci U S A. 2021 Feb 16;118(7): PMID: 32234785; PMCID: PMC7165443.</td>
      <td>rAp,rCp,rGp,rUp,rAp,rUp,rCp,rCp,rCp,rUp,rUp,rUp,rUp,rCp,rGp,rCp,rGp,rUp,rAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>277</th>
      <td>4-1BB ligand (M12-23)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/358.png" width="120" ></td>
      <td>4-1BB ligand on CD8+ T Cells</td>
      <td>Protein</td>
      <td>40 nM (reported value)</td>
      <td>None</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>31154.72 g/mole</td>
      <td>58.33%</td>
      <td>952100 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>108</td>
      <td>None</td>
      <td>rGp,rGp,rGp,rAp,rGp,rAp,rGp,rAp,rGp,rGp,rAp,rAp,rGp,rAp,rGp,rGp,rGp,rAp,fUp,rGp,rGp,rGp,fCp,rGp,rAp,fCp,fCp,rGp,rAp,rAp,fCp,rGp,fUp,rGp,fCp,fCp,fCp,fUp,fUp,fCp,rAp,rAp,rAp,rGp,fCp,fCp,rGp,fUp,fUp,fCp,rAp,fCp,fUp,rAp,rAp,fCp,fCp,rAp,rGp,fUp,rGp,rGp,fCp,rAp,fUp,rAp,rAp,fCp,fCp,fCp,rAp,rGp,rAp,rGp,rGp,fUp,fCp,rGp,rAp,fUp,rAp,rGp,fUp,rAp,fCp,fUp,rGp,rGp,rAp,fUp,fCp,fCp,fCp,fCp,fCp,fCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>278</th>
      <td>RBP4 binding aptamer (RBA)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/408.png" width="120" ></td>
      <td>Retinol-binding protein-4 (RBP4)</td>
      <td>Protein</td>
      <td>201 nM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td>No significant signal was observed from Vaspin, Nampt, adiponectin, human serum albumin, human IgG, and fibrinogen.</td>
      <td>23679.45 g/mole</td>
      <td>47.37%</td>
      <td>755800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Limit of detection using ELAAS assay was 78 ng/ml. See SJ Lee et al (2012) for more details.</td>
      <td>78</td>
      <td>None</td>
      <td>dAp,dTp,dAp,dCp,dCp,dAp,dGp,dCp,dTp,dTp,dAp,dTp,dTp,dCp,dAp,dAp,dTp,dTp,dAp,dCp,dAp,dGp,dTp,dAp,dGp,dTp,dGp,dAp,dGp,dGp,dGp,dGp,dTp,dCp,dCp,dGp,dTp,dCp,dGp,dTp,dGp,dGp,dGp,dGp,dTp,dAp,dGp,dTp,dTp,dGp,dGp,dGp,dTp,dCp,dGp,dTp,dGp,dGp,dAp,dGp,dAp,dTp,dAp,dGp,dTp,dAp,dAp,dGp,dTp,dGp,dCp,dAp,dAp,dTp,dCp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>279</th>
      <td>EGFR (CL4)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/446.png" width="120" ></td>
      <td>EGFR - Extracellular Domain</td>
      <td>Protein</td>
      <td>10 nM (reported value)</td>
      <td>Buffer used for Binding Assays: PBS with 0.01% BSA.  Incubation Time: 15 minutes.</td>
      <td>37°C</td>
      <td>\n                                85°C for 5 minutes in DMEM serum free, snap-cooled on ice for 2 minutes, gradual warming to 37°C (from article reference [34]) \n</td>
      <td>High affinity for human extracellular EGFR (both disulfide-linked EGFR dimer and reduced EGFR monomer).  No appreciable binding to extracellular ErbB3.</td>
      <td>12561.44 g/mole</td>
      <td>51.28%</td>
      <td>386200 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>This apatamer was shown to induce apoptosis in A549 cells and inhibit tumor growth in human non-small-cell lung cancer xenograft mice.</td>
      <td>39</td>
      <td>None</td>
      <td>rGp,fCp,fCp,fUp,fUp,rAp,rGp,fUp,rAp,rAp,fCp,rGp,fUp,rGp,fCp,fUp,fUp,fUp,rGp,rAp,fUp,rGp,fUp,fCp,rGp,rAp,fUp,fUp,fCp,rGp,rAp,fCp,rAp,rGp,rGp,rAp,rGp,rGp,fCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>280</th>
      <td>PP3</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7125.jpg" width="120" ></td>
      <td>Hemagglutinin</td>
      <td>Protein</td>
      <td>2.61-3.87 nM (reported value)</td>
      <td>4.5 g/L glucose\nand 5 mM MgCl2 in Dulbecco’s PBS (Sigma),BSA (1 mg/mL from Fisher)\nwith t-RNA (0.1 mg/mL)</td>
      <td>4°C</td>
      <td>None</td>
      <td></td>
      <td>23387.2 g/mole</td>
      <td>50.00%</td>
      <td>730700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>84</td>
      <td>Parekh, P., Tang, Z., Turner, P. C., Moyer, R. W., & Tan, W. (2010). Aptamers Recognizing Glycosylated Hemagglutinin Expressed on the Surface of Vaccinia Virus-Infected Cells. Analytical Chemistry, 82(20), 8642–8649. doi:10.1021/ac101801j</td>
      <td>dAp,dTp,dCp,dCp,dAp,dGp,dAp,dGp,dTp,dGp,dAp,dCp,dGp,dCp,dAp,dGp,dCp,dAp,dCp,dGp,dAp,dGp,dCp,dCp,dAp,dGp,dAp,dCp,dAp,dTp,dCp,dTp,dCp,dAp,dCp,dAp,dCp,dCp,dTp,dGp,dTp,dTp,dGp,dCp,dAp,dTp,dAp,dTp,dAp,dCp,dAp,dTp,dTp,dTp,dTp,dGp,dCp,dAp,dTp,dGp,dGp,dAp,dCp,dAp,dCp,dGp,dGp,dTp,dGp,dGp,dCp,dTp,dTp,dAp,dGp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>281</th>
      <td>TROP2 (Tac B1)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7219.jpg" width="120" ></td>
      <td>TROP2 FcHis / Human trophoblast antigen 2</td>
      <td>Protein</td>
      <td>153 nM (reported value)</td>
      <td>Buffer, Annealing - 1 mM MgCl2 in PBS\n\nBuffer, Binding - 20 mM Tris-HCl buffer [pH 7.4], 10 mM NaCl, 1 mM MgCl2, and 0.05% Tween 20)</td>
      <td>37°C</td>
      <td>\n                                 On day 2, the Utrp-modified library (0.1 nmol) from round 5 of CE-SELEX was dissolved in the annealing buffer (1 mM MgCl2 in phosphate-buffered saline [PBS]). The resulting mixture was heated at 95 ◦C for 1 min, then allowed to cool slowly to 25 ◦C for 60 min to refold the aptamers. \n</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>The last four "T" bases in the secondary structure image are actually supposed to be "Utrp" as seen in the primary sequence code</td>
      <td>70</td>
      <td>Uemachi, H.; Kasahara, Y.; Tanaka, K.; Okuda, T.; Yoneda, Y.; and S. Obika. 2021 . Hybrid-Type SELEX for the Selection of Artificial Nucleic Acid\nAptamers Exhibiting Cell Internalization Activity. Pharmaceutics 2021, 13, 888. https://doi.org/10.3390/pharmaceutics13060888</td>
      <td>dTp,dCp,dGp,dCp,dCp,dTp,dTp,dGp,dCp,dCp,dGp,dGp,dAp,dTp,dCp,dGp,dCp,dAp,dGp,dAp,dTp,dGp,dCp,dTp,dGp,dTp,dTp,dGp,dTp,dCp,dAp,dCp,dCp,dTp,dGp,dCp,dCp,dTp,dCp,dGp,dTp,dCp,dTp,dCp,dCp,dCp,dTp,dCp,dGp,dTp,fUp,dGp,dGp,fUp,dCp,dCp,dGp,fUp,dGp,dAp,dGp,dCp,dCp,fUp,dGp,dAp,dCp,dAp,dCp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>282</th>
      <td>Plasminogen Activator Inhibitor-1 (paionap-40)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/465.png" width="120" ></td>
      <td>Plasminogen Activator Inhibitor-1</td>
      <td>Protein</td>
      <td>1.23 nM (reported value)</td>
      <td>HBS buffer (pH 7.4), 2mM MgCl2, 0.05% Tween 20.</td>
      <td>N/A°C</td>
      <td>None</td>
      <td>E.Coli active PAI-1 variant</td>
      <td>25454.12 g/mole</td>
      <td>45.57%</td>
      <td>800200 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>79</td>
      <td>Madsen, J.B., et al. "RNA Aptamers as Conformational Probes and Regulatory Agents for Plasminogen Activator Inhibitor-1." Biochemistry, 49, (2010), 4103-4115.</td>
      <td>rGp,rGp,rGp,rGp,fCp,fCp,rAp,fCp,fCp,rAp,rAp,fCp,rGp,rAp,fCp,rAp,fUp,fUp,fUp,rAp,fUp,fCp,rGp,rAp,rAp,fUp,fUp,rGp,rAp,fUp,rAp,rAp,fCp,fCp,fUp,fUp,rAp,fCp,rGp,fCp,rGp,rAp,rGp,rAp,rGp,fCp,rGp,fUp,rAp,rGp,fUp,fUp,fCp,rGp,fUp,fUp,rGp,rAp,fUp,rAp,fUp,rAp,rAp,rAp,fUp,rAp,rGp,fUp,rGp,fCp,fCp,fCp,rAp,fUp,rGp,rGp,rAp,fUp,fCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>283</th>
      <td>Platelet-Derived Growth Factor (PDGF) B-chain (36t)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/142.jpg" width="120" ></td>
      <td>Platelet-Derived Growth Factor -AB (PDGF-AB)</td>
      <td>Protein</td>
      <td>0.094 nM (reported value)</td>
      <td>None</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>12266 g/mole</td>
      <td>56.41%</td>
      <td>382300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Contains a terminal 3'-3'-linked thymidine residue to protect from 3' exonuclease degradation.</td>
      <td>40</td>
      <td>Green, L., et al. "Inhibitory DNA Ligands to Platelet-Derived Growth Factor B-Chain." Journal of Biochemistry, 35(1996): 14413-14424.</td>
      <td>dCp,dAp,dCp,dAp,dGp,dGp,dCp,dTp,dAp,dCp,dGp,dGp,dCp,dAp,dCp,dGp,dTp,dAp,dGp,dAp,dGp,dCp,dAp,dTp,dCp,dAp,dCp,dCp,dAp,dTp,dGp,dAp,dTp,dCp,dCp,dTp,dGp,dTp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>284</th>
      <td>Aptamer 1.12</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/545.jpg" width="120" ></td>
      <td>Ochratoxin A (OTA)</td>
      <td>Protein</td>
      <td>0.36 µM (reported value)</td>
      <td>Selection Buffer (SB): 10mM HEPES, pH 7.0, 120mM NaCl, 5mM KCl, 5mM MgCl2</td>
      <td>°C</td>
      <td>\n                                Heat to 90C for 5 minutes, followed by 30 minutes at room temperature. \n</td>
      <td>Equilibrium dialysis of 100nM OTA in selection buffer showed approximately 95% binding to Aptamer 1.12 at an aptamer concentration of 10uM. Addition of 20nM NAP or 20nM warfarin to the mix had no significant effect on the binding of OTA to aptamer. With the same conditions as OTA, OTB (Ochratoxin B)</td>
      <td>19102.42 g/mole</td>
      <td>59.02%</td>
      <td>597700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>30NT Random Region: GCATCTGATCGGGTGTGGGTGGCGTAAAGG ,     Forward Primer: Cy3-TGGTGGCTGTAGGTCA ,     Reverse Primer: Biotin-CGTTGTCCGATGCTC</td>
      <td>61</td>
      <td>Jorge A. Cruz-Aguado, Gregory Penner, "Determination of Ochratoxin A with a DNA Aptamer", J. Agric. Food Chem. (2008), 56, 10456-10461.</td>
      <td>dTp,dGp,dGp,dTp,dGp,dGp,dCp,dTp,dGp,dTp,dAp,dGp,dGp,dTp,dCp,dAp,dGp,dCp,dAp,dTp,dCp,dTp,dGp,dAp,dTp,dCp,dGp,dGp,dGp,dTp,dGp,dTp,dGp,dGp,dGp,dTp,dGp,dGp,dCp,dGp,dTp,dAp,dAp,dAp,dGp,dGp,dGp,dAp,dGp,dCp,dAp,dTp,dCp,dGp,dGp,dAp,dCp,dAp,dAp,dCp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>285</th>
      <td>Histone (H4K16Ac)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/356.png" width="120" ></td>
      <td>H4K16Ac</td>
      <td>Peptide</td>
      <td>47 nM (reported value)</td>
      <td>None</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>14926.78 g/mole</td>
      <td>45.83%</td>
      <td>476300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>48</td>
      <td>None</td>
      <td>dAp,dGp,dAp,dCp,dGp,dTp,dAp,dAp,dGp,dTp,dTp,dAp,dAp,dTp,dTp,dGp,dGp,dAp,dCp,dTp,dTp,dGp,dGp,dTp,dCp,dGp,dTp,dGp,dTp,dGp,dCp,dGp,dGp,dCp,dAp,dCp,dAp,dGp,dCp,dGp,dAp,dTp,dTp,dGp,dAp,dAp,dAp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>286</th>
      <td>Heteroaryl dihydropyrimidine (Aptamer 21)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/201.png" width="120" ></td>
      <td>Heteroaryldihydropyrimidine (HAP)</td>
      <td>Small Organic</td>
      <td>50 nM (reported value)</td>
      <td>None</td>
      <td>20°C</td>
      <td>None</td>
      <td></td>
      <td>25553.41 g/mole</td>
      <td>56.96%</td>
      <td>774800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>RNA was refolded prior to selection process.</td>
      <td>79</td>
      <td>None</td>
      <td>rGp,rGp,rGp,rUp,rAp,rGp,rGp,rCp,rCp,rAp,rGp,rGp,rCp,rAp,rGp,rCp,rCp,rAp,rAp,rCp,rUp,rAp,rGp,rCp,rGp,rAp,rGp,rAp,rGp,rCp,rUp,rUp,rAp,rAp,rAp,rUp,rCp,rUp,rCp,rUp,rGp,rAp,rGp,rCp,rCp,rCp,rGp,rAp,rGp,rAp,rGp,rGp,rGp,rUp,rUp,rCp,rAp,rGp,rUp,rGp,rCp,rUp,rGp,rCp,rUp,rUp,rAp,rUp,rGp,rUp,rGp,rGp,rAp,rCp,rGp,rGp,rCp,rUp,rUp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>287</th>
      <td>TAFapt1-S</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7135.png" width="120" ></td>
      <td>AML1-MTG8</td>
      <td>Protein</td>
      <td>4.47 nM (reported value)</td>
      <td>Binding buffer [20 mM sodium phosphate (pH 6.5), 2 mM magnesium acetate, 200 mM potassium acetate, 5% glycerol, 0.05% Triton X-100, 5 mM b-mercaptoethanol].\n\nBinding buffer [20 mM Tris/HCl (pH 7.5) containing 50 mM KCl and 1 mM DTT].</td>
      <td>25°C</td>
      <td>None</td>
      <td></td>
      <td>11413.3 g/mole</td>
      <td>62.16%</td>
      <td>348600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>37</td>
      <td>Fukunaga, Junichi et al. “A G-quadruplex-forming RNA aptamer binds to the MTG8 TAFH domain and dissociates the leukemic AML1-MTG8 fusion protein from DNA.” FEBS letters vol. 594,21 (2020): 3477-3489. doi:10.1002/1873-3468.13914</td>
      <td>rGp,rGp,rGp,rUp,rUp,rUp,rUp,rCp,rGp,rGp,rGp,rAp,rUp,rAp,rCp,rGp,rCp,rGp,rGp,rAp,rGp,rGp,rGp,rUp,rGp,rGp,rGp,rCp,rAp,rAp,rUp,rAp,rAp,rCp,rCp,rCp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>288</th>
      <td>Sp1</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7209.jpg" width="120" ></td>
      <td>Shigella sonnei</td>
      <td>Other</td>
      <td>5.98 nM (reported value)</td>
      <td>Tris BB</td>
      <td>37°C</td>
      <td>None</td>
      <td>binding with S. sonnei is six times greater than other, related bacteria species.</td>
      <td>24547.59 g/mole</td>
      <td>48.75%</td>
      <td>720600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Dual aptamer with Sp20</td>
      <td>80</td>
      <td>None</td>
      <td>dTp,dGp,dAp,dGp,dCp,dCp,dCp,dAp,dAp,dGp,dCp,dCp,dCp,dTp,dGp,dGp,dTp,dAp,dTp,dGp,dTp,dTp,dCp,dTp,dTp,dCp,dCp,dCp,dTp,dTp,dTp,dTp,dAp,dTp,dTp,dAp,dGp,dTp,dCp,dCp,dTp,dGp,dTp,dAp,dTp,dTp,dCp,dCp,dTp,dCp,dTp,dAp,dCp,dTp,dGp,dTp,dTp,dGp,dCp,dCp,dGp,dGp,dCp,dAp,dGp,dGp,dTp,dCp,dTp,dAp,dCp,dTp,dTp,dTp,dGp,dGp,dGp,dAp,dTp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>289</th>
      <td>Salmonella Enteritidis (S25)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/288.png" width="120" ></td>
      <td>Salmonella Enteritidis</td>
      <td>Cells</td>
      <td>N/A nM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td></td>
      <td>28847.27 g/mole</td>
      <td>50.00%</td>
      <td>875300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>99</td>
      <td>Hyeon, J. et al. Development of RNA Aptamers for Detection of Salmonellas Enteritidis. Journal of Microbiological Methods, 89(2012), 79-82.</td>
      <td>rGp,rGp,rGp,rUp,rUp,rCp,rAp,rCp,rUp,rGp,rCp,rAp,rGp,rAp,rCp,rUp,rUp,rGp,rAp,rCp,rGp,rAp,rAp,rGp,rCp,rUp,rUp,rGp,rAp,rGp,rAp,rGp,rAp,rUp,rGp,rCp,rCp,rCp,rCp,rCp,rUp,rGp,rAp,rUp,rGp,rUp,rGp,rCp,rAp,rUp,rUp,rCp,rUp,rUp,rGp,rUp,rUp,rGp,rUp,rGp,rUp,rUp,rGp,rCp,rGp,rGp,rCp,rAp,rAp,rUp,rGp,rGp,rAp,rUp,rCp,rCp,rAp,rCp,rAp,rUp,rCp,rUp,rAp,rCp,rGp,rAp,rAp,rUp,rUp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>290</th>
      <td>Bovine Catalase (CAT 1)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/307.png" width="120" ></td>
      <td>Bovine Catalase</td>
      <td>Protein</td>
      <td>237 nM (reported value)</td>
      <td>None</td>
      <td>N/A°C</td>
      <td>\n                                Heated to 94°C for 10 min and then cooled at 0.5° C/s to 20°C. \n</td>
      <td>At least a 100-fold decrease in affinity to lysozyme, trypsinogen, chymotrypsinogenA, and myoglobin relative to the target.</td>
      <td>25356.41 g/mole</td>
      <td>60.98%</td>
      <td>774800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>The Kd value listed was determined using affinity capillary electrophoresis. The Kd value using a fluorescence intensity assay was 430 nM.</td>
      <td>87</td>
      <td>None</td>
      <td>dCp,dTp,dTp,dCp,dTp,dGp,dCp,dCp,dCp,dGp,dCp,dCp,dTp,dCp,dCp,dTp,dTp,dCp,dCp,dGp,dAp,dCp,dCp,dTp,dAp,dGp,dCp,dAp,dGp,dTp,dGp,dGp,dAp,dCp,dAp,dTp,dGp,dTp,dGp,dGp,dCp,dAp,dGp,dGp,dGp,dTp,dGp,dAp,dAp,dGp,dTp,dGp,dGp,dCp,dAp,dTp,dCp,dGp,dTp,dCp,dGp,dGp,dAp,dGp,dAp,dCp,dGp,dAp,dGp,dAp,dTp,dAp,dGp,dGp,dCp,dGp,dGp,dAp,dCp,dAp,dCp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>291</th>
      <td>Apta 3</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7180.png" width="120" ></td>
      <td>Carcinoembryonic antigen (CEA)</td>
      <td>Protein</td>
      <td>60.4 nM (reported value)</td>
      <td>Not given.</td>
      <td>37°C</td>
      <td>None</td>
      <td>Apta 3 displayed high specificity and affinity for T84 cells (human carcinoma cell line derived from a lung metastasis of a colon carcinoma).</td>
      <td>21611.9 g/mole</td>
      <td>67.14%</td>
      <td>631300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>70</td>
      <td>Melo, M.; Correa, C.; Cunha, P.; Goes, A.; Gomes, D.; Andrade, A. DNA aptamers selection for carcinoembryonic antigen (CEA). Bioorg. Med. Chem. Lett. 2020 May 23; 30(15): 127278. DOI: 10.1016/j.bmcl.2020.127278</td>
      <td>dTp,dCp,dGp,dCp,dGp,dCp,dGp,dAp,dGp,dTp,dCp,dGp,dTp,dCp,dTp,dGp,dGp,dGp,dGp,dGp,dGp,dGp,dTp,dGp,dTp,dAp,dTp,dCp,dGp,dTp,dTp,dGp,dAp,dCp,dGp,dAp,dGp,dTp,dTp,dGp,dCp,dGp,dCp,dGp,dTp,dGp,dCp,dGp,dTp,dCp,dTp,dCp,dGp,dTp,dGp,dCp,dCp,dGp,dCp,dAp,dTp,dCp,dGp,dTp,dCp,dCp,dTp,dCp,dCp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>292</th>
      <td>Trypanosoma brucei (2-16)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/145.png" width="120" ></td>
      <td>Trypanosoma brucei</td>
      <td>Cells</td>
      <td>0.75 nM (reported value)</td>
      <td>Binding Buffer (20 mM NaxHyPO4 (pH 7.4),2 mM MgCl2,130 mM NaCl,5 mM KCl, 20 mM glucose,0.2 mM ??-mercaptoethanol)</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>27369.42 g/mole</td>
      <td>63.53%</td>
      <td>805600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>85</td>
      <td>Homann and Goringer. "Combinatorial selection of high affinity RNA ligands to live African trypanosome." Nucleic Acid Research, 27(1999): 2006-2014.</td>
      <td>rGp,rGp,rGp,rAp,rGp,rAp,rCp,rGp,rAp,rUp,rAp,rUp,rUp,rCp,rGp,rUp,rCp,rCp,rAp,rUp,rUp,rCp,rGp,rGp,rGp,rUp,rGp,rGp,rCp,rCp,rCp,rGp,rUp,rGp,rUp,rCp,rUp,rGp,rAp,rGp,rCp,rGp,rGp,rGp,rGp,rAp,rCp,rGp,rGp,rCp,rCp,rAp,rCp,rUp,rUp,rGp,rAp,rGp,rCp,rGp,rCp,rCp,rGp,rCp,rUp,rGp,rUp,rCp,rCp,rGp,rAp,rCp,rUp,rGp,rAp,rAp,rUp,rUp,rCp,rUp,rCp,rGp,rAp,rCp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>293</th>
      <td>Nucleolin aptamer (AS1411)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/290.png" width="120" ></td>
      <td>Nucleolin</td>
      <td>Protein</td>
      <td>N/A nM (reported value)</td>
      <td>None</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>8272.41 g/mole</td>
      <td>65.38%</td>
      <td>250800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>28</td>
      <td>Kotula J, Sullenger B, et al. 2012. Aptamer-mediated delivery of splice-switching oligonucleotides to the nuclei of cancer cells. Nucleic Acid and Therapeutics 22.3:187-195.  Li, L. et al. 2014. Aptamer photoregulation in vivo. PNAS 11.48: 17099-103.</td>
      <td>dGp,dGp,dTp,dGp,dGp,dTp,dGp,dGp,dTp,dGp,dGp,dTp,dTp,dGp,dTp,dGp,dGp,dTp,dGp,dGp,dTp,dGp,dGp,dTp,dGp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>294</th>
      <td>Sgc-3b</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7126.jpg" width="120" ></td>
      <td>Selectin L.</td>
      <td>Protein</td>
      <td>nM (reported value)</td>
      <td>PBS with 5 mm MgCl2, 1 mm CaCl2, and 0.1 mg/ml herring sperm DNA</td>
      <td>4°C</td>
      <td>None</td>
      <td></td>
      <td>17627.5 g/mole</td>
      <td>40.35%</td>
      <td>558100 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>63</td>
      <td>Bing, T., Shangguan, D., & Wang, Y. (2015). Facile Discovery of Cell-Surface Protein Targets of Cancer Cell Aptamers. Molecular & Cellular Proteomics, 14(10), 2692–2700. doi:10.1074/mcp.m115.051243</td>
      <td>dTp,dTp,dTp,dAp,dCp,dTp,dTp,dAp,dTp,dTp,dCp,dAp,dAp,dTp,dTp,dCp,dCp,dCp,dGp,dTp,dGp,dGp,dGp,dAp,dAp,dGp,dGp,dCp,dTp,dAp,dTp,dAp,dGp,dAp,dGp,dGp,dGp,dGp,dCp,dCp,dAp,dGp,dTp,dCp,dTp,dAp,dTp,dGp,dAp,dAp,dTp,dAp,dAp,dGp,dTp,dTp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>295</th>
      <td>Antibac1</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/630.jpg" width="120" ></td>
      <td>Peptidoglycan</td>
      <td>Other</td>
      <td>415 nM (reported value)</td>
      <td>PBS 1x</td>
      <td>23°C</td>
      <td>\n                                N/A \n</td>
      <td>0.415 uM for peptidoglycan  31.82 nM for E. coli</td>
      <td>7395.81 g/mole</td>
      <td>72.73%</td>
      <td>220800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>This aptamer was originally selected for peptidoglycan, but it is referenced in an apta-report because it was also demonstrated to have affinity for a variety of bacteria cells, including E. coli by Ana Claudia Graziani, Maria Isabel Stets, Ana Luisa Kalb Lopes, Pedro Henrique Caires Schluga, Soledad Marton, Ieda Ferreira Mendes, Antero Silva Ribeiro de Andrade, Marco Aur??lio Krieger, Josiane Cardoso. J. Microbiol. Biotechnol.2017 ; 27(4): 838~843</td>
      <td>55</td>
      <td>Ferreira IM, de Souza Lacerda CM, de Faria LS, Corrêa CR, de Andrade AS. Selection of peptidoglycan-specific aptamers for bacterial cells identification. Appl Biochem Biotechnol. 2014 Dec;174(7):2548-56. doi: 10.1007/s12010-014-1206-6. Epub 2014 Sep 4. PMID: 25185503.</td>
      <td>dTp,dCp,dGp,dCp,dGp,dCp,dGp,dAp,dGp,dTp,dCp,dGp,dTp,dCp,dTp,dGp,dGp,dGp,dGp,dAp,dCp,dAp,dGp,dGp,dGp,dAp,dGp,dTp,dGp,dCp,dGp,dCp,dTp,dGp,dCp,dTp,dCp,dCp,dCp,dCp,dCp,dCp,dGp,dCp,dAp,dTp,dCp,dGp,dTp,dCp,dCp,dTp,dCp,dCp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>296</th>
      <td>Mouse White Fat</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/248.png" width="120" ></td>
      <td>Mouse White Fat</td>
      <td>Tissue</td>
      <td>N/A nM (reported value)</td>
      <td>N/A</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>1019.27 g/mole</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>9</td>
      <td>Kolonin et al. "Teratogenicity induced by targeting a placental  immunoglobulin transporter." PNAS, 99(2002): 13055-13060.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>297</th>
      <td>HIV-1 R5 SU Glycoprotein gp120 (B40t77)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/366.png" width="120" ></td>
      <td>HIV-1 R5  gp120</td>
      <td>Protein</td>
      <td>31 nM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td></td>
      <td>24701.55 g/mole</td>
      <td>54.55%</td>
      <td>737700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>84</td>
      <td>None</td>
      <td>rGp,rGp,rGp,rAp,rGp,rAp,fCp,rAp,rAp,rGp,rAp,fCp,fUp,rAp,rGp,rAp,fCp,rGp,fCp,fUp,fCp,rAp,rAp,fUp,rGp,fUp,rGp,rGp,rGp,fCp,fCp,rAp,fCp,rGp,fCp,fCp,fCp,rGp,rAp,fUp,fUp,fUp,fUp,rAp,fCp,rGp,fCp,fUp,fUp,fUp,fUp,rAp,fCp,fCp,fCp,rGp,fCp,rAp,fCp,rGp,fCp,rGp,rAp,fUp,fUp,rGp,rGp,fUp,fUp,fUp,rGp,fUp,fUp,fUp,fCp,fCp,fCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>298</th>
      <td>Yeast phenylalanine tRNA (B2)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/428" width="120" ></td>
      <td>Yeast phenylalanine tRNA</td>
      <td>Small Organic</td>
      <td>12 nM (reported value)</td>
      <td>Selection Buffer (250 mM NaCl, 50 mM Tris-HCl (pH 7.5), 10 mM MgCl2, 2 mM spermidine, 0.2 mM EDTA)</td>
      <td>25°C</td>
      <td>None</td>
      <td></td>
      <td>35011.9 g/mole</td>
      <td>53.21%</td>
      <td>1060400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>109</td>
      <td>Scarabino, Daniela, et al. "tRNA prefers to kiss." EMBO Journal 18 (1999): 4571-4578</td>
      <td>rGp,rGp,rGp,rAp,rAp,rUp,rUp,rCp,rCp,rGp,rCp,rGp,rUp,rGp,rUp,rGp,rCp,rUp,rAp,rCp,rGp,rUp,rAp,rUp,rCp,rUp,rUp,rCp,rAp,rGp,rGp,rCp,rGp,rGp,rUp,rAp,rAp,rCp,rUp,rAp,rAp,rCp,rUp,rGp,rUp,rGp,rCp,rUp,rGp,rAp,rGp,rUp,rCp,rUp,rAp,rAp,rUp,rCp,rUp,rUp,rUp,rGp,rUp,rGp,rAp,rGp,rGp,rGp,rAp,rCp,rGp,rGp,rUp,rAp,rAp,rCp,rAp,rUp,rAp,rUp,rGp,rGp,rUp,rUp,rCp,rCp,rCp,rGp,rCp,rGp,rUp,rGp,rGp,rUp,rCp,rCp,rGp,rUp,rUp,rCp,rGp,rGp,rGp,rAp,rUp,rCp,rCp,rUp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>299</th>
      <td>Anionic Porphyrins (ST1)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/485.png" width="120" ></td>
      <td>N-methylmesoporphyrin IX (NMM), mesoporphyrin IX (MPIX), and other metalloderivatives</td>
      <td>Small Organic</td>
      <td>0.7 µM (reported value)</td>
      <td>SB Buffer: 100 mM Tris-acetate (pH 7.4), 200 mM sodium acetate, 25 mM potassium acetate, 10 mM magnesium acetate, 0.5% Triton X-100, and 5% dimethyl sulfoxide.</td>
      <td>RT°C</td>
      <td>\n                                Heated to 90°C and then slowly cooled (over an hour) to room temperature. \n</td>
      <td>Kd listed is for NMM. Aptamer also binds to MPIX (Kd = 3.6 μM), Fe(III)-MPIX (2.6 μM), hemin (1.7 μM), Ni(II)-MPIX (3.6 μM), and Zn(II)-MPIX (4.9 μM).</td>
      <td>7883.18 g/mole</td>
      <td>60.00%</td>
      <td>253400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Thought to form a G-quartet</td>
      <td>25</td>
      <td>None</td>
      <td>dAp,dAp,dCp,dGp,dTp,dGp,dGp,dGp,dAp,dGp,dGp,dGp,dCp,dGp,dGp,dTp,dGp,dGp,dTp,dGp,dTp,dTp,dGp,dAp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>300</th>
      <td>Tb-L8</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7119.jpg" width="120" ></td>
      <td>Thrombin</td>
      <td>Protein</td>
      <td>33 nM (reported value)</td>
      <td>50 mM HEPES, 100 mM NaCl, 1 mM MgCl2, 5 mM KCl, 1 mM,CaCl2, pH 7.4</td>
      <td>20°C</td>
      <td>None</td>
      <td></td>
      <td>12601.1 g/mole</td>
      <td>65.00%</td>
      <td>389600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>44</td>
      <td>Qiao, N., Li, J., Wu, X., Diao, D., Zhao, J., Li, J., … Lou, X. (2019). Speeding up in vitro Discovery of Structure-Switching Aptamers via Magnetic Cross-Linking Precipitation. Analytical Chemistry. doi:10.1021/acs.analchem.9b00081</td>
      <td>dGp,dGp,dTp,dGp,dTp,dGp,dGp,dGp,dCp,dGp,dAp,dCp,dAp,dTp,dGp,dGp,dGp,dGp,dGp,dTp,dTp,dGp,dGp,dTp,dGp,dGp,dTp,dGp,dTp,dAp,dGp,dGp,dAp,dGp,dCp,dGp,dCp,dCp,dAp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>301</th>
      <td>AptD2</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7220.png" width="120" ></td>
      <td>DEN-NS1</td>
      <td>Protein</td>
      <td>104 nM (reported value)</td>
      <td>None</td>
      <td>23°C</td>
      <td>None</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>"Dsp" and "Pap" are expanded alphabet bases - see Reference for more details.</td>
      <td>63</td>
      <td>Ken-ichiro Matsunaga, Michiko Kimoto, Vanessa Weixun Lim, Hui Pen Tan, Yu Qian Wong, William Sun, Shawn Vasoo, Yee Sin Leo, Ichiro Hirao, High-affinity five/six-letter DNA aptamers with superior specificity enabling the detection of dengue NS1 protein variants beyond the serotype identification, Nucleic Acids Research, 2021;, gkab515, https://doi.org/10.1093/nar/gkab515</td>
      <td>dGp,dGp,dCp,dTp,dGp,dGp,dTp,dCp,dCp,dGp,dCp,dTp,dGp,dGp,dGp,dAp,dAp,dCp,dAp,dAp,dGp,dGp,dGp,dCp,dGp,dGp,dGp,dAp,dGp,dGp,dGp,dAp,dGp,dGp,dGp,dTp,dGp,dTp,dGp,dGp,dGp,dTp,dGp,dCp,dGp,dAp,dCp,dAp,dAp,dGp,dCp,dGp,dGp,dAp,dCp,dCp,dAp,dGp,dCp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>302</th>
      <td>i-6</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7211.png" width="120" ></td>
      <td>Glycan modification</td>
      <td>Other</td>
      <td>29.5 µM (reported value)</td>
      <td>40 mM Tris-HCl, 240 mM NaCl, 10 mM KCl, 2 mM MgCl2, 2 mM CaCl2, and 0.01% Tween-20 in nuclease-free water</td>
      <td>22°C</td>
      <td>None</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Contains indole-modified bases. See reference article and its supplemental information for details.</td>
      <td>90</td>
      <td>Discovery of indole-modified aptamers for highly specific recognition of protein glycoforms\nAlex M. Yoshikawa, Alexandra Rangel, Trevor Feagin, Elizabeth M. Chun, Leighton Wan, Anping Li, Leonhard Moekl, Michael Eisenstein, Sharon Pitteri, H. Tom Soh\nbioRxiv 2021.03.13.435263; doi: https://doi.org/10.1101/2021.03.13.435263</td>
      <td>dAp,dGp,dCp,dAp,dGp,dCp,dAp,dCp,dAp,dGp,dAp,dGp,dGp,dTp,dCp,dAp,dGp,dAp,dTp,dGp,dAp,dTp,dCp,dTp,dCp,dGp,dTp,dGp,dTp,dGp,dTp,dTp,dTp,dGp,dGp,dTp,dTp,dTp,dTp,dAp,dGp,dAp,dTp,dTp,dTp,dTp,dTp,dGp,dGp,dAp,dAp,dAp,dTp,dTp,dAp,dTp,dGp,dAp,dTp,dTp,dCp,dCp,dTp,dGp,dTp,dTp,dGp,dGp,dTp,dCp,dCp,dCp,dTp,dAp,dTp,dGp,dCp,dGp,dTp,dGp,dCp,dTp,dAp,dCp,dCp,dGp,dTp,dGp,dAp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>303</th>
      <td>Interleukin-17A/F (IL-17A/F) (APTAF42)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/119.jpg" width="120" ></td>
      <td>Human Interleukin-17A/F</td>
      <td>Protein</td>
      <td>72.2 nM (reported value)</td>
      <td>Binding Buffer (145 mM sodium chloride, 5.4 mM potassium chloride, 0.8 mM magnesium chloride, 1.8 mM calcium chloride, 20 mM Trisƒ??HCl (pH 7.6) supplemented with 0.05% Tween20</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>22313.31 g/mole</td>
      <td>59.42%</td>
      <td>670800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>69</td>
      <td>Adachi, H., et al. "Antagonistic RNA aptamer specific to a heterodimeric form of human interleukin-17A/F." Biochemie, 93 (2011): 1081-1088.</td>
      <td>rGp,rGp,rGp,fCp,fUp,rAp,rGp,fCp,fUp,rGp,rAp,fUp,fCp,rGp,fUp,rAp,fCp,fCp,fCp,rAp,rGp,fUp,rAp,rGp,fCp,rGp,fUp,rGp,rGp,fCp,rAp,fUp,rGp,rGp,rGp,rGp,fUp,rGp,fCp,fCp,fUp,rAp,rGp,fUp,fCp,rGp,rGp,rGp,fCp,rGp,rAp,fUp,rAp,fCp,fUp,rAp,rAp,fCp,rAp,rGp,fCp,fUp,rAp,rAp,fCp,rAp,fCp,fCp,fCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>304</th>
      <td>Sp1</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7225.png" width="120" ></td>
      <td>Shigella sonnei</td>
      <td>Cells</td>
      <td>5.980 nM (reported value)</td>
      <td>50 mM Tris–HCl, 5 mM KCl, 100 mM NaCl, 1 mM MgCl2, pH\n7.4</td>
      <td>37°C</td>
      <td>\n                                mFold \n</td>
      <td>Highly Specific to the pathogenic enterobacteria Shigella sonnei</td>
      <td>12061.73 g/mole</td>
      <td>40.00%</td>
      <td>345300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>40</td>
      <td>DOI: 10.1039/c5ay00214a\n</td>
      <td>dTp,dTp,dCp,dTp,dTp,dCp,dCp,dCp,dTp,dTp,dTp,dTp,dAp,dTp,dTp,dAp,dGp,dTp,dCp,dCp,dTp,dGp,dTp,dAp,dTp,dTp,dCp,dCp,dTp,dCp,dTp,dAp,dCp,dTp,dGp,dTp,dTp,dGp,dCp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>305</th>
      <td>PCB77 (II)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/304.png" width="120" ></td>
      <td>3,3,4,4-tetrachlorobiphenyl (PCB77)</td>
      <td>Small Organic</td>
      <td>8.32 µM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td></td>
      <td>11783.7 g/mole</td>
      <td>55.26%</td>
      <td>374200 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>38</td>
      <td>Shengmin, X. (2012). Selection of dna aptamers against polychlorinated biphenyls as potential biorecognition elements for environmental analysis. Analytical Biochemistry, 423(2012), 195-201.</td>
      <td>dAp,dAp,dGp,dCp,dGp,dCp,dGp,dCp,dGp,dAp,dGp,dAp,dCp,dTp,dAp,dCp,dGp,dTp,dTp,dTp,dGp,dTp,dAp,dGp,dGp,dGp,dAp,dTp,dAp,dCp,dCp,dGp,dAp,dTp,dGp,dTp,dCp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>306</th>
      <td>Heparanase (1.5M NaCl)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/425.png" width="120" ></td>
      <td>Heparanase (HPSE1)</td>
      <td>Protein</td>
      <td>12 nM nM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td></td>
      <td>22348.62 g/mole</td>
      <td>45.83%</td>
      <td>727000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>74</td>
      <td>None</td>
      <td>dGp,dGp,dGp,dAp,dGp,dAp,dCp,dAp,dAp,dGp,dAp,dAp,dTp,dAp,dAp,dAp,dCp,dGp,dCp,dTp,dCp,dAp,dAp,dAp,dTp,dGp,dGp,dAp,dCp,dTp,dTp,dTp,dTp,dGp,dAp,dAp,dTp,dGp,dTp,dGp,dGp,dCp,dAp,dAp,dCp,dAp,dAp,dAp,dTp,dTp,dCp,dGp,dAp,dCp,dAp,dGp,dGp,dAp,dGp,dGp,dCp,dTp,dCp,dAp,dCp,dAp,dAp,dCp,dAp,dGp,dGp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>307</th>
      <td>L-Valine</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/153.jpg" width="120" ></td>
      <td>L-Valine</td>
      <td>Small Organic</td>
      <td>12 mM (reported value)</td>
      <td>Selection Buffer (25 mM HEPES, 200 mM NaCl, 6 mM MgCl2, 1 mM EDTA (pH 7.3))</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>26818.29 g/mole</td>
      <td>55.42%</td>
      <td>821800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>83</td>
      <td>Majerfeld and Yarus. "An RNA Pocket for an Aliphatic Hyrdrophobe." Nature Structural Biology, 1(1994): 287-292.</td>
      <td>rGp,rGp,rGp,rAp,rGp,rCp,rUp,rCp,rAp,rGp,rAp,rAp,rUp,rAp,rAp,rAp,rCp,rGp,rCp,rUp,rCp,rAp,rAp,rAp,rUp,rCp,rCp,rGp,rUp,rGp,rGp,rAp,rCp,rAp,rGp,rGp,rGp,rCp,rGp,rUp,rAp,rAp,rGp,rCp,rGp,rCp,rCp,rUp,rUp,rCp,rGp,rAp,rCp,rAp,rUp,rGp,rAp,rGp,rAp,rCp,rAp,rCp,rGp,rGp,rAp,rUp,rCp,rCp,rUp,rGp,rCp,rGp,rAp,rCp,rGp,rAp,rAp,rUp,rUp,rCp,rAp,rGp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>308</th>
      <td>Osteopontin (OPN-R3)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/370.png" width="120" ></td>
      <td>Human Osteopontin</td>
      <td>Protein</td>
      <td>18 nM (reported value)</td>
      <td>None</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>12862.72 g/mole</td>
      <td>45.00%</td>
      <td>402900 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>41</td>
      <td>None</td>
      <td>fCp,rGp,rGp,fCp,fCp,rAp,fCp,rAp,rGp,rAp,rAp,fUp,rGp,rAp,rAp,rAp,rAp,rAp,fCp,fCp,fUp,fCp,rAp,fUp,fCp,rGp,rAp,fUp,rGp,fUp,fUp,rGp,fCp,rAp,fUp,rAp,rGp,fUp,fUp,rGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>309</th>
      <td>Bacillus thuringiensis spores</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/391.png" width="120" ></td>
      <td>Bacillus thuringiensis spores</td>
      <td>Cells</td>
      <td>N/A nM (reported value)</td>
      <td>PBS</td>
      <td>RT°C</td>
      <td>None</td>
      <td></td>
      <td>18379.9 g/mole</td>
      <td>60.00%</td>
      <td>548400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>61</td>
      <td>None</td>
      <td>dCp,dAp,dTp,dCp,dCp,dGp,dTp,dCp,dAp,dCp,dAp,dCp,dCp,dTp,dGp,dCp,dTp,dCp,dTp,dGp,dGp,dCp,dCp,dAp,dCp,dTp,dAp,dAp,dCp,dAp,dTp,dGp,dGp,dGp,dGp,dAp,dCp,dCp,dAp,dGp,dGp,dTp,dGp,dGp,dTp,dGp,dTp,dTp,dGp,dGp,dCp,dTp,dCp,dCp,dCp,dGp,dTp,dAp,dTp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>310</th>
      <td>Anti-HSA Structure-switching Aptamer</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7117.jpg" width="120" ></td>
      <td>HSA (Human Serum Albumin)</td>
      <td>Protein</td>
      <td>20 ± 4 nM (reported value)</td>
      <td>Binding buffer for anti-HSA aptamer selection: 50 mM HEPES, 100 mM NaCl, 1 mM MgCl2, 5 mM KCl, 1 mM CaCl2, pH 7.4.\n</td>
      <td>23°C</td>
      <td>\n                                95°C for 10 minutes, slow cooling to room temperature \n</td>
      <td>Demonstrated specificity to HSA over streptavidin, short peptides, and BSA (bovine serum albumin).</td>
      <td>18459 g/mole</td>
      <td>48.33%</td>
      <td>582100 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>60</td>
      <td>Qiao, N., Li, J., Wu, X., Diao, D., Zhao, J., Li, J., … Lou, X. (2019). Speeding up in vitro Discovery of Structure-Switching Aptamers via Magnetic Cross-Linking Precipitation. Analytical Chemistry. doi:10.1021/acs.analchem.9b00081</td>
      <td>dCp,dGp,dTp,dAp,dCp,dCp,dGp,dGp,dCp,dCp,dAp,dGp,dTp,dGp,dAp,dTp,dTp,dAp,dCp,dGp,dAp,dCp,dGp,dAp,dGp,dAp,dCp,dGp,dAp,dGp,dCp,dTp,dTp,dAp,dTp,dGp,dCp,dGp,dTp,dAp,dTp,dTp,dGp,dAp,dTp,dGp,dCp,dCp,dTp,dAp,dAp,dCp,dTp,dAp,dTp,dCp,dTp,dAp,dCp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>311</th>
      <td>AMPA GluR2 (AN58)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/444.png" width="120" ></td>
      <td>AMPA GluR2 Glutamate Receptor Channel</td>
      <td>Protein</td>
      <td>419 pM (reported value)</td>
      <td>None</td>
      <td>22°C</td>
      <td>None</td>
      <td></td>
      <td>18676.31 g/mole</td>
      <td>50.00%</td>
      <td>584600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>This aptamer forms two different structures, both of which bind to GluR2 and both of which are necessary to inhibit the receptor channel. See Huang (2009) for more details.</td>
      <td>58</td>
      <td>None</td>
      <td>rGp,rGp,rGp,rCp,rGp,rAp,rAp,rUp,rUp,rCp,rAp,rAp,rCp,rUp,rGp,rCp,rCp,rAp,rUp,rCp,rUp,rAp,rGp,rGp,rCp,rAp,rGp,rUp,rAp,rAp,rCp,rCp,rAp,rGp,rGp,rAp,rGp,rUp,rUp,rAp,rGp,rUp,rAp,rGp,rGp,rAp,rCp,rAp,rAp,rGp,rUp,rUp,rUp,rCp,rGp,rUp,rCp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>312</th>
      <td>Sialyl Lewis X (5)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/48.png" width="120" ></td>
      <td>Sialyl Lewis X</td>
      <td>Other</td>
      <td>0.085 nM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td></td>
      <td>42804.72 g/mole</td>
      <td>46.62%</td>
      <td>1336400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>The reported sequence contains Y(pyrimidine) at site 46 and 67. We've chosen uracil (U) to replace the arbitrary Y.  The reported sequence contains R(purine) at site 101. We've chosen adenine(A) to replace the arbitrary R.</td>
      <td>133</td>
      <td>Jeong, Sunjoo, et al. "In Vitro Selection of the RNA Aptamer against the Sialyl Lewis X and Its Inhibition of the Cell Adhesion." Biochemical and Biophysical Research Communications, 281 (2001): 237ƒ??243.</td>
      <td>rCp,rCp,rGp,rUp,rAp,rAp,rUp,rAp,rCp,rGp,rAp,rCp,rUp,rCp,rAp,rCp,rUp,rAp,rUp,rAp,rGp,rGp,rGp,rGp,rAp,rGp,rCp,rUp,rCp,rGp,rGp,rUp,rAp,rCp,rCp,rGp,rAp,rAp,rUp,rUp,rCp,rAp,rAp,rGp,rGp,rUp,rAp,rCp,rUp,rCp,rUp,rGp,rUp,rGp,rCp,rUp,rUp,rGp,rUp,rCp,rGp,rAp,rUp,rGp,rUp,rGp,rUp,rAp,rUp,rUp,rGp,rAp,rUp,rGp,rGp,rCp,rAp,rCp,rUp,rUp,rUp,rCp,rGp,rAp,rGp,rUp,rCp,rAp,rAp,rCp,rGp,rAp,rGp,rUp,rUp,rGp,rAp,rCp,rAp,rGp,rAp,rAp,rCp,rAp,rAp,rGp,rUp,rAp,rGp,rUp,rCp,rAp,rAp,rGp,rCp,rUp,rUp,rUp,rGp,rCp,rAp,rGp,rAp,rGp,rAp,rGp,rGp,rAp,rUp,rCp,rCp,rUp,rUp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>313</th>
      <td>G12msi</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7130.jpg" width="120" ></td>
      <td>GPC3</td>
      <td>Protein</td>
      <td>0.17 nM (reported value)</td>
      <td>4.5 g/L glucose, 5 mM MgCl2, 0.1 mg/mL yeast tRNA, and 1 mg/mL\nbovine serum albumin</td>
      <td>25°C</td>
      <td>None</td>
      <td></td>
      <td>11399.4 g/mole</td>
      <td>35.29%</td>
      <td>358601.769336 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>In the sequence: D - Gemcitabine</td>
      <td>37</td>
      <td>Park, J. Y., Chae, J. R., Cho, Y. L., Kim, Y., Lee, D., Lee, J. K., & Kang, W. J. (2020). Targeted Therapy of Hepatocellular Carcinoma Using Gemcitabine-Incorporated GPC3 Aptamer. Pharmaceutics, 12(10), 985. doi:10.3390/pharmaceutics12100985</td>
      <td>dGp,dTp,dGp,dAp,dAp,dTp,dTp,dGp,dCp,dGp,dTp,dTp,dGp,dAp,dAp,dTp,dTp,dAp,dTp,dGp,dTp,dTp,dTp,dCp,dAp,dGp,dTp,dCp,dAp,dTp,dCp,dAp,dCp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>314</th>
      <td>Tetracycline</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/191.png" width="120" ></td>
      <td>Tetracycline</td>
      <td>Small Organic</td>
      <td>63.6 nM (reported value)</td>
      <td>None</td>
      <td>N/A°C</td>
      <td>None</td>
      <td>Weakly binds to doxycycline and oxytetracycline.</td>
      <td>23367.05 g/mole</td>
      <td>68.42%</td>
      <td>688400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Aptamer was immobilized on steptavidin-modified screen-printed gold electrodes to create a sensitive aptasensor for tetracycline (Kim et al. 2010). The range of detection was 0.01-10 µM.</td>
      <td>76</td>
      <td>None</td>
      <td>dCp,dGp,dTp,dAp,dCp,dGp,dGp,dAp,dAp,dTp,dTp,dCp,dGp,dCp,dTp,dAp,dGp,dCp,dCp,dCp,dCp,dCp,dCp,dGp,dGp,dCp,dAp,dGp,dGp,dCp,dCp,dAp,dCp,dGp,dGp,dCp,dTp,dTp,dGp,dGp,dGp,dTp,dTp,dGp,dGp,dTp,dCp,dCp,dCp,dAp,dCp,dTp,dGp,dCp,dGp,dCp,dGp,dTp,dGp,dGp,dAp,dTp,dCp,dCp,dGp,dAp,dGp,dCp,dTp,dCp,dCp,dAp,dCp,dGp,dTp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>315</th>
      <td>VEGF-Stimulated HUVECs</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/232.jpg" width="120" ></td>
      <td>VEGF-Stimulated Human Umbilical Vein Endothelial Cells (HUVECs)</td>
      <td>Tissue</td>
      <td>N/A nM (reported value)</td>
      <td>None</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>985.29 g/mole</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>8</td>
      <td>None</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>316</th>
      <td>Chagas Apt-79</td>
      <td>Chimeric</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/597.jpg" width="120" ></td>
      <td>T. cruzi Excreted Secreted Antigens (TESA)</td>
      <td>Protein</td>
      <td>See source nM (reported value)</td>
      <td>PBS, mouse plasma</td>
      <td>23°C</td>
      <td>\n                                65 C for 10 minutes. 23 C for 1 hour. \n</td>
      <td></td>
      <td>22341.44 g/mole</td>
      <td>40.28%</td>
      <td>722800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>216</td>
      <td>de Araujo FF, Nagarkatti R, Gupta C, Marino AP, Debrabant A (2015) Aptamer-Based Detection of Disease Biomarkers in Mouse Models for Chagas Drug  Discovery. PLoS Negl Trop Dis 9(1): e3451. doi:10.1371/journal.pntd.0003451</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>317</th>
      <td>#AP-Cell 3</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7204.png" width="120" ></td>
      <td>Aspartate β-Hydroxylase (ASPH)</td>
      <td>Protein</td>
      <td>65.23 ± 9.61 nM (reported value)</td>
      <td>None</td>
      <td>°C</td>
      <td>\n                                m-fold \n</td>
      <td></td>
      <td>16017.28 g/mole</td>
      <td>59.62%</td>
      <td>498100 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>52</td>
      <td>Hadi Bakhtiari, Abbas Ali Palizban, Hossein Khanahmad, and Mohammad Reza Mofid\nACS Omega 2021 6 (16), 11005-11014\nDOI: 10.1021/acsomega.1c00876</td>
      <td>dCp,dCp,dGp,dTp,dAp,dTp,dCp,dGp,dCp,dCp,dCp,dAp,dGp,dGp,dCp,dAp,dAp,dCp,dTp,dGp,dGp,dGp,dCp,dTp,dAp,dAp,dAp,dCp,dTp,dTp,dCp,dCp,dCp,dAp,dGp,dAp,dGp,dGp,dGp,dAp,dAp,dCp,dGp,dAp,dAp,dAp,dCp,dCp,dTp,dGp,dGp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>318</th>
      <td>Human Norovirus SMV-21</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/578.jpg" width="120" ></td>
      <td>Human Norovirus</td>
      <td>Other</td>
      <td>101 nM (reported value)</td>
      <td>500 ml of binding buffer [consisting of phosphate buffered saline (pH 7.1) supplemented with 100 mg/L CaCl2, 100 mg/L MgCl2 and 0.05% Tween 20] at Room Temperature</td>
      <td>24°C</td>
      <td>None</td>
      <td>See page 8 of source material. Aptamer specific to several strains of the Human Norovirus</td>
      <td>25046.34 g/mole</td>
      <td>43.21%</td>
      <td>781000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>245</td>
      <td>Escudero-Abarca BI, Suh SH, Moore MD, Dwivedi HP, Jaykus L-A (2014) Selection, Characterization and Application of Nucleic Acid Aptamers for the Capture and Detection of Human Norovirus Strains. PLoS ONE 9(9): e106805. doi:10.1371/journal.pone.0106805</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>319</th>
      <td>Heparanase (3M NaSCN)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/424.png" width="120" ></td>
      <td>Heparanase (HPSE1)</td>
      <td>Protein</td>
      <td>13 nM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td></td>
      <td>22240.55 g/mole</td>
      <td>43.06%</td>
      <td>719300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>74</td>
      <td>None</td>
      <td>dGp,dGp,dGp,dAp,dGp,dAp,dCp,dAp,dAp,dGp,dAp,dAp,dTp,dAp,dAp,dAp,dCp,dGp,dCp,dTp,dCp,dAp,dAp,dTp,dTp,dTp,dAp,dAp,dCp,dGp,dTp,dAp,dTp,dTp,dTp,dAp,dTp,dTp,dCp,dAp,dAp,dGp,dCp,dTp,dCp,dGp,dTp,dAp,dTp,dTp,dCp,dGp,dAp,dCp,dAp,dGp,dGp,dAp,dGp,dGp,dCp,dTp,dCp,dAp,dCp,dAp,dAp,dCp,dAp,dGp,dGp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>320</th>
      <td>??-Catenin</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/468.png" width="120" ></td>
      <td>Arm 1-12 of B-catenin</td>
      <td>Protein</td>
      <td>5 nM (reported value)</td>
      <td>None</td>
      <td>N/A°C</td>
      <td>\n                                N/A \n</td>
      <td></td>
      <td>26107.74 g/mole</td>
      <td>56.79%</td>
      <td>792100 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Expression of this aptamer in cells using the U6 promoter produced nuclear-localized aptamers that bound to ??-catenin, inhibited transcription of cyclin D1 and c-myc, and prevented alternative splicing.</td>
      <td>89</td>
      <td>Lee HK, Choi YS, Park YA, et al. (2006) Modulation of oncogenic transcription and alternative splicing by beta-catenin and an RNA aptamer in colon cancer cells.</td>
      <td>rGp,rGp,rAp,rCp,rGp,rCp,rGp,rUp,rGp,rGp,rUp,rAp,rCp,rCp,rAp,rGp,rGp,rCp,rCp,rGp,rAp,rUp,rCp,rUp,rAp,rUp,rGp,rGp,rAp,rCp,rGp,rCp,rUp,rAp,rUp,rAp,rGp,rGp,rCp,rAp,rCp,rAp,rCp,rCp,rGp,rGp,rAp,rUp,rAp,rCp,rUp,rUp,rUp,rAp,rAp,rCp,rGp,rAp,rUp,rUp,rGp,rGp,rCp,rUp,rAp,rAp,rGp,rCp,rUp,rUp,rCp,rCp,rGp,rCp,rGp,rGp,rGp,rGp,rAp,rUp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>321</th>
      <td>Adenine (12E4)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/96.png" width="120" ></td>
      <td>Adenine</td>
      <td>Small Organic</td>
      <td>10 nM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td></td>
      <td>29362.78 g/mole</td>
      <td>47.25%</td>
      <td>927600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>91</td>
      <td>None</td>
      <td>rGp,rGp,rGp,rAp,rGp,rAp,rGp,rGp,rAp,rUp,rAp,rCp,rUp,rAp,rCp,rAp,rCp,rGp,rUp,rGp,rAp,rUp,rAp,rGp,rGp,rAp,rCp,rGp,rAp,rUp,rUp,rAp,rUp,rCp,rGp,rAp,rAp,rAp,rAp,rUp,rCp,rAp,rCp,rCp,rAp,rGp,rAp,rUp,rUp,rGp,rGp,rAp,rCp,rCp,rCp,rUp,rGp,rGp,rUp,rUp,rAp,rAp,rCp,rGp,rAp,rUp,rCp,rCp,rAp,rUp,rUp,rGp,rCp,rAp,rUp,rGp,rUp,rAp,rGp,rCp,rAp,rGp,rAp,rAp,rGp,rCp,rUp,rUp,rCp,rCp,rGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>322</th>
      <td>SARS-CoV-1 3'-inverted Thymidine Aptamer NG8</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7085.png" width="120" ></td>
      <td>SARS-CoV-1 Helicase</td>
      <td>Protein</td>
      <td>26.8 nM (reported value)</td>
      <td>Buffer A (Binding Buffer and Wash Buffer): 50 mM Tris-HCl pH 6.8, 150 mM KCl, 20 mM imidazole, 0.05% Tween-20, 0.1% Triton X-100  \nBuffer B (Elution Buffer): 30 mL; 50 mM Tris-HCl pH 6.8, 150 mM KCl, 500 mM; imidazole, 0.02% Tween-20, 0.1% Triton\nIncubation Time: 30 min</td>
      <td>25°C</td>
      <td>\n                                Heated to 90 C for 5 min and cooled at 48 C in Buffer A \n</td>
      <td></td>
      <td>28490.55 g/mole</td>
      <td>46.74%</td>
      <td>972910 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>IC50: 17.5 nM</td>
      <td>92</td>
      <td>Shum, K. T., & Tanner, J. A. (2008). Differential Inhibitory Activities and Stabilisation of DNA Aptamers against the SARS Coronavirus Helicase. ChemBioChem, 9(18), 3037-3045. doi:10.1002/cbic.200800491</td>
      <td>dCp,dCp,dGp,dTp,dAp,dAp,dTp,dAp,dCp,dGp,dAp,dCp,dTp,dCp,dAp,dCp,dTp,dAp,dTp,dAp,dGp,dGp,dGp,dGp,dAp,dGp,dCp,dTp,dCp,dGp,dGp,dTp,dAp,dCp,dCp,dGp,dAp,dAp,dTp,dTp,dCp,dAp,dTp,dGp,dTp,dTp,dGp,dGp,dTp,dAp,dGp,dTp,dTp,dGp,dGp,dCp,dTp,dTp,dGp,dTp,dGp,dTp,dTp,dCp,dGp,dTp,dGp,dTp,dGp,dTp,dTp,dAp,dAp,dGp,dCp,dTp,dTp,dTp,dCp,dAp,dGp,dAp,dGp,dAp,dGp,dGp,dAp,dTp,dCp,dCp,dTp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>323</th>
      <td>Protective Antigen (PA) of Bacilus anthracis (MMS_105)</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/255.png" width="120" ></td>
      <td>Protective Antigen (PA) of Bacilus anthracis</td>
      <td>Cells</td>
      <td>19.1 nM (reported value)</td>
      <td>Binding Buffer: PBSB (PBS buffer plus 0.5% BSA), 4°C</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>1967.27 g/mole</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>15</td>
      <td>Kogot et al. "Screening of Peptide Libraries against Protective Antigen of Bacillus anthracis in a Disposable Microfluidic Cartridge." PloS One, 6(11): e26925. doi:10.1371/journal.pone.0026925</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>324</th>
      <td>CH1S-3</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7179.png" width="120" ></td>
      <td>Trastuzumab</td>
      <td>Antibody</td>
      <td>10.3 nM (reported value)</td>
      <td>Binding buffer was prepared by adding 1 mg/mL BSA and 0.1 mg/mL yeast tRNA into the Dulbecco's PBS (DPBS) containing 0.55 mM MgCl2.</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>9030 g/mole</td>
      <td>60.71%</td>
      <td>262400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>The CH1S-3 aptamer was modified with biotin on the 5' end.</td>
      <td>28</td>
      <td>Chen, K.; Zhou, J.; Shao, Z.; Liu, J.; Song, J.; Wang, R.; Li, J.; Tan, W. Aptamers as Versatile Molecular Tools for Antibody Production Monitoring and Quality Control. J. Am. Chem. Soc. 2020 Jun 9; 142(28): 12079-12086. PMID 32516525. DOI: 10.1021/jacs.9b13370.</td>
      <td>dGp,dTp,dCp,dCp,dAp,dGp,dGp,dGp,dTp,dTp,dCp,dCp,dAp,dAp,dGp,dGp,dTp,dGp,dCp,dTp,dTp,dCp,dGp,dTp,dGp,dGp,dAp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>325</th>
      <td>THC1.2</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7213.png" width="120" ></td>
      <td>THC</td>
      <td>Small Organic</td>
      <td>40 nM (reported value)</td>
      <td>20 mM Tris-HCl, \n0.5 mM MgCl2, 20 mM NaCl, pH 7.4</td>
      <td>°C</td>
      <td>\n                                mFold \n</td>
      <td>Not cross-reactive with synthetic cannabinoids or other illicit drugs. Cross-reactive with THC-OOH.</td>
      <td>14787 g/mole</td>
      <td>68.09%</td>
      <td>464000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>47</td>
      <td>Yu, H., Luo, Y., Alkhamis, O., Canoura, J., Yu, B., & Xiao, Y. (2021). Isolation of Natural DNA Aptamers for Challenging Small-Molecule Targets, Cannabinoids. Analytical chemistry, 93(6), 3172–3180. https://doi.org/10.1021/acs.analchem.0c04592</td>
      <td>dCp,dTp,dTp,dAp,dCp,dGp,dAp,dCp,dCp,dCp,dAp,dGp,dGp,dGp,dGp,dGp,dGp,dTp,dGp,dGp,dAp,dCp,dAp,dGp,dGp,dCp,dGp,dGp,dGp,dGp,dGp,dTp,dTp,dAp,dGp,dGp,dGp,dGp,dGp,dGp,dTp,dCp,dGp,dTp,dAp,dAp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>326</th>
      <td>HIV-1 TAR RNA Hairpin Loop (B22-19)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/566" width="120" ></td>
      <td>HIV-1 TAR RNA Hairpin Loop</td>
      <td>Small Organic</td>
      <td>50 nM (reported value)</td>
      <td>R  buffer (20 mM HEPES (pH  7.4), 140 mM  potassium acetate, 20 mM sodium acetate, 3 mM magnesium acetate)</td>
      <td>RT°C</td>
      <td>None</td>
      <td></td>
      <td>5659.71 g/mole</td>
      <td>57.89%</td>
      <td>166500 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>19</td>
      <td>None</td>
      <td>dCp,dCp,dCp,dTp,dAp,dGp,dTp,dTp,dAp,dGp,dCp,dCp,dAp,dTp,dCp,dTp,dCp,dCp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>327</th>
      <td>Apramycin (LGA11)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/426.png" width="120" ></td>
      <td>Apramycin and other aminolycosides (AMGs)</td>
      <td>Small Organic</td>
      <td>28.4 nM (reported value)</td>
      <td>Binding Buffer: PBS with 10% (v/v) methanol</td>
      <td>RT°C</td>
      <td>None</td>
      <td>Binds to several aminoglycosides: kanamycin, paromycin streptomycin, apramycin, tobramycin, neomycin, and gentamycin.</td>
      <td>25568.23 g/mole</td>
      <td>56.96%</td>
      <td>771400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>87</td>
      <td>None</td>
      <td>rGp,rGp,rGp,rAp,rGp,rAp,rAp,rGp,rGp,fCp,rGp,rGp,fCp,rGp,fCp,rGp,fUp,rAp,rGp,rGp,fCp,rGp,rAp,rGp,fCp,fUp,fUp,fUp,rAp,rGp,rGp,fCp,fCp,fCp,rGp,rAp,fCp,rAp,fUp,fUp,fCp,fCp,fCp,fCp,fUp,rAp,rAp,rAp,rAp,rAp,rAp,rGp,fCp,fUp,fUp,rGp,fUp,fUp,fCp,fUp,rAp,fUp,fUp,fCp,rGp,fUp,rGp,rGp,rAp,fCp,rGp,rAp,fUp,rGp,fCp,rGp,fCp,rAp,fCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>328</th>
      <td>S. Enterica Serovar Typhi IVP Pili Protein (S-PS8.4)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/127.jpg" width="120" ></td>
      <td>S. Enterica Serovar Typhi IVP Pili Protein</td>
      <td>Protein</td>
      <td>7.2 nM (reported value)</td>
      <td>Binding Buffer (25 mM Tris-HCl, 50 mM KCl, 200 mM NaCl, 0.2 mM EDTA, 5% [vol/vol] glycerol, 0.5 mM DTT)</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>22923.86 g/mole</td>
      <td>59.15%</td>
      <td>696500 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>71</td>
      <td>Pan, Q., et al. "Aptamers That Preferentially Bind Type IVB Pili and Inhibit Human Monocytic-Cell Invasion by Salmonella enterica Serovar Typhi." Antimicrobial Agents and Chemotherapy, 49(2005): 4052-4060.</td>
      <td>rGp,rGp,rGp,rAp,rAp,rCp,rAp,rGp,rUp,rCp,rCp,rGp,rAp,rGp,rCp,rCp,rUp,rCp,rAp,rCp,rUp,rGp,rUp,rUp,rAp,rUp,rCp,rCp,rGp,rAp,rUp,rAp,rGp,rCp,rAp,rGp,rCp,rGp,rCp,rGp,rGp,rGp,rAp,rUp,rGp,rAp,rGp,rGp,rGp,rUp,rCp,rAp,rAp,rUp,rGp,rCp,rGp,rUp,rCp,rAp,rUp,rAp,rGp,rGp,rAp,rUp,rCp,rCp,rCp,rGp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>329</th>
      <td>V7t1 targeting VEGF-121</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/367.png" width="120" ></td>
      <td>VEGF-121</td>
      <td>Protein</td>
      <td>1.1 nM (reported value)</td>
      <td>10% (v/v) human serum in TBSTE (TBSE containing 0.05% (v/v) of Tween-20).</td>
      <td>24°C</td>
      <td>None</td>
      <td>V7t1 is a truncated DNA aptamer that targets both VEGF-121 and VEGF-165.</td>
      <td>7900.16 g/mole</td>
      <td>72.00%</td>
      <td>248000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>25</td>
      <td>Nonaka,Y., Sode, K., and Ikebukuro, K.(2010) "Screening and Improvement of an Anti-VEGF DNA Aptamer." Molecules, 15, 215-225.</td>
      <td>dTp,dGp,dTp,dGp,dGp,dGp,dGp,dGp,dTp,dGp,dGp,dAp,dCp,dGp,dGp,dGp,dCp,dCp,dGp,dGp,dGp,dTp,dAp,dGp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>330</th>
      <td>Daunomycin (10.10)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/190.png" width="120" ></td>
      <td>Daunomycin</td>
      <td>Small Organic</td>
      <td>272 nM (reported value)</td>
      <td>Binding Buffer (20 mM Tris-HCl (pH 7.4), 140 mM NaCl, 5 mM KCl, 1 mM MgCl2, 1 mM CaCl2, and 0.05% v/v Tween-20)</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>11581.57 g/mole</td>
      <td>56.76%</td>
      <td>366800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>37</td>
      <td>Wochner et al. "A DNA aptamer with high affinity and specificity for therapeutic anthracyclines." Analytical Biochemistry, 373(2008): 34-42.</td>
      <td>dAp,dCp,dCp,dAp,dTp,dCp,dTp,dGp,dTp,dGp,dTp,dAp,dAp,dGp,dGp,dGp,dGp,dTp,dAp,dAp,dGp,dGp,dGp,dGp,dTp,dGp,dGp,dGp,dGp,dGp,dTp,dAp,dCp,dGp,dTp,dCp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>331</th>
      <td>Activated Protein C (APC 99)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/134.jpg" width="120" ></td>
      <td>Human Activated Protein C</td>
      <td>Protein</td>
      <td></td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td></td>
      <td>31865.02 g/mole</td>
      <td>62.63%</td>
      <td>931000 L/(mole·cm)</td>
      <td>137 nM (reported value)</td>
      <td></td>
      <td>Strong serine protease inhibitor of APC.</td>
      <td>110</td>
      <td>Gal, S., et al. "Selection of a RNA aptamer that binds to human activated protein C and inhibits its protease function." European Journal of Biochemistry, 252(1998): 553-562.</td>
      <td>rGp,rUp,rGp,rAp,rGp,rAp,rCp,rCp,rAp,rGp,rCp,rCp,rGp,rAp,rGp,rUp,rGp,rGp,rUp,rGp,rUp,rCp,rUp,rGp,rGp,rCp,rUp,rAp,rUp,rUp,rCp,rAp,rCp,rUp,rGp,rGp,rAp,rGp,rCp,rGp,rUp,rGp,rGp,rGp,rUp,rGp,rGp,rAp,rAp,rCp,rCp,rCp,rCp,rUp,rGp,rCp,rGp,rCp,rAp,rCp,rUp,rCp,rGp,rUp,rUp,rUp,rGp,rGp,rCp,rUp,rGp,rUp,rCp,rCp,rGp,rGp,rGp,rCp,rCp,rUp,rUp,rCp,rGp,rGp,rGp,rCp,rCp,rGp,rGp,rGp,rAp,rUp,rUp,rAp,rUp,rCp,rUp,rCp,rUp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>332</th>
      <td>Nickel (N1)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/93.jpg" width="120" ></td>
      <td>Nickel</td>
      <td>Small Organic</td>
      <td>800 nM (reported value)</td>
      <td>Ni-NTA resin (200 uL) equilibrated with a buffer (50 mM NaCl, 7 mM MgCl2, 50 mM Tris-HCl (pH 7.5) was heated at 75C for 3 min and then slowly cool to RT. The Sample was loaded on the Ni-NTA matrix for 5 min under gentle agitation.</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>28782.5 g/mole</td>
      <td>44.94%</td>
      <td>912100 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>This may be a biphasic binding. 1-2 strong binding sites with Kd 0.8 uM, 14~16 weak binding sites with Kd 29 uM.</td>
      <td>89</td>
      <td>Hoffman, H., et al. "Ni2+-binding RNA motifs with an asymmetric purine-rich internal loop and a G-A base pair." RNA Journal, 3 (1997): 1289-1300.</td>
      <td>rGp,rGp,rGp,rAp,rGp,rAp,rGp,rGp,rAp,rUp,rAp,rCp,rUp,rAp,rCp,rAp,rCp,rGp,rUp,rGp,rGp,rAp,rAp,rAp,rAp,rAp,rCp,rCp,rAp,rAp,rCp,rAp,rAp,rAp,rUp,rUp,rGp,rGp,rGp,rAp,rAp,rAp,rAp,rAp,rUp,rGp,rUp,rUp,rAp,rAp,rGp,rGp,rGp,rUp,rCp,rCp,rAp,rCp,rUp,rUp,rCp,rAp,rUp,rGp,rCp,rCp,rAp,rUp,rUp,rGp,rCp,rAp,rUp,rGp,rUp,rAp,rGp,rCp,rAp,rGp,rAp,rAp,rGp,rCp,rUp,rUp,rCp,rCp,rGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>333</th>
      <td>Plasmodium falciparum lactate dehydrogenase (pL1)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/309.png" width="120" ></td>
      <td>Plasmodium falciparum lactate dehydrogenase  (PfLDH)</td>
      <td>Protein</td>
      <td>38.7 nM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>\n                                Denatured at 90°C for 3 min and then incubated at 4°C for 1 h. \n</td>
      <td></td>
      <td>11163.29 g/mole</td>
      <td>55.56%</td>
      <td>340000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>For details on pL1's use in colorimetric testing, see: Lee S, Manjunatha DH, Jeon W, Ban C (2014) Cationic Surfactant-Based Colorimetric Detection of Plasmodium Lactate Dehydrogenase, a Biomarker for Malaria, Using the Specific DNA Aptamer. PLoS ONE 9(7): e100847. doi:10.1371/journal.pone.0100847</td>
      <td>36</td>
      <td>None</td>
      <td>dGp,dTp,dTp,dCp,dGp,dAp,dTp,dTp,dGp,dGp,dAp,dTp,dTp,dGp,dTp,dGp,dCp,dCp,dGp,dGp,dAp,dAp,dGp,dTp,dGp,dCp,dTp,dGp,dGp,dCp,dTp,dCp,dGp,dAp,dAp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>334</th>
      <td>Epithelial cell adhesion molecule (EpCAM) (EpDT3)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/374.png" width="120" ></td>
      <td>Cancer stem cell marker epithelial cell adhesion molecule</td>
      <td>Protein</td>
      <td>54.5 nM (reported value)</td>
      <td>None</td>
      <td>37°C</td>
      <td>None</td>
      <td>None</td>
      <td>6392.85 g/mole</td>
      <td>68.42%</td>
      <td>188200 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>This aptamer is a modified version of Truncated Clone D.  Also, it appears to be internalized via clathrin-dependent endocytosis following binding. See ref. for more details.</td>
      <td>20</td>
      <td>None</td>
      <td>rGp,fCp,rGp,rAp,fCp,fUp,rGp,rGp,fUp,fUp,rAp,fCp,fCp,fCp,rGp,rGp,fUp,fCp,rGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>335</th>
      <td>C10</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7082.png" width="120" ></td>
      <td>CD19+</td>
      <td>Protein</td>
      <td>49.6 nM (reported value)</td>
      <td>SELEX buffer (1X Hank’s buffered salt solution, 1.4 mM MgCl2), 37C with 1 ug/uL bovine serum albumin and 1 ug/uL salmon-sperm DNA.</td>
      <td>37°C</td>
      <td>\n                                Not applicable. \n</td>
      <td></td>
      <td>15227.85 g/mole</td>
      <td>69.39%</td>
      <td>510660 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Structure not confirmed.  Refolding program not available.</td>
      <td>49</td>
      <td>Mayer, Gunter, et al.  “Fluorescence-activated cell sorting for aptamer SELEX with cell mixtures.”  Nature Protocols 5, (2010): 1993-2004.</td>
      <td>dCp,dCp,dCp,dGp,dGp,dGp,dTp,dGp,dTp,dGp,dGp,dTp,dGp,dGp,dGp,dTp,dGp,dGp,dGp,dCp,dAp,dGp,dGp,dGp,dGp,dGp,dGp,dTp,dTp,dAp,dGp,dCp,dAp,dTp,dCp,dTp,dCp,dAp,dCp,dCp,dAp,dGp,dCp,dTp,dCp,dCp,dGp,dCp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>336</th>
      <td>Sulforhodamine B (Clone 73 - min)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/148.png" width="120" ></td>
      <td>Sulforhodamine B</td>
      <td>Small Organic</td>
      <td>190 nM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td></td>
      <td>9155.94 g/mole</td>
      <td>82.76%</td>
      <td>284400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Kd value listed is for sulforhodamine B immobilized to a column. Free sulforhodamine B binds with Kd= 660 nM.</td>
      <td>29</td>
      <td>None</td>
      <td>dCp,dCp,dGp,dGp,dCp,dCp,dAp,dAp,dGp,dGp,dGp,dTp,dGp,dGp,dGp,dAp,dGp,dGp,dGp,dAp,dGp,dGp,dGp,dGp,dGp,dCp,dCp,dGp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>337</th>
      <td>Enterotoxin B (APT SEB1)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/300.png" width="120" ></td>
      <td>Staphylococcus aureus Enterotoxin B</td>
      <td>Protein</td>
      <td>N/A nM (reported value)</td>
      <td>Binding buffer: 10 mM phosphate buffer, 2.7 mM KCl, 140 mM NaCl, 0.05% Tween, pH 7.4,</td>
      <td>RT°C</td>
      <td>None</td>
      <td></td>
      <td>24022.61 g/mole</td>
      <td>47.44%</td>
      <td>707600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>78</td>
      <td>None</td>
      <td>dGp,dGp,dTp,dAp,dTp,dTp,dGp,dAp,dGp,dGp,dGp,dTp,dCp,dGp,dCp,dAp,dTp,dCp,dCp,dAp,dCp,dTp,dGp,dGp,dTp,dCp,dGp,dTp,dTp,dGp,dTp,dTp,dGp,dTp,dCp,dTp,dGp,dTp,dTp,dGp,dTp,dCp,dTp,dGp,dTp,dTp,dAp,dTp,dGp,dTp,dTp,dGp,dTp,dTp,dTp,dCp,dGp,dTp,dGp,dAp,dTp,dGp,dGp,dCp,dTp,dCp,dTp,dAp,dAp,dCp,dTp,dCp,dTp,dCp,dCp,dTp,dCp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>338</th>
      <td>M4</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7222.png" width="120" ></td>
      <td>Pb2</td>
      <td>Other</td>
      <td>nM (reported value)</td>
      <td>15 μL of NaCl solution (0.5 mM), 30 μL of aptamer (1 μM)</td>
      <td>23°C</td>
      <td>\n                                mFold \n</td>
      <td>Highly specific, more specific than its original aptamer T30695.</td>
      <td>5114.29 g/mole</td>
      <td>75.00%</td>
      <td>161700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>This aptamer is a Pb^2 binding aptamer that is a mutant variation of T30695. The other mutant variants are designated as M16, and M4-16. Most of its binding ability was determined in silico. The calculated binding free energy of this aptamer to Pb^2 is −1483.39 −kj /mol.</td>
      <td>16</td>
      <td>https://doi.org/10.1016/j.molliq.2019.111159.\n</td>
      <td>dGp,dGp,dGp,dAp,dGp,dGp,dGp,dTp,dGp,dGp,dGp,dTp,dGp,dGp,dGp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>339</th>
      <td>A15</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7159.jpg" width="120" ></td>
      <td>Brain Endothelia Cell</td>
      <td>Cells</td>
      <td>48.74 ± 3.11 nM (reported value)</td>
      <td>10 mmol/l HEPES, pH 7.4, 50 mmol/l NaCl, 1 mmol/l MgCl2, 1 mmol/l CaCl2, 0.01% bovine serum albumin</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>21775.7 g/mole</td>
      <td>59.15%</td>
      <td>650400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>71</td>
      <td>Cheng C, Chen YH, Lennox KA, Behlke MA, Davidson BL. In vivo SELEX for Identification of Brain-penetrating Aptamers. Mol Ther Nucleic Acids. 2013;2(1):e67. Published 2013 Jan 8. doi:10.1038/mtna.2012.59</td>
      <td>rGp,rGp,rGp,rAp,rGp,rGp,rAp,rCp,rGp,rAp,rUp,rGp,rCp,rGp,rGp,rCp,rGp,rUp,rAp,rUp,rUp,rGp,rCp,rGp,rCp,rGp,rAp,rGp,rGp,rAp,rUp,rUp,rAp,rUp,rCp,rCp,rGp,rCp,rUp,rCp,rAp,rUp,rCp,rGp,rUp,rUp,rGp,rUp,rUp,rGp,rUp,rUp,rGp,rUp,rGp,rCp,rAp,rGp,rAp,rCp,rGp,rAp,rCp,rUp,rCp,rGp,rCp,rCp,rCp,rGp,rAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>340</th>
      <td>CoV2-RBD-4C</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7218.jpeg" width="120" ></td>
      <td>SARS-CoV-2 S protein / SARS-CoV-2 RBD</td>
      <td>Protein</td>
      <td>13.6 nM (reported value)</td>
      <td>Buffer, Binding - PBS with 0.55 mM MgCl2</td>
      <td>°C</td>
      <td>\n                                Annealed at 95 °C, after which it was immediately cooled on\nice for 5 min and then at 25 °C for 10 min. \n</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>67</td>
      <td>Song, Y., Song, J., Wei, X., Huang, M., Sun, M., Zhu, L., Lin, B., Shen, H., Zhu, Z., and C. Yang. 2020. Discovery of Aptamers Targeting the Receptor-Binding Domain of the SARS-CoV‑2 Spike Glycoprotein. Analytical Chemistry 2020, 92, 9895−9900. https://dx.doi.org/10.1021/acs.analchem.0c01394</td>
      <td>dAp,dTp,dCp,dCp,dAp,dGp,dAp,dGp,dTp,dGp,dAp,dCp,dGp,dCp,dAp,dGp,dCp,dAp,dTp,dTp,dTp,dCp,dAp,dTp,dCp,dGp,dGp,dGp,dTp,dCp,dCp,dAp,dAp,dAp,dAp,dGp,dGp,dGp,dGp,dCp,dTp,dGp,dCp,dTp,dCp,dGp,dGp,dGp,dAp,dTp,dTp,dGp,dCp,dGp,dGp,dAp,dTp,dAp,dTp,dGp,dGp,dAp,dCp,dAp,dCp,dGp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>341</th>
      <td>Neuropeptide Y (DP3)</td>
      <td>2'-NH2-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/441.png" width="120" ></td>
      <td>Neuropeptide Y (NPY)</td>
      <td>Peptide</td>
      <td>370 nM (reported value)</td>
      <td>None</td>
      <td>37°C</td>
      <td>None</td>
      <td>The aptamer did not bind to the similar human pancreatic polypeptide (hPP; ~50% sequence homology).</td>
      <td>45033.34 g/mole</td>
      <td>57.78%</td>
      <td>928300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>92</td>
      <td>None</td>
      <td>rGp,rGp,rGp,rAp,rGp,rAp,rAp,rAp,rGp,rGp,rGp,rAp,rAp,rGp,rGp,rAp,rGp,rAp,rGp,rAp,rGp,rGp,rAp,rGp,rGp,rGp,rGp,rGp,rGp,rAp,rGp,rGp,rGp,rAp,rGp,rGp,rAp,rGp,rGp,rAp,rGp,rAp,rAp,rAp,rGp,rAp,rAp,rGp,rAp,rAp,rGp,rGp,rAp,rAp,rGp,rAp,rGp,rGp,rAp,rGp,rGp,rAp,rGp,rAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>342</th>
      <td>MUC1 (MA3)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/359.png" width="120" ></td>
      <td>MUC1</td>
      <td>Protein</td>
      <td>38.3 nM (reported value)</td>
      <td>Binding Buffer: Hanks buffer with 0.1 mg/ml salmon sperm DNA and 1 mg/ml BSA.</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>26266.08 g/mole</td>
      <td>50.00%</td>
      <td>827500 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>86</td>
      <td>None</td>
      <td>dAp,dAp,dCp,dCp,dGp,dCp,dCp,dCp,dAp,dAp,dAp,dTp,dCp,dCp,dCp,dTp,dAp,dAp,dGp,dAp,dGp,dTp,dCp,dGp,dGp,dAp,dCp,dTp,dGp,dCp,dAp,dAp,dCp,dCp,dTp,dAp,dTp,dGp,dCp,dTp,dAp,dTp,dCp,dGp,dTp,dTp,dGp,dAp,dTp,dGp,dTp,dCp,dTp,dGp,dTp,dCp,dCp,dAp,dAp,dGp,dCp,dAp,dAp,dCp,dAp,dCp,dAp,dGp,dAp,dCp,dAp,dCp,dAp,dCp,dTp,dAp,dCp,dAp,dCp,dAp,dCp,dGp,dCp,dAp,dCp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>343</th>
      <td>XA1</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7215.jpg" width="120" ></td>
      <td>XLR-11</td>
      <td>Small Organic</td>
      <td>310 nM (reported value)</td>
      <td>20 mM Tris-HCl, \n0.5 mM MgCl2, 20 mM NaCl, pH 7.4</td>
      <td>°C</td>
      <td>\n                                mFold \n</td>
      <td>Also binds to synthetic cannabinoid UR-144 with a KD of 127 nM. No cross-reactivity with THC or other illicit drug substances.</td>
      <td>14365 g/mole</td>
      <td>60.87%</td>
      <td>440100 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>46</td>
      <td>Yu, H., Luo, Y., Alkhamis, O., Canoura, J., Yu, B., & Xiao, Y. (2021). Isolation of Natural DNA Aptamers for Challenging Small-Molecule Targets, Cannabinoids. Analytical chemistry, 93(6), 3172–3180. https://doi.org/10.1021/acs.analchem.0c04592</td>
      <td>dCp,dTp,dTp,dAp,dCp,dGp,dAp,dCp,dTp,dGp,dTp,dGp,dGp,dTp,dCp,dGp,dGp,dGp,dTp,dGp,dGp,dTp,dGp,dGp,dGp,dCp,dCp,dTp,dCp,dTp,dAp,dGp,dAp,dGp,dGp,dGp,dGp,dTp,dGp,dTp,dCp,dGp,dTp,dAp,dAp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>344</th>
      <td>Phenylalanine-Cp*Rh(III) sensor</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7105.jpg" width="120" ></td>
      <td>Phenylalanine</td>
      <td>Small Organic</td>
      <td>3.2 nM (reported value)</td>
      <td>100 μM concentration of monomeric Cp*Rh(III)</td>
      <td>24 °C°C</td>
      <td>\n                                Mix and incubate for 5 min at 95oC, and cool down at room temperature > 30 min \n</td>
      <td>N/A</td>
      <td>15707.21 g/mole</td>
      <td>56.86%</td>
      <td>531650 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>51</td>
      <td>Yang KA, Barbu M, Halim M, et al. Recognition and sensing of low-epitope targets via ternary complexes with oligonucleotides and synthetic receptors. Nat Chem. 2014;6(11):1003-1008. doi:10.1038/nchem.2058</td>
      <td>dCp,dTp,dCp,dTp,dCp,dGp,dGp,dGp,dAp,dCp,dGp,dAp,dCp,dGp,dGp,dAp,dCp,dGp,dCp,dTp,dAp,dAp,dTp,dCp,dTp,dTp,dAp,dCp,dAp,dAp,dGp,dGp,dGp,dCp,dGp,dTp,dAp,dGp,dTp,dGp,dTp,dAp,dTp,dGp,dTp,dCp,dGp,dTp,dCp,dCp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>345</th>
      <td>Anti-IgE Aptamer, D 17.4 ext</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/276.png" width="120" ></td>
      <td>IgE (human)</td>
      <td>Protein</td>
      <td>3.6 nM (reported value)</td>
      <td>PBS containing 1 mM MgCl2 (pH 7.4)</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>13758.86 g/mole</td>
      <td>73.33%</td>
      <td>394100 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Aptamer binding was improved by Liss et al by extending the stem loop. The original aptamer lacked the last four base pairs in the stem loop (Kd = 8.4 nM).</td>
      <td>45</td>
      <td>None</td>
      <td>dGp,dCp,dGp,dCp,dGp,dGp,dGp,dGp,dCp,dAp,dCp,dGp,dTp,dTp,dTp,dAp,dTp,dCp,dCp,dGp,dTp,dCp,dCp,dCp,dTp,dCp,dCp,dTp,dAp,dGp,dTp,dGp,dGp,dCp,dGp,dTp,dGp,dCp,dCp,dCp,dCp,dGp,dCp,dGp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>346</th>
      <td>Acetylcholine Receptor Antibody (mAB198) (SE RNA VII)</td>
      <td>2'-NH2-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/67.jpg" width="120" ></td>
      <td>Acetylcholine Receptor Antibody (mAB198)</td>
      <td>Protein</td>
      <td>60 nM (reported value)</td>
      <td>Binding Buffer (30mM Tris-HCl (pH 7.5), 150 mM NaCl, 10mM MgCl2, 2mM dithiothreitol, 1% BSA)</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>28339.52 g/mole</td>
      <td>68.75%</td>
      <td>455500 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>48</td>
      <td>Lee, Seong-Wook and Sullenger, Bruce. "Isolation of a nuclease-resistant decoy RNA that can protect human acetylcholine receptors from myasthenic antibodies." Nature Biotechnology, 15 (1997): 41-45.</td>
      <td>rGp,rGp,rGp,rGp,rGp,rAp,rGp,rGp,rAp,rGp,rGp,rAp,rGp,rGp,rAp,rAp,rGp,rAp,rGp,rGp,rGp,rGp,rAp,rGp,rGp,rAp,rAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>347</th>
      <td>NAD</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/151.jpg" width="120" ></td>
      <td>NAD</td>
      <td>Small Organic</td>
      <td>2.5 nM (reported value)</td>
      <td>Selection Buffer (20 mM HEPES (pH 7.5), 300 mM KC1, and 5 mM MgC12) at room temperature</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>15748.56 g/mole</td>
      <td>59.18%</td>
      <td>474700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>49</td>
      <td>Lauhon and Szostak. "RNA Aptamers That Bind Flavin and Nicotinamide Redox Cofactors." Journal of the American Chemical Society, 117(1994): 1246-1257.</td>
      <td>rGp,rGp,rAp,rAp,rCp,rCp,rCp,rAp,rAp,rCp,rUp,rAp,rGp,rGp,rCp,rGp,rUp,rUp,rUp,rGp,rAp,rGp,rGp,rGp,rGp,rAp,rUp,rUp,rCp,rGp,rGp,rCp,rCp,rAp,rCp,rGp,rGp,rUp,rAp,rAp,rCp,rAp,rAp,rCp,rCp,rCp,rCp,rUp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>348</th>
      <td>ATP C27</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/573.jpg" width="120" ></td>
      <td>ATP</td>
      <td>Small Organic</td>
      <td>21 µM (reported value)</td>
      <td>(50 mM Trisƒ??HCl, pH 7.5, 150 mM NaCl, 5 mM MgCl2) + 1 mg/ml tRNA and 0.1 mg/ml BSA</td>
      <td>24°C</td>
      <td>\n                                80C for 5 minutes, cool at 24C \n</td>
      <td></td>
      <td>34469.89 g/mole</td>
      <td>51.89%</td>
      <td>1081600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>353</td>
      <td>Weill L., Louis D., and Sargueil B., 2004.Selection and Evolution of NTP-Specific Aptamers. Nucleic Acids Research, 2004 Vol. 32 No. 17, 5045-5058</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>349</th>
      <td>P-Selectin (PF377)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/223.png" width="120" ></td>
      <td>P-Selectin</td>
      <td>Protein</td>
      <td>N/A nM (reported value)</td>
      <td>None</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>18386.21 g/mole</td>
      <td>57.89%</td>
      <td>559700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>57</td>
      <td>None</td>
      <td>rAp,fCp,rGp,fCp,fUp,fCp,rAp,rAp,fCp,rGp,rAp,rGp,fCp,fCp,rAp,rGp,rGp,rAp,rAp,fCp,rAp,fUp,fCp,rGp,rAp,fCp,rGp,fUp,fCp,rAp,rGp,fCp,rAp,rAp,rAp,fCp,rGp,fCp,rGp,rAp,rGp,fCp,rGp,fCp,rAp,rAp,fCp,fCp,rAp,rGp,fUp,rAp,rAp,fCp,rAp,fCp,fCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>350</th>
      <td>Aptamer-2</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7229.jpg" width="120" ></td>
      <td>S/RBD receptor binding</td>
      <td>Protein</td>
      <td>4.5 nM (reported value)</td>
      <td>Dulbecco's phosphate-buffered saline (DPBS) containing 1 mM MgCl2 (DPBS/Mg)</td>
      <td>23°C</td>
      <td>\n                                mFold \n</td>
      <td>Highly Specific</td>
      <td>25234.36 g/mole</td>
      <td>48.15%</td>
      <td>786735 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Binds to the receptor-binding domain (RBD) of COVID-19 virus and inhibits the S/RBD interaction between virus and host cell.</td>
      <td>243</td>
      <td>https://doi.org/10.1002/anie.202100345</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>351</th>
      <td>Nanobioprobe mediated DNA aptamer</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/539.png" width="120" ></td>
      <td>Trinitrotoluene (TNT)</td>
      <td>Other</td>
      <td>100 nM (reported value)</td>
      <td>10 mM Tris/HCl buffer, pH 7.6, 150 mM NaCl, 5 mM MgCl2</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>27415.9 g/mole</td>
      <td>46.59%</td>
      <td>890400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>The 5' end of the designed primer was labeled with FITC and thiol (-SHC6) to monitorthe progression of selection rounds and the binding of aptamers on a gold substrate respectively.  The 3' end of the designed primer was labeled with biotin.</td>
      <td>264</td>
      <td>Priyanka, M. S., et al.  "Nanobioprobe mediated DNA aptamers for explosive detection."  Chem. Commun., 2014, 50, 1080 - 1082.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>352</th>
      <td>Feline Immunodeficiency Virus (F1a)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/172.png" width="120" ></td>
      <td>Feline Immunodeficiency Virus (FIV)</td>
      <td>Protein</td>
      <td>1.9 nM (reported value)</td>
      <td>Binding Buffer (50 mM Tris-HCl (pH 7.7), 200 mM potassium acetate, and 10 mM dithiothreitol)</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>29248.51 g/mole</td>
      <td>51.65%</td>
      <td>892700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>42</td>
      <td>Chen, et al. "Inhibitory RNA Ligand to Reverse Transcriptase from Feline Immunodeficiency  Virus." Journal of Biochemistry, 35(1996): 6923-6930.</td>
      <td>rGp,rGp,rGp,rAp,rGp,rGp,rAp,rUp,rAp,rUp,rUp,rUp,rUp,rCp,rUp,rCp,rAp,rGp,rAp,rCp,rCp,rGp,rUp,rAp,rAp,rGp,rUp,rAp,rCp,rCp,rGp,rAp,rAp,rUp,rGp,rUp,rGp,rCp,rUp,rUp,rUp,rUp,rGp,rGp,rCp,rCp,rGp,rAp,rUp,rUp,rUp,rUp,rUp,rGp,rGp,rCp,rCp,rCp,rCp,rUp,rGp,rCp,rAp,rGp,rUp,rUp,rGp,rCp,rAp,rGp,rCp,rAp,rUp,rCp,rGp,rUp,rGp,rAp,rAp,rCp,rUp,rAp,rGp,rGp,rAp,rUp,rCp,rCp,rGp,rGp,rGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>353</th>
      <td>ex-50.T</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7200.jpg" width="120" ></td>
      <td>Breast cancer exosomes</td>
      <td>Other</td>
      <td>0.8 nM (reported value)</td>
      <td>For Binding Assays on Exosomes: \n200 nM of aptamer in 1X PBS\nPBS buffer used to wash\n\nFor Binding Assays on Cells:\n200 nM of aptamer in serum-free medium\nSerum-free medium used to wash</td>
      <td>37°C</td>
      <td>\n                                N/A \n</td>
      <td>Specific for breast cancer exosomes. Able to discriminate from two other cancer exosome types. Does not react with HSA</td>
      <td>10.652 g/mole</td>
      <td>42.42%</td>
      <td>333200 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>33</td>
      <td>None</td>
      <td>fUp,rGp,fUp,rGp,rGp,fCp,rAp,rGp,fUp,fUp,rAp,rAp,rGp,rAp,rAp,fUp,rAp,rGp,rAp,fUp,fCp,fUp,fUp,fCp,rGp,fCp,fUp,rGp,fCp,rGp,rAp,fUp,fUp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>354</th>
      <td>L-arginine (12-28 hairpin)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/483.png" width="120" ></td>
      <td>L-arginine</td>
      <td>Small Organic</td>
      <td>~2.5 nM (reported value)</td>
      <td>Buffer for CD binding assay: 10 mM sodium phosphate buffer (pH 7.5) and 100 mM NaCl.</td>
      <td>4°C</td>
      <td>\n                                Heated to 95°C and cooled to 4°C for over 2h. \n</td>
      <td>The aptamer's affinity for argininamide and agmatine is 20-fold more than for arginine. Affinities for D-arginine, guanidine, and ethylguanidine are about the same.</td>
      <td>8574.6 g/mole</td>
      <td>60.71%</td>
      <td>264800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>30</td>
      <td>None</td>
      <td>dGp,dGp,dGp,dAp,dTp,dCp,dGp,dAp,dAp,dAp,dCp,dGp,dTp,dAp,dGp,dCp,dGp,dCp,dCp,dTp,dTp,dCp,dGp,dAp,dTp,dCp,dCp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>355</th>
      <td>Human Neutrophil Elastase (DNA I)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/175.png" width="120" ></td>
      <td>Human Neutrophil Elastase (HNE)</td>
      <td>Protein</td>
      <td>17 nM (reported value)</td>
      <td>None</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>14046.12 g/mole</td>
      <td>64.44%</td>
      <td>428800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>45</td>
      <td>None</td>
      <td>dTp,dAp,dGp,dCp,dGp,dAp,dTp,dAp,dCp,dTp,dGp,dCp,dGp,dTp,dGp,dGp,dGp,dTp,dTp,dGp,dGp,dGp,dGp,dCp,dGp,dGp,dGp,dTp,dAp,dGp,dGp,dGp,dCp,dCp,dAp,dGp,dCp,dAp,dGp,dTp,dCp,dTp,dCp,dGp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>356</th>
      <td>14-16</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/501.png" width="120" ></td>
      <td>Mouse P68 RNA Helicase (Ddx5)</td>
      <td>Protein</td>
      <td>30.8 nM (reported value)</td>
      <td>For selection: 5 nmol of aptamer pool in 200 µl of PBS, injected systemically and allowed to circulate for 20 minutes  For binding assay: 10 mM Tris-HCl (pH 7.0), 150 mM NaCl, 10% (v/v) glycerol, 2.5 µg/ml tRNA. 10 minute incubation at 37°C  For affinity purification: 20 mM HEPES (pH 7.4), 150 mM NaCl, 2 mM CaCl2, 0.1% BSA, 0.1 µM tRNA at 4°C for 2 hours</td>
      <td>37°C</td>
      <td>\n                                N/A \n</td>
      <td>See additional comments. Initially demonstrated twofold increased affinity (30.8 nM) for aggregate CT26 tumor proteins over normal colon tissue (68.9 nM).</td>
      <td>25949.66 g/mole</td>
      <td>65.00%</td>
      <td>769500 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Has better affinity for HA-tagged version of human p68 protein (13.8 nM), but only has an affinity of 190 nM for HA-tagged Tenascin CD. ATPase activity was significantly decreased in the presence of aptamer 14-16. IC50 of aptamer was 0.78 µM compared to that of the library (3.63 µM).  An additional ƒ??Uƒ? nucleotide is present at position 55 in supplementary Figure 1-b and in the later-published version of the paper available at http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2795795/. This sequence was used to predict the secondary structure.</td>
      <td>88</td>
      <td>Mi et al. ƒ??In vivo selection of tumor-targeting RNA motifsƒ?. Nature Chemical Biology 1(2010): 22-24. DOI: 10.1038/NCHEMBIO.277</td>
      <td>rGp,rGp,rGp,rAp,rGp,rGp,rAp,fCp,rGp,rAp,fUp,rGp,fCp,rGp,rGp,fCp,rAp,rGp,fUp,rGp,fCp,fCp,fCp,rAp,rAp,fCp,fCp,rGp,rGp,rAp,rAp,fCp,rAp,rAp,fCp,rAp,rAp,fCp,fCp,rAp,fCp,fCp,rGp,rGp,fCp,rGp,rGp,fCp,fUp,fCp,fCp,fUp,rGp,fCp,fUp,fCp,rAp,rGp,rAp,fCp,rGp,rAp,fCp,fUp,fCp,rGp,fCp,fUp,rGp,rAp,rGp,rGp,rAp,fUp,fCp,fCp,rGp,rAp,rGp,rAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>357</th>
      <td>D-Tryptophan (MF-10)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/107.png" width="120" ></td>
      <td>D-Tryptophan</td>
      <td>Small Organic</td>
      <td>18 nM (reported value)</td>
      <td>Column Buffer (4 mM Mg2+, 250 mM NaCl, 50 mM Tris C1 (pH 7.6))  Binding Buffer is  (250 mM  NaCl, 4 mM MgCl2, 50 mM Tris (pH 7.6))   Elution Buffer (20 mM EDTA (pH 8.0))</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>49445.7 g/mole</td>
      <td>53.59%</td>
      <td>1515800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>153</td>
      <td>Famulok and Szostak. "Stereospecific Recognition of Tryptophan Agarose by in Vitro Selected RNA." Journal of American Chemical Society, 114 (1992): 3990</td>
      <td>rAp,rGp,rUp,rAp,rAp,rUp,rAp,rCp,rGp,rAp,rCp,rUp,rCp,rAp,rCp,rUp,rAp,rUp,rAp,rGp,rGp,rGp,rAp,rGp,rAp,rAp,rUp,rUp,rCp,rCp,rGp,rAp,rCp,rCp,rAp,rGp,rAp,rAp,rGp,rUp,rUp,rGp,rGp,rCp,rGp,rUp,rUp,rGp,rGp,rCp,rAp,rUp,rGp,rAp,rCp,rGp,rCp,rGp,rGp,rGp,rGp,rAp,rAp,rUp,rCp,rGp,rGp,rGp,rUp,rGp,rCp,rAp,rUp,rCp,rGp,rAp,rUp,rGp,rAp,rCp,rUp,rAp,rCp,rUp,rCp,rCp,rUp,rGp,rGp,rGp,rCp,rCp,rCp,rAp,rCp,rGp,rUp,rCp,rUp,rGp,rUp,rUp,rGp,rUp,rUp,rGp,rAp,rCp,rGp,rUp,rCp,rAp,rCp,rAp,rGp,rCp,rUp,rUp,rGp,rAp,rUp,rUp,rUp,rAp,rGp,rGp,rAp,rUp,rAp,rGp,rCp,rGp,rCp,rUp,rUp,rGp,rGp,rGp,rCp,rAp,rGp,rUp,rCp,rGp,rUp,rGp,rCp,rAp,rGp,rUp,rGp,rGp,rAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>358</th>
      <td>GNP DV2</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7129.jpg" width="120" ></td>
      <td>Dichlorvos (DV)</td>
      <td>Small Organic</td>
      <td>0.85 nM (reported value)</td>
      <td>10 mM Tris, 10 mM MgCl2, 50 mM KCl, 25 mM NaCl, pH 7.5</td>
      <td>25°C</td>
      <td>None</td>
      <td></td>
      <td>20258.2 g/mole</td>
      <td>55.38%</td>
      <td>655400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>65</td>
      <td>Chatterjee, B., Kalyani, N., Anand, A., Khan, E., Das, S., Bansal, V., … Sharma, T. K. (2020). GOLD SELEX: a novel SELEX approach for the development of high-affinity aptamers against small molecules without residual activity. Microchimica Acta, 187(11). doi:10.1007/s00604-020-04577-0</td>
      <td>dGp,dGp,dAp,dAp,dGp,dAp,dAp,dCp,dAp,dTp,dGp,dTp,dGp,dAp,dGp,dCp,dAp,dAp,dAp,dAp,dGp,dGp,dCp,dCp,dAp,dGp,dCp,dAp,dAp,dAp,dAp,dGp,dGp,dCp,dCp,dAp,dGp,dGp,dAp,dAp,dCp,dCp,dGp,dTp,dAp,dAp,dAp,dAp,dAp,dGp,dGp,dCp,dCp,dGp,dCp,dGp,dTp,dTp,dGp,dCp,dTp,dGp,dGp,dCp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>359</th>
      <td>Glycated Hemoglobin Aptamer</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/617.jpg" width="120" ></td>
      <td>Glycated Hemoglobin</td>
      <td>Protein</td>
      <td>7.3 nM (reported value)</td>
      <td>Aptamer binding took place in human blood sample produced by centrifuging 100 µL of whole human blood, collecting the pellet of red blood cells, resuspending the pellet in 50 µL of deionized water, and straining the resulting suspension.</td>
      <td>°C</td>
      <td>None</td>
      <td></td>
      <td>22588.62 g/mole</td>
      <td>66.67%</td>
      <td>694400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>72</td>
      <td>J Li et al. On chip, aptamer-based sandwich assay for detection of glycated hemoglobins via magnetic beads. Biosensors and Bioelectronics 79 (2016): 887-893</td>
      <td>dTp,dGp,dGp,dCp,dAp,dGp,dGp,dAp,dAp,dGp,dAp,dCp,dAp,dAp,dAp,dCp,dAp,dCp,dAp,dTp,dCp,dGp,dTp,dCp,dGp,dCp,dGp,dGp,dCp,dCp,dTp,dTp,dAp,dGp,dGp,dAp,dGp,dGp,dGp,dGp,dCp,dGp,dGp,dAp,dCp,dGp,dGp,dGp,dGp,dGp,dGp,dGp,dGp,dGp,dCp,dGp,dTp,dGp,dGp,dTp,dCp,dTp,dGp,dTp,dGp,dGp,dTp,dGp,dCp,dTp,dGp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>360</th>
      <td>Beta 2-Integrin (D20)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/139.png" width="120" ></td>
      <td>Beta 2-Integrin</td>
      <td>Protein</td>
      <td>500 nM (reported value)</td>
      <td>Binding Buffer (4.3 mM K2HPO4, 1.4 mM NaH2PO4, 150 mM NaCl, 1.0 mM MgCl2, 0.1 uM CaCl2)</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>23599.21 g/mole</td>
      <td>64.38%</td>
      <td>700300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>73</td>
      <td>Blind, M., et al. "Cytoplasmic RNA modulators of an inside-out signaltransduction cascade." Proceedings of National Academy of Sciences of the United States of America, 96(1999):3606ƒ??3610.</td>
      <td>rGp,rGp,rGp,rCp,rGp,rCp,rUp,rAp,rAp,rGp,rUp,rCp,rCp,rUp,rCp,rGp,rCp,rUp,rCp,rAp,rUp,rGp,rCp,rGp,rCp,rGp,rUp,rCp,rCp,rCp,rAp,rUp,rGp,rGp,rGp,rGp,rUp,rAp,rUp,rAp,rGp,rAp,rGp,rGp,rGp,rGp,rUp,rCp,rGp,rAp,rAp,rGp,rUp,rGp,rGp,rAp,rCp,rGp,rCp,rGp,rCp,rGp,rAp,rCp,rUp,rCp,rGp,rGp,rAp,rUp,rCp,rCp,rUp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>361</th>
      <td>BC1</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7156.jpg" width="120" ></td>
      <td>Osteocalcin (OC)</td>
      <td>Protein</td>
      <td>69.34 nM (reported value)</td>
      <td>2 mM MgCl2, 50 mM Tris (pH 7.5) and 150 mM NaCl in distilled water</td>
      <td>22°C</td>
      <td>None</td>
      <td></td>
      <td>18336.9 g/mole</td>
      <td>49.15%</td>
      <td>579500 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>59</td>
      <td>Chinnappan R, Zaghloul NS, AlZabn R, Malkawi A, Abdel Rahman A, Abu-Salah KM, Zourob M. Aptamer selection and aptasensor construction for bone density biomarkers. Talanta. 2021 Mar 1;224:121818. doi: 10.1016/j.talanta.2020.121818. Epub 2020 Oct 29. PMID: 33379043.</td>
      <td>dAp,dTp,dGp,dAp,dCp,dGp,dGp,dGp,dGp,dGp,dTp,dCp,dTp,dAp,dGp,dGp,dCp,dAp,dAp,dGp,dTp,dAp,dAp,dTp,dAp,dAp,dCp,dGp,dGp,dGp,dGp,dGp,dCp,dAp,dAp,dGp,dCp,dTp,dTp,dTp,dTp,dCp,dTp,dAp,dTp,dCp,dTp,dCp,dGp,dTp,dTp,dCp,dTp,dAp,dGp,dGp,dGp,dTp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>362</th>
      <td>Phorate (SS4-54)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/296.png" width="120" ></td>
      <td>Phorate</td>
      <td>Small Organic</td>
      <td>1.43 µM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td>This aptamer binds preferentially to the organophosphates phorate, profenofos, benzoate and omethoate. It has reduced binding to imidacloprid, acetamiprid, chlorpyrifos, methamidophos, dichlorvos, parathion, dimethoate, monocrotophos and phoxim.</td>
      <td>16675.9 g/mole</td>
      <td>46.30%</td>
      <td>517600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>"A molecular beacon (5' FAM-CTGCACAAGAATCGCTGCAG-3') was designed, which was partly complementary with the middle constant nucleotides-sequence of the selected ssDNAs."</td>
      <td>54</td>
      <td>None</td>
      <td>dAp,dAp,dGp,dCp,dTp,dTp,dTp,dTp,dTp,dTp,dGp,dAp,dCp,dTp,dGp,dAp,dCp,dTp,dGp,dCp,dAp,dGp,dCp,dGp,dAp,dTp,dTp,dCp,dTp,dTp,dGp,dAp,dTp,dCp,dGp,dCp,dCp,dAp,dCp,dGp,dGp,dTp,dCp,dTp,dGp,dGp,dAp,dAp,dAp,dAp,dAp,dGp,dAp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>363</th>
      <td>M03</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7184.jpeg" width="120" ></td>
      <td>Myoglobin (Mb)</td>
      <td>Protein</td>
      <td>46.3 nM (reported value)</td>
      <td>Ammonium bicarbonate buffer (50 mM, pH 8.5) containing 500 mM NaCl.</td>
      <td>RT°C</td>
      <td>None</td>
      <td></td>
      <td>24588.9 g/mole</td>
      <td>55.00%</td>
      <td>744700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>80</td>
      <td>Ma, Y.; Li, W.; Xing, R.; Li, P.; Liu, Z. Epitope-Imprinted Magnetic Nanoparticles as a General Platform for Efficient In Vitro Evolution of Protein-Binding Aptamers. ACS Sens. 2020 Jul 7; 5(8): 2537-2544. DOI: 10.1012/acssensors.0c00846</td>
      <td>dCp,dTp,dTp,dCp,dTp,dGp,dCp,dCp,dCp,dGp,dCp,dCp,dTp,dCp,dCp,dTp,dTp,dCp,dCp,dCp,dGp,dGp,dGp,dTp,dTp,dAp,dCp,dAp,dGp,dGp,dAp,dTp,dCp,dGp,dTp,dAp,dAp,dAp,dTp,dCp,dTp,dTp,dAp,dTp,dTp,dCp,dGp,dTp,dTp,dGp,dCp,dTp,dCp,dGp,dAp,dTp,dCp,dGp,dGp,dGp,dAp,dGp,dAp,dCp,dGp,dAp,dGp,dAp,dTp,dAp,dGp,dGp,dCp,dGp,dGp,dAp,dCp,dAp,dCp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>364</th>
      <td>Alpha V Beta 5 Integrin</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/242.png" width="120" ></td>
      <td>Alpha V Beta 5 Integrin</td>
      <td>Cells</td>
      <td>N/A nM (reported value)</td>
      <td>Binding Buffer (50uL PBS/1.5% BSA)</td>
      <td>RT°C</td>
      <td>None</td>
      <td></td>
      <td>1092.33 g/mole</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>9</td>
      <td>Cardo-Vila et al. "Alpha v beta 5 integrin-dependent programmed cell death triggered by a peptide mimic of annexin V." Molecular Cell, 11(2003): 1151-1162.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>365</th>
      <td>Rev Peptide</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/258.png" width="120" ></td>
      <td>Rev Peptide</td>
      <td>Peptide</td>
      <td>18 µM (reported value)</td>
      <td>Buffer (50 mM Tris, pH 7.5, 100 mM KCl, 10 mM MgCl2), r. t.</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>25129.29 g/mole</td>
      <td>59.74%</td>
      <td>774600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Contains an L1 ligase ribozyme motif.</td>
      <td>77</td>
      <td>Hesselberth et al. "Simultaneous detection of diverse analytes with  an aptazyme ligase array." Analytical Biochemistry, 312(2003): 106-112.</td>
      <td>rGp,rGp,rAp,rCp,rCp,rUp,rCp,rGp,rGp,rCp,rGp,rAp,rAp,rAp,rGp,rCp,rCp,rGp,rGp,rUp,rAp,rAp,rCp,rGp,rCp,rCp,rAp,rCp,rAp,rAp,rGp,rUp,rCp,rGp,rGp,rAp,rGp,rGp,rGp,rUp,rAp,rAp,rGp,rAp,rUp,rCp,rUp,rGp,rAp,rCp,rAp,rGp,rGp,rAp,rAp,rCp,rUp,rGp,rGp,rGp,rUp,rGp,rAp,rCp,rGp,rGp,rAp,rGp,rGp,rUp,rUp,rAp,rGp,rGp,rUp,rGp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>366</th>
      <td>HA12-16</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/549.jpg" width="120" ></td>
      <td>gHA1 protein</td>
      <td>Protein</td>
      <td>- nM (reported value)</td>
      <td>20 mM sodium phosphate, pH 7.5, 500 mM NaCl, and 20 mM imidazole</td>
      <td>24°C</td>
      <td>\n                                denatured at 85 C for 5 min and then incubated at room temperature \n</td>
      <td>N/A</td>
      <td>29999.09 g/mole</td>
      <td>50.54%</td>
      <td>926600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Affinity measured using radiological absorbency: see Figure 5 of Source Material. HA12-16 displays a high degree of absorbency at 450 nm.</td>
      <td>279</td>
      <td>Kwon H-M, Lee KH, Han BW, Han MR, Kim DH, et al. (2014) An RNA Aptamer That Specifically Binds to the Glycosylated Hemagglutinin of Avian Influenza Virus and Suppresses Viral Infection in Cells. PLoS ONE 9(5): e97574. doi:10.1371/journal.pone.0097574</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>367</th>
      <td>Anti Glioblastoma Cells (GBM128)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/509.png" width="120" ></td>
      <td>Glioblastoma Cell</td>
      <td>Cells</td>
      <td>20 nM (reported value)</td>
      <td>25 µM glucose    4 µM yeast tRNA     15 µM BSA  5 mM MgCl2 in PBS (pH = 7.4)    1 hour incubation with cells</td>
      <td>4°C</td>
      <td>\n                                heated at 95C for 5 min and cooled on ice for 10 min  immediately \n</td>
      <td>human astroglial cell line SVGp12 was used as a  counter target</td>
      <td>25689.73 g/mole</td>
      <td>49.40%</td>
      <td>812600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>GMB128 only showed binding with cells at 4C while GMB131 showed binding at 4C as well as physiological temperature despite less affinity than GMB128</td>
      <td>0</td>
      <td>Kang et al. Selection of DNA Aptamers against Glioblastoma Cells  with High Affinity and Specificity (PLOSone)</td>
      <td>dGp,dAp,dAp,dTp,dTp,dCp,dAp,dGp,dTp,dCp,dGp,dGp,dAp,dCp,dAp,dGp,dCp,dGp,dAp,dCp,dGp,dGp,dTp,dGp,dGp,dGp,dAp,dGp,dCp,dCp,dCp,dCp,dAp,dAp,dAp,dTp,dAp,dAp,dTp,dTp,dCp,dTp,dTp,dGp,dCp,dGp,dAp,dTp,dTp,dAp,dTp,dTp,dAp,dGp,dTp,dGp,dTp,dAp,dAp,dGp,dCp,dGp,dGp,dAp,dTp,dGp,dGp,dAp,dCp,dGp,dAp,dAp,dTp,dAp,dTp,dCp,dGp,dTp,dCp,dTp,dCp,dCp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>368</th>
      <td>Streptavidin (31)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/548" width="120" ></td>
      <td>Streptavidin</td>
      <td>Protein</td>
      <td>56.7 nM (reported value)</td>
      <td>None</td>
      <td>21°C</td>
      <td>None</td>
      <td></td>
      <td>29413.16 g/mole</td>
      <td>42.71%</td>
      <td>903200 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>In binding buffer, heat aptamer to 90°C for 10 minutes, rapidly cool to 4°C for 15 minutes, and allow the solution to incubate at room temperature for 5-8 minutes.   This sequence contains the primer regions. The core sequence of the aptamer is: 5'-CTATACTCCACTTTGCTATTTCTCGGTTCCTTCACGCGCCGATCGCAGGCTGATGAATTG-3'.</td>
      <td>96</td>
      <td>None</td>
      <td>dAp,dTp,dAp,dCp,dCp,dAp,dGp,dCp,dTp,dTp,dAp,dTp,dTp,dCp,dAp,dAp,dTp,dTp,dCp,dTp,dAp,dTp,dAp,dCp,dTp,dCp,dCp,dAp,dCp,dTp,dTp,dTp,dGp,dCp,dTp,dAp,dTp,dTp,dTp,dCp,dTp,dCp,dGp,dGp,dTp,dTp,dCp,dCp,dTp,dTp,dCp,dAp,dCp,dGp,dCp,dGp,dCp,dCp,dGp,dAp,dTp,dCp,dGp,dCp,dAp,dGp,dGp,dCp,dTp,dGp,dAp,dTp,dGp,dAp,dAp,dTp,dTp,dGp,dAp,dGp,dAp,dTp,dAp,dGp,dTp,dAp,dAp,dGp,dTp,dGp,dCp,dAp,dAp,dTp,dCp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>369</th>
      <td>H1</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7143.jpg" width="120" ></td>
      <td>Poly(histidine) tag (His-tag)</td>
      <td>Protein</td>
      <td>86 nM (reported value)</td>
      <td>20 mM Tris-HCl, 100 mM NaCl, 5 mM MgCl2, and 0.1% bovine\nserum albumin with 10µg/mL yeast tRNA and 10µg/mL salmon sperm DNA</td>
      <td>25°C</td>
      <td>None</td>
      <td></td>
      <td>24224.8 g/mole</td>
      <td>37.97%</td>
      <td>777800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>79</td>
      <td>Wang Y, Li Z, Yu H. Aptamer-Based Western Blot for Selective Protein Recognition. Front Chem. 2020 Oct 29;8:570528. doi: 10.3389/fchem.2020.570528. PMID: 33195056; PMCID: PMC7658645.</td>
      <td>dAp,dTp,dAp,dCp,dGp,dAp,dCp,dTp,dCp,dAp,dCp,dTp,dAp,dTp,dTp,dAp,dGp,dGp,dGp,dAp,dTp,dGp,dTp,dCp,dCp,dAp,dCp,dCp,dAp,dTp,dAp,dTp,dAp,dGp,dAp,dTp,dCp,dGp,dAp,dTp,dTp,dTp,dAp,dAp,dGp,dTp,dCp,dCp,dCp,dTp,dCp,dGp,dTp,dTp,dAp,dTp,dTp,dAp,dAp,dTp,dGp,dCp,dTp,dAp,dGp,dAp,dCp,dTp,dAp,dCp,dTp,dGp,dAp,dCp,dTp,dAp,dCp,dAp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>370</th>
      <td>Ricin A-chain (RTA) Aptamer (RA80.1.d2 - 31RA)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/601.png" width="120" ></td>
      <td>Ricin A-chain (RTA)</td>
      <td>Protein</td>
      <td>7.4 nM (reported value)</td>
      <td>1X PBS, 5 mM MgCl2</td>
      <td>23°C</td>
      <td>None</td>
      <td></td>
      <td>8089.29 g/mole</td>
      <td>69.23%</td>
      <td>261800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>26</td>
      <td>(1) Bogomolova, A., Aldissi, M. ƒ??Real-time and label-free analyte detection in a flow-through mode using immobilized fluorescent aptamer/quantum dots molecular switches.ƒ? Biosensors and Bioelecgtronics, 66 (2015): 290-296.      (2) Hesselberth, J., et al. ƒ??In vitro Selection of RNA Molecules That Inhibit the Activity of Ricin A-chain.ƒ? Journal of Biological Chemistry, 275, 7 (2000): 4937-4942.</td>
      <td>dGp,dGp,dCp,dGp,dAp,dAp,dCp,dAp,dGp,dGp,dGp,dGp,dAp,dCp,dGp,dAp,dGp,dCp,dAp,dAp,dGp,dAp,dCp,dGp,dCp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>371</th>
      <td>Microcystin (RC6)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/462.png" width="120" ></td>
      <td>Microcystin</td>
      <td>Small Organic</td>
      <td>28-61 nM (reported value)</td>
      <td>None</td>
      <td>25°C</td>
      <td>None</td>
      <td>Kd values for Microcystin-LR, -LA, and -YR were 61, 43, and 28 nM respectively</td>
      <td>18280.89 g/mole</td>
      <td>53.33%</td>
      <td>573300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>60</td>
      <td>None</td>
      <td>dCp,dAp,dCp,dGp,dCp,dAp,dAp,dCp,dAp,dAp,dCp,dAp,dCp,dAp,dAp,dCp,dAp,dTp,dGp,dCp,dCp,dCp,dAp,dGp,dCp,dGp,dCp,dCp,dTp,dGp,dGp,dAp,dAp,dCp,dAp,dTp,dAp,dTp,dCp,dCp,dTp,dAp,dTp,dGp,dAp,dGp,dTp,dTp,dAp,dGp,dTp,dCp,dCp,dGp,dCp,dCp,dCp,dAp,dCp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>372</th>
      <td>PrP Fibrils (SAF-93)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/72.jpg" width="120" ></td>
      <td>PrP Fibrils</td>
      <td>Protein</td>
      <td>23.4 nM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td></td>
      <td>37278.21 g/mole</td>
      <td>49.14%</td>
      <td>1146800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>The oligonucleotides used in this study were documented in this reference: Tahiri-Alaoui, A., et al. "High affinity nucleic acid aptamers for streptavidin incorporated into bi-specific capture ligands." Nucleic Acids Research, 30 (2002): e45</td>
      <td>116</td>
      <td>Rhie, A., et al. "Characterization of 2'-fluoro-RNA aptamers that bind preferentially to disease-associated conformations of prion protein and inhibit conversion." Journal of Biological Chemistry, 278 (2003): 39697-705.</td>
      <td>rGp,rGp,rGp,rAp,rGp,rAp,fCp,rAp,rAp,rGp,rAp,fCp,fUp,rAp,rGp,rAp,fCp,rGp,fCp,fUp,fCp,rAp,rAp,fCp,fUp,rAp,fCp,rGp,rAp,rAp,fCp,fUp,fCp,rAp,fUp,rGp,rAp,fCp,rAp,fCp,rAp,rAp,rGp,rGp,rAp,fUp,rGp,fCp,rAp,rAp,fUp,fCp,fUp,fCp,rAp,fUp,fCp,fCp,fCp,rGp,fCp,fCp,rAp,rGp,fCp,fCp,fCp,rAp,fCp,fCp,rGp,fUp,fUp,fUp,fCp,rGp,rAp,fCp,rAp,fUp,rGp,rAp,rGp,rAp,fCp,fUp,fCp,rAp,fCp,rAp,rAp,fCp,rAp,rGp,fUp,fUp,fCp,fCp,fCp,fUp,fUp,fUp,rAp,rGp,fUp,rGp,rAp,rGp,rGp,rGp,fUp,fUp,rAp,rAp,fUp,fUp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>373</th>
      <td>HAE N55</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/588.jpg" width="120" ></td>
      <td>Inflamed Human Aortic Endothelial Cells</td>
      <td>Cells</td>
      <td>82 nM (reported value)</td>
      <td>Wash buffer: 5 mmol/L MgCl2, 4.5 g/L glucose, in  Dulbecco PBS. Binding buffer: 0.1 % bovine serum albumin (BSA)   0.1 g/L yeast tRNA + Wash Buffer</td>
      <td>24°C</td>
      <td>None</td>
      <td></td>
      <td>26239.18 g/mole</td>
      <td>34.88%</td>
      <td>837100 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>258</td>
      <td>Ji, Kaili., Lim, Wee Siang., Li, Sam Fong Yau., Bhakoo, Kishore. (2013) A two-step stimulus-response cell-SELEX method to generate a DNA aptamer to recognize inflamed human aortic endothelial cells as a potential in vivo molecular probe for atherosclerosis plaque detection. Anal Bioanal Chem (2013) 405:6853-6861. DOI 10.1007//s00216-013-7155-z</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>374</th>
      <td>Ky2 Aptamer</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/553.png" width="120" ></td>
      <td>Kanamycin</td>
      <td>Other</td>
      <td>78.8 nM (reported value)</td>
      <td>20mM Tris-HCl, 50mM NaCl, 5mM KCl, 5mM MgCl2, pH 8.0</td>
      <td>37°C</td>
      <td>\n                                500pmole of ssDNA library dissolved in 100mL binding buffer and preincubated at 85C for 3 min, then 4C for 1 hour. \n</td>
      <td></td>
      <td>6567.32 g/mole</td>
      <td>61.90%</td>
      <td>207500 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>69</td>
      <td>Kyung-Mi Song, Minseon Cho, Hunho Jo, Kyoungin Min, Sung Ho Jeon, Taisun Kim, Min Su Han, Ja Kang Ku, Changill Ban, "Gold nanoparticle-based colorimetric detection of kanamycin using a DNA aptamer". Anal. Biochem. 415 (2011) 175-181.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>375</th>
      <td>Methylenedianiline (M1)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/81.png" width="120" ></td>
      <td>Methylenedianiline</td>
      <td>Small Organic</td>
      <td>450 nM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td></td>
      <td>28102.08 g/mole</td>
      <td>57.47%</td>
      <td>852400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>87</td>
      <td>None</td>
      <td>rGp,rGp,rGp,rAp,rGp,rAp,rCp,rAp,rAp,rGp,rAp,rAp,rUp,rAp,rAp,rAp,rCp,rGp,rCp,rUp,rCp,rAp,rAp,rCp,rUp,rGp,rCp,rGp,rAp,rUp,rCp,rAp,rGp,rGp,rGp,rGp,rUp,rAp,rAp,rAp,rUp,rUp,rUp,rCp,rCp,rGp,rCp,rGp,rCp,rAp,rGp,rGp,rCp,rUp,rCp,rCp,rAp,rCp,rGp,rCp,rCp,rGp,rCp,rUp,rUp,rCp,rGp,rAp,rCp,rAp,rGp,rGp,rAp,rGp,rGp,rCp,rUp,rCp,rAp,rCp,rAp,rAp,rCp,rAp,rGp,rGp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>376</th>
      <td>Lup an 1 allergen, ??-Conglutin (#40)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/419.png" width="120" ></td>
      <td>??-Conglutin</td>
      <td>Protein</td>
      <td>360 nM (reported value)</td>
      <td>None</td>
      <td>21°C</td>
      <td>None</td>
      <td>Aptamer did not bind to gliadin or ?-conglutin negative controls.</td>
      <td>28903.79 g/mole</td>
      <td>53.76%</td>
      <td>915500 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>104</td>
      <td>None</td>
      <td>dAp,dGp,dCp,dTp,dGp,dAp,dCp,dAp,dCp,dAp,dGp,dCp,dAp,dGp,dGp,dTp,dTp,dGp,dGp,dTp,dGp,dGp,dGp,dGp,dGp,dTp,dGp,dGp,dCp,dTp,dTp,dCp,dCp,dAp,dGp,dTp,dTp,dGp,dGp,dGp,dTp,dTp,dGp,dAp,dCp,dAp,dAp,dTp,dAp,dCp,dGp,dTp,dAp,dGp,dGp,dGp,dAp,dCp,dAp,dCp,dGp,dAp,dAp,dGp,dTp,dCp,dCp,dAp,dAp,dCp,dCp,dAp,dCp,dGp,dAp,dGp,dTp,dCp,dGp,dAp,dGp,dCp,dAp,dAp,dTp,dCp,dTp,dCp,dGp,dAp,dAp,dAp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>377</th>
      <td>Malachite Green (MG-4)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/65.jpg" width="120" ></td>
      <td>Malchite Green</td>
      <td>Small Organic</td>
      <td>1 µM (reported value)</td>
      <td>None</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>12300.52 g/mole</td>
      <td>60.53%</td>
      <td>377500 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>38</td>
      <td>None</td>
      <td>rGp,rGp,rAp,rUp,rCp,rCp,rCp,rGp,rAp,rCp,rUp,rGp,rGp,rCp,rGp,rAp,rGp,rAp,rGp,rCp,rCp,rAp,rGp,rGp,rUp,rAp,rAp,rCp,rGp,rAp,rAp,rUp,rGp,rGp,rAp,rUp,rCp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>378</th>
      <td>ApoE 06</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/611.jpg" width="120" ></td>
      <td>Apolipoprotein E</td>
      <td>Protein</td>
      <td>938 pM (reported value)</td>
      <td>PBSMCT (DPBS with 2.5 mM MgCl2, 1 mM CaCl2, 0.05% TWEEN-20)</td>
      <td>23°C</td>
      <td>None</td>
      <td></td>
      <td>18119.84 g/mole</td>
      <td>44.07%</td>
      <td>563700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>59</td>
      <td>Wang, J., Gong, Q., Maheshwari, N., Eisenstein, M., Luz Arcila, M., Kosik, K., & Soh, T. Particle Display: A Quantitative Screening Method for Generating High-Affinity Aptamers. Angew. Chem. Int. Ed. 2014, 53, 4796ƒ??4801</td>
      <td>dGp,dAp,dTp,dAp,dAp,dAp,dCp,dGp,dCp,dCp,dTp,dTp,dGp,dAp,dTp,dTp,dAp,dAp,dAp,dGp,dGp,dCp,dCp,dCp,dAp,dGp,dTp,dTp,dCp,dTp,dTp,dTp,dAp,dGp,dGp,dCp,dCp,dTp,dAp,dCp,dAp,dCp,dGp,dTp,dGp,dCp,dTp,dGp,dCp,dGp,dAp,dCp,dAp,dTp,dTp,dAp,dAp,dTp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>379</th>
      <td>S3-2-3</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7235.png" width="120" ></td>
      <td>Esophageal cancer cells.</td>
      <td>Cells</td>
      <td>265 nM (reported value)</td>
      <td>4.5 g L−1 glucose, 5 mM MgCl2, 1 mg mL−1 bovine serum albumin (BSA) and 0.1 mg mL−1 yeast tRNA in 0.01 M PBS</td>
      <td>4°C</td>
      <td>\n                                Denatured in binding buffer at 95 °C for 5 min and quickly cooled on ice for 10 min \n</td>
      <td>Binding to cell lines: KYSE150, other ESCC\nNo binding to: hepatocellular cancer cell line (SMMC-7721), gastric cancer cell line (SNU-5), lung cancer cell line (EBC-1), breast cancer cell line (MCF-7) and embryonic esophageal cell line (SHEE)</td>
      <td>5710.7 g/mole</td>
      <td>77.78%</td>
      <td>184000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>18</td>
      <td>Chen, X., Zhang, Y., Shi, Y., Niu, T., Li, B., & Guo, L. et al. (2021). Evolution of DNA aptamers against esophageal squamous cell carcinoma using cell-SELEX. The Analyst, 146(13), 4180-4187. doi: 10.1039/d1an00634g</td>
      <td>dAp,dTp,dGp,dGp,dCp,dCp,dAp,dGp,dGp,dGp,dGp,dGp,dGp,dAp,dGp,dGp,dGp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>380</th>
      <td>FMN-induced molecular switch</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/396.png" width="120" ></td>
      <td>flavin mononucleotide (FMN)</td>
      <td>Small Organic</td>
      <td>na nM (reported value)</td>
      <td>Reaction buffer [50 mM Tris.HCl (pH 7.5) and 20 mM MgCl2] for 20 hr</td>
      <td>23°C</td>
      <td>None</td>
      <td></td>
      <td>23333.13 g/mole</td>
      <td>58.33%</td>
      <td>710900 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>72</td>
      <td>Garrett A. Soukup and Ronald R. Breaker. "Engineering precision RNA molecular switches" Proc. Natl. Acad. Sci. 96(1999):3584-3589.</td>
      <td>rGp,rGp,rGp,rCp,rGp,rAp,rCp,rCp,rCp,rUp,rGp,rAp,rUp,rGp,rAp,rGp,rCp,rCp,rUp,rUp,rAp,rGp,rGp,rAp,rUp,rAp,rUp,rGp,rCp,rUp,rUp,rCp,rGp,rGp,rCp,rAp,rGp,rAp,rAp,rGp,rGp,rAp,rCp,rGp,rUp,rCp,rGp,rAp,rAp,rAp,rCp,rGp,rGp,rUp,rGp,rAp,rAp,rAp,rGp,rCp,rCp,rGp,rUp,rAp,rGp,rGp,rUp,rUp,rGp,rCp,rCp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>381</th>
      <td>N5</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7137.jpg" width="120" ></td>
      <td>PD-L1</td>
      <td>Protein</td>
      <td>441 nM (reported value)</td>
      <td>136 mM NaCl, 2.6 mM KCl, 4 mM Na2HPO4, 1.8 mM KH2PO4, 2 mM MgCl2, pH 7.4</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>12550.2 g/mole</td>
      <td>42.50%</td>
      <td>414300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>40</td>
      <td>Li, X., Li, Z., & Yu, H. (2020). Selection of threose nucleic acid aptamers to block PD-1/PD-L1 interaction for cancer immunotherapy. Chemical Communications. doi:10.1039/d0cc06032a</td>
      <td>dGp,dAp,dTp,dTp,dGp,dAp,dGp,dTp,dAp,dGp,dAp,dTp,dAp,dGp,dTp,dGp,dGp,dTp,dTp,dCp,dTp,dGp,dTp,dAp,dCp,dGp,dTp,dAp,dGp,dTp,dGp,dAp,dAp,dAp,dGp,dAp,dGp,dTp,dGp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>382</th>
      <td>Cyclic Cytidine Monophosphate (AR3)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/197.jpg" width="120" ></td>
      <td>cCMP</td>
      <td>Small Organic</td>
      <td>N/A nM (reported value)</td>
      <td>None</td>
      <td>23°C</td>
      <td>None</td>
      <td></td>
      <td>29205.73 g/mole</td>
      <td>53.33%</td>
      <td>908200 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>90</td>
      <td>Seetharaman et al. "Immobolized RNA switches for the analysis of complex chemical and biological mixture." Nature Biotechnology, 19(2001):336-341.</td>
      <td>rGp,rGp,rAp,rUp,rAp,rAp,rUp,rAp,rGp,rCp,rCp,rGp,rUp,rAp,rGp,rGp,rUp,rUp,rGp,rCp,rGp,rAp,rAp,rAp,rGp,rCp,rGp,rAp,rCp,rCp,rCp,rUp,rGp,rAp,rUp,rGp,rAp,rGp,rCp,rCp,rUp,rUp,rUp,rAp,rGp,rGp,rGp,rCp,rCp,rAp,rAp,rGp,rUp,rGp,rUp,rGp,rGp,rUp,rGp,rAp,rAp,rAp,rGp,rAp,rCp,rAp,rCp,rAp,rCp,rUp,rCp,rGp,rAp,rAp,rAp,rCp,rGp,rGp,rUp,rAp,rGp,rCp,rGp,rAp,rGp,rAp,rGp,rCp,rUp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>383</th>
      <td>Mitomer2</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/575.jpg" width="120" ></td>
      <td>Mitochondria</td>
      <td>Other</td>
      <td>. nM (reported value)</td>
      <td>20mM Tris-HCl, 2mM KCl, 40mM NaCl, 1mM MgCl2, 250mM Sucrose, pH=7.5</td>
      <td>37°C</td>
      <td>\n                                80C for 10 min, and cooled down slowly to 4C \n</td>
      <td></td>
      <td>9011.31 g/mole</td>
      <td>71.43%</td>
      <td>259100 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Binding activity measured by fluorescence intensity within referenced paper</td>
      <td>95</td>
      <td>Yuri Tawaraya, Mamoru Hyodo, Mst Naznin Ara, Yuma Yamada, and Hideyoshi Harashima, ƒ??RNA Aptamers for Targeting Mitochondria Using a Mitochondria-Based SELEX Methodƒ? Biol. Pharm. Bull. 37(8) 1411-1415 (2014)</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>384</th>
      <td>XA-B217</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7152.jpg" width="120" ></td>
      <td>Thy-1 membrane glycoprotein</td>
      <td>Protein</td>
      <td>2 nM (reported value)</td>
      <td>10 mL of 20 mM Tris buffer (150 mM NaCl, pH 8.0)</td>
      <td>22°C</td>
      <td>None</td>
      <td></td>
      <td>12868.8 g/mole</td>
      <td>59.52%</td>
      <td>389250 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>42</td>
      <td>Wang H, Li X, Lai LA, Brentnall TA, Dawson DW, Kelly KA, Chen R, Pan S. X-aptamers targeting Thy-1 membrane glycoprotein in pancreatic ductal adenocarcinoma. Biochimie. 2021 Feb;181:25-33. doi: 10.1016/j.biochi.2020.11.018. Epub 2020 Nov 23. PMID: 33242496; PMCID: PMC7863625.</td>
      <td>dTp,dTp,dGp,dCp,dCp,dCp,dAp,dCp,dCp,dGp,dTp,dAp,dCp,dTp,dGp,dTp,dGp,dCp,dAp,dGp,dGp,dTp,dCp,dGp,dAp,dAp,dCp,dTp,dAp,dCp,dAp,dGp,dGp,dCp,dAp,dCp,dGp,dTp,dCp,dGp,dTp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>385</th>
      <td>Myo40-7-27</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/602.png" width="120" ></td>
      <td>Myoglobin</td>
      <td>Protein</td>
      <td>4.93 nM (reported value)</td>
      <td>20 mM Hepes, 120 mM NaCl, 5 mM KCl, 1 mM CaCl2, 1mM  MgCl2, pH 7.3</td>
      <td>23°C</td>
      <td>None</td>
      <td></td>
      <td>12131.88 g/mole</td>
      <td>55.00%</td>
      <td>350000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>40</td>
      <td>Wang, Q., et al. ƒ??Sensitive point-of-care monitoring of cardiac biomarker myoglobin using aptamer and ubiquitous person glucose meter.ƒ? Biosensors and Bioelecgtronics, 64 (2015): 161-164.   Wang, Q., et al. ƒ??Screening of DNA Aptamers against Myoglobin Using a Positive and Negative Selection Units Integrated Microfluid Chip and its Biosensing Application.ƒ? Analytical Chemistry, 86 (2014): 6572-6579.</td>
      <td>dCp,dCp,dCp,dTp,dCp,dCp,dTp,dTp,dTp,dCp,dCp,dTp,dTp,dCp,dGp,dAp,dCp,dGp,dTp,dAp,dGp,dAp,dTp,dCp,dTp,dGp,dCp,dTp,dGp,dCp,dGp,dTp,dTp,dGp,dTp,dTp,dCp,dCp,dGp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>386</th>
      <td>Aptamin C</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7199.png" width="120" ></td>
      <td>Vitamin C</td>
      <td>Small Organic</td>
      <td>987.9 nM (reported value)</td>
      <td>50mM sodium acetate, 1% (w/v) BSA, 0.05% (v/v) Tween 20, 1mM MgCl2 (Sigma, all components) adjusted to pH 5.5</td>
      <td>22°C</td>
      <td>\n                                ViennaRNA \n</td>
      <td></td>
      <td>9995.4 g/mole</td>
      <td>78.13%</td>
      <td>296700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>32</td>
      <td>Chiu, A.S., Sankarapani, V., Drabek, R., Jackson, G.W., Batchelor, R.H. and Kim, Y., 2018. Inhibition of vitamin C oxidation by DNA aptamers. Aptamers, 2, pp.1-20.</td>
      <td>dGp,dTp,dGp,dGp,dAp,dGp,dGp,dCp,dGp,dGp,dTp,dGp,dGp,dCp,dCp,dAp,dGp,dTp,dCp,dTp,dCp,dGp,dCp,dGp,dGp,dTp,dGp,dGp,dCp,dGp,dGp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>387</th>
      <td>E. Coli 5S RNA (Helix 89 RNA)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/108.png" width="120" ></td>
      <td>E. Coli 5S RNA</td>
      <td>Small Organic</td>
      <td>3 µM (reported value)</td>
      <td>Binding buffer: 30 mM Trisƒ??acetate(pH 7.5), 60 mM magnesium acetate, 120 mM potassium acetate, and 120 mM ammonium acetate.</td>
      <td>RT°C</td>
      <td>\n                                in Binding buffer, heat to 55° C for 15 minutes and slowly let cool to RT. \n</td>
      <td></td>
      <td>11910.22 g/mole</td>
      <td>56.76%</td>
      <td>366600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>37</td>
      <td>None</td>
      <td>rGp,rGp,rUp,rGp,rAp,rUp,rAp,rCp,rCp,rGp,rCp,rCp,rCp,rAp,rAp,rGp,rAp,rGp,rUp,rUp,rCp,rAp,rUp,rAp,rUp,rCp,rGp,rAp,rCp,rGp,rGp,rCp,rGp,rGp,rUp,rGp,rUp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>388</th>
      <td>A-10 Excised Adenosine Aptamer</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7100.png" width="120" ></td>
      <td>Adenosine</td>
      <td>Small Organic</td>
      <td>12.7-16.9 nM (reported value)</td>
      <td>Column Buffer: 300 mM NaC1, 5 mM MgC12, 20 mM Tris (pH 7.6)</td>
      <td>37°C</td>
      <td>\n                                Heated to 75°C for 5 min and then cooled to room temperature for 20-30 min. \n</td>
      <td>This aptamer has high specificity to adenosine and low specificity to ATP, AMP, Guanosine, Cytidine, Uridine and Theophylline</td>
      <td>9966.5 g/mole</td>
      <td>50.00%</td>
      <td>311000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>32</td>
      <td>Li, Y., Liu, B., Huang, Z., & Liu, J. (2020). Engineering base-excised aptamers for highly specific recognition of adenosine. Chemical Science. doi:10.1039/d0sc00086h</td>
      <td>dGp,dTp,dAp,dTp,dTp,dGp,dCp,dGp,dGp,dAp,dGp,dGp,dAp,dAp,dGp,dGp,dTp,dTp,dTp,dTp,dTp,dAp,dAp,dCp,dCp,dTp,dTp,dCp,dGp,dGp,dGp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>389</th>
      <td>A5</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7148.jpg" width="120" ></td>
      <td>GLUT1 Glucose transporter</td>
      <td>Small Organic</td>
      <td>160 ± 49 nM nM (reported value)</td>
      <td>100 mg/mL of yeast tRNA (Thermo Fisher, AM7119) and 1 mg/mL of bovine serum albumin (RPI, A30075).</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>24647.9 g/mole</td>
      <td>47.50%</td>
      <td>755700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>80</td>
      <td>Rosch, J.C., Neal, E.H., Balikov, D.A. et al. CRISPR-Mediated Isogenic Cell-SELEX Approach for Generating Highly Specific Aptamers Against Native Membrane Proteins. Cel. Mol. Bioeng. 13, 559–574 (2020). https://doi.org/10.1007/s12195-020-00651-y</td>
      <td>dTp,dCp,dGp,dCp,dAp,dCp,dAp,dTp,dTp,dCp,dCp,dGp,dCp,dTp,dTp,dCp,dTp,dAp,dCp,dCp,dCp,dAp,dTp,dGp,dGp,dCp,dTp,dAp,dGp,dGp,dTp,dGp,dTp,dTp,dTp,dAp,dTp,dTp,dAp,dAp,dTp,dCp,dCp,dTp,dGp,dTp,dAp,dGp,dGp,dAp,dTp,dTp,dTp,dGp,dCp,dGp,dGp,dAp,dAp,dTp,dCp,dGp,dTp,dAp,dAp,dGp,dTp,dCp,dCp,dGp,dTp,dGp,dTp,dGp,dTp,dGp,dCp,dGp,dAp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>390</th>
      <td>ATP-induced molecular switch</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/393.png" width="120" ></td>
      <td>ATP</td>
      <td>Small Organic</td>
      <td>NA nM (reported value)</td>
      <td>reaction buffer [50 mM Tris.HCl (pH 7.5) and 20 mM MgCl2] for 20 hr</td>
      <td>23°C</td>
      <td>None</td>
      <td></td>
      <td>26224.86 g/mole</td>
      <td>60.49%</td>
      <td>785500 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>81</td>
      <td>None</td>
      <td>rGp,rGp,rGp,rCp,rGp,rAp,rCp,rCp,rCp,rUp,rGp,rAp,rUp,rGp,rAp,rGp,rCp,rCp,rUp,rUp,rGp,rGp,rGp,rAp,rAp,rGp,rAp,rAp,rAp,rCp,rUp,rGp,rUp,rGp,rGp,rCp,rAp,rCp,rUp,rUp,rCp,rGp,rGp,rUp,rGp,rCp,rCp,rAp,rGp,rCp,rAp,rCp,rGp,rUp,rCp,rGp,rAp,rAp,rAp,rCp,rGp,rGp,rUp,rGp,rAp,rAp,rAp,rGp,rCp,rCp,rGp,rUp,rAp,rGp,rGp,rUp,rUp,rGp,rCp,rCp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>391</th>
      <td>Cocaine Aptamer</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/614.jpg" width="120" ></td>
      <td>Cocaine</td>
      <td>Small Organic</td>
      <td>N/A nM (reported value)</td>
      <td>The buffer for immobilization of ssDNA probe consisted of 0.8M phosphate buffer (PB) with 1.0M NaCl, 5 mM MgCl2 and 1 mM ethylenediamine tetraacetic acid (EDTA), pH 7.0. The DNA hybridization buffer and cocaine incubation buffer was composed of 50mM PB with 100mM K2SO4 (pH 7.4).</td>
      <td>25°C</td>
      <td>\n                                N/A \n</td>
      <td>N/A</td>
      <td>13274.72 g/mole</td>
      <td>41.86%</td>
      <td>429800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>43</td>
      <td>Yang, Z. et al. Community Sewage Sensors towards Evaluation of Drug Use Trends: Detection of Cocaine in Wastewater with DNA-Directed Immobilization Aptamer Sensors. Sci. Rep. 6, 21024; doi: 10.1038/srep21024 (2016).</td>
      <td>dGp,dAp,dCp,dAp,dAp,dGp,dGp,dAp,dTp,dAp,dAp,dAp,dTp,dCp,dCp,dTp,dTp,dCp,dAp,dAp,dTp,dGp,dAp,dAp,dGp,dTp,dGp,dGp,dGp,dTp,dCp,dAp,dCp,dTp,dCp,dAp,dTp,dCp,dTp,dGp,dTp,dGp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>392</th>
      <td>a-Synuclein Oligomers (T-SO508)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/362.png" width="120" ></td>
      <td>a-synuclein oligomers</td>
      <td>Protein</td>
      <td>68 nM (reported value)</td>
      <td>TBS-T buffer for ELONA assay: 10 mM Tris-HCl, 150 mM NaCl, 5 mM KCl, pH 7.4 with 0.05% Tween 20.</td>
      <td>25°C</td>
      <td>\n                                Heated to 95°C for 10 min, then cooled at a rate of 2°C/min to a final temperature of 25°C. \n</td>
      <td>None</td>
      <td>7528.9 g/mole</td>
      <td>75.00%</td>
      <td>220200 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>CD spectrum suggests that this aptamer forms a G-quadruplex.</td>
      <td>24</td>
      <td>None</td>
      <td>dGp,dCp,dCp,dTp,dGp,dTp,dGp,dGp,dTp,dGp,dTp,dTp,dGp,dGp,dGp,dGp,dCp,dGp,dGp,dGp,dTp,dGp,dCp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>393</th>
      <td>Terminal 8-oxodG DNA Lesion (R10-B35)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/488.png" width="120" ></td>
      <td>DNA Lesion 7,8-dihydro-8-hydroxy-2'-deoxyguanosine (8-oxodG)</td>
      <td>Small Organic</td>
      <td>5 µM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td></td>
      <td>25157.29 g/mole</td>
      <td>50.00%</td>
      <td>782200 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>78</td>
      <td>None</td>
      <td>rGp,rGp,rGp,rCp,rGp,rAp,rAp,rUp,rUp,rCp,rCp,rCp,rGp,rAp,rGp,rGp,rAp,rCp,rCp,rAp,rAp,rAp,rUp,rAp,rGp,rUp,rAp,rCp,rCp,rAp,rCp,rCp,rCp,rGp,rGp,rGp,rAp,rAp,rAp,rAp,rCp,rAp,rGp,rCp,rUp,rAp,rAp,rUp,rGp,rCp,rCp,rGp,rAp,rAp,rAp,rCp,rGp,rGp,rAp,rGp,rAp,rUp,rUp,rUp,rUp,rUp,rUp,rCp,rUp,rGp,rCp,rAp,rGp,rAp,rAp,rGp,rCp,rUp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>394</th>
      <td>QA-36</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7146.jpg" width="120" ></td>
      <td>Soft-Shelled Turtle Iridovirus (STIV)</td>
      <td>Other</td>
      <td>53.8 nM (reported value)</td>
      <td>Binding Buffer: 100 mM NaCl, 2.5 mM MgCl2, 20mM Tris-HCl (pH 7.5)</td>
      <td>°C</td>
      <td>\n                                Not given. \n</td>
      <td>The aptamer only recognized the STIV virus when its binding was tested for other viruses.</td>
      <td>15794.2 g/mole</td>
      <td>68.00%</td>
      <td>478800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>50</td>
      <td>Li et al. "Characterization of DNA aptamers generated against the soft-shelled turtle iridovirus with antiviral effects." BMC Veterinary Research, 11 (2015): Article number 245.</td>
      <td>dTp,dGp,dTp,dGp,dCp,dGp,dGp,dGp,dGp,dGp,dAp,dGp,dGp,dGp,dGp,dAp,dGp,dTp,dGp,dGp,dCp,dGp,dCp,dTp,dGp,dTp,dTp,dGp,dGp,dTp,dGp,dCp,dGp,dGp,dGp,dTp,dAp,dTp,dAp,dGp,dCp,dGp,dCp,dGp,dTp,dGp,dGp,dTp,dGp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>395</th>
      <td>Interleukin 32 (IL-32) (AC3-3)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/118.jpg" width="120" ></td>
      <td>Interleukin-32 (IL-32)</td>
      <td>Protein</td>
      <td>78 nM (reported value)</td>
      <td>None</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>29155.58 g/mole</td>
      <td>51.11%</td>
      <td>905300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>90</td>
      <td>Kim et al. "Generation of Antagonistic RNA Aptamers Specific to Proinflammatory Cytokine Interleukin-32." Bulletin of the  Korean Chemical Society, 31 (2010):  3561-3566.</td>
      <td>rGp,rGp,rGp,rUp,rUp,rCp,rAp,rCp,rUp,rGp,rCp,rAp,rGp,rAp,rCp,rUp,rUp,rGp,rAp,rCp,rGp,rAp,rAp,rGp,rCp,rUp,rUp,rCp,rCp,rGp,rGp,rAp,rGp,rAp,rGp,rAp,rAp,rGp,rGp,rGp,rUp,rCp,rAp,rAp,rAp,rGp,rUp,rUp,rGp,rUp,rGp,rCp,rGp,rGp,rGp,rAp,rGp,rUp,rGp,rUp,rGp,rUp,rUp,rGp,rUp,rGp,rGp,rAp,rAp,rUp,rGp,rGp,rAp,rUp,rCp,rCp,rAp,rCp,rAp,rUp,rCp,rUp,rAp,rCp,rGp,rAp,rAp,rUp,rUp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>396</th>
      <td>Concanavalin A Lectin</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/266.png" width="120" ></td>
      <td>Concanavalin A Lectin</td>
      <td>Protein</td>
      <td>46 µM (reported value)</td>
      <td>Con A Buffer (CAB: 50 mM NaCl/20 mM Mops (pH 6.8)/2 mM MgCl2/2 mM CaCl2/0.2 mM EDTA)], room temperature</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>1501.65 g/mole</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>12</td>
      <td>Oldenburg et al. "  Peptide ligands for a sugar-binding protein isolated from a random peptide library." PNAS, 89(1992):5393-5397.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>397</th>
      <td>Hepatitis C Virus Non-Structural Protein 3 (G9-1)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/124.jpg" width="120" ></td>
      <td>Hepatitis C Virus Non-Structural Protein 3</td>
      <td>Protein</td>
      <td>11.6 nM (reported value)</td>
      <td>Binding Buffer (50 mM Tris/HCl (pH 7.8), 30 mM NaCl, 5 mM CaCl2 and 10 mM dithiothreitol)    Elution Buffer (0.4 m sodium acetate, 5 mm EDTA and 7 m urea (pH 5.5))</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>23294.92 g/mole</td>
      <td>50.68%</td>
      <td>704000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>73</td>
      <td>Fukuda, et al. "Isolation and characterization of RNA aptamers specific for the hepatitis C virus nonstructural protein 3 protease." European Journal of Biochemistry, 267(2000): 3685-3694.</td>
      <td>rGp,rGp,rGp,rAp,rGp,rAp,rAp,rUp,rUp,rCp,rCp,rGp,rAp,rCp,rCp,rAp,rGp,rAp,rAp,rGp,rCp,rUp,rUp,rCp,rGp,rGp,rGp,rAp,rUp,rUp,rUp,rGp,rAp,rGp,rGp,rGp,rUp,rAp,rGp,rAp,rAp,rUp,rGp,rGp,rGp,rAp,rCp,rUp,rAp,rCp,rCp,rUp,rUp,rUp,rCp,rCp,rUp,rCp,rUp,rCp,rUp,rCp,rCp,rUp,rUp,rCp,rCp,rUp,rCp,rUp,rUp,rCp,rUp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>398</th>
      <td>VCAM-1</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/237.jpg" width="120" ></td>
      <td>VCAM-1</td>
      <td>Tissue</td>
      <td>N/A nM (reported value)</td>
      <td>Washed with PBS containing 0.1% Tween-20.</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>1097.39 g/mole</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>9</td>
      <td>None</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>399</th>
      <td>HGF (H38-15)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/471" width="120" ></td>
      <td>Human  Hepatocyte Growth Factor</td>
      <td>Protein</td>
      <td>19 nM (reported value)</td>
      <td>None</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>31366.29 g/mole</td>
      <td>55.88%</td>
      <td>928300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>102</td>
      <td>Saito, Takesh et al. "Generation of Inhibitory DNA Aptamers Against Human  Hepatocyte Growth Factor." DNA and Cell Biology, 24 (2005): 624-633.</td>
      <td>dAp,dGp,dAp,dTp,dGp,dCp,dCp,dTp,dGp,dTp,dCp,dCp,dAp,dGp,dCp,dAp,dTp,dCp,dGp,dAp,dGp,dCp,dGp,dCp,dCp,dAp,dGp,dCp,dTp,dTp,dTp,dGp,dCp,dTp,dGp,dAp,dTp,dGp,dGp,dGp,dTp,dGp,dGp,dCp,dCp,dAp,dCp,dCp,dCp,dTp,dTp,dGp,dCp,dCp,dCp,dTp,dGp,dGp,dGp,dTp,dTp,dTp,dGp,dAp,dAp,dTp,dTp,dTp,dCp,dGp,dAp,dTp,dCp,dCp,dTp,dAp,dTp,dCp,dGp,dGp,dTp,dAp,dGp,dCp,dTp,dAp,dGp,dAp,dCp,dTp,dGp,dCp,dTp,dTp,dTp,dGp,dTp,dCp,dCp,dTp,dCp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>400</th>
      <td>Phorate (SS2-55)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/292.png" width="120" ></td>
      <td>Phorate</td>
      <td>Small Organic</td>
      <td>1.11 µM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td>This aptamer binds preferentially to the organophosphates phorate, profenofos, benzoate and omethoate. It has reduced binding to imidacloprid, acetamiprid, chlorpyrifos, methamidophos, dichlorvos, parathion, dimethoate, monocrotophos and phoxim.</td>
      <td>16990.09 g/mole</td>
      <td>49.09%</td>
      <td>526900 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>"A molecular beacon (5' FAM-CTGCACAAGAATCGCTGCAG-3') was designed, which was partly complementary with the middle constant nucleotides-sequence of the selected ssDNAs."</td>
      <td>55</td>
      <td>None</td>
      <td>dAp,dAp,dGp,dCp,dTp,dTp,dGp,dCp,dTp,dTp,dTp,dAp,dTp,dAp,dGp,dCp,dCp,dTp,dGp,dCp,dAp,dGp,dCp,dGp,dAp,dTp,dTp,dCp,dTp,dTp,dGp,dAp,dTp,dCp,dGp,dGp,dAp,dAp,dAp,dAp,dGp,dGp,dCp,dTp,dGp,dAp,dGp,dAp,dGp,dCp,dTp,dAp,dCp,dGp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>401</th>
      <td>EphA2 Receptor (SWL peptide)</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/236.png" width="120" ></td>
      <td>EphA2 Receptor</td>
      <td>Tissue</td>
      <td>678 nM (reported value)</td>
      <td>Blocked with 0.5% BSA in TBS and washed with binding buffer (TBS, 1 mM CaCl2, 0.1% Tween 20).</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>1505.73 g/mole</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>12</td>
      <td>None</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>402</th>
      <td>Lysozyme (Clone 1)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/458.png" width="120" ></td>
      <td>Hen egg white lysozyme</td>
      <td>Protein</td>
      <td>29-31 nM (reported value)</td>
      <td>None</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>24750.16 g/mole</td>
      <td>45.00%</td>
      <td>788000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>81</td>
      <td>None</td>
      <td>dGp,dGp,dGp,dAp,dAp,dTp,dGp,dGp,dAp,dTp,dCp,dCp,dAp,dCp,dAp,dTp,dCp,dTp,dAp,dCp,dGp,dAp,dAp,dTp,dTp,dCp,dAp,dTp,dCp,dAp,dGp,dGp,dGp,dCp,dTp,dAp,dAp,dAp,dGp,dAp,dGp,dTp,dGp,dCp,dAp,dGp,dAp,dGp,dTp,dTp,dAp,dCp,dTp,dTp,dAp,dGp,dTp,dTp,dCp,dAp,dCp,dTp,dGp,dCp,dAp,dGp,dAp,dCp,dTp,dTp,dGp,dAp,dCp,dGp,dAp,dAp,dGp,dCp,dTp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>403</th>
      <td>Endothelial Cells (P36)</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/199.png" width="120" ></td>
      <td>Endothelial Cells</td>
      <td>Cells</td>
      <td>2.2 nM (reported value)</td>
      <td>Binding Buffer (1 mM glucose, 1% BSA, 10 mM MgSO4)    Selection Buffer (0.1% BSA, 1 mM CaCl2, and 10 mM MgCl2)</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>1667.95 g/mole</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>12</td>
      <td>Herrmann et al. "Selection of cell specific peptides in a rat carotid injury model using a random peptide-presenting bacterial library." Bichemica et Biophysica Acta, 1472(1999): 529-536.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>404</th>
      <td>1G8-14</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/595.jpg" width="120" ></td>
      <td>Ebola Virus VP35</td>
      <td>Protein</td>
      <td>3.7 nM (reported value)</td>
      <td>10 mM HEPES (pH 7.0), 150 mM NaCl, 2mM TCEP (tris(2-carboxyethyl)phosphine), and 1 mM MgCl2 and filtered through nitrocellulose membrane</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>29057.6 g/mole</td>
      <td>46.67%</td>
      <td>914300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>270</td>
      <td>Binning, J., Wang, T., Luthra, P., et al. (2013) Development of RNA aptamers targeting Ebola Virus VP35. Biochemistry, 52(47):8406-8419. doi: 10.1021/bi400704d</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>405</th>
      <td>GL21.T</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/552.png" width="120" ></td>
      <td>Axl tyrosine kinase receptor</td>
      <td>Protein</td>
      <td>12 nM (reported value)</td>
      <td>Dulbecco's Modified Eagle's Medium (DMEM), 10% FBS, 2mM L-glutamine</td>
      <td>37°C</td>
      <td>\n                                85C for 5 minutes, snap cooled on ice for 2 minutes, and then warmed up to 37C. \n</td>
      <td></td>
      <td>10875.53 g/mole</td>
      <td>50.00%</td>
      <td>335000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Note that the sequence is 2'-fluoropyrimidine, 2'-ribopurine (see reference paper).</td>
      <td>34</td>
      <td>Laura Cerchia, Carla L Esposito, Simona Camorani, Anna Rienzo, Loredana Stasio,  Luigi Insabato, Andrea Affuso and Vittorio de Franciscis,"Targeting Axl With an High-affinity Inhibitory Aptamer". Molecular Therapy vol.20 no.12, 2291-2303, Dec2012.</td>
      <td>rAp,fUp,rGp,rAp,fUp,fCp,rAp,rAp,fUp,fCp,rGp,fCp,fCp,fUp,fCp,rAp,rAp,fUp,fUp,fCp,rGp,rAp,fCp,rAp,rGp,rGp,rAp,rGp,rGp,fCp,fUp,fCp,rAp,fCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>406</th>
      <td>SARS Coronavirus NTPase/Helicase (ES15-1)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/126.jpg" width="120" ></td>
      <td>SARS Coronavirus NTPase/Helicase</td>
      <td>Protein</td>
      <td>1.2 nM (reported value)</td>
      <td>Binding Buffer (30 mM Trisƒ??HCl (pH 7.5), 150 mM NaCl, 1.5 mM MgCl2  , 2 mM dithiothreitol (DTT) and  1% (w/v) BSA)    Termination Buffer (0.2 M EDTA, 0.2 M NaCl, and 20 mM HEPES (pH  7.4))    Buffer A (25 mM Trisƒ??HCl (pH 7.5), and 0.3 mM NaCl)</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>34719.1 g/mole</td>
      <td>46.73%</td>
      <td>1104500 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>107</td>
      <td>Jang, K., et al. "Isolation of inhibitory RNA aptamers against severe acute respiratory  syndrome (SARS) coronavirus NTPase/Helicase." Biochemical and Biophysical Research Communications, 366(2008): 738-744.</td>
      <td>rGp,rAp,rUp,rAp,rAp,rUp,rAp,rCp,rGp,rAp,rCp,rUp,rCp,rAp,rCp,rUp,rAp,rUp,rAp,rGp,rGp,rGp,rUp,rUp,rCp,rAp,rCp,rUp,rGp,rCp,rAp,rGp,rAp,rCp,rUp,rUp,rGp,rAp,rCp,rGp,rAp,rAp,rGp,rCp,rUp,rUp,rGp,rCp,rAp,rGp,rAp,rAp,rAp,rAp,rGp,rGp,rGp,rGp,rGp,rAp,rAp,rGp,rAp,rAp,rGp,rAp,rGp,rGp,rGp,rUp,rGp,rAp,rUp,rUp,rCp,rAp,rGp,rGp,rCp,rGp,rAp,rGp,rAp,rGp,rAp,rAp,rUp,rGp,rGp,rAp,rUp,rCp,rCp,rAp,rCp,rAp,rUp,rCp,rUp,rAp,rCp,rGp,rAp,rAp,rUp,rUp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>407</th>
      <td>C-Reactive Protein Monomer</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/164.png" width="120" ></td>
      <td>Monomeric C-Reactive Protein (CRP)</td>
      <td>Protein</td>
      <td>187.7 nM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td>Selectively binds to monomeric CRP and not pentameric CRP.</td>
      <td>14346.65 g/mole</td>
      <td>56.82%</td>
      <td>444700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>M S Wang (2012) confirmed monomeric binding using a gel shift assay.</td>
      <td>44</td>
      <td>None</td>
      <td>rGp,rCp,rCp,rUp,rGp,rUp,rAp,rAp,rGp,rGp,rUp,rGp,rGp,rUp,rCp,rGp,rGp,rUp,rGp,rUp,rGp,rGp,rCp,rGp,rAp,rGp,rUp,rGp,rUp,rGp,rUp,rUp,rAp,rGp,rGp,rAp,rGp,rAp,rGp,rAp,rUp,rUp,rGp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>408</th>
      <td>Carcinoembryonic Antigen Aptamer (YJ-1)</td>
      <td>Chimeric</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/330.png" width="120" ></td>
      <td>Carcinoembryonic Antigen (CEA)</td>
      <td>Protein</td>
      <td>0.77 nM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td></td>
      <td>15869.54 g/mole</td>
      <td>42.86%</td>
      <td>507000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>49</td>
      <td>None</td>
      <td>rGp,fCp,rGp,rGp,rAp,rAp,rGp,fCp,rGp,fUp,rGp,fCp,fUp,rGp,rGp,rGp,fCp,fUp,rAp,rGp,rAp,rAp,fUp,rAp,rAp,fUp,rAp,rAp,fUp,rAp,rAp,rGp,rAp,rAp,rAp,rAp,fCp,fCp,rAp,rGp,fUp,rAp,fCp,fUp,fUp,fUp,fCp,rGp,fUp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>409</th>
      <td>Hepatitis C Virus RdRp (r10/43)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/599" width="120" ></td>
      <td>Hepatitis C Virus RdRp</td>
      <td>Protein</td>
      <td>1.3 nM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td></td>
      <td>25933.89 g/mole</td>
      <td>53.01%</td>
      <td>827900 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>83</td>
      <td>None</td>
      <td>dGp,dGp,dGp,dAp,dGp,dAp,dCp,dAp,dAp,dGp,dAp,dAp,dTp,dAp,dAp,dAp,dCp,dGp,dCp,dTp,dCp,dAp,dAp,dGp,dGp,dGp,dCp,dGp,dTp,dGp,dGp,dTp,dGp,dGp,dGp,dTp,dGp,dGp,dGp,dGp,dTp,dAp,dCp,dTp,dAp,dAp,dTp,dAp,dAp,dTp,dGp,dTp,dGp,dCp,dGp,dTp,dTp,dTp,dGp,dTp,dTp,dCp,dGp,dAp,dCp,dAp,dGp,dGp,dAp,dGp,dGp,dCp,dTp,dCp,dAp,dCp,dAp,dAp,dCp,dAp,dGp,dGp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>410</th>
      <td>L-Histidine (His 945)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/114.png" width="120" ></td>
      <td>L-Histidine</td>
      <td>Small Organic</td>
      <td>12 µM (reported value)</td>
      <td>Selection Buffer (50 mM Hepes (pH 7.0), 250 mM NaCl, 5 mM each CaCl2  and MgCl2, and 1 mM glycine)</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>13017.94 g/mole</td>
      <td>52.50%</td>
      <td>410400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>40</td>
      <td>Majerfeld, et al. "RNA Affnity for Molecular L-Histidine; Genetic Code Origins." Journal of Molecular Evolution, 61 (2005): 226-235.</td>
      <td>rGp,rGp,rCp,rAp,rUp,rCp,rGp,rGp,rAp,rAp,rAp,rGp,rUp,rGp,rGp,rGp,rUp,rUp,rGp,rAp,rUp,rGp,rUp,rAp,rAp,rGp,rUp,rAp,rAp,rCp,rAp,rGp,rGp,rCp,rGp,rAp,rUp,rGp,rCp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>411</th>
      <td>WT62</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7128.jpg" width="120" ></td>
      <td>HIV-1 Reverse Transcriptase</td>
      <td>Protein</td>
      <td>7.51±0.29x10^-8 nM (reported value)</td>
      <td>10 mM HEPES pH 7.5, 150 mM NaCl, 80 mM KCl, and \n5 mM MgCl2</td>
      <td>25°C</td>
      <td>None</td>
      <td></td>
      <td>12687.3 g/mole</td>
      <td>39.02%</td>
      <td>412100 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>41</td>
      <td>Hannongbua, S., Ratanabunyong, S., Aeksiri, N., Yanaka, S., Yagi-Utsumi, M., Kato, K., & Choowongkomol, K. (2020). Characterization of new DNA Aptamers for anti‐HIV‐1 Reverse Transcriptase. ChemBioChem. doi:10.1002/cbic.202000633</td>
      <td>dTp,dCp,dGp,dGp,dTp,dTp,dGp,dAp,dGp,dTp,dGp,dAp,dGp,dAp,dTp,dTp,dAp,dGp,dTp,dCp,dAp,dAp,dAp,dTp,dGp,dTp,dTp,dTp,dAp,dGp,dAp,dAp,dGp,dAp,dCp,dAp,dAp,dCp,dCp,dTp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>412</th>
      <td>Antibac 2</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/625.jpg" width="120" ></td>
      <td>peptidoglycan</td>
      <td>Other</td>
      <td>195.9 nM (reported value)</td>
      <td>Selection buffer (PBS containing 1.4 mM MgCl2 and 0.05% BSA); Room temperature (45 mins)</td>
      <td>°C</td>
      <td>None</td>
      <td></td>
      <td>16912.91 g/mole</td>
      <td>69.09%</td>
      <td>497500 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Antibac 2 was tested on A. baumanni, L. monocitogenes, E. coli, S. aureus, and K. pneumonia, among others. The Kd listed above is for K. Pneumonia.</td>
      <td>55</td>
      <td>A Graziani et al. High efficiency binding aptamers for a wide range of sepsis 1 bacterial agents. JMB Papers in Press. 2017: JMB16-11004.</td>
      <td>dTp,dCp,dGp,dCp,dGp,dCp,dGp,dAp,dGp,dTp,dCp,dGp,dTp,dCp,dTp,dGp,dGp,dGp,dGp,dGp,dAp,dCp,dTp,dAp,dGp,dAp,dGp,dGp,dAp,dCp,dTp,dTp,dGp,dTp,dGp,dCp,dGp,dGp,dCp,dCp,dCp,dCp,dGp,dCp,dAp,dTp,dCp,dGp,dTp,dCp,dCp,dTp,dCp,dCp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>413</th>
      <td>Ricin Toxin (C5)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/406" width="120" ></td>
      <td>Ricin Toxin</td>
      <td>Small Organic</td>
      <td>58 nM (reported value)</td>
      <td>None</td>
      <td>N/A°C</td>
      <td>\n                                Denature at 94°C for 5 minutes, cool slowly to room temperature \n</td>
      <td></td>
      <td>26766.37 g/mole</td>
      <td>56.98%</td>
      <td>837700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>86</td>
      <td>None</td>
      <td>dAp,dTp,dAp,dGp,dGp,dAp,dGp,dTp,dCp,dAp,dCp,dGp,dAp,dCp,dGp,dAp,dCp,dCp,dAp,dGp,dAp,dAp,dCp,dCp,dGp,dTp,dAp,dGp,dGp,dTp,dTp,dCp,dGp,dGp,dGp,dGp,dCp,dGp,dGp,dAp,dGp,dTp,dGp,dGp,dTp,dCp,dCp,dGp,dGp,dAp,dAp,dGp,dGp,dTp,dGp,dGp,dCp,dGp,dTp,dGp,dGp,dTp,dAp,dTp,dGp,dTp,dGp,dCp,dGp,dTp,dCp,dTp,dAp,dCp,dCp,dTp,dCp,dTp,dTp,dGp,dAp,dCp,dTp,dAp,dAp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>414</th>
      <td>Streptomycin (LS11)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/438.png" width="120" ></td>
      <td>Streptomycin and other Aminoglycosides (AMGs)</td>
      <td>Small Organic</td>
      <td>19.8 nM (reported value)</td>
      <td>Binding Buffer: PBS with 10% (v/v) methanol</td>
      <td>RT°C</td>
      <td>None</td>
      <td>Binds to several aminoglycosides: kanamycin, paromycin streptomycin, apramycin, tobramycin, neomycin, and gentamycin.</td>
      <td>25474.89 g/mole</td>
      <td>46.84%</td>
      <td>788000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>87</td>
      <td>None</td>
      <td>rGp,rGp,rGp,rAp,rGp,rAp,rAp,rGp,rGp,fCp,rGp,rGp,fCp,rGp,fCp,rGp,fUp,rAp,rGp,rGp,fCp,rGp,rAp,rGp,fCp,fUp,fUp,fUp,rAp,fCp,fCp,rAp,rGp,fUp,fUp,fUp,fUp,rAp,fUp,fUp,fUp,rGp,fUp,fUp,fUp,fUp,rAp,fUp,fUp,rGp,fUp,fUp,rAp,fUp,rAp,fUp,rGp,fCp,fUp,fUp,rAp,fUp,fUp,fCp,rGp,fUp,rGp,rGp,rAp,fCp,rGp,rAp,fUp,rGp,fCp,rGp,fCp,rAp,fCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>415</th>
      <td>Fructose-bisboronic acid aptamer-sensor</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7103.jpg" width="120" ></td>
      <td>Fructose</td>
      <td>Small Organic</td>
      <td>4 nM (reported value)</td>
      <td>Shinkai's receptor 50uM with 2% methanol.</td>
      <td>24 °C°C</td>
      <td>\n                                Mix and incubate for 5 min at 95°C, and cool down at room temperature 24°C > 30 min \n</td>
      <td>N/A</td>
      <td>15770.23 g/mole</td>
      <td>60.78%</td>
      <td>523430 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>sensor mixtures (fluorescein-labeled aptamer + competitor dabcyl-labeled oligonucleotide) to the addition of Shinkai’s receptor in the presence of increasing amounts of sugar, with the actual concentration of a receptor*complex at any particular glucose concentrations calculated from the Kd of Shinkai’s receptor.</td>
      <td>51</td>
      <td>Yang KA, Barbu M, Halim M, et al. Recognition and sensing of low-epitope targets via ternary complexes with oligonucleotides and synthetic receptors. Nat Chem. 2014;6(11):1003-1008. doi:10.1038/nchem.2058</td>
      <td>dCp,dTp,dCp,dTp,dCp,dGp,dGp,dGp,dAp,dCp,dGp,dAp,dCp,dGp,dGp,dCp,dTp,dGp,dGp,dCp,dAp,dCp,dGp,dTp,dTp,dTp,dGp,dGp,dTp,dTp,dCp,dAp,dAp,dGp,dAp,dAp,dTp,dGp,dTp,dGp,dGp,dGp,dTp,dGp,dTp,dCp,dGp,dTp,dCp,dCp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>416</th>
      <td>HPA-2</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7164.jpg" width="120" ></td>
      <td>Helicobacter pylori</td>
      <td>Other</td>
      <td>19.3 ± 3.2 nM (reported value)</td>
      <td>100 mM NaCl, 5 mM KCl, 50 mM Tris−HCl, and 1 mM MgCl2, pH 7.5</td>
      <td>30°C</td>
      <td>None</td>
      <td></td>
      <td>12297 g/mole</td>
      <td>55.00%</td>
      <td>385700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>40</td>
      <td>Wu, Hangjie et al. “Rapid Detection of Helicobacter pylori by the Naked Eye Using DNA Aptamers.” ACS omega vol. 6,5 3771-3779. 21 Jan. 2021, doi:10.1021/acsomega.0c05374</td>
      <td>dCp,dCp,dAp,dGp,dGp,dAp,dGp,dGp,dAp,dCp,dCp,dCp,dTp,dAp,dTp,dTp,dCp,dTp,dCp,dGp,dTp,dGp,dTp,dAp,dTp,dCp,dGp,dAp,dCp,dGp,dAp,dGp,dAp,dTp,dCp,dCp,dAp,dGp,dTp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>417</th>
      <td>PSMA Aptamer (A10-3)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/83.png" width="120" ></td>
      <td>PSMA</td>
      <td>Protein</td>
      <td></td>
      <td>None</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>17966.62 g/mole</td>
      <td>55.36%</td>
      <td>539200 L/(mole·cm)</td>
      <td>20.5 nM (reported value)</td>
      <td></td>
      <td>None</td>
      <td>56</td>
      <td>Lupold, Shawn, et al. "Identification and Characterization of Nuclease-stabilized RNA Molecules That Bind Human Prostate Cancer Cells via the Prostate-specific  Membrane Antigen." American Association for Cancer Research Journal 62, (2002): 4029-4033</td>
      <td>rGp,rGp,rGp,rAp,rGp,rGp,rAp,fCp,rGp,rAp,fUp,rGp,fCp,rGp,rGp,rAp,fUp,fCp,rAp,rGp,fCp,fCp,rAp,fUp,rGp,fUp,fUp,fUp,rAp,fCp,rGp,fUp,fCp,rAp,fCp,fUp,fCp,fCp,fUp,fUp,rGp,fUp,fCp,rAp,rAp,fUp,fCp,fCp,fUp,fCp,rAp,fUp,fCp,rGp,rGp,fCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>418</th>
      <td>Digoxin (Truncated D1)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/405.png" width="120" ></td>
      <td>Digoxin</td>
      <td>Protein</td>
      <td>0.05 nM (reported value)</td>
      <td>None</td>
      <td>N/A°C</td>
      <td>None</td>
      <td>Kd for ouabain was 6.4 nM.</td>
      <td>15635.16 g/mole</td>
      <td>62.00%</td>
      <td>482000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>This aptamer reverses the inotropic effect of digoxin in isolated guinea-pig atria at a concentration of 400 nM. No effect on ouabain was observed at the same concentration.</td>
      <td>51</td>
      <td>None</td>
      <td>dAp,dGp,dCp,dGp,dAp,dGp,dGp,dGp,dCp,dGp,dGp,dTp,dGp,dTp,dCp,dCp,dAp,dAp,dCp,dAp,dGp,dCp,dGp,dGp,dTp,dTp,dTp,dTp,dTp,dTp,dCp,dAp,dCp,dGp,dAp,dGp,dGp,dAp,dGp,dGp,dTp,dTp,dGp,dGp,dCp,dGp,dGp,dTp,dGp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>419</th>
      <td>Interferon-γ (2'F-1)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/406.png" width="120" ></td>
      <td>Interferon-y/IFN-y/IFNG</td>
      <td>Protein</td>
      <td>4.4 nM (reported value)</td>
      <td>None</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>18947.24 g/mole</td>
      <td>57.63%</td>
      <td>566400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>This is a biphasic binding described by a two-Kd model, 4.4/716 nM.</td>
      <td>59</td>
      <td>Kubik, Mark F., et al. "Isolation and Characterization of  2ƒ??-Fluoro-,  2ƒ??-Amino-,  and 2ƒ??-Fluoro-/Amino-Modified  RNA Ligands to  Human  IFN-7 That Inhibit  Receptor  Binding." Journal of Immunology, 159 (1997): 259-267.</td>
      <td>rGp,rGp,rGp,rAp,rGp,rGp,rAp,fCp,rGp,rAp,fUp,rGp,fCp,rGp,rGp,rAp,fCp,rAp,fCp,fCp,rGp,fUp,fUp,rAp,rAp,fUp,fCp,fUp,rGp,rAp,rGp,rGp,fCp,fCp,fCp,fUp,rGp,fUp,fCp,fCp,fUp,rAp,fUp,fUp,fCp,fCp,fUp,fUp,fCp,rAp,fCp,rGp,fCp,fCp,fUp,fCp,rAp,rGp,rAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>420</th>
      <td>YFL-6</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7237.jpg" width="120" ></td>
      <td>Lactoferrin</td>
      <td>Protein</td>
      <td>4.97 nM (reported value)</td>
      <td>1x PBSMC</td>
      <td>37°C</td>
      <td>\n                                IDT \n</td>
      <td>Does not interact with related proteins BSA, casein, beta-lactoglobulin, or alpha-lactalbumin.</td>
      <td>15297 g/mole</td>
      <td>56.00%</td>
      <td>462100 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>50</td>
      <td>Yu, F., Li, H., Sun, W., Xu, D. and He, F., 2019. Rapid selection of aptamers based on protein microarray. RSC advances, 9(17), pp.9762-9768.</td>
      <td>dAp,dGp,dGp,dCp,dAp,dGp,dGp,dAp,dCp,dAp,dCp,dCp,dGp,dTp,dAp,dAp,dCp,dCp,dGp,dGp,dTp,dGp,dCp,dAp,dTp,dCp,dTp,dAp,dTp,dGp,dGp,dCp,dTp,dAp,dCp,dTp,dAp,dGp,dCp,dTp,dCp,dTp,dTp,dCp,dCp,dTp,dGp,dCp,dCp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>421</th>
      <td>Tryptophan-Cp*Rh(III) sensor</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7107.jpg" width="120" ></td>
      <td>Tryptophan</td>
      <td>Small Organic</td>
      <td>1.2 µM (reported value)</td>
      <td>100 μM concentration of monomeric Cp*Rh(III)</td>
      <td>24 °C°C</td>
      <td>\n                                Mix and incubate for 5 min at 95oC, and cool down at room temperature > 30 min \n</td>
      <td>N/A</td>
      <td>15668.16 g/mole</td>
      <td>60.78%</td>
      <td>522150 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>51</td>
      <td>Yang KA, Barbu M, Halim M, et al. Recognition and sensing of low-epitope targets via ternary complexes with oligonucleotides and synthetic receptors. Nat Chem. 2014;6(11):1003-1008. doi:10.1038/nchem.2058</td>
      <td>dCp,dTp,dCp,dTp,dCp,dGp,dGp,dGp,dAp,dCp,dGp,dAp,dCp,dCp,dGp,dCp,dGp,dGp,dTp,dAp,dGp,dTp,dCp,dTp,dTp,dAp,dAp,dCp,dCp,dTp,dAp,dAp,dAp,dGp,dCp,dGp,dGp,dTp,dGp,dTp,dCp,dAp,dGp,dGp,dTp,dCp,dGp,dTp,dCp,dCp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>422</th>
      <td>ACG03</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/532.png" width="120" ></td>
      <td>gastric cancer cell-line HGC-27</td>
      <td>Cells</td>
      <td>16.49 nM (reported value)</td>
      <td>4.5 g/l glucose, 5 mM MgCl2,2 mg/ml BSA and 0.2 mg/ml yeast tRNA dissolved in PBS buffer</td>
      <td>4°C</td>
      <td>None</td>
      <td></td>
      <td>24321.75 g/mole</td>
      <td>60.26%</td>
      <td>730600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>HGC-27 was used as target (positive cell line) and GES-1 as counter selection (negative cell line)</td>
      <td>78</td>
      <td>Zhang, X. et al.  "A cell-based single-stranded DNA aptamer specifically targets gastric cancer."  The International Journal of Biochemistry & Cell Biology 2014, 46: 1-8.</td>
      <td>dAp,dCp,dGp,dCp,dTp,dCp,dGp,dGp,dAp,dTp,dGp,dCp,dCp,dAp,dCp,dTp,dAp,dCp,dAp,dGp,dGp,dGp,dGp,dGp,dGp,dTp,dGp,dGp,dTp,dCp,dCp,dTp,dGp,dAp,dGp,dGp,dGp,dTp,dGp,dGp,dTp,dGp,dTp,dGp,dGp,dTp,dTp,dGp,dGp,dTp,dTp,dTp,dGp,dGp,dTp,dTp,dTp,dCp,dCp,dTp,dCp,dAp,dTp,dGp,dGp,dAp,dCp,dGp,dTp,dGp,dCp,dTp,dGp,dGp,dTp,dGp,dAp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>423</th>
      <td>XA-B216</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7151.jpg" width="120" ></td>
      <td>Thy-1 membrane glycoprotein</td>
      <td>Protein</td>
      <td>36 nM (reported value)</td>
      <td>10 mL of 20 mM Tris buffer (150 mM NaCl, pH 8.0)</td>
      <td>22°C</td>
      <td>None</td>
      <td></td>
      <td>12868.3 g/mole</td>
      <td>66.67%</td>
      <td>380700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>42</td>
      <td>Wang H, Li X, Lai LA, Brentnall TA, Dawson DW, Kelly KA, Chen R, Pan S. X-aptamers targeting Thy-1 membrane glycoprotein in pancreatic ductal adenocarcinoma. Biochimie. 2021 Feb;181:25-33. doi: 10.1016/j.biochi.2020.11.018. Epub 2020 Nov 23. PMID: 33242496; PMCID: PMC7863625.</td>
      <td>dTp,dTp,dGp,dCp,dCp,dCp,dAp,dCp,dCp,dTp,dCp,dCp,dCp,dTp,dGp,dTp,dGp,dCp,dGp,dGp,dGp,dCp,dCp,dAp,dCp,dAp,dGp,dAp,dGp,dCp,dAp,dGp,dCp,dAp,dGp,dTp,dGp,dTp,dCp,dGp,dTp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>424</th>
      <td>Thrombin (15mer)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/64.jpg" width="120" ></td>
      <td>Thrombin</td>
      <td>Protein</td>
      <td>84 nM (reported value)</td>
      <td>None</td>
      <td>37°C</td>
      <td>None</td>
      <td>Specific to human thrombin. Does not bind to ovalbumin, human fibrinogen or bovine thrombin.</td>
      <td>4726.13 g/mole</td>
      <td>60.00%</td>
      <td>143300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>15</td>
      <td>None</td>
      <td>dGp,dGp,dTp,dTp,dGp,dGp,dTp,dGp,dTp,dGp,dGp,dTp,dTp,dGp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>425</th>
      <td>Anti Influenza NS1 protein</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/504.png" width="120" ></td>
      <td>NS1</td>
      <td>Protein</td>
      <td>19.91 nM (reported value)</td>
      <td>50 mM Tris/Cl (pH 8.0), 150 mM NaCl, 1.5 mM MgCl2, 2 mM DTT  and 1% (w/v) BSA</td>
      <td>23°C</td>
      <td>\n                                95C for 10min  Denaturing  Cold-shock by placing on ice for 10min to renature \n</td>
      <td></td>
      <td>27570.81 g/mole</td>
      <td>67.05%</td>
      <td>834900 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>ƒ??NS1 is a multifunctional proteinƒ??However, the major role of NS1 is to Antagonize Interferonsƒ??inhibiting the hostƒ??s Innate immunityƒ? (Woo)  This aptamer has shown activity  that blocks NS1 interaction with hosts (woo)</td>
      <td>88</td>
      <td>Hye-Min Woo. Single-stranded DNA aptamer that specifically binds to the influenza  Virus NS1 protein suppresses interferon antagonism</td>
      <td>dGp,dCp,dAp,dAp,dTp,dGp,dGp,dTp,dAp,dCp,dGp,dGp,dTp,dAp,dCp,dTp,dTp,dCp,dCp,dGp,dCp,dGp,dGp,dTp,dCp,dCp,dGp,dGp,dGp,dGp,dTp,dGp,dGp,dGp,dTp,dGp,dGp,dGp,dTp,dGp,dGp,dTp,dGp,dGp,dGp,dGp,dGp,dGp,dTp,dGp,dCp,dGp,dGp,dGp,dGp,dGp,dGp,dGp,dCp,dGp,dGp,dCp,dCp,dGp,dCp,dAp,dAp,dAp,dAp,dGp,dTp,dGp,dCp,dAp,dCp,dGp,dCp,dTp,dAp,dCp,dTp,dTp,dTp,dGp,dCp,dTp,dAp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>426</th>
      <td>L-Selectin (LD201t1)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/407.png" width="120" ></td>
      <td>L-Selectin binding to sialyl Lewis X</td>
      <td>Protein</td>
      <td></td>
      <td>None</td>
      <td>22°C</td>
      <td>None</td>
      <td></td>
      <td>15113.89 g/mole</td>
      <td>46.94%</td>
      <td>481400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>49</td>
      <td>Hicke, Brain, et al. "DNA Aptamers Block L-Selectin Function In Vivo Inhibition of Human Lymphocyte Traf?cking in SCID Mic." Journal of Clinical Investigation, 98 (1996): 2688-2692</td>
      <td>dTp,dAp,dGp,dCp,dCp,dAp,dAp,dGp,dGp,dTp,dAp,dAp,dCp,dCp,dAp,dGp,dTp,dAp,dCp,dAp,dAp,dGp,dGp,dTp,dGp,dCp,dTp,dAp,dAp,dAp,dCp,dGp,dTp,dAp,dAp,dTp,dGp,dGp,dCp,dTp,dTp,dCp,dGp,dGp,dCp,dTp,dTp,dAp,dCp</td>
      <td>3 nM (reported value)</td>
    </tr>
    <tr>
      <th>427</th>
      <td>Theophylline + Flavin Mononucleotide (AR7)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/385.png" width="120" ></td>
      <td>Theophylline and FMN</td>
      <td>Protein</td>
      <td>N/A nM (reported value)</td>
      <td>None</td>
      <td>23°C</td>
      <td>None</td>
      <td></td>
      <td>37838.86 g/mole</td>
      <td>51.28%</td>
      <td>1175900 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Cleaves only in the presence of both theophylline and FMN.</td>
      <td>118</td>
      <td>Seetharaman et al. "Immobilized RNA switches for the analysis of complex chemical and biological mixture." Nature Biotechnology, 19(2001):336-341.</td>
      <td>rGp,rGp,rAp,rUp,rAp,rAp,rUp,rAp,rGp,rCp,rCp,rGp,rUp,rAp,rGp,rGp,rUp,rUp,rGp,rCp,rGp,rAp,rAp,rAp,rGp,rCp,rGp,rAp,rCp,rCp,rCp,rUp,rGp,rAp,rUp,rGp,rAp,rGp,rCp,rCp,rUp,rUp,rAp,rGp,rGp,rAp,rUp,rAp,rUp,rGp,rUp,rCp,rUp,rGp,rGp,rAp,rUp,rAp,rCp,rCp,rAp,rUp,rGp,rCp,rAp,rUp,rGp,rAp,rUp,rGp,rCp,rAp,rCp,rCp,rUp,rUp,rGp,rGp,rCp,rAp,rGp,rUp,rCp,rUp,rUp,rAp,rCp,rAp,rGp,rAp,rAp,rGp,rGp,rAp,rCp,rGp,rUp,rCp,rGp,rAp,rAp,rAp,rCp,rGp,rGp,rUp,rAp,rGp,rCp,rGp,rAp,rGp,rAp,rGp,rCp,rUp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>428</th>
      <td>1G8-14</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/580.jpg" width="120" ></td>
      <td>Ebola virus inhibitory domain (eVP35 IID) (EBOV) (eVP35)</td>
      <td>Protein</td>
      <td>3.7 nM (reported value)</td>
      <td>10mM HEPES (pH 7.0), 150mM NaCl, 2mM TCEP (tris(2-carboxyethyl)phosphine), 1mM MgCl2</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>22934.2 g/mole</td>
      <td>60.00%</td>
      <td>715100 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>230</td>
      <td>Jennifer M. Binning, Tianjiao Wang, Priya Luthra, Reed S. Shabman, Dominika M. Borek, Gai  Liu, Wei Xu, Daisy W. Leung, Christopher F. Basler, and Gaya K. Amarasinghe, ƒ??Development of RNA aptamers targeting ebolavirus VP35ƒ? Biochemistry 2013, 52, 8406-8419.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>429</th>
      <td>Celiac disease Aptamer (Gli4)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/540.jpg" width="120" ></td>
      <td>33-mer (immunodominant peptide from alpha2-gliadin)</td>
      <td>Peptide</td>
      <td>61 nM (reported value)</td>
      <td>50 mM TRIS-HCl, 0.25 M NaCl, 5mM MgCl2</td>
      <td>°C</td>
      <td>\n                                N/A \n</td>
      <td>None</td>
      <td>12103.86 g/mole</td>
      <td>57.50%</td>
      <td>357500 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>120</td>
      <td>Gonzalez, S, et. al. "Aptamer Binding to Celiac Disease-Triggering Hydrophobic Proteins A Sensitive Gluten Detection Approach." 2014. Analytical Chemistry 86: 2733-39.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>430</th>
      <td>HIV-1 Rev Peptide (L1-Rev)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/413.png" width="120" ></td>
      <td>HIV-1 Rev Peptide</td>
      <td>Peptide</td>
      <td>18 µM (reported value)</td>
      <td>None</td>
      <td>25°C</td>
      <td>None</td>
      <td></td>
      <td>25129.29 g/mole</td>
      <td>59.74%</td>
      <td>774600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>77</td>
      <td>None</td>
      <td>rGp,rGp,rAp,rCp,rCp,rUp,rCp,rGp,rGp,rCp,rGp,rAp,rAp,rAp,rGp,rCp,rCp,rGp,rGp,rUp,rAp,rAp,rCp,rGp,rCp,rCp,rAp,rCp,rAp,rAp,rGp,rUp,rCp,rGp,rGp,rAp,rGp,rGp,rGp,rUp,rAp,rAp,rGp,rAp,rUp,rCp,rUp,rGp,rAp,rCp,rAp,rGp,rGp,rAp,rAp,rCp,rUp,rGp,rGp,rGp,rUp,rGp,rAp,rCp,rGp,rGp,rAp,rGp,rGp,rUp,rUp,rAp,rGp,rGp,rUp,rGp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>431</th>
      <td>TD05.17</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7108.png" width="120" ></td>
      <td>B-cells</td>
      <td>Cells</td>
      <td>43 nM (reported value)</td>
      <td>RPMI 1640 and 4.5 g/l glucose and 5 mM MgCl2</td>
      <td>37°C</td>
      <td>\n                                None \n</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>BNA residues in red</td>
      <td>37</td>
      <td>Mallikaratchy, P. R., Ruggiero, A., Gardner, J. R., Kuryavyi, V., Maguire, W. F., Heaney, M. L., McDevitt, M. R., Patel, D. J., & Scheinberg, D. A. (2011). A multivalent DNA aptamer specific for the B-cell receptor on human lymphoma and leukemia. Nucleic acids research, 39(6), 2458–2469. https://doi.org/10.1093/nar/gkq996</td>
      <td>dAp,dGp,dGp,dAp,dGp,dGp,dAp,dTp,dAp,dGp,dTp,dTp,dCp,dGp,dGp,dTp,dGp,dGp,dCp,dTp,dGp,dTp,dTp,dCp,dAp,dGp,dGp,dGp,dTp,dCp,dTp,dCp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>432</th>
      <td>Trypanosoma cruzi-Heparan Parasite Receptor (HS6)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/169.jpg" width="120" ></td>
      <td>Trypanosoma cruzi-Heparan Parasite Receptor</td>
      <td>Cells</td>
      <td>40 nM (reported value)</td>
      <td>50 uM RNA containing 20 mM glucose.</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>34422.25 g/mole</td>
      <td>52.34%</td>
      <td>1047600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>107</td>
      <td>None</td>
      <td>rAp,fCp,fCp,rGp,rAp,rGp,fUp,fCp,fCp,rAp,rGp,rAp,rAp,rGp,fCp,fUp,fUp,rGp,fUp,rAp,rGp,fUp,rAp,fCp,fUp,rAp,fCp,fCp,rGp,fCp,rAp,rAp,fCp,rGp,fUp,fUp,rGp,rGp,fCp,rGp,fUp,fUp,fCp,rGp,rGp,fCp,rGp,fUp,fUp,rGp,rGp,fCp,fCp,rGp,fCp,rGp,fUp,fCp,rAp,fUp,fCp,rGp,fCp,fUp,rAp,rGp,fCp,fCp,fUp,rAp,rGp,rAp,fUp,rGp,rGp,rAp,rGp,fUp,fUp,rGp,rAp,rAp,fUp,fUp,fCp,fUp,fCp,fCp,fCp,fUp,rAp,fUp,rAp,rGp,fUp,rGp,rAp,rGp,fUp,fCp,rGp,fUp,rAp,fUp,fUp,rAp,fCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>433</th>
      <td>Tumor Endothelial Cells (AraHH001)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/492.png" width="120" ></td>
      <td>Mouse and Human Tumour Endothelial Cells (mTECs and hTECs)</td>
      <td>Cells</td>
      <td>43.8 nM (reported value)</td>
      <td>None</td>
      <td>4°C</td>
      <td>\n                                Heated at 80°C for 10 min, then slowly cooled. \n</td>
      <td>Did not bind to skin-endothelial cells, OS-RC-EC, RFP-SM, HUVEC, and HMVEC cell lines. Bound to mOS-RC-EC from a different origin.</td>
      <td>12071.86 g/mole</td>
      <td>52.50%</td>
      <td>364700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Additional testing by Ara et al. revealed AraHH001 impaired the ability of mTECs to form capillary-like structures in a matrigel. Also, confocal microscopy showed that AraHH001 is internalized by the cell into  acid compartments.</td>
      <td>40</td>
      <td>None</td>
      <td>dAp,dCp,dGp,dTp,dAp,dCp,dCp,dGp,dAp,dCp,dTp,dTp,dCp,dGp,dTp,dAp,dTp,dGp,dCp,dCp,dAp,dAp,dCp,dAp,dGp,dCp,dCp,dCp,dTp,dTp,dTp,dAp,dTp,dCp,dCp,dAp,dCp,dCp,dTp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>434</th>
      <td>Adenosine</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/209.jpg" width="120" ></td>
      <td>ATP</td>
      <td>Small Organic</td>
      <td>1 mM (reported value)</td>
      <td>Selection Buffer (300 mM NaCl, 50 mM Tris-acetate (pH 7.2))</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>39076.43 g/mole</td>
      <td>48.41%</td>
      <td>1243100 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>126</td>
      <td>Liu and Lu. "Adenosine-Dependent Assembly of Aptazyme-Functionalized Gold Nanoparticles and Its Application as a Colorimetric Biosensor." Analytical Chemistry, 76 (2004): 1627-1632.</td>
      <td>dAp,dCp,dTp,dCp,dAp,dTp,dCp,dTp,dGp,dTp,dGp,dAp,dGp,dAp,dCp,dTp,dCp,dAp,dCp,dTp,dAp,dTp,dAp,dGp,dGp,dAp,dAp,dGp,dAp,dGp,dAp,dTp,dGp,dTp,dCp,dAp,dAp,dCp,dTp,dCp,dGp,dTp,dGp,dCp,dAp,dCp,dGp,dAp,dGp,dTp,dTp,dGp,dAp,dCp,dAp,dTp,dCp,dTp,dCp,dTp,dTp,dCp,dTp,dCp,dCp,dGp,dAp,dGp,dCp,dCp,dGp,dGp,dTp,dCp,dGp,dAp,dAp,dAp,dTp,dAp,dTp,dTp,dGp,dGp,dAp,dGp,dGp,dAp,dAp,dGp,dCp,dTp,dCp,dGp,dAp,dGp,dCp,dTp,dGp,dGp,dAp,dGp,dGp,dAp,dAp,dAp,dAp,dGp,dTp,dGp,dAp,dGp,dTp,dCp,dTp,dCp,dAp,dCp,dAp,dGp,dAp,dTp,dGp,dAp,dGp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>435</th>
      <td>XQ-P3</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7194.jpg" width="120" ></td>
      <td>PD-L1</td>
      <td>Protein</td>
      <td>15.36 nM (reported value)</td>
      <td>D-PBS supplemented with 4.5 g/L of glucose, 5 mM of MgCl2, 0.1 mg/mL of yeast tRNA, and 1 mg/mL of BSA</td>
      <td>22°C</td>
      <td>None</td>
      <td></td>
      <td>19261.5 g/mole</td>
      <td>57.14%</td>
      <td>573400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>63</td>
      <td>Wu X, Li F, Li Y, Yu Y, Liang C, Zhang B, Zhao C, Lu A, Zhang G. A PD-L1 Aptamer Selected by Loss-Gain Cell-SELEX Conjugated with Paclitaxel for Treating Triple-Negative Breast Cancer. Med Sci Monit. 2020 Jun 23;26:e925583. doi: 10.12659/MSM.925583. PMID: 32574155; PMCID: PMC7331476.</td>
      <td>dAp,dCp,dCp,dGp,dAp,dCp,dCp,dGp,dTp,dGp,dCp,dTp,dGp,dGp,dAp,dCp,dTp,dCp,dAp,dTp,dCp,dTp,dCp,dGp,dCp,dTp,dTp,dTp,dTp,dTp,dTp,dCp,dAp,dCp,dGp,dGp,dTp,dCp,dCp,dAp,dCp,dAp,dCp,dTp,dAp,dCp,dTp,dAp,dTp,dGp,dAp,dGp,dCp,dGp,dAp,dGp,dCp,dCp,dTp,dGp,dGp,dCp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>436</th>
      <td>Clone 1</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/546.jpg" width="120" ></td>
      <td>Protein E (PE)</td>
      <td>Protein</td>
      <td>nM (reported value)</td>
      <td>165 ug (~5 nmol) purified truncated His-tagged PE immobilized on Niƒ??NTA agarose (Qiagen) in 1.6 mL PBSM (PBS with 1 mM MgCl2)</td>
      <td>23°C</td>
      <td>None</td>
      <td></td>
      <td>22454.14 g/mole</td>
      <td>63.77%</td>
      <td>674000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>227</td>
      <td>Barfod, Anders et al. "In Vitro Selection of RNA Aptamers Directed Against Protein E: A Haemophilus influenzae Adhesin." Mol Biotech, 2014: 1-12.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>437</th>
      <td>EphA2 Receptor (YSA peptide)</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/235.png" width="120" ></td>
      <td>EphA2 Receptor</td>
      <td>Tissue</td>
      <td>186 nM (reported value)</td>
      <td>Blocked with 0.5% BSA in TBS and washed with binding buffer (TBS, 1 mM CaCl2, 0.1% Tween 20).</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>1483.71 g/mole</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>12</td>
      <td>None</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>438</th>
      <td>L-Isoleucine (IL 42-32b)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/112.png" width="120" ></td>
      <td>L-Isoleucine</td>
      <td>Small Organic</td>
      <td>200 µM (reported value)</td>
      <td>Selection Buffer (10 mM HEPES (pH 7.0), 300 mM NaCl, and 7.5 mM MgCl2  (buffer B) and 0.1 mM each of ZnCl2, CaCl2, and MnCl2 and 0.01 mM CuCl)  Reaction Buffer (50 mM sodium  borate (pH 8.0), 42 mM KCl, 7.5 mM MgCl2, and 0.4 mM ZnCl2)</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>10233.16 g/mole</td>
      <td>56.25%</td>
      <td>310800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>0</td>
      <td>Majerfeld and Yarus. "Isoleucine:RNA sites with associated coding sequence." RNA, 4 (1998): 471-478.</td>
      <td>rGp,rGp,rUp,rCp,rUp,rUp,rAp,rCp,rGp,rUp,rCp,rGp,rUp,rUp,rCp,rGp,rCp,rGp,rAp,rCp,rUp,rAp,rUp,rUp,rGp,rGp,rGp,rAp,rGp,rAp,rCp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>439</th>
      <td>RA16</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7113.png" width="120" ></td>
      <td>NCl-H460</td>
      <td>Cells</td>
      <td>24.75 ± 2.28 nM (reported value)</td>
      <td>10× transcription buffer (400 mM Tris-Cl, 80 mM MgCl2, and 20 mM spermidine), 10 mM dithiothreitol, 20 U T7 mutant (Y639F) RNA polymerase, 10 mM ATP, 10 mM GTP, 10 mM 2′-F-dCTP/UTP, 2 mM 16-Biotin-UTP, 20 U RiboLock RNase Inhibitor, and 0.05 U inorganic pyrophosphatase. The resulting reaction mixture was treated with 2 μL DNase I at 37 °C for 1 h, followed by phenol-chloroform extraction.</td>
      <td>37 °C°C</td>
      <td>\n                                RNA pellets were suspended in RNA refolding buffer (10 mM HEPES pH 7.4, 50 mM NaCl, 1 mM CaCl2,1 mM MgCl2, and 2.7 mM KCl), followed by refolding at 90 °C for 3 min and slowly cooling to room temperature. \n</td>
      <td></td>
      <td>26830.1 g/mole</td>
      <td>54.22%</td>
      <td>826700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>83</td>
      <td>Wang, Hanlu, et al. “In Vivo SELEX of an Inhibitory NSCLC-Specific RNA Aptamer from PEGylated RNA Library.” Molecular Therapy - Nucleic Acids, Cell Press, 9 Dec. 2017, www.sciencedirect.com/science/article/pii/S2162253117303013.</td>
      <td>rGp,rGp,rGp,rAp,rGp,rAp,rGp,rAp,rAp,rCp,rAp,rAp,rUp,rGp,rAp,rCp,rCp,rUp,rGp,rCp,rGp,rGp,rUp,rGp,rCp,rCp,rAp,rAp,rGp,rCp,rCp,rGp,rUp,rCp,rGp,rGp,rGp,rUp,rUp,rAp,rUp,rGp,rUp,rUp,rGp,rAp,rUp,rCp,rUp,rCp,rCp,rUp,rCp,rAp,rAp,rGp,rGp,rAp,rCp,rGp,rAp,rGp,rUp,rGp,rCp,rAp,rUp,rUp,rGp,rCp,rAp,rUp,rCp,rAp,rCp,rGp,rUp,rCp,rAp,rGp,rUp,rAp,rGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>440</th>
      <td>PC12 Cells (Aptamer17a)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/227.png" width="120" ></td>
      <td>PC12 Cells</td>
      <td>Cells</td>
      <td>N/A nM (reported value)</td>
      <td>Washed with selection buffer (5 mM MgCl in PBS) from and 1 mg/ml yeast tRNAfor 30 minutes.</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>7954.23 g/mole</td>
      <td>60.00%</td>
      <td>255700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Binding conditions from Wang et al. (2003) and truncated aptamer 17 (Aptamer 17a) from Liu et al. (2011).</td>
      <td>25</td>
      <td>None</td>
      <td>dGp,dGp,dTp,dTp,dGp,dGp,dAp,dTp,dGp,dTp,dAp,dAp,dGp,dGp,dTp,dTp,dGp,dGp,dAp,dGp,dGp,dGp,dGp,dGp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>441</th>
      <td>MUC1-5TR-1</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/561.jpg" width="120" ></td>
      <td>human breast adenocarcinoma (MCF7)</td>
      <td>Cells</td>
      <td>47.3 nM (reported value)</td>
      <td>0.5M NaCl, 10mM Tris-HCl, 1mM MgCl2, in Nuclease Free Water, pH 7.5-7.6</td>
      <td>25°C</td>
      <td>\n                                2hr, room temperature \n</td>
      <td></td>
      <td>22353.64 g/mole</td>
      <td>45.83%</td>
      <td>748000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>239</td>
      <td>John R. Stecker, Alissa A. Savage, John G. Bruno, Dana M. Garcia, Joseph R. Koke. "Dynamics and Visualization of MCF7 Adenocarcinoma Cell Death by Aptamer-C1q-Mediated Membrane Attack" Nucleic Acid Therapeutics, Vol.22, Num.4 (2012)</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>442</th>
      <td>EF508</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7153.jpg" width="120" ></td>
      <td>Enterococcus faecalis bacteria</td>
      <td>Other</td>
      <td>37 nM (reported value)</td>
      <td>10 mM of Tris–EDTA bufer (pH 8.5)</td>
      <td>22°C</td>
      <td>None</td>
      <td></td>
      <td>26627.3 g/mole</td>
      <td>43.02%</td>
      <td>853800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>86</td>
      <td>Kolm C, Cervenka I, Aschl UJ, et al. DNA aptamers against bacterial cells can be efficiently selected by a SELEX process using state-of-the art qPCR and ultra-deep sequencing. Sci Rep. 2020;10(1):20917. Published 2020 Dec 1. doi:10.1038/s41598-020-77221-9</td>
      <td>dTp,dAp,dGp,dGp,dGp,dAp,dAp,dGp,dAp,dGp,dAp,dAp,dGp,dGp,dAp,dCp,dAp,dTp,dAp,dTp,dGp,dAp,dTp,dAp,dCp,dTp,dGp,dGp,dCp,dCp,dTp,dTp,dGp,dAp,dCp,dAp,dCp,dCp,dCp,dTp,dGp,dTp,dTp,dGp,dTp,dGp,dGp,dCp,dTp,dTp,dGp,dAp,dTp,dGp,dAp,dCp,dAp,dAp,dTp,dAp,dAp,dCp,dAp,dTp,dTp,dGp,dAp,dCp,dTp,dAp,dGp,dTp,dAp,dCp,dAp,dTp,dGp,dAp,dCp,dCp,dAp,dCp,dTp,dTp,dGp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>443</th>
      <td>Mouse Transferrin Receptor (FB4)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/451.png" width="120" ></td>
      <td>Mouse Transferrin Receptor (mTfR)</td>
      <td>Protein</td>
      <td>N/A nM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td>Does not bind to human TfR cells.</td>
      <td>25651.39 g/mole</td>
      <td>62.03%</td>
      <td>765200 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>The RNA aptamer-streptavidin conjugate was mouse-specific.  Human cells needed to first be transfected with mouse transferrin receptor in order to be taken up by human cells.  The FB4 aptamer does not inhibit transferrin suggesting that binding is not competitive.</td>
      <td>79</td>
      <td>E F Neufeld et al. Aptamer-based endocytosis of a lysosomal enzyme. Proc. Natl. Acad. Sci. USA 105(2008): 15908-15913</td>
      <td>rGp,rGp,rGp,rCp,rGp,rAp,rAp,rUp,rUp,rCp,rCp,rGp,rCp,rGp,rUp,rGp,rUp,rGp,rCp,rUp,rGp,rAp,rGp,rGp,rGp,rCp,rGp,rGp,rAp,rAp,rGp,rAp,rAp,rCp,rUp,rAp,rAp,rUp,rUp,rUp,rGp,rGp,rGp,rAp,rCp,rGp,rGp,rAp,rUp,rUp,rGp,rCp,rGp,rGp,rCp,rCp,rGp,rUp,rUp,rGp,rUp,rCp,rUp,rGp,rUp,rGp,rGp,rCp,rGp,rUp,rCp,rCp,rGp,rUp,rUp,rCp,rGp,rGp,rGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>444</th>
      <td>B03</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7186.jpeg" width="120" ></td>
      <td>β2-Microglobulin (B2M)</td>
      <td>Protein</td>
      <td>36.7 nM (reported value)</td>
      <td>Ammonium Bicarbonate buffer (50 mM, pH 8.5) containing 500 mM NaCl.</td>
      <td>RT°C</td>
      <td>None</td>
      <td></td>
      <td>24620.9 g/mole</td>
      <td>51.25%</td>
      <td>754000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>80</td>
      <td>Ma, Y.; Li, W.; Xing, R.; Li, P.; Liu, Z. Epitope-Imprinted Magnetic Nanoparticles as a General Platform for Efficient In Vitro Evolution of Protein-Binding Aptamers. ACS Sens. 2020 Jul 7; 5(8): 2537-2544. DOI: 10.1012/acssensors.0c00846</td>
      <td>dCp,dTp,dTp,dCp,dTp,dGp,dCp,dCp,dCp,dGp,dCp,dCp,dTp,dCp,dCp,dTp,dTp,dCp,dCp,dTp,dAp,dTp,dTp,dTp,dAp,dTp,dTp,dGp,dAp,dAp,dTp,dGp,dCp,dTp,dGp,dTp,dAp,dAp,dTp,dTp,dTp,dCp,dGp,dCp,dCp,dAp,dCp,dGp,dCp,dGp,dAp,dAp,dAp,dGp,dTp,dAp,dGp,dGp,dGp,dGp,dAp,dGp,dAp,dCp,dGp,dAp,dGp,dAp,dTp,dAp,dGp,dGp,dCp,dGp,dGp,dAp,dCp,dAp,dCp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>445</th>
      <td>Tobramycin</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/439.png" width="120" ></td>
      <td>Tobramycin and other Aminoglycosides (AMGs)</td>
      <td>Small Organic</td>
      <td>24.8 nM (reported value)</td>
      <td>Binding Buffer: PBS with 10% (v/v) methanol</td>
      <td>RT°C</td>
      <td>None</td>
      <td>Binds to several aminoglycosides: kanamycin, paromycin streptomycin, apramycin, tobramycin, neomycin, and gentamycin.</td>
      <td>25474.89 g/mole</td>
      <td>46.84%</td>
      <td>788000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>87</td>
      <td>None</td>
      <td>rGp,rGp,rGp,rAp,rGp,rAp,rAp,rGp,rGp,fCp,rGp,rGp,fCp,rGp,fCp,rGp,fUp,rAp,rGp,rGp,fCp,rGp,rAp,rGp,fCp,fUp,fUp,fUp,rAp,fCp,fCp,rAp,rGp,fUp,fUp,fUp,fUp,rAp,fUp,fUp,fUp,rGp,fUp,fUp,fUp,fUp,rAp,fUp,fUp,rGp,fUp,fUp,rAp,fUp,rAp,fUp,rGp,fCp,fUp,fUp,rAp,fUp,fUp,fCp,rGp,fUp,rGp,rGp,rAp,fCp,rGp,rAp,fUp,rGp,fCp,rGp,fCp,rAp,fCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>446</th>
      <td>VEGF receptor KDR/Flk-1 (K237)</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/238.png" width="120" ></td>
      <td>VEGF receptor KDR/Flk-1</td>
      <td>Protein</td>
      <td>N/A nM (reported value)</td>
      <td>Washing Buffer: 0.9% NaCl, 0.01 M phosphate buffer, pH 6.2, blocked with 0.3% BSA, 25 mM HEPES (pH 7.6) for 3 h.</td>
      <td>RT°C</td>
      <td>\n                                N/A \n</td>
      <td></td>
      <td>1803.06 g/mole</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Disrupts the interaction between VEGF and and the KDR receptor and causes inhibition of angiogenesis and tumor growth.</td>
      <td>12</td>
      <td>None</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>447</th>
      <td>Aflatoxin B1 Aptamer (AFB1)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/526.jpg" width="120" ></td>
      <td>Aflatoxin B1</td>
      <td>Small Organic</td>
      <td>11.39 nM (reported value)</td>
      <td>pH 7.0: 100 mmol/L NaCl, 20 mmol/L Trisƒ??HCl pH 7.6, 2 mmol/L MgCl2, 5 mmol/L KCl, 1 mmol/L CaCl2, 0.02 % Tween 20</td>
      <td>°C</td>
      <td>\n                                N/A \n</td>
      <td>N/A</td>
      <td>24665.01 g/mole</td>
      <td>55.00%</td>
      <td>762900 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>80</td>
      <td>Ma, X., Wang, W., Chen, X., Xia, Y., Wu, S., Duan, N., and Wang, Z. "Selection, identification, and application of Aflatoxin B1 aptamer." European Food Research and Technology 238.6 (2014): 919-925.</td>
      <td>dAp,dGp,dCp,dAp,dGp,dCp,dAp,dCp,dAp,dGp,dAp,dGp,dGp,dTp,dCp,dAp,dGp,dAp,dTp,dGp,dGp,dTp,dGp,dCp,dTp,dAp,dTp,dCp,dAp,dTp,dGp,dCp,dGp,dCp,dTp,dCp,dAp,dAp,dTp,dGp,dGp,dGp,dAp,dGp,dAp,dCp,dTp,dTp,dTp,dAp,dGp,dCp,dTp,dGp,dCp,dCp,dCp,dCp,dCp,dAp,dCp,dCp,dTp,dAp,dTp,dGp,dCp,dGp,dTp,dGp,dCp,dTp,dAp,dCp,dCp,dGp,dTp,dGp,dAp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>448</th>
      <td>1-PA</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/562.jpg" width="120" ></td>
      <td>Anthrax Protective Antigen (PA)</td>
      <td>Protein</td>
      <td>8.53 nM (reported value)</td>
      <td>50 uL Human Serum, 5 mM HEPES</td>
      <td>37°C</td>
      <td>\n                                n/a \n</td>
      <td>n/a</td>
      <td>9124.01 g/mole</td>
      <td>48.28%</td>
      <td>294200 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>96</td>
      <td>Byul Nim Oh, Sungeun Lee, Hye-Yeon Park, Jin-Ook Baeg, Moon-Young Yoon, and Jinheung Kim. 2011. Reference: Sensitive fluorescence assay of anthrax protective antigen with two new DNA aptamers and their binding properties. Analyst, 2011, 136, 3384. DOI: 10.1039/c0an00978d</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>449</th>
      <td>AS1411</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/639.png" width="120" ></td>
      <td>non-small lung cancer cell A549</td>
      <td>Cells</td>
      <td></td>
      <td>A549 PBS solutions or whole blood</td>
      <td>37°C</td>
      <td>\n                                Room Temperature for 4 hours incubation with ion-selective-electrode (ISE) sensor \n</td>
      <td>unspecific reaction and accumulation observed with erythrocyte, platelet and protein molecules</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>10 CFU/mL (reported value)</td>
      <td>Best cell-sensitivity in buffer is -?25.21?mV/log10CA549.    3-aminopropyltriethoxysilane (APTES) and glutaraldehyde (GA) based covalent anchoring method is used to immobilize AS1411 onto the PGO decorated I-  -ISE (PGO-I-ISE)</td>
      <td>26</td>
      <td>Zhang R, Gu Y, Wang Z, Li Y, Fan Q, Jia Y. Aptamer cell sensor based on porous graphene oxide decorated ion-selective-electrode: Double sensing platform for cell and ion. Biosensors and Bioelectronics. 2018 Jun 15.</td>
      <td>dGp,dGp,dTp,dGp,dGp,dTp,dGp,dGp,dTp,dGp,dGp,dTp,dTp,dGp,dTp,dGp,dGp,dTp,dGp,dGp,dTp,dGp,dGp,dTp,dGp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>450</th>
      <td>Lactobacillum acidophilius (hemag1P)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/212.png" width="120" ></td>
      <td>Lactobacillum acidophilius</td>
      <td>Cells</td>
      <td>13 nM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td></td>
      <td>24010.64 g/mole</td>
      <td>47.44%</td>
      <td>751500 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Most likely target is the S-layer protein expressed on the cell surface.</td>
      <td>78</td>
      <td>None</td>
      <td>dAp,dGp,dCp,dAp,dGp,dCp,dAp,dCp,dAp,dGp,dAp,dGp,dGp,dTp,dCp,dAp,dGp,dAp,dTp,dGp,dTp,dAp,dGp,dCp,dCp,dCp,dTp,dTp,dCp,dAp,dAp,dCp,dAp,dTp,dAp,dGp,dTp,dAp,dAp,dTp,dAp,dTp,dCp,dTp,dCp,dTp,dGp,dCp,dAp,dTp,dTp,dCp,dTp,dGp,dTp,dGp,dTp,dGp,dCp,dCp,dTp,dAp,dTp,dGp,dCp,dGp,dTp,dGp,dCp,dTp,dAp,dCp,dCp,dGp,dTp,dGp,dAp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>451</th>
      <td>HA 12-16</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/533.jpg" width="120" ></td>
      <td>HA protein (gHA1)</td>
      <td>Protein</td>
      <td>nM (reported value)</td>
      <td>20 mM sodium phosphate, pH 7.5, 500 mM NaCl and 20 mM imidazole</td>
      <td>23°C</td>
      <td>None</td>
      <td></td>
      <td>29132.96 g/mole</td>
      <td>50.00%</td>
      <td>915800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>282</td>
      <td>Kwon, H. et al.  "An RNA Aptamer That Specifically Binds to the Glycosylated Hemagglutinin of Avian Influenza Virus and Suppresses Viral Infection in Cells."  PLOS One 2014, 9 1-9.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>452</th>
      <td>B01</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7185.jpeg" width="120" ></td>
      <td>β2-Microglobulin (B2M)</td>
      <td>Protein</td>
      <td>52.9 nM (reported value)</td>
      <td>Ammonium bicarbonate buffer (50 mM, pH 8.5) containing 500 mM NaCl.</td>
      <td>RT°C</td>
      <td>None</td>
      <td></td>
      <td>24550.8 g/mole</td>
      <td>60.00%</td>
      <td>732200 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>80</td>
      <td>Ma, Y.; Li, W.; Xing, R.; Li, P.; Liu, Z. Epitope-Imprinted Magnetic Nanoparticles as a General Platform for Efficient In Vitro Evolution of Protein-Binding Aptamers. ACS Sens. 2020 Jul 7; 5(8): 2537-2544. DOI: 10.1012/acssensors.0c00846</td>
      <td>dCp,dTp,dTp,dCp,dTp,dGp,dCp,dCp,dCp,dGp,dCp,dCp,dTp,dCp,dCp,dTp,dTp,dCp,dCp,dTp,dCp,dGp,dTp,dGp,dAp,dCp,dGp,dCp,dCp,dGp,dTp,dGp,dCp,dAp,dGp,dTp,dTp,dAp,dCp,dCp,dGp,dAp,dTp,dAp,dCp,dGp,dAp,dGp,dCp,dCp,dTp,dCp,dGp,dTp,dTp,dTp,dTp,dGp,dGp,dGp,dAp,dGp,dAp,dCp,dGp,dAp,dGp,dAp,dTp,dAp,dGp,dGp,dCp,dGp,dGp,dAp,dCp,dAp,dCp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>453</th>
      <td>VEGF receptor Flt-1 (F56)</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/239.png" width="120" ></td>
      <td>VEGF receptor Flt-1</td>
      <td>Cells</td>
      <td>N/A nM (reported value)</td>
      <td>Washing Buffer (150 mM NaCl and 10 mM phosphate buffer, pH 6.2) and then blocked in 3% BSA (25 mM HEPES, pH 7.6) for 3 hours at room temperature.</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>1759.02 g/mole</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Selectively binds VEGF receptor Flt-1</td>
      <td>12</td>
      <td>None</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>454</th>
      <td>Mouse Lung Vasculature</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/243.png" width="120" ></td>
      <td>Mouse Lung Vasculature</td>
      <td>Tissue</td>
      <td>N/A nM (reported value)</td>
      <td>None</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>1684.03 g/mole</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>13</td>
      <td>D Rajotte et al. "  Molecular Heterogeneity of the Vascular Endothelium Revealed by In Vivo  Phage Display." Journal of Clinical Investigations, 102(1998): 430-437.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>455</th>
      <td>Trypanosoma cruzi-Laminin Parasite Receptor (L28)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/165.png" width="120" ></td>
      <td>Trypanosoma cruz-Laminin Parasite Receptor</td>
      <td>Cells</td>
      <td>209 nM (reported value)</td>
      <td>50 uM RNA containing 20 mM glucose.</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>34263.1 g/mole</td>
      <td>46.73%</td>
      <td>1054100 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>107</td>
      <td>None</td>
      <td>rAp,fCp,fCp,rGp,rAp,rGp,fUp,fCp,fCp,rAp,rGp,rAp,rAp,rGp,fCp,fUp,fUp,rGp,fUp,rAp,rGp,fUp,rAp,fCp,fUp,fUp,rAp,fUp,rGp,fUp,fCp,fCp,fCp,rAp,fUp,fCp,rGp,rAp,rAp,fCp,rGp,fCp,rAp,rGp,fUp,rGp,fUp,rAp,fUp,fCp,fUp,fUp,rGp,fCp,rAp,fCp,fCp,rGp,rAp,fCp,fUp,fCp,fUp,fCp,fUp,rGp,fCp,fCp,fUp,rAp,rGp,rAp,fUp,rGp,rGp,rAp,rGp,fUp,fUp,rGp,rAp,rAp,fUp,fUp,fCp,fUp,fCp,fCp,fCp,fUp,rAp,fUp,rAp,rGp,fUp,rGp,rAp,rGp,fUp,fCp,rGp,fUp,rAp,fUp,fUp,rAp,fCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>456</th>
      <td>Trypanosoma cruzi-Thrombospondin Parasite Receptor (T6)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/167.png" width="120" ></td>
      <td>Trypanosoma cruzi-Thrombospondin Parasite Receptor</td>
      <td>Cells</td>
      <td>400 nM (reported value)</td>
      <td>50 uM RNA containing 20 mM glucose.</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>33037.62 g/mole</td>
      <td>49.51%</td>
      <td>1016600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>103</td>
      <td>None</td>
      <td>rAp,fCp,fCp,rGp,rAp,rGp,fUp,fCp,fCp,rAp,rGp,rAp,rAp,rGp,fCp,fUp,fUp,rGp,fUp,rAp,rGp,fUp,rAp,fCp,fUp,fCp,rAp,rAp,fUp,rAp,fCp,fUp,fCp,rAp,rAp,rGp,fCp,rAp,rAp,rGp,fCp,fCp,fCp,rAp,rGp,fCp,fCp,fCp,fUp,rAp,fCp,fCp,rAp,rAp,fCp,fCp,fCp,fCp,rGp,rGp,rAp,rGp,fCp,fCp,fUp,rAp,rGp,rAp,fUp,rGp,rGp,rAp,rGp,fUp,fUp,rGp,rAp,rAp,fUp,fUp,fCp,fUp,fCp,fCp,fCp,fUp,rAp,fUp,rAp,rGp,fUp,rGp,rAp,rGp,fUp,fCp,rGp,fUp,rAp,fUp,fUp,rAp,fCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>457</th>
      <td>PC-3 (G1)</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/251.png" width="120" ></td>
      <td>PC-3 Human Prostate Carcinoma</td>
      <td>Tissue</td>
      <td>N/A nM (reported value)</td>
      <td>None</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>1783.1 g/mole</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>15</td>
      <td>Newton et al. "In Vivo Selection of Phage for the Optical Imaging of PC-3 Human Prostate Carcinoma in Mice."</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>458</th>
      <td>nDNA-547</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7198.jpg" width="120" ></td>
      <td>nano-sized DNA</td>
      <td>Small Organic</td>
      <td>363 ± 135 nM nM (reported value)</td>
      <td>7 M urea, 10 mM EDTA, 300 mM NaOAc, pH 5.2</td>
      <td>26°C</td>
      <td>None</td>
      <td></td>
      <td>21887.2 g/mole</td>
      <td>51.39%</td>
      <td>666800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>72</td>
      <td>Ogawa, A., & Itoh, Y. (2020). In Vitro Selection of RNA Aptamers Binding to Nanosized DNA for Constructing Artificial Riboswitches. ACS Synthetic Biology. doi:10.1021/acssynbio.0c00384</td>
      <td>dAp,dCp,dCp,dAp,dCp,dAp,dAp,dCp,dGp,dGp,dTp,dTp,dTp,dCp,dCp,dCp,dAp,dGp,dGp,dAp,dCp,dGp,dGp,dAp,dTp,dGp,dCp,dTp,dTp,dCp,dCp,dGp,dTp,dAp,dAp,dTp,dGp,dAp,dCp,dGp,dCp,dTp,dTp,dCp,dCp,dCp,dAp,dTp,dTp,dAp,dCp,dCp,dCp,dGp,dTp,dCp,dCp,dAp,dAp,dTp,dCp,dAp,dAp,dCp,dTp,dCp,dAp,dAp,dCp,dTp,dTp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>459</th>
      <td>ATP C32</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/571.jpg" width="120" ></td>
      <td>ATP</td>
      <td>Small Organic</td>
      <td>27 µM (reported value)</td>
      <td>(50 mM Trisƒ??HCl, pH 7.5, 150 mM NaCl, 5 mM MgCl2) + 1 mg/ml tRNA and 0.1 mg/ml BSA</td>
      <td>24°C</td>
      <td>\n                                80C for 5 min, cool at 24C \n</td>
      <td>n/a</td>
      <td>33707.44 g/mole</td>
      <td>49.04%</td>
      <td>1065100 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>312</td>
      <td>Weill L., Louis D., and Sargueil B., 2004.Selection and Evolution of NTP-Specific Aptamers. Nucleic Acids Research, 2004 Vol. 32 No. 17, 5045-5058</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>460</th>
      <td>Tumor-derived endothelial cells (PepMW3)</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/383.png" width="120" ></td>
      <td>Tumor-derived endothelial cells (TDECs)</td>
      <td>Cells</td>
      <td>N/A nM (reported value)</td>
      <td>None</td>
      <td>4°C</td>
      <td>None</td>
      <td>None</td>
      <td>960.21 g/mole</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>9</td>
      <td>None</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>461</th>
      <td>R10C5</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7171.jpg" width="120" ></td>
      <td>Legionella pneumophila (Lp)</td>
      <td>Other</td>
      <td>116 nM (reported value)</td>
      <td>1X PBS with 0.1 mg/ml salmon sperm DNA, 1% bovine serum albumin, and 0.05% Tween 20</td>
      <td>25°C</td>
      <td>None</td>
      <td></td>
      <td>21213.8 g/mole</td>
      <td>50.72%</td>
      <td>658800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>69</td>
      <td>Saad M, Chinerman D, Tabrizian M, Faucher SP. Identification of two aptamers binding to Legionella pneumophila with high affinity and specificity. Sci Rep. 2020 Jun 4;10(1):9145. doi: 10.1038/s41598-020-65973-3. PMID: 32499557; PMCID: PMC7272621.</td>
      <td>dGp,dCp,dAp,dAp,dTp,dGp,dGp,dTp,dAp,dCp,dGp,dGp,dTp,dAp,dCp,dTp,dTp,dCp,dCp,dGp,dGp,dAp,dCp,dAp,dGp,dTp,dGp,dCp,dTp,dGp,dAp,dAp,dAp,dAp,dCp,dTp,dGp,dTp,dGp,dAp,dCp,dCp,dCp,dCp,dCp,dCp,dAp,dAp,dAp,dAp,dGp,dTp,dGp,dCp,dAp,dCp,dGp,dCp,dTp,dAp,dCp,dTp,dTp,dTp,dGp,dCp,dTp,dAp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>462</th>
      <td>Brain-homing peptide (PEPC7)</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/312.png" width="120" ></td>
      <td>Murine Brain</td>
      <td>Tissue</td>
      <td>N/A nM (reported value)</td>
      <td>None</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>1014.19 g/mole</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>This peptide is capable of penetrating the blood brain barrier and ushering through the attached bacteriophage.</td>
      <td>9</td>
      <td>None</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>463</th>
      <td>GVI-1</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7138.png" width="120" ></td>
      <td>GCRV-infected CIK</td>
      <td>Cells</td>
      <td>220.86 nM (reported value)</td>
      <td>Binding Buffer [10% FBS] from Life Technologies.</td>
      <td>4°C</td>
      <td>None</td>
      <td></td>
      <td>27920.1 g/mole</td>
      <td>52.22%</td>
      <td>874000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Grass Carp Reovirus Infected Cytokine-Induced Killer Cells (GCRV-infected CIK cells).\nGrass carp (Ctenopharyngon idellus) kidney cells (CIK).</td>
      <td>90</td>
      <td>Yu, Qing et al. “Generation and characterization of aptamers against grass carp reovirus infection for the development of rapid detection assay.” Journal of fish diseases vol. 44,1 (2021): 33-44. doi:10.1111/jfd.13265</td>
      <td>dGp,dTp,dCp,dTp,dGp,dAp,dAp,dGp,dTp,dAp,dGp,dAp,dCp,dGp,dCp,dAp,dGp,dGp,dAp,dGp,dGp,dGp,dGp,dTp,dGp,dTp,dAp,dGp,dCp,dTp,dCp,dGp,dTp,dTp,dAp,dTp,dGp,dAp,dTp,dTp,dCp,dGp,dGp,dAp,dCp,dAp,dAp,dGp,dAp,dCp,dTp,dTp,dAp,dCp,dCp,dTp,dTp,dGp,dCp,dGp,dCp,dCp,dTp,dCp,dTp,dGp,dGp,dGp,dAp,dTp,dAp,dGp,dTp,dCp,dAp,dCp,dAp,dCp,dCp,dTp,dGp,dAp,dGp,dTp,dAp,dAp,dGp,dCp,dGp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>464</th>
      <td>TGF-??1 (T18_1_3)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/58.png" width="120" ></td>
      <td>Transforming Growth Factor-??1 (TGF-??1)</td>
      <td>Protein</td>
      <td>94 nM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td></td>
      <td>23064.62 g/mole</td>
      <td>58.11%</td>
      <td>673000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>74</td>
      <td>Kang, Jonghoon, et al. "Combinatorial selection of a single stranded DNA thioaptamer targeting TGF-??1 protein." Bioorganic & Medicinal Chemistry Letters, 18 (2008): 1835-1839.</td>
      <td>dCp,dGp,dCp,dTp,dCp,dGp,dGp,dCp,dTp,dTp,dCp,dAp,dCp,dGp,dAp,dGp,dAp,dTp,dTp,dCp,dGp,dTp,dGp,dTp,dGp,dTp,dTp,dGp,dTp,dGp,dTp,dTp,dGp,dTp,dGp,dTp,dTp,dGp,dGp,dTp,dTp,dTp,dGp,dGp,dTp,dGp,dGp,dTp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>465</th>
      <td>VEGF receptor KDR and Flt-1 (v107)</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/240.png" width="120" ></td>
      <td>VEGF receptor KDR and Flt-1</td>
      <td>Cells</td>
      <td></td>
      <td>Binding Buffer (0.5% (v/v) Tween20, 0.5% BSA in PBS)</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>2606.94 g/mole</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>19</td>
      <td>None</td>
      <td></td>
      <td>0.6 µM (reported value)</td>
    </tr>
    <tr>
      <th>466</th>
      <td>Small-Cell Lung Cancer (HCH07)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/69.jpg" width="120" ></td>
      <td>Small-cell lung cancer (SCLC) cell-surface molecular markers</td>
      <td>Cells</td>
      <td>38 nM (reported value)</td>
      <td>None</td>
      <td>4°C</td>
      <td>None</td>
      <td></td>
      <td>21768.15 g/mole</td>
      <td>49.30%</td>
      <td>650200 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>71</td>
      <td>Chen, Hui, et al. "Molecular Recognition of Small-Cell Lung Cancer Cells Using Aptamers." CHEMMEDCHEM, 3 (2008): 991-1001.</td>
      <td>dTp,dAp,dCp,dCp,dAp,dGp,dTp,dGp,dCp,dGp,dAp,dTp,dGp,dCp,dTp,dCp,dAp,dGp,dGp,dCp,dCp,dGp,dAp,dTp,dGp,dTp,dCp,dAp,dAp,dCp,dTp,dTp,dTp,dTp,dTp,dCp,dTp,dAp,dAp,dCp,dTp,dCp,dAp,dCp,dTp,dGp,dGp,dTp,dTp,dTp,dTp,dGp,dCp,dCp,dTp,dGp,dAp,dCp,dGp,dCp,dAp,dTp,dTp,dCp,dGp,dGp,dTp,dTp,dGp,dAp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>467</th>
      <td>Hepatitis C Virus Dependent RNA Polymerase (B.2)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/170.png" width="120" ></td>
      <td>Hepatitis C Virus Dependent RNA Polymerase</td>
      <td>Protein</td>
      <td>1.5 nM (reported value)</td>
      <td>None</td>
      <td>23°C</td>
      <td>None</td>
      <td></td>
      <td>29744.87 g/mole</td>
      <td>67.39%</td>
      <td>868400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>92</td>
      <td>None</td>
      <td>rGp,rGp,rGp,rAp,rUp,rGp,rCp,rUp,rUp,rCp,rGp,rGp,rCp,rAp,rUp,rCp,rCp,rCp,rCp,rGp,rAp,rAp,rGp,rCp,rCp,rGp,rCp,rUp,rAp,rUp,rGp,rGp,rAp,rCp,rCp,rAp,rGp,rUp,rGp,rGp,rCp,rGp,rCp,rGp,rGp,rCp,rUp,rUp,rCp,rGp,rGp,rCp,rCp,rCp,rGp,rAp,rCp,rGp,rGp,rAp,rGp,rUp,rGp,rGp,rUp,rAp,rCp,rCp,rGp,rCp,rUp,rUp,rCp,rGp,rGp,rCp,rGp,rGp,rUp,rAp,rCp,rGp,rUp,rAp,rAp,rGp,rCp,rUp,rUp,rGp,rGp,rGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>468</th>
      <td>Bacillus anthracis spores (BAS-6F)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/487.png" width="120" ></td>
      <td>Bacillus anthracis spores</td>
      <td>Other</td>
      <td>N/A nM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>\n                                Heated to 95° C for 5 minutes and added hot. \n</td>
      <td>binds to bacillus antracis over anthrose sugar.</td>
      <td>22213.45 g/mole</td>
      <td>54.17%</td>
      <td>706300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>72</td>
      <td>None</td>
      <td>dAp,dTp,dAp,dCp,dGp,dGp,dGp,dAp,dGp,dCp,dCp,dAp,dAp,dCp,dAp,dCp,dCp,dAp,dTp,dCp,dCp,dCp,dTp,dCp,dTp,dTp,dAp,dGp,dGp,dAp,dTp,dAp,dCp,dAp,dAp,dAp,dGp,dCp,dCp,dAp,dAp,dAp,dCp,dTp,dGp,dAp,dGp,dCp,dCp,dCp,dGp,dTp,dGp,dCp,dAp,dGp,dAp,dGp,dCp,dAp,dGp,dGp,dTp,dGp,dTp,dGp,dAp,dCp,dGp,dGp,dAp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>469</th>
      <td>Interleukin 8 (8A-35)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/618.png" width="120" ></td>
      <td>Interleukin-8 (CXCL-8)</td>
      <td>Protein</td>
      <td>1.72 pM (reported value)</td>
      <td>30 mM Tris-HCl (pH 7.5), 150 mM NaCl, 1.5 mM MgCl2, 2mM DTT, and 1% BSA</td>
      <td>23°C</td>
      <td>\n                                N/A \n</td>
      <td>Counter Targets: CXCL-1 and CXCL-2</td>
      <td>11182.56 g/mole</td>
      <td>40.00%</td>
      <td>353800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>IL-8</td>
      <td>35</td>
      <td>Sung HJ, Choi S, Lee JW, Ok CY, Bae YS, Kim YH, Lee W, Heo K, Kim IH. Inhibition of human neutrophil activity by an RNA aptamer bound to interleukin-8. Biomaterials. 2014 Jan;35(1):578-89. doi: 10.1016/j.biomaterials.2013.09.107. Epub 2013 Oct 13.</td>
      <td>rGp,rGp,rGp,rGp,rGp,fCp,fUp,fUp,rAp,fUp,fCp,rAp,fUp,fUp,fCp,fCp,rAp,fUp,fUp,fUp,rAp,rGp,fUp,rGp,fUp,fUp,rAp,fUp,rGp,rAp,fUp,rAp,rAp,fCp,fCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>470</th>
      <td>Tobramycin (LGA11)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/433.png" width="120" ></td>
      <td>Tobramycin and other Aminoglycosides (AMGs)</td>
      <td>Small Organic</td>
      <td>19.2 nM (reported value)</td>
      <td>Binding Buffer: PBS with 10% (v/v) methanol</td>
      <td>RT°C</td>
      <td>None</td>
      <td>Binds to several aminoglycosides: kanamycin, paromycin streptomycin, apramycin, tobramycin, neomycin, and gentamycin.</td>
      <td>25568.23 g/mole</td>
      <td>56.96%</td>
      <td>771400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>87</td>
      <td>None</td>
      <td>rGp,rGp,rGp,rAp,rGp,rAp,rAp,rGp,rGp,fCp,rGp,rGp,fCp,rGp,fCp,rGp,fUp,rAp,rGp,rGp,fCp,rGp,rAp,rGp,fCp,fUp,fUp,fUp,rAp,rGp,rGp,fCp,fCp,fCp,rGp,rAp,fCp,rAp,fUp,fUp,fCp,fCp,fCp,fCp,fUp,rAp,rAp,rAp,rAp,rAp,rAp,rGp,fCp,fUp,fUp,rGp,fUp,fUp,fCp,fUp,rAp,fUp,fUp,fCp,rGp,fUp,rGp,rGp,rAp,fCp,rGp,rAp,fUp,rGp,fCp,rGp,fCp,rAp,fCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>471</th>
      <td>HER3 (A30)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/62.png" width="120" ></td>
      <td>HER3</td>
      <td>Protein</td>
      <td>45 nM (reported value)</td>
      <td>None</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>25442.28 g/mole</td>
      <td>59.49%</td>
      <td>761100 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>79</td>
      <td>None</td>
      <td>rGp,rAp,rAp,rUp,rUp,rCp,rCp,rGp,rCp,rGp,rUp,rGp,rUp,rGp,rCp,rCp,rAp,rGp,rCp,rGp,rAp,rAp,rAp,rGp,rUp,rUp,rGp,rCp,rGp,rUp,rAp,rUp,rGp,rGp,rGp,rUp,rCp,rAp,rCp,rAp,rUp,rCp,rGp,rCp,rAp,rGp,rGp,rCp,rAp,rCp,rAp,rUp,rGp,rUp,rCp,rAp,rUp,rCp,rUp,rGp,rGp,rGp,rCp,rGp,rGp,rUp,rCp,rCp,rGp,rUp,rUp,rCp,rGp,rGp,rGp,rAp,rUp,rCp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>472</th>
      <td>Abrin Toxin (TA6)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/80.png" width="120" ></td>
      <td>Abrin Toxin</td>
      <td>Protein</td>
      <td>~28 nM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>\n                                Denature at 94°C for 5 minutes, cool slowly  to room temperature. \n</td>
      <td></td>
      <td>10506.89 g/mole</td>
      <td>50.00%</td>
      <td>331300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>37</td>
      <td>Tang, J., et al. "In vitro selection of DNA aptamer against abrin toxin and aptamer-based abrin direct detection." Biosensors and Bioelectronics, 22 (2007): 2456-2463.</td>
      <td>dAp,dTp,dCp,dCp,dTp,dGp,dTp,dGp,dAp,dGp,dGp,dAp,dAp,dTp,dGp,dCp,dTp,dCp,dAp,dTp,dGp,dCp,dAp,dTp,dAp,dGp,dCp,dAp,dAp,dGp,dGp,dGp,dCp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>473</th>
      <td>Bfl-1 Anti-Apoptotoc Protein (Apt50)</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/412.png" width="120" ></td>
      <td>Bfl-1 Antio-Apoptotic Protein</td>
      <td>Protein</td>
      <td>N/A nM (reported value)</td>
      <td>N/A</td>
      <td>N/A°C</td>
      <td>None</td>
      <td>Does not interact with Bcl-2, Bcl-xL, or Mcl-1 in yeast.</td>
      <td>1601.02 g/mole</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>13</td>
      <td>None</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>474</th>
      <td>Thrombin 03</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/610.jpg" width="120" ></td>
      <td>Thrombin</td>
      <td>Protein</td>
      <td>7.03 pM (reported value)</td>
      <td>PBSMCT (DPBS with 2.5 mM MgCl2, 1 mM CaCl2, 0.05% TWEEN-20)</td>
      <td>23°C</td>
      <td>None</td>
      <td></td>
      <td>18794.24 g/mole</td>
      <td>55.00%</td>
      <td>585800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>60</td>
      <td>Wang, J., Gong, Q., Maheshwari, N., Eisenstein, M., Luz Arcila, M., Kosik, K., & Soh, T. Particle Display: A Quantitative Screening Method for Generating High-Affinity Aptamers. Angew. Chem. Int. Ed. 2014, 53, 4796ƒ??4801</td>
      <td>dCp,dAp,dGp,dCp,dGp,dCp,dTp,dAp,dGp,dGp,dGp,dCp,dTp,dTp,dTp,dTp,dAp,dGp,dCp,dGp,dTp,dAp,dAp,dTp,dGp,dGp,dGp,dTp,dAp,dGp,dGp,dGp,dTp,dGp,dGp,dTp,dGp,dCp,dGp,dGp,dTp,dGp,dCp,dAp,dGp,dAp,dTp,dAp,dTp,dCp,dGp,dGp,dAp,dAp,dTp,dTp,dGp,dGp,dTp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>475</th>
      <td>Insulin-Linked Polymorphic Region Two-Repeat (ILPR2)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/499.png" width="120" ></td>
      <td>Insulin</td>
      <td>Peptide</td>
      <td>N/A nM (reported value)</td>
      <td>Tris buffer (20 mM Tris, 5mM KCL, pH 7.3)</td>
      <td>RT°C</td>
      <td>None</td>
      <td>In capillary protein capture, this aptamer captured 5 times more insulin than thrombin.</td>
      <td>8911.82 g/mole</td>
      <td>71.43%</td>
      <td>283900 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>28</td>
      <td>None</td>
      <td>dAp,dCp,dAp,dGp,dGp,dGp,dGp,dTp,dGp,dTp,dGp,dGp,dGp,dGp,dAp,dCp,dAp,dGp,dGp,dGp,dGp,dTp,dGp,dTp,dGp,dGp,dGp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>476</th>
      <td>Daunomycin (10.10v)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/157.jpg" width="120" ></td>
      <td>Daunomycin</td>
      <td>Small Organic</td>
      <td>272 nM (reported value)</td>
      <td>Binding Buffer ((20 mM Trisƒ??HCl [pH 7.4], 140 mM NaCl, 5 mM KCl, 1 mM MgCl2, 1 mM CaCl2, and 0.05% [v/v]  Tween 20)</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>12873.4 g/mole</td>
      <td>58.54%</td>
      <td>405800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>41</td>
      <td>Wochner, et al. "A DNA aptamer with high affinity and specificity for therapeutic anthracyclines." Analytical Biochemistry, 373(2008): 34-42.</td>
      <td>dAp,dCp,dCp,dAp,dTp,dCp,dTp,dGp,dTp,dGp,dTp,dAp,dAp,dGp,dGp,dGp,dGp,dTp,dAp,dAp,dGp,dGp,dGp,dGp,dTp,dGp,dGp,dGp,dGp,dGp,dTp,dGp,dGp,dGp,dTp,dAp,dCp,dGp,dTp,dCp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>477</th>
      <td>S6-1</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7118.jpg" width="120" ></td>
      <td>human malignant glioma cell line SHG44</td>
      <td>Cells</td>
      <td>44 ± 15 nM (reported value)</td>
      <td>4.5 g/L of glucose, 0.01 M phosphate buffered saline, and 5 mM of MgCl2</td>
      <td>37 °C°C</td>
      <td>None</td>
      <td></td>
      <td>18474 g/mole</td>
      <td>46.67%</td>
      <td>583800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>60</td>
      <td>None</td>
      <td>dTp,dCp,dAp,dAp,dGp,dTp,dCp,dAp,dCp,dAp,dGp,dGp,dTp,dTp,dCp,dCp,dAp,dGp,dGp,dTp,dAp,dAp,dTp,dAp,dCp,dCp,dTp,dAp,dAp,dGp,dGp,dGp,dTp,dAp,dTp,dGp,dCp,dTp,dCp,dTp,dCp,dGp,dCp,dCp,dTp,dAp,dTp,dTp,dAp,dTp,dAp,dTp,dGp,dGp,dAp,dGp,dCp,dAp,dCp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>478</th>
      <td>Neurotensin receptor NTS-1 (P19)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/180.png" width="120" ></td>
      <td>Neurotensin receptor NTS-1</td>
      <td>Protein</td>
      <td>0.37 nM (reported value)</td>
      <td>Binding Buffer (SHMCK buffer containing 0.1% (w/v) HSA, 0.2% (w/v)  CHAPS, 0.04% (w/v) CHS, and 0.1% (w/v) LM)</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>22801.8 g/mole</td>
      <td>60.00%</td>
      <td>701900 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>70</td>
      <td>Daniels et al. "Generation of RNA Aptamers to the G-Protein-Coupled  Receptor for Neurotensin, NTS-1." Journal of Analytical Biochemistry, 305(2002): 214-226.</td>
      <td>rGp,rGp,rGp,rAp,rGp,rGp,rAp,fCp,rGp,rAp,fUp,rGp,fCp,rGp,rGp,rAp,fCp,rAp,rGp,rAp,fUp,rAp,fCp,rGp,rGp,rAp,rAp,fCp,fUp,rAp,fCp,rAp,rGp,rAp,rGp,rGp,fUp,fCp,rAp,rAp,fUp,fUp,rAp,fCp,rGp,rGp,fUp,rGp,rGp,fCp,fCp,rAp,fCp,rGp,fCp,rAp,rGp,rAp,fCp,rGp,rAp,fCp,fUp,fCp,rGp,fCp,fCp,fCp,rGp,rAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>479</th>
      <td>Uranium DNAzyme</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/598.jpg" width="120" ></td>
      <td>Uranium</td>
      <td>Other</td>
      <td>n/a nM (reported value)</td>
      <td>0.4 M NaCl and 20 mM MES buffer solution (pH 5.5) at 20?C for 15 min.     To quench: 5 uL of 0.5 M Tris base solution added then sample was vortexed to shift the pH from 5.5 to 8.0.</td>
      <td>20°C</td>
      <td>\n                                1.0 mL 9.6 ?? 10-5 M DNAzyme   1.0 mL 6.4 ?? 10-5 M TAMRA-modified substrate  Mixed with 1.0 mL of 2.0 M NaCl and 1.0 mL of 20 mM MES buffer solution (pH 5.5).  Heat at 90?C for 15 min and cooled down t \n</td>
      <td>Tested against Barium, Calcium, Cadmium, Chromium, Iron 2+, Iron 3+, Mercury, Magnesium, Manganese, Nickel, and Lead. Even in 100-fold excess concentrations over that used for Uranium, fluorescence intensity did not increase significantly, except in the presence of Uranium.</td>
      <td>16630.84 g/mole</td>
      <td>51.85%</td>
      <td>524500 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Limit of Detection: 0.41 nM Uranium    Substrate Sequence:  TAMRA-CCTC ACTAT rA GGAAG AGATG GACGT G</td>
      <td>171</td>
      <td>Sai Jin Xiao, Jun Zuo, Zhi Qiang Zhu, Yong Zhong Ouyang, Xing Lei Zhang, Huan Wen Chen, & Li Zhang (2015) Highly Sensitive DNAzyme Sensor for Selective Detection of Trace Uranium in Ore and Natural Water Samples. Sensors and Actuators B 210 (2015) 656-660</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>480</th>
      <td>IgE (4.4.12)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/505" width="120" ></td>
      <td>IgE</td>
      <td>Protein</td>
      <td>23 nM (reported value)</td>
      <td>None</td>
      <td>25°C</td>
      <td>None</td>
      <td>Selectively bind human IgE over human IgG and mouse IgE.</td>
      <td>24050.67 g/mole</td>
      <td>47.44%</td>
      <td>753800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>The selectivity was not selected for, but rather a byproduct of the high affinity for human IgE.</td>
      <td>78</td>
      <td>None</td>
      <td>dAp,dGp,dCp,dAp,dGp,dCp,dAp,dCp,dAp,dGp,dAp,dGp,dGp,dTp,dCp,dAp,dGp,dAp,dTp,dGp,dTp,dGp,dAp,dAp,dAp,dCp,dAp,dTp,dAp,dGp,dCp,dAp,dTp,dAp,dTp,dTp,dTp,dAp,dCp,dTp,dTp,dAp,dTp,dGp,dTp,dCp,dGp,dCp,dCp,dTp,dTp,dGp,dCp,dCp,dGp,dGp,dTp,dTp,dCp,dCp,dTp,dAp,dTp,dGp,dCp,dGp,dTp,dGp,dCp,dTp,dAp,dCp,dCp,dGp,dTp,dGp,dAp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>481</th>
      <td>Tumour Marker MUC1 (S1.6)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/76.jpg" width="120" ></td>
      <td>Tumour Marker MUC</td>
      <td>Peptide</td>
      <td>0.2131 nM (reported value)</td>
      <td>Binding Buffer (0.2 M  NaHCO3 and 0.5 M NaCl (pH 7.4))</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>22336.6 g/mole</td>
      <td>48.61%</td>
      <td>737600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Kd values given in Table 2 of reference article may have been miscalculated. Using the kd and ka given in Table 2 of the reference, we have calculated the above Kd value from Kd=(kd/ka).</td>
      <td>72</td>
      <td>Ferreira, C., et al. "DNA Aptamers That Bi nd to MUC1 Tumour Marker: Design and Characterization of MUC1-Binding Single-Stranded DNA Aptamers." Tumor Biology, 27(2006): 289ƒ??301.</td>
      <td>dGp,dGp,dGp,dAp,dGp,dAp,dCp,dAp,dAp,dGp,dAp,dAp,dTp,dAp,dAp,dAp,dCp,dGp,dCp,dTp,dCp,dAp,dAp,dGp,dCp,dAp,dAp,dCp,dAp,dGp,dGp,dGp,dTp,dAp,dTp,dCp,dCp,dAp,dAp,dAp,dGp,dGp,dAp,dTp,dCp,dAp,dAp,dAp,dTp,dTp,dCp,dGp,dAp,dCp,dAp,dGp,dGp,dAp,dGp,dGp,dCp,dTp,dCp,dAp,dCp,dAp,dAp,dCp,dAp,dGp,dGp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>482</th>
      <td>YPEN-1 Endothelial Cells (III.1)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/210.png" width="120" ></td>
      <td>YPEN-1 Endothelial Cells</td>
      <td>Cells</td>
      <td>N/A nM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td></td>
      <td>29854.53 g/mole</td>
      <td>39.58%</td>
      <td>961400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>96</td>
      <td>None</td>
      <td>dAp,dTp,dAp,dCp,dCp,dAp,dGp,dCp,dTp,dTp,dAp,dTp,dTp,dCp,dAp,dAp,dTp,dTp,dAp,dGp,dGp,dCp,dGp,dGp,dTp,dGp,dCp,dAp,dTp,dTp,dGp,dTp,dGp,dGp,dTp,dTp,dGp,dGp,dTp,dAp,dGp,dTp,dAp,dTp,dAp,dCp,dAp,dTp,dGp,dAp,dGp,dGp,dTp,dTp,dTp,dGp,dGp,dTp,dTp,dGp,dAp,dGp,dAp,dCp,dTp,dAp,dGp,dTp,dCp,dGp,dCp,dAp,dAp,dGp,dAp,dTp,dAp,dTp,dAp,dGp,dAp,dTp,dAp,dGp,dTp,dAp,dAp,dGp,dTp,dGp,dCp,dAp,dAp,dTp,dCp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>483</th>
      <td>Aptamer Beacon</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7231.jpg" width="120" ></td>
      <td>adenosine</td>
      <td>Protein</td>
      <td>6.0 nM (reported value)</td>
      <td>150 mmol L−1 NaCl, 1 mmol L−1 MgCl2, 5 mmol L−1 KCl and Tris\nbuffer of 20 mmol L−1 (pH 7.4)</td>
      <td>37 °C</td>
      <td>None</td>
      <td></td>
      <td>8459.31 g/mole</td>
      <td>59.26%</td>
      <td>279432 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>27</td>
      <td>Zhang JQ, Wang YS, Xue JH, He Y, Yang HX, Liang J, Shi LF, Xiao XL. A gold nanoparticles-modified aptamer beacon for urinary adenosine detection based on structure-switching/fluorescence-"turning on" mechanism. J Pharm Biomed Anal. 2012 Nov;70:362-8. doi: 10.1016/j.jpba.2012.05.032. Epub 2012 Jun 1. PMID: 22717140.\n</td>
      <td>dAp,dCp,dCp,dTp,dGp,dGp,dGp,dGp,dGp,dAp,dGp,dTp,dAp,dTp,dTp,dGp,dCp,dGp,dGp,dAp,dGp,dGp,dAp,dAp,dGp,dGp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>484</th>
      <td>Unphosphorylated ERK2</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/264.jpg" width="120" ></td>
      <td>Unphosphorylated ERK2</td>
      <td>Protein</td>
      <td></td>
      <td>None</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>24170.59 g/mole</td>
      <td>57.33%</td>
      <td>732700 L/(mole·cm)</td>
      <td></td>
      <td>2 nM (reported value)</td>
      <td>None</td>
      <td>83</td>
      <td>Vaish et al. "Monitoring post-translational modifications of proteins with allosteric ribozymes." Nature Biotechnology, 20(2002): 810-815.</td>
      <td>rGp,rGp,rCp,rGp,rUp,rGp,rAp,rCp,rCp,rUp,rGp,rAp,rUp,rGp,rAp,rGp,rUp,rCp,rAp,rCp,rGp,rCp,rUp,rAp,rAp,rGp,rGp,rAp,rGp,rGp,rAp,rUp,rUp,rUp,rCp,rCp,rGp,rAp,rAp,rAp,rGp,rCp,rGp,rGp,rCp,rUp,rAp,rCp,rGp,rGp,rUp,rCp,rCp,rGp,rCp,rCp,rAp,rGp,rUp,rGp,rUp,rUp,rAp,rCp,rGp,rAp,rAp,rAp,rCp,rGp,rUp,rUp,rCp,rCp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>485</th>
      <td>Ampicillin (AMP4)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/489.png" width="120" ></td>
      <td>Ampicillin</td>
      <td>Small Organic</td>
      <td>9.4 nM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>\n                                Heated at 90°C for 3 min and then cooled at 4°C for 1 hour. \n</td>
      <td>Only AMP17 was evaluated for specificity.</td>
      <td>6519.26 g/mole</td>
      <td>71.43%</td>
      <td>196500 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>21</td>
      <td>None</td>
      <td>dCp,dAp,dCp,dGp,dGp,dCp,dAp,dTp,dGp,dGp,dTp,dGp,dGp,dGp,dCp,dGp,dTp,dCp,dGp,dTp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>486</th>
      <td>Plasminogen Activator Inhibitor-1 (paionap-5)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/464.png" width="120" ></td>
      <td>Plasminogen Activator Inhibitor-1</td>
      <td>Protein</td>
      <td>1.58 nM (reported value)</td>
      <td>HBS buffer (pH 7.4), 2mM MgCl2, 0.05% Tween 20.</td>
      <td>N/A°C</td>
      <td>None</td>
      <td>E.Coli active PAI-1 variant</td>
      <td>25425.08 g/mole</td>
      <td>50.63%</td>
      <td>779600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>79</td>
      <td>Madsen, J.B., et al. "RNA Aptamers as Conformational Probes and Regulatory Agents for Plasminogen Activator Inhibitor-1." Biochemistry, 49, (2010), 4103-4115.</td>
      <td>rGp,rGp,rGp,rGp,fCp,fCp,rAp,fCp,fCp,rAp,rAp,fCp,rGp,rAp,fCp,rAp,fUp,fUp,rGp,rAp,rAp,fCp,fCp,rAp,fCp,rGp,fUp,rAp,rGp,rGp,fCp,fUp,fCp,rGp,fUp,fUp,fUp,fCp,fUp,rGp,rAp,rGp,fCp,fCp,rGp,rAp,fUp,fCp,fUp,fCp,rGp,rAp,fUp,rGp,fUp,fUp,rGp,rAp,fUp,rAp,fUp,rAp,rAp,rAp,fUp,rAp,rGp,fUp,rGp,fCp,fCp,fCp,rAp,fUp,rGp,rGp,rAp,fUp,fCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>487</th>
      <td>Hepatitis C NS3 Protein (10G-1)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/130.jpg" width="120" ></td>
      <td>Hepatitis C NS3 Protein</td>
      <td>Protein</td>
      <td>990 pM (reported value)</td>
      <td>None</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>21495.01 g/mole</td>
      <td>56.06%</td>
      <td>665000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>An assay of the inhibition  of helicase activity by the aptamer showed an IC50 of 12.5 nM.</td>
      <td>68</td>
      <td>Urvil, P., et al. "Selection of RNA  aptamers that bind specifically to the NS3 protease of hepatitis C virus." European Journal of Biochemistry, 248(1997):130-138.</td>
      <td>rGp,rGp,rGp,rAp,rGp,rAp,rGp,rCp,rGp,rGp,rAp,rAp,rGp,rCp,rGp,rUp,rGp,rCp,rUp,rGp,rGp,rGp,rCp,rCp,rAp,rGp,rUp,rAp,rGp,rUp,rGp,rUp,rAp,rUp,rAp,rGp,rGp,rGp,rCp,rUp,rCp,rGp,rAp,rAp,rAp,rUp,rGp,rUp,rUp,rCp,rAp,rUp,rGp,rGp,rCp,rUp,rCp,rAp,rGp,rUp,rGp,rGp,rAp,rCp,rAp,rUp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>488</th>
      <td>Neuropeptide Y (Clone 4.31)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/440.png" width="120" ></td>
      <td>Neuropeptide Y (NPY)</td>
      <td>Peptide</td>
      <td>300 nM (reported value)</td>
      <td>Capillary Electrophoresis Separation Buffer: 20 mM Hepes (pH 7.3) with 10 mM NaCl</td>
      <td>RT°C</td>
      <td>None</td>
      <td>The specificity for neuropeptide Y over the similar human pancreatic polypeptide (hPP; ~50% sequence homology)was 42-fold.</td>
      <td>24737.1 g/mole</td>
      <td>51.25%</td>
      <td>783500 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>80</td>
      <td>None</td>
      <td>dAp,dGp,dCp,dAp,dGp,dCp,dAp,dCp,dAp,dGp,dAp,dGp,dGp,dTp,dCp,dAp,dGp,dAp,dTp,dGp,dCp,dAp,dAp,dAp,dCp,dCp,dAp,dCp,dAp,dGp,dCp,dCp,dTp,dGp,dAp,dGp,dTp,dGp,dGp,dTp,dTp,dAp,dGp,dCp,dGp,dTp,dAp,dTp,dGp,dTp,dCp,dAp,dTp,dTp,dTp,dAp,dCp,dGp,dGp,dAp,dCp,dCp,dTp,dAp,dTp,dGp,dCp,dGp,dTp,dGp,dCp,dTp,dAp,dCp,dCp,dGp,dTp,dGp,dAp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>489</th>
      <td>CD63-1</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7154.jpg" width="120" ></td>
      <td>CD63</td>
      <td>Protein</td>
      <td>38.71 nM (reported value)</td>
      <td>pH 7.4 PBS containing 2.0 mM MgCl2, 0.02% Tween 20, and 100 µg/mL\nSalmon Sperm DNA</td>
      <td>22°C</td>
      <td>None</td>
      <td></td>
      <td>12998.4 g/mole</td>
      <td>59.52%</td>
      <td>411800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>42</td>
      <td>Song Z, Mao J, Barrero RA, Wang P, Zhang F, Wang T. Development of a CD63 Aptamer for Efficient Cancer Immunochemistry and Immunoaffinity-Based Exosome Isolation. Molecules. 2020 Nov 27;25(23):5585. doi: 10.3390/molecules25235585. PMID: 33261145; PMCID: PMC7730289.</td>
      <td>dTp,dAp,dAp,dCp,dAp,dCp,dGp,dAp,dCp,dAp,dGp,dAp,dCp,dGp,dTp,dTp,dCp,dGp,dGp,dAp,dGp,dGp,dTp,dCp,dGp,dAp,dAp,dCp,dCp,dCp,dTp,dGp,dAp,dCp,dAp,dGp,dCp,dGp,dTp,dGp,dGp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>490</th>
      <td>Ramos Cells (TD05)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/273.png" width="120" ></td>
      <td>Ramos Cells (B-cell lymphoma cell line)</td>
      <td>Cells</td>
      <td>74.7 nM (reported value)</td>
      <td>Wash Buffer (4.5 g/L glucose and 5 mM MgCl2 in Dulbeccoƒ??s PBS).    Binding Buffer (yeast tRNA (0.1 mg/mL) and BSA (1 mg/mL) into the wash buffer).    On ice.</td>
      <td>N/A°C</td>
      <td>\n                                Denature by heating to 95°  for 5 minutes. Then, cool on ice for 10 minutes. \n</td>
      <td>TD05 was specific for Ramos cells relative to CCRF-CEM and Toledo cells</td>
      <td>14847.64 g/mole</td>
      <td>60.42%</td>
      <td>451600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>48</td>
      <td>Tang et al. "Selection of Aptamers for Molecular Recognition  and Characterization of Cancer Cells." Analytical Chemistry, 79(2007): 4900-4907.</td>
      <td>dAp,dAp,dCp,dAp,dCp,dCp,dGp,dTp,dGp,dGp,dAp,dGp,dGp,dAp,dTp,dAp,dGp,dTp,dTp,dCp,dGp,dGp,dTp,dGp,dGp,dCp,dTp,dGp,dTp,dTp,dCp,dAp,dGp,dGp,dGp,dTp,dCp,dTp,dCp,dCp,dTp,dCp,dCp,dCp,dGp,dGp,dTp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>491</th>
      <td>HIV Reverse Transcriptase (PF1)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/416.png" width="120" ></td>
      <td>HIV RT</td>
      <td>Protein</td>
      <td>80 nM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td></td>
      <td>9326.14 g/mole</td>
      <td>46.67%</td>
      <td>300100 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>30</td>
      <td>None</td>
      <td>dAp,dGp,dGp,dAp,dAp,dGp,dGp,dCp,dTp,dTp,dTp,dAp,dGp,dGp,dTp,dCp,dTp,dGp,dAp,dGp,dAp,dTp,dCp,dTp,dCp,dGp,dGp,dAp,dAp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>492</th>
      <td>TEPP</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7201.png" width="120" ></td>
      <td>bEnd.3 and MDA-MB-231</td>
      <td>Cells</td>
      <td>110-85.6 nM (reported value)</td>
      <td>PBS with 1mM MgCl2, 0.1 mg/mL tRNA, 1mg/mL bovine serum albumin, and 10% FCS</td>
      <td>37°C</td>
      <td>\n                                Mfold \n</td>
      <td></td>
      <td>10266.58 g/mole</td>
      <td>69.70%</td>
      <td>304900 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>BIFUNCTIONAL aptamer. Affinity value is not a range. First value is affinity with bEnd.3, second value is affinity with MDA</td>
      <td>33</td>
      <td>Macdonald, J., Denoyer, D., Henri, J., Jamieson, A., Burvenich, I. J., Pouliot, N., & Shigdar, S. (2020). Bifunctional aptamer–doxorubicin conjugate crosses the blood–brain barrier and selectively delivers its payload to EpCAM-positive tumor cells. Nucleic acid therapeutics, 30(2), 117-128.</td>
      <td>dGp,dCp,dGp,dCp,dGp,dGp,dTp,dAp,dCp,dCp,dGp,dCp,dGp,dCp,dTp,dAp,dAp,dCp,dGp,dGp,dAp,dGp,dGp,dTp,dTp,dGp,dCp,dGp,dTp,dCp,dCp,dGp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>493</th>
      <td>Epithelial Cell Adhesion Molecule (EpCAM) SYL3C</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/628.jpg" width="120" ></td>
      <td>Cancer Cells</td>
      <td>Cells</td>
      <td>38 nM (reported value)</td>
      <td>1 x PBS (pH-7.4) and 5 mM MgCl2</td>
      <td>37°C</td>
      <td>\n                                N/A \n</td>
      <td>38 nM against MDA-MB-231 breast cancer cell line and 38 nM against Kato III human gastric carcinoma cell line</td>
      <td>13916.04 g/mole</td>
      <td>60.00%</td>
      <td>418900 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>45</td>
      <td>Selection of DNA Aptamers against Epithelial Cell Adhesion Molecule for Cancer Cell Imaging and Circulating Tumor Cell Capture  Yanling Song, Zhi Zhu, Yuan An, Weiting Zhang, Huimin Zhang, Dan Liu, Chundong Yu, Wei Duan, and Chaoyong James Yang  Analytical Chemistry 2013 85 (8), 4141-4149  DOI: 10.1021/ac400366b</td>
      <td>dCp,dAp,dCp,dTp,dAp,dCp,dAp,dGp,dAp,dGp,dGp,dTp,dGp,dTp,dCp,dTp,dGp,dTp,dCp,dCp,dCp,dAp,dCp,dGp,dTp,dTp,dGp,dTp,dCp,dAp,dTp,dGp,dGp,dGp,dGp,dGp,dGp,dTp,dTp,dGp,dGp,dCp,dCp,dTp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>494</th>
      <td>L-vasopressin</td>
      <td>L-DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/455.png" width="120" ></td>
      <td>L-vasopressin/arginine vasopressin/AVP/ADH</td>
      <td>Peptide</td>
      <td>1.2 µM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td>Does not bind to the D-enantiomer of vassopressin and there is at least a 100-fold decrease in binding with the similar molecular oxytocin.</td>
      <td>44304.78 g/mole</td>
      <td>0.00%</td>
      <td>527800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>61</td>
      <td>None</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>495</th>
      <td>GRK2 (C13.28)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/314.jpg" width="120" ></td>
      <td>G protein-coupled receptor kinase 2</td>
      <td>Protein</td>
      <td>1.2 nM (reported value)</td>
      <td>Binding Buffer 20 mM Tris (pH 7.0), 50 mM NaCl, 5 mM MgCl2,2 mM DTT, and 0.1% Lubrol (without nonspecific RNA)</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>9062.59 g/mole</td>
      <td>57.14%</td>
      <td>282300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Activity IC50 - 11 nM  Binding IC50 - 1.5 nM</td>
      <td>28</td>
      <td>Tesmer et al., Molecular Mechanism for Inhibition of G Protein-Coupled Receptor Kinase 2 by a Selective RNA Aptamer, Structure (2012), doi:10.1016/j.str.2012.05.002</td>
      <td>rGp,rGp,rCp,rAp,rGp,rAp,rCp,rCp,rAp,rUp,rAp,rCp,rGp,rGp,rGp,rAp,rGp,rAp,rGp,rAp,rAp,rAp,rCp,rUp,rUp,rGp,rCp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>496</th>
      <td>A1</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/557.png" width="120" ></td>
      <td>Staphylococcus aureus enterotoxin A (SEA)</td>
      <td>Protein</td>
      <td>92.17 nM (reported value)</td>
      <td>20 mM Tris, 100 mM NaCl, 5 mM KCl, 1 mM CaCl2 and 1 mM MgCl2 ƒ?› 6H2O, pH 7.4</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>24607.02 g/mole</td>
      <td>48.75%</td>
      <td>771100 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>ssDNA aptamers specific for SEA were selected using a 12-round SELEX procedure</td>
      <td>240</td>
      <td>Huang, Y. et al. "Selection, identification and application of a DNA aptamer against Staphylococcus aureus enterotoxin A." Anal. Methods, 2014, 6, 690-697.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>497</th>
      <td>Kanamycin (LS13)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/435.png" width="120" ></td>
      <td>Kanamycin and other Aminoglycosides (AMGs)</td>
      <td>Small Organic</td>
      <td>69.9 nM (reported value)</td>
      <td>Binding Buffer: PBS with 10% (v/v) methanol</td>
      <td>RT°C</td>
      <td>None</td>
      <td>Binds to several aminoglycosides: kanamycin, paromycin streptomycin, apramycin, tobramycin, neomycin, and gentamycin.</td>
      <td>25474.89 g/mole</td>
      <td>46.84%</td>
      <td>788000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>87</td>
      <td>None</td>
      <td>rGp,rGp,rGp,rAp,rGp,rAp,rAp,rGp,rGp,fCp,rGp,rGp,fCp,rGp,fCp,rGp,fUp,rAp,rGp,rGp,fCp,rGp,rAp,rGp,fCp,fUp,fUp,fUp,rAp,fCp,fCp,rAp,rGp,fUp,fUp,fUp,fUp,rAp,fUp,fUp,fUp,rGp,fUp,fUp,fUp,fUp,rAp,fUp,fUp,rGp,fUp,fUp,rAp,fUp,rAp,fUp,rGp,fCp,fUp,fUp,rAp,fUp,fUp,fCp,rGp,fUp,rGp,rGp,rAp,fCp,rGp,rAp,fUp,rGp,fCp,rGp,fCp,rAp,fCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>498</th>
      <td>Malaria.1</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7110.jpg" width="120" ></td>
      <td>Plasmodium falciparum</td>
      <td>Cells</td>
      <td>60 nM (reported value)</td>
      <td>Aptamer Binding buffer [AB buffer, RPMI-1640 (pH 7.2) supplemented with BSA (1 mg/mL, Sigma, St Louis, MO), yeast tRNA (0.1 mg/mL, Invitrogen), sheared salmon sperm DNA (0.1 mg/mL, Invitrogen)]</td>
      <td>RT°C</td>
      <td>\n                                Reactions were amplified for 13–20 cycles of 0.5 min at 94 °C, 0.5 min at 56 °C, and 0.5 min at 72 °C, followed by 5 min at 72 °C \n</td>
      <td></td>
      <td>13891.09 g/mole</td>
      <td>46.67%</td>
      <td>483970 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>For alkaline denaturation-based SELEX, the strand separation after PCR was performed with streptavidin-coated magnetic beads (M-270 Dynabeads, Invitrogen, Carlsbad, CA) according to manufacturer’s instructions. This brand was chosen because many groups working with SELEX commonly use these beads. Briefly, PCR product was incubated for 25 minutes on a rotator with 5 mg beads. After 2 washes with binding and washing buffer (10 mM Tris–HCl pH 7.5, 1 mM EDTA, 0.2 M NaCl) an alkaline denaturation was performed with 500 μL freshly prepared 200 mM NaOH to melt the DNA. The tubes were applied to a magnet and the eluted ssDNA was desalted with a Sephadex G-25 column (NAP-5, GE Healthcare, UK). In the lambda SELEX experiment, PCR products featuring a phosphorylated antisense strand were treated with 5 units of lambda exonuclease for 30 min at 37 °C to digest the antisense strand. The regenerated single-stranded aptamers were isolated by ethanol precipitation for the next round of counter-selection, selection, and amplification.</td>
      <td>45</td>
      <td>Oteng, Eugene, et al. "High-efficiency enrichment enables identification of aptamers to circulating Plasmodium falciparum-infected erythrocytes" Scientific Reports 10, 9706</td>
      <td>dCp,dCp,dAp,dAp,dAp,dCp,dGp,dCp,dGp,dCp,dTp,dGp,dGp,dTp,dTp,dTp,dGp,dGp,dAp,dAp,dTp,dGp,dCp,dTp,dTp,dAp,dGp,dAp,dGp,dTp,dTp,dTp,dGp,dAp,dTp,dCp,dGp,dAp,dTp,dGp,dCp,dCp,dTp,dAp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>499</th>
      <td>4-1BB 07</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/613.jpg" width="120" ></td>
      <td>4-1BB Ligand</td>
      <td>Protein</td>
      <td>2.32 nM (reported value)</td>
      <td>PBSMCT (DPBS with 2.5 mM MgCl2, 1 mM CaCl2, 0.05% TWEEN-20)</td>
      <td>23°C</td>
      <td>None</td>
      <td></td>
      <td>18793.26 g/mole</td>
      <td>51.67%</td>
      <td>600100 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>60</td>
      <td>Wang, J., Gong, Q., Maheshwari, N., Eisenstein, M., Luz Arcila, M., Kosik, K., & Soh, T. Particle Display: A Quantitative Screening Method for Generating High-Affinity Aptamers. Angew. Chem. Int. Ed. 2014, 53, 4796ƒ??4801</td>
      <td>dGp,dTp,dCp,dAp,dGp,dAp,dTp,dTp,dCp,dCp,dAp,dCp,dTp,dAp,dTp,dAp,dGp,dTp,dAp,dGp,dGp,dTp,dTp,dGp,dGp,dGp,dTp,dAp,dGp,dGp,dGp,dTp,dGp,dGp,dTp,dCp,dGp,dCp,dAp,dGp,dTp,dGp,dGp,dAp,dTp,dGp,dAp,dTp,dAp,dTp,dGp,dTp,dCp,dGp,dTp,dAp,dGp,dGp,dGp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>500</th>
      <td>Isocarbophos (SS2-55)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/294.png" width="120" ></td>
      <td>Isocarbophos</td>
      <td>Small Organic</td>
      <td>0.83 µM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td>This aptamer binds preferentially to the organophosphates phorate, profenofos, benzoate and omethoate. It has reduced binding to imidacloprid, acetamiprid, chlorpyrifos, methamidophos, dichlorvos, parathion, dimethoate, monocrotophos and phoxim.</td>
      <td>16990.09 g/mole</td>
      <td>49.09%</td>
      <td>526900 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>"A molecular beacon (5' FAM-CTGCACAAGAATCGCTGCAG-3') was designed, which was partly complementary with the middle constant nucleotides-sequence of the selected ssDNAs."</td>
      <td>55</td>
      <td>None</td>
      <td>dAp,dAp,dGp,dCp,dTp,dTp,dGp,dCp,dTp,dTp,dTp,dAp,dTp,dAp,dGp,dCp,dCp,dTp,dGp,dCp,dAp,dGp,dCp,dGp,dAp,dTp,dTp,dCp,dTp,dTp,dGp,dAp,dTp,dCp,dGp,dGp,dAp,dAp,dAp,dAp,dGp,dGp,dCp,dTp,dGp,dAp,dGp,dAp,dGp,dCp,dTp,dAp,dCp,dGp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>501</th>
      <td>U19</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/523.jpg" width="120" ></td>
      <td>U87-EGFRvIII</td>
      <td>Cells</td>
      <td>16.7 nM (reported value)</td>
      <td>0.45 g of glucose, 10 mg yeast tRNA, 5 mg salmon sperm DNA, 0.1 g BSA and 0.5 ml of 1 M MgCl2 in 100 ml of 1X phosphate-buffered saline [PBS]</td>
      <td>°C</td>
      <td>None</td>
      <td></td>
      <td>23496.35 g/mole</td>
      <td>40.79%</td>
      <td>730300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>228</td>
      <td>Wu, X. et al. "Cell-SELEX Aptamer for Highly Specific Radionuclide Molecular Imaging of Glioblastoma In Vivo." PLOS One 2014, 9:3.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>502</th>
      <td>Dopamine (dopa1.30/c.30)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/402.png" width="120" ></td>
      <td>Dopamine</td>
      <td>Small Organic</td>
      <td>N/A nM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td></td>
      <td>18784.41 g/mole</td>
      <td>60.34%</td>
      <td>567600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>65</td>
      <td>Mannironi, C., et al. "In Vitro Selection of Dopamine RNA Ligands." Biochemistry, 36 (1997):  9726-9734.</td>
      <td>rGp,rUp,rCp,rUp,rCp,rUp,rGp,rUp,rGp,rUp,rGp,rCp,rGp,rCp,rCp,rAp,rGp,rAp,rGp,rAp,rAp,rCp,rAp,rCp,rUp,rGp,rGp,rGp,rGp,rCp,rAp,rGp,rAp,rUp,rAp,rUp,rGp,rGp,rGp,rCp,rCp,rAp,rGp,rCp,rAp,rCp,rAp,rGp,rAp,rAp,rUp,rGp,rAp,rGp,rGp,rCp,rCp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>503</th>
      <td>Zn2+ (AR1)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/207.png" width="120" ></td>
      <td>Zn2+</td>
      <td>Small Organic</td>
      <td>~100 µM (reported value)</td>
      <td>None</td>
      <td>23°C</td>
      <td>None</td>
      <td></td>
      <td>31742.3 g/mole</td>
      <td>54.08%</td>
      <td>985700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>98</td>
      <td>Seetharaman et al. "Immobolized RNA switches for the analysis of complex chemical and biological mixtures." Nature Biotechnology, 19(2001): 336-341.</td>
      <td>rGp,rGp,rAp,rUp,rAp,rAp,rUp,rAp,rGp,rCp,rCp,rGp,rUp,rAp,rGp,rGp,rUp,rUp,rGp,rCp,rGp,rAp,rAp,rAp,rGp,rCp,rGp,rAp,rCp,rCp,rCp,rUp,rGp,rAp,rUp,rGp,rAp,rGp,rAp,rAp,rAp,rGp,rCp,rCp,rAp,rAp,rAp,rGp,rCp,rCp,rGp,rUp,rAp,rGp,rCp,rGp,rCp,rAp,rGp,rAp,rUp,rGp,rAp,rUp,rCp,rUp,rCp,rGp,rCp,rCp,rAp,rUp,rCp,rAp,rGp,rUp,rAp,rCp,rCp,rGp,rAp,rAp,rAp,rCp,rGp,rGp,rUp,rAp,rGp,rCp,rGp,rAp,rGp,rAp,rGp,rCp,rUp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>504</th>
      <td>Lup an 1 allergen β-conglutin (SGQ)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/420.png" width="120" ></td>
      <td>β-conglutin</td>
      <td>Protein</td>
      <td>0.58 nM (reported value)</td>
      <td>None</td>
      <td>22°C</td>
      <td>None</td>
      <td>Showed no cross-reactivity with gliadin or γ-conglutin.</td>
      <td>3509.33 g/mole</td>
      <td>81.82%</td>
      <td>109700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>This is a truncated version of the clone 40 aptamer listed in the related aptamers section below. It is predicted to form a single G-quartet.</td>
      <td>12</td>
      <td>None</td>
      <td>dGp,dGp,dTp,dGp,dGp,dGp,dGp,dGp,dTp,dGp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>505</th>
      <td>Pb(II)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/198.png" width="120" ></td>
      <td>Pb(II)</td>
      <td>Small Organic</td>
      <td>4 nM (reported value)</td>
      <td>Buffer A (50 mM HEPES, 500 mM NaCl, 20 uM EDTA (pH 7.0)</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>17288.28 g/mole</td>
      <td>50.00%</td>
      <td>547100 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>9</td>
      <td>Lu et al. "New highly sensitive and selective catalytic DNA biosensors for metal ions." Biosensors and Bioelectronics, 18(2003): 529-540.</td>
      <td>dAp,dCp,dTp,dCp,dAp,dCp,dTp,dAp,dTp,dGp,dGp,dAp,dAp,dGp,dAp,dGp,dAp,dTp,dGp,dGp,dCp,dGp,dAp,dCp,dAp,dTp,dCp,dTp,dCp,dTp,dTp,dCp,dTp,dCp,dCp,dGp,dAp,dGp,dCp,dCp,dGp,dGp,dTp,dCp,dGp,dAp,dAp,dAp,dTp,dAp,dGp,dTp,dGp,dAp,dGp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>506</th>
      <td>L-arginine (12-79)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/484.png" width="120" ></td>
      <td>L-arginine</td>
      <td>Small Organic</td>
      <td>~2.5 mM (reported value)</td>
      <td>Buffer for CD binding assay: 10 mM sodium phosphate buffer (pH 7.5) and 100 mM NaCl.</td>
      <td>4°C</td>
      <td>\n                                Heated to 95°C and cooled to 4°C for over 2h. \n</td>
      <td>Apparent Kd values for guanidine were the same as arginine and argininamide bound 10-fold more tightly.</td>
      <td>8439.53 g/mole</td>
      <td>62.96%</td>
      <td>273400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Forms a G-quartet structure upon binding to arginine.</td>
      <td>28</td>
      <td>None</td>
      <td>dGp,dAp,dCp,dCp,dAp,dGp,dGp,dGp,dCp,dAp,dAp,dAp,dCp,dGp,dGp,dTp,dAp,dGp,dGp,dGp,dAp,dGp,dTp,dGp,dGp,dTp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>507</th>
      <td>Human Matrix Metalloprotease 9 (hMMP-9) (F3)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/377.png" width="120" ></td>
      <td>Human matrix metalloprotease 9 (hMMP-9)</td>
      <td>Protein</td>
      <td>8.1 nM (reported value)</td>
      <td>None</td>
      <td>23°C</td>
      <td>None</td>
      <td>Does not bind to hMMP-2 or -7.</td>
      <td>21667.8 g/mole</td>
      <td>66.18%</td>
      <td>615600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>See also the truncated and modified aptamer, FB3omf.</td>
      <td>75</td>
      <td>None</td>
      <td>rGp,rGp,fUp,fUp,rAp,fCp,fCp,rAp,rGp,fCp,fCp,fUp,fUp,fCp,rAp,fCp,fUp,rGp,fCp,fUp,fCp,fCp,fCp,rGp,rGp,rAp,fCp,rGp,fCp,rGp,fUp,fCp,fUp,rGp,fUp,rAp,fUp,fCp,fCp,rGp,fCp,fUp,rAp,fCp,fCp,fCp,rGp,fCp,fCp,rGp,fCp,rAp,fCp,fCp,rAp,fCp,rGp,rGp,fUp,fCp,rGp,rGp,fUp,fCp,rAp,fCp,rAp,fCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>508</th>
      <td>Interleukin-6 receptor (AIR-3A)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/474.png" width="120" ></td>
      <td>Cells presenting IL-6R</td>
      <td>Cells</td>
      <td>8.5 nM (reported value)</td>
      <td>None</td>
      <td>37°C</td>
      <td>\n                                None Listed \n</td>
      <td>No binding to IL-6 or lysozyme observed.</td>
      <td>6307.88 g/mole</td>
      <td>73.68%</td>
      <td>193400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>19</td>
      <td>None</td>
      <td>rGp,rGp,rGp,rGp,rAp,rGp,rGp,rCp,rUp,rGp,rUp,rGp,rGp,rUp,rGp,rAp,rGp,rGp,rGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>509</th>
      <td>M64CA</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/282.png" width="120" ></td>
      <td>MPT64 Protein</td>
      <td>Protein</td>
      <td>N/A nM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td></td>
      <td>10421.87 g/mole</td>
      <td>35.29%</td>
      <td>326100 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Was used in a sandwich assay with M64CA.</td>
      <td>35</td>
      <td>None</td>
      <td>dTp,dTp,dCp,dGp,dGp,dGp,dAp,dAp,dTp,dGp,dAp,dTp,dTp,dAp,dTp,dCp,dAp,dAp,dAp,dTp,dTp,dTp,dAp,dTp,dGp,dCp,dCp,dCp,dTp,dCp,dTp,dGp,dAp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>510</th>
      <td>CTBA4</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7116.jpg" width="120" ></td>
      <td>Thrombin</td>
      <td>Protein</td>
      <td>0.15 nM (reported value)</td>
      <td>1× PBSMT, pH 7.2, contains 137 mM NaCl,\n2.68 mM KCl, 8.1 mM Na2HPO4, 1.76 mM KH2PO4, 1\nmM MgCl2, and 0.025% Tween-20</td>
      <td>20°C</td>
      <td>None</td>
      <td></td>
      <td>24520.8 g/mole</td>
      <td>51.90%</td>
      <td>751900 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>87</td>
      <td>Yu Mao, Jimmy Gu, Dingran Chang, Lei Wang, Lili Yao, Qihui Ma, Zhaofeng Luo, Hao Qu, Yingfu Li, Lei Zheng, Evolution of a highly functional circular DNA aptamer in serum, Nucleic Acids Research, Volume 48, Issue 19, 4 November 2020, Pages 10680–10690</td>
      <td>dAp,dTp,dCp,dTp,dCp,dGp,dAp,dCp,dTp,dAp,dGp,dTp,dCp,dAp,dTp,dAp,dGp,dGp,dGp,dGp,dGp,dCp,dGp,dCp,dGp,dAp,dAp,dCp,dAp,dTp,dAp,dCp,dGp,dCp,dGp,dGp,dTp,dTp,dGp,dGp,dTp,dGp,dTp,dGp,dGp,dTp,dTp,dGp,dGp,dCp,dTp,dGp,dAp,dCp,dAp,dAp,dTp,dAp,dCp,dTp,dCp,dGp,dTp,dTp,dTp,dTp,dTp,dGp,dGp,dTp,dGp,dTp,dCp,dTp,dCp,dGp,dGp,dAp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>511</th>
      <td>M13a</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/556.jpg" width="120" ></td>
      <td>PSL</td>
      <td>Peptide</td>
      <td>4.59 µM (reported value)</td>
      <td>200 µL of binding buffer (137 mM NaCl, 0.5 mM MgCl2, 2.7 mM KCl, 2 mM KH2PO4, 10 mM Na2HPO4, pH 7.4)</td>
      <td>24°C</td>
      <td>\n                                N/A \n</td>
      <td>High, see Figure 4 of source material</td>
      <td>22343.54 g/mole</td>
      <td>52.78%</td>
      <td>696800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>216</td>
      <td>Hongcheng Mei, Tao Bing, Xiaojuan Yang, Cui Qi, Tianjun Chang, Xiangjun Liu, Zehui Cao, and Dihua Shangguan. 2012. Functional-Group Specific Aptamers Indirectly Recognizing Compounds with Alkyl Amino Group. Anal. Chem. 2012, 84, 7323-7329</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>512</th>
      <td>Clone 2</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/547.jpg" width="120" ></td>
      <td>Protein E (PE)</td>
      <td>Protein</td>
      <td>1.2 nM (reported value)</td>
      <td>165 ug (~5 nmol) purified truncated His-tagged PE immobilized on Niƒ??NTA agarose (Qiagen) in 1.6 mL PBSM (PBS with 1 mM MgCl2)</td>
      <td>23°C</td>
      <td>None</td>
      <td></td>
      <td>20369.18 g/mole</td>
      <td>68.18%</td>
      <td>643700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>221</td>
      <td>Barfod, Anders et al. "In Vitro Selection of RNA Aptamers Directed Against Protein E: A Haemophilus influenzae Adhesin." Mol Biotech, 2014: 1-12.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>513</th>
      <td>Protein Tyrosine Phosphatase (N71yc16)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/137.jpg" width="120" ></td>
      <td>Protein tyrosine phosphatase (PTPase)</td>
      <td>Protein</td>
      <td>18 nM (reported value)</td>
      <td>Selection Buffer (20 mM Tris (pH 7.6), 150 mM NaCl, 5 mM MgCl2, 1 mM dithiothreitol; 100 µl final volume)</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>36151.85 g/mole</td>
      <td>50.00%</td>
      <td>1126100 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>112</td>
      <td>Bell, S., et al. "RNA Molecules That Bind to and Inhibit the Active Site of a Tyrosine Phosphatase." The Journal of Biological Chemistry, 273(1998):14309-14314.</td>
      <td>rGp,rGp,rGp,rAp,rGp,rAp,rUp,rAp,rCp,rCp,rAp,rGp,rCp,rUp,rUp,rAp,rUp,rUp,rCp,rAp,rAp,rUp,rUp,rCp,rUp,rGp,rGp,rCp,rAp,rAp,rUp,rGp,rGp,rGp,rCp,rUp,rAp,rUp,rCp,rCp,rCp,rAp,rAp,rGp,rUp,rGp,rCp,rUp,rAp,rGp,rGp,rCp,rUp,rUp,rCp,rAp,rGp,rGp,rGp,rAp,rGp,rCp,rGp,rAp,rGp,rGp,rAp,rCp,rCp,rAp,rGp,rAp,rCp,rGp,rAp,rCp,rGp,rUp,rAp,rCp,rCp,rUp,rAp,rAp,rCp,rCp,rCp,rUp,rAp,rAp,rGp,rGp,rUp,rGp,rAp,rGp,rAp,rUp,rAp,rGp,rUp,rAp,rAp,rGp,rUp,rGp,rCp,rAp,rAp,rUp,rCp,rUp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>514</th>
      <td>DNAzymeMB (Part of Lysozyme Aptasensor)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/181.png" width="120" ></td>
      <td>N/A: See additional comments.</td>
      <td>Protein</td>
      <td></td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td>BSA was used as a negative control.</td>
      <td>8798.78 g/mole</td>
      <td>57.14%</td>
      <td>285700 L/(mole·cm)</td>
      <td></td>
      <td>0.1 fM (reported value)</td>
      <td>Contains a horseradish peroxidase-like DNAzyme with a region complementary to the Lysozyme Aptamer DNA Blocker (highlighted green in the structure). In the presence of the blocker, the two oligonucleotides hybridize, inactivating the DNAzyme. When the target lysozyme is added, DNAzyme-Blocker complex dissociates and the DNAzyme is activated. For more details, consult ref.</td>
      <td>28</td>
      <td>Fu et al. "An ultrasensitive peroxidase DNAzyme-associated aptasensor that utilizes a target-triggered enzymatic signal amplification strategy." Chemical Communications, 47(2011): 9876-9878.</td>
      <td>dAp,dGp,dGp,dGp,dAp,dCp,dGp,dGp,dGp,dCp,dTp,dAp,dAp,dGp,dTp,dAp,dAp,dCp,dTp,dGp,dTp,dGp,dGp,dAp,dGp,dGp,dGp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>515</th>
      <td>C5s Anti-Antibody</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/607.jpg" width="120" ></td>
      <td>Anti-Vesicular Stomatitis Antibody</td>
      <td>Antibody</td>
      <td>n/a nM (reported value)</td>
      <td>90% High Glucose DMEM, 10% Fetal Bovine Serum. Tested in vivo.</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>12587.16 g/mole</td>
      <td>72.50%</td>
      <td>385000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>40</td>
      <td>Muharemagic D, Zamay A, Ghobadloo SM, Evgin L, Savitskaya A, et al. (2014) Aptamer-Facilitated Protection of Oncolytic Virus from Neutralizing Antibodies. Molecular Therapy--Nucleic Acids 3, e167. doi: 10.1038/mtna.2014.19</td>
      <td>dAp,dCp,dTp,dAp,dGp,dGp,dAp,dCp,dGp,dCp,dTp,dTp,dGp,dGp,dGp,dAp,dGp,dGp,dGp,dGp,dGp,dGp,dGp,dTp,dGp,dGp,dGp,dGp,dTp,dGp,dTp,dCp,dCp,dGp,dGp,dTp,dCp,dGp,dCp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>516</th>
      <td>A2</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7182.png" width="120" ></td>
      <td>M0-like and M2-like macrophages and monocytes</td>
      <td>Cells</td>
      <td>20-45 nM (reported value)</td>
      <td>For individual aptamer binding, cells were incubated with aptamer that was in Wash Buffer supplemented with 10% FBS and 100 ug/mL tRNA.</td>
      <td>4°C</td>
      <td>None</td>
      <td></td>
      <td>16149.5 g/mole</td>
      <td>46.15%</td>
      <td>526500 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>52</td>
      <td>Sylvestre, M.; Saxby, C.; Kacherovsky, N.; Gustafson, H.; Salipante, S.; Pun, S. Identification of a DNA Aptamer That Binds to Human Monocytes and Macrophages. Bioconjug. Chem. 2020 Aug 9; 31(8): 1899-1907. DOI: 10.1021/acs.bioconjchem.0c00247.</td>
      <td>dGp,dAp,dAp,dGp,dAp,dGp,dTp,dAp,dGp,dAp,dTp,dGp,dAp,dAp,dAp,dCp,dGp,dTp,dTp,dTp,dTp,dTp,dTp,dCp,dGp,dCp,dCp,dCp,dGp,dAp,dTp,dAp,dAp,dAp,dAp,dGp,dGp,dGp,dAp,dCp,dGp,dTp,dGp,dCp,dGp,dTp,dCp,dAp,dGp,dAp,dCp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>517</th>
      <td>C42</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7166.jpg" width="120" ></td>
      <td>Human PD1</td>
      <td>Cells</td>
      <td>23 nM (reported value)</td>
      <td>5 ml of PBS-T containing 1 μg/ml bovine serum albumin (BSA) and 0.1 μg/ml tRNA</td>
      <td>22°C</td>
      <td>None</td>
      <td></td>
      <td>25005.1 g/mole</td>
      <td>56.25%</td>
      <td>772200 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>80</td>
      <td>Khedri M, Abnous K, Rafatpanah H, Nabavinia MS, Taghdisi SM, Ramezani M. Development and Evaluation of Novel Aptamers Specific for Human PD1 Using Hybrid Systematic Evolution of Ligands by Exponential Enrichment Approach. Immunol Invest. 2020 Jul;49(5):535-554. doi: 10.1080/08820139.2020.1744639. Epub 2020 May 19. PMID: 32429721.</td>
      <td>dGp,dCp,dTp,dGp,dTp,dGp,dTp,dGp,dAp,dCp,dTp,dCp,dCp,dTp,dGp,dCp,dAp,dAp,dTp,dGp,dGp,dTp,dAp,dGp,dCp,dGp,dGp,dGp,dTp,dAp,dGp,dGp,dGp,dGp,dAp,dGp,dGp,dGp,dAp,dGp,dGp,dGp,dTp,dGp,dAp,dAp,dTp,dGp,dGp,dAp,dGp,dGp,dAp,dTp,dGp,dTp,dTp,dCp,dAp,dTp,dGp,dGp,dCp,dAp,dGp,dCp,dTp,dGp,dTp,dAp,dTp,dCp,dTp,dTp,dGp,dTp,dCp,dTp,dCp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>518</th>
      <td>Cellobiose (Cel#16)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/82.png" width="120" ></td>
      <td>Cellobiose</td>
      <td>Small Organic</td>
      <td>600 nM (reported value)</td>
      <td>None</td>
      <td>N/A°C</td>
      <td>None</td>
      <td>Binds to cellobiose, but not to related disaccharides lactose, maltose, or gentiobiose.</td>
      <td>12988.4 g/mole</td>
      <td>78.05%</td>
      <td>389300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Other selected clones tended to also be G-rich, with the total pool possessing 50-60% G content.</td>
      <td>41</td>
      <td>None</td>
      <td>dGp,dCp,dGp,dGp,dGp,dGp,dTp,dTp,dGp,dGp,dGp,dCp,dGp,dGp,dGp,dTp,dGp,dGp,dGp,dTp,dTp,dCp,dGp,dCp,dTp,dGp,dGp,dGp,dCp,dAp,dGp,dGp,dGp,dGp,dGp,dCp,dGp,dAp,dGp,dTp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>519</th>
      <td>Aptamer ID 3</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/543.jpg" width="120" ></td>
      <td>PDGF-BB</td>
      <td>Protein</td>
      <td>2.6 nM (reported value)</td>
      <td>0.1 mM Na2HPO4, 1.8 mM KH2PO4, 137 mM NaCl, 2.7 mM KCl, and 1 mM MgCl2 (PBSM), pH 7.4</td>
      <td>23°C</td>
      <td>None</td>
      <td></td>
      <td>24315.72 g/mole</td>
      <td>56.25%</td>
      <td>715600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>3 rounds of positive selection at increasing selection stringencies</td>
      <td>240</td>
      <td>Cho et al. "Quantitative selection of DNA aptamers through microfluidic selection and high-throughput sequencing." PNAS 2010, 1-6.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>520</th>
      <td>Xanthine (XAB)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/103.png" width="120" ></td>
      <td>Xanthine</td>
      <td>Small Organic</td>
      <td>3.3 nM (reported value)</td>
      <td>Binding Buffer (20 mM Trisƒ??HCl (pH 7.5), 0.3 M NaCl and 5 mM MgCl2)</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>10232.17 g/mole</td>
      <td>59.38%</td>
      <td>307700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>32</td>
      <td>Kiga, D., et al. "An RNA aptamer to the xanthine/guanine base with a  distinctive mode of purine recognition." Nucleic Acids Research, 26 (1998): 1755-1760.</td>
      <td>rGp,rGp,rCp,rAp,rCp,rGp,rUp,rGp,rUp,rAp,rUp,rUp,rAp,rCp,rCp,rCp,rUp,rAp,rGp,rUp,rGp,rGp,rUp,rCp,rGp,rAp,rCp,rGp,rUp,rGp,rCp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>521</th>
      <td>Lysozyme (Apt 80)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/456.png" width="120" ></td>
      <td>Hen egg white lysozyme and recombinant human lysozyme</td>
      <td>Protein</td>
      <td>2.8 nM (reported value)</td>
      <td>None</td>
      <td>25°C</td>
      <td>None</td>
      <td>Tran et al. found good specificity against BSA, ovomucoid, and ovotransferrin in capillary electrophoresis assays.</td>
      <td>24742.06 g/mole</td>
      <td>57.50%</td>
      <td>777200 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>85</td>
      <td>None</td>
      <td>dAp,dGp,dCp,dAp,dGp,dCp,dAp,dCp,dAp,dGp,dAp,dGp,dGp,dTp,dCp,dAp,dGp,dAp,dTp,dGp,dGp,dCp,dAp,dGp,dGp,dTp,dAp,dAp,dGp,dCp,dAp,dGp,dGp,dCp,dGp,dGp,dCp,dTp,dCp,dAp,dCp,dAp,dAp,dAp,dAp,dCp,dCp,dAp,dTp,dTp,dCp,dGp,dCp,dAp,dTp,dGp,dCp,dGp,dGp,dCp,dCp,dCp,dTp,dAp,dTp,dGp,dCp,dGp,dTp,dGp,dCp,dTp,dAp,dCp,dCp,dGp,dTp,dGp,dAp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>522</th>
      <td>CAP/CAM Aptamer</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/629.png" width="120" ></td>
      <td>Chloramphenicol</td>
      <td>Other</td>
      <td>7.7 µM (reported value)</td>
      <td>100 mM NaCl, 20 mM Trisƒ??HCl, 2 mM MgCl2, 5 mM KCl, 1 mM  CaCl2, pH 7.6</td>
      <td>23°C</td>
      <td>None</td>
      <td></td>
      <td>25106.28 g/mole</td>
      <td>58.02%</td>
      <td>781800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>AptaReport Article Reference:   Luan Q., Xi Y., et al. (2017) A facile colorimetric aptamer assay for small molecule detection in food based on a magnetic single-stranded DNA binding protein-linked composite probe. Sensors and Actuators B: Chemical 239: 979-978.</td>
      <td>81</td>
      <td>Mehta J, Van Dorst B, Rouah-Martin E, et al. (2010) In vitro selection and characterication of DNA aptamers recognizing chloramphenicol. Journal of Biotechnology 155: 361-369.</td>
      <td>dAp,dGp,dCp,dAp,dGp,dCp,dAp,dCp,dAp,dGp,dAp,dGp,dGp,dTp,dCp,dAp,dGp,dAp,dTp,dGp,dAp,dCp,dTp,dTp,dCp,dAp,dGp,dTp,dGp,dAp,dGp,dTp,dTp,dGp,dTp,dCp,dCp,dCp,dAp,dCp,dGp,dGp,dTp,dCp,dGp,dGp,dCp,dGp,dAp,dGp,dTp,dCp,dGp,dGp,dTp,dGp,dGp,dTp,dAp,dGp,dCp,dCp,dTp,dAp,dTp,dGp,dCp,dGp,dTp,dGp,dCp,dTp,dAp,dCp,dCp,dCp,dGp,dTp,dGp,dAp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>523</th>
      <td>8-OHdG (22)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/302.jpg" width="120" ></td>
      <td>8-OHdG</td>
      <td>Other</td>
      <td>0.1 nM (reported value)</td>
      <td>50 mM Trisƒ??HCl (pH 7.6), 300 mM NaCl, 30 mM KCl, and 5 mM MgCl2</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>37780.41 g/mole</td>
      <td>63.64%</td>
      <td>1161100 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>121</td>
      <td>Miyachi, Y. (2009). Selection of a dna aptmaer that binds 8-ohdg using gmp-agarose. Bioorganic & Medicinal Chemistry Letters, 19(2009), 3619-3622.</td>
      <td>dTp,dAp,dAp,dTp,dAp,dCp,dGp,dAp,dCp,dTp,dCp,dAp,dCp,dGp,dTp,dAp,dTp,dAp,dGp,dGp,dGp,dAp,dAp,dTp,dTp,dCp,dGp,dTp,dCp,dGp,dAp,dCp,dGp,dGp,dAp,dTp,dGp,dCp,dGp,dGp,dGp,dCp,dGp,dAp,dTp,dCp,dGp,dGp,dCp,dGp,dGp,dGp,dGp,dGp,dGp,dTp,dGp,dCp,dGp,dTp,dGp,dCp,dGp,dCp,dTp,dCp,dTp,dGp,dTp,dGp,dCp,dCp,dAp,dGp,dGp,dGp,dGp,dGp,dTp,dGp,dGp,dGp,dAp,dCp,dAp,dGp,dAp,dTp,dCp,dAp,dTp,dAp,dTp,dGp,dGp,dGp,dGp,dGp,dTp,dGp,dCp,dTp,dCp,dAp,dGp,dGp,dTp,dCp,dGp,dAp,dCp,dGp,dCp,dAp,dTp,dGp,dCp,dGp,dCp,dCp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>524</th>
      <td>JH19</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/519.jpg" width="120" ></td>
      <td>NB4 AML cell line</td>
      <td>Cells</td>
      <td>7.57 nM (reported value)</td>
      <td>4.5 g/L glucose, 5 mM MgCl2, 0.1 mg/ml yeast tRNA and 1 mg/ml Bovine Serum Albumin (BSA) into Phosphate buffered saline (PBS)</td>
      <td>°C</td>
      <td>None</td>
      <td></td>
      <td>27806.05 g/mole</td>
      <td>56.18%</td>
      <td>858700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>267</td>
      <td>Yang, M. et al. "Developing aptamer probes for acute myelogenous leukemia detection and surface protein biomarker discovery." Journal of Hematology & Oncology 2014, 7:5.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>525</th>
      <td>2F11-14</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/594.jpg" width="120" ></td>
      <td>Ebola Virus VP35</td>
      <td>Protein</td>
      <td>7.1 nM (reported value)</td>
      <td>10 mM HEPES (pH 7.0), 150 mM NaCl, 2mM TCEP (tris(2-carboxyethyl)phosphine), and 1 mM MgCl2 and filtered through nitrocellulose membrane</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>29726.14 g/mole</td>
      <td>47.83%</td>
      <td>939900 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>276</td>
      <td>Binning, J., Wang, T., Luthra, P., et al. (2013) Development of RNA aptamers targeting Ebola Virus VP35. Biochemistry, 52(47):8406-8419. doi: 10.1021/bi400704d</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>526</th>
      <td>S3</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7114.jpg" width="120" ></td>
      <td>CD109</td>
      <td>Protein</td>
      <td>10.5332 - 13.3326  nM (reported value)</td>
      <td>Dulbecco’s phosphate buffered saline (D-PBS, Ca2+/Mg2+-free) supplemented with 4.5 g/L of glucose and 5 mM of MgCl2</td>
      <td>4°C</td>
      <td>None</td>
      <td></td>
      <td>13834 g/mole</td>
      <td>50.00%</td>
      <td>437600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>44</td>
      <td>Xiong, H., Yan, J., Cai, S., He, Q., Peng, D., Liu, Z., & Liu, Y. (2019). Cancer protein biomarker discovery based on nucleic acid aptamers. International Journal of Biological Macromolecules. doi:10.1016/j.ijbiomac.2019.03.165</td>
      <td>dTp,dCp,dTp,dGp,dAp,dGp,dAp,dAp,dTp,dAp,dGp,dTp,dGp,dGp,dTp,dTp,dTp,dGp,dCp,dTp,dGp,dTp,dAp,dTp,dGp,dGp,dTp,dGp,dGp,dGp,dCp,dGp,dTp,dTp,dGp,dAp,dAp,dAp,dGp,dAp,dGp,dGp,dGp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>527</th>
      <td>S132B-C22</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/565.jpg" width="120" ></td>
      <td>Botulinum Neuroxin Type A (BoNT/A)</td>
      <td>Protein</td>
      <td>0.8 nM (reported value)</td>
      <td>PBS, pH 7.4, 300mM NaCl, 3mM MgCl2, 5mM DTT, 1mg/ml BSA, 0.2% Tween, 1mg/ml heparin</td>
      <td>°C</td>
      <td>None</td>
      <td></td>
      <td>12930.02 g/mole</td>
      <td>60.00%</td>
      <td>395100 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>127</td>
      <td>Tzuu-Wang Chang, Michael Blank, Pavithra Janardhanan, Bal Ram Singh, Charlene Mello,  Michael Blind, Shuowei Cai, "In vitro selection of RNA aptamers that inhibit the activity of type  A botulinum neurotoxin" Biochemical and Biophysical Research Communications 396 (2010) 854ƒ??860.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>528</th>
      <td>TLR3-ECD (Family-1)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/211.png" width="120" ></td>
      <td>Toll-like receptor 3 Ectodomain</td>
      <td>Protein</td>
      <td>2.1 nM (reported value)</td>
      <td>None</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>20703.58 g/mole</td>
      <td>60.94%</td>
      <td>628900 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>64</td>
      <td>None</td>
      <td>rGp,rGp,rUp,rAp,rGp,rAp,rUp,rAp,rCp,rGp,rAp,rUp,rGp,rGp,rAp,rUp,rAp,rCp,rCp,rCp,rCp,rCp,rUp,rGp,rUp,rGp,rGp,rCp,rCp,rCp,rGp,rUp,rCp,rAp,rAp,rCp,rAp,rCp,rAp,rGp,rGp,rGp,rGp,rAp,rAp,rGp,rUp,rGp,rGp,rCp,rAp,rUp,rGp,rAp,rCp,rGp,rCp,rGp,rCp,rAp,rGp,rCp,rCp,rAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>529</th>
      <td>p-Nosyl Protected Alkyl Amino Group (M6b-M14)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/459.png" width="120" ></td>
      <td>p-nosyl protected alkyl amino groups</td>
      <td>Small Organic</td>
      <td>2.27 nM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td>Competition binding assay performed by Mei et al. showed that the aptamer requires the entire p-nosyl-amide moiety.</td>
      <td>9917.5 g/mole</td>
      <td>50.00%</td>
      <td>306300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Forms an anti-parallel G-quadruplex.</td>
      <td>33</td>
      <td>None</td>
      <td>dAp,dTp,dCp,dAp,dTp,dGp,dGp,dTp,dGp,dGp,dGp,dTp,dAp,dTp,dCp,dGp,dGp,dCp,dAp,dCp,dTp,dCp,dGp,dTp,dTp,dGp,dGp,dTp,dTp,dGp,dAp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>530</th>
      <td>Neomycin (LGA11)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/430.png" width="120" ></td>
      <td>Neomycin and other aminogycosides (AMGs)</td>
      <td>Small Organic</td>
      <td>40.3 nM (reported value)</td>
      <td>Binding Buffer: PBS with 10% (v/v) methanol</td>
      <td>RT°C</td>
      <td>None</td>
      <td>Binds to several aminoglycosides: kanamycin, paromycin streptomycin, apramycin, tobramycin, neomycin, and gentamycin.</td>
      <td>25568.23 g/mole</td>
      <td>56.96%</td>
      <td>771400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>87</td>
      <td>None</td>
      <td>rGp,rGp,rGp,rAp,rGp,rAp,rAp,rGp,rGp,fCp,rGp,rGp,fCp,rGp,fCp,rGp,fUp,rAp,rGp,rGp,fCp,rGp,rAp,rGp,fCp,fUp,fUp,fUp,rAp,rGp,rGp,fCp,fCp,fCp,rGp,rAp,fCp,rAp,fUp,fUp,fCp,fCp,fCp,fCp,fUp,rAp,rAp,rAp,rAp,rAp,rAp,rGp,fCp,fUp,fUp,rGp,fUp,fUp,fCp,fUp,rAp,fUp,fUp,fCp,rGp,fUp,rGp,rGp,rAp,fCp,rGp,rAp,fUp,rGp,fCp,rGp,fCp,rAp,fCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>531</th>
      <td>Bevacizumab-Responsive Peptide (BRP)</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/220.png" width="120" ></td>
      <td>Bevacizumab-treated Tumor</td>
      <td>Tissue</td>
      <td>N/A nM (reported value)</td>
      <td>Intracardiac injection of phage libraries. Amplified phages were partially purified by polyethyleneglycol(PEG) precipitation and resuspended in tris-buffered saline (TBS).</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>1583.81 g/mole</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>12</td>
      <td>Cao et al. "Phage display peptide probes for imaging early response to bevacizumab treatment." Amino Acids, 41(2011): 1103-1112.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>532</th>
      <td>Colicin E3 (F1-1)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/75.png" width="120" ></td>
      <td>Colicin E3</td>
      <td>Protein</td>
      <td>2 nM (reported value)</td>
      <td>Binding Buffer (10 mM Tris-HCl (pH 7.6), 10 mM MgCl2, 30 mM NH4Cl)</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>38734.22 g/mole</td>
      <td>56.20%</td>
      <td>1159200 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>121</td>
      <td>Hirao, I., et al. "In Vitro Selection of RNA Aptamers that Bind to Colicin E3 and Structurally  Resemble the Decoding Site of 16S Ribosomal RNA." Biochemistry, 43 (2004): 3214-3221.</td>
      <td>rGp,rGp,rGp,rAp,rGp,rAp,rAp,rUp,rUp,rCp,rCp,rGp,rAp,rCp,rCp,rAp,rGp,rAp,rAp,rGp,rCp,rUp,rUp,rAp,rCp,rUp,rGp,rUp,rCp,rCp,rUp,rCp,rCp,rCp,rUp,rUp,rCp,rAp,rGp,rAp,rGp,rAp,rGp,rCp,rGp,rCp,rGp,rGp,rGp,rAp,rCp,rCp,rCp,rUp,rUp,rAp,rAp,rCp,rUp,rUp,rGp,rGp,rGp,rGp,rCp,rCp,rCp,rAp,rCp,rGp,rAp,rAp,rCp,rAp,rGp,rCp,rUp,rUp,rCp,rAp,rGp,rUp,rUp,rCp,rCp,rGp,rUp,rCp,rUp,rCp,rGp,rGp,rCp,rGp,rUp,rCp,rAp,rUp,rAp,rUp,rGp,rUp,rGp,rCp,rGp,rUp,rCp,rUp,rAp,rCp,rAp,rUp,rGp,rGp,rAp,rUp,rCp,rCp,rUp,rCp,rAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>533</th>
      <td>Kanamycin A (sla 110)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/466.png" width="120" ></td>
      <td>Kanamycin A</td>
      <td>Protein</td>
      <td><300 nM (reported value)</td>
      <td>Binding Buffer (500 mM NaCl, 50 mM Tris Cl (pH 7.6))</td>
      <td>RT°C</td>
      <td>None</td>
      <td>Moderate binding to dibekacin, amikacin, and ribostamycin observed.  No binding to bekanamycin or streptomycin observed.</td>
      <td>25654.43 g/mole</td>
      <td>48.75%</td>
      <td>787300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>88</td>
      <td>Lato et al. "In vitro selection of RNA lectins: using combinatorial chemistry to interpret ribozyme evolution." Chemistry & Biology, 2(1995): 291-303.</td>
      <td>rGp,rGp,rGp,rAp,rAp,rUp,rGp,rGp,rAp,rUp,rCp,rCp,rAp,rCp,rAp,rUp,rCp,rUp,rAp,rCp,rGp,rAp,rAp,rUp,rUp,rCp,rGp,rAp,rAp,rUp,rUp,rGp,rCp,rCp,rGp,rUp,rAp,rAp,rUp,rUp,rUp,rCp,rCp,rCp,rGp,rUp,rGp,rGp,rAp,rGp,rCp,rGp,rAp,rUp,rGp,rCp,rUp,rUp,rCp,rAp,rCp,rUp,rGp,rCp,rAp,rGp,rAp,rCp,rUp,rUp,rGp,rAp,rCp,rGp,rAp,rAp,rGp,rCp,rUp,rUp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>534</th>
      <td>Cyclophilin B (M9-5)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/318.png" width="120" ></td>
      <td>Cyclophilin B (CypB) pancreatic cell line</td>
      <td>Protein</td>
      <td></td>
      <td>None</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>23031.81 g/mole</td>
      <td>69.01%</td>
      <td>674400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>78</td>
      <td>None</td>
      <td>rGp,rGp,rGp,rAp,rGp,rGp,rAp,fCp,rGp,rAp,fUp,rGp,fCp,rGp,rGp,rGp,rGp,rAp,fCp,fCp,fUp,rAp,fUp,rGp,fCp,rAp,rGp,fUp,rAp,rGp,fCp,fCp,rAp,rGp,fUp,rGp,fUp,rGp,rGp,rAp,fCp,fUp,rGp,rGp,rGp,fCp,fUp,rGp,fCp,fCp,fCp,fCp,fCp,fCp,fCp,fCp,rAp,rGp,rAp,fCp,rGp,rAp,fCp,fUp,fCp,rGp,fCp,fCp,fCp,rGp,rAp</td>
      <td>1.5 nM (reported value)</td>
    </tr>
    <tr>
      <th>535</th>
      <td>Tumor necrosis factor-alpha (TNFa) (VR11)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/376.png" width="120" ></td>
      <td>Tumor necrosis factor-alpha (TNFa)</td>
      <td>Protein</td>
      <td>7 nM (reported value)</td>
      <td>Selection Buffer: PBS, 0.005% (v/v) Tween-20.</td>
      <td>37°C</td>
      <td>None</td>
      <td>Does not bind to TNF??.</td>
      <td>7788.1 g/mole</td>
      <td>64.00%</td>
      <td>244400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>This aptamer blocked TNFa signaling in a NF-?B luciferase reporter assays and also prevented TNFa-induced apoptosis and reduced NO production in cultured cells.</td>
      <td>26</td>
      <td>None</td>
      <td>dTp,dGp,dGp,dTp,dGp,dGp,dAp,dTp,dGp,dGp,dCp,dGp,dCp,dAp,dGp,dTp,dCp,dGp,dGp,dCp,dGp,dAp,dCp,dAp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>536</th>
      <td>Thrombin (31mer) (D2-5)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/422.png" width="120" ></td>
      <td>Thrombin</td>
      <td>Protein</td>
      <td>N/A nM (reported value)</td>
      <td>Buffer for Clotting Inhibition assay: Tris-HCl pH 8.0, 5 mM KCl, 100 mM NaCl.</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>9710.34 g/mole</td>
      <td>61.29%</td>
      <td>295700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Aptamer consists of a stem-loop (see figure) with the loop forming a G-quartet identical to the 15-mer thrombin-binding aptamer in the Related Aptamers section below. The authors optimized the stem for clotting inhibition.</td>
      <td>31</td>
      <td>None</td>
      <td>dAp,dGp,dCp,dGp,dAp,dGp,dTp,dAp,dGp,dGp,dTp,dTp,dGp,dGp,dTp,dGp,dTp,dGp,dGp,dTp,dTp,dGp,dGp,dGp,dGp,dCp,dTp,dCp,dGp,dCp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>537</th>
      <td>GVI-11</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7141.png" width="120" ></td>
      <td>GCRV-infected CIK</td>
      <td>Cells</td>
      <td>278.66 nM (reported value)</td>
      <td>Binding Buffer [10% FBS] from Life Technologies.</td>
      <td>4°C</td>
      <td>None</td>
      <td></td>
      <td>29874.3 g/mole</td>
      <td>57.73%</td>
      <td>917700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Grass Carp Reovirus Infected Cytokine-Induced Killer Cells (GCRV-infected CIK cells). Grass carp (Ctenopharyngon idellus) kidney cells (CIK).</td>
      <td>97</td>
      <td>Yu, Qing et al. “Generation and characterization of aptamers against grass carp reovirus infection for the development of rapid detection assay.” Journal of fish diseases vol. 44,1 (2021): 33-44. doi:10.1111/jfd.13265</td>
      <td>dGp,dTp,dCp,dTp,dGp,dAp,dAp,dGp,dTp,dAp,dGp,dAp,dCp,dGp,dCp,dAp,dGp,dGp,dAp,dGp,dTp,dCp,dCp,dGp,dTp,dCp,dGp,dGp,dTp,dTp,dCp,dGp,dCp,dCp,dTp,dTp,dGp,dCp,dGp,dAp,dGp,dTp,dGp,dCp,dAp,dCp,dGp,dCp,dCp,dGp,dCp,dCp,dAp,dGp,dCp,dCp,dGp,dTp,dTp,dAp,dCp,dAp,dTp,dAp,dAp,dAp,dGp,dTp,dCp,dTp,dGp,dCp,dCp,dCp,dAp,dCp,dTp,dAp,dGp,dTp,dCp,dAp,dCp,dAp,dCp,dCp,dTp,dGp,dAp,dGp,dTp,dAp,dAp,dGp,dCp,dGp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>538</th>
      <td>A10</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7195.jpg" width="120" ></td>
      <td>Bovine pregnancy-associated glycoproteins (bPAGs)</td>
      <td>Protein</td>
      <td>11.7 nM (reported value)</td>
      <td>150 mM NaCl, 20 mM HEPES, 1 mM KCl, 1 mM CaCl2, 1 mM MgCl2, pH 7.4</td>
      <td>94°C</td>
      <td>None</td>
      <td></td>
      <td>17849.5 g/mole</td>
      <td>63.16%</td>
      <td>545500 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>57</td>
      <td>Liu, C., Lu, C., & Shi, G. (2020). Selection, identification, and application of DNA aptamers against bovine pregnancy-associated glycoproteins 4. Analytical and Bioanalytical Chemistry. doi:10.1007/s00216-020-02666-w</td>
      <td>dTp,dTp,dGp,dAp,dAp,dGp,dTp,dGp,dAp,dCp,dTp,dCp,dCp,dGp,dCp,dAp,dCp,dTp,dGp,dGp,dGp,dTp,dGp,dGp,dGp,dTp,dGp,dGp,dGp,dAp,dGp,dGp,dGp,dTp,dCp,dGp,dTp,dGp,dCp,dGp,dGp,dCp,dTp,dGp,dGp,dTp,dCp,dAp,dTp,dAp,dGp,dCp,dAp,dGp,dGp,dGp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>539</th>
      <td>Human Immunoglobulin G (IgG) (Apt 8-2)</td>
      <td>Chimeric</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/375.png" width="120" ></td>
      <td>hlgG-Fc</td>
      <td>Protein</td>
      <td>90.9 nM (reported value)</td>
      <td>Binding Buffer (145mM NaCl, 5.4 mM KCl, 0.8mM MgCl, 1.8 mM CaCl, and 20 mM Tris-HCl (pH 7.6))</td>
      <td>N/A°C</td>
      <td>None</td>
      <td>None</td>
      <td>7224.54 g/mole</td>
      <td>60.87%</td>
      <td>226000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>23</td>
      <td>Miyakawa, Shin, et al. "Structural and molecular basis for hyperspecificity of RNA aptamer to human immunoglobulin G." RNA Journal 14,(2008): 1154-1163.</td>
      <td>dGp,dGp,rAp,rGp,rGp,fUp,rGp,dCp,fUp,dCp,dCp,dGp,dAp,dAp,dAp,rGp,rGp,dAp,dAp,fCp,fUp,dCp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>540</th>
      <td>Peroxidase DNAzyme ([B7]-3-0)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/469.png" width="120" ></td>
      <td>Hemin</td>
      <td>Protein</td>
      <td>29 nM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td></td>
      <td>6349.19 g/mole</td>
      <td>60.00%</td>
      <td>203600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Forms a G-quadruplex structure that enhances the natural peroxidase activity of hemin</td>
      <td>20</td>
      <td>None</td>
      <td>dAp,dTp,dTp,dGp,dGp,dGp,dAp,dGp,dGp,dGp,dAp,dTp,dTp,dGp,dGp,dGp,dTp,dGp,dGp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>541</th>
      <td>Vascular Permeability Factor/Vascular Endothelial Growth Factor (NX-213)</td>
      <td>Chimeric</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/144.jpg" width="120" ></td>
      <td>Vascular Permeability  Factor/Vascular  Endothelial  Growth  Factor (VPF/VEGF)</td>
      <td>Protein</td>
      <td>140 pM (reported value)</td>
      <td>None</td>
      <td>37°C</td>
      <td>\n                                Heated at 90°C for 2 min and then cooled on ice. \n</td>
      <td>Binding with Reduced VPF/VEGF, platelet-derived growth factor AB, basic fibroblast growth factor, antithrombin III, and thrombin was measured. Aptamer bound with >650- fold preference to target.</td>
      <td>16446.5 g/mole</td>
      <td>50.00%</td>
      <td>300400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>32</td>
      <td>None</td>
      <td>rGp,rAp,rGp,rAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>542</th>
      <td>Aptamer A1</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7088.pdf" width="120" ></td>
      <td>HA protein of CJ01 H1N1 Influenza Virus</td>
      <td>Protein</td>
      <td>3.36 nM (reported value)</td>
      <td>S Buffer (Washing and Binding): 40nM HEPES (pH 7.5); 102mM NaCl; 5mM KCl; 5mM MgCl2, 0.05% Tween-20</td>
      <td>25°C</td>
      <td>\n                                95 C for 5 min, slowly cooled to 25 C \n</td>
      <td></td>
      <td>14125.15 g/mole</td>
      <td>67.39%</td>
      <td>488790 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>46</td>
      <td>Kwon, J., Narayan, C., Kim, C., Han, M. J., Kim, M., & Jang, S. K. (2019). Development of a Subtype-Specific Diagnostic System for Influenza Virus H3N2 Using a Novel Virus-Based Systematic Evolution of Ligands by Exponential Enrichment (Viro-SELEX). Journal of Biomedical Nanotechnology, 15(7), 1609–1621. https://doi.org/10.1166/jbn.2019.2789 \nGold L, Ayers D, Bertino J, Bock C, Bock A, Brody EN, et al. (2010) Aptamer-Based Multiplexed Proteomic Technology for Biomarker Discovery. PLoS ONE 5(12): e15004. https://doi.org/10.1371/journal.pone.0015004</td>
      <td>dCp,dCp,dTp,dGp,dCp,dCp,dTp,dTp,dTp,dGp,dGp,dAp,dAp,dCp,dCp,dGp,dAp,dCp,dCp,dGp,dCp,dCp,dAp,dGp,dGp,dCp,dCp,dGp,dAp,dGp,dGp,dAp,dAp,dAp,dGp,dGp,dCp,dAp,dCp,dCp,dGp,dAp,dCp,dAp,dGp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>543</th>
      <td>Anti Staphylococcus aureus (SA20)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/505.png" width="120" ></td>
      <td>Staphylococcus aureus (S.Aureus)</td>
      <td>Cells</td>
      <td>70.86 nM (reported value)</td>
      <td>0.1mg/ml salmonsperm DNA (0.1mg/ml tRNA, 1%BSA, 1PBS, 0.05%  (v/v) Tween-20)   Incubated with cells for 1 hour</td>
      <td>37°C</td>
      <td>\n                                denatured  by heating at  100C for 5 min then cooled immediately on ice for 10 min \n</td>
      <td>Streptococcus and S.epidermidis were used as counter targets</td>
      <td>27053.55 g/mole</td>
      <td>53.41%</td>
      <td>822700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Aptamers were originally characterized as a ƒ??panelƒ? of aptamers as a possible rapid diagnostic test. The strongest binding sequence was officially cataloged here.</td>
      <td>88</td>
      <td>Mitsuma et al. Promising New Assays and Technologies for  the Diagnosis and Management of Infectious  Diseases(Clinical Infectious Diseases)</td>
      <td>dGp,dCp,dAp,dAp,dTp,dGp,dGp,dTp,dAp,dCp,dGp,dGp,dTp,dAp,dCp,dTp,dTp,dCp,dCp,dGp,dCp,dGp,dCp,dCp,dCp,dTp,dCp,dTp,dCp,dAp,dCp,dGp,dTp,dGp,dGp,dCp,dAp,dCp,dTp,dCp,dAp,dGp,dAp,dGp,dTp,dGp,dCp,dCp,dGp,dGp,dAp,dAp,dGp,dTp,dTp,dCp,dTp,dGp,dCp,dGp,dTp,dTp,dAp,dTp,dCp,dAp,dAp,dAp,dAp,dGp,dTp,dGp,dCp,dAp,dCp,dGp,dCp,dTp,dAp,dCp,dTp,dTp,dTp,dGp,dCp,dTp,dAp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>544</th>
      <td>VEGF-KDR</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/233.jpg" width="120" ></td>
      <td>VEGF-KDR</td>
      <td>Tissue</td>
      <td></td>
      <td>Binding Buffer: DMEM supplemented with 20 mM HEPES (pH 7.4) and 2 mg/mL gelatin</td>
      <td>4°C</td>
      <td>None</td>
      <td></td>
      <td>886.11 g/mole</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Reduces VEGF-induced angiogenesis and blocks VEGF-KDR interaction.</td>
      <td>7</td>
      <td>Binetruy-Tournaire et al. "Identification of a peptide blocking vascular endothelial growth factor (VEGF)-mediated angiogenesis." The EMBO Journal, 19(2000): 1525-1533.</td>
      <td></td>
      <td>~80 µM (reported value)</td>
    </tr>
    <tr>
      <th>545</th>
      <td>T2</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7196.jpg" width="120" ></td>
      <td>Plasmodium vivax lactate dehydrogenase (PvLDH)</td>
      <td>Protein</td>
      <td>670 nM (reported value)</td>
      <td>25 mM Tris·HCl at pH 7.5 containing 0.1 M NaCl and 20 mM imidazole</td>
      <td>22°C</td>
      <td>None</td>
      <td></td>
      <td>24238.7 g/mole</td>
      <td>50.63%</td>
      <td>755500 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>79</td>
      <td>Cheung YW, Röthlisberger P, Mechaly AE, et al. Evolution of abiotic cubane chemistries in a nucleic acid aptamer allows selective recognition of a malaria biomarker. Proc Natl Acad Sci U S A. 2020;117(29):16790-16798. doi:10.1073/pnas.2003267117</td>
      <td>dCp,dAp,dCp,dTp,dCp,dAp,dCp,dGp,dTp,dCp,dAp,dGp,dTp,dGp,dAp,dCp,dAp,dTp,dGp,dCp,dAp,dTp,dGp,dCp,dCp,dGp,dAp,dTp,dGp,dAp,dCp,dTp,dAp,dGp,dTp,dCp,dGp,dTp,dCp,dAp,dCp,dTp,dAp,dGp,dTp,dGp,dCp,dAp,dCp,dGp,dTp,dAp,dAp,dCp,dGp,dTp,dGp,dCp,dTp,dAp,dGp,dTp,dCp,dAp,dGp,dAp,dAp,dAp,dTp,dTp,dTp,dCp,dGp,dCp,dAp,dCp,dCp,dAp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>546</th>
      <td>Vaspin binding aptamer (VBA)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/409.png" width="120" ></td>
      <td>Vaspin</td>
      <td>Protein</td>
      <td>303 nM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td>No significant signal was observed from RBP4, Nampt, adiponectin, human serum albumin, human IgG, and fibrinogen.</td>
      <td>23487.31 g/mole</td>
      <td>50.67%</td>
      <td>743100 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Limit of detection using ELAAS assay was 39 ng/ml. See SJ Lee et al (2012) for more details.</td>
      <td>83</td>
      <td>None</td>
      <td>dAp,dTp,dAp,dCp,dCp,dAp,dGp,dCp,dTp,dTp,dAp,dTp,dTp,dCp,dAp,dAp,dTp,dTp,dGp,dGp,dGp,dCp,dGp,dGp,dTp,dGp,dGp,dGp,dGp,dGp,dGp,dGp,dGp,dTp,dAp,dGp,dTp,dGp,dGp,dGp,dTp,dGp,dTp,dTp,dAp,dTp,dGp,dGp,dCp,dGp,dAp,dTp,dCp,dGp,dTp,dGp,dGp,dAp,dGp,dAp,dTp,dAp,dGp,dTp,dAp,dAp,dGp,dTp,dGp,dCp,dAp,dAp,dTp,dCp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>547</th>
      <td>RGD motifs in Fibronectin and Vitronectin</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/268.png" width="120" ></td>
      <td>RGD motifs in Fibronectin and Vitronectin</td>
      <td>Protein</td>
      <td>N/A nM (reported value)</td>
      <td>TBS Wash Buffer</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>1061.26 g/mole</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>This peptide aptamer may be cyclic because of the C- and N-terminal cysteine residues.</td>
      <td>8</td>
      <td>Pasqualini et al. "A Peptide Isolated from Phage  Display Libraries Is a Structural and Functional Mimic of  an RGD-binding Site on Integrins." The Journal of Cell Biology, 130(1995): 1189-1196.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>548</th>
      <td>V7t1 targeting VEGF-165</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/368.png" width="120" ></td>
      <td>VEGF-165</td>
      <td>Protein</td>
      <td>1.4 nM (reported value)</td>
      <td>10% (v/v) human serum in TBSTE (TBSE containing 0.05% (v/v) of Tween-20).</td>
      <td>24°C</td>
      <td>None</td>
      <td>V7t1 is a truncated DNA aptamer that targets both VEGF-121 and VEGF-165.</td>
      <td>7900.16 g/mole</td>
      <td>72.00%</td>
      <td>248000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>25</td>
      <td>None</td>
      <td>dTp,dGp,dTp,dGp,dGp,dGp,dGp,dGp,dTp,dGp,dGp,dAp,dCp,dGp,dGp,dGp,dCp,dCp,dGp,dGp,dGp,dTp,dAp,dGp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>549</th>
      <td>Human Breast Cancer Xenografts (RGD-4C)</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/213.png" width="120" ></td>
      <td>Human Breast Cancer Xenografts</td>
      <td>Tissue</td>
      <td>N/A nM (reported value)</td>
      <td>None</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>1103.34 g/mole</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>9</td>
      <td>None</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>550</th>
      <td>Non-small Cell Lung Cancer (S6)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/271.jpg" width="120" ></td>
      <td>Non-small Cell Lung Cancer</td>
      <td>Cells</td>
      <td>28.2 nM (reported value)</td>
      <td>Wash Buffer (5 mmol MgCl2 and 4.5 g  glucose into 1 L of Dulbeccoƒ??s PBS without calcium and  magnesium)     The binding buffer was prepared by adding 1 mg/mL bovine serum albumin (BSA) and 0.1 mg/mL yeast tRNA to washing buffer</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>26491.17 g/mole</td>
      <td>58.82%</td>
      <td>809200 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>85</td>
      <td>Zhao et al. "Recognition of subtype non-small cell lung cancer by DNA aptamers selected  from living cells." Analyst 134(2009):1808-1814.</td>
      <td>dAp,dCp,dGp,dCp,dTp,dCp,dGp,dGp,dAp,dTp,dGp,dCp,dCp,dAp,dCp,dTp,dAp,dCp,dAp,dGp,dGp,dTp,dGp,dGp,dCp,dCp,dAp,dGp,dTp,dCp,dAp,dCp,dTp,dCp,dAp,dAp,dTp,dTp,dGp,dGp,dGp,dTp,dGp,dTp,dAp,dGp,dGp,dGp,dGp,dTp,dGp,dGp,dGp,dGp,dAp,dTp,dTp,dGp,dTp,dGp,dGp,dGp,dTp,dTp,dGp,dCp,dTp,dCp,dAp,dTp,dGp,dGp,dAp,dCp,dGp,dTp,dGp,dCp,dTp,dGp,dGp,dTp,dGp,dAp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>551</th>
      <td>CD133-A15</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7206.png" width="120" ></td>
      <td>AC133 epitope (CD133)</td>
      <td>Protein</td>
      <td>83.2 ± 82.9 nM (reported value)</td>
      <td>None</td>
      <td>°C</td>
      <td>\n                                m-Fold \n</td>
      <td></td>
      <td>4710.86 g/mole</td>
      <td>60.00%</td>
      <td>140700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>15</td>
      <td>Shigdar, S. et al. Nov 19, 2012. "RNA aptamers targeting cancer stem cell marker CD133." Elsevier. Cancer Letters.</td>
      <td>rCp,rCp,rCp,rUp,rCp,rCp,rUp,rAp,rCp,rAp,rUp,rAp,rGp,rGp,rGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>552</th>
      <td>Rous Sarcoma Virus (A)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/168.jpg" width="120" ></td>
      <td>Rous Sarcoma Virus</td>
      <td>Cells</td>
      <td>20 nM (reported value)</td>
      <td>None</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>28169.78 g/mole</td>
      <td>55.17%</td>
      <td>861600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>87</td>
      <td>None</td>
      <td>rGp,rGp,rGp,rAp,rGp,fCp,fUp,fCp,rAp,rGp,rAp,rAp,fUp,rAp,rAp,rAp,fCp,rGp,fCp,fUp,fCp,rAp,rAp,rGp,rGp,rGp,fUp,rAp,rGp,rGp,rGp,rAp,fUp,fCp,rGp,fUp,fUp,rAp,fCp,fCp,fCp,fCp,rGp,rAp,fCp,rAp,fUp,fUp,fUp,fUp,rAp,rAp,fUp,rGp,rGp,rGp,fCp,fCp,rGp,rAp,fUp,rGp,fUp,fUp,fUp,fCp,rGp,rAp,fCp,rAp,fUp,rGp,rAp,rGp,rGp,fCp,fCp,fCp,rGp,rGp,rAp,fUp,fCp,fCp,rGp,rGp,fCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>553</th>
      <td>PS1</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/531.png" width="120" ></td>
      <td>HDLM2</td>
      <td>Cells</td>
      <td>10 nM (reported value)</td>
      <td>4.5 g/L glucose, 5 mM MgCl2, 1 mg/mL BSA and 0.1 mg/mL t-RNA</td>
      <td>37°C</td>
      <td>\n                                95°C for 5 minutes \n</td>
      <td></td>
      <td>20312.25 g/mole</td>
      <td>46.97%</td>
      <td>631700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Counter selection with CD30-positive ALCL K299 cells were performed after every four rounds of positive SELEX</td>
      <td>198</td>
      <td>Parekh, P. et al. "Biostable ssDNA Aptamers Specific for Hodgkin Lymphoma." Sensors 2013, 13: 14543-14557.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>554</th>
      <td>Apt_NDV03</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7191.png" width="120" ></td>
      <td>Newcastle disease virus (NDV) (avian disease)</td>
      <td>Other</td>
      <td>78.1 nM (reported value)</td>
      <td>PBS (pH 7.2) with 150 mM NaCl and 5 mM MgCl2.</td>
      <td>RT°C</td>
      <td>None</td>
      <td></td>
      <td>24714 g/mole</td>
      <td>51.25%</td>
      <td>756100 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>80</td>
      <td>Reference: Marnissi, B.; Kamali-Moghaddam, M.; Ghram, A.; Hmila, I. Generation of ssDNA aptamers as diagnostic tool for Newcastle avian virus. PLoS One. 2020 Aug 13; 15(8): e0237253. DOI: 10.1371/journal.pone.0237253.</td>
      <td>dAp,dGp,dTp,dGp,dCp,dAp,dAp,dGp,dCp,dAp,dGp,dTp,dAp,dTp,dTp,dCp,dGp,dGp,dTp,dCp,dCp,dGp,dAp,dTp,dGp,dGp,dAp,dGp,dGp,dAp,dCp,dCp,dTp,dCp,dCp,dGp,dGp,dTp,dTp,dTp,dAp,dCp,dCp,dGp,dTp,dGp,dTp,dCp,dGp,dTp,dTp,dTp,dTp,dAp,dCp,dTp,dCp,dTp,dTp,dGp,dTp,dAp,dAp,dAp,dGp,dCp,dTp,dGp,dAp,dTp,dGp,dCp,dGp,dTp,dGp,dAp,dTp,dGp,dCp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>555</th>
      <td>NS13</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7131.jpg" width="120" ></td>
      <td>N‑Cadherin Protein</td>
      <td>Protein</td>
      <td>93 nM (reported value)</td>
      <td>1X PBS</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>40 g/mole</td>
      <td>45.00%</td>
      <td>378300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>40</td>
      <td>Yang, L., Gao, T., Li, W., Luo, Y., Ullah, S., Fang, X., … Pei, R. (2020). Ni-Nitrilotriacetic Acid Affinity SELEX Method for Selection of DNA Aptamers Specific to the N-Cadherin Protein. ACS Combinatorial Science. doi:10.1021/acscombsci.0c00165</td>
      <td>dAp,dCp,dTp,dTp,dGp,dCp,dTp,dCp,dTp,dGp,dAp,dTp,dGp,dCp,dGp,dAp,dCp,dAp,dAp,dCp,dCp,dTp,dTp,dTp,dTp,dGp,dCp,dCp,dCp,dAp,dGp,dAp,dTp,dAp,dGp,dTp,dAp,dAp,dGp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>556</th>
      <td>Aptamer 25</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/587.png" width="120" ></td>
      <td>Human norovirus (SMV-VLP)</td>
      <td>Other</td>
      <td>232 nM (reported value)</td>
      <td>Phosphate buffered saline (pH 7.1) supplemented with 100 mg/L CaCl2, 100 mg/L MgCl2 and 0.05% Tween 20</td>
      <td>23°C</td>
      <td>\n                                Heat at 90°C for 5 min \n</td>
      <td></td>
      <td>24864.21 g/mole</td>
      <td>43.21%</td>
      <td>770500 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>The library was amplified using a fluorescein (FAM)-labeled forward constant region primer [5'-56-FAM/-AGTATACGTATTACCTGCAGC-3']  and a biotinylated reverse constant  region primer [5'-/5Bios/-GCAAGATCTCCGAGATATCG-3']</td>
      <td>243</td>
      <td>Escudero-Abarca, B. I. et al.  "Selection, Characterization and Application of Nucleic Acid Aptamers for the Capture and Detection of Human Norovirus Strains."  PLOS One 2014: 1-11.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>557</th>
      <td>Basic Fibroblast Growth Factor (26A-t)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/136.jpg" width="120" ></td>
      <td>Basic Fibroblast Growth Factor (bFGF)</td>
      <td>Protein</td>
      <td>190 pM (reported value)</td>
      <td>Selection Buffer (1X PBS = 10.1 mM Na2HPO4/1.8 mMKH2PO4/137 mM NaCl/2.7 mM KCl,  pH 7.4)</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>10943.71 g/mole</td>
      <td>52.94%</td>
      <td>340500 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Heat-denatured bFGF (90C for 15 min) binds to aptamer with Kd > 1uM. This is a biphasic binding with Kd 0.19+-0.02 nM.</td>
      <td>34</td>
      <td>Jellinek, D., et al. "High-affinity RNA ligands to basic fibroblast growth factor inhibit receptor binding." Proceedings of the National Academy of Sciences of the  United States of America, 90(1993): 11227ƒ??11231.</td>
      <td>rGp,rGp,rUp,rGp,rAp,rAp,rGp,rGp,rCp,rAp,rAp,rCp,rGp,rUp,rAp,rUp,rAp,rGp,rGp,rCp,rAp,rAp,rGp,rCp,rAp,rCp,rAp,rCp,rUp,rUp,rCp,rAp,rCp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>558</th>
      <td>Mouse Retina</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/244.png" width="120" ></td>
      <td>Mouse Retina</td>
      <td>Tissue</td>
      <td>N/A nM (reported value)</td>
      <td>None</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>1117.41 g/mole</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>9</td>
      <td>D Rajotte et al. "  Molecular Heterogeneity of the Vascular Endothelium Revealed by In Vivo  Phage Display." Journal of Clinical Investigations, 102(1998): 430-437.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>559</th>
      <td>Mouse Placenta</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/247.png" width="120" ></td>
      <td>Neonatal Fc Receptor</td>
      <td>Tissue</td>
      <td>N/A nM (reported value)</td>
      <td>None</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>778.96 g/mole</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Has been shown to interfere with the progression of pregnancy in mice.</td>
      <td>7</td>
      <td>Kolonin et al. "Teratogenicity induced by targeting a placental  immunoglobulin transporter." PNAS, 99(2002): 13055-13060.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>560</th>
      <td>Trypsin</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/177.png" width="120" ></td>
      <td>Trypsin</td>
      <td>Protein</td>
      <td>N/A pM (reported value)</td>
      <td>89% Inhibition      Buffer (0.1M Tris-HCl, 0.025M CaCl2 (pH 7.8))</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>737.96 g/mole</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>6</td>
      <td>Yokobayashi et al. "Directed evolution of trypsin inhibiting peptides using a genetic algorithm." Journal of the Chemical Society, 1(1996): 2435-2437.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>561</th>
      <td>Mouse Transferrin Receptor (GS24)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/449.png" width="120" ></td>
      <td>Mouse Transferrin Receptor (mTfR)</td>
      <td>Protein</td>
      <td>N/A nM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td>Does not bind to human TfR.</td>
      <td>19730.83 g/mole</td>
      <td>51.56%</td>
      <td>603000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>64</td>
      <td>None</td>
      <td>dGp,dAp,dAp,dTp,dTp,dCp,dCp,dGp,dCp,dGp,dTp,dGp,dTp,dGp,dCp,dAp,dCp,dAp,dCp,dGp,dGp,dTp,dCp,dAp,dCp,dAp,dGp,dTp,dTp,dAp,dGp,dTp,dAp,dTp,dCp,dGp,dCp,dTp,dAp,dCp,dGp,dTp,dTp,dCp,dTp,dTp,dTp,dGp,dGp,dTp,dAp,dGp,dTp,dCp,dCp,dGp,dTp,dTp,dCp,dGp,dGp,dGp,dAp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>562</th>
      <td>PCB72 (Aptamer 9.3)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/229.png" width="120" ></td>
      <td>Polychlorinated biphenyl 72 (PCB)</td>
      <td>Small Organic</td>
      <td>66 nM (reported value)</td>
      <td>None</td>
      <td>r.t.°C</td>
      <td>None</td>
      <td></td>
      <td>24336.74 g/mole</td>
      <td>57.50%</td>
      <td>735000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>90</td>
      <td>Mehta et al. "Selection and characterization of PCB-binding DNA aptamers." Analytical Chemistry 84(2012): 1669-1676.</td>
      <td>dAp,dGp,dCp,dAp,dGp,dCp,dAp,dCp,dAp,dGp,dAp,dGp,dGp,dTp,dCp,dAp,dGp,dAp,dTp,dGp,dCp,dAp,dCp,dTp,dCp,dGp,dGp,dAp,dCp,dCp,dCp,dCp,dAp,dTp,dTp,dCp,dTp,dCp,dCp,dTp,dTp,dCp,dCp,dAp,dTp,dCp,dCp,dCp,dTp,dCp,dAp,dTp,dCp,dCp,dGp,dTp,dCp,dCp,dAp,dCp,dCp,dCp,dTp,dAp,dTp,dGp,dCp,dGp,dTp,dGp,dCp,dTp,dAp,dCp,dCp,dGp,dTp,dGp,dAp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>563</th>
      <td>Anti-His Tag (6H7)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/163.png" width="120" ></td>
      <td>6X His Tag</td>
      <td>Protein</td>
      <td>4.6 nM (reported value)</td>
      <td>Selection Buffer (50mM K2HPO4, 150mM NaCl, 0.05% Tween 20 (pH 7.5))</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>12488.1 g/mole</td>
      <td>67.50%</td>
      <td>370000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>40</td>
      <td>Kokpinar, et al. "Aptamer-Based Downstream Processing of His-Tagged Proteins Utilizing Magnetic Beads." Biotechnology and Bioengineering, 108(2011):2371-2379.</td>
      <td>dGp,dCp,dTp,dAp,dTp,dGp,dGp,dGp,dTp,dGp,dGp,dTp,dCp,dTp,dGp,dGp,dTp,dTp,dGp,dGp,dGp,dAp,dTp,dTp,dGp,dGp,dCp,dCp,dCp,dCp,dGp,dGp,dGp,dAp,dGp,dCp,dTp,dGp,dGp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>564</th>
      <td>EpCAM EP166</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/585.jpg" width="120" ></td>
      <td>EpCAM</td>
      <td>Protein</td>
      <td>5.48 nM (reported value)</td>
      <td>Binding buffer [4.5 g/L glucose, 5 mM MgCl2 and 10% FBS in Dulbeccoƒ??s PBS (Sigma)] Bovine Serum Albumin (1 mg/ml; Thermo Inc., USA)</td>
      <td>37°C</td>
      <td>None</td>
      <td>Binds strongly HepG2 cells expressing EpCAM but only binds very weakly to HepG2 cells that do not express EpCAM</td>
      <td>23973.58 g/mole</td>
      <td>58.44%</td>
      <td>767300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>231</td>
      <td>Kim, J., W.; Kim, E., Y.; Kim, S., Y; Byun, S., K.; Lee, D.; Oh, K.; Kim, W., K.; Han, B., S.; Chi, S.; Lee, S., C.; Bae, K. Identification of DNA Aptamers toward Epithelial Cell Adhesion Molecule via Cell-SELEX. Molecules and Cells, 2014, 37(10), 742-746 ISSN: 0219-1032</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>565</th>
      <td>SP6</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7226.png" width="120" ></td>
      <td>SARS-Co V-2 Spike (RBD independent mechanism to neutralize SARS-CoV-2)</td>
      <td>Protein</td>
      <td>21 ± 4.6 nM (reported value)</td>
      <td>PBS/3mM MgCl2/0.8 mg/mL BSA (PBS: 137 mM NaCl, 2.7 mM KCl, 10 nM NH4PO4, 1.76 mM KH2PO4 pH 7.4)</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Truncated version of Aptamer was showing higher binding affinity.\nKD value of truncated version was not disclosed.\nSequence: 5'-dCpdCpdCpdApdTpdGpdGpdTpdApdGpdGpdTpdApdTpdTpdGpdCpdTpdTpdGpdGpdTpdApdGpdGpdGpdApdTpdApdGpdTpdGpdGpdGp-3' \n</td>
      <td>86</td>
      <td>Schmitz, A., Weber, A., Bayin, M., Breuers, S., Fieberg, V., Famulok, M., & Mayer, G. (2021). A SARS‐CoV‐2 Spike Binding DNA Aptamer that Inhibits Pseudovirus Infection by an RBD‐Independent Mechanism. Angewandte Chemie International Edition, 60(18), 10279-10285.</td>
      <td>dCp,dCp,dCp,dAp,dTp,dGp,dGp,dTp,dAp,dGp,dGp,dTp,dAp,dTp,dTp,dGp,dCp,dTp,dTp,dGp,dGp,dTp,dAp,dGp,dGp,dGp,dAp,dTp,dAp,dGp,dTp,dGp,dGp,dGp,dGp,dGp,dGp,dAp,dGp,dAp,dGp,dGp,dAp,dGp,dGp,dGp,dAp,dGp,dAp,dTp,dAp,dGp,dAp,dTp,dAp,dTp,dCp,dAp,dAp,dCp,dCp,dCp,dAp,dTp,dGp,dGp,dTp,dAp,dGp,dGp,dTp,dAp,dTp,dTp,dGp,dCp,dTp,dTp,dGp,dGp,dTp,dAp,dGp,dGp,dGp,dAp,dTp,dAp,dGp,dTp,dGp,dGp,dGp,dCp,dTp,dTp,dGp,dAp,dTp,dGp,dTp,dTp,dTp,dCp,dGp,dTp,dGp,dGp,dAp,dTp,dGp,dCp,dCp,dAp,dCp,dAp,dGp,dGp,dAp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>566</th>
      <td>Thyroxine (ApT4-A')</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/189.png" width="120" ></td>
      <td>Thyroxine</td>
      <td>Small Organic</td>
      <td>50 µM (reported value)</td>
      <td>None</td>
      <td>22°C</td>
      <td>None</td>
      <td></td>
      <td>14313.6 g/mole</td>
      <td>65.91%</td>
      <td>424000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>44</td>
      <td>None</td>
      <td>rGp,rGp,rUp,rGp,rGp,rAp,rGp,rGp,rGp,rGp,rGp,rAp,rCp,rGp,rUp,rGp,rCp,rUp,rGp,rCp,rAp,rUp,rCp,rCp,rGp,rCp,rAp,rGp,rUp,rGp,rCp,rGp,rUp,rCp,rUp,rUp,rGp,rGp,rGp,rUp,rUp,rGp,rUp,rGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>567</th>
      <td>CCRF-CEM (sgc8c)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/448.png" width="120" ></td>
      <td>Acute lymphoblastic leukemia (CCRF-CEM)</td>
      <td>Cells</td>
      <td>0.78 nM (reported value)</td>
      <td>None</td>
      <td>4°C</td>
      <td>None</td>
      <td>Recognizes and binds to cell lines Molt-4 (T cell Acute Lymphoblastic Leukemia [ALL]), Sup-T1 (T cell ALL), and Jurkat (T cell ALL).</td>
      <td>12634.26 g/mole</td>
      <td>51.22%</td>
      <td>397600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>46</td>
      <td>None</td>
      <td>dAp,dTp,dCp,dTp,dAp,dAp,dCp,dTp,dGp,dCp,dTp,dGp,dCp,dGp,dCp,dCp,dGp,dCp,dCp,dGp,dGp,dGp,dAp,dAp,dAp,dAp,dTp,dAp,dCp,dTp,dGp,dTp,dAp,dCp,dGp,dGp,dTp,dTp,dAp,dGp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>568</th>
      <td>Tr-7</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7095.jpg" width="120" ></td>
      <td>IL2-CD25 Complex</td>
      <td>Protein</td>
      <td>45.0 nM (reported value)</td>
      <td>HEPES-buffered saline, pH=7.5 with 2mmol/L CaCl2</td>
      <td>37°C</td>
      <td>\n                                Transcribed RNA aptamer library was folded at 2 µM concentration in AIM-V (BSA) medium by step-wise incubation at 95°C for 5 minutes, 65°C for 10 minutes, followed by 37°C for 20 minutes \n</td>
      <td></td>
      <td>13628.35 g/mole</td>
      <td>66.67%</td>
      <td>599100 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>51</td>
      <td>Veeramani, Suresh,et al."An RNA aptamer-based biomarker platform demonstrates high soluble CD25 occupancy by IL2 in the serum of follicular lymphoma patients" American Association for Cancer Research 7, (2019) 1511-1522</td>
      <td>rGp,rGp,rGp,rAp,rGp,rGp,rAp,rCp,rGp,rAp,rUp,rGp,rCp,rGp,rGp,rUp,rGp,rAp,rGp,rUp,rCp,rGp,rUp,rUp,rCp,rCp,rCp,rUp,rUp,rCp,rGp,rUp,rCp,rCp,rCp,rCp,rAp,rGp,rAp,rCp,rGp,rAp,rCp,rUp,rCp,rGp,rCp,rCp,rCp,rGp,rAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>569</th>
      <td>PSMA Dimeric Aptamer</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7083.png" width="120" ></td>
      <td>Prostate Specific Membrane Antigen (PSMA)</td>
      <td>Protein</td>
      <td>nM (reported value)</td>
      <td>SELEX forward rounds were performed by adding 1 mg of PSMA bound to Dynabeads Talon (Invitrogen, Oslo, Norway) to 700 μl of binding buffer (100 mmol/l NaCl, 20 mmol/l Tris, 2 mmol/l MgCl2, 5 mmol/l KCl, 1 mmol/l CaCl2, 0.2% Tween-20, pH 8). The beads were removed using a Dynal magnet (Invitrogen) and were washed four times with binding buffer. At least 10 μg of ssDNA was annealed by heating to 95 °C followed by gentle cooling and was added to the PSMA matrix followed by vortexing and incubation for 1 hour at 37 °C with mild agitation.</td>
      <td>25°C</td>
      <td>\n                                At least 10 μg of ssDNA was annealed by heating to 95 °C followed by gentle cooling and was added to the PSMA matrix followed by vortexing and incubation for 1 hour at 37 °C with mild agitation.  \n</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Dimeric aptamer anneals with partial complement poly-T16 sequence.</td>
      <td>72</td>
      <td>Boyacioglu, O., Stuart, C. H., Kulik, G., & Gmeiner, W. H. (2013). Dimeric DNA Aptamer Complexes for High-capacity-targeted Drug Delivery Using pH-sensitive Covalent Linkages. Molecular therapy. Nucleic acids, 2(7), e107. https://doi.org/10.1038/mtna.2013.37</td>
      <td>dGp,dCp,dGp,dTp,dTp,dTp,dTp,dCp,dGp,dCp,dTp,dTp,dTp,dTp,dGp,dCp,dGp,dTp,dTp,dTp,dTp,dGp,dGp,dGp,dTp,dCp,dAp,dTp,dCp,dTp,dGp,dCp,dTp,dTp,dAp,dCp,dGp,dAp,dTp,dAp,dGp,dCp,dAp,dAp,dTp,dGp,dCp,dTp,dCp,dGp,dGp,dCp,dAp,dAp,dAp,dAp,dAp,dAp,dAp,dAp,dAp,dAp,dAp,dAp,dAp,dAp,dAp,dAp,dGp,dCp,dCp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>570</th>
      <td>GR-3</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/632.jpg" width="120" ></td>
      <td>glucagon receptor (GCGR)</td>
      <td>Protein</td>
      <td>52.7?ñ5.1 nM (reported value)</td>
      <td>5mM MgCl2 and 4.5 g/L glucose in Dulbeccoƒ??s Phosphate Bufered Saline (D-PBS, pH=7.4), and  0.1mg/mL yeast tRNA and 1mg/L bovine serum albumin (BSA) to reduce nonspecifc binding.</td>
      <td>4-37°C</td>
      <td>\n                                denatured at 95 °C for 5 min and then cooled immediately on ice for 10 min \n</td>
      <td></td>
      <td>24099.71 g/mole</td>
      <td>46.91%</td>
      <td>760100 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>81</td>
      <td>Wang, Guodong, et al. "Selection and characterization of DNA aptamer against glucagon  receptor by cell-SELEX". Nature: Scientific Reports (7): 7179. DOI:10.1038/s41598-017-05840-w</td>
      <td>dAp,dTp,dCp,dCp,dAp,dGp,dAp,dGp,dTp,dGp,dAp,dCp,dGp,dCp,dAp,dGp,dCp,dAp,dGp,dAp,dTp,dAp,dAp,dGp,dTp,dAp,dGp,dGp,dTp,dAp,dTp,dCp,dCp,dGp,dTp,dTp,dTp,dGp,dAp,dAp,dAp,dAp,dAp,dCp,dTp,dTp,dTp,dTp,dCp,dTp,dGp,dAp,dCp,dCp,dGp,dTp,dCp,dCp,dGp,dAp,dCp,dTp,dAp,dTp,dGp,dGp,dAp,dCp,dAp,dCp,dGp,dGp,dTp,dGp,dGp,dCp,dTp,dTp,dAp,dGp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>571</th>
      <td>Co2+ (AR1)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/193.png" width="120" ></td>
      <td>Co2+</td>
      <td>Small Organic</td>
      <td>~100 µM (reported value)</td>
      <td>None</td>
      <td>23°C</td>
      <td>None</td>
      <td></td>
      <td>31742.3 g/mole</td>
      <td>54.08%</td>
      <td>985700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>98</td>
      <td>Seetharaman et al. "Immobolized RNA switches for the analysis of complex chemical and biological mixtures." Nature Biotechnology, 19(2001): 336-341.</td>
      <td>rGp,rGp,rAp,rUp,rAp,rAp,rUp,rAp,rGp,rCp,rCp,rGp,rUp,rAp,rGp,rGp,rUp,rUp,rGp,rCp,rGp,rAp,rAp,rAp,rGp,rCp,rGp,rAp,rCp,rCp,rCp,rUp,rGp,rAp,rUp,rGp,rAp,rGp,rAp,rAp,rAp,rGp,rCp,rCp,rAp,rAp,rAp,rGp,rCp,rCp,rGp,rUp,rAp,rGp,rCp,rGp,rCp,rAp,rGp,rAp,rUp,rGp,rAp,rUp,rCp,rUp,rCp,rGp,rCp,rCp,rAp,rUp,rCp,rAp,rGp,rUp,rAp,rCp,rCp,rGp,rAp,rAp,rAp,rCp,rGp,rGp,rUp,rAp,rGp,rCp,rGp,rAp,rGp,rAp,rGp,rCp,rUp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>572</th>
      <td>Human Transferrin Receptor (T7)</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/450.png" width="120" ></td>
      <td>Human Transferrin Receptor (hTfR)</td>
      <td>Protein</td>
      <td>N/A nM (reported value)</td>
      <td>Serum-free Opti-MEM.</td>
      <td>37°C</td>
      <td>None</td>
      <td>Does not bind to mouse TfR.</td>
      <td>939.15 g/mole</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>This peptide aptamer is internalized through receptor-mediated endocytosis.</td>
      <td>7</td>
      <td>None</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>573</th>
      <td>Anti-EGFR (E07)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/328.png" width="120" ></td>
      <td>hEGFR and mEGFR</td>
      <td>Protein</td>
      <td>2.4 nM (reported value)</td>
      <td>None</td>
      <td>25°C</td>
      <td>None</td>
      <td></td>
      <td>29987.87 g/mole</td>
      <td>52.69%</td>
      <td>915000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>93</td>
      <td>N. Li et al. "Inhibition of Cell Proliferation by an Anti-EGFR Aptamer." PLoS One 6(2011): e20299.</td>
      <td>rGp,rGp,fCp,rGp,fCp,fUp,fCp,fCp,rGp,rAp,fCp,fCp,fUp,fUp,rAp,rGp,fUp,fCp,fUp,fCp,fUp,rGp,fUp,rGp,fCp,fCp,rGp,fCp,fUp,rAp,fUp,rAp,rAp,fUp,rGp,fCp,rAp,fCp,rGp,rGp,rAp,fUp,fUp,fUp,rAp,rAp,fUp,fCp,rGp,fCp,fCp,rGp,fUp,rAp,rGp,rAp,rAp,rAp,rAp,rGp,fCp,rAp,fUp,rGp,fUp,fCp,rAp,rAp,rAp,rGp,fCp,fCp,rGp,rGp,rAp,rAp,fCp,fCp,rGp,fUp,rGp,fUp,rAp,rGp,fCp,rAp,fCp,rAp,rGp,fCp,rAp,rGp,rAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>574</th>
      <td>1.5 M Short Anti-Heparanase Aptamer</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/604.jpg" width="120" ></td>
      <td>Heparanase</td>
      <td>Protein</td>
      <td>n/a nM (reported value)</td>
      <td>Cell Culture Medium: 50% DMEM,  50% Hamƒ??s F-12  Cell culture medium supplemented with: 50 mg/ml ascorbic acid, 250 ng/ml amphotericin B, 5mg/ml insulin (bovine pancreas), 0.4 ng/ml hydrocortisone and 10% heat-inactivated fetal bovine serum.</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>9254.11 g/mole</td>
      <td>40.00%</td>
      <td>298000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>30</td>
      <td>Simmons SC, Jamsa H, Silva D, Cortez CM, McKenzie EA, et al. (2014) Anti-Heparanase Aptamers as Potential Diagnostic and Therapeutic Agents for Oral Cancer. PLoS ONE 9(10): e96846. doi:10.1371/journal.pone.0096846</td>
      <td>dAp,dCp,dTp,dTp,dTp,dTp,dGp,dAp,dAp,dTp,dGp,dTp,dGp,dGp,dCp,dAp,dAp,dCp,dAp,dAp,dAp,dTp,dTp,dCp,dGp,dAp,dCp,dAp,dGp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>575</th>
      <td>Anti Fumonsin B1 Toxin</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/506.png" width="120" ></td>
      <td>Fumonsin Toxin B1</td>
      <td>Protein</td>
      <td>100 nM (reported value)</td>
      <td>100 mM NaCl, 20 mM Tris,   2 mM MgCl2, 5 mM KCl, 1 mM CaCl2, pH 7.6 incubate with target for 60 minutes</td>
      <td>N/A°C</td>
      <td>\n                                10 minute denature at 90C   4 minute Cold shock at 4oC and incubation at room Temperature for a minimum of 7 minutes \n</td>
      <td>N/A</td>
      <td>29340.21 g/mole</td>
      <td>33.33%</td>
      <td>932400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>96</td>
      <td>Mckeague et al. Screening and Initial Binding Assessment of Fumonisin B1   Aptamers: International Journal of Molecular Sciences</td>
      <td>dAp,dTp,dAp,dCp,dCp,dAp,dGp,dCp,dTp,dTp,dAp,dTp,dTp,dCp,dAp,dAp,dTp,dTp,dAp,dAp,dTp,dCp,dGp,dCp,dAp,dTp,dTp,dAp,dCp,dCp,dTp,dTp,dAp,dTp,dAp,dCp,dCp,dAp,dGp,dCp,dTp,dTp,dAp,dTp,dTp,dCp,dAp,dAp,dTp,dTp,dAp,dCp,dGp,dTp,dCp,dTp,dGp,dCp,dAp,dCp,dAp,dTp,dAp,dCp,dCp,dAp,dGp,dCp,dTp,dTp,dAp,dTp,dTp,dCp,dAp,dAp,dTp,dTp,dAp,dGp,dAp,dTp,dAp,dGp,dTp,dAp,dAp,dGp,dTp,dGp,dCp,dAp,dAp,dTp,dCp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>576</th>
      <td>Swine Influenza A Virus - H3N2 (HA68)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/494.jpg" width="120" ></td>
      <td>Recombinant H3 cluster IV</td>
      <td>Protein</td>
      <td>7.1 nM (reported value)</td>
      <td>None</td>
      <td>25°C</td>
      <td>\n                                95°C for 10 minutes followed by a 10 minute incubation at room temperature \n</td>
      <td>Shown to bind specifically to the H3 subtype compared to recombinant HA proteins from other subtypes.</td>
      <td>12157.93 g/mole</td>
      <td>50.00%</td>
      <td>371300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Binding region located at residues 8-40. At 2.5 uM, completely inhibited agglutination of turkey red blood cells in presence of 16 HA units/50 ul.</td>
      <td>40</td>
      <td>Wongphatcharachai, Manoosak, et al. "Neutralizing DNA Aptamers against Swine Influenza H3N2 Viruses." Journal of Clinical Microbiology 2013, 51(1):46.</td>
      <td>dAp,dCp,dTp,dCp,dCp,dGp,dTp,dCp,dAp,dTp,dCp,dTp,dTp,dTp,dAp,dGp,dTp,dGp,dGp,dCp,dCp,dCp,dCp,dAp,dAp,dTp,dGp,dTp,dCp,dGp,dTp,dTp,dAp,dTp,dCp,dAp,dCp,dCp,dGp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>577</th>
      <td>Interferon-γ (2'NH2-17)</td>
      <td>2'-NH2-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/479.png" width="120" ></td>
      <td>Interferon-y/IFN-y/IFNG</td>
      <td>Protein</td>
      <td>1.7 nM (reported value)</td>
      <td>None</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>38504.53 g/mole</td>
      <td>63.77%</td>
      <td>682800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>75</td>
      <td>Kubik, Mark F., et al. "Isolation and Characterization of  2ƒ??-Fluoro-,  2ƒ??-Amino-,  and 2ƒ??-Fluoro-/Amino-Modified  RNA Ligands to  Human  IFN-7 That Inhibit  Receptor  Binding." Journal of Immunology, 159 (1997): 259-267.</td>
      <td>rGp,rGp,rGp,rAp,rGp,rGp,rAp,rGp,rAp,rGp,rGp,rGp,rGp,rGp,rAp,rGp,rGp,rGp,rAp,rAp,rAp,rGp,rGp,rGp,rGp,rAp,rGp,rGp,rGp,rGp,rGp,rGp,rGp,rAp,rAp,rGp,rAp,rGp,rAp,rGp,rGp,rAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>578</th>
      <td>EAg3-Py</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7091.png" width="120" ></td>
      <td>Hepatitis B - E Antigen (HBeAg)</td>
      <td>Protein</td>
      <td>0.4 nM (reported value)</td>
      <td>1× PBS buffer (10 mM sodium\nphosphate, 137 mM NaCl, and 4.5 mM KCl, pH 7.4)\n7 mM MgCl2 added after refolding\n\nValidated against patent samples in blood serum.</td>
      <td>37°C</td>
      <td>\n                                95 °C for 10 min and then cooled to 0 °C for 10 min \n</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Modifying C16 with a pyrene improved affinity by a factor of 100 over the unmodified aptamer.</td>
      <td>48</td>
      <td>Aptamer Binding Assay for the E Antigen of Hepatitis B Using Modified Aptamers with G-Quadruplex Structures\nYanming Liu, Connie Le, D. Lorne Tyrrell, X. Chris Le, and Xing-Fang Li\nAnalytical Chemistry 2020 92 (9), 6495-6501\nDOI: 10.1021/acs.analchem.9b05740</td>
      <td>dTp,dTp,dTp,dTp,dTp,dTp,dTp,dTp,dGp,dGp,dGp,dCp,dGp,dAp,dAp,dGp,dAp,dCp,dCp,dGp,dGp,dGp,dAp,dCp,dGp,dGp,dGp,dAp,dGp,dGp,dAp,dAp,dAp,dGp,dAp,dGp,dAp,dTp,dGp,dTp,dTp,dTp,dGp,dGp,dTp,dTp,dTp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>579</th>
      <td>hVap1_Stem</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/567.jpg" width="120" ></td>
      <td>VEGF</td>
      <td>Protein</td>
      <td>5.6 nM (reported value)</td>
      <td>APBB; 10 mM HEPES, pH 7.9, 125 mM NaCl, 25 mM KCl, and 1 mM MgCl2</td>
      <td>23-25°C</td>
      <td>None</td>
      <td></td>
      <td>14775.61 g/mole</td>
      <td>56.25%</td>
      <td>453400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>145</td>
      <td>Meng, Hsein-Wei, and John Pagano. "Discovering Aptamers by Cell-SELEX against Human Soluble Growth Factors Ectopically Expressed on Yeast Cell Surface." Plos-One 9.3 (2014): 1-12. Print.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>580</th>
      <td>Gonadotropin-releasing hormone 1 (A10)</td>
      <td>L-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/77.png" width="120" ></td>
      <td>Gonadoliberin/Gonadotropin-releasing hormone 1 (GnRH)</td>
      <td>Peptide</td>
      <td>190 nM (reported value)</td>
      <td>None</td>
      <td>25°C</td>
      <td>None</td>
      <td>None</td>
      <td>46731.86 g/mole</td>
      <td>0.00%</td>
      <td>498100 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>50</td>
      <td>Leva, S., et al. "GnRH Binding RNA and DNA Spiegelmers: A Novel Approach toward GnRH Antagonism." Chemistry and Biology, 9 (2002): 351-359.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>581</th>
      <td>theophylline-inhibited molecular switch</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/398.png" width="120" ></td>
      <td>theophylline</td>
      <td>Small Organic</td>
      <td>na nM (reported value)</td>
      <td>Reaction buffer [50 mM Tris.HCl (pH 7.5) and 20 mM MgCl2] for 20 hr</td>
      <td>23°C</td>
      <td>None</td>
      <td></td>
      <td>24231.68 g/mole</td>
      <td>60.00%</td>
      <td>727900 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>76</td>
      <td>None</td>
      <td>rGp,rGp,rGp,rCp,rGp,rAp,rCp,rCp,rCp,rUp,rGp,rAp,rUp,rGp,rAp,rGp,rAp,rUp,rGp,rAp,rUp,rAp,rCp,rCp,rAp,rGp,rCp,rCp,rGp,rAp,rAp,rAp,rGp,rGp,rCp,rCp,rCp,rUp,rUp,rGp,rGp,rCp,rAp,rGp,rCp,rUp,rCp,rUp,rCp,rGp,rAp,rAp,rAp,rCp,rGp,rGp,rUp,rGp,rAp,rAp,rAp,rGp,rCp,rCp,rGp,rUp,rAp,rGp,rGp,rUp,rUp,rGp,rCp,rCp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>582</th>
      <td>Pro-Gastrin Releasing Peptide</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/471.png" width="120" ></td>
      <td>Pro-gastrin releasing peptide (proGRP)</td>
      <td>Protein</td>
      <td>1.2 µM (reported value)</td>
      <td>None</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>18685.27 g/mole</td>
      <td>32.79%</td>
      <td>593700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>61</td>
      <td>None</td>
      <td>dAp,dTp,dAp,dCp,dCp,dAp,dGp,dCp,dTp,dTp,dAp,dTp,dTp,dCp,dAp,dAp,dTp,dTp,dTp,dGp,dCp,dAp,dCp,dCp,dAp,dCp,dTp,dTp,dAp,dTp,dTp,dGp,dTp,dTp,dAp,dCp,dTp,dAp,dAp,dTp,dCp,dTp,dGp,dAp,dGp,dAp,dTp,dAp,dGp,dTp,dAp,dAp,dGp,dTp,dGp,dCp,dAp,dAp,dTp,dCp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>583</th>
      <td>ERK 1/ ERK2 (Family II)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/143.jpg" width="120" ></td>
      <td>Extracellular Regulated Kinase 1 and 2 (ERK 1 and ERK2)</td>
      <td>Protein</td>
      <td>4.7 nM (reported value)</td>
      <td>None</td>
      <td>30°C</td>
      <td>\n                                Denatured in water at 95°C for 3 min and slowly cooling in binding buffer from 55°C to 30°C. \n</td>
      <td>Preferentially binds to double phosphorylated ERK2 (ppERK2) with 10-fold greater affinity that to unphosphorylated ERK2 (50 nM). No inhibition of JNK or P38.</td>
      <td>35091.26 g/mole</td>
      <td>45.87%</td>
      <td>1100300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>119</td>
      <td>Seiwert, S., et al. "RNA aptamers as pathway-specific MAP kinase inhibitors." Chemistry & Biology, 7(2000): 833-834.</td>
      <td>rGp,rGp,rGp,rAp,rGp,rAp,rGp,rCp,rCp,rAp,rUp,rAp,rCp,rCp,rUp,rGp,rAp,rCp,rAp,rAp,rAp,rGp,rAp,rCp,rGp,rCp,rUp,rAp,rGp,rCp,rGp,rAp,rAp,rUp,rUp,rGp,rGp,rUp,rUp,rCp,rCp,rUp,rCp,rAp,rCp,rUp,rCp,rAp,rAp,rAp,rAp,rGp,rUp,rAp,rGp,rGp,rGp,rGp,rAp,rAp,rAp,rGp,rCp,rGp,rUp,rUp,rAp,rUp,rUp,rAp,rAp,rGp,rAp,rAp,rAp,rCp,rCp,rAp,rAp,rAp,rAp,rUp,rUp,rUp,rGp,rAp,rCp,rAp,rGp,rGp,rUp,rUp,rAp,rCp,rGp,rCp,rAp,rUp,rCp,rCp,rUp,rGp,rCp,rAp,rUp,rCp,rCp,rUp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>584</th>
      <td>Phenylalanine</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/158.jpg" width="120" ></td>
      <td>Phenylalanine</td>
      <td>Small Organic</td>
      <td>34 µM (reported value)</td>
      <td>Selection Buffer (50 mM Hepes (pH 7.0), 0.3 M NaCl, 5 mM CaCl2, and  5 mM MgCl2)</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>25864.53 g/mole</td>
      <td>50.00%</td>
      <td>800800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>80</td>
      <td>Illangasekare and Marus. "Phenylalanine-Binding RNAs and Genetic Code Evolution." Journal of Molecular Evolution, 54(2002): 298-311.</td>
      <td>rAp,rUp,rUp,rGp,rGp,rAp,rUp,rCp,rGp,rGp,rUp,rAp,rGp,rUp,rAp,rUp,rUp,rUp,rAp,rGp,rGp,rGp,rUp,rGp,rAp,rGp,rAp,rCp,rAp,rCp,rUp,rUp,rCp,rAp,rUp,rGp,rCp,rCp,rUp,rUp,rUp,rGp,rUp,rUp,rGp,rCp,rAp,rGp,rGp,rCp,rUp,rGp,rGp,rGp,rGp,rUp,rGp,rAp,rAp,rGp,rGp,rCp,rGp,rCp,rUp,rAp,rCp,rAp,rUp,rGp,rGp,rCp,rGp,rUp,rCp,rUp,rGp,rAp,rAp,rAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>585</th>
      <td>Reactive Green 19 (GR-30)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/91.png" width="120" ></td>
      <td>Reactive Green 19</td>
      <td>Small Organic</td>
      <td>33 µM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>\n                                Heat to 95°C and then cool to room temperature. \n</td>
      <td></td>
      <td>8414.46 g/mole</td>
      <td>77.78%</td>
      <td>250300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>27</td>
      <td>Ellington and Szostak. "Selection in vitro of single-stranded DNA molecules that fold into specific ligand-binding structures." Nature, 355 (1992): 850-852.</td>
      <td>dGp,dCp,dAp,dGp,dGp,dGp,dGp,dCp,dGp,dTp,dTp,dCp,dGp,dGp,dGp,dGp,dGp,dGp,dTp,dAp,dCp,dCp,dGp,dCp,dTp,dGp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>586</th>
      <td>Omethoate (SS4-54)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/299.png" width="120" ></td>
      <td>Omethoate</td>
      <td>Small Organic</td>
      <td>2 µM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td>This aptamer binds preferentially to the organophosphates phorate, profenofos, benzoate and omethoate. It has reduced binding to imidacloprid, acetamiprid, chlorpyrifos, methamidophos, dichlorvos, parathion, dimethoate, monocrotophos and phoxim.</td>
      <td>16675.9 g/mole</td>
      <td>46.30%</td>
      <td>517600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>"A molecular beacon (5' FAM-CTGCACAAGAATCGCTGCAG-3') was designed, which was partly complementary with the middle constant nucleotides-sequence of the selected ssDNAs."</td>
      <td>54</td>
      <td>None</td>
      <td>dAp,dAp,dGp,dCp,dTp,dTp,dTp,dTp,dTp,dTp,dGp,dAp,dCp,dTp,dGp,dAp,dCp,dTp,dGp,dCp,dAp,dGp,dCp,dGp,dAp,dTp,dTp,dCp,dTp,dTp,dGp,dAp,dTp,dCp,dGp,dCp,dCp,dAp,dCp,dGp,dGp,dTp,dCp,dTp,dGp,dGp,dAp,dAp,dAp,dAp,dAp,dGp,dAp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>587</th>
      <td>Lysozyme</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/162.png" width="120" ></td>
      <td>Lysozyme</td>
      <td>Protein</td>
      <td>31 nM (reported value)</td>
      <td>Selection Buffer (20 mM Tris (pH 7.5) 100 mM NaCl,5 mM MgCl2)</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>25443.39 g/mole</td>
      <td>45.57%</td>
      <td>800700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>41</td>
      <td>Cox and Ellington. "Automated Selection of Anti-Protein Aptamers." Bioorganic & Medicinal Chemsitry, 9 (2001): 2525-2531/</td>
      <td>rGp,rGp,rGp,rAp,rAp,rUp,rGp,rGp,rAp,rUp,rCp,rCp,rAp,rCp,rAp,rUp,rCp,rUp,rAp,rCp,rGp,rAp,rAp,rUp,rUp,rCp,rAp,rUp,rCp,rAp,rGp,rGp,rGp,rCp,rUp,rAp,rAp,rAp,rGp,rAp,rGp,rGp,rCp,rAp,rGp,rAp,rGp,rUp,rUp,rAp,rCp,rUp,rUp,rAp,rGp,rUp,rUp,rCp,rAp,rCp,rUp,rGp,rCp,rAp,rGp,rAp,rCp,rUp,rUp,rGp,rAp,rCp,rGp,rAp,rAp,rGp,rCp,rUp,rUp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>588</th>
      <td>M02</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7183.jpeg" width="120" ></td>
      <td>Myoglobin (Mb)</td>
      <td>Protein</td>
      <td>46.3 nM (reported value)</td>
      <td>Ammonium bicarbonate buffer (50 mM, pH 8.5) containing 500 mM NaCl.</td>
      <td>RT°C</td>
      <td>None</td>
      <td></td>
      <td>24606.9 g/mole</td>
      <td>61.25%</td>
      <td>736400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>80</td>
      <td>Ma, Y.; Li, W.; Xing, R.; Li, P.; Liu, Z. Epitope-Imprinted Magnetic Nanoparticles as a General Platform for Efficient In Vitro Evolution of Protein-Binding Aptamers. ACS Sens. 2020 Jul 7; 5(8): 2537-2544. DOI: 10.1012/acssensors.0c00846</td>
      <td>dCp,dTp,dTp,dCp,dTp,dGp,dCp,dCp,dCp,dGp,dCp,dCp,dTp,dCp,dCp,dTp,dTp,dCp,dCp,dCp,dGp,dGp,dTp,dCp,dGp,dAp,dGp,dAp,dCp,dCp,dCp,dGp,dGp,dGp,dTp,dTp,dTp,dAp,dCp,dTp,dTp,dCp,dCp,dTp,dCp,dGp,dGp,dTp,dAp,dGp,dTp,dGp,dTp,dTp,dCp,dGp,dAp,dGp,dGp,dGp,dAp,dGp,dAp,dCp,dGp,dAp,dGp,dAp,dTp,dAp,dGp,dGp,dCp,dGp,dGp,dAp,dCp,dAp,dCp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>589</th>
      <td>Hodgkin Lymphoma aptamer</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/525.png" width="120" ></td>
      <td>HDLM2</td>
      <td>Cells</td>
      <td>10 nM (reported value)</td>
      <td>Washing buffer for selection: Dulbeccoƒ??s PBS with 4.5 g/L glucose and 5 mM MgCl2s.   Binding buffer to reduce nonspecific background binding: 1 mg/mL BSA and 0.1 mg/mL t-RNA</td>
      <td>°C</td>
      <td>\n                                Heat at 95°C for 5 mins \n</td>
      <td></td>
      <td>20312.25 g/mole</td>
      <td>46.97%</td>
      <td>631700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>ssDNA library for SELEX contains a 18 mer primer binding region at the 5' and 3' end with a 30 mer randomized region in the middle.</td>
      <td>198</td>
      <td>Parekh, P. et al. "Biostable ssDNA Aptamers Specific for Hodgkin Lymphoma." Sensors 2013, 13: 14543-14557.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>590</th>
      <td>MT32 protein of Apple Stem Pitting Virus (ASPV)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/159.png" width="120" ></td>
      <td>MT32 coat protein</td>
      <td>Protein</td>
      <td>55 nM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td>Binds to the PSA-H coat protein with similar affinity (83 nM).</td>
      <td>12619.23 g/mole</td>
      <td>57.50%</td>
      <td>380300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Most likely forms a g-quartet.</td>
      <td>40</td>
      <td>Balogh, et al. "Selection and versatile application of virus-specific aptamers." FASEB Journal, 24(2010): 4187-4195.</td>
      <td>dGp,dGp,dGp,dGp,dTp,dGp,dGp,dTp,dGp,dGp,dGp,dTp,dTp,dCp,dTp,dTp,dTp,dTp,dTp,dGp,dTp,dGp,dGp,dTp,dAp,dTp,dTp,dGp,dGp,dTp,dGp,dTp,dGp,dGp,dGp,dGp,dGp,dGp,dCp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>591</th>
      <td>Aflatoxin M1 Aptamer (AFAS3)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/529.jpg" width="120" ></td>
      <td>Aflatoxin M1</td>
      <td>Small Organic</td>
      <td>35.6 nM (reported value)</td>
      <td>20 mM Tris-HCL, 100 mM NaCl, 2 mM MgCl2, 5 mM KCl, 1 mM CaCl2</td>
      <td>25°C</td>
      <td>None</td>
      <td>Checked with GNP-based assay. Found ten aptamers specific for aflatoxin M1 and five that cross reacted with aflatoxin B1.</td>
      <td>22383.47 g/mole</td>
      <td>60.27%</td>
      <td>664700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>73</td>
      <td>Malhotra, S., Pandey, A., Rajput, S. and Sharma, R. "Selection of aptamers for Aflatoxin M1 and their characterization." Journal of Molecular Recognition 27 (2014): 493-500.</td>
      <td>dAp,dTp,dCp,dCp,dGp,dTp,dCp,dAp,dCp,dAp,dCp,dCp,dTp,dGp,dCp,dTp,dCp,dTp,dGp,dAp,dCp,dGp,dCp,dTp,dGp,dGp,dGp,dGp,dTp,dCp,dGp,dAp,dCp,dCp,dCp,dGp,dGp,dAp,dGp,dAp,dAp,dAp,dTp,dCp,dGp,dCp,dAp,dTp,dTp,dCp,dCp,dCp,dCp,dTp,dGp,dTp,dGp,dGp,dTp,dGp,dTp,dTp,dGp,dGp,dCp,dTp,dCp,dCp,dCp,dGp,dTp,dAp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>592</th>
      <td>AP-1</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7168.jpg" width="120" ></td>
      <td>Proprotein Convertase Subtilisin/Kexin 9</td>
      <td>Protein</td>
      <td>294 nM (reported value)</td>
      <td>25 mM Tris-HCl, 192 mM glycine, 5 mM KH2PO4, and pH 8.3</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>27041.5 g/mole</td>
      <td>46.59%</td>
      <td>835800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>88</td>
      <td>Sattari R, Palizban A, Khanahmad H. Single-Strand DNA-Like Oligonucleotide Aptamer Against Proprotein Convertase Subtilisin/Kexin 9 Using CE-SELEX: PCSK9 Targeting Selection. Cardiovasc Drugs Ther. 2020 Aug;34(4):475-485. doi: 10.1007/s10557-020-06986-y. PMID: 32415571.</td>
      <td>dAp,dTp,dAp,dCp,dCp,dAp,dGp,dCp,dTp,dTp,dAp,dTp,dTp,dCp,dAp,dAp,dTp,dTp,dGp,dAp,dCp,dCp,dCp,dGp,dTp,dTp,dTp,dCp,dGp,dTp,dTp,dCp,dCp,dCp,dTp,dCp,dTp,dGp,dGp,dGp,dAp,dAp,dGp,dTp,dTp,dTp,dAp,dGp,dCp,dCp,dCp,dAp,dGp,dTp,dTp,dGp,dCp,dCp,dTp,dGp,dGp,dGp,dCp,dGp,dAp,dTp,dAp,dCp,dCp,dAp,dAp,dGp,dAp,dTp,dAp,dGp,dTp,dAp,dAp,dGp,dTp,dGp,dCp,dAp,dAp,dTp,dCp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>593</th>
      <td>Botulinum neurotoxin type A (BoNT A) Aptamer</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/600.png" width="120" ></td>
      <td>BoNT A Light Chain (BoNT/A-rLc)</td>
      <td>Protein</td>
      <td>34 nM (reported value)</td>
      <td>1X Binding Buffer: 20 mM Hepes, 150 mM NaCl, 2 mM KCl, 2 mM MgCl2, 2mM CaCl2 (pH 7.4)</td>
      <td>23°C</td>
      <td>None</td>
      <td></td>
      <td>18544.08 g/mole</td>
      <td>46.67%</td>
      <td>547900 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>60</td>
      <td>(1) Bogomolova, A., Aldissi, M. ƒ??Real-time and label-free analyte detection in a flow-through mode using immobilized fluorescent aptamer/quantum dots molecular switches.ƒ? Biosensors and Bioelecgtronics, 66 (2015): 290-296. (2) Lou, X., et al. ƒ??Micromagnetic selection of aptamers in microfluidic channels.ƒ? PNAS, 106, 9 (2009): 2989-2994.</td>
      <td>dCp,dTp,dTp,dGp,dAp,dGp,dTp,dGp,dTp,dCp,dAp,dTp,dGp,dGp,dAp,dCp,dGp,dTp,dTp,dCp,dCp,dGp,dGp,dTp,dCp,dTp,dTp,dGp,dGp,dGp,dCp,dGp,dGp,dGp,dAp,dTp,dAp,dTp,dTp,dTp,dGp,dTp,dTp,dTp,dGp,dTp,dTp,dTp,dTp,dCp,dTp,dGp,dCp,dCp,dTp,dAp,dTp,dGp,dTp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>594</th>
      <td>Dopamine (dopa2/c.1)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/95.jpg" width="120" ></td>
      <td>Dopamine</td>
      <td>Small Organic</td>
      <td>1.6 µM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td></td>
      <td>30112.25 g/mole</td>
      <td>58.06%</td>
      <td>921600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>103</td>
      <td>Mannironi, C., et al. "In Vitro Selection of Dopamine RNA Ligands." Biochemistry, 36 (1997):  9726-973.</td>
      <td>rGp,rGp,rGp,rAp,rAp,rUp,rUp,rCp,rCp,rGp,rCp,rGp,rUp,rGp,rUp,rGp,rCp,rGp,rCp,rCp,rGp,rCp,rGp,rGp,rAp,rAp,rGp,rAp,rCp,rGp,rUp,rUp,rGp,rGp,rAp,rAp,rGp,rGp,rAp,rUp,rAp,rGp,rAp,rUp,rAp,rCp,rCp,rUp,rAp,rCp,rAp,rAp,rCp,rGp,rGp,rGp,rGp,rAp,rAp,rUp,rAp,rUp,rAp,rGp,rAp,rGp,rGp,rCp,rCp,rAp,rGp,rCp,rAp,rCp,rAp,rUp,rAp,rGp,rUp,rGp,rAp,rGp,rGp,rCp,rCp,rCp,rUp,rCp,rCp,rUp,rCp,rCp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>595</th>
      <td>Renal Tissue (Glomerulus and Intertubular tissue)</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/225.png" width="120" ></td>
      <td>Glomerular vessels and renal intertubular tissue</td>
      <td>Tissue</td>
      <td>N/A nM (reported value)</td>
      <td>None</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>737.99 g/mole</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>7</td>
      <td>None</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>596</th>
      <td>HMBA (aptabeacon)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/334.png" width="120" ></td>
      <td>6X His-tag</td>
      <td>Protein</td>
      <td></td>
      <td>1X DPBS without Ca and Mg (pH 7.1-7.5), 20 °C</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>12990.45 g/mole</td>
      <td>57.14%</td>
      <td>382400 L/(mole·cm)</td>
      <td></td>
      <td>~4.2 nM (reported value)</td>
      <td>Aptabeacon specific for proteins containing a terminal His-tag.</td>
      <td>1</td>
      <td>None</td>
      <td>dCp,dAp,dGp,dGp,dTp,dTp,dGp,dGp,dTp,dCp,dTp,dGp,dGp,dTp,dTp,dGp,dGp,dGp,dTp,dTp,dTp,dGp,dGp,dCp,dTp,dCp,dCp,dTp,dGp,dTp,dGp,dTp,dAp,dCp,dGp,dCp,dAp,dAp,dCp,dCp,dTp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>597</th>
      <td>Tenascin-C (peptide #1)</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/382.png" width="120" ></td>
      <td>Tenascin C (TNC)</td>
      <td>Protein</td>
      <td>4.58 µM (reported value)</td>
      <td>None</td>
      <td>4°C</td>
      <td>None</td>
      <td>The biotin-labeled peptide bound to the TNC positive cell lines U251 and U11MG (glioblastoma). It did not bind to the TNC negative control cell lines HT29 and HCT116 (colorectal cancer).</td>
      <td>1708.99 g/mole</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>15</td>
      <td>None</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>598</th>
      <td>Ampicillin (AMP17)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/490.png" width="120" ></td>
      <td>Ampicillin</td>
      <td>Small Organic</td>
      <td>13.4 nM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>\n                                Heated at 90°C for 3 min and then cooled at 4°C for 1 hour. \n</td>
      <td>Did not bind to related antibiotics amoxicillin and benzylpenicillin, or unrelated antibiotics chloramphenicol, tetracycline, sulfadimethoxine, streptomycin, and kanamycin.</td>
      <td>5940.9 g/mole</td>
      <td>68.42%</td>
      <td>183800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Because of AMP17's ability to aggregate on gold nanoparticles at lower salt concentrations than the other two aptamers, AMP17 was further evaluated in a colorimetric-fluorometric assay.</td>
      <td>19</td>
      <td>None</td>
      <td>dGp,dCp,dGp,dGp,dGp,dCp,dGp,dGp,dTp,dTp,dGp,dTp,dAp,dTp,dAp,dGp,dCp,dGp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>599</th>
      <td>CBSA-5335 LF</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7176.jpg" width="120" ></td>
      <td>Cocaine</td>
      <td>Small Organic</td>
      <td>36 nM (reported value)</td>
      <td>10% saliva in 10 mM Tris, 0.1 mM MgCl2, pH 7.4</td>
      <td>23°C</td>
      <td>\n                                N/A \n</td>
      <td>Demonstrated specificity to cocaine over benzoylecgonine, anhydroecgonine methyl ester, cocaethylene and nicotine</td>
      <td>12552.2 g/mole</td>
      <td>53.66%</td>
      <td>379500 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>The sequence below is one fragment of the cocaine split aptamer that contains two separate fragments, please view "Related Aptamers" for second fragment of the sequence.</td>
      <td>41</td>
      <td>Yu, H., Canoura, J., Guntupalli, B., Lou, X., & Xiao, Y. (2017). A cooperative-binding split aptamer assay for rapid, specific and ultra-sensitive fluorescence detection of cocaine in saliva. Chemical Science, 8(1), 131–141. doi:10.1039/c6sc01833e</td>
      <td>dCp,dTp,dCp,dCp,dTp,dTp,dCp,dAp,dAp,dCp,dGp,dAp,dAp,dGp,dTp,dGp,dGp,dGp,dTp,dCp,dTp,dCp,dCp,dTp,dTp,dCp,dAp,dAp,dCp,dGp,dAp,dAp,dGp,dTp,dGp,dGp,dGp,dTp,dCp,dTp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>600</th>
      <td>Angiogenic endothelial cells</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/241.png" width="120" ></td>
      <td>Aminopeptidase N</td>
      <td>Tissue</td>
      <td>N/A nM (reported value)</td>
      <td>None</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>319.43 g/mole</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Authors conjugated the NGR motif to liposomal doxorubicin and used the conjugate to selectively target xenograft neuroblastoma tumors in mice. See reference for details.</td>
      <td>3</td>
      <td>Pastorino et al. "Vascular Damage and Anti-angiogenic Effects of Tumor Vessel-Targeted Liposomal Chemotherapy." Cancer Research, 63(2003): 7400-7409.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>601</th>
      <td>Lysozyme Aptamer</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/623.jpg" width="120" ></td>
      <td>Lysozyme</td>
      <td>Protein</td>
      <td>N/A µM (reported value)</td>
      <td>Borate buffer (pH 8.4); Room Temperature (30 min)</td>
      <td>°C</td>
      <td>None</td>
      <td></td>
      <td>11144.35 g/mole</td>
      <td>36.11%</td>
      <td>356800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>36</td>
      <td>J Zhao et al. High-Resolution and Universal Visualization of Latent Fingerprints Based on Aptamer-Functionalized Core-Shell Nanoparticles with Embedded SERS Reporters. ACS Appl. Mater. Interfaces 8(2016): 14389-14395.</td>
      <td>dTp,dTp,dTp,dTp,dTp,dTp,dAp,dTp,dCp,dAp,dGp,dGp,dGp,dCp,dTp,dAp,dAp,dAp,dGp,dAp,dGp,dTp,dGp,dCp,dAp,dGp,dAp,dGp,dTp,dTp,dAp,dCp,dTp,dTp,dAp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>602</th>
      <td>f-4</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7212.png" width="120" ></td>
      <td>fetuin</td>
      <td>Other</td>
      <td>6.2 µM (reported value)</td>
      <td>100 mM NaCl, 2 mM MgCl2, 5 mM KCl, 1 mM CaCl2, 0.02% Tween 20, 20 mM Tris-HCl, pH 7.5</td>
      <td>22°C</td>
      <td>None</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Contains indole-modified bases. See reference article and supplement for details.</td>
      <td>40</td>
      <td>Discovery of indole-modified aptamers for highly specific recognition of protein glycoforms\nAlex M. Yoshikawa, Alexandra Rangel, Trevor Feagin, Elizabeth M. Chun, Leighton Wan, Anping Li, Leonhard Moekl, Michael Eisenstein, Sharon Pitteri, H. Tom Soh\nbioRxiv 2021.03.13.435263; doi: https://doi.org/10.1101/2021.03.13.435263</td>
      <td>dCp,dGp,dCp,dCp,dAp,dAp,dAp,dGp,dCp,dTp,dCp,dGp,dAp,dTp,dCp,dAp,dAp,dTp,dAp,dAp,dCp,dCp,dGp,dAp,dAp,dAp,dGp,dAp,dAp,dAp,dAp,dGp,dGp,dCp,dAp,dCp,dAp,dTp,dCp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>603</th>
      <td>Pepocin (9-41U22)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/171.png" width="120" ></td>
      <td>Pepocin</td>
      <td>Protein</td>
      <td>18 nM (reported value)</td>
      <td>None</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>13103.89 g/mole</td>
      <td>43.90%</td>
      <td>410700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>41</td>
      <td>None</td>
      <td>rGp,rGp,rGp,rAp,rGp,rUp,rCp,rUp,rGp,rAp,rAp,rGp,rUp,rCp,rGp,rGp,rAp,rCp,rUp,rUp,rGp,rUp,rUp,rAp,rUp,rCp,rAp,rAp,rUp,rUp,rCp,rAp,rCp,rUp,rUp,rCp,rAp,rGp,rAp,rCp,rUp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>604</th>
      <td>Rapt1</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7112.jpg" width="120" ></td>
      <td>Recombinant Clostridium Difficile Toxin B</td>
      <td>Protein</td>
      <td>8.9 nM (reported value)</td>
      <td>20 mM Tris-HCl, 100 mM NaCl, 5 mM KCl and 10 mM MgCl2, 1 mM DTT, 0.05%\nTween-20, pH 7.5.</td>
      <td>RT°C</td>
      <td>\n                                denatured at 90 °C for 15 min and then cooled on ice for 15 min \n</td>
      <td></td>
      <td>12266.98 g/mole</td>
      <td>60.00%</td>
      <td>419360 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>40</td>
      <td>Liu, Meng et. al. "In Vitro Selection of a DNA Aptamer Targeting Degraded Protein Fragments for Biosensing" Angewandte Chemie 10.(2020)</td>
      <td>dCp,dTp,dCp,dCp,dCp,dAp,dTp,dGp,dCp,dTp,dAp,dTp,dTp,dAp,dTp,dGp,dGp,dCp,dTp,dAp,dGp,dCp,dGp,dCp,dAp,dGp,dGp,dAp,dGp,dGp,dCp,dCp,dAp,dCp,dAp,dCp,dCp,dGp,dGp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>605</th>
      <td>PCB77 (VIII)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/305.png" width="120" ></td>
      <td>3,3,4,4-tetrachlorobiphenyl (PCB77)</td>
      <td>Small Organic</td>
      <td>4.02 µM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td></td>
      <td>12415.09 g/mole</td>
      <td>57.50%</td>
      <td>378800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>40</td>
      <td>Shengmin, X. (2012). Selection of dna aptamers against polychlorinated biphenyls as potential biorecognition elements for environmental analysis. Analytical Biochemistry, 423(2012), 195-201.</td>
      <td>dGp,dGp,dCp,dGp,dGp,dGp,dGp,dCp,dTp,dAp,dCp,dGp,dAp,dAp,dGp,dTp,dAp,dGp,dTp,dGp,dAp,dTp,dTp,dTp,dTp,dTp,dTp,dCp,dCp,dGp,dAp,dTp,dGp,dGp,dCp,dCp,dCp,dGp,dTp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>606</th>
      <td>Aptamer ID 1</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/541.jpg" width="120" ></td>
      <td>PDGF-BB</td>
      <td>Protein</td>
      <td>2.7 nM (reported value)</td>
      <td>0.1 mM Na2HPO4, 1.8 mM KH2PO4, 137 mM NaCl, 2.7 mM KCl, and 1 mM MgCl2 (PBSM), pH 7.4</td>
      <td>23°C</td>
      <td>None</td>
      <td></td>
      <td>24316.76 g/mole</td>
      <td>52.50%</td>
      <td>730900 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>3 rounds of positive selection at increasing selection stringencies</td>
      <td>240</td>
      <td>Cho et al. "Quantitative selection of DNA aptamers through microfluidic selection and high-throughput sequencing." PNAS 2010, 1-6/</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>607</th>
      <td>L-Selectin Multivalent Aptamer</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/599.jpg" width="120" ></td>
      <td>L-Selectin</td>
      <td>Protein</td>
      <td></td>
      <td>PBS</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>15113.89 g/mole</td>
      <td>46.94%</td>
      <td>481400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Made multivalent through Rolling Circle Amplification.   Circle Template: ACC TTG TAC TGG TTA CCT TGG CTA AAA AAA AAA AAA AAA AAA AAG TAA GCC GAA GCC ATT ACG TTT AGC  See reference for further details.</td>
      <td>147</td>
      <td>Chang EK, Eckert MA, Ali MM, Riazifar H, Pone EJ, Liu L, et al. (2015) Facile Supermolecular Aptamer Inhibitors of L-Selectin. PLoS ONE 10(3): e0123034. doi:10.1371/journal.pone.0123034</td>
      <td></td>
      <td>(Multivalent) 0.75 nM (reported value)</td>
    </tr>
    <tr>
      <th>608</th>
      <td>Receptor Tyrosine Kinase (RET) Mutant (D4)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/88.png" width="120" ></td>
      <td>Receptor Tyrosine Kinase (RET) Mutant</td>
      <td>Protein</td>
      <td>35 nM (reported value)</td>
      <td>Selection buffer: RPMI 1640</td>
      <td>37°C</td>
      <td>None</td>
      <td>Binds to both PC12/MEN2A and PC12/MEN2B.</td>
      <td>16281.84 g/mole</td>
      <td>54.00%</td>
      <td>511900 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>50</td>
      <td>None</td>
      <td>rGp,fCp,rGp,fCp,rGp,rGp,rGp,rAp,rAp,fUp,rAp,rGp,fUp,rAp,fUp,rGp,rGp,rAp,rAp,rGp,rGp,rAp,fUp,rAp,fCp,rGp,fUp,rAp,fUp,rAp,fCp,fCp,rGp,fUp,rGp,fCp,rAp,rAp,fUp,fCp,fCp,rAp,rGp,rGp,rGp,fCp,rAp,rAp,fCp,rGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>609</th>
      <td>Glycated human serum albumin (GHSA)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/615.jpg" width="120" ></td>
      <td>GHSA</td>
      <td>Protein</td>
      <td>5.78 µM (reported value)</td>
      <td>1X PBS, 137mM [Na+], 2mM [Mg2+]</td>
      <td>25°C</td>
      <td>\n                                Incubated at 65°C for 5 min followed by 4°C for 1 min \n</td>
      <td>Shows specific binding to only the Glycated form of human serum albumin</td>
      <td>7102.65 g/mole</td>
      <td>60.87%</td>
      <td>213200 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>23</td>
      <td>Apiwat et al. Graphene based aptasensor for glycated albuminin diabetes mellitus   diagnosis and monitoring. Biosensors and Bioelectronics 82(2016)140ƒ??145.</td>
      <td>dTp,dGp,dCp,dGp,dGp,dTp,dTp,dGp,dTp,dAp,dGp,dTp,dAp,dCp,dTp,dCp,dGp,dTp,dGp,dGp,dCp,dCp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>610</th>
      <td>Endothelial Integrin Alpha-V Beta-3 (Clone 17.16)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/222.png" width="120" ></td>
      <td>Alpha-V Beta-3</td>
      <td>Protein</td>
      <td>N/A nM (reported value)</td>
      <td>None</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>11855.18 g/mole</td>
      <td>51.35%</td>
      <td>364200 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>37</td>
      <td>None</td>
      <td>rUp,rUp,rCp,rAp,rAp,rCp,rGp,rCp,rUp,rGp,rUp,rGp,rAp,rAp,rGp,rGp,rGp,rCp,rUp,rUp,rAp,rUp,rAp,rCp,rGp,rAp,rGp,rCp,rGp,rGp,rAp,rUp,rUp,rAp,rCp,rCp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>611</th>
      <td>Theophylline (AR6)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/203.png" width="120" ></td>
      <td>Theophylline</td>
      <td>Small Organic</td>
      <td>N/A nM (reported value)</td>
      <td>None</td>
      <td>23°C</td>
      <td>None</td>
      <td></td>
      <td>30366.35 g/mole</td>
      <td>52.13%</td>
      <td>938500 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>95</td>
      <td>Seetharaman et al. "Immobilized RNA switches for the analysis of complex chemical and biological mixture." Nature Biotechnology, 19(2001):336-341.</td>
      <td>rGp,rGp,rAp,rUp,rAp,rAp,rUp,rAp,rGp,rCp,rCp,rGp,rUp,rAp,rGp,rGp,rUp,rUp,rGp,rCp,rGp,rAp,rAp,rAp,rGp,rCp,rGp,rAp,rCp,rCp,rCp,rUp,rGp,rAp,rUp,rGp,rAp,rGp,rUp,rCp,rUp,rGp,rGp,rAp,rUp,rAp,rCp,rCp,rAp,rUp,rGp,rCp,rAp,rUp,rGp,rAp,rUp,rGp,rCp,rAp,rCp,rCp,rUp,rUp,rGp,rGp,rCp,rAp,rGp,rUp,rCp,rUp,rUp,rAp,rCp,rGp,rAp,rAp,rAp,rCp,rGp,rGp,rUp,rAp,rGp,rCp,rGp,rAp,rGp,rAp,rGp,rCp,rUp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>612</th>
      <td>ErbB2 Tyrosine Kinase Receptor (Ph-12-1)</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/415.png" width="120" ></td>
      <td>ErbB2 Tyrosine Kinase Receptor</td>
      <td>Protein</td>
      <td>N/A nM (reported value)</td>
      <td>N/A</td>
      <td>N/A°C</td>
      <td>None</td>
      <td>Binds strongly to breast cancer cell lines SKBR3, T47D, MCF-7, amd PM-1, prostate Du-145, Glioma U87MG, colon HT-2933, and lung DMS-273.  Exhibits some binding to human lymphoblast lymphoma cell line K562.</td>
      <td>1648.88 g/mole</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>12</td>
      <td>None</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>613</th>
      <td>C10</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7134.jpg" width="120" ></td>
      <td>Rituximab</td>
      <td>Protein</td>
      <td>17.6 nM (reported value)</td>
      <td>20 mM Tris pH 7.6, 100 mM NaCl, 2 mM MgCl2, 5 mM KCl, 1 mM CaCl2, 0.02% v/v Tween 20</td>
      <td>22°C</td>
      <td>None</td>
      <td></td>
      <td>12348 g/mole</td>
      <td>45.00%</td>
      <td>386600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>44</td>
      <td>Kohlberger M, Wildner S, Regl C, Huber CG, Gadermaier G. Rituximab-specific DNA aptamers are able to selectively recognize heat-treated antibodies. PLoS One. 2020 Nov 5;15(11):e0241560. doi: 10.1371/journal.pone.0241560. PMID: 33151990; PMCID: PMC7644011.</td>
      <td>dAp,dCp,dTp,dTp,dCp,dGp,dGp,dCp,dTp,dAp,dGp,dTp,dTp,dAp,dGp,dGp,dGp,dGp,dGp,dTp,dAp,dGp,dTp,dTp,dTp,dAp,dGp,dAp,dTp,dCp,dGp,dTp,dCp,dTp,dCp,dTp,dAp,dCp,dAp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>614</th>
      <td>L-tryptophan(Trp 70-727)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/195.jpg" width="120" ></td>
      <td>L-tryptophan</td>
      <td>Protein</td>
      <td>15 nM (reported value)</td>
      <td>None</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>33718.43 g/mole</td>
      <td>59.62%</td>
      <td>1026700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>104</td>
      <td>None</td>
      <td>rGp,rGp,rGp,rAp,rUp,rCp,rCp,rUp,rAp,rAp,rGp,rCp,rUp,rCp,rUp,rAp,rUp,rCp,rGp,rGp,rCp,rUp,rGp,rGp,rAp,rCp,rGp,rAp,rCp,rGp,rGp,rGp,rGp,rAp,rCp,rGp,rCp,rCp,rAp,rCp,rUp,rGp,rGp,rAp,rCp,rUp,rAp,rGp,rGp,rUp,rAp,rAp,rGp,rCp,rCp,rAp,rGp,rGp,rAp,rCp,rCp,rGp,rUp,rAp,rCp,rGp,rUp,rCp,rGp,rGp,rGp,rAp,rGp,rCp,rCp,rGp,rUp,rCp,rAp,rGp,rAp,rAp,rUp,rAp,rAp,rAp,rAp,rGp,rCp,rGp,rGp,rCp,rCp,rUp,rAp,rGp,rCp,rGp,rAp,rUp,rCp,rGp,rAp,rUp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>615</th>
      <td>Hematoporphyrin IX</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/92.png" width="120" ></td>
      <td>Hematoporphyrin IX (HPIX)</td>
      <td>Small Organic</td>
      <td>1.6 µM (reported value)</td>
      <td>Selection Buffer: 100 mM Trisƒ??acetate (pH 8.0), 200 mM sodium acetate, 50 mM potassium acetate, 25 mM magnesium acetate, 5% dioxane</td>
      <td>RT°C</td>
      <td>\n                                Heated to 95°C for 5 min and then slowly cooled to room temperature. \n</td>
      <td></td>
      <td>31939.63 g/mole</td>
      <td>66.67%</td>
      <td>986100 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Forms a parallel G-quartet structure.</td>
      <td>109</td>
      <td>None</td>
      <td>dTp,dAp,dGp,dGp,dGp,dAp,dAp,dTp,dTp,dCp,dGp,dTp,dCp,dGp,dAp,dCp,dGp,dGp,dAp,dTp,dCp,dCp,dCp,dAp,dAp,dTp,dGp,dGp,dGp,dGp,dTp,dCp,dGp,dGp,dGp,dCp,dGp,dGp,dGp,dCp,dCp,dGp,dGp,dGp,dTp,dGp,dTp,dCp,dAp,dTp,dGp,dGp,dTp,dGp,dGp,dAp,dCp,dGp,dGp,dAp,dGp,dAp,dTp,dGp,dGp,dGp,dAp,dCp,dGp,dTp,dAp,dGp,dAp,dGp,dGp,dGp,dCp,dGp,dGp,dTp,dCp,dTp,dGp,dCp,dAp,dGp,dGp,dTp,dCp,dGp,dAp,dCp,dGp,dCp,dAp,dTp,dGp,dCp,dGp,dCp,dCp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>616</th>
      <td>M64RA</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/283.png" width="120" ></td>
      <td>MPT64 Protein</td>
      <td>Protein</td>
      <td>N/A nM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td></td>
      <td>10873.13 g/mole</td>
      <td>48.57%</td>
      <td>339000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Was used in a sandwich assay with M64CA.</td>
      <td>36</td>
      <td>None</td>
      <td>dTp,dGp,dGp,dGp,dAp,dGp,dCp,dTp,dGp,dAp,dTp,dGp,dTp,dCp,dGp,dCp,dAp,dTp,dGp,dGp,dGp,dTp,dTp,dTp,dTp,dGp,dAp,dTp,dCp,dAp,dCp,dAp,dTp,dGp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>617</th>
      <td>Anti -?1-Adrenergic Receptor-Autoantibodies (Aptamer 110)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/274.jpg" width="120" ></td>
      <td>?1-Adrenoceptor Autoantibodies(AABs)</td>
      <td>Protein</td>
      <td>n/a nM (reported value)</td>
      <td>Buffer 1: Tris buffer pH 7.3, 0.1% Tween-20, DMEM, 0.1% Tween-20  Buffer 2: Phosphate buffer pH 7.5, 0.1% Tween-20</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>6480.27 g/mole</td>
      <td>57.14%</td>
      <td>212600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Low affinity aptamers were removed from the column using both buffers.</td>
      <td>21</td>
      <td>Haberland, A., et al. "Aptamer Neutralization of Beta1-Adrenoceptor Autoantibodies Isolated From Patients With Cardiomyopathies." Circulation Research, 109 (2011): 986-992.</td>
      <td>dAp,dCp,dAp,dGp,dTp,dAp,dAp,dCp,dCp,dGp,dCp,dGp,dTp,dGp,dAp,dGp,dGp,dTp,dCp,dGp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>618</th>
      <td>Anti-human platelet antigen antibodies (Trx-HPA-1a)</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/421.png" width="120" ></td>
      <td>Anti-human platelet antigen antibodies (HPA-1a)</td>
      <td>Protein</td>
      <td>N/A nM (reported value)</td>
      <td>Buffer for ELISA assay: PBS.</td>
      <td>20-25°C</td>
      <td>None</td>
      <td></td>
      <td>16956.67 g/mole</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Aptamer consists of a selected dodecapeptide (VVAGDDPREDTW) inserted into the thioredoxin protein with a C-terminal poly-histone tail. The dodecapeptide mimics human platelet antigen-1 and binds to many of the antibodies that target HPA-1a, though not all of them.</td>
      <td>141</td>
      <td>None</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>619</th>
      <td>P2MC</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7174.jpg" width="120" ></td>
      <td>Doxycycline</td>
      <td>Other</td>
      <td>0.780 ± 1.76 nM (reported value)</td>
      <td>50 mM Tris-HCl (pH 8.3), 100 mM KCl, and 20 mM MgCl2</td>
      <td>65°C</td>
      <td>None</td>
      <td></td>
      <td>30882 g/mole</td>
      <td>50.00%</td>
      <td>963900 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>100</td>
      <td>Tickner, Zachary J et al. “Selection of High-Affinity RNA Aptamers That Distinguish between Doxycycline and Tetracycline.” Biochemistry vol. 59,37 (2020): 3473-3486. doi:10.1021/acs.biochem.0c00586</td>
      <td>dGp,dGp,dGp,dAp,dGp,dAp,dCp,dCp,dGp,dCp,dTp,dTp,dAp,dAp,dGp,dCp,dTp,dTp,dAp,dAp,dAp,dGp,dGp,dAp,dGp,dCp,dGp,dGp,dTp,dTp,dAp,dCp,dGp,dAp,dAp,dTp,dGp,dCp,dGp,dAp,dTp,dGp,dAp,dCp,dTp,dTp,dAp,dGp,dGp,dTp,dAp,dCp,dCp,dAp,dTp,dTp,dGp,dCp,dAp,dCp,dTp,dAp,dCp,dGp,dGp,dTp,dAp,dCp,dCp,dTp,dAp,dAp,dAp,dCp,dAp,dGp,dTp,dTp,dCp,dCp,dTp,dTp,dTp,dGp,dGp,dAp,dTp,dCp,dCp,dGp,dAp,dAp,dTp,dTp,dCp,dGp,dCp,dCp,dGp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>620</th>
      <td>FANA R8-3</td>
      <td>Chimeric</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7230.png" width="120" ></td>
      <td>SARS-CoV-2 S Protein</td>
      <td>Protein</td>
      <td>68.1 nM (reported value)</td>
      <td>Buffer: 200 µl PBS (137 mM NaCl, 2.7 mM KCl, 8 mM Na2HPO4, and 2 mM KH2PO4)\npH: 7.4\nIncubation time: 30 mins\n\n</td>
      <td>23°C</td>
      <td>\n                                About 200 pmol of 5′ 32P-labeled FANA starting pool was heat to 90⁰C then snap-cooled on ice. The material was then incubated with 20 pmol of SARS-COV-2 RBD protein that had been attached to Dynabeads™ using the C-terminal His-tag. Incubations were in 200 µl PBS (137 mM NaCl, 2.7 mM KCl, 8 mM Na2HPO4, and 2 mM KH2PO4, pH 7.4) for 30 min with agitation at room temperature.\n \n</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>These are not the common 2'-F-RNA nucleotides, these are 2'-Fluoro-Arabino Nucleic Acid (FANA) nucleotides.</td>
      <td>79</td>
      <td>Ferreira-Bravo, I. A. and J. J. DeStefano. 2021. Xeno-nucleic Acid (XNA) 2'-Fluoro-Arabino Nucleic Acid (FANA) Aptamers to the Receptor Binding Domain of SARS-CoV-2 S Protein Block ACE2 Binding. bioRxiv\nhttps://doi.org/10.1101/2021.07.13.452259\n</td>
      <td>dAp,dAp,dAp,dAp,dGp,dGp,dTp,dAp,dGp,dTp,dGp,dCp,dTp,dGp,dAp,dAp,dTp,dTp,dCp,dGp,dGp,dTp,dCp,dGp,dCp,dGp,dAp,dTp,dTp,dAp,dAp,dCp,dAp,dTp,dTp,dAp,dAp,dAp,dCp,dCp,dGp,dCp,dAp,dTp,dAp,dAp,dAp,dAp,dAp,dGp,dGp,dGp,dTp,dGp,dGp,dCp,dCp,dGp,dGp,dAp,fUp,fUp,fCp,fGp,fCp,fUp,fAp,fUp,fCp,fCp,fAp,fGp,fUp,fUp,fGp,fGp,fCp,fCp,fUp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>621</th>
      <td>Dopamine (dopa2/c.4)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/401.png" width="120" ></td>
      <td>Dopamine</td>
      <td>Small Organic</td>
      <td></td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td>Cross-reactivity was performed on 3-methoxytyramine, epinephrine, homovanillic acid, 3,4-dihydroxyphenylacetic acid (cross-reactivity less than 1% for these reactants) and norepinephrine (3.68%).</td>
      <td>21676.14 g/mole</td>
      <td>62.69%</td>
      <td>652400 L/(mole·cm)</td>
      <td></td>
      <td>1 pM (reported value)</td>
      <td>None</td>
      <td>74</td>
      <td>None</td>
      <td>rGp,rGp,rGp,rAp,rAp,rUp,rUp,rCp,rCp,rGp,rCp,rGp,rUp,rGp,rUp,rGp,rCp,rGp,rCp,rCp,rGp,rCp,rGp,rGp,rAp,rAp,rGp,rAp,rGp,rGp,rGp,rAp,rAp,rUp,rAp,rUp,rAp,rGp,rAp,rGp,rGp,rCp,rCp,rAp,rGp,rCp,rAp,rCp,rAp,rUp,rAp,rGp,rUp,rGp,rAp,rGp,rGp,rCp,rCp,rCp,rUp,rCp,rCp,rUp,rCp,rCp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>622</th>
      <td>Microcystin-LR (AN6)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/463.png" width="120" ></td>
      <td>Microcystin-LR</td>
      <td>Small Organic</td>
      <td>50 nM (reported value)</td>
      <td>None</td>
      <td>25°C</td>
      <td>None</td>
      <td>Binds to congener microcystin-LA with three-fold reduced affinity (Kd of 158). No binding observed to microcystin-YR.</td>
      <td>18167.79 g/mole</td>
      <td>55.00%</td>
      <td>557700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>60</td>
      <td>None</td>
      <td>dGp,dGp,dCp,dGp,dCp,dCp,dAp,dAp,dAp,dCp,dAp,dGp,dGp,dAp,dCp,dCp,dAp,dCp,dCp,dAp,dTp,dGp,dAp,dCp,dAp,dAp,dTp,dTp,dAp,dCp,dCp,dCp,dAp,dTp,dAp,dCp,dCp,dAp,dCp,dCp,dTp,dCp,dAp,dTp,dTp,dAp,dTp,dGp,dCp,dCp,dCp,dCp,dAp,dTp,dCp,dTp,dCp,dCp,dGp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>623</th>
      <td>Plasmodium vivax lactate dehydrogenase (pL1)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/308.png" width="120" ></td>
      <td>Plasmodium vivax lactate dehydrogenase  (PvLDH)</td>
      <td>Protein</td>
      <td>16.8 nM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td></td>
      <td>11163.29 g/mole</td>
      <td>55.56%</td>
      <td>340000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>For details on pL1's use in colorimetric testing, see:   Lee S, Manjunatha DH, Jeon W, Ban C (2014) Cationic Surfactant-Based Colorimetric Detection of Plasmodium Lactate Dehydrogenase, a Biomarker for Malaria, Using the Specific DNA Aptamer. PLoS ONE 9(7): e100847. doi:10.1371/journal.pone.0100847</td>
      <td>36</td>
      <td>None</td>
      <td>dGp,dTp,dTp,dCp,dGp,dAp,dTp,dTp,dGp,dGp,dAp,dTp,dTp,dGp,dTp,dGp,dCp,dCp,dGp,dGp,dAp,dAp,dGp,dTp,dGp,dCp,dTp,dGp,dGp,dCp,dTp,dCp,dGp,dAp,dAp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>624</th>
      <td>ErbB2 Tyrosine Kinase Receptor (Ph7-2)</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/414.png" width="120" ></td>
      <td>ErbB2 Tyrosine Kinase Receptor</td>
      <td>Protein</td>
      <td>N/A nM (reported value)</td>
      <td>N/A</td>
      <td>N/A°C</td>
      <td>None</td>
      <td>Binds strongly to breast cancer cell lines SKBR3 and T47D.</td>
      <td>911.12 g/mole</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>7</td>
      <td>None</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>625</th>
      <td>NF-kappa B (Aptamer 3)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/141.png" width="120" ></td>
      <td>NF-kappa B</td>
      <td>Protein</td>
      <td>1 nM (reported value)</td>
      <td>Buffer A (20 mM HEPES (pH 7.5), 0.4 M NaCl, 2 mM EDTA, 5% v/v glycerol, 1 mM DTT, 0.5 mM PMSF, 1 ug/mL pepstatin, 1 ug/mL leupeptin)    Buffer B (20 mM HEPES (pH 7.5), 2 mM EDTA, 5% v/v glycerol, 1 mM DTT, 0.5 mM PMSF, 1 ug/mL pepstatin, 1 ug/mL leupeptin)    Selection Buffer (10 mM HEPES (pH 7.5) 0.1 M NaCl, 1 mM DTT)</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>31122.71 g/mole</td>
      <td>41.24%</td>
      <td>981700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>97</td>
      <td>Lebriska and Maher. "Selection and Characterization of an RNA Decoy for Transcription Factor NF-B." Journal of Biochemistry, 38(1999): 3168-3174.</td>
      <td>rGp,rGp,rGp,rAp,rUp,rAp,rUp,rCp,rCp,rUp,rCp,rGp,rAp,rGp,rCp,rAp,rUp,rAp,rAp,rGp,rAp,rAp,rAp,rCp,rAp,rAp,rGp,rAp,rUp,rAp,rGp,rAp,rUp,rCp,rCp,rUp,rGp,rAp,rAp,rAp,rCp,rUp,rGp,rUp,rUp,rUp,rUp,rAp,rAp,rGp,rGp,rUp,rUp,rGp,rGp,rCp,rCp,rGp,rAp,rUp,rCp,rUp,rUp,rCp,rUp,rGp,rCp,rUp,rCp,rGp,rAp,rGp,rAp,rAp,rUp,rGp,rCp,rAp,rUp,rGp,rAp,rAp,rGp,rCp,rGp,rUp,rUp,rCp,rCp,rAp,rUp,rAp,rUp,rUp,rUp,rUp,rUp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>626</th>
      <td>Bisphenol A(#3)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/178.png" width="120" ></td>
      <td>Bisphenol A(BPA)</td>
      <td>Small Organic</td>
      <td>8.3 nM (reported value)</td>
      <td>None</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>33800.83 g/mole</td>
      <td>63.30%</td>
      <td>1036700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>109</td>
      <td>None</td>
      <td>dGp,dGp,dGp,dCp,dCp,dGp,dTp,dTp,dCp,dGp,dAp,dAp,dCp,dAp,dCp,dGp,dAp,dGp,dCp,dAp,dTp,dGp,dCp,dCp,dGp,dGp,dTp,dGp,dGp,dGp,dTp,dGp,dGp,dTp,dCp,dAp,dGp,dGp,dTp,dGp,dGp,dGp,dAp,dTp,dAp,dGp,dCp,dGp,dTp,dTp,dCp,dCp,dGp,dCp,dGp,dTp,dAp,dTp,dGp,dGp,dCp,dCp,dCp,dAp,dGp,dCp,dGp,dCp,dAp,dTp,dCp,dAp,dCp,dGp,dGp,dGp,dTp,dTp,dCp,dGp,dCp,dAp,dCp,dCp,dAp,dGp,dGp,dAp,dCp,dAp,dGp,dTp,dAp,dCp,dTp,dCp,dAp,dGp,dGp,dTp,dCp,dAp,dTp,dCp,dCp,dTp,dAp,dGp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>627</th>
      <td>Mouse Skin Vasculature</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/246.png" width="120" ></td>
      <td>Mouse hypodermis</td>
      <td>Tissue</td>
      <td>N/A nM (reported value)</td>
      <td>None</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>1467.78 g/mole</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>13</td>
      <td>D Rajotte et al. "  Molecular Heterogeneity of the Vascular Endothelium Revealed by In Vivo  Phage Display." Journal of Clinical Investigations, 102(1998): 430-437.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>628</th>
      <td>HIV-RT 3.17</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7160.jpg" width="120" ></td>
      <td>HIV reverse transcriptase (HIV RT)</td>
      <td>Cells</td>
      <td>0.38 nM (reported value)</td>
      <td>20 mM Tris-HCl [pH 7.4], 140 mM NaCl, 5 mM KCl, 1 mM MgCl2, and 1 mM CaCl2</td>
      <td>25°C</td>
      <td>None</td>
      <td></td>
      <td>15383.1 g/mole</td>
      <td>24.00%</td>
      <td>491500 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>50</td>
      <td>Dunn MR, McCloskey CM, Buckley P, Rhea K, Chaput JC. Generating Biologically Stable TNA Aptamers that Function with High Affinity and Thermal Stability. J Am Chem Soc. 2020 Apr 29;142(17):7721-7724. doi: 10.1021/jacs.0c00641. Epub 2020 Apr 20. PMID: 32298104.</td>
      <td>dTp,dTp,dAp,dTp,dGp,dTp,dAp,dGp,dCp,dAp,dTp,dTp,dTp,dAp,dTp,dGp,dAp,dAp,dAp,dTp,dTp,dTp,dTp,dTp,dAp,dAp,dAp,dTp,dCp,dAp,dAp,dTp,dTp,dTp,dAp,dCp,dTp,dAp,dTp,dTp,dGp,dGp,dTp,dGp,dGp,dTp,dAp,dTp,dCp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>629</th>
      <td>A15</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/558.png" width="120" ></td>
      <td>Staphylococcus aureus enterotoxin A (SEA)</td>
      <td>Protein</td>
      <td>48.57 nM (reported value)</td>
      <td>20 mM Tris, 100 mM NaCl, 5 mM KCl, 1 mM CaCl2 and 1 mM MgCl2 ƒ?› 6H2O, pH 7.4</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>24604.02 g/mole</td>
      <td>47.50%</td>
      <td>768000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>ssDNA aptamers specific for SEA were selected using a 12-round SELEX procedure.  A15 was used successfully in fluorescent bioassay to detect SEA in the food sample with a detection limit of 8.7 x 10^(-3) ug/mL.</td>
      <td>240</td>
      <td>Huang, Y. et al. "Selection, identification and application of a DNA aptamer against Staphylococcus aureus enterotoxin A." Anal. Methods, 2014, 6, 690-697.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>630</th>
      <td>M1</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7142.jpg" width="120" ></td>
      <td>Maltose-binding protein (MBP)</td>
      <td>Protein</td>
      <td>173 nM (reported value)</td>
      <td>20 mM Tris-HCl, 100 mM NaCl, 5 mM MgCl2, and 0.1% bovine\nserum albumin with 10µg/mL yeast tRNA and 10µg/mL salmon sperm DNA</td>
      <td>25°C</td>
      <td>None</td>
      <td></td>
      <td>24292.8 g/mole</td>
      <td>40.51%</td>
      <td>782600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>79</td>
      <td>Wang Y, Li Z, Yu H. Aptamer-Based Western Blot for Selective Protein Recognition. Front Chem. 2020 Oct 29;8:570528. doi: 10.3389/fchem.2020.570528. PMID: 33195056; PMCID: PMC7658645.</td>
      <td>dAp,dTp,dAp,dCp,dGp,dAp,dCp,dTp,dCp,dAp,dCp,dTp,dAp,dTp,dTp,dAp,dGp,dGp,dGp,dAp,dCp,dCp,dAp,dCp,dAp,dTp,dAp,dAp,dAp,dCp,dAp,dTp,dGp,dCp,dAp,dTp,dCp,dGp,dCp,dTp,dCp,dAp,dAp,dTp,dAp,dAp,dCp,dGp,dGp,dGp,dAp,dTp,dAp,dTp,dTp,dGp,dTp,dTp,dGp,dTp,dGp,dCp,dTp,dAp,dGp,dAp,dCp,dTp,dAp,dCp,dTp,dGp,dAp,dCp,dTp,dAp,dCp,dAp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>631</th>
      <td>HIV co-receptor CCR5 (BY6M4)</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/461.png" width="120" ></td>
      <td>CCR5</td>
      <td>Protein</td>
      <td>N/A nM (reported value)</td>
      <td>RPMI-10 medium</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>6018.13 g/mole</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Decreased induction of R5-tropic reporter virus by 95%. Did not affect CD4 or CXCR4 expression.</td>
      <td>44</td>
      <td>None</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>632</th>
      <td>Protein Kinase C-d</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/74.png" width="120" ></td>
      <td>Protein Kinase C-d</td>
      <td>Protein</td>
      <td>122 nM (reported value)</td>
      <td>N/A</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>23218.06 g/mole</td>
      <td>60.00%</td>
      <td>732900 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>High-affinity DNA aptamers for PKC-delta were selected by capillary electrophoresis based on SELEX.</td>
      <td>75</td>
      <td>Mallikaratchy, P., et al. "Selection of DNA ligands for protein kinase C-d." Chemical Communications, 2006 (2006): 3229-3231.</td>
      <td>dGp,dCp,dCp,dAp,dGp,dGp,dGp,dGp,dTp,dTp,dCp,dCp,dAp,dCp,dTp,dAp,dCp,dGp,dTp,dAp,dGp,dAp,dAp,dCp,dAp,dCp,dGp,dAp,dCp,dGp,dGp,dGp,dAp,dAp,dTp,dAp,dCp,dTp,dGp,dAp,dCp,dTp,dCp,dTp,dCp,dCp,dCp,dCp,dCp,dAp,dTp,dGp,dTp,dAp,dCp,dCp,dAp,dGp,dGp,dGp,dGp,dGp,dCp,dAp,dGp,dAp,dGp,dAp,dGp,dAp,dAp,dGp,dGp,dGp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>633</th>
      <td>Anti-NF-kB p65 (D1)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/389.png" width="120" ></td>
      <td>Anti-NF-kB p65</td>
      <td>Protein</td>
      <td>149 nM (reported value)</td>
      <td>Selection buffer: 10mM HEPES at pH 7.5, 100mM NaCl, 1mM DTT</td>
      <td>37°C</td>
      <td>None</td>
      <td>Binds to both p65 and p50, with kd of 149nM and 190nM respectively.</td>
      <td>27468.57 g/mole</td>
      <td>49.41%</td>
      <td>858400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>85</td>
      <td>Wurster, S. E, and Maher, L. J. "Selection and characterization of anti-NF-kB p65 aptamers." RNA, 14(2008): 1037-1047.</td>
      <td>rGp,rCp,rAp,rUp,rGp,rCp,rAp,rGp,rUp,rGp,rUp,rCp,rUp,rAp,rUp,rUp,rCp,rUp,rCp,rGp,rAp,rGp,rUp,rAp,rGp,rCp,rGp,rAp,rUp,rCp,rGp,rUp,rUp,rGp,rAp,rAp,rGp,rGp,rGp,rGp,rUp,rAp,rUp,rAp,rAp,rGp,rGp,rUp,rUp,rGp,rGp,rCp,rAp,rGp,rAp,rUp,rCp,rGp,rCp,rUp,rAp,rGp,rCp,rAp,rUp,rGp,rCp,rAp,rAp,rCp,rUp,rGp,rAp,rCp,rUp,rCp,rGp,rGp,rAp,rUp,rAp,rAp,rGp,rCp,rAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>634</th>
      <td>YFL-4</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7236.png" width="120" ></td>
      <td>Lactoferrin</td>
      <td>Protein</td>
      <td>0.953 nM (reported value)</td>
      <td>1x PBSMC</td>
      <td>37°C</td>
      <td>\n                                IDT \n</td>
      <td>Does not interact with related proteins BSA, casein, beta-lactoglobulin, or alpha-lactalbumin.</td>
      <td>18095 g/mole</td>
      <td>45.76%</td>
      <td>558200 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>59</td>
      <td>Yu, F., Li, H., Sun, W., Xu, D. and He, F., 2019. Rapid selection of aptamers based on protein microarray. RSC advances, 9(17), pp.9762-9768.\n</td>
      <td>dGp,dCp,dAp,dGp,dGp,dAp,dCp,dAp,dCp,dCp,dGp,dTp,dAp,dAp,dCp,dAp,dCp,dGp,dGp,dGp,dCp,dTp,dGp,dAp,dTp,dGp,dCp,dTp,dCp,dTp,dCp,dTp,dTp,dTp,dAp,dTp,dTp,dTp,dTp,dAp,dCp,dCp,dTp,dAp,dAp,dAp,dTp,dAp,dAp,dAp,dGp,dTp,dGp,dTp,dCp,dCp,dTp,dGp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>635</th>
      <td>Cd2+ (AR1)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/205.png" width="120" ></td>
      <td>Cd2+</td>
      <td>Small Organic</td>
      <td>~100 µM (reported value)</td>
      <td>None</td>
      <td>23°C</td>
      <td>None</td>
      <td></td>
      <td>31742.3 g/mole</td>
      <td>54.08%</td>
      <td>985700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>98</td>
      <td>Seetharaman et al. "Immobolized RNA switches for the analysis of complex chemical and biological mixtures." Nature Biotechnology, 19(2001): 336-341.</td>
      <td>rGp,rGp,rAp,rUp,rAp,rAp,rUp,rAp,rGp,rCp,rCp,rGp,rUp,rAp,rGp,rGp,rUp,rUp,rGp,rCp,rGp,rAp,rAp,rAp,rGp,rCp,rGp,rAp,rCp,rCp,rCp,rUp,rGp,rAp,rUp,rGp,rAp,rGp,rAp,rAp,rAp,rGp,rCp,rCp,rAp,rAp,rAp,rGp,rCp,rCp,rGp,rUp,rAp,rGp,rCp,rGp,rCp,rAp,rGp,rAp,rUp,rGp,rAp,rUp,rCp,rUp,rCp,rGp,rCp,rCp,rAp,rUp,rCp,rAp,rGp,rUp,rAp,rCp,rCp,rGp,rAp,rAp,rAp,rCp,rGp,rGp,rUp,rAp,rGp,rCp,rGp,rAp,rGp,rAp,rGp,rCp,rUp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>636</th>
      <td>Tr-8</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7093.jpg" width="120" ></td>
      <td>CD25</td>
      <td>Protein</td>
      <td>35.7 nM (reported value)</td>
      <td>HEPES-buffered saline, pH=7.5 with 2mmol/L CaCl2</td>
      <td>37°C</td>
      <td>\n                                Transcribed RNA aptamer library was folded at 2 µM concentration in AIM-V (BSA) medium by step-wise incubation at 95°C for 5 minutes, 65°C for 10 minutes, followed by 37°C for 20\nminutes \n</td>
      <td></td>
      <td>13869.47 g/mole</td>
      <td>70.59%</td>
      <td>605800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Structure Unconfirmed</td>
      <td>51</td>
      <td>Veeramani, Suresh,et al."An RNA aptamer-based biomarker platform demonstrates high soluble CD25 occupancy by IL2 in the serum of follicular lymphoma patients" American Association for Cancer Research 7, (2019) 1511-1522</td>
      <td>rGp,rGp,rGp,rAp,rGp,rGp,rAp,rCp,rGp,rAp,rUp,rGp,rCp,rGp,rGp,rGp,rCp,rCp,rGp,rUp,rUp,rGp,rUp,rUp,rGp,rUp,rGp,rUp,rGp,rCp,rCp,rGp,rCp,rCp,rCp,rCp,rAp,rGp,rAp,rCp,rGp,rAp,rCp,rUp,rCp,rGp,rCp,rCp,rCp,rGp,rAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>637</th>
      <td>A14</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/566.jpg" width="120" ></td>
      <td>Penetrating the Blood Brain Barrier (BBB)</td>
      <td>Other</td>
      <td>nM (reported value)</td>
      <td>10mM HEPES, pH 7.4, 50mM NaCl, 1mM MgCl2, 1mM CaCl2, 0.01% BSA</td>
      <td>37°C</td>
      <td>\n                                Denature at 94C for 3 minutes, and then slowly refold at room temperature \n</td>
      <td></td>
      <td>17891.96 g/mole</td>
      <td>78.18%</td>
      <td>521500 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>180</td>
      <td>Congsheng Cheng, Yong Hong Chen, Kim A Lennox, Mark A Behlke, Beverly L Davidson, "In vivo SELEX for Identification of Brain-penetrating Aptamers" Molecular Therapy-Nucleic Acids (2013)2, e67, American Society of Gene & Cell Therapy</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>638</th>
      <td>Apt37</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/645.jpg" width="120" ></td>
      <td>Thymidine Kinase 1</td>
      <td>Protein</td>
      <td>570 pM (reported value)</td>
      <td>PBS</td>
      <td>°C</td>
      <td>None</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>40</td>
      <td>Aptamer-Based Sandwich Assay for Measurement of Thymidine Kinase 1 in Serum of Cancerous Patients  Mahmood Nazari, Seyed Latif Mousavi Gargari, Abbas Sahebghadam Lotfi, Mohammad Javad Rassaee, and Ramezan Ali Taheri  Biochemistry 2019 58 (18), 2373-2383</td>
      <td>dCp,dGp,dGp,dTp,dGp,dCp,dAp,dGp,dTp,dGp,dGp,dAp,dTp,dAp,dCp,dAp,dTp,dGp,dCp,dCp,dAp,dGp,dCp,dCp,dGp,dTp,dAp,dGp,dCp,dCp,dAp,dTp,dCp,dGp,dTp,dGp,dGp,dAp,dTp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>639</th>
      <td>Histone H3R8Me2sym (Clone 1)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/224.png" width="120" ></td>
      <td>Histone H3R8Me2sym</td>
      <td>Peptide</td>
      <td>12 nM (reported value)</td>
      <td>N/A</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>26323.83 g/mole</td>
      <td>64.20%</td>
      <td>784200 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Truncated versions of the aptamer were investigated, but exhibited lower affinities for the target.</td>
      <td>81</td>
      <td>Hyun et al. "An RNA Aptamer That Selectively Recognizes Symmetric Dimethylation of Arginine 8 in the Histone H3 N-Terminal Peptide." Nucleic Acid Therapeutics, 21 (2011): 157-163.</td>
      <td>rGp,rGp,rGp,rAp,rCp,rGp,rCp,rGp,rUp,rGp,rGp,rUp,rAp,rCp,rCp,rGp,rAp,rUp,rGp,rGp,rGp,rUp,rCp,rAp,rGp,rCp,rAp,rUp,rGp,rUp,rAp,rGp,rCp,rCp,rAp,rGp,rGp,rCp,rAp,rGp,rGp,rGp,rCp,rCp,rGp,rUp,rGp,rUp,rGp,rAp,rGp,rCp,rUp,rUp,rGp,rUp,rGp,rCp,rUp,rGp,rAp,rUp,rGp,rUp,rGp,rAp,rGp,rCp,rUp,rUp,rCp,rCp,rGp,rCp,rGp,rGp,rGp,rGp,rAp,rUp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>640</th>
      <td>Cyanocobalamin (35-mer)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/105.png" width="120" ></td>
      <td>Cyanocobalamin</td>
      <td>Small Organic</td>
      <td>88 nM (reported value)</td>
      <td>Column Buffers (1 M LiC1, 5 mM MgC12, and 25 mM HEPES (pH 7.4) or 1 M NaCl,  5 mM MgC12, and 25 mM HEPES (pH  7.4))</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>11263.9 g/mole</td>
      <td>60.00%</td>
      <td>341600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>35</td>
      <td>Lorsch and Szostak. "In vitro selection of RNA aptamers specific for cyanocobalamin." Biochemistry, 33 (1994): 973-982.</td>
      <td>rCp,rCp,rGp,rGp,rUp,rGp,rCp,rGp,rCp,rAp,rUp,rAp,rAp,rCp,rCp,rAp,rCp,rCp,rUp,rCp,rAp,rGp,rUp,rGp,rCp,rGp,rAp,rGp,rCp,rAp,rAp,rGp,rGp,rAp,rAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>641</th>
      <td>21st clone</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7096.png" width="120" ></td>
      <td>Shp2</td>
      <td>Protein</td>
      <td>24.9 nM (reported value)</td>
      <td>PBS with 0.5 mM MgCl2 and 0.1 mg/mL yeast tRNA, RT</td>
      <td>23°C</td>
      <td>\n                                Not applicable \n</td>
      <td></td>
      <td>24459.9 g/mole</td>
      <td>48.75%</td>
      <td>822950 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Structure not confirmed, no refolding program.</td>
      <td>80</td>
      <td>Zhang, Wei Yun, et al.  “Highly parallel single-molecule amplification approach based on agarose droplet polymerase chain reaction for efficient and cost-effective aptamer selection.”  Analytical Chemistry 84, (2012): 350-355.</td>
      <td>dAp,dGp,dCp,dGp,dTp,dCp,dGp,dAp,dAp,dTp,dAp,dCp,dCp,dAp,dCp,dTp,dAp,dCp,dAp,dGp,dCp,dCp,dCp,dGp,dCp,dAp,dCp,dTp,dCp,dAp,dAp,dCp,dCp,dAp,dCp,dCp,dGp,dTp,dTp,dCp,dCp,dTp,dTp,dTp,dGp,dTp,dTp,dTp,dAp,dAp,dTp,dTp,dTp,dTp,dGp,dCp,dAp,dCp,dAp,dTp,dCp,dTp,dAp,dAp,dTp,dGp,dGp,dAp,dGp,dCp,dTp,dCp,dGp,dTp,dGp,dGp,dTp,dCp,dAp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>642</th>
      <td>Insulin Receptor Antibody (MA20) (#1)</td>
      <td>2'-NH2-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/572" width="120" ></td>
      <td>Insulin Receptor Antibody (MA20)</td>
      <td>Antibody</td>
      <td>30 nM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td></td>
      <td>50020.06 g/mole</td>
      <td>58.14%</td>
      <td>843600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>86</td>
      <td>None</td>
      <td>rGp,rGp,rGp,rAp,rGp,rAp,rGp,rGp,rGp,rAp,rAp,rGp,rGp,rGp,rGp,rGp,rGp,rGp,rGp,rGp,rGp,rAp,rAp,rAp,rAp,rAp,rAp,rGp,rAp,rAp,rGp,rAp,rAp,rGp,rGp,rAp,rAp,rAp,rAp,rAp,rGp,rAp,rGp,rGp,rGp,rAp,rGp,rGp,rAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>643</th>
      <td>HPV-HF cell line (20)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/317.png" width="120" ></td>
      <td>nontumorigenic HF cell line</td>
      <td>Cells</td>
      <td>1.6 nM (reported value)</td>
      <td>1% BSA in PBS with salts</td>
      <td>4°C</td>
      <td>None</td>
      <td></td>
      <td>27123.72 g/mole</td>
      <td>44.83%</td>
      <td>880300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Aptamer 20 has higher affinity than aptamer 14 against the HF cell line but does not internalize.</td>
      <td>87</td>
      <td>Graham JC, Zarbl H (2012) Use of Cell-SELEX to Generate DNA Aptamers as Molecular Probes of HPV-Associated Cervical Cancer Cells. PLoS ONE 7(4):  e36103. doi:10.1371/journal.pone.0036103</td>
      <td>dAp,dTp,dAp,dCp,dCp,dAp,dGp,dCp,dTp,dTp,dAp,dTp,dTp,dCp,dAp,dAp,dTp,dTp,dGp,dGp,dGp,dGp,dAp,dGp,dGp,dGp,dAp,dGp,dAp,dCp,dAp,dCp,dAp,dGp,dTp,dCp,dAp,dTp,dGp,dGp,dAp,dGp,dCp,dAp,dGp,dTp,dTp,dAp,dTp,dTp,dAp,dGp,dGp,dGp,dTp,dGp,dTp,dAp,dCp,dCp,dGp,dGp,dGp,dTp,dGp,dTp,dAp,dGp,dTp,dAp,dGp,dAp,dTp,dAp,dGp,dTp,dAp,dAp,dGp,dTp,dGp,dCp,dAp,dAp,dTp,dCp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>644</th>
      <td>Anti Lung Carcinoma ( EJ4)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/507.png" width="120" ></td>
      <td>Lung Adenocarcinoma (H23)</td>
      <td>Cells</td>
      <td>60 nM (reported value)</td>
      <td>0.45% w/v glucose   5 mM  MgCl2 in PBS    1 mg/mL tRNA    1 mg/mL BSA     incubate with cells for 30 minutes</td>
      <td>4°C</td>
      <td>\n                                denature by heating at 95C for 5  minutes, followed by rapid  cooling on ice \n</td>
      <td>HBE 135-E6/E7 (normal epithelial lung) cells used as counter targets</td>
      <td>12668.28 g/mole</td>
      <td>53.66%</td>
      <td>402800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Reference did not list what primer sequences they used. The Author only mentioned that the primers were designed using IDTƒ??s oligo analyzer and met the following characteristics: a minimum hairpin structure, similar melting temperatures and minimal base pairing.</td>
      <td>41</td>
      <td>Jimenez et al. Generation of Lung Adenocarcinoma DNA Aptamers for  Cancer Studies (PLOSone)</td>
      <td>dGp,dAp,dAp,dGp,dAp,dCp,dGp,dAp,dGp,dCp,dGp,dGp,dCp,dGp,dAp,dGp,dTp,dGp,dTp,dTp,dAp,dTp,dTp,dAp,dCp,dGp,dCp,dTp,dTp,dGp,dGp,dAp,dAp,dAp,dCp,dAp,dAp,dCp,dCp,dCp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>645</th>
      <td>SMZ1</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7147.jpg" width="120" ></td>
      <td>Sulfamethazine (SMZ)</td>
      <td>Small Organic</td>
      <td>167.8 nM (reported value)</td>
      <td>20 mM Tris-HCl, 100 mM NaCl, 2 mM MgCl2, 5 mM KCl, 1 mM\nCaCl2, pH 7.4</td>
      <td>25°C</td>
      <td>None</td>
      <td></td>
      <td>24543.9 g/mole</td>
      <td>58.75%</td>
      <td>738000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>80</td>
      <td>Kou, Q., Wu, P., Sun, Q., Li, C., Zhang, L., Shi, H., … Le, T. (2020). Selection and truncation of aptamers for ultrasensitive detection of sulfamethazine using a fluorescent biosensor based on graphene oxide. Analytical and Bioanalytical Chemistry. doi:10.1007/s00216-020-03044-2</td>
      <td>dGp,dAp,dCp,dAp,dGp,dGp,dCp,dAp,dGp,dGp,dAp,dCp,dAp,dCp,dCp,dGp,dTp,dAp,dAp,dCp,dGp,dTp,dTp,dAp,dGp,dAp,dCp,dGp,dCp,dTp,dGp,dGp,dAp,dCp,dCp,dGp,dCp,dGp,dTp,dGp,dTp,dGp,dAp,dTp,dCp,dGp,dGp,dGp,dTp,dCp,dAp,dGp,dCp,dTp,dGp,dCp,dAp,dAp,dTp,dTp,dCp,dTp,dGp,dCp,dTp,dAp,dCp,dCp,dTp,dCp,dCp,dCp,dTp,dCp,dCp,dTp,dCp,dTp,dTp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>646</th>
      <td>Profenofos (SS4-54)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/297.png" width="120" ></td>
      <td>Profenofos</td>
      <td>Small Organic</td>
      <td>1.25 µM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td>This aptamer binds preferentially to the organophosphates phorate, profenofos, benzoate and omethoate. It has reduced binding to imidacloprid, acetamiprid, chlorpyrifos, methamidophos, dichlorvos, parathion, dimethoate, monocrotophos and phoxim.</td>
      <td>16675.9 g/mole</td>
      <td>46.30%</td>
      <td>517600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>"A molecular beacon (5' FAM-CTGCACAAGAATCGCTGCAG-3') was designed, which was partly complementary with the middle constant nucleotides-sequence of the selected ssDNAs."</td>
      <td>54</td>
      <td>None</td>
      <td>dAp,dAp,dGp,dCp,dTp,dTp,dTp,dTp,dTp,dTp,dGp,dAp,dCp,dTp,dGp,dAp,dCp,dTp,dGp,dCp,dAp,dGp,dCp,dGp,dAp,dTp,dTp,dCp,dTp,dTp,dGp,dAp,dTp,dCp,dGp,dCp,dCp,dAp,dCp,dGp,dGp,dTp,dCp,dTp,dGp,dGp,dAp,dAp,dAp,dAp,dAp,dGp,dAp,dGp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>647</th>
      <td>Hen Egg White Lysozyme (cyt7-2)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/259.png" width="120" ></td>
      <td>Hen Egg White Lysozyme</td>
      <td>Protein</td>
      <td>2.5 µM (reported value)</td>
      <td>None</td>
      <td>25°C</td>
      <td>\n                                Denatured at 70° C for 3 minutes in 5 uL of water. Then, allowed to cool to RT. \n</td>
      <td></td>
      <td>30571.4 g/mole</td>
      <td>49.47%</td>
      <td>942500 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Contains a L1 ligase ribozyme motif.</td>
      <td>106</td>
      <td>Robertson and Ellington. "In vitro selection of nucleoprotein enzymes." Nature Biotechnology, 19(2001): 650-655.    Hesselberth et al. "Simultaneous detection of diverse analytes with an aptazyme ligase array." Analytical Biochemistry, 312(2003): 106-112.</td>
      <td>rGp,rGp,rAp,rCp,rCp,rUp,rCp,rGp,rGp,rCp,rGp,rAp,rAp,rAp,rGp,rCp,rUp,rAp,rAp,rCp,rGp,rUp,rCp,rUp,rCp,rAp,rUp,rGp,rGp,rCp,rUp,rAp,rAp,rAp,rUp,rUp,rGp,rCp,rCp,rAp,rUp,rGp,rUp,rUp,rGp,rCp,rUp,rAp,rCp,rAp,rAp,rAp,rUp,rGp,rAp,rUp,rAp,rUp,rGp,rAp,rCp,rUp,rAp,rGp,rAp,rGp,rAp,rGp,rGp,rUp,rUp,rAp,rGp,rGp,rUp,rGp,rCp,rCp,rUp,rCp,rGp,rUp,rGp,rAp,rUp,rGp,rUp,rCp,rCp,rAp,rGp,rUp,rCp,rGp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>648</th>
      <td>HBA1</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7187.png" width="120" ></td>
      <td>Avian Influenza Virus Subtype H5N1</td>
      <td>Other</td>
      <td>70 nM (reported value)</td>
      <td>Tris buffer (20 mM, pH 7.4).</td>
      <td>Not given.°C</td>
      <td>None</td>
      <td></td>
      <td>18601 g/mole</td>
      <td>70.00%</td>
      <td>559900 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>HBA1 was found to inhibit hemagglutination, which is caused by the virus surface protein hemagglutinin. This aptamer can be used as both a diagnostic and a therapeutic agent for H5N1.</td>
      <td>60</td>
      <td>Kim, S.; Choi, J.; Kim, A.; Lee, S.; Yoon, M. Development of ssDNA Aptamers for Diagnosis and Inhibition of the Highly Pathogenic Avian Influenza Virus Subtype H5N1. Biomolecules 2020 July 28; 10(8): 1116. DOI: 10.3390/biom10081116.</td>
      <td>dAp,dTp,dGp,dCp,dGp,dGp,dAp,dTp,dCp,dCp,dCp,dGp,dCp,dGp,dCp,dGp,dCp,dGp,dAp,dCp,dCp,dGp,dTp,dGp,dTp,dCp,dAp,dGp,dCp,dGp,dGp,dGp,dGp,dAp,dCp,dTp,dAp,dGp,dCp,dGp,dGp,dTp,dGp,dTp,dAp,dGp,dCp,dGp,dCp,dGp,dAp,dAp,dGp,dCp,dTp,dTp,dGp,dCp,dGp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>649</th>
      <td>ppERK2/ERK2</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/73.jpg" width="120" ></td>
      <td>ppERK2/ERK2</td>
      <td>Protein</td>
      <td>4.7 nM (reported value)</td>
      <td>Selection Buffer (10 mM HEPES (pH 7.5) and 10 mM MgCl2)</td>
      <td>30°C</td>
      <td>None</td>
      <td></td>
      <td>39558.93 g/mole</td>
      <td>47.15%</td>
      <td>1233700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>123</td>
      <td>Seiwert, S., et al. "RNA aptamers as pathway-specific MAP kinase inhibitors." Chemistry and Biology, 7 (2000): 833-843.</td>
      <td>rGp,rGp,rGp,rAp,rGp,rAp,rGp,rCp,rCp,rAp,rUp,rAp,rCp,rCp,rUp,rGp,rAp,rCp,rAp,rAp,rAp,rGp,rAp,rCp,rGp,rCp,rUp,rAp,rGp,rCp,rGp,rAp,rAp,rUp,rUp,rGp,rGp,rUp,rUp,rCp,rCp,rUp,rCp,rAp,rCp,rUp,rCp,rAp,rAp,rAp,rAp,rGp,rUp,rAp,rGp,rGp,rGp,rGp,rAp,rAp,rAp,rGp,rCp,rGp,rUp,rUp,rAp,rUp,rUp,rAp,rAp,rGp,rAp,rAp,rAp,rCp,rCp,rAp,rAp,rAp,rAp,rUp,rUp,rUp,rGp,rAp,rCp,rAp,rGp,rGp,rUp,rUp,rAp,rCp,rGp,rCp,rAp,rUp,rCp,rCp,rUp,rGp,rCp,rAp,rUp,rCp,rCp,rUp,rCp,rCp,rAp,rGp,rGp,rUp,rUp,rAp,rCp,rGp,rCp,rAp,rUp,rCp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>650</th>
      <td>Trypanosoma cruzi-Fibronectin Parasite Receptor (F4)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/166.png" width="120" ></td>
      <td>Trypanosoma cruzi-Fibronectin Parasite Receptor</td>
      <td>Cells</td>
      <td>124 nM (reported value)</td>
      <td>50 uM RNA containing 20 mM glucose.</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>34315.32 g/mole</td>
      <td>53.27%</td>
      <td>1039800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>107</td>
      <td>None</td>
      <td>rAp,fCp,fCp,rGp,rAp,rGp,fUp,fCp,fCp,rAp,rGp,rAp,rAp,rGp,fCp,fUp,fUp,rGp,fUp,rAp,rGp,fUp,rAp,fCp,fUp,rGp,rAp,fCp,fCp,fCp,fUp,fUp,fCp,fCp,fCp,rGp,rAp,fCp,rGp,fCp,rAp,fCp,rGp,rGp,rAp,fCp,fCp,rAp,rGp,fUp,rGp,rAp,fUp,rGp,fCp,fCp,rAp,rAp,fCp,fCp,fCp,rAp,fCp,fCp,fCp,rGp,fCp,fCp,fUp,rAp,rGp,rAp,fUp,rGp,rGp,rAp,rGp,fUp,fUp,rGp,rAp,rAp,fUp,fUp,fCp,fUp,fCp,fCp,fCp,fUp,rAp,fUp,rAp,rGp,fUp,rGp,rAp,rGp,fUp,fCp,rGp,fUp,rAp,fUp,fUp,rAp,fCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>651</th>
      <td>Aspartame</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/261.png" width="120" ></td>
      <td>Aspartame</td>
      <td>Protein</td>
      <td></td>
      <td>Selection Buffer (150 mM NaCl, 10 mM MgCl2, 1 mM EDTA, 10 mM Sodium Phosphate, 90 mM HEPES (pH 7.1))    Assay Buffer (180mM HEPES (pH 7.0?ñ8.0), 300 mM NaCl, 20 or 50 mM MgCl2, 2 mM EDTA, 20 mM Na phosphate), r.t.</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>31837.04 g/mole</td>
      <td>51.52%</td>
      <td>970700 L/(mole·cm)</td>
      <td></td>
      <td>5 mM (reported value)</td>
      <td>None</td>
      <td>99</td>
      <td>Ferguson et al. "A novel strategy for selection of allosteric ribozymes yields RiboReporter TM sensors for caffeine and aspartame." Nucleic Acids Research, 32(2004): 1756-1766.</td>
      <td>rGp,rGp,rAp,rUp,rGp,rUp,rCp,rCp,rAp,rGp,rUp,rCp,rGp,rCp,rUp,rUp,rGp,rCp,rAp,rAp,rUp,rGp,rCp,rCp,rCp,rUp,rUp,rUp,rUp,rAp,rGp,rAp,rCp,rCp,rCp,rUp,rGp,rAp,rUp,rGp,rAp,rGp,rCp,rGp,rGp,rUp,rGp,rCp,rUp,rAp,rGp,rUp,rUp,rAp,rGp,rUp,rUp,rGp,rCp,rAp,rGp,rUp,rUp,rUp,rCp,rGp,rGp,rUp,rUp,rGp,rUp,rUp,rAp,rCp,rGp,rCp,rGp,rAp,rAp,rAp,rCp,rGp,rGp,rUp,rGp,rAp,rAp,rAp,rGp,rCp,rCp,rGp,rUp,rAp,rGp,rGp,rUp,rCp,rUp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>652</th>
      <td>CD59 (sp22)</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/381.png" width="120" ></td>
      <td>CD59</td>
      <td>Protein</td>
      <td>N/A nM (reported value)</td>
      <td>Washed with cold PBS and glycine buffer added. Chilled on ice for 15 minutes.</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>2922.43 g/mole</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Inhibits CD59 expression; Concomitantly, sp22 increases complement-mediated lysis and apoptosis signals.</td>
      <td>22</td>
      <td>None</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>653</th>
      <td>Adipose Tissue (adipo-8)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/326.jpg" width="120" ></td>
      <td>Mature 3T3-L1 adipocytes (adipose tissue)</td>
      <td>Cells</td>
      <td>17.8 nM (reported value)</td>
      <td>None</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>27226.7 g/mole</td>
      <td>56.32%</td>
      <td>862800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Aptamer binds to adipose cells at 4 °C and 37°C. Trypsin assay suggests that aptamer binds to a surface protein.</td>
      <td>87</td>
      <td>J. Liu et. al. "Selection of aptamers specific for adipose tissue." PLoS One, 7(2012): e37789.</td>
      <td>dAp,dTp,dGp,dAp,dGp,dAp,dAp,dGp,dCp,dGp,dTp,dCp,dGp,dGp,dTp,dGp,dTp,dGp,dGp,dTp,dTp,dAp,dAp,dAp,dCp,dAp,dCp,dGp,dGp,dAp,dAp,dCp,dGp,dAp,dAp,dGp,dGp,dTp,dGp,dCp,dAp,dGp,dGp,dAp,dAp,dGp,dAp,dTp,dTp,dTp,dGp,dTp,dCp,dGp,dAp,dTp,dGp,dCp,dGp,dGp,dTp,dGp,dCp,dCp,dTp,dGp,dAp,dGp,dCp,dGp,dGp,dGp,dCp,dTp,dGp,dGp,dCp,dAp,dAp,dGp,dGp,dCp,dGp,dCp,dAp,dTp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>654</th>
      <td>HIV-1 R5 Glycoprotein gp120 (B4)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/125.jpg" width="120" ></td>
      <td>HIV-1 R5 SU Glycoprotein</td>
      <td>Protein</td>
      <td>5 nM (reported value)</td>
      <td>HBS Buffer (10 mM HEPES (pH 7.4), 150 mM NaCl, 1 mM CaCl2, 1 mM MgCl2, 2.7 mM KCl)</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>36897.78 g/mole</td>
      <td>47.37%</td>
      <td>1148700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>114</td>
      <td>Khati, M., et al. "Neutralization of Infectivity of Diverse R5 Clinical Isolates of Human Immunodeficiency Virus Type 1 by gp120-Binding 2'F-RNA Aptamers." Journal of Virology, 77(2003): 12692-12698.</td>
      <td>rAp,rAp,fUp,fUp,rAp,rAp,fCp,fCp,fCp,fUp,fCp,rAp,fCp,fUp,rAp,rAp,rAp,rGp,rGp,rGp,rAp,rAp,fCp,fUp,rGp,fUp,fUp,rGp,fUp,rGp,rAp,rGp,fUp,fCp,fUp,fCp,rAp,fUp,rGp,fUp,fCp,rGp,rAp,rAp,rGp,rAp,rGp,fCp,rGp,rGp,fUp,fUp,rAp,rAp,rGp,rGp,rGp,rAp,rGp,rAp,fUp,fUp,fUp,rAp,rGp,rGp,fCp,rAp,rGp,fCp,rAp,rGp,fCp,fUp,fUp,rGp,rGp,rAp,fCp,rAp,rGp,fUp,rGp,fUp,rAp,fUp,fCp,rGp,rGp,fCp,fUp,rGp,rAp,rGp,fUp,fUp,rGp,rAp,rGp,fCp,rGp,fUp,fCp,fUp,rAp,rGp,fUp,fCp,fUp,fUp,rGp,fUp,fCp,fUp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>655</th>
      <td>Internalin A of Listeria monocytogenes</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/404.png" width="120" ></td>
      <td>Internalin A (InlA)</td>
      <td>Protein</td>
      <td></td>
      <td>TBS: Tris 10 mM, NaCl 0.85%, pH 8.0</td>
      <td>4°C</td>
      <td>None</td>
      <td>Limited cross-reactivity with non-virulent strains of Listeria (L innocua, L grayi, L ivanovii, L seeligeri, and L welshimeri), E coli, S enterica, L acidophilus, S aureus, and P aeruginos.</td>
      <td>14811.63 g/mole</td>
      <td>65.96%</td>
      <td>470400 L/(mole·cm)</td>
      <td></td>
      <td>None</td>
      <td>None</td>
      <td>48</td>
      <td>None</td>
      <td>dAp,dTp,dCp,dCp,dAp,dTp,dGp,dGp,dGp,dGp,dCp,dGp,dGp,dAp,dGp,dAp,dTp,dGp,dAp,dGp,dGp,dGp,dGp,dGp,dAp,dGp,dGp,dAp,dGp,dGp,dGp,dCp,dGp,dGp,dGp,dTp,dAp,dCp,dCp,dCp,dGp,dGp,dTp,dTp,dGp,dAp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>656</th>
      <td>Apt-3</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7172.jpg" width="120" ></td>
      <td>C promoter binding factor 1 (CBF1)</td>
      <td>Protein</td>
      <td>34 nM (reported value)</td>
      <td>100 mM Tris-HCl (pH 8.0), 100 mM NaCl, 5 mM KCl, 1 mM MgCl2, 0.1% Tween 20, 1 mg/mL BSA</td>
      <td>22°C</td>
      <td>None</td>
      <td></td>
      <td>22004.3 g/mole</td>
      <td>54.17%</td>
      <td>693600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>72</td>
      <td>Tezuka-Kagajo, Mari et al. “Development of Human CBF1-Targeting Single-Stranded DNA Aptamers with Antiangiogenic Activity In Vitro.” Nucleic acid therapeutics vol. 30,6 (2020): 365-378. doi:10.1089/nat.2020.0875</td>
      <td>dAp,dTp,dAp,dGp,dGp,dAp,dGp,dTp,dCp,dGp,dAp,dCp,dCp,dGp,dAp,dCp,dCp,dAp,dGp,dAp,dCp,dCp,dAp,dAp,dCp,dCp,dCp,dTp,dAp,dCp,dGp,dCp,dGp,dTp,dAp,dCp,dCp,dAp,dAp,dCp,dCp,dAp,dGp,dAp,dTp,dGp,dAp,dCp,dCp,dTp,dAp,dCp,dGp,dTp,dGp,dCp,dGp,dTp,dCp,dTp,dAp,dCp,dAp,dTp,dCp,dTp,dAp,dGp,dAp,dCp,dTp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>657</th>
      <td>Escherichia coli (E. coli) fimbriae protein K88 (clone 37)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/570.jpg" width="120" ></td>
      <td>K88</td>
      <td>Protein</td>
      <td>25 nM (reported value)</td>
      <td>20 mmol/L HEPES, 1 mmol/L MgCl2; 1 mmol/L CaCl2; and 120 mmol/L KCl</td>
      <td>25°C</td>
      <td>None</td>
      <td></td>
      <td>18431.95 g/mole</td>
      <td>56.67%</td>
      <td>545200 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>180</td>
      <td>Li, H. et al. Aptamer selection for the detection of Escherichia coli K88. 2011. Canadian Journal of Microbiology 57.6: 453-9.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>658</th>
      <td>GBN2</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7167.jpg" width="120" ></td>
      <td>Nervous necrosis virus (NNV)</td>
      <td>Other</td>
      <td>27.96 nM (reported value)</td>
      <td>5 g/L glucose,10% FBS,  1.0 g/L bovine serum albumin, 0.1 mg/ml yeast tRNA and 5 mM MgCl2</td>
      <td>4°C</td>
      <td>None</td>
      <td></td>
      <td>29267.9 g/mole</td>
      <td>54.74%</td>
      <td>894300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>95</td>
      <td>Zhou L, Wang S, Yu Q, et al. Characterization of Novel Aptamers Specifically Directed to Red-Spotted Grouper Nervous Necrosis Virus (RGNNV)-Infected Cells for Mediating Targeted siRNA Delivery. Front Microbiol. 2020;11:660. Published 2020 Apr 30. doi:10.3389/fmicb.2020.00660</td>
      <td>dGp,dAp,dCp,dGp,dCp,dTp,dTp,dAp,dCp,dTp,dCp,dAp,dGp,dGp,dTp,dGp,dTp,dGp,dAp,dCp,dTp,dCp,dGp,dCp,dTp,dCp,dCp,dAp,dCp,dTp,dGp,dGp,dTp,dCp,dCp,dGp,dGp,dAp,dTp,dCp,dAp,dCp,dTp,dTp,dGp,dAp,dTp,dAp,dTp,dGp,dTp,dCp,dGp,dTp,dGp,dCp,dGp,dGp,dCp,dAp,dGp,dTp,dCp,dGp,dTp,dTp,dTp,dAp,dCp,dAp,dTp,dCp,dCp,dCp,dGp,dAp,dAp,dGp,dGp,dAp,dCp,dGp,dCp,dAp,dGp,dAp,dTp,dGp,dAp,dAp,dGp,dTp,dCp,dTp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>659</th>
      <td>Atrazine R12.28</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/583.jpg" width="120" ></td>
      <td>Atrazine</td>
      <td>Small Organic</td>
      <td>5.1 nM (reported value)</td>
      <td>500 µL of buffer consisting of 100 mM sodium chloride, 20 mM Tris-HCl, and 2 mM magnesium chloride</td>
      <td>24°C</td>
      <td>None</td>
      <td></td>
      <td>24384.85 g/mole</td>
      <td>50.63%</td>
      <td>749500 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>237</td>
      <td>Williams, R.; Crihfield, C.; Gattu, S.; Holland, L.; Sooter, L. In Vitro Selection of a Single-Stranded DNA Molecular Recognition Element against Atrazine. International Journal of Molecular Sciences, 2014, 15, 14332-14347. ISSN 1422-0067 doi: 10.3990/ijms150814332</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>660</th>
      <td>SP6 RNA polymerase (S05)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/287.jpg" width="120" ></td>
      <td>SP6 RNA Polymerase</td>
      <td>Protein</td>
      <td>9.5 nM (reported value)</td>
      <td>Binding Buffer (50mM Tris-HCl, pH 7.6, 1 M NaCl, 10mM 2-mercaptoethanol, 10% glycerol) at 4°C overnight at -20°C until use</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>11178.67 g/mole</td>
      <td>40.00%</td>
      <td>352800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>35</td>
      <td>Mori, Y. (2012). Inhibitory RNA aptamer against sp6 RNA polymerase. Biochemical and Biophysical Resarch Communications, 420 (2012), 440-443.</td>
      <td>rUp,rUp,rGp,rCp,rUp,rUp,rGp,rGp,rAp,rAp,rUp,rGp,rCp,rGp,rUp,rUp,rAp,rUp,rAp,rGp,rUp,rCp,rUp,rCp,rUp,rUp,rAp,rGp,rGp,rUp,rGp,rUp,rGp,rUp,rAp,rUp,rUp,rGp,rCp,rUp,rUp,rGp,rGp,rAp,rAp,rUp,rGp,rCp,rGp,rUp,rUp,rAp,rUp,rAp,rGp,rUp,rCp,rUp,rCp,rUp,rUp,rAp,rGp,rGp,rUp,rGp,rUp,rGp,rUp,rAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>661</th>
      <td>Caffeine (S2.caf.D11)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/260.png" width="120" ></td>
      <td>Caffeine</td>
      <td>Protein</td>
      <td></td>
      <td>Selection Buffer (500 mM NaCl, 10 mM  MgCl2, 1 mM EDTA, 10 mM sodium phosphate, 90 mM HEPES (pH 7.5))    Assay Buffer (180mM HEPES (pH 7.0?ñ8.0), 300 mM NaCl, 20 or  50 mM MgCl2, 2 mM EDTA, 20 mM Na phosphate), r.t.</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>31866.13 g/mole</td>
      <td>51.52%</td>
      <td>976100 L/(mole·cm)</td>
      <td></td>
      <td>1mM mM (reported value)</td>
      <td>None</td>
      <td>99</td>
      <td>Ferguson et al. "A novel strategy for selection of allosteric ribozymes yields RiboReporter  TM sensors for caffeine and aspartame." Nucleic Acids Research, 32(2004): 1756-1766.</td>
      <td>rGp,rGp,rAp,rUp,rGp,rUp,rCp,rCp,rAp,rGp,rUp,rCp,rGp,rCp,rUp,rUp,rGp,rCp,rAp,rAp,rUp,rGp,rCp,rCp,rCp,rUp,rUp,rUp,rUp,rAp,rGp,rAp,rCp,rCp,rCp,rUp,rGp,rAp,rUp,rGp,rAp,rGp,rGp,rAp,rUp,rCp,rAp,rUp,rCp,rGp,rGp,rAp,rCp,rUp,rUp,rUp,rGp,rUp,rCp,rCp,rUp,rGp,rUp,rGp,rGp,rAp,rGp,rUp,rAp,rAp,rGp,rAp,rUp,rCp,rGp,rCp,rGp,rAp,rAp,rAp,rCp,rGp,rGp,rUp,rGp,rAp,rAp,rAp,rGp,rCp,rCp,rGp,rUp,rAp,rGp,rGp,rUp,rCp,rUp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>662</th>
      <td>NADH</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/280.png" width="120" ></td>
      <td>NADH</td>
      <td>Small Organic</td>
      <td>37 nM (reported value)</td>
      <td>Selection Buffer (20 mM HEPES (pH 7.5), 300 mM KC1, and 5 mM MgC12) at room temperature.</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>15748.56 g/mole</td>
      <td>59.18%</td>
      <td>474700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>49</td>
      <td>Lauhon and Szostak. "RNA Aptamers That Bind Flavin and Nicotinamide Redox Cofactors." Journal of the American Chemical Society, 117(1994): 1246-1257.</td>
      <td>rGp,rGp,rAp,rAp,rCp,rCp,rCp,rAp,rAp,rCp,rUp,rAp,rGp,rGp,rCp,rGp,rUp,rUp,rUp,rGp,rAp,rGp,rGp,rGp,rGp,rAp,rUp,rUp,rCp,rGp,rGp,rCp,rCp,rAp,rCp,rGp,rGp,rUp,rAp,rAp,rCp,rAp,rAp,rCp,rCp,rCp,rCp,rUp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>663</th>
      <td>Apple Stem Pitting Virus of PSAH PearIsolate</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/160.png" width="120" ></td>
      <td>Apple Stem Pitting Virus (ASPV)</td>
      <td>Protein</td>
      <td>8 nM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td>Binds to the MT32 coat protein with reduced affinity (290 nM).</td>
      <td>12158.96 g/mole</td>
      <td>46.15%</td>
      <td>361100 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>39</td>
      <td>Balogh, et al. "Selection and versatile application of virus-specific aptamers." FASEB Journal, 24(2010): 4187-4195.</td>
      <td>dAp,dGp,dCp,dAp,dAp,dGp,dGp,dTp,dTp,dTp,dGp,dGp,dTp,dGp,dTp,dTp,dGp,dGp,dTp,dTp,dGp,dGp,dTp,dTp,dGp,dCp,dTp,dGp,dGp,dTp,dTp,dTp,dTp,dGp,dGp,dTp,dTp,dTp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>664</th>
      <td>Adenosine Triphosphate (1-1MIN)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/99.jpg" width="120" ></td>
      <td>Adenosine Triphosphate / ATP</td>
      <td>Small Organic</td>
      <td>4.8 µM (reported value)</td>
      <td>None</td>
      <td>N/A°C</td>
      <td>\n                                N/A \n</td>
      <td>None</td>
      <td>18459.16 g/mole</td>
      <td>54.39%</td>
      <td>571900 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>57</td>
      <td>None</td>
      <td>rGp,rGp,rGp,rAp,rGp,rAp,rUp,rCp,rUp,rAp,rCp,rGp,rGp,rAp,rUp,rCp,rUp,rCp,rAp,rGp,rGp,rGp,rCp,rUp,rCp,rUp,rUp,rAp,rCp,rGp,rGp,rGp,rAp,rGp,rCp,rUp,rAp,rCp,rAp,rUp,rGp,rGp,rAp,rAp,rGp,rGp,rAp,rGp,rUp,rCp,rCp,rAp,rUp,rGp,rUp,rGp,rUp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>665</th>
      <td>Chloramphenicol (Cm1)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/155.jpg" width="120" ></td>
      <td>Chloramphenicol</td>
      <td>Small Organic</td>
      <td>110 nM (reported value)</td>
      <td>Binding Buffer (20 mM MgCl2, 400 mM NaCl, 100 mM bis-Tris (pH  6.4)</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>16189.95 g/mole</td>
      <td>42.00%</td>
      <td>522900 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>50</td>
      <td>Burke et al. "RNA aptamers to the peptidyl transferase inhibitor chloramphenicol." Chemistry & Biology, 4(1997): 833-843.</td>
      <td>rGp,rGp,rGp,rAp,rUp,rCp,rAp,rCp,rAp,rGp,rUp,rGp,rAp,rAp,rAp,rAp,rAp,rAp,rGp,rAp,rCp,rGp,rUp,rGp,rUp,rGp,rAp,rAp,rUp,rGp,rUp,rCp,rAp,rCp,rAp,rCp,rUp,rGp,rAp,rAp,rAp,rAp,rAp,rAp,rGp,rAp,rUp,rCp,rCp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>666</th>
      <td>Acute myeloid leukemia cells (KH1C12)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/68.jpg" width="120" ></td>
      <td>Acute myeloid leukemia cells (HL60)</td>
      <td>Cells</td>
      <td>4.5 nM (reported value)</td>
      <td>None</td>
      <td>4°C</td>
      <td>None</td>
      <td></td>
      <td>20896.62 g/mole</td>
      <td>47.06%</td>
      <td>647700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>68</td>
      <td>Sefah, Kwame, et al. "Molecular recognition of acute myeloid leukemia using aptamers." Leukemia, 23 (2009):235-244</td>
      <td>dAp,dTp,dCp,dCp,dAp,dGp,dAp,dGp,dTp,dGp,dAp,dCp,dGp,dCp,dAp,dGp,dCp,dAp,dTp,dGp,dCp,dCp,dCp,dTp,dAp,dGp,dTp,dTp,dAp,dCp,dTp,dAp,dCp,dTp,dAp,dCp,dTp,dCp,dTp,dTp,dTp,dTp,dTp,dAp,dGp,dCp,dAp,dAp,dAp,dCp,dTp,dGp,dGp,dAp,dCp,dAp,dCp,dGp,dGp,dTp,dGp,dGp,dCp,dTp,dTp,dAp,dGp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>667</th>
      <td>Neurotrophin receptor, TrkB (C4-3)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/306.png" width="120" ></td>
      <td>Neurotrophin receptor, TrkB</td>
      <td>Protein</td>
      <td>2.1 nM (reported value)</td>
      <td>None</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>16478.87 g/mole</td>
      <td>64.71%</td>
      <td>490000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>This aptamer is a TrkB agonist and was shown to increase the phosphorylation of TrkB and downstream signaling protein, Akt.</td>
      <td>51</td>
      <td>None</td>
      <td>rGp,rGp,rGp,rAp,rGp,rGp,rAp,fCp,rGp,rAp,fUp,rGp,fCp,rGp,rGp,fUp,fCp,rGp,fUp,rAp,fUp,fUp,rAp,fUp,fCp,fCp,rGp,fCp,fUp,rGp,fCp,rAp,fCp,rGp,fCp,fCp,rAp,rGp,rAp,fCp,rGp,rAp,fCp,fUp,fCp,rGp,fCp,fCp,fCp,rGp,rAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>668</th>
      <td>Interleukin 17A/F (IL-17A/F) (AptAF42dope1)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/323.png" width="120" ></td>
      <td>Interleukin-17A/F</td>
      <td>Protein</td>
      <td>35.8 nM (reported value)</td>
      <td>Binding buffer-145mM sodium chloride, 5.4 mM potassium chloride, 0.8 mM magnesium chloride, 1.8 mM calcium chloride, 20 mM Tris-HCl (pH 7.6) supplemented with 0.05% Tween20</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>21984.1 g/mole</td>
      <td>60.29%</td>
      <td>657400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>75</td>
      <td>Adachi, H et al. (2011). Antagonistic rna aptamer specific to a heterodimeric form of human interleukin-17a/f. Biochimie, 93(2011), 1081-1088. doi: 10.1016/j.biochi.2011.04.003</td>
      <td>rGp,rGp,rGp,fCp,fUp,rAp,rGp,fCp,fUp,rGp,rAp,fUp,fCp,rGp,fUp,rAp,fCp,fCp,rAp,rGp,fUp,rAp,rGp,fCp,rGp,fUp,rGp,rGp,fCp,fCp,fUp,rGp,rGp,rGp,rGp,rGp,rGp,fCp,fCp,fUp,rAp,rGp,fUp,fCp,rGp,fUp,rGp,fCp,rGp,rAp,fUp,rAp,fCp,fUp,rAp,rAp,fCp,rAp,rGp,fCp,fUp,rAp,rAp,fCp,rAp,fCp,fCp,fCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>669</th>
      <td>PSA aptamer (PSap4#5)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/332.jpg" width="120" ></td>
      <td>PSA</td>
      <td>Protein</td>
      <td><100 nM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>\n                                The DNA library was heated at 95 °C for 10 min and then gradually cooled to 25 °C over a period of 30 min in TBS buffer (10mM Tris/HCl, 150mM NaCl, 5mM KCl, 5mM MgCl2, pH 7.4). \n</td>
      <td></td>
      <td>9747.45 g/mole</td>
      <td>28.13%</td>
      <td>305200 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Binding was not detectable during the course of internal investigation. Gel-Shift and Forte-Bio Octet methods to attempt to determine Kd did not generate results indicative of aptamer-target binding.</td>
      <td>32</td>
      <td>None</td>
      <td>dTp,dTp,dTp,dTp,dTp,dAp,dAp,dTp,dTp,dAp,dAp,dAp,dGp,dCp,dTp,dCp,dGp,dCp,dCp,dAp,dTp,dCp,dAp,dAp,dAp,dTp,dAp,dGp,dCp,dTp,dTp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>670</th>
      <td>h-Ras-1</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7202.png" width="120" ></td>
      <td>h-Ras (Human Ras protein)</td>
      <td>Protein</td>
      <td>200 nM (reported value)</td>
      <td>50 mM Tris-HCl</td>
      <td>23°C</td>
      <td>\n                                m-fold \n</td>
      <td></td>
      <td>25479.04 g/mole</td>
      <td>52.50%</td>
      <td>766100 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>80</td>
      <td>Berezovski, M. Musheev, M. Drabovich, A. and Krylov, S. (2006) Non-SELEX Selection of Aptamers. Department of Chemistry, York University. https://doi.org/10.1021/ja056943j</td>
      <td>rCp,rTp,rTp,rCp,rTp,rGp,rCp,rCp,rCp,rGp,rCp,rCp,rTp,rCp,rCp,rTp,rTp,rCp,rCp,rTp,rAp,rTp,rTp,rAp,rGp,rGp,rGp,rTp,rGp,rCp,rTp,rTp,rCp,rGp,rCp,rGp,rAp,rAp,rGp,rTp,rTp,rCp,rAp,rTp,rTp,rTp,rTp,rAp,rCp,rAp,rTp,rCp,rCp,rCp,rCp,rAp,rTp,rAp,rGp,rGp,rAp,rGp,rAp,rCp,rGp,rAp,rGp,rAp,rTp,rAp,rGp,rGp,rCp,rGp,rGp,rAp,rCp,rAp,rCp,rTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>671</th>
      <td>Mycobacterium tuberculosis (strain H37Rv) (aptamer NK2)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/423.png" width="120" ></td>
      <td>Virulent mycobacterium tuberculosis (H37Rv)</td>
      <td>Protein</td>
      <td>31 nM (reported value)</td>
      <td>Binding buffer: 25 mM Tris-HCl, 50 mM KCl, 200 mM NaCl, 0.2 mM EDTA, 5.0% (v/v) glycerol, and 0.5 mM DTT.</td>
      <td>37°C</td>
      <td>\n                                ssDNA was heated at 85°C for 15 min and snap-cooled on ice for 3 min. \n</td>
      <td>None</td>
      <td>29764.31 g/mole</td>
      <td>51.55%</td>
      <td>909900 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>97</td>
      <td>None</td>
      <td>dGp,dCp,dGp,dGp,dGp,dAp,dTp,dCp,dCp,dTp,dAp,dTp,dGp,dAp,dCp,dGp,dCp,dAp,dTp,dTp,dGp,dAp,dCp,dCp,dCp,dAp,dCp,dAp,dAp,dCp,dAp,dCp,dAp,dCp,dTp,dAp,dCp,dTp,dGp,dTp,dCp,dGp,dCp,dTp,dCp,dGp,dGp,dTp,dTp,dCp,dGp,dAp,dAp,dCp,dTp,dTp,dCp,dGp,dTp,dGp,dCp,dGp,dAp,dCp,dTp,dGp,dTp,dTp,dCp,dCp,dCp,dTp,dAp,dTp,dAp,dGp,dTp,dGp,dAp,dGp,dTp,dCp,dGp,dTp,dAp,dTp,dTp,dAp,dGp,dAp,dAp,dTp,dTp,dCp,dCp,dGp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>672</th>
      <td>Signal Recognition Particle 19 (SRP19)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/452.jpg" width="120" ></td>
      <td>Signal Recognition Particle 19 (SRP19)</td>
      <td>Protein</td>
      <td>0.6 nM (reported value)</td>
      <td>None</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>28497.44 g/mole</td>
      <td>60.23%</td>
      <td>870000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>SRP19 protein from hyperthermophilic archaeon Pyrococcus furiosus</td>
      <td>0</td>
      <td>Haraguchi, Y. Et al. Characterization of RNA aptamers against SRP19 protein having sequences different from SRP RNA.  Nucleic Acids Symposium Series, 2009, 53, 265-266.</td>
      <td>rGp,rGp,rGp,rAp,rGp,rAp,rCp,rAp,rAp,rGp,rAp,rAp,rUp,rAp,rAp,rAp,rCp,rGp,rCp,rUp,rCp,rAp,rAp,rCp,rAp,rCp,rAp,rGp,rAp,rAp,rCp,rGp,rCp,rGp,rGp,rUp,rCp,rCp,rCp,rCp,rAp,rCp,rAp,rCp,rAp,rGp,rGp,rAp,rCp,rAp,rGp,rGp,rAp,rGp,rCp,rCp,rAp,rGp,rCp,rCp,rCp,rCp,rGp,rGp,rUp,rUp,rCp,rGp,rAp,rCp,rAp,rGp,rGp,rAp,rGp,rGp,rCp,rUp,rCp,rAp,rCp,rAp,rAp,rCp,rAp,rGp,rGp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>673</th>
      <td>B1-4</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/548.jpg" width="120" ></td>
      <td>IFN-Gamma</td>
      <td>Protein</td>
      <td>74.5 nM (reported value)</td>
      <td>200 uL (PBS), 0.05 pM Aptamer</td>
      <td>24°C</td>
      <td>\n                                N/A \n</td>
      <td>Tested against BSA, Trypsin, IgG and Albumin, displays a high degree of specificity</td>
      <td>18001.76 g/mole</td>
      <td>47.46%</td>
      <td>585100 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>177</td>
      <td>Cao B, Hu Y, Duan J, Ma J, Xu D, et al. (2014) Selection of a Novel DNA Aptamer for Assay of Intracellular Interferon-Gamma. PLoS ONE 9(5): e98214.  doi:10.1371/journal.pone.0098214</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>674</th>
      <td>Lysergamine (M3.2)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/470.png" width="120" ></td>
      <td>Lysergamine, metergoline, and other small ergot alkaloids</td>
      <td>Protein</td>
      <td>44 nM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td>None</td>
      <td>24516.84 g/mole</td>
      <td>62.50%</td>
      <td>747200 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>The aptamer contains two cytosine-repeat binding sites.</td>
      <td>88</td>
      <td>None</td>
      <td>dAp,dGp,dCp,dAp,dGp,dCp,dAp,dCp,dAp,dGp,dAp,dGp,dGp,dTp,dCp,dAp,dGp,dAp,dTp,dGp,dGp,dCp,dGp,dTp,dCp,dAp,dGp,dCp,dCp,dCp,dCp,dGp,dAp,dTp,dCp,dGp,dCp,dCp,dAp,dTp,dCp,dCp,dAp,dGp,dGp,dGp,dAp,dCp,dTp,dCp,dCp,dCp,dCp,dCp,dCp,dTp,dAp,dCp,dTp,dGp,dCp,dCp,dTp,dAp,dTp,dGp,dCp,dGp,dTp,dGp,dCp,dTp,dAp,dCp,dCp,dGp,dTp,dGp,dAp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>675</th>
      <td>Advanced Glycation End Products (Clone 9)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/128.png" width="120" ></td>
      <td>Advanced Glycation End products (AGE)</td>
      <td>Protein</td>
      <td>1 pM (reported value)</td>
      <td>N/A</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>32346.02 g/mole</td>
      <td>48.11%</td>
      <td>1004700 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>106</td>
      <td>Higashimoto, Y., et al. "In vitro selection of DNA aptamers that block toxic effects of AGE on cultured retinal pericytes." Microvascular Research, 74 (2007): 65-69.</td>
      <td>dAp,dGp,dCp,dTp,dCp,dAp,dGp,dAp,dAp,dTp,dGp,dGp,dAp,dTp,dCp,dCp,dAp,dAp,dAp,dCp,dGp,dCp,dTp,dCp,dAp,dTp,dAp,dAp,dCp,dTp,dCp,dAp,dCp,dTp,dCp,dCp,dAp,dTp,dAp,dCp,dTp,dCp,dAp,dCp,dTp,dTp,dGp,dCp,dTp,dGp,dAp,dTp,dTp,dCp,dGp,dCp,dCp,dAp,dAp,dCp,dAp,dAp,dCp,dAp,dCp,dAp,dCp,dCp,dCp,dTp,dTp,dAp,dAp,dAp,dCp,dAp,dGp,dTp,dCp,dCp,dCp,dTp,dTp,dCp,dGp,dAp,dCp,dAp,dTp,dGp,dAp,dGp,dAp,dAp,dTp,dTp,dCp,dGp,dGp,dCp,dCp,dGp,dGp,dAp,dTp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>676</th>
      <td>HIV-1 Tat protein</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/364.png" width="120" ></td>
      <td>HIV-1 Tat protein</td>
      <td>Protein</td>
      <td>120 pM (reported value)</td>
      <td>Binding Buffer: 50 mM Tris-HCl (pH 7.8) and 50 mM KCl</td>
      <td>r.t.°C</td>
      <td>None</td>
      <td></td>
      <td>11784.08 g/mole</td>
      <td>56.76%</td>
      <td>352600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>38</td>
      <td>None</td>
      <td>rAp,rCp,rGp,rAp,rAp,rGp,rCp,rUp,rUp,rGp,rAp,rUp,rCp,rCp,rCp,rGp,rUp,rUp,rUp,rGp,rCp,rCp,rGp,rGp,rUp,rCp,rGp,rAp,rUp,rCp,rGp,rCp,rUp,rUp,rCp,rGp,rAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>677</th>
      <td>Omethoate (SS2-55)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/295.png" width="120" ></td>
      <td>Omethoate</td>
      <td>Small Organic</td>
      <td>2.5 µM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td>This aptamer binds preferentially to the organophosphates phorate, profenofos, benzoate and omethoate. It has reduced binding to imidacloprid, acetamiprid, chlorpyrifos, methamidophos, dichlorvos, parathion, dimethoate, monocrotophos and phoxim.</td>
      <td>16990.09 g/mole</td>
      <td>49.09%</td>
      <td>526900 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>"A molecular beacon (5' FAM-CTGCACAAGAATCGCTGCAG-3') was designed, which was partly complementary with the middle constant nucleotides-sequence of the selected ssDNAs."</td>
      <td>55</td>
      <td>None</td>
      <td>dAp,dAp,dGp,dCp,dTp,dTp,dGp,dCp,dTp,dTp,dTp,dAp,dTp,dAp,dGp,dCp,dCp,dTp,dGp,dCp,dAp,dGp,dCp,dGp,dAp,dTp,dTp,dCp,dTp,dTp,dGp,dAp,dTp,dCp,dGp,dGp,dAp,dAp,dAp,dAp,dGp,dGp,dCp,dTp,dGp,dAp,dGp,dAp,dGp,dCp,dTp,dAp,dCp,dGp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>678</th>
      <td>PFOA Aptamer</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7234.jpg" width="120" ></td>
      <td>Polyfluoroalkyl acid (Perfluorooctanoic acid, PFOA)</td>
      <td>Small Organic</td>
      <td>5.5 nM (reported value)</td>
      <td>pH 7.5 buffer (20 mM HEPES, 1 M NaCl, 10 mM MgCl2, and 5 mM KCl)</td>
      <td>23°C</td>
      <td>None</td>
      <td></td>
      <td>9488.22 g/mole</td>
      <td>60.00%</td>
      <td>308100 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>30</td>
      <td>Park, J., Yang, K.-A., Choi, Y., & Choe, J. K. (2022). Novel ssdna aptamer-based fluorescence sensor for perfluorooctanoic acid detection in water. Environment International, 158, 107000. https://doi.org/10.1016/j.envint.2021.107000</td>
      <td>dGp,dGp,dAp,dAp,dTp,dCp,dGp,dTp,dGp,dGp,dGp,dTp,dGp,dGp,dTp,dAp,dGp,dGp,dGp,dTp,dAp,dGp,dGp,dGp,dGp,dAp,dTp,dGp,dCp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>679</th>
      <td>Kanamycin A (sla 16)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/154.jpg" width="120" ></td>
      <td>Kanamycin A</td>
      <td>Small Organic</td>
      <td>300 nM (reported value)</td>
      <td>Binding Buffer (500 mM NaCl, 50 mM Tris (pH 7.6))</td>
      <td>RT°C</td>
      <td>None</td>
      <td></td>
      <td>25761.54 g/mole</td>
      <td>57.50%</td>
      <td>773600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Structures displayed in the reference are DNA sequences.</td>
      <td>80</td>
      <td>Lato et al. "In vitro selection of RNA lectins: using combinatorial chemistry to interpret ribozyme evolution." Chemistry & Biology, 2(1995): 291-303.</td>
      <td>rGp,rGp,rGp,rAp,rAp,rUp,rGp,rGp,rAp,rUp,rCp,rCp,rAp,rCp,rAp,rUp,rCp,rUp,rAp,rCp,rGp,rAp,rAp,rUp,rUp,rCp,rAp,rCp,rCp,rGp,rCp,rGp,rGp,rGp,rGp,rUp,rUp,rGp,rCp,rGp,rGp,rAp,rCp,rCp,rGp,rGp,rGp,rAp,rGp,rCp,rUp,rCp,rCp,rAp,rGp,rCp,rUp,rUp,rCp,rAp,rCp,rUp,rGp,rCp,rAp,rGp,rAp,rCp,rUp,rUp,rGp,rAp,rCp,rGp,rAp,rAp,rGp,rCp,rUp,rUp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>680</th>
      <td>RNA Tobramycin Molecular Beacon (BA 14-2)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/459" width="120" ></td>
      <td>Tobramycin</td>
      <td>Small Organic</td>
      <td>16 µM (reported value)</td>
      <td>Binding buffer: 50 mM Tris (pH 7.4), 500 mM NaCl</td>
      <td>4°C</td>
      <td>None</td>
      <td></td>
      <td>33690.28 g/mole</td>
      <td>48.08%</td>
      <td>1052600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>104</td>
      <td>None</td>
      <td>rGp,rGp,rAp,rAp,rUp,rGp,rGp,rAp,rUp,rCp,rCp,rAp,rCp,rAp,rUp,rCp,rUp,rAp,rCp,rGp,rAp,rAp,rGp,rGp,rCp,rUp,rUp,rUp,rGp,rAp,rAp,rGp,rGp,rUp,rGp,rAp,rGp,rAp,rCp,rCp,rGp,rUp,rGp,rCp,rAp,rAp,rAp,rUp,rGp,rAp,rGp,rGp,rAp,rUp,rGp,rGp,rUp,rGp,rUp,rGp,rGp,rAp,rUp,rGp,rAp,rUp,rUp,rAp,rGp,rGp,rGp,rUp,rUp,rGp,rUp,rCp,rGp,rGp,rUp,rUp,rUp,rUp,rCp,rAp,rCp,rUp,rGp,rCp,rAp,rGp,rAp,rCp,rUp,rUp,rGp,rAp,rCp,rGp,rAp,rAp,rGp,rCp,rUp,rUp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>681</th>
      <td>CTBA4T-B1</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7145.png" width="120" ></td>
      <td>Thrombin</td>
      <td>Peptide</td>
      <td>19 pM (reported value)</td>
      <td>Binding buffer (1× PBSMT, pH 7.2) containing 137 mM NaCl, 2.68 mM KCl, 8.1 mM Na2HPO4, 1.76 mM KH2PO4, 1 mM MgCl2, and 0.025% Tween-20.</td>
      <td>95, 25°C</td>
      <td>None</td>
      <td></td>
      <td>17110.1 g/mole</td>
      <td>58.18%</td>
      <td>525200 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Letters in yellow are nucleotides in the original central constant domain. Letters in blue are nucleotides in the two random-sequence regions. Those in red are nucleotides in the primer domains.</td>
      <td>55</td>
      <td>Mao, Yu et al. “Evolution of a highly functional circular DNA aptamer in serum.” Nucleic acids research vol. 48,19 (2020): 10680-10690. doi:10.1093/nar/gkaa800</td>
      <td>dAp,dTp,dCp,dTp,dCp,dGp,dAp,dGp,dTp,dCp,dAp,dTp,dAp,dGp,dGp,dGp,dGp,dGp,dCp,dGp,dCp,dGp,dAp,dAp,dCp,dAp,dTp,dAp,dCp,dGp,dCp,dGp,dGp,dTp,dTp,dGp,dGp,dTp,dGp,dTp,dGp,dGp,dTp,dTp,dGp,dGp,dCp,dTp,dGp,dAp,dCp,dTp,dCp,dGp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>682</th>
      <td>Aptamer 1 - Switch antisense</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7097.png" width="120" ></td>
      <td>SARS-CoV nucleocapsid, SARS-CoV-2 nucleocapsid</td>
      <td>Protein</td>
      <td>4.93 nM (reported value)</td>
      <td>10 mM Tris, 0.5 mM NaCl, 1 mM EDTA, pH 7.4\n</td>
      <td>23°C</td>
      <td>\n                                Main aptamer: 80°C for 3 minutes, gentle cooling at 23°C for 5 minutes.\nAntisense Apta-switch component: 40°C for 10 minutes, then combine with refolded main aptamer and incubate for 2 hours at 23°C. Wash with 1X DPBS, store at 4°C. \n</td>
      <td>N/A. This is an antisense complementary sequence to Nucleocapsid aptamer ID #: 7074.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Apta-switch component (antisense strand) to Nucleocapsid protein aptamer (ID #: 7074).\nOriginally reported with poly(dT) spacer and linker for functionalization.</td>
      <td>14</td>
      <td>Singh NK, Ray P, Carlin AF, Magallanes C, Morgan SC, Laurent LC, Aronoff-Spencer ES, Hall DA. "Hitting the diagnostic sweet spot: Pont-of-care SARS-CoV-2 salivary antigen testing with an off-the-shelf gluocometer". medRxiv preprint. doi: https://doi.org/10.1101/2020.09.24.20200394.</td>
      <td>dGp,dAp,dCp,dCp,dGp,dCp,dCp,dCp,dCp,dAp,dGp,dCp,dCp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>683</th>
      <td>ADCC Bi-specific Aptamer (bsA17)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/497.png" width="120" ></td>
      <td>CD16a positive cell lines</td>
      <td>Protein</td>
      <td>N/A nM (reported value)</td>
      <td>Binding Buffer: DPBS with 0.05% BSA</td>
      <td>37°C</td>
      <td>None</td>
      <td></td>
      <td>41337.79 g/mole</td>
      <td>57.58%</td>
      <td>1284900 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>132</td>
      <td>None</td>
      <td>dCp,dCp,dAp,dCp,dTp,dGp,dCp,dGp,dGp,dGp,dGp,dGp,dTp,dCp,dTp,dAp,dTp,dAp,dCp,dGp,dTp,dGp,dAp,dGp,dGp,dAp,dAp,dGp,dAp,dAp,dGp,dTp,dGp,dGp,dGp,dCp,dAp,dGp,dGp,dTp,dCp,dGp,dCp,dAp,dGp,dGp,dTp,dCp,dGp,dGp,dAp,dGp,dGp,dGp,dAp,dAp,dAp,dAp,dGp,dTp,dTp,dAp,dTp,dCp,dAp,dGp,dGp,dCp,dTp,dGp,dGp,dAp,dTp,dGp,dGp,dTp,dAp,dGp,dCp,dTp,dCp,dGp,dGp,dTp,dCp,dGp,dGp,dGp,dGp,dTp,dGp,dGp,dGp,dTp,dGp,dGp,dGp,dTp,dTp,dGp,dGp,dCp,dAp,dAp,dGp,dTp,dCp,dTp,dGp,dAp,dTp,dTp,dAp,dGp,dTp,dTp,dTp,dTp,dGp,dGp,dAp,dGp,dTp,dAp,dCp,dTp,dCp,dGp,dCp,dTp,dCp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>684</th>
      <td>Histone H4 N-terminal tail (4.33)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/355.png" width="120" ></td>
      <td>H4 N-terminal tail</td>
      <td>Protein</td>
      <td>1.3 nM (reported value)</td>
      <td>None</td>
      <td>N/A°C</td>
      <td>None</td>
      <td>Kd values for other histones (low-/high-salt concentrations): H3 (14/273 nM), H2 A (204/4050 nM), and H2 B (84/825 nM).</td>
      <td>15592.13 g/mole</td>
      <td>60.00%</td>
      <td>473300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Affinities for H3, H2 A, and H2 B histones are also provided.</td>
      <td>50</td>
      <td>None</td>
      <td>dTp,dGp,dGp,dTp,dGp,dGp,dGp,dGp,dTp,dTp,dCp,dCp,dCp,dGp,dGp,dGp,dAp,dGp,dGp,dGp,dCp,dGp,dGp,dCp,dTp,dAp,dCp,dGp,dGp,dGp,dTp,dTp,dCp,dCp,dGp,dTp,dAp,dAp,dTp,dCp,dAp,dGp,dAp,dTp,dTp,dTp,dGp,dTp,dGp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>685</th>
      <td>OC10</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7157.jpg" width="120" ></td>
      <td>beta-crosslap (BC)</td>
      <td>Protein</td>
      <td>148.4 nM (reported value)</td>
      <td>2 mM MgCl2, 50 mM Tris (pH 7.5) and 150 mM NaCl in distilled water</td>
      <td>22°C</td>
      <td>None</td>
      <td></td>
      <td>18376.9 g/mole</td>
      <td>49.15%</td>
      <td>585800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>59</td>
      <td>Chinnappan R, Zaghloul NS, AlZabn R, Malkawi A, Abdel Rahman A, Abu-Salah KM, Zourob M. Aptamer selection and aptasensor construction for bone density biomarkers. Talanta. 2021 Mar 1;224:121818. doi: 10.1016/j.talanta.2020.121818. Epub 2020 Oct 29. PMID: 33379043.</td>
      <td>dAp,dTp,dGp,dAp,dGp,dTp,dAp,dGp,dTp,dGp,dCp,dGp,dGp,dAp,dGp,dGp,dGp,dAp,dTp,dGp,dTp,dGp,dAp,dAp,dTp,dAp,dCp,dTp,dGp,dAp,dCp,dGp,dCp,dGp,dGp,dTp,dCp,dAp,dTp,dAp,dGp,dTp,dCp,dGp,dCp,dTp,dGp,dTp,dTp,dGp,dTp,dAp,dCp,dCp,dAp,dTp,dTp,dGp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>686</th>
      <td>ATP-inhibited molecular switch</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/399.png" width="120" ></td>
      <td>ATP</td>
      <td>Small Organic</td>
      <td>na nM (reported value)</td>
      <td>Reaction buffer [50 mM Tris.HCl (pH 7.5) and 20 mM MgCl2] for 20 hr</td>
      <td>23°C</td>
      <td>None</td>
      <td></td>
      <td>25919.68 g/mole</td>
      <td>60.00%</td>
      <td>778900 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>80</td>
      <td>None</td>
      <td>rGp,rGp,rGp,rCp,rGp,rAp,rCp,rCp,rCp,rUp,rGp,rAp,rUp,rGp,rAp,rGp,rAp,rUp,rGp,rGp,rGp,rGp,rAp,rAp,rGp,rAp,rAp,rAp,rCp,rUp,rGp,rUp,rGp,rGp,rCp,rAp,rCp,rUp,rUp,rCp,rGp,rGp,rUp,rGp,rCp,rCp,rAp,rGp,rCp,rCp,rUp,rCp,rUp,rCp,rGp,rAp,rAp,rAp,rCp,rGp,rGp,rUp,rGp,rAp,rAp,rAp,rGp,rCp,rCp,rGp,rUp,rAp,rGp,rGp,rUp,rUp,rGp,rCp,rCp,rCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>687</th>
      <td>Nampt binding aptamer (NBA)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/410.png" width="120" ></td>
      <td>Nampt/Nicotinamide phosphoribosyltransferase</td>
      <td>Protein</td>
      <td>72.52 nM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td>No significant signal was observed from RBP4, Vaspin, adiponectin, human serum albumin, human IgG, and fibrinogen.</td>
      <td>23679.45 g/mole</td>
      <td>47.37%</td>
      <td>748600 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Limit of detection using ELAAS assay was 19 ng/ml. See SJ Lee et al (2012) for more details.</td>
      <td>84</td>
      <td>None</td>
      <td>dAp,dTp,dAp,dCp,dCp,dAp,dGp,dCp,dTp,dTp,dAp,dTp,dTp,dCp,dAp,dAp,dTp,dTp,dGp,dGp,dGp,dCp,dAp,dGp,dGp,dAp,dCp,dAp,dGp,dGp,dTp,dGp,dTp,dCp,dGp,dGp,dCp,dTp,dTp,dGp,dAp,dTp,dAp,dGp,dGp,dCp,dTp,dGp,dGp,dGp,dGp,dTp,dGp,dTp,dGp,dTp,dGp,dTp,dAp,dGp,dAp,dTp,dAp,dGp,dTp,dAp,dAp,dGp,dTp,dGp,dCp,dAp,dAp,dTp,dCp,dTp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>688</th>
      <td>39SGP1A</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7208.jpg" width="120" ></td>
      <td>Ebola virus Soluble glycoprotein (sGP)</td>
      <td>Protein</td>
      <td>30 nM (reported value)</td>
      <td>137 mM NaCl, 2.7 mM KCl, 10 mM Na2HPO4, 2 mM KH2PO4, 5 mM MgCl2, pH 7.4)</td>
      <td>23–24 °C°C</td>
      <td>\n                                Denature at 95°C for 5 minutes, place on ice (4°C) for 5 minutes, room temperature (23°C) for 10 minutes. \n</td>
      <td>high affinity and specificity for sGP</td>
      <td>12.297 g/mole</td>
      <td>43.59%</td>
      <td>369000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Also has high affinity and specificity to the Sudan virus (SUDV) sGP.</td>
      <td>39</td>
      <td>Shubham, S., Hoinka, J., Banerjee, S. et al. A 2′FY-RNA Motif Defines an Aptamer for Ebolavirus Secreted Protein. Sci Rep 8, 12373 (2018). https://doi.org/10.1038/s41598-018-30590-8</td>
      <td>rGp,rGp,rGp,fCp,rGp,fCp,fUp,fCp,rAp,rAp,fUp,fUp,fUp,fUp,fUp,fUp,rAp,fUp,fUp,rGp,fCp,rAp,fUp,fUp,fUp,fUp,fUp,fCp,fUp,fUp,fUp,rGp,rAp,rGp,fCp,rGp,fCp,fCp,fCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>689</th>
      <td>Human Nonpancreatic Secretory Phospholipase A2 (Aptamer 15)</td>
      <td>2'-NH2-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/131.jpg" width="120" ></td>
      <td>Human Nonpancreatic Secretory Phospholipase A2  (hnps-PLA2)</td>
      <td>Small Organic</td>
      <td>1.7 nM (reported value)</td>
      <td>Tris-buffered saline (25 mM Tris, pH 7.4, 150 mM NaCl)    Tris potassium calcium buffer (25 mM Tris, 2.5 mM CaCl2, 100 mM KCl)</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>26999.39 g/mole</td>
      <td>62.75%</td>
      <td>513200 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>51</td>
      <td>Bridonneau, P., et al. "High-Affinity Aptamers Selectively Inhibit Human Nonpancreatic Secretory Phospholipase A2 (hnps-PLA2)." Journal of Medicinal Chemistry, 41(1998): 778-786.</td>
      <td>rGp,rGp,rGp,rAp,rAp,rAp,rAp,rGp,rAp,rGp,rAp,rGp,rGp,rGp,rGp,rGp,rAp,rAp,rGp,rGp,rAp,rGp,rAp,rGp,rAp,rGp,rGp,rGp,rGp,rGp,rAp,rGp,rAp,rAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>690</th>
      <td>Hepatocyte Growth Factor Receptor (CLN0003)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/496.png" width="120" ></td>
      <td>HGF-R / c-Met</td>
      <td>Protein</td>
      <td>0.091 nM (reported value)</td>
      <td>Binding Buffer: DPBS with 0.05% BSA</td>
      <td>37°C</td>
      <td>None</td>
      <td>The aptamer bound to c-Met positive cell lines GTL-16, MKN-45, and EBC-1 and not to the c-Met negative cell line Jurkat E6.1.</td>
      <td>26296.09 g/mole</td>
      <td>54.76%</td>
      <td>815800 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>84</td>
      <td>None</td>
      <td>dGp,dGp,dAp,dGp,dGp,dGp,dAp,dAp,dAp,dAp,dGp,dTp,dTp,dAp,dTp,dCp,dAp,dGp,dGp,dCp,dTp,dGp,dGp,dAp,dTp,dGp,dGp,dTp,dAp,dGp,dCp,dTp,dCp,dGp,dGp,dTp,dCp,dGp,dGp,dGp,dGp,dTp,dGp,dGp,dGp,dTp,dGp,dGp,dGp,dTp,dTp,dGp,dGp,dCp,dAp,dAp,dGp,dTp,dCp,dTp,dGp,dAp,dTp,dTp,dAp,dGp,dTp,dTp,dTp,dTp,dGp,dGp,dAp,dGp,dTp,dAp,dCp,dTp,dCp,dGp,dCp,dTp,dCp,dCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>691</th>
      <td>TV06</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7124.jpg" width="120" ></td>
      <td>STIP1 (TOV-21G, Ovarian Cancer)</td>
      <td>Protein</td>
      <td>nM (reported value)</td>
      <td>4.5 g/L glucose and 5 mM MgCl2 in Dulbecco’s phosphate-buffered saline (PBS) with CaCl2 and MgCl2 (Sigma), baker’s yeast tRNA</td>
      <td>4°C</td>
      <td>None</td>
      <td></td>
      <td>23311.1 g/mole</td>
      <td>50.00%</td>
      <td>708300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>84</td>
      <td>Van Simaeys, D., Turek, D., Champanhac, C., Vaizer, J., Sefah, K., Zhen, J., … Tan, W. (2014). Identification of Cell Membrane Protein Stress-Induced Phosphoprotein 1 as a Potential Ovarian Cancer Biomarker Using Aptamers Selected by Cell Systematic Evolution of Ligands by Exponential Enrichment. Analytical Chemistry, 86(9), 4521–4527. doi:10.1021/ac500466x</td>
      <td>dAp,dTp,dCp,dCp,dAp,dGp,dAp,dGp,dTp,dGp,dAp,dCp,dGp,dCp,dAp,dGp,dCp,dAp,dCp,dGp,dGp,dCp,dAp,dCp,dTp,dCp,dAp,dCp,dTp,dCp,dTp,dTp,dTp,dGp,dTp,dTp,dAp,dAp,dGp,dTp,dGp,dGp,dTp,dCp,dTp,dGp,dCp,dTp,dTp,dCp,dTp,dTp,dAp,dAp,dCp,dCp,dTp,dTp,dCp,dAp,dTp,dCp,dGp,dAp,dCp,dAp,dCp,dGp,dGp,dTp,dGp,dGp,dCp,dTp,dTp,dAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>692</th>
      <td>Peritoneal tumors of gastric cancer</td>
      <td>Peptide</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/252.png" width="120" ></td>
      <td>Peritoneal tumors of gastric cancer</td>
      <td>Tissue</td>
      <td>N/A nM (reported value)</td>
      <td>None</td>
      <td>N/A°C</td>
      <td>None</td>
      <td></td>
      <td>1252.56 g/mole</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>KLP motif showed homology to laminin-5, which binds to a3??1 integrin, suggesting that this might be the target of the aptamer.</td>
      <td>10</td>
      <td>Akita et al. "Identification of oligopeptides binding to peritoneal tumors of gastric cancer." Japanese Cancer Association, 97(2006): 1075-1081.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>693</th>
      <td>Okadaic Acid Aptamer (OA-34)</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/515.jpg" width="120" ></td>
      <td>Okadaic acid</td>
      <td>Small Organic</td>
      <td>77 nM (reported value)</td>
      <td>50 mM Tris, pH 7.5, 150 mM NaCl, 2 mM MgCl2</td>
      <td>°C</td>
      <td>None</td>
      <td></td>
      <td>18680.15 g/mole</td>
      <td>58.33%</td>
      <td>590200 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>180</td>
      <td>Eissa, S., Ng, A., Siaj, M., Tavares, A., and Zourob, M. "Selection and Identification of DNA Aptamers against Okadaic Acid for Biosensing Application." Analytical Chemistry 85.24 (2013): 11794-801.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>694</th>
      <td>CD4 (aptamer 12)</td>
      <td>2'-F-RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/369.png" width="120" ></td>
      <td>CD4 protein found on CD4+ T-Cells</td>
      <td>Protein</td>
      <td>0.5 nM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td></td>
      <td>27852.63 g/mole</td>
      <td>51.16%</td>
      <td>861400 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>89</td>
      <td>None</td>
      <td>rGp,rGp,rGp,rAp,rGp,rAp,fCp,rAp,rAp,rGp,rAp,rAp,fUp,rAp,rAp,rAp,fCp,rGp,fCp,fUp,fCp,rAp,rAp,rGp,fUp,rGp,rAp,fCp,rGp,fUp,fCp,fCp,fUp,rGp,rAp,fUp,fCp,rGp,rAp,fUp,fUp,rGp,fUp,rGp,fCp,rAp,fUp,fUp,fCp,rGp,rGp,fUp,rGp,fUp,rGp,rAp,fCp,rGp,rAp,fUp,fCp,fUp,fUp,fUp,fCp,rGp,rAp,fCp,rAp,rGp,rGp,rAp,rGp,rGp,fCp,fUp,fCp,rAp,fCp,rAp,rAp,fCp,rAp,rGp,rGp,fCp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>695</th>
      <td>Tobramycin (J6)</td>
      <td>RNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/412" width="120" ></td>
      <td>Tobramycin</td>
      <td>Small Organic</td>
      <td>2 nM (reported value)</td>
      <td>None</td>
      <td>RT°C</td>
      <td>None</td>
      <td></td>
      <td>35049.02 g/mole</td>
      <td>51.38%</td>
      <td>1078300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>109</td>
      <td>Wang, Yong, et al. "Specific binding of aminoglycoside antibiotics to RNA." Chemistry & Biology 2 (1995): 281-290</td>
      <td>rGp,rGp,rGp,rAp,rGp,rAp,rAp,rUp,rUp,rCp,rCp,rGp,rAp,rCp,rCp,rAp,rGp,rAp,rAp,rGp,rCp,rUp,rUp,rAp,rGp,rUp,rAp,rUp,rAp,rGp,rCp,rGp,rAp,rGp,rGp,rUp,rUp,rUp,rAp,rGp,rCp,rUp,rAp,rCp,rAp,rCp,rUp,rCp,rGp,rUp,rGp,rCp,rUp,rGp,rAp,rUp,rCp,rGp,rUp,rUp,rUp,rGp,rGp,rUp,rAp,rCp,rGp,rGp,rGp,rAp,rCp,rCp,rUp,rGp,rCp,rGp,rUp,rGp,rUp,rAp,rGp,rCp,rCp,rCp,rAp,rUp,rAp,rUp,rGp,rUp,rGp,rCp,rGp,rUp,rCp,rUp,rAp,rCp,rAp,rUp,rGp,rGp,rAp,rUp,rCp,rCp,rUp,rCp,rAp</td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>696</th>
      <td>U2</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/521.jpg" width="120" ></td>
      <td>U87-EGFRvIII</td>
      <td>Cells</td>
      <td>3.37 nM (reported value)</td>
      <td>0.45 g of glucose, 10 mg yeast tRNA, 5 mg salmon sperm DNA, 0.1 g BSA and 0.5 ml of 1 M MgCl2 in 100 ml of 1X phosphate-buffered saline [PBS]</td>
      <td>°C</td>
      <td>None</td>
      <td></td>
      <td>23483.29 g/mole</td>
      <td>47.37%</td>
      <td>717000 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>None</td>
      <td>228</td>
      <td>Wu, X. et al. "Cell-SELEX Aptamer for Highly Specific Radionuclide Molecular Imaging of Glioblastoma In Vivo." PLOS One 2014, 9:3.</td>
      <td></td>
      <td>NaN</td>
    </tr>
    <tr>
      <th>697</th>
      <td>Rd29C33-T7</td>
      <td>DNA</td>
      <td><img src="/Users/jiamingxu/Downloads/Aptahtmls/images/7127.jpg" width="120" ></td>
      <td>Melamine</td>
      <td>Small Organic</td>
      <td>0.92-0.98 nM (reported value)</td>
      <td>Cyanuric acid, Ammeline, Ammelide.</td>
      <td>59°C</td>
      <td>None</td>
      <td></td>
      <td>10546.8 g/mole</td>
      <td>64.71%</td>
      <td>317300 L/(mole·cm)</td>
      <td></td>
      <td></td>
      <td>Truncated from aptamer Rd29C33.</td>
      <td>38</td>
      <td>Gu, Chunmei et al. “Label-free fluorescence detection of melamine with a truncated aptamer.” The Analyst vol. 141,14 (2016): 4511-7. doi:10.1039/c6an00537c</td>
      <td>dGp,dCp,dAp,dCp,dAp,dCp,dCp,dGp,dAp,dTp,dGp,dGp,dCp,dGp,dGp,dTp,dCp,dCp,dTp,dGp,dTp,dTp,dTp,dAp,dGp,dGp,dGp,dGp,dGp,dTp,dGp,dTp,dGp,dCp</td>
      <td>NaN</td>
    </tr>
  </tbody>
</table>